<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Front Plant Sci</span> <span class="iso-abbrev" title="iso-abbrev">Front Plant Sci</span> <span class="publisher-id" title="publisher-id">Front. Plant Sci.</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Frontiers in Plant Science</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1664-462X</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Frontiers Media S.A.</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5504242">5504242</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3389/fpls.2017.01206">10.3389/fpls.2017.01206</a> </span> <div class="article-categories" title="article-categories"> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Science</span>: <span class="subject" title="subject">Original Research</span> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Antarctic Moss Multiprotein Bridging Factor 1c Overexpression in Arabidopsis Resulted in Enhanced Tolerance to Salt Stress </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alavilli</span> <span class="given-names" tagx="given-names" title="given-names">Hemasundar</span> </span> <a href="#aff1"> <sup>1</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/336454/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">Hyoungseok</span> </span> <a href="#aff2"> <sup>2</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/399648/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">Mira</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#aff2"> <sup>2</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/456181/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">Byeong-ha</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#fn001"> <sup>*</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/305134/overview" /> </span> </span> </div> <span class="citation_author_institution" id="aff1">[], <sup>1</sup> <span class="institution" title="institution">Department of Life Science, Sogang University</span> <span class="country" tagx="country" title="country">Seoul, South Korea</span> </span> <span class="citation_author_institution" id="aff2">[], <sup>2</sup> <span class="institution" title="institution">Division of Life Sciences, Korea Polar Research Institute</span> <span class="country" tagx="country" title="country">Incheon, South Korea</span> </span> <div class="author-notes" title="author-notes"> <div class="fn-type-edited-by" title="edited-by"> <p>Edited by: <i>Bernd Mueller-Roeber, University of Potsdam, Germany</i> </p> </div> <div class="fn-type-edited-by" title="edited-by"> <p>Reviewed by: <i>Dirk K. Hincha, Max Planck Institute of Molecular Plant Physiology (MPG), Germany; Narendra Singh Yadav, Ben-Gurion University of the Negev, Israel </i> </p> </div> <div class="corresp" title="corresp">
*Correspondence: <i>Byeong-ha Lee, <span class="email" tagx="email" title="email">byeongha@sogang.ac.kr</span> </i> </div> <div class="fn-type-other" title="other"> <p>This article was submitted to Plant Abiotic Stress, a section of the journal Frontiers in Plant Science </p> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-7-7</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2017</span> </span> <span class="volume" tagx="volume" title="volume">8</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">1206</span> <span class="history" title="history"> <span class="received" title="received">received: 2017-3-31</span> <span class="accepted" title="accepted">accepted: 2017-6-26</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Alavilli, Lee, Park and Lee</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2017</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) or licensor are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p> <i>Polytrichastrum alpinum</i> is one of the moss species that survives extreme conditions in the Antarctic. In order to explore the functional benefits of moss genetic resources, <i>P. alpinum</i> multiprotein-bridging factor 1c gene ( <i>PaMBF1c</i>) was isolated and characterized. The deduced amino acid sequence of PaMBF1c comprises of a multiprotein-bridging factor (MBF1) domain and a helix-turn-helix (HTH) domain. <i>PaMBF1c</i> expression was induced by different abiotic stresses in <i>P. alpinum</i>, implying its roles in stress responses. We overexpressed <i>PaMBF1c</i> in Arabidopsis and analyzed the resulting phenotypes in comparison with wild type and/or Arabidopsis <i>MBF1c</i> ( <i>AtMBF1c</i>) overexpressors. Overexpression of <i>PaMBF1c</i> in Arabidopsis resulted in enhanced tolerance to salt and osmotic stress, as well as to cold and heat stress. More specifically, enhanced salt tolerance was observed in <i>PaMBF1c</i> overexpressors in comparison to wild type but not clearly observable in <i>AtMBF1c</i> overexpressing lines. Thus, these results implicate the evolution of <i>PaMBF1c</i> under salt-enriched Antarctic soil. RNA-Seq profiling of NaCl-treated plants revealed that 10 salt-stress inducible genes were already up-regulated in <i>PaMBF1c</i> overexpressing plants even before NaCl treatment. Gene ontology enrichment analysis with salt up-regulated genes in each line uncovered that the terms lipid metabolic process, ion transport, and cellular amino acid biosynthetic process were significantly enriched in <i>PaMBF1c</i> overexpressors. Additionally, gene enrichment analysis with salt down-regulated genes in each line revealed that the enriched categories in wild type were not significantly overrepresented in <i>PaMBF1c</i> overexpressing lines. The up-regulation of several genes only in <i>PaMBF1c</i> overexpressing lines suggest that enhanced salt tolerance in <i>PaMBF1c</i>-OE might involve reactive oxygen species detoxification, maintenance of ATP homeostasis, and facilitation of Ca <sup>2+</sup> signaling. Interestingly, many salt down-regulated ribosome- and translation-related genes were not down-regulated in <i>PaMBF1c</i> overexpressing lines under salt stress. These differentially regulated genes by <i>PaMBF1c</i> overexpression could contribute to the enhanced tolerance in <i>PaMBF1c</i> overexpressing lines under salt stress. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
stress tolerance</div> <div class="kwd" title="kwd">
salt stress</div> <div class="kwd" title="kwd">
MBF1c</div> <div class="kwd" title="kwd">
RNA sequencing</div> <div class="kwd" title="kwd">
Antarctic moss</div> <div class="kwd" title="kwd"> <i>Polytrichastrum alpinum</i> </div> </div> <div class="counts" title="counts"> <span class="fig-count" title="fig-count">fig-count: </span> <span class="table-count" title="table-count">table-count: </span> <span class="equation-count" title="equation-count">equation-count: </span> <span class="ref-count" title="ref-count">ref-count: </span> <span class="page-count" title="page-count">page-count: </span> <span class="word-count" title="word-count">word-count: </span> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>In the Antarctic landscape, mosses constitute the dominant flora. They are capable of coping with multiple abiotic stress factors such as low temperatures, high radiation, high salts, strong winds and prolonged desiccation, and unpredictable cycles of freezing and thawing ( <a href="#B51">Turetsky et al., 2012</a>; <a href="#B56">Zuniga-Gonzalez et al., 2016</a>). <i>Polytrichastrum alpinum</i> (Hedw.) G.L.Sm., also known as alpine haircap moss, is distributed over a large area of arctic, sub-arctic and montane temperate regions ( <a href="#B7">Bhattarai et al., 2009</a>; <a href="#B6">Bell and Hyvonen, 2010</a>; <a href="#B52">Victoria et al., 2013</a>). Despite its strong stress tolerance, few reports till date have described the utilization of moss genetic resources in crop improvement. </p> <p>Multiprotein bridging factor 1 (MBF1) was first purified from posterior silk gland extracts of <i>Bombyx mori</i> ( <a href="#B26">Li et al., 1994</a>). MBF1 proteins are highly conserved from archaea to humans. Each of the three Arabidopsis <i>MBF1</i> paralogs could restore <i>MBF1</i> functions in <i>mbf1</i> deficient yeast ( <i>mbf1</i>Δ) ( <a href="#B49">Tsuda et al., 2004</a>). In addition, the <i>mbf1</i>Δ yeast were also rescued by expressing human or silkworm <i>MBF1</i> ( <a href="#B45">Takemaru et al., 1997</a>), suggesting conservation of <i>MBF1</i> gene function. MBF1 proteins function as a non-DNA binding transcriptional co-activators ( <a href="#B2">Aravind and Koonin, 1999</a>; <a href="#B21">Kabe et al., 1999</a>) that are involved in diverse physiological and developmental processes ( <a href="#B8">Brendel et al., 2002</a>; <a href="#B29">Liu et al., 2003</a>). </p> <p>In plants, <i>MBF1</i> genes are known to be involved in abiotic and biotic stress tolerance. <i>MBF1</i> expression in <i>Solanum tuberosum</i> ( <i>StMBF1</i>) is induced by wounding ( <a href="#B14">Godoy et al., 2001</a>) and pathogen attack in tubers ( <a href="#B4">Arce et al., 2006</a>). Expression levels of <i>Vitis vinifera MBF1</i> ( <i>VvMBF1</i>) were increased in leaf tissues in response to ABA and dehydration stress, and overexpression of <i>VvMBF1</i> in Arabidopsis resulted in enhanced drought stress tolerance ( <a href="#B55">Yan et al., 2014</a>); also, ectopic expression of <i>Triticum aestivum MBF1c</i> ( <i>TaMBF1c</i>) in rice improved its thermal tolerance under vegetative and reproductive stages ( <a href="#B39">Qin et al., 2015</a>). However, <i>MBF1</i> overexpression does not always appear to result in improved stress tolerance in plants. When the <i>AtMBF1a</i> and <i>AtMBF1b</i> homolog, <i>Capsicum annum MBF1</i> ( <i>CaMBF1</i>) was overexpressed in Arabidopsis, the resulting transgenic plants produced large leaves but displayed reduced tolerance to abiotic stress ( <a href="#B16">Guo et al., 2014</a>). </p> <p>The model plant <i>Arabidopsis thaliana</i> has three <i>MBF1</i> paralogs – <i>AtMBF1a, AtMBF1b</i>, and <i>AtMBF1c</i>. Phylogenetic analysis suggests that <i>AtMBF1a</i> and <i>AtMBF1b</i> are more closely related, while <i>AtMBF1c</i> belongs to a separate group ( <a href="#B50">Tsuda and Yamazaki, 2004</a>). Overexpression of the <i>AtMBF1a</i> gene enhanced salt tolerance, glucose insensitivity and fungal resistance in transgenic Arabidopsis plants ( <a href="#B22">Kim et al., 2007</a>). Compared to the other two Arabidopsis <i>MBF1</i>s, <i>AtMBF1c</i> was highly induced in response to pathogen infection, dehydration, high salt, methyl viologen, hydrogen peroxide (H <sub>2</sub>O <sub>2</sub>), and heat treatment ( <a href="#B40">Rizhsky et al., 2002</a>; <a href="#B50">Tsuda and Yamazaki, 2004</a>; <a href="#B42">Suzuki et al., 2005</a>). The Arabidopsis <i>mbf1</i> triple knock-down mutant ( <i>mbf1 abc-</i>) was hypersensitive to oxidative and osmotic stress agents such as methyl viologen, H <sub>2</sub>O <sub>2</sub> and high concentrations of sorbitol. These <i>mbf1 abc-</i> mutant stress-sensitive phenotypes were either partially or fully restored by <i>AtMBF1c</i> cDNA overexpression, implicating the predominance of <i>AtMBF1c</i> gene function in stress tolerance ( <a href="#B3">Arce et al., 2010</a>). Overexpression of the <i>AtMBF1c</i> gene in Arabidopsis also enhanced tolerance to bacterial infection as well as to heat and osmotic stress ( <a href="#B42">Suzuki et al., 2005</a>). In fact, functionality of the <i>AtMBF1c</i> gene is well established in plant heat stress response; it controls heat stress-related gene expression to improve basal heat tolerance during heat stress ( <a href="#B43">Suzuki et al., 2011</a>). </p> <p>In order to investigate the functional benefits of moss genetic resources, we have isolated the stress responsive <i>PaMBF1c</i> gene from a <i>P. alpinum</i> cDNA library and examined its functions under salt and other abiotic stress by overexpression and RNA-Seq profiling. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="conserveddomainanalysisandphylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Conserved Domain Analysis and Phylogenetic Analysis</div> <p>The MBF protein homologs sequences from diverse species were retrieved from GenBank database and Phytozome database <sup> <a href="#fn01">1</a> </sup> ( <a href="#B15">Goodstein et al., 2012</a>) with BlastP using PaMBF1c amino acid sequences as a query. Multiple sequence alignment was performed using ClustalW ( <a href="#B46">Thompson et al., 1994</a>; <a href="#B23">Kumar et al., 2016</a>). Conserved domains in each sequence were identified using NCBI conserved domain finder <sup> <a href="#fn02">2</a> </sup>. The phylogenetic tree was constructed with MEGA7 software ( <a href="#B46">Thompson et al., 1994</a>; <a href="#B23">Kumar et al., 2016</a>) from the data sets by using the maximum likelihood method based on the JTT matrix-based model. The initial tree for the heuristic search was obtained by applying the neighbor-joining method to a matrix of pair-wise distances, estimated using a JTT model. Supports for internal branches were tested by bootstrap analyses of 1000 replications. </p> </div> <div class="plantmaterialandgrowthconditions" title="sec"> <div class="title" tagx="title" title="title">
Plant Material and Growth Conditions</div> <p> <i>Polytrichastrum alpinum</i> (Hedw.) G.L.Sm. samples were collected from the King Sejong Antarctic station (62°14′29′′S; 58°44′18′′W), at the Barton Peninsula of King George Island. <i>In vitro</i> culture was carried out on BCD solid media ( <a href="#B5">Ashton and Cove, 1977</a>), which are successfully being used in a model moss <i>Physcomitrella patens</i> culture ( <a href="#B13">Du et al., 2016</a>), in a growth room operating at 23°C with 16-h light/8-h dark light cycle with a light intensity of 150 μmol m <sup>-2</sup>S <sup>-1</sup>. </p> <p>The plates were placed in a growth chamber that operates at 22 ± 1°C and 70% relative humidity with continuous light (80–100 μmol m <sup>-2</sup>S <sup>-1</sup>). The seeds planted on soil (Sungro propagation mixture, Canada) were maintained in a controlled growth chamber that operates with 16-h light/8-h dark cycle. All seeds were stratified for at least 2 days at 4°C before being transferred to the growth chamber. When necessary, seedlings raised on MS were transferred to soil pots in a growth chamber at 22 ± 1°C and 50–70% relative humidity programmed with 16-h light/8-h dark light cycle (the light intensity of 80–100 μmol m <sup>-2</sup>S <sup>-1</sup>). </p> </div> <div class="cloningofpambf1cgeneandgenerationoftransgenicarabidopsisplants" title="sec"> <div class="title" tagx="title" title="title">
Cloning of <i>PaMBF1c</i> Gene and Generation of Transgenic Arabidopsis Plants </div> <p> <i>Polytrichastrum alpinum</i> first strand cDNA was synthesized from total RNA with MMLV reverse transcriptase (Enzynomics, South Korea) and oligo(dT) primer. Full-length <i>PaMBF1c</i> coding sequences were amplified using gene specific primers (Supplementary Table <a href="#SM8">S1</a>). The resulting PCR products were cloned into the pENTR/D/TOPO entry vector (Invitrogen, United States) and sequenced using M-13 primers. After sequencing confirmation, the entry plasmids were LR-recombined with the gateway compatible binary destination vector, pMDC32 ( <a href="#B12">Curtis and Grossniklaus, 2003</a>), which resulted in the overexpression construct, pMDC32-35s: <i>PaMBF1c.</i> The construct was then transferred into <i>Agrobacterium tumefaciens</i> strain GV3101 via electroporation. Agrobacterium-mediated transformation of <i>Arabidopsis thaliana</i> (Col-0) with the <i>PaMBF1c</i> gene was performed through the floral dipping method ( <a href="#B11">Clough and Bent, 1998</a>). For selection of <i>PaMBF1c</i>-OE lines, T1 seeds were harvested from floral-dipped plants and selected on MS/agar plates containing hygromycin (25 μg/mL). Using hygromycin selection and transgene detection, homozygotes for the <i>PaMBF1c</i> transgene of two independent events were selected at T2 generation after segregation analysis at T3 generation ( <i>PaMBF1c-</i>OE1 and <i>PaMBF1c-</i>OE2). Hygromycin resistance from each line at T2 generation was segregated to a 3:1 ratio of resistant-to-sensitive, indicating a single locus of insertion. </p> </div> <div class="stresstreatmentandanalysisofstresstoleranceinplants" title="sec"> <div class="title" tagx="title" title="title">
Stress Treatment and Analysis of Stress Tolerance in Plants</div> <p> <i>Polytrichastrum alpinum</i> gametophores were transferred onto fresh agar plates of BCD medium containing mannitol (150, 300 mM) or NaCl (75, 150 mM), respectively, and incubated at 15°C for 6 h. Heat treatment was carried out by transferring colonies grown at 15°C to chambers of 37 or 42°C for 2 h. </p> <p>For <i>Arabidopsis thaliana</i> germination experiments, at least 100–120 seeds of each genotype were planted in a media with or without stress agents and germination trends were recorded from the day after planting until all the seeds in control plates were germinated. The number of germinated seeds was expressed as the percentage of total number of seeds plated. For root growth experiments, the seeds were planted in vertical MS plates with 0.6% gelrite and allowed to grow for 4 days. Seedlings with a 1–1.5 cm long root were transferred onto a second MS vertical plate supplemented with different concentrations of salts or stress agents. For survival index under NaCl and LiCl, the shoot phenotypes of seedlings were observed every day after transferring seedlings from the control medium. Hypocotyl elongation under heat stress was assessed as previously described ( <a href="#B19">Hong and Vierling, 2000</a>). The malondialdehyde (MDA), chlorophyll, and anthocyanin content in control and stress treated plants were also determined as previously described ( <a href="#B17">Heath and Packer, 1968</a>; <a href="#B28">Lichtenthaler, 1987</a>; <a href="#B37">Neff and Chory, 1998</a>). </p> </div> <div class="measurementofelectrolyteleakage" title="sec"> <div class="title" tagx="title" title="title">
Measurement of Electrolyte Leakage</div> <p>Fully expanded fourth or fifth rosette leaf with petiole from 3-week-old seedlings was placed in a 15 mL test tube containing 100 μL of deionized water and placed at 0°C in a refrigerated circulating bath (Gaon Science Instrument, South Korea). The remaining steps were carried out according to methods described previously ( <a href="#B25">Lee et al., 2002</a>). Conductivity was measured using a conductivity meter (Control Company, United States) before and after autoclaving, and electrolyte leakage was expressed as a percentage of conductivity before autoclaving over conductivity after autoclaving. </p> </div> <div class="geneexpressionanalysis" title="sec"> <div class="title" tagx="title" title="title">
Gene Expression Analysis</div> <p>Total RNAs were isolated from plant materials using an RNA purification kit (NanoHelix, South Korea) and treated with RNase free DNase I (Qiagen, Germany). For semi-quantitative one step RT-PCR, total RNAs were added to the Hi-pure one step RT-PCR master mix (Genepole, South Korea) and the PCR reaction was performed according to manufacturer instructions. <i>Protein phosphatase 2A</i> (At1g13320) was used as an internal control. Quantitative real-time PCR was performed using KAPA SYBR FAST qPCR kit (Kapa Biosystems, United States) according to manufacturer instructions and run on the ABI 7500 system (Applied Biosystems, United States). The relative expression of <i>PaMBF1c</i> in <i>P. alpinum</i> was calculated by normalizing expression values with those of the housekeeping gene <i>PaTubulin</i>. For the Arabidopsis qRT experiment, <i>AtClathrin</i> gene (At4g24550) was used as an internal control. The ΔΔCt method was adapted to calculate relative gene expression ( <a href="#B31">Livak and Schmittgen, 2001</a>). The primer pairs used for amplification are shown in Supplementary Table <a href="#SM8">S1</a>. </p> </div> <div class="rnasequencing" title="sec"> <div class="title" tagx="title" title="title">
RNA Sequencing</div> <p>RNA-sequencing was carried out using total RNA from 2-week-old WT, two independent <i>PaMBF1c</i>-OE lines (two events) and two independent <i>AtMBF1c</i>-OE lines (two events) under normal and salt stressed conditions (300 mM NaCl, 6 h). We used this salt stress condition because 300 mM NaCl treatment for 4–6 h in Arabidopsis brought about maximum expression of a very well characterized stress inducible gene, <i>RD29A</i> ( <a href="#B20">Ishitani et al., 1997</a>). Total RNAs were extracted from at least 25–30 plants of each genotype and treatment using Pure Helix total RNA purification kit (NanoHelix, South Korea) and RNase free DNase I (Qiagen, Germany). Three different biological replicates were prepared. The integrity and concentration of RNA was determined using Bioanalyzer (RIN &amp;gt; 6) and Qubit <sup>®</sup> RNA BR assay kit (Life Technologies, United States). To construct the sequencing library, 1.5 μg of total RNA of each sample was used as input for the TruSeq RNA sample prep kit v2 (Illumina, United States). The libraries were validated and quantified by Bioanalyzer and qPCR quantification method, and then multiplexed with equal ratio and loaded on the flowcell of the Illumina MiSeq Reagent Kit v3 (150 cycles). Afterward, sequencing was performed on a MiSeq Sequencer system (Illumina, United States) and total 3 Gb (40M paired end reads) of sequencing data was generated (Q <sub>30</sub> &amp;gt; 98%). The RNA-Seq data have been deposited to the Sequence Read Archive (SRA <sup> <a href="#fn03">3</a> </sup>) under accession number <a href="SRP110226">SRP110226</a>. </p> </div> <div class="transcriptomicdataanalysis" title="sec"> <div class="title" tagx="title" title="title">
Transcriptomic Data Analysis</div> <p>Basically all analyses were performed using the CLC Genomics Workbench v7 module (Qiagen, Germany). After quality and adapter trimming, raw reads were mapped to the Arabidopsis gene model annotation file of the Gene Ontology Consortium (released at August 8th, 2014). The expression values were measured in FPKM (Fragments per Kilobase of exon model per Million mapped reads) normalized values in gene level ( <a href="#B35">Mortazavi et al., 2008</a>). For statistical analysis, <i>t</i>-test and Baggerley’s test were performed using original and normalized read counts, and several relevant values for analysis ( <i>p</i>-value, FDR corrected <i>p</i>-value, test-statistic, etc.) were calculated using the "multi-group comparison" option of the program. Through statistical analysis, differentially expressed genes were determined from a cutoff value ( <i>p</i>-value &amp;lt; 0.05, corrected <i>p</i>-value of FDR &amp;lt; 0.05 and absolute value of fold change &amp;gt; 1.5) from pairwise comparison of normalized FPKM values between samples. Gene ontology (GO) enrichment analysis was performed using the PANTHER overrepresentation test (PANTHER version 10 <sup> <a href="#fn04">4</a> </sup>) ( <a href="#B33">Mi et al., 2013</a>, <a href="#B34">2016</a>). <i>Arabidopsis thaliana</i> was selected as a reference organism with default settings and Bonferroni correction for multiple testing was used ( <i>p</i> &amp;lt; 0.05). </p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical Analysis</div> <p>All statistical comparisons between variants were determined by ANOVA (analysis of variance) and least significant differences (LSD) between variants were calculated using Statistix 8.1 computation software. Statistically significant mean values were denoted as <sup>∗</sup> ( <i>p</i>-value ≤ 0.05). </p> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="sequenceandphylogeneticanalysisofpambf1c" title="sec"> <div class="title" tagx="title" title="title">
Sequence and Phylogenetic Analysis of PaMBF1c</div> <p>The <i>Polytrichastrum alpinum</i> DNA sequence encoding multi-protein bridging factor 1c (MBF1c) protein was retrieved from our unpublished transcriptome data, based on its sequence homology with <i>AtMBF1c</i>. The gene was designated <i>PaMBF1c</i> and its sequence was submitted to GenBank (Accession number, <a href="KM978992">KM978992</a>). The total length of the 432 nucleotide coding sequence comprise of 143 deduced amino acids with an estimated molecular mass of 15.7 kDa and an isoelectric point of 10.08 predicted by ExPasy bioinformatics tools for protein structure analysis <sup> <a href="#fn05">5</a> </sup>. It contains two distinctive conserved domains – an MBF1 domain at the N-terminal region and a helix-turn-helix (HTH) domain at the C-terminal region ( <b>Figure <a href="#F1">1A</a> </b>). <i>PaMBF1c</i> shares 86% identity with its homolog from <i>Physcomitrella patens</i> (XP_001771731), 64% with <i>Medicago truncatula</i> (AES76734), 66% with <i>Arabidopsis thaliana</i> (NP_189093), 59% with <i>Triticum aestivum</i> (ACU43593), and 59% identity with <i>Oryza sativa</i> (NP_001057974). Sequence alignment analysis of PaMBF1c also suggests the presence of glutamic acid (E) at the 115th amino acid position in the HTH domain ( <b>Figure <a href="#F1">1A</a> </b>). This particular amino acid is reportedly conserved across the plant species and is essential for the binding to TATA-box binding protein ( <a href="#B30">Liu et al., 2007</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">FIGURE 1</span> </a> </div>   <p>Sequence alignment and phylogenetic analysis of PaMBF1c. <b>(A)</b> Amino acid sequences of PaMBF1c protein (AJG41867) and homologs from <i>Physcomitrella patens</i> (XP_001771731), <i>Oryza sativa</i> (NP_001057974), <i>Medicago truncatula</i> (AES76734), <i>Triticum aestivum</i> (ACU43593), and <i>Arabidopsis thaliana</i> (NP_189093) were used for amino acid sequence alignment. The dotted lines below the sequence alignment indicate multiprotein bridging factor 1 (MBF1) domain and helix-turn-helix (HTH) domain. In the HTH domain, the amino acid residue glutamic acid (E) (arrow head) is conserved among all plant MBF1 proteins. <b>(B)</b> Phylogenetic tree of MBF1 proteins from diverse species. MBF1 proteins from mosses ( <i>Polytrichastrum alpinum</i>; Phpat, <i>Physcomitrella patens</i>; Sphfalx, <i>Sphagnum fallax</i>; Mapoly, <i>Marchantia polymorpha</i>), Lycophyte (Selmo, <i>Selaginella moellendorffii</i>), algae (Chlre, <i>Chlamydomonas reinhardtii</i>; Cocsu, <i>Coccomyxa subellipsoidea</i>; Vocar, <i>Volvox carteri</i>), monocots (Zeama, <i>Zea mays</i>; Bradi, <i>Brachypodium distachyon</i>; Orysa, <i>Oryza sativa</i>; Sobic, <i>Sorghum bicolor</i>), and dicots (Arath, <i>Arabidopsis thaliana</i>; Vitvi, <i>Vitis vinifera</i>; Solyc, <i>Solanum lycopersicum</i>; Potri, <i>Populus trichocarpa</i>) were included. The phylogenetic tree was constructed using the Neighbor-Joining method. </p>   </div> <p>The PaMBF1c protein sequence was queried against proteome datasets of various plants and chlorophytes in Phytozome <sup> <a href="#fn06">6</a> </sup> to better understand the phylogenetic relations of MBF1 family proteins in plants. The phylogenetic analysis demonstrated a clear divergence of MBF1c proteins from other MBF1 family proteins (MBF1a/b) ( <b>Figure <a href="#F1">1B</a> </b>). Interestingly, algae have only one MBF1 gene while most land plants contain at least two types MBF1 genes ( <b>Figure <a href="#F1">1B</a> </b>), suggesting a gene duplication event early in the evolution of land plant. Within the MBF1c clade, MBF1c orthologs of all non-vascular plant species were clustered together away from those of vascular plant ( <b>Figure <a href="#F1">1B</a> </b>) implying MBF1c differentiation among land plants. </p> </div> <div class="expressionofpambf1cgeneinp.alpinumunderabioticstress" title="sec"> <div class="title" tagx="title" title="title">
Expression of <i>PaMBF1c</i> Gene in <i>P. alpinum</i> under Abiotic Stress </div> <p>A number of <i>MBF1</i> group genes in various species were reported to be differentially induced by various abiotic stresses ( <a href="#B40">Rizhsky et al., 2002</a>; <a href="#B50">Tsuda and Yamazaki, 2004</a>; <a href="#B22">Kim et al., 2007</a>). We examined <i>PaMBF1c</i> transcript abundance under salt, osmotic, and heat stress conditions in <i>P. alpinum</i>. To this end, quantitative real-time PCR (qRT-PCR) analysis was carried out using RNA isolated from <i>P. alpinum</i> gametophores treated with NaCl (75 mM or 150 mM for 6 h), mannitol (150 or 300 mM for 6 h), or high temperature (37 or 42°C for 2 h). The results revealed that <i>PaMBF1c</i> transcript levels were increased in response to various abiotic stress treatments ( <b>Figure <a href="#F2">2</a> </b>); thus, we concluded that <i>PaMBF1c</i> is a heat, salt, and osmotic stress-responsive gene in <i>P. alpinum</i>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">FIGURE 2</span> </a> </div>   <p>Expression of <i>PaMBF1c</i> in <i>P. alpinum</i> under various abiotic stress conditions. The <i>PaMBF1c</i> expression levels were measured by quantitative real-time PCR with total RNA from <i>P. alpinum</i> gametophores under osmotic stress (150 mM or 300 mM mannitol for 6 h), salt stress (75 or 150 mM NaCl for 6 h) or heat stress (37 or 42°C for 2 h). The <i>P. alpinum</i> tubulin gene was used as an internal control for normalization. The expression level of <i>PaMBF1c</i> grown on normal BCD was used as a control (calibrator for quantification) and was assumed as 1. Error bars represents standard deviation of means ( <i>n</i> = 3). Asterisks indicate statistical significance in LSD test ( <i>p</i> &amp;lt; 0.05). </p>   </div> </div> <div class="generationandgrowthphenotypeofpambf1coverexpressinglines" title="sec"> <div class="title" tagx="title" title="title">
Generation and Growth Phenotype of <i>PaMBF1c</i> Overexpressing Lines </div> <p>To further investigate the functional roles of <i>PaMBF1c</i> in plants, we generated transgenic lines overexpressing the <i>PaMBF1c</i> gene under control of the 35S cauliflower mosaic virus promoter (35S: <i>PaMBF1c</i>). Using hygromycin resistance selection and the presence of the 35S: <i>PaMBF1c</i> transgene, we selected stable homozygous transgenic lines for 35S: <i>PaMBF1c</i> in the T4 generation (hereafter referred to as <i>PaMBF1c</i>-OE lines, where OE stands for overexpressing). Semi-quantitative RT-PCR analysis for <i>PaMBF1c</i> transcripts indeed confirmed that <i>PaMBF1c</i>-OE lines overexpressed <i>PaMBF1c</i>, as <i>PaMBF1c</i> transcripts were highly accumulated in transgenic plants ( <b>Figure <a href="#F3">3A</a> </b>). With confirmed lines, we examined the growth and development of <i>PaMBF1c</i>-OE plants. Two-week old <i>PaMBF1c</i>-OE plants displayed better growth and development than WT plants under normal growth conditions. Fresh weight of 2-week-old transgenic plants was about 16–17 mg/seedling while that of WT was about 11 mg/seedling ( <b>Figures <a href="#F3">3B,C</a> </b>). In addition, <i>PaMBF1c</i>-OE lines bolted 1–2 days earlier than did WT ( <b>Figures <a href="#F3">3D,E</a> </b>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">FIGURE 3</span> </a> </div>   <p>Growth and development of <i>PaMBF1c-</i>OE lines. <b>(A)</b> Higher accumulation of transgene transcripts in <i>PaMBF1c</i>-OE lines were confirmed by semi-quantitative RT-PCR. <b>(B)</b> The size of <i>PaMBF1c</i>-OE lines was bigger than WT. Pictures were taken 12 days after germination. <b>(C)</b> Measurement of fresh weights per seedling revealed a better growth of <i>PaMBF1c</i>-OE lines in comparison to WT. <b>(D,E)</b> <i>PaMBF1c</i>-OE lines bolted earlier than WT. Error bars represents standard deviation of means ( <i>n</i> = 20). Asterisks indicate statistical significance in LSD test ( <i>p</i> &amp;lt; 0.05). </p>   </div> </div> <div class="evaluationofpambf1coverexpressinglinesundervariousabioticstressconditions" title="sec"> <div class="title" tagx="title" title="title">
Evaluation of <i>PaMBF1c</i> Overexpressing Lines under Various Abiotic Stress Conditions </div> <p>To study the responses of <i>PaMBF1c</i>-OE plants to abiotic stress conditions, two <i>PaMBF1c</i>-OE lines from two independent transgenic events were subjected to different stress treatments. Germination of each line was tested on MS medium containing different kinds of stress agents: salt stress (200 mM NaCl), ionic stress (15 mM LiCl) or osmotic stress (200 mM mannitol). On the control medium, all lines germinated with percentages of 95–100% after 2–3 days of planting ( <b>Supplementary Figure <a href="#SM1">S1</a> </b>). On mediums supplemented with 200 mM NaCl, the germination ratio of WT was decreased to 35.8%, whereas <i>PaMBF1c</i>-OE lines displayed 72.9–86.4% germination ratios ( <b>Figure <a href="#F4">4A</a> </b>). Similarly, <i>PaMBF1c</i>-OE lines displayed 94–94.5% germination ratios on 15 mM LiCl MS medium, whereas WT germination ratio was reduced to 62.2% ( <b>Figure <a href="#F4">4A</a> </b>). In the medium containing 200 mM mannitol, <i>PaMBF1c</i>-OE lines displayed slightly better germination than in WT ( <b>Figure <a href="#F4">4A</a> </b>). Taken together, the results showed that <i>PaMBF1c</i>-OE lines germinate better than WT under salt, ionic, and osmotic stress. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">FIGURE 4</span> </a> </div>   <p>Evaluation of <i>PaMBF1c</i>-OE lines under different abiotic stress conditions. <b>(A)</b> <i>PaMBF1c</i>-OE lines germinated better than did WT under salt stress (200 mM NaCl), ionic stress (15 mM LiCl), and osmotic stress (200 mM mannitol). The number of germinated seeds was expressed as a percentage of total number of seeds planted ( <i>n</i> ≥ 100). Radicle emergence was considered germination and the germination was scored after 4 days of planting. <b>(B)</b> Roots of <i>PaMBF1c</i>-OE lines elongated longer than that of WT under salt stress (150 mM NaCl), ionic stress (15 mM LiCl), and osmotic stress (200 mM mannitol). Root growth was scored after 6 days of transfer of seedlings from normal medium to stress medium. Root elongation of seedlings under stress conditions was expressed as a percentage of each stress control grown on normal MS medium after 6 days of transfer ( <i>n</i> ≥ 10). <b>(C)</b> <i>PaMBF1c</i>-OE lines survived better than WT under salt stress (200 mM NaCl), ionic stress (20 mM LiCl) and heat stress (45°C for 60 min) ( <i>n</i> ≥ 15). Error bars represents standard deviation of the mean values of three independent experiments. Asterisks indicate statistical significance in LSD test ( <i>p</i> &amp;lt; 0.05). </p>   </div> <p>Root growth is affected by various stress conditions and is often considered an index for stress sensitivity ( <a href="#B24">Lee et al., 2006</a>). Thus, we examined levels of stress tolerance by analyzing root growth of <i>PaMBF1c</i>-OE lines. Similarly sized seedlings (3- to 4-day-old) grown under normal condition were transferred to MS medium for salt stress (150 mM NaCl), ionic stress (15 mM LiCl), or osmotic stress (200 mM mannitol); and root elongation was measured 6 days after transfer. All lines showed similar root lengths in the control MS media ( <b>Supplementary Figure <a href="#SM2">S2</a> </b>); however, relative root elongation in <i>PaMBF1c</i>-OE lines was significantly greater in the presence of 150 mM NaCl (48–48.7% vs. 34.2%), 200 mM mannitol (51.5–54.8% vs. 38.1%), or 15 mM LiCl (52.3–53.3% vs. 37%) when compared to WT ( <b>Figure <a href="#F4">4B</a> </b>). Additionally, seedling survival was examined under stress conditions. Three- to four-day-old seedlings grown under normal conditions were transferred to MS medium supplemented with 200 mM NaCl for salt stress or 20 mM LiCl for ionic stress. For heat stress, 8-day-old seedlings grown on MS plates were heat treated at 45°C ( <b>Figure <a href="#F4">4C</a> </b> and <b>Supplementary Figures <a href="#SM3">S3A,B</a> </b>). The <i>PaMBF1c</i>-OE lines showed significantly better survival than did WT under the presence of 200 mM NaCl (41–69% vs. 16.6%), 20 mM LiCl (73–86% vs. 40%), or heat (53–62% vs. 15%) ( <b>Figure <a href="#F4">4C</a> </b> and <b>Supplementary Figures <a href="#SM3">S3C–E</a> </b>). We also investigated responses to cold stress in <i>PaMBF1c</i>-OE lines by measuring ion leakage after freezing stress treatment. At -6°C, both WT and <i>PaMBF1c</i>-OE showed similar ion leakage levels ( <b>Supplementary Figure <a href="#SM4">S4</a> </b>); however, <i>PaMBF1c</i>-OE lines showed lower ion leakage at -9 and -12°C than WT, suggesting enhanced freezing tolerance in <i>PaMBF1c</i>-OE lines ( <b>Supplementary Figure <a href="#SM4">S4</a> </b>). </p> </div> <div class="examinationofsaltstresstoleranceinpambf1coverexpressinglines" title="sec"> <div class="title" tagx="title" title="title">
Examination of Salt Stress Tolerance in <i>PaMBF1c</i> Overexpressing Lines </div> <p>We examined in detail the tolerance phenotypes of <i>PaMBF1c</i>-OE under salt stress by analyzing chlorophyll content, lipid peroxidation level, and anthocyanin accumulation. Chlorophyll (Chl) degradation is among the manifestations caused by salt and osmotic stress ( <a href="#B9">Claeys et al., 2014</a>; <a href="#B1">Alavilli et al., 2016</a>). To further examine the effect of salt stress, we measured Chl content in WT and <i>PaMBF1c-</i>OE lines with or without salt stress. No notable differences were observed between WT and <i>PaMBF1c</i>-OE grown under control condition ( <b>Figures <a href="#F5">5A,B</a> </b>); however, stress treatment with 75 mM or 150 mM NaCl showed that <i>PaMBF1c</i>-OE lines maintained higher Chl content than that of WT, as also evidenced by visual phenotypes (i.e., leaf bleaching) ( <b>Figures <a href="#F5">5A,B</a> </b>). Abiotic stresses usually lead to lipid peroxidation of cell membranes which can cause irreversible damage to its functionality. MDA is considered an indicator of lipid peroxidation level ( <a href="#B17">Heath and Packer, 1968</a>). After salt stress (120 mM NaCl), the MDA content of <i>PaMBF1c</i>-OE lines was significantly lower in than that of WT, which suggests lower lipid peroxidation in <i>PaMBF1c</i>-OE ( <b>Figure <a href="#F5">5C</a> </b>). Anthocyanin accumulation is known to be associated with improved drought and salt stress tolerance ( <a href="#B38">Oh et al., 2011</a>; <a href="#B36">Nakabayashi et al., 2014</a>). Accordingly, 10-day-old <i>PaMBF1c</i>-OE lines grown under salt stress medium (150 mM NaCl) exhibited higher anthocyanin accumulation than WT ( <b>Figure <a href="#F5">5D</a> </b> and <b>Supplementary Figure <a href="#SM5">S5</a> </b>). Taken all together, these results clearly demonstrated the higher salt-stress tolerance of <i>PaMBF1c</i>-OE lines than WT. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">FIGURE 5</span> </a> </div>   <p>Chlorophyll, malondialdehyde (MDA), and anthocyanin levels in <i>PaMBF1c</i>-OE lines. <b>(A,B)</b> <i>PaMBF1c</i>-OE lines retained more chlorophyll content than WT at 75 and 150 mM NaCl. <b>(C)</b> <i>PaMBF1c</i>-OE lines showed lower MDA level than WT. MDA levels were examined with 21-day-old seedlings grown on MS plates with 0 or 120 mM NaCl. <b>(D)</b> <i>PaMBF1c</i>-OE lines accumulated higher amount of anthocyanin than WT. Anthocyanin content was measured with 10-day-old seedlings grown on MS plates with 0 or 150 mM NaCl. Error bars represents standard deviation of mean values of at least three independent experiments ( <i>n</i> = 25 seedlings per each treatment). Asterisks indicate statistical significance in LSD test ( <i>p</i> &amp;lt; 0.05). </p>   </div> </div> <div class="comparisonofstresstolerancebetweenpambf1candatmbf1coverexpressinglines" title="sec"> <div class="title" tagx="title" title="title">
Comparison of Stress Tolerance between <i>PaMBF1c</i> and <i>AtMBF1c</i> Overexpressing Lines </div> <p> <i>AtMBF1c</i> overexpression in Arabidopsis resulted in improved heat and osmotic stress tolerance ( <a href="#B42">Suzuki et al., 2005</a>). <i>AtMBF1c</i>-OE lines appeared to exhibit tolerance at only low levels of salt stress (50 mM NaCl) when compared to WT ( <a href="#B42">Suzuki et al., 2005</a>). Thus, we compared the stress tolerance phenotypes of our <i>PaMBF1c</i>-OE lines and the <i>AtMBF1c</i>-OE lines under our experimental conditions. We first assessed heat tolerance with two independent lines for each overexpressor. The hypocotyl elongation assay makes use of the characteristic heat inhibition of hypocotyl elongation ( <a href="#B19">Hong and Vierling, 2000</a>). <i>PaMBF1c</i>- <i>OE</i> lines showed higher hypocotyl elongation than did WT at both 41 and 43°C ( <b>Figure <a href="#F6">6A</a> </b>), suggesting hyposensitivity to heat stress in <i>PaMBF1c</i>-OE. Similarly, <i>AtMBF1c</i>-OE lines displayed higher hypocotyl elongation when compared to WT in agreement with previous reports ( <a href="#B42">Suzuki et al., 2005</a>). Hypocotyl lengths of both OE lines were comparable after heat treatment suggesting that both <i>PaMBF1c</i> and <i>AtMBF1c</i> have similar levels of increased heat tolerance ( <b>Figure <a href="#F6">6A</a> </b>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">FIGURE 6</span> </a> </div>   <p>Comparison of stress tolerance between <i>PaMBF1c</i>-OE lines and <i>AtMBF1c</i>-OE lines under various abiotic stress conditions. <b>(A)</b> Comparison of hypocotyl elongation between <i>PaMBF1c</i>-OE and <i>AtMBF1c</i>-OE lines under heat stress. Both OE lines showed similar levels of heat tolerance but higher than WT ( <i>n</i> ≥ 10). <b>(B)</b> Comparison of germination between <i>PaMBF1c</i>-OE and <i>AtMBF1c</i>-OE lines under salt stress (200 mM NaCl), ionic stress (15 mM LiCl), and osmotic stress (200 mM mannitol). <i>PaMBF1c</i>-OE displayed higher germination ratio than WT and <i>AtMBF1c</i>-OE lines under salt and ionic stresses ( <i>n</i> ≥ 100). <b>(C)</b> Comparison of root elongation between <i>PaMBF1c</i>-OE and <i>AtMBF1c</i>-OE lines under salt stress (100 mM NaCl), ionic stress (15 mM LiCl), and osmotic stress (200 mM mannitol). <i>PaMBF1c</i>-OE showed higher root elongation than <i>AtMBF1c</i>-OE and WT under salt and ionic stresses ( <i>n</i> ≥ 10). Error bars represents standard deviation of the mean values of three independent experiments. Asterisks indicate statistical significance in LSD test ( <i>p</i> &amp;lt; 0.05). </p>   </div> <p>We further extended the comparison to other types of stress. We treated <i>PaMBF1c-</i>OE and <i>AtMBF1c-</i>OE lines with salt (100–200 mM NaCl), ionic (15 mM LiCl) and osmotic stress (200 mM mannitol) at germination and post-germination stages. Under salt stress, <i>PaMBF1c</i>-OE lines consistently displayed higher germination ratios than WT, whereas <i>AtMBF1c</i>-OE lines germinated at almost the same ratio as WT ( <b>Figure <a href="#F6">6B</a> </b>). In addition, the germination of <i>PaMBF1c</i>-OE lines was better than those of WT and <i>AtMBF1c</i>-OE lines under ionic stress (15 mM LiCl) ( <b>Figure <a href="#F6">6B</a> </b>). However, germination of each line at osmotic stress (200 and 400 mM mannitol) were largely comparable, with one <i>PaMBF1c</i>-OE line demonstrating a slightly higher germination ratio than the other genotypes ( <b>Figure <a href="#F6">6B</a> </b> and <b>Supplementary Figure <a href="#SM6">S6</a> </b>). These results suggest that <i>PaMBF1c</i>-OE performs better in terms of germination than WT or even <i>AtMBF1c</i>-OE, particularly under salt and ionic stress. </p> <p>Root growth comparisons also revealed that the root of <i>PaMBF1c</i>-OE lines grew longer under salt and ionic stresses than that of both WT and <i>AtMBF1c</i>-OE ( <b>Figure <a href="#F6">6C</a> </b>). <i>AtMBF1c</i>-OE lines exhibited root growth similar to that of WT under salt and ionic stresses ( <b>Figure <a href="#F6">6C</a> </b>). Under osmotic stress (200 mM mannitol), root elongation was similar between <i>PaMBF1-</i>OE and <i>AtMBF1c</i>-OE lines but was significantly enhanced in comparison with WT ( <b>Figure <a href="#F6">6C</a> </b>). Improved osmotic stress tolerance in the <i>AtMBF1c</i>-OE line was consistent with previous reports ( <a href="#B42">Suzuki et al., 2005</a>). These results revealed that except for germination under osmotic stress, <i>PaMBF1c</i>-OE lines showed enhanced tolerance at both germination and post-germination stage under all tested conditions including stress conditions in which <i>AtMBF1c</i> did not show increased tolerance. Conclusively, these results implicate <i>PaMBF1c</i> as an <i>MBF1c</i> allele which affords better coping against multiple stresses than <i>AtMBF1c</i>. </p> </div> <div class="transcriptomeanalysisofpambf1coverexpressinglinesundersaltstressconditions" title="sec"> <div class="title" tagx="title" title="title">
Transcriptome Analysis of <i>PaMBF1c</i> Overexpressing Lines under Salt Stress Conditions </div> <p>In order to understand the gene expression patterns of <i>PaMBF1c</i>- <i>OE</i> lines in salt stress response and identify genes important for <i>PaMBF1c</i>-specific stress tolerance, we carried out RNA-Seq analysis. After sequencing with total RNA from salt-treated samples, we first defined salt-regulated genes in WT as the genes with statistically altered expression in WT under salt stress. A total of 5,360 genes were salt-regulated with 1,845 genes up-regulated and 3,515 down-regulated ( <b>Figure <a href="#F7">7A</a> </b> and Supplementary Table <a href="#SM9">S2</a>). Among these salt-regulated genes, we then identified the up- or down-regulated genes in <i>PaMBF1c</i>-OE and <i>AtMBF1c</i>-OE lines under normal condition. We found that 10 genes in the <i>PaMBF1c</i>-OE line were already up-regulated, which included three genes that were also already up-regulated in the <i>AtMBF1c</i>-OE line ( <b>Table <a href="#T1">1</a> </b> and <b>Figure <a href="#F7">7B</a> </b>). We validated the expression of several genes in <b>Table <a href="#T1">1</a> </b> by real-time PCR and found that gene expression of all tested genes was consistent with RNA-Seq results ( <b>Supplementary Figure <a href="#SM7">S7</a> </b>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">FIGURE 7</span> </a> </div>   <p>Transcriptome analysis of WT, <i>PaMBF1c</i>-OE, and <i>AtMBF1c</i>-OE lines. <b>(A)</b> Numbers of salt-regulated genes ( <i>p</i>-value &amp;lt; 0.05, FDR corrected <i>p</i>-value &amp;lt; 0.05, absolute value of fold change &amp;gt; 1.5) in WT, <i>PaMBF1c</i>-OE, and <i>AtMBF1c</i>-OE lines. Shaded regions indicate genes commonly regulated by salt in all three lines. Numbers above the pink bars denote the total number of up-regulated genes and ones below the blue bars indicate the total number of down-regulated genes in each genotype. Numbers by the shaded regions in the middle of the bar are the total number of genes that were commonly up-/down-regulated in all variants. <b>(B)</b> Venn diagram showing 10 salt up-regulated genes that were already up-regulated in <i>PaMBF1c</i>-OE and 7 genes in <i>AtMBF1c</i>-OE plants under normal conditions. </p>   </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <p>Salt up-regulated genes that were already up-regulated in the <i>PaMBF1</i>c-OE lines under normal conditions. </p>  <table> <thead> <tr> <th>Gene locus</th> <th>Annotation</th> <th> <i>PaMBF1c</i>-OE vs. WT under normal conditions <hr /> </th> <th>Salt treated vs. normal in WT <hr /> </th> </tr> <tr> <td /> <td /> <th>Fold change</th> <th> <i>p</i>-value </th> <th>Fold change</th> <th> <i>p</i>-value </th> </tr> </thead> <tbody> <tr> <td>At1g33480</td> <td>RING/U-box superfamily protein</td> <td>22.29</td> <td>0</td> <td>6.93</td> <td>3.2.E - 05</td> </tr> <tr> <td>At5g59310</td> <td>Lipid transfer protein 4 (LTP4)</td> <td>14.47</td> <td>5.4.E - 03</td> <td>542.72</td> <td>0</td> </tr> <tr> <td>At5g26970</td> <td>Unknown protein</td> <td>5.61</td> <td>4.2.E - 02</td> <td>7.70</td> <td>8.3.E - 04</td> </tr> <tr> <td>At5g51720 <sup>∗</sup> </td> <td>NEET group protein</td> <td>2.29</td> <td>4.9.E - 04</td> <td>1.99</td> <td>1.1.E - 03</td> </tr> <tr> <td>At2g14610 <sup>∗</sup> </td> <td>Pathogenesis related gene1 (PR1)</td> <td>2.13</td> <td>1.6.E - 06</td> <td>2.49</td> <td>3.4.E - 12</td> </tr> <tr> <td>At1g77120 <sup>∗</sup> </td> <td>Alcohol dehydrogenase 1 (ADH1)</td> <td>1.98</td> <td>2.3.E - 04</td> <td>14.99</td> <td>0</td> </tr> <tr> <td>At1g29395</td> <td>COR414-TM1</td> <td>1.68</td> <td>1.8.E - 02</td> <td>4.04</td> <td>0</td> </tr> <tr> <td>At1g23130</td> <td>Bet vl allergen family protein</td> <td>1.62</td> <td>0</td> <td>1.53</td> <td>6.2.E - 14</td> </tr> <tr> <td>At4g25100</td> <td>Fe-superoxide dismutase1 (FeSOD1)</td> <td>1.57</td> <td>0</td> <td>2.27</td> <td>0</td> </tr> <tr> <td>At1g56580</td> <td>Smaller with variable branches (SVB)</td> <td>1.53</td> <td>8.5.E - 07</td> <td>3.52</td> <td>0</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <span class="attrib" tagx="attrib" title="attrib"> <i> <sup>∗</sup>Genes already up-regulated also in <i>AtMBF1c</i>-OE under normal conditions. The list of genes shown were determined from a cut-off value ( <i>p</i>-value &amp;lt; 0.05, corrected <i>p</i>-value of FDR &amp;lt; 0.05, fold change &amp;gt; 1.5). </i> </span> </div> </div> <p>None of the salt down-regulated genes were found already down-regulated in either <i>PaMBF1c</i> or <i>AtMBF1c</i> overexpressing lines (data not shown). The already up-regulated genes, particularly <i>PaMBF1c</i>-OE specific genes, might elicit quicker response to salt stress in <i>PaMBF1c</i>-OE lines, resulting in enhanced salt tolerance in overexpressors. Among the already up-regulated genes in <i>PaMBF1c</i>-OE, the <i>FeSOD1</i> gene (At4g25100) was identified in a transcriptome comparison study by <a href="#B44">Taji et al. (2004)</a> in which 77 genes were identified that showed higher expression under normal conditions in a halophyte <i>Thellungiella halophila</i> than in Arabidopsis. This suggests that early establishment of <i>FeSOD1</i>-mediated detoxification of reactive oxygen species (ROS) in <i>PaMBF1c</i>-OE might be an important step in the enhanced tolerance to salt stress which usually causes secondary oxidative stress. </p> <p>We also compared profiles of salt-regulated genes from WT, <i>PaMBF1c</i>-OE and <i>AtMBF1c</i>-OE lines. The two overexpressors showed similar numbers of genes with altered expression under salt stress as did wild type (1,808 and 1,826up-regulated genes in <i>PaMBF1c</i>-OE and <i>AtMBF1c</i>-OE, respectively; 3,394 and 3,472 down-regulated genes in <i>PaMBF1c</i>-OE and <i>AtMBF1c</i>-OE, respectively) ( <b>Figure <a href="#F7">7A</a> </b> and Supplementary Table <a href="#SM10">S3</a>). Expression of the majority of these genes was commonly regulated by salt stress in all genotypes; i.e., WT, <i>PaMBF1c</i>-OE, and <i>AtMBF1c</i>-OE lines each possessed the same set of 1,430 up-regulated and 2,475 down-regulated genes out of a total of 1845, 1,808 and 1,826 up-regulated and 3515, 3,394 and 3,472 down-regulated genes in WT, <i>PaMBF1c</i>-OE and <i>AtMBF1c</i>-OE, respectively ( <b>Figure <a href="#F7">7A</a> </b> and Supplementary Table <a href="#SM10">S3</a>). At least in part, the genes with altered expression unique to <i>PaMBF1c</i>-OE might contribute to the enhanced salt tolerance of the <i>PaMBF1c</i>-OE lines. </p> <p>Gene ontology enrichment analysis within the category ‘biological processes’ was conducted with salt-regulated genes using the whole Arabidopsis genome set as reference. The overall distribution pattern of salt-regulated genes in GO terms was well conserved among WT, <i>PaMBF1c</i>-OE, and <i>AtMBF1c</i>-OE lines under salt stress conditions ( <b>Figure <a href="#F8">8A</a> </b>). The terms RNA metabolic process, response to stress, response to stimulus, DNA metabolic process, and carbohydrate metabolic process were significantly over- or under-represented in all three plants compared to the Arabidopsis genome ( <i>p</i>-value of the Bonferroni correction for multiple testing &amp;lt; 0.05) ( <b>Figure <a href="#F8">8A</a> </b>). The term protein metabolic process was significantly over-represented only in WT plants but not in <i>PaMBF1c</i>-OE or <i>AtMBF1c</i>-OE, suggesting that protein metabolism is a major biological process affected in WT plants under salt stress. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">FIGURE 8</span> </a> </div>   <p>Gene ontology enrichment analysis of salt regulated genes in WT, <i>PaMBF1c</i>-OE, and <i>AtMBF1c</i>-OE lines. <b>(A)</b> Gene Ontology (GO) terms with significance (corrected <i>p</i>-value of Bonferroni correction &amp;lt; 0.05) from the GO enrichment analysis with salt-regulated genes using Arabidopsis whole genome as a comparison reference. <b>(B)</b> Functional GO classification of genes salt up-regulated genes based on plant GO slim terms. <b>(C)</b> Functional GO classification of genes salt down-regulated genes based on plant GO slim terms. Only GO terms with significance (corrected <i>p</i>-value of Bonferroni correction &amp;lt; 0.05) from the GO enrichment analysis were presented. For suffix of each line in the graph legends, "-S" indicates "salt-treated," "up" and "down" mean "up-regulated" and "down-regulated," respectively. Asterisks indicate statistical significance. </p>   </div> <p>To increase the resolution of GO enrichment analysis, we divided salt-regulated genes into salt up-regulated and salt down-regulated groups and conducted separate GO enrichment analyses for each with the whole Arabidopsis genome as reference. For salt up-regulated genes, GO terms in biological processes that were significantly over-represented in <i>PaMBF1c</i>-OE plants were ‘lipid metabolic process,’ ‘ion transport,’ and ‘cellular amino acid biosynthetic process’ ( <b>Figure <a href="#F8">8B</a> </b>). The corresponding salt up-regulated genes included in these terms were listed in Supplementary Table <a href="#SM11">S4</a>. Given the possibility that improved salt tolerance in <i>PaMBF1c</i>-OE might be mainly due to enhanced ionic stress tolerance, we manually surveyed genes for the term ‘ion transport’ that were significantly up-regulated in <i>PaMBF1c</i>-OE only. Notably, genes involved in ATP production and transport ( <i>Mitochondrial phosphate transporter</i> [At3g48850] and <i>ADP/ATP carrier 3</i> [At4g28390]) as well as ATP-dependent Ca <sup>2+</sup> pumping ( <i>ECA1</i> [At1g07810]) were among the <i>PaMBF1c</i>-OE specific salt up-regulated genes (Supplementary Table <a href="#SM11">S4</a>) ( <a href="#B27">Liang et al., 1997</a>; <a href="#B53">Wu et al., 2002</a>). Thus, enhanced salt tolerance in <i>PaMBF1c</i>-OE might be caused in part by promotions of ATP synthesis and Ca <sup>2+</sup> signaling. </p> <p>For salt down-regulated genes, the GO terms ‘cellular component biogenesis,’ ‘response to stress,’ ‘protein metabolic process,’ and ‘cholesterol metabolic process’ were significantly over-represented in WT plants under salt-stress condition but not so in <i>PaMBF1c</i>-OE or <i>AtMBF1c</i>-OE plants with the exception of ‘cellular component biogenesis’ which was over-represented in both WT and <i>AtMBF1c</i>-OE plants ( <b>Figure <a href="#F8">8C</a> </b>). Salt down-regulated genes in these GO terms were listed in Supplementary Table <a href="#SM12">S5</a>. We reasoned that among the genes of these GO terms, those that are either salt up-regulated or not down-regulated only in <i>PaMBF1c</i>-OE lines would be more highly associated with enhanced salt stress tolerance in <i>PaMBF1c</i>-OE. And thus, we manually surveyed those groups of genes. Among the interesting findings that we noticed was that many ribosome and translation-related genes were not salt down-regulated only in <i>PaMBF1c</i>-OE (Supplementary Table <a href="#SM12">S5</a>), which suggests that <i>PaMBF1c</i> might function in promoting protein synthesis processes particularly to acquire enhanced salt stress tolerance. </p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Developing crop plants with inbuilt tolerance for multiple stresses is a requisite for mitigating damage to global agriculture productivity. So far, numerous efforts have been made to clone novel genes from a diverse array of species acclimated to various adverse environments. Our study demonstrated that ectopic expression of <i>PaMBF1c</i> enhanced the adaption of transgenic Arabidopsis to various abiotic stresses and in particular, to salt stress. </p> <p>Phylogenetic analysis of PaMBF1c and MBF1s from other plant species ranging from algae to dicot plants suggested a clear divergence between MBF1c and MBF1a/b (MBF1a and MBF1b) proteins among both vascular and non-vascular plants ( <b>Figure <a href="#F1">1B</a> </b>). Other studies have also concluded the same ( <a href="#B50">Tsuda and Yamazaki, 2004</a>; <a href="#B30">Liu et al., 2007</a>). This denotes that MBF1c proteins of plants might possess non-redundant functions which differ from those of MBF1a/b proteins despite sharing similar conservative domains among the MBF1 family proteins. </p> <p>Another interesting finding was the general maintenance of <i>MBF1c</i> as a single gene versus the variability of <i>MBF1a</i> or <i>MBF1b</i> group genes depending on species ( <b>Figure <a href="#F1">1B</a> </b>). Phylogenetic analysis revealed that this divergence between <i>MBF1a/b</i> and <i>MBF1c</i> is apparent in both vascular and non-vascular plants. Algae species (i.e., <i>Chlamydomonas reinhardtii, Coccomyxa subellipsoidea</i>, and <i>Volvox carteri</i>) contain only one <i>MBF1</i> gene while most land plants possess two ( <i>MBF1a/b</i> and <i>MBF1c</i>), suggesting that gene duplication occurred early in land plant emergence. In some species such as <i>Marchantia polymorpha, Oryza sativa</i>, and <i>Sorghum bicolor</i>, only two copies of <i>MBF1</i> genes (one <i>MBF1a</i> or <i>MBF1b</i>, and one <i>MBF1c</i>) exist. Species such as <i>A. thaliana</i>, <i>Populus trichocarpa</i>, <i>Physcomitrella patens</i>, and <i>Brachypodium distachyon</i> appeared to have experienced a single round of recent gene duplication in the <i>MBF1a</i>/ <i>b</i> gene which resulted in three <i>MBF1</i> genes including the <i>MBF1c</i> gene. </p> <p>Thus, it is likely that <i>MBF1c</i> is functionally distinct from <i>MBF1a/b</i> since its evolution and mainly function in stress tolerance regulation. This notion has been supported by many studies including our own; <i>PaMBF1c</i>-OE demonstrated similar enhanced stress tolerance to heat and osmotic stress as <i>AtMBF1c</i>-OE ( <a href="#B42">Suzuki et al., 2005</a>). In addition, <i>PaMBF1c</i> overexpression brought about enhanced salt stress tolerance which was not observed in the case of <i>AtMBF1c</i> overexpression. Thus, <i>PaMBF1c</i> may have obtained additional function during evolution to cope with the high salt conditions of Antarctic soil (see below). </p> <p>Consistent with its function, expression of <i>MBF1c</i> was highly induced by different abiotic stresses in plants. For example, transcript levels of <i>TaMBF1c</i> was increased under drought, H <sub>2</sub>O <sub>2</sub>, and heat stress conditions in <i>Triticum aestivum</i> ( <a href="#B39">Qin et al., 2015</a>). Similarly, expression of <i>PaMBF1c</i> was induced by mannitol, NaCl, and heat treatment ( <b>Figure <a href="#F2">2</a> </b>). In Arabidopsis, expression of <i>AtMBF1c</i> was also induced in response to heat and drought or a combination of both heat and osmotic stress ( <a href="#B42">Suzuki et al., 2005</a>). In contrast, the expression of <i>AtMBF1a</i> and <i>AtMBF1b</i> was not altered by abiotic stress, but rather developmentally regulated ( <a href="#B50">Tsuda and Yamazaki, 2004</a>). These observations suggest that <i>MBF1</i> genes have evolved not only at the coding sequence level, but also at the level of regulatory sequence for the diversification of function. </p> <p>In addition to their functions in abiotic stress responses, <i>PaMBF1c</i> and <i>AtMBF1c</i> appear to have some role in development. In comparison to WT, both <i>PaMBF1c</i>-OE and <i>AtMBF1c</i>-OE displayed higher fresh weight and flowered earlier. The enhanced growth by <i>MBF1</i> overexpression was also reported in a <i>CaMBF1</i> overexpression study even in cases showing stress sensitive phenotypes ( <a href="#B16">Guo et al., 2014</a>). This robust growth of <i>MBF1</i> overexpression plants might be due to elevated endoreduplication and promotion of cell expansion throughout the leaves ( <a href="#B48">Tojo et al., 2009</a>). These observations indicate the functional similarity of <i>PaMBF1c</i> and <i>AtMBF1c</i> during development. Thus, the fact that enhanced salt stress tolerance was achieved only by <i>PaMBF1c</i> overexpression, despite these similarities between <i>PaMBF1c</i> and <i>AtMBF1c</i>, underscores a broader functional spectrum of <i>PaMBF1c</i> from the polar moss. </p> <p>Adverse effects of salt stress on plants are the combined result of osmotic stress and ionic stress. Ionic stress tolerance was enhanced only in <i>PaMBF1c</i>-OE at 15–20 mM LiCl, while similar levels of osmotic stress tolerance was measured in <i>PaMBF1c</i>-OE and <i>AtMBF1c</i>-OE at 200 mM mannitol ( <b>Figure <a href="#F6">6</a> </b>). This indicates that the <i>PaMBF1c</i> gene might cope better against ionic stress imposed by salt treatment in comparison with the <i>AtMBF1c</i> gene. The moss <i>P. alpinum</i> was collected from Antarctic soils which contain characteristically high contents of soluble salts (i.e., sulfates, chlorides, nitrates, potassium, calcium, and magnesium) resulting from chemical weathering of rocks, marine salt deposits and sedimentary rock leaching ( <a href="#B10">Claridge and Campbell, 1977</a>). Thus, <i>PaMBF1c</i> might have evolved to function better in the high salt conditions of Antarctic, probably through optimization of amino acid residues. We found that there are 14 amino acid residues in PaMBF1c different from in <i>Triticum aestivum MBF1c</i> (ACU43593) and <i>AtMBF1c</i> (NP_189093) whose functions were shown to be involved in abiotic stress tolerance ( <a href="#B42">Suzuki et al., 2005</a>; <a href="#B39">Qin et al., 2015</a>). However, we cannot rule out the possibility that foreign gene (i.e., <i>PaMBF1c</i>) overexpression could have escaped from the endogenous gene regulation system of Arabidopsis to outperform endogenous gene (i.e., <i>AtMBF1c</i>) overexpression in stress response. Still, it cannot be denied that the case of <i>PaMBF1c</i> shows a good example of the advantage of using foreign genes to improve stress tolerance in plants. A question may arise as to whether the stress tolerance phenotypes in <i>PaMBF1c</i>-OE might be due to its enhanced growth; however, bigger sized plants do not always result in improved stress tolerance. For example, <i>CaMBF1</i> overexpression resulted in Arabidopsis with large leaves but with reduced stress tolerance to cold and salt stress ( <a href="#B16">Guo et al., 2014</a>). In our <i>PaMBF1c</i>-OE lines, we did not observe enhanced root growth under normal conditions ( <b>Supplementary Figure <a href="#SM2">S2</a> </b>) despite the fact that <i>PaMBF1c</i>-OE lines showed longer root length under stress conditions than WT and, in some cases, both WT and <i>AtMBF1c</i>-OE ( <b>Figures <a href="#F4">4</a> </b>, <b> <a href="#F6">6</a> </b>). In addition, analysis of germination, MDA content, and anthocyanin content, which normally are not directly related to enhanced growth, suggested improved salt tolerance in <i>PaMBF1c</i>-OE. It should be noted that germination, MDA content and anthocyanin content were very similar among the tested lines under control conditions. Therefore, we believe that improved stress tolerance by <i>PaMBF1c</i> overexpression is not likely due to enhanced growth. </p> <p>Thus, how does <i>PaMBF1c</i>, but not <i>AtMBF1c</i>, mechanistically bring about enhanced salt stress tolerance? A simple answer to this question would be that the genes with salt-altered expression only in <i>PaMBF1c</i>-OE account for the beneficial <i>PaMBF1c</i> function. According to this view, attention needs to be given to the genes involved in the over-represented GO terms of lipid metabolic process, ion transport, and cellular amino acid biosynthetic process among salt up-regulated genes in <i>PaMBF1c</i>-OE ( <b>Figure <a href="#F8">8B</a> </b>). Another candidate gene group for better salt tolerance in <i>PaMBF1c</i>-OE could be the salt-regulated genes that are already up-regulated under normal conditions ( <b>Table <a href="#T1">1</a> </b>). Ten and seven salt-induced genes were already highly expressed in <i>PaMBF1c</i>-OE and <i>AtMBF1c</i>-OE under normal conditions, respectively ( <b>Table <a href="#T1">1</a> </b> and <b>Figure <a href="#F7">7B</a> </b>). The 10 up-regulated genes in <i>PaMBF1c</i>-OE included three genes that were also up-regulated in <i>AtMBF1c</i>-OE ( <b>Table <a href="#T1">1</a> </b> and <b>Figure <a href="#F7">7B</a> </b>). </p> <p>Thus, these already up-regulated genes in <i>PaMBF1c</i>-OE might prime the OE lines for quicker responses to salt stress resulting in enhanced salt tolerance in the <i>PaMBF1c</i>-OE lines. One such rapid response could be the salt-induced removal of ROS by <i>FeSOD1</i> (At4g25100) whose expression was already high under normal conditions also in the salt cress <i>T. halophila</i> ( <a href="#B44">Taji et al., 2004</a>). It is known that salt stress induces oxidative stress ( <a href="#B18">Hernández et al., 1993</a>). Also, oxidative stress is shown to cause ATP depletion ( <a href="#B47">Tiwari et al., 2002</a>). Thus, given the fact that ATP production and transport genes (At3g48850, At4g28390) and ATP-dependent Ca <sup>2+</sup> pump, <i>ECA1</i> (At1g07810) were uniquely salt up-regulated only in <i>PaMBF1c</i>-OE (Supplementary Table <a href="#SM11">S4</a>), the enhancement of salt tolerance in <i>PaMBF1c</i>-OE could involve ROS detoxification, maintenance of ATP homeostasis, and facilitation of Ca <sup>2+</sup> signaling. Additionally, <i>PaMBF1c</i>-OE function in salt stress might be associated with protein synthesis, given that many ribosome and translation-related genes, which were otherwise down-regulated by salt, were shown not down-regulated solely in <i>PaMBF1c</i>-OE lines. Indeed, several studies have reported that some ribosome- and translation-related genes were shown to be involved in abiotic stress responses; for example, knock-down of eukaryotic translation initiation factor 5A (eIF5A) in Arabidopsis brought about hypersensitivity to heat, oxidative and osmotic stresses, while overexpression of eIF5A resulted in osmotic stress tolerance ( <a href="#B32">Ma et al., 2010</a>; <a href="#B54">Xu et al., 2011</a>). In addition, Arabidopsis mutants defective in cytosolic 60S ribosomal maturation factor <i>REIL</i> displayed cold-sensitive phenotypes ( <a href="#B41">Schmidt et al., 2013</a>). Further molecular and physiological studies will be needed to uncover if and how these genes function in <i>PaMBF1c</i>-mediated salt tolerance. </p> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>HL and B-hL conceived and designed the research; HA and MP performed the experiments; HA, HL, and B-hL discussed the results and wrote the paper. </p> </div> <div class="conflictofintereststatement" title="sec"> <div class="title" tagx="title" title="title">
Conflict of Interest Statement</div> <p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest. </p> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p> <b>Funding.</b> This work was supported by Next-Generation BioGreen21 Program (PJ011006) to B-hL, Rural Development Administration, South Korea, and Polar Genomics 101 Project (PE17080) to HL, the Korea Polar Research Institute. </p> </div> </div> <div class="ack" title="ack"> <p>The authors thank Dr. Ron Mittler (University of North Texas, United States) for the <i>AtMBF1c</i> overexpressing lines. </p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">1</span> <p> <a href="https://phytozome.jgi.doe.gov/pz/portal.html">https://phytozome.jgi.doe.gov/pz/portal.html</a> </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">2</span> <p> <a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi">http://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi</a> </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">3</span> <p> <a href="http://www.ncbi.nlm.nih.gov/sra/">www.ncbi.nlm.nih.gov/sra/</a> </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">4</span> <p> <a href="http://pantherdb.org/">http://pantherdb.org/</a> </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">5</span> <p> <a href="http://web.expasy.org/compute_pi">http://web.expasy.org/compute_pi</a> </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">6</span> <p> <a href="http://www.phytozome.net">http://www.phytozome.net</a> </p> </div> </div> <div class="supplementary material" title="supplementary material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <p>The Supplementary Material for this article can be found online at: <a href="http://journal.frontiersin.org/article/10.3389/fpls.2017.01206/full#supplementary-material">http://journal.frontiersin.org/article/10.3389/fpls.2017.01206/full#supplementary-material</a> </p> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">FIGURE S1</span>  <p>Germination of WT, <i>PaMBF1c</i>-OE, and <i>AtMBF1c</i>-OE lines on the control media for each stress treatment. All lines showed near 100% germination levels on the control media. The control MS/agar plates were prepared separately for each stress treatment. </p>  <div class="media" title="media"> <a href="Image_1.JPEG">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Image_1.JPEG">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">FIGURE S2</span>  <p>Root elongation of WT, <i>PaMBF1c</i>-OE, and <i>AtMBF1c</i>-OE lines on the control media for each stress treatment. Root growth on control media (MS only) for <b>(A)</b> salt stress (NaCl), <b>(B)</b> ionic stress (LiCl) and <b>(C)</b> osmotic stress (mannitol). All lines showed similar root elongation on the control media (average of 2.56 ± 0.22 cm on 6 days after transfer). The control MS/agar plates were prepared separately for each stress treatment. </p>  <div class="media" title="media"> <a href="Image_2.JPEG">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Image_2.JPEG">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">FIGURE S3</span>  <p>Survival test of WT and <i>PaMBF1c</i>-OE lines under different abiotic stress conditions. <b>(A)</b> Seedlings on MS/agar plates before heat stress treatment. <b>(B)</b> Seedlings on MS/agar plates after heat stress treatment (45°C for 60 min). <b>(C)</b> Seedlings on control MS/agar plates for salt and ionic stress treatments. <b>(D)</b> Seedlings on 200 mM NaCl plates (salt stress). <b>(E)</b> Seedlings on 20 mM LiCl plates (ionic stress). </p>  <div class="media" title="media"> <a href="Image_3.JPEG">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Image_3.JPEG">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">FIGURE S4</span>  <p> <i>PaMBF1c</i>-OE lines showed lower ion leakage than WT at low temperatures. Fully expanded fourth or fifth rosette leaves with petiole from 3-week-old seedlings were used for the experiment. Ion leakage was expressed as a percentage of total electrolytes at a given temperature. Error bars represents standard deviation of the mean values of five independent experiments. </p>  <div class="media" title="media"> <a href="Image_4.JPEG">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Image_4.JPEG">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">FIGURE S5</span>  <p>Anthocyanin accumulation under salt stress. <b>(A)</b> Ten-day-old seedlings grown on control and 150 mM NaCl plates (salt stress) <b>(B)</b> Quantification of anthocyanin accumulation in each line. Error bars represents standard deviation ( <i>n</i> = 25 seedlings per each treatment). Asterisks indicate statistical significance in LSD test ( <i>p</i> &amp;lt; 0.05). </p>  <div class="media" title="media"> <a href="Image_5.JPEG">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Image_5.JPEG">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">FIGURE S6</span>  <p>Comparison of germination between <i>PaMBF1c</i>-OE and <i>AtMBF1c</i>-OE lines under different concentrations of mannitol. Error bars represents standard deviation of the mean values of three independent experiments. </p>  <div class="media" title="media"> <a href="Image_6.JPEG">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Image_6.JPEG">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">FIGURE S7</span>  <p>Validation of the gene expression profiles obtained in RNA-sequencing by qRT-PCR. Relative expression levels of selected genes were determined by qRT-PCR using the cDNA synthesized from RNA isolated from the 14-day-old seedlings treated to high salt stress (300 mM NaCl for 6 h). <b>(A)</b> RING/U-box super family protein (At1g33480); <b>(B)</b> lipid transfer protein 4 (At5g59310); <b>(C)</b> Fe-superoxide dismutase (At4g25100); <b>(D)</b> hypothetical protein (At5g26970). Arabidopsis Clathrin (At4g24550) gene was used as an internal control for normalization of different cDNA samples. Three biological replicates were averaged and error bars represent standard deviation. </p>  <div class="media" title="media"> <a href="Image_7.JPEG">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Image_7.JPEG">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Table_1.XLSX">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Table_2.XLSX">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Table_3.XLSX">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Table_4.XLSX">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Table_5.XLSX">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="B1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alavilli</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Awasthi</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rout</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sahoo</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">B.-H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panda</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Overexpression of a barley aquaporin gene, HvPIP2;5 confers salt and osmotic stress tolerance in yeast and plants. </span> <span class="source" tagx="source" title="source"> <i>Front. Plant Sci.</i> </span> <span class="volume" tagx="volume" title="volume">7</span>: <span class="issue" tagx="issue" title="issue">1566</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3389/fpls.2016.01566">10.3389/fpls.2016.01566</a> </span> </span> </li> <li tag="ref"> <a name="B2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aravind</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koonin</span> <span class="given-names" tagx="given-names" title="given-names">E. V.</span> </span> </span> ( <span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">DNA-binding proteins and evolution of transcription regulation in the archaea.</span> <span class="source" tagx="source" title="source"> <i>Nucleic Acids Res.</i> </span> <span class="volume" tagx="volume" title="volume">27</span> <span class="fpage" tagx="fpage" title="fpage">4658</span>– <span class="lpage" tagx="lpage" title="lpage">4670</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/27.23.4658">10.1093/nar/27.23.4658</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10556324">10556324</a> </span> </span> </li> <li tag="ref"> <a name="B3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arce</span> <span class="given-names" tagx="given-names" title="given-names">D. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Godoy</span> <span class="given-names" tagx="given-names" title="given-names">A. V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsuda</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamazaki</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Valle</span> <span class="given-names" tagx="given-names" title="given-names">E. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iglesias</span> <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">The analysis of an Arabidopsis triple knock-down mutant reveals functions for MBF1 genes under oxidative stress conditions. </span> <span class="source" tagx="source" title="source"> <i>J. Plant Physiol.</i> </span> <span class="volume" tagx="volume" title="volume">167</span> <span class="fpage" tagx="fpage" title="fpage">194</span>– <span class="lpage" tagx="lpage" title="lpage">200</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jplph.2009.09.003">10.1016/j.jplph.2009.09.003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19783066">19783066</a> </span> </span> </li> <li tag="ref"> <a name="B4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arce</span> <span class="given-names" tagx="given-names" title="given-names">D. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tonon</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zanetti</span> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Godoy</span> <span class="given-names" tagx="given-names" title="given-names">A. V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirose</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Casalongue</span> <span class="given-names" tagx="given-names" title="given-names">C. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">The potato transcriptional co-activator StMBF1 is up-regulated in response to oxidative stress and interacts with the TATA-box binding protein. </span> <span class="source" tagx="source" title="source"> <i>J. Biochem. Mol. Biol.</i> </span> <span class="volume" tagx="volume" title="volume">39</span> <span class="fpage" tagx="fpage" title="fpage">355</span>– <span class="lpage" tagx="lpage" title="lpage">360</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.5483/bmbrep.2006.39.4.355">10.5483/bmbrep.2006.39.4.355</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16889677">16889677</a> </span> </span> </li> <li tag="ref"> <a name="B5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ashton</span> <span class="given-names" tagx="given-names" title="given-names">N. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cove</span> <span class="given-names" tagx="given-names" title="given-names">D. J.</span> </span> </span> ( <span class="year" tagx="year" title="year">1977</span>). <span class="mixed-article-title" title="mixed-article-title">The isolation and preliminary characterisation of auxotrophic and analogue resistant mutants of the moss, <i>Physcomitrella patens</i>. </span> <span class="source" tagx="source" title="source"> <i>Mol. Gen. Genet.</i> </span> <span class="volume" tagx="volume" title="volume">154</span> <span class="fpage" tagx="fpage" title="fpage">87</span>– <span class="lpage" tagx="lpage" title="lpage">95</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/BF00265581">10.1007/BF00265581</a> </span> </span> </li> <li tag="ref"> <a name="B6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bell</span> <span class="given-names" tagx="given-names" title="given-names">N. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hyvonen</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Phylogeny of the moss class Polytrichopsida (BRYOPHYTA): generic-level structure and incongruent gene trees. </span> <span class="source" tagx="source" title="source"> <i>Mol. Phylogenet. Evol.</i> </span> <span class="volume" tagx="volume" title="volume">55</span> <span class="fpage" tagx="fpage" title="fpage">381</span>– <span class="lpage" tagx="lpage" title="lpage">398</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ympev.2010.02.004">10.1016/j.ympev.2010.02.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20152915">20152915</a> </span> </span> </li> <li tag="ref"> <a name="B7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhattarai</span> <span class="given-names" tagx="given-names" title="given-names">H. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paudel</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">H. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oh</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yim</span> <span class="given-names" tagx="given-names" title="given-names">J. H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">In vitro antioxidant capacities of two Benzonaphthoxanthenones: ohioensins F and G, isolated from the antarctic moss <i>Polytrichastrum alpinum</i>. </span> <span class="source" tagx="source" title="source"> <i>Z. Naturforsch. C</i> </span> <span class="volume" tagx="volume" title="volume">64</span> <span class="fpage" tagx="fpage" title="fpage">197</span>– <span class="lpage" tagx="lpage" title="lpage">200</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1515/znc-2009-3-408">10.1515/znc-2009-3-408</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19526712">19526712</a> </span> </span> </li> <li tag="ref"> <a name="B8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brendel</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gelman</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Auwerx</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Multiprotein bridging factor-1 (MBF-1) is a cofactor for nuclear receptors that regulate lipid metabolism. </span> <span class="source" tagx="source" title="source"> <i>Mol. Endocrinol.</i> </span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">1367</span>– <span class="lpage" tagx="lpage" title="lpage">1377</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1210/mend.16.6.0843">10.1210/mend.16.6.0843</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12040021">12040021</a> </span> </span> </li> <li tag="ref"> <a name="B9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Claeys</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Landeghem</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dubois</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maleux</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Inze</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">What is stress? dose-response effects in commonly used in vitro stress assays.</span> <span class="source" tagx="source" title="source"> <i>Plant Physiol.</i> </span> <span class="volume" tagx="volume" title="volume">165</span> <span class="fpage" tagx="fpage" title="fpage">519</span>– <span class="lpage" tagx="lpage" title="lpage">527</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.113.234641">10.1104/pp.113.234641</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24710067">24710067</a> </span> </span> </li> <li tag="ref"> <a name="B10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Claridge</span> <span class="given-names" tagx="given-names" title="given-names">G. G. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Campbell</span> <span class="given-names" tagx="given-names" title="given-names">I. B.</span> </span> </span> ( <span class="year" tagx="year" title="year">1977</span>). <span class="mixed-article-title" title="mixed-article-title">THE salts in antarctic soils, their distribution and relationship to soil processes.</span> <span class="source" tagx="source" title="source"> <i>Soil Sci.</i> </span> <span class="volume" tagx="volume" title="volume">123</span> <span class="fpage" tagx="fpage" title="fpage">377</span>– <span class="lpage" tagx="lpage" title="lpage">384</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1097/00010694-197706000-00006">10.1097/00010694-197706000-00006</a> </span> </span> </li> <li tag="ref"> <a name="B11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clough</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bent</span> <span class="given-names" tagx="given-names" title="given-names">A. F.</span> </span> </span> ( <span class="year" tagx="year" title="year">1998</span>). <span class="mixed-article-title" title="mixed-article-title">Floral dip: a simplified method for <i>Agrobacterium</i>-mediated transformation of <i>Arabidopsis thaliana</i>. </span> <span class="source" tagx="source" title="source"> <i>Plant J.</i> </span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">735</span>– <span class="lpage" tagx="lpage" title="lpage">743</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1365-313x.1998.00343.x">10.1046/j.1365-313x.1998.00343.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10069079">10069079</a> </span> </span> </li> <li tag="ref"> <a name="B12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Curtis</span> <span class="given-names" tagx="given-names" title="given-names">M. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grossniklaus</span> <span class="given-names" tagx="given-names" title="given-names">U.</span> </span> </span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">A gateway cloning vector set for high-throughput functional analysis of genes in planta.</span> <span class="source" tagx="source" title="source"> <i>Plant Physiol.</i> </span> <span class="volume" tagx="volume" title="volume">133</span> <span class="fpage" tagx="fpage" title="fpage">462</span>– <span class="lpage" tagx="lpage" title="lpage">469</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.103.027979">10.1104/pp.103.027979</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14555774">14555774</a> </span> </span> </li> <li tag="ref"> <a name="B13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Du</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiao</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Heterologous expression of two <i>Physcomitrella</i> patens group 3 late embryogenesis abundant protein (LEA3) genes confers salinity tolerance in arabidopsis. </span> <span class="source" tagx="source" title="source"> <i>J. Plant Biol.</i> </span> <span class="volume" tagx="volume" title="volume">59</span> <span class="fpage" tagx="fpage" title="fpage">182</span>– <span class="lpage" tagx="lpage" title="lpage">193</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s12374-016-0565-7">10.1007/s12374-016-0565-7</a> </span> </span> </li> <li tag="ref"> <a name="B14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Godoy</span> <span class="given-names" tagx="given-names" title="given-names">A. V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zanetti</span> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">San Segundo</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Casalongue</span> <span class="given-names" tagx="given-names" title="given-names">C. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">Identification of a putative <i>Solanum tuberosum</i> transcriptional coactivator up-regulated in potato tubers by <i>Fusarium solani</i> f. sp eumartii infection and wounding. </span> <span class="source" tagx="source" title="source"> <i>Physiol. Plant.</i> </span> <span class="volume" tagx="volume" title="volume">112</span> <span class="fpage" tagx="fpage" title="fpage">217</span>– <span class="lpage" tagx="lpage" title="lpage">222</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1034/j.1399-3054.2001.1120210.x">10.1034/j.1399-3054.2001.1120210.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11454227">11454227</a> </span> </span> </li> <li tag="ref"> <a name="B15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goodstein</span> <span class="given-names" tagx="given-names" title="given-names">D. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shu</span> <span class="given-names" tagx="given-names" title="given-names">S. Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Howson</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neupane</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hayes</span> <span class="given-names" tagx="given-names" title="given-names">R. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fazo</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Phytozome: a comparative platform for green plant genomics.</span> <span class="source" tagx="source" title="source"> <i>Nucleic Acids Res.</i> </span> <span class="volume" tagx="volume" title="volume">40</span> <span class="fpage" tagx="fpage" title="fpage">D1178</span>– <span class="lpage" tagx="lpage" title="lpage">D1186</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkr944">10.1093/nar/gkr944</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22110026">22110026</a> </span> </span> </li> <li tag="ref"> <a name="B16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">W. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">R. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Du</span> <span class="given-names" tagx="given-names" title="given-names">X. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yin</span> <span class="given-names" tagx="given-names" title="given-names">Y. X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gong</span> <span class="given-names" tagx="given-names" title="given-names">Z. H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Reduced tolerance to abiotic stress in transgenic Arabidopsis overexpressing a <i>Capsicum annuum</i> multiprotein bridging factor 1. </span> <span class="source" tagx="source" title="source"> <i>BMC Plant Biol.</i> </span> <span class="volume" tagx="volume" title="volume">14</span>: <span class="issue" tagx="issue" title="issue">138</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2229-14-138">10.1186/1471-2229-14-138</a> </span> </span> </li> <li tag="ref"> <a name="B17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heath</span> <span class="given-names" tagx="given-names" title="given-names">R. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Packer</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> </span> ( <span class="year" tagx="year" title="year">1968</span>). <span class="mixed-article-title" title="mixed-article-title">Photoperoxidation in isolated chloroplasts.</span> <span class="source" tagx="source" title="source"> <i>Arch. Biochem. Biophys.</i> </span> <span class="volume" tagx="volume" title="volume">125</span> <span class="fpage" tagx="fpage" title="fpage">189</span>– <span class="lpage" tagx="lpage" title="lpage">198</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0003-9861(68)90654-1">10.1016/0003-9861(68)90654-1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/5655425">5655425</a> </span> </span> </li> <li tag="ref"> <a name="B18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hernández</span> <span class="given-names" tagx="given-names" title="given-names">J. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Corpas</span> <span class="given-names" tagx="given-names" title="given-names">F. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gómez</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Del Río</span> <span class="given-names" tagx="given-names" title="given-names">L. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sevilla</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> </span> ( <span class="year" tagx="year" title="year">1993</span>). <span class="mixed-article-title" title="mixed-article-title">Salt-induced oxidative stress mediated by activated oxygen species in pea leaf mitochondria.</span> <span class="source" tagx="source" title="source"> <i>Physiol. Plant.</i> </span> <span class="volume" tagx="volume" title="volume">89</span> <span class="fpage" tagx="fpage" title="fpage">103</span>– <span class="lpage" tagx="lpage" title="lpage">110</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1399-3054.1993.tb01792.x">10.1111/j.1399-3054.1993.tb01792.x</a> </span> </span> </li> <li tag="ref"> <a name="B19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hong</span> <span class="given-names" tagx="given-names" title="given-names">S. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vierling</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> </span> ( <span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">Mutants of <i>Arabidopsis thaliana</i> defective in the acquisition of tolerance to high temperature stress. </span> <span class="source" tagx="source" title="source"> <i>Proc. Natl. Acad. Sci. U.S.A.</i> </span> <span class="volume" tagx="volume" title="volume">97</span> <span class="fpage" tagx="fpage" title="fpage">4392</span>– <span class="lpage" tagx="lpage" title="lpage">4397</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.97.8.4392">10.1073/pnas.97.8.4392</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10760305">10760305</a> </span> </span> </li> <li tag="ref"> <a name="B20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishitani</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiong</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stevenson</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">J. K.</span> </span> </span> ( <span class="year" tagx="year" title="year">1997</span>). <span class="mixed-article-title" title="mixed-article-title">Genetic analysis of osmotic and cold stress signal transduction in Arabidopsis: interactions and convergence of abscisic acid-dependent and abscisic acid-independent pathways. </span> <span class="source" tagx="source" title="source"> <i>Plant Cell</i> </span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">1935</span>– <span class="lpage" tagx="lpage" title="lpage">1949</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.9.11.1935">10.1105/tpc.9.11.1935</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9401119">9401119</a> </span> </span> </li> <li tag="ref"> <a name="B21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kabe</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goto</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shima</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Imai</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wada</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morohashi</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">The role of human MBF1 as a transcriptional coactivator.</span> <span class="source" tagx="source" title="source"> <i>J. Biol. Chem.</i> </span> <span class="volume" tagx="volume" title="volume">274</span> <span class="fpage" tagx="fpage" title="fpage">34196</span>– <span class="lpage" tagx="lpage" title="lpage">34202</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.274.48.34196">10.1074/jbc.274.48.34196</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10567391">10567391</a> </span> </span> </li> <li tag="ref"> <a name="B22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">G. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">E. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ko</span> <span class="given-names" tagx="given-names" title="given-names">C. B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">K. Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jeong</span> <span class="given-names" tagx="given-names" title="given-names">J. A.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Abiotic and biotic stress tolerance in Arabidopsis overexpressing the <i>Multiprotein bridging factor 1a (MBF1a)</i> transcriptional coactivator gene. </span> <span class="source" tagx="source" title="source"> <i>Biochem. Biophys. Res. Commun.</i> </span> <span class="volume" tagx="volume" title="volume">354</span> <span class="fpage" tagx="fpage" title="fpage">440</span>– <span class="lpage" tagx="lpage" title="lpage">446</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bbrc.2006.12.212">10.1016/j.bbrc.2006.12.212</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17234157">17234157</a> </span> </span> </li> <li tag="ref"> <a name="B23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stecher</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tamura</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets.</span> <span class="source" tagx="source" title="source"> <i>Mol. Biol. Evolut.</i> </span> <span class="volume" tagx="volume" title="volume">33</span> <span class="fpage" tagx="fpage" title="fpage">1870</span>– <span class="lpage" tagx="lpage" title="lpage">1874</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msw054">10.1093/molbev/msw054</a> </span> </span> </li> <li tag="ref"> <a name="B24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">B. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kapoor</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">J. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">J. K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">STABILIZED1, a stress-upregulated nuclear protein, is required for pre-mRNA splicing, mRNA turnover, and stress tolerance in Arabidopsis. </span> <span class="source" tagx="source" title="source"> <i>Plant Cell</i> </span> <span class="volume" tagx="volume" title="volume">18</span> <span class="fpage" tagx="fpage" title="fpage">1736</span>– <span class="lpage" tagx="lpage" title="lpage">1749</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.106.042184">10.1105/tpc.106.042184</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16751345">16751345</a> </span> </span> </li> <li tag="ref"> <a name="B25" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">B.-H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiong</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">J.-K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">A mitochondrial complex i defect impairs cold-regulated nuclear gene expression.</span> <span class="source" tagx="source" title="source"> <i>Plant Cell</i> </span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">1235</span>– <span class="lpage" tagx="lpage" title="lpage">1251</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.010433">10.1105/tpc.010433</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12084824">12084824</a> </span> </span> </li> <li tag="ref"> <a name="B26" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">F. Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ueda</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirose</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> ( <span class="year" tagx="year" title="year">1994</span>). <span class="mixed-article-title" title="mixed-article-title">Mediators of activation of Fushi-Tarazu Gene-transcription by Bmftz-F1.</span> <span class="source" tagx="source" title="source"> <i>Mol. Cell. Biol.</i> </span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">3013</span>– <span class="lpage" tagx="lpage" title="lpage">3021</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1128/MCB.14.5.3013">10.1128/MCB.14.5.3013</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8164657">8164657</a> </span> </span> </li> <li tag="ref"> <a name="B27" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liang</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cunningham</span> <span class="given-names" tagx="given-names" title="given-names">K. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harper</span> <span class="given-names" tagx="given-names" title="given-names">J. F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sze</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> ( <span class="year" tagx="year" title="year">1997</span>). <span class="mixed-article-title" title="mixed-article-title">ECA1 complements yeast mutants defective in Ca <sup>2+</sup> pumps and encodes an endoplasmic reticulum-type Ca <sup>2+</sup>-ATPase in <i>Arabidopsis thaliana</i>. </span> <span class="source" tagx="source" title="source"> <i>Proc. Natl. Acad. Sci. U.S.A.</i> </span> <span class="volume" tagx="volume" title="volume">94</span> <span class="fpage" tagx="fpage" title="fpage">8579</span>– <span class="lpage" tagx="lpage" title="lpage">8584</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.94.16.8579">10.1073/pnas.94.16.8579</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9238019">9238019</a> </span> </span> </li> <li tag="ref"> <a name="B28" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lichtenthaler</span> <span class="given-names" tagx="given-names" title="given-names">H. K.</span> </span> </span> ( <span class="year" tagx="year" title="year">1987</span>). <span class="mixed-article-title" title="mixed-article-title">Chlorophyll fluorescence signatures of leaves during the autumnal chlorophyll breakdown.</span> <span class="source" tagx="source" title="source"> <i>J. Plant Physiol.</i> </span> <span class="volume" tagx="volume" title="volume">131</span> <span class="fpage" tagx="fpage" title="fpage">101</span>– <span class="lpage" tagx="lpage" title="lpage">110</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0176-1617(87)80271-7">10.1016/S0176-1617(87)80271-7</a> </span> </span> </li> <li tag="ref"> <a name="B29" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Q. X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jindra</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ueda</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hiromi</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirose</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Drosophila MBF1 is a co-activator for tracheae defective and contributes to the formation of tracheal and nervous systems. </span> <span class="source" tagx="source" title="source"> <i>Development</i> </span> <span class="volume" tagx="volume" title="volume">130</span> <span class="fpage" tagx="fpage" title="fpage">719</span>– <span class="lpage" tagx="lpage" title="lpage">728</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1242/dev.00297">10.1242/dev.00297</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12506002">12506002</a> </span> </span> </li> <li tag="ref"> <a name="B30" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Q. X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakashima-Kamimura</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ikeo</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirose</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gojobori</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Compensatory change of interacting amino acids in the coevolution of transcriptional coactivator MBF1 and TATA-box-binding protein. </span> <span class="source" tagx="source" title="source"> <i>Mol. Biol. Evol.</i> </span> <span class="volume" tagx="volume" title="volume">24</span> <span class="fpage" tagx="fpage" title="fpage">1458</span>– <span class="lpage" tagx="lpage" title="lpage">1463</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msm073">10.1093/molbev/msm073</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17440176">17440176</a> </span> </span> </li> <li tag="ref"> <a name="B31" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Livak</span> <span class="given-names" tagx="given-names" title="given-names">K. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmittgen</span> <span class="given-names" tagx="given-names" title="given-names">T. D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">Analysis of relative gene expression data using real-time quantitative PCR and the 2-ΔΔCT method. </span> <span class="source" tagx="source" title="source"> <i>Methods</i> </span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">402</span>– <span class="lpage" tagx="lpage" title="lpage">408</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1006/meth.2001.1262">10.1006/meth.2001.1262</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11846609">11846609</a> </span> </span> </li> <li tag="ref"> <a name="B32" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">T.-W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hopkins</span> <span class="given-names" tagx="given-names" title="given-names">M. T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peterson</span> <span class="given-names" tagx="given-names" title="given-names">C. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">J. E.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Arabidopsis eIF5A3 influences growth and the response to osmotic and nutrient stress.</span> <span class="source" tagx="source" title="source"> <i>Plant Cell Environ.</i> </span> <span class="volume" tagx="volume" title="volume">33</span> <span class="fpage" tagx="fpage" title="fpage">1682</span>– <span class="lpage" tagx="lpage" title="lpage">1696</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-3040.2010.02173.x">10.1111/j.1365-3040.2010.02173.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20492553">20492553</a> </span> </span> </li> <li tag="ref"> <a name="B33" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mi</span> <span class="given-names" tagx="given-names" title="given-names">H. Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Muruganujan</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Casagrande</span> <span class="given-names" tagx="given-names" title="given-names">J. T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thomas</span> <span class="given-names" tagx="given-names" title="given-names">P. D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Large-scale gene function analysis with the PANTHER classification system.</span> <span class="source" tagx="source" title="source"> <i>Nat. Protoc.</i> </span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">1551</span>– <span class="lpage" tagx="lpage" title="lpage">1566</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nprot.2013.092">10.1038/nprot.2013.092</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23868073">23868073</a> </span> </span> </li> <li tag="ref"> <a name="B34" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mi</span> <span class="given-names" tagx="given-names" title="given-names">H. Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Poudel</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Muruganujan</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Casagrande</span> <span class="given-names" tagx="given-names" title="given-names">J. T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thomas</span> <span class="given-names" tagx="given-names" title="given-names">P. D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">PANTHER version 10: expanded protein families and functions, and analysis tools.</span> <span class="source" tagx="source" title="source"> <i>Nucleic Acids Res.</i> </span> <span class="volume" tagx="volume" title="volume">44</span> <span class="fpage" tagx="fpage" title="fpage">D336</span>– <span class="lpage" tagx="lpage" title="lpage">D342</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkv1194">10.1093/nar/gkv1194</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26578592">26578592</a> </span> </span> </li> <li tag="ref"> <a name="B35" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mortazavi</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williams</span> <span class="given-names" tagx="given-names" title="given-names">B. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mccue</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schaeffer</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wold</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Mapping and quantifying mammalian transcriptomes by RNA-Seq.</span> <span class="source" tagx="source" title="source"> <i>Nat. Methods</i> </span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">621</span>– <span class="lpage" tagx="lpage" title="lpage">628</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth.1226">10.1038/nmeth.1226</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18516045">18516045</a> </span> </span> </li> <li tag="ref"> <a name="B36" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakabayashi</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yonekura-Sakakibara</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Urano</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suzuki</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamada</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nishizawa</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Enhancement of oxidative and drought tolerance in Arabidopsis by overaccumulation of antioxidant flavonoids. </span> <span class="source" tagx="source" title="source"> <i>Plant J.</i> </span> <span class="volume" tagx="volume" title="volume">77</span> <span class="fpage" tagx="fpage" title="fpage">367</span>– <span class="lpage" tagx="lpage" title="lpage">379</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/tpj.12388">10.1111/tpj.12388</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24274116">24274116</a> </span> </span> </li> <li tag="ref"> <a name="B37" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neff</span> <span class="given-names" tagx="given-names" title="given-names">M. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chory</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">1998</span>). <span class="mixed-article-title" title="mixed-article-title">Genetic interactions between phytochrome A, phytochrome B, and cryptochrome 1 during Arabidopsis development. </span> <span class="source" tagx="source" title="source"> <i>Plant Physiol.</i> </span> <span class="volume" tagx="volume" title="volume">118</span> <span class="fpage" tagx="fpage" title="fpage">27</span>– <span class="lpage" tagx="lpage" title="lpage">36</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.118.1.27">10.1104/pp.118.1.27</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9733523">9733523</a> </span> </span> </li> <li tag="ref"> <a name="B38" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oh</span> <span class="given-names" tagx="given-names" title="given-names">J. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">Y. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">J. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kwon</span> <span class="given-names" tagx="given-names" title="given-names">Y. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Enhanced level of anthocyanin leads to increased salt tolerance in Arabidopsis <i>PAP1-D</i> plants upon sucrose treatment. </span> <span class="source" tagx="source" title="source"> <i>J. Korean Soc. Appl. Biol. Chem.</i> </span> <span class="volume" tagx="volume" title="volume">54</span> <span class="fpage" tagx="fpage" title="fpage">79</span>– <span class="lpage" tagx="lpage" title="lpage">88</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.3839/jksabc.2011.011">10.3839/jksabc.2011.011</a> </span> </span> </li> <li tag="ref"> <a name="B39" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qin</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Geng</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yao</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ni</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Overexpression of heat stress-responsive TaMBF1c, a wheat ( <i>Triticum aestivum</i> L.) Multiprotein Bridging Factor, confers heat tolerance in both yeast and rice. </span> <span class="source" tagx="source" title="source"> <i>Plant Mol. Biol.</i> </span> <span class="volume" tagx="volume" title="volume">87</span> <span class="fpage" tagx="fpage" title="fpage">31</span>– <span class="lpage" tagx="lpage" title="lpage">45</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11103-014-0259-9">10.1007/s11103-014-0259-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25326264">25326264</a> </span> </span> </li> <li tag="ref"> <a name="B40" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rizhsky</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liang</span> <span class="given-names" tagx="given-names" title="given-names">H. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mittler</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">The combined effect of drought stress and heat shock on gene expression in tobacco.</span> <span class="source" tagx="source" title="source"> <i>Plant Physiol.</i> </span> <span class="volume" tagx="volume" title="volume">130</span> <span class="fpage" tagx="fpage" title="fpage">1143</span>– <span class="lpage" tagx="lpage" title="lpage">1151</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.006858">10.1104/pp.006858</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12427981">12427981</a> </span> </span> </li> <li tag="ref"> <a name="B41" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmidt</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dethloff</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beine-Golovchuk</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kopka</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">The REIL1 and REIL2 proteins of <i>Arabidopsis thaliana</i> are required for leaf growth in the cold. </span> <span class="source" tagx="source" title="source"> <i>Plant Physiol.</i> </span> <span class="volume" tagx="volume" title="volume">163</span> <span class="fpage" tagx="fpage" title="fpage">1623</span>– <span class="lpage" tagx="lpage" title="lpage">1639</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.113.223925">10.1104/pp.113.223925</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24038679">24038679</a> </span> </span> </li> <li tag="ref"> <a name="B42" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suzuki</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rizhsky</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liang</span> <span class="given-names" tagx="given-names" title="given-names">H. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shuman</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shulaev</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mittler</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Enhanced tolerance to environmental stress in transgenic plants expressing the transcriptional coactivator multiprotein bridging factor 1c. </span> <span class="source" tagx="source" title="source"> <i>Plant Physiol.</i> </span> <span class="volume" tagx="volume" title="volume">139</span> <span class="fpage" tagx="fpage" title="fpage">1313</span>– <span class="lpage" tagx="lpage" title="lpage">1322</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.105.070110">10.1104/pp.105.070110</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16244138">16244138</a> </span> </span> </li> <li tag="ref"> <a name="B43" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suzuki</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sejima</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tam</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schlauch</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mittler</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Identification of the MBF1 heat-response regulon of <i>Arabidopsis thaliana</i>. </span> <span class="source" tagx="source" title="source"> <i>Plant J.</i> </span> <span class="volume" tagx="volume" title="volume">66</span> <span class="fpage" tagx="fpage" title="fpage">844</span>– <span class="lpage" tagx="lpage" title="lpage">851</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2011.04550.x">10.1111/j.1365-313X.2011.04550.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21457365">21457365</a> </span> </span> </li> <li tag="ref"> <a name="B44" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taji</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seki</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Satou</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakurai</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kobayashi</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishiyama</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Comparative genomics in salt tolerance between Arabidopsis and Arabidopsis-related halophyte salt cress using Arabidopsis microarray. </span> <span class="source" tagx="source" title="source"> <i>Plant Physiol.</i> </span> <span class="volume" tagx="volume" title="volume">135</span> <span class="fpage" tagx="fpage" title="fpage">1697</span>– <span class="lpage" tagx="lpage" title="lpage">1709</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.104.039909">10.1104/pp.104.039909</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15247402">15247402</a> </span> </span> </li> <li tag="ref"> <a name="B45" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takemaru</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">F. Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ueda</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirose</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> ( <span class="year" tagx="year" title="year">1997</span>). <span class="mixed-article-title" title="mixed-article-title">Multiprotein bridging factor 1 (MBF1) is an evolutionarily conserved transcriptional coactivator that connects a regulatory factor and TATA element-binding protein. </span> <span class="source" tagx="source" title="source"> <i>Proc. Natl. Acad. Sci. U.S.A.</i> </span> <span class="volume" tagx="volume" title="volume">94</span> <span class="fpage" tagx="fpage" title="fpage">7251</span>– <span class="lpage" tagx="lpage" title="lpage">7256</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.94.14.7251">10.1073/pnas.94.14.7251</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9207077">9207077</a> </span> </span> </li> <li tag="ref"> <a name="B46" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">J. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Higgins</span> <span class="given-names" tagx="given-names" title="given-names">D. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gibson</span> <span class="given-names" tagx="given-names" title="given-names">T. J.</span> </span> </span> ( <span class="year" tagx="year" title="year">1994</span>). <span class="mixed-article-title" title="mixed-article-title">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. </span> <span class="source" tagx="source" title="source"> <i>Nucleic Acids Res.</i> </span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">4673</span>– <span class="lpage" tagx="lpage" title="lpage">4680</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/22.22.4673">10.1093/nar/22.22.4673</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7984417">7984417</a> </span> </span> </li> <li tag="ref"> <a name="B47" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tiwari</span> <span class="given-names" tagx="given-names" title="given-names">B. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Belenghi</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Levine</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Oxidative stress increased respiration and generation of reactive oxygen species, resulting in ATP depletion, opening of mitochondrial permeability transition, and programmed cell death. </span> <span class="source" tagx="source" title="source"> <i>Plant Physiol.</i> </span> <span class="volume" tagx="volume" title="volume">128</span> <span class="fpage" tagx="fpage" title="fpage">1271</span>– <span class="lpage" tagx="lpage" title="lpage">1281</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.010999">10.1104/pp.010999</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11950976">11950976</a> </span> </span> </li> <li tag="ref"> <a name="B48" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tojo</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsuda</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoshizumi</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ikeda</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamaguchi</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matsui</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Arabidopsis MBF1s control leaf cell cycle and its expansion.</span> <span class="source" tagx="source" title="source"> <i>Plant Cell Physiol.</i> </span> <span class="volume" tagx="volume" title="volume">50</span> <span class="fpage" tagx="fpage" title="fpage">254</span>– <span class="lpage" tagx="lpage" title="lpage">264</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/pcp/pcn187">10.1093/pcp/pcn187</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19050034">19050034</a> </span> </span> </li> <li tag="ref"> <a name="B49" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsuda</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsuji</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirose</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamazaki</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Three Arabidopsis MBF1 homologs with distinct expression profiles play roles as transcriptional co-activators. </span> <span class="source" tagx="source" title="source"> <i>Plant Cell Physiol.</i> </span> <span class="volume" tagx="volume" title="volume">45</span> <span class="fpage" tagx="fpage" title="fpage">225</span>– <span class="lpage" tagx="lpage" title="lpage">231</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/pcp/pch017">10.1093/pcp/pch017</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14988493">14988493</a> </span> </span> </li> <li tag="ref"> <a name="B50" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsuda</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamazaki</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Structure and expression analysis of three subtypes of Arabidopsis MBF1 genes.</span> <span class="source" tagx="source" title="source"> <i>Biochim. Biophys. Acta</i> </span> <span class="volume" tagx="volume" title="volume">1680</span> <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">10</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bbaexp.2004.08.004">10.1016/j.bbaexp.2004.08.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15451167">15451167</a> </span> </span> </li> <li tag="ref"> <a name="B51" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turetsky</span> <span class="given-names" tagx="given-names" title="given-names">M. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bond-Lamberty</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Euskirchen</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Talbot</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frolking</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mcguire</span> <span class="given-names" tagx="given-names" title="given-names">A. D.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">The resilience and functional role of moss in boreal and arctic ecosystems.</span> <span class="source" tagx="source" title="source"> <i>New Phytol.</i> </span> <span class="volume" tagx="volume" title="volume">196</span> <span class="fpage" tagx="fpage" title="fpage">49</span>– <span class="lpage" tagx="lpage" title="lpage">67</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1469-8137.2012.04254.x">10.1111/j.1469-8137.2012.04254.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22924403">22924403</a> </span> </span> </li> <li tag="ref"> <a name="B52" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Victoria</span> <span class="given-names" tagx="given-names" title="given-names">F. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De Albuquerque</span> <span class="given-names" tagx="given-names" title="given-names">M. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pereira</span> <span class="given-names" tagx="given-names" title="given-names">A. B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simas</span> <span class="given-names" tagx="given-names" title="given-names">F. N. B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Spielmann</span> <span class="given-names" tagx="given-names" title="given-names">A. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schaefer</span> <span class="given-names" tagx="given-names" title="given-names">C. E. G. R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). Characterization and mapping of plant communities at Hennequin Point, King George Island, Antarctica. <i>Polar Res.</i> <span class="volume" tagx="volume" title="volume">32</span> <span class="issue" tagx="issue" title="issue">19261</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3402/polar.V32i0.19261">10.3402/polar.V32i0.19261</a> </span> </span> </li> <li tag="ref"> <a name="B53" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liang</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hong</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Young</span> <span class="given-names" tagx="given-names" title="given-names">J. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sussman</span> <span class="given-names" tagx="given-names" title="given-names">M. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harper</span> <span class="given-names" tagx="given-names" title="given-names">J. F.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">An endoplasmic reticulum-bound Ca <sup>2+</sup>/Mn <sup>2+</sup> pump, ECA1, supports plant growth and confers tolerance to Mn <sup>2+</sup> Stress. </span> <span class="source" tagx="source" title="source"> <i>Plant Physiol.</i> </span> <span class="volume" tagx="volume" title="volume">130</span> <span class="fpage" tagx="fpage" title="fpage">128</span>– <span class="lpage" tagx="lpage" title="lpage">137</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.004440">10.1104/pp.004440</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12226493">12226493</a> </span> </span> </li> <li tag="ref"> <a name="B54" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ming</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">RceIF5A, encoding an eukaryotic translation initiation factor 5A in <i>Rosa chinensis</i>, can enhance thermotolerance, oxidative and osmotic stress resistance of <i>Arabidopsis thaliana</i>. </span> <span class="source" tagx="source" title="source"> <i>Plant Mol. Biol.</i> </span> <span class="volume" tagx="volume" title="volume">75</span> <span class="fpage" tagx="fpage" title="fpage">167</span>– <span class="lpage" tagx="lpage" title="lpage">178</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11103-010-9716-2">10.1007/s11103-010-9716-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21107886">21107886</a> </span> </span> </li> <li tag="ref"> <a name="B55" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yan</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hou</span> <span class="given-names" tagx="given-names" title="given-names">H. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singer</span> <span class="given-names" tagx="given-names" title="given-names">S. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yan</span> <span class="given-names" tagx="given-names" title="given-names">X. X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">R. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">X. P.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">The grape VvMBF1 gene improves drought stress tolerance in transgenic <i>Arabidopsis thaliana</i>. </span> <span class="source" tagx="source" title="source"> <i>Plant Cell Tissue Organ Cult.</i> </span> <span class="volume" tagx="volume" title="volume">118</span> <span class="fpage" tagx="fpage" title="fpage">571</span>– <span class="lpage" tagx="lpage" title="lpage">582</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11240-014-0508-2">10.1007/s11240-014-0508-2</a> </span> </span> </li> <li tag="ref"> <a name="B56" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zuniga-Gonzalez</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zuniga</span> <span class="given-names" tagx="given-names" title="given-names">G. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pizarro</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Casanova-Katny</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Soluble carbohydrate content variation in <i>Sanionia uncinata</i> and <i>Polytrichastrum alpinum</i>, two Antarctic mosses with contrasting desiccation capacities. </span> <span class="source" tagx="source" title="source"> <i>Biol. Res.</i> </span> <span class="volume" tagx="volume" title="volume">49</span>: <span class="issue" tagx="issue" title="issue">6</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s40659-015-0058-z">10.1186/s40659-015-0058-z</a> </span> </span> </li> </ul> </div> </div>  </body></html>