<?xml version="1.0" encoding="UTF-8"?><html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
   <div class="sec" title="sec">
      <div xmlns="http://www.w3.org/1999/xhtml">Glycosylation site variation between African and Asian strains</div>
      <p xmlns="http://www.w3.org/1999/xhtml">Deletions in a potential glycosylation site of several strains were observed following
         their alignment (
         <a>Figure 2</a>). A 4-codon deletion was observed beginning at amino acid position 153 of the E protein
         of the MR766 strain (GenBank accession number AY632535), a 6-codon deletion at position
         156 of another MR 766 strain (GenBank accession number DQ859059), and a 6-codon deletion
         at position 156 of the IbH 30656 strain sequenced in this study. The MR 766 strain
         (Passage 147) sequenced here did not exhibit any deletions in the predicted amino
         acid sequence and provided evidence of passage-associated changes in potential glycosylation
         site(s).
         
      </p>
      <div xmlns="http://www.w3.org/1999/xhtml"><span>10.1371/journal.pntd.0001477.g002</span><span>Figure 2</span><div>
            <div>Zika virus phylogeny.</div>
            <p>Nucleotide (top) and amino acid (bottom) sequences of the envelope protein/gene of
               Zika virus strains showing the deletions in the potential glycosylation sites of the
               MR 766 (Uganda, 1947) and the IbH 30656 (Nigeria, 1968) strains. Deletions are indicated
               by dashes. The “N” at position 467 of the P6-740 strain (Malaysia, 1966) represents
               an equally weighted double population of the nucleotides “C” and “T”. This translates
               to an “X” at position 165 of the amino acid alignment.
            </p>
         </div>
         <div></div>
      </div>
   </div>
</html>