<?xml version="1.0" encoding="UTF-8"?><html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
   <div class="sec" title="sec">
      <div xmlns="http://www.w3.org/1999/xhtml">Phlyogenetic analyses</div>
      <p xmlns="http://www.w3.org/1999/xhtml">All three methods of phylogenetic inference (NJ, ML and MP) identified two major lineages
         (African and Asian) (
         <a>Figure 1</a>). The most recent common ancestor of MR 766 (Uganda, 1947) diverged first, followed
         by the divergence of the most recent common ancestor of the ArD 41519 (Senegal, 1984)
         and IbH 30656 (Nigeria, 1968) strains, the P6-740 strain (Malaysia, 1966), and lastly
         the EC Yap (Micronesia, 2007) and FS13025 (Cambodia, 2010) strains.
         
      </p>
      <div xmlns="http://www.w3.org/1999/xhtml"><span>10.1371/journal.pntd.0001477.g001</span><span>Figure 1</span><div>
            <div>Zika virus nucleotide and amino acid alignments.</div>
            <p>Neighbor-joining phylogeny generated from open reading frame nucleotide sequences
               of Zika virus strains. The tree was rooted with Spondweni virus (GenBank accession
               number DQ859064). The scale at the bottom of the tree represents genetic distance
               in nucleotide substitutions per site. Numbers at the nodes represent percent bootstrap
               support values based on 1,000 replicates. Isolates are represented according to strain
               name, country of origin, and year of isolation. The lineage of each virus is indicated
               to the right of the tree. *Strains sequenced in this study.
            </p>
         </div>
         <div></div>
      </div>
   </div>
</html>