<?xml version="1.0" encoding="UTF-8"?><html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
   <div class="sec" title="sec">
      <div xmlns="http://www.w3.org/1999/xhtml">Genetic and phylogenetic analyses</div>
      <p xmlns="http://www.w3.org/1999/xhtml">Nucleotide sequences derived from the five ZIKV strains were assembled and aligned
         with three other sequences of ZIKV and one sequence of Spondweni virus that were retrieved
         from GenBank using the Vector NTI Suite (Invitrogen, USA) (
         <a>Table 2</a>). The two GenBank sequences of the MR766 strain exhibited considerable nucleotide
         and amino acid variation; therefore we resequenced this strain (
         <a>Table 2</a>). Phylogenies were generated using neighbor-joining (NJ), maximum-likelihood (ML)
         and maximum-parsimony (MP) methods using the default settings implemented in the PHYLIP
         package 
         <a>[18]</a>. The Spondweni virus strain SM-6 V-1 was used as the outgroup for all phylogenies,
         as Spondweni virus is the most closely related flavivirus (antigenically and genetically)
         to ZIKV 
         <a>[1]</a>, 
         <a>[19]</a>, 
         <a>[20]</a>, 
         <a>[21]</a>, 
         <a>[22]</a>. Robustness of the phylogenies was evaluated by resampling with 1,000 bootstrap replicates
         and horizontal branches were scaled according to the number of nucleotide substitutions
         per site.
         
      </p>
   </div>
</html>