<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">PLoS One</span> <span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span> <span class="publisher-id" title="publisher-id">plos</span> <span class="pmc" title="pmc">plosone</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, USA</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/3149072">3149072</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21829715">21829715</a> </span> <span class="publisher-id" title="publisher-id">PONE-D-11-04119</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0023167">10.1371/journal.pone.0023167</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span>: <span class="subject" title="subject">Proteins</span> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Science</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Cell Biology</span>: <span class="subject" title="subject">Plant Cell Wall</span> </div>: <span class="subject" title="subject">Plant Evolution</span> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Global Analysis of Proline-Rich Tandem Repeat Proteins Reveals Broad Phylogenetic Diversity in Plant Secretomes </div> <div class="alt-title" title="alt-title">
Plant Pro-Rich Proteins Are Unexpectedly Diverse</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Newman</span> <span class="given-names" tagx="given-names" title="given-names">Aaron M.</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#fn1"> <sup>Â¤</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cooper</span> <span class="given-names" tagx="given-names" title="given-names">James B.</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#aff2"> <sup>2</sup> </a> <a href="#cor1"> <sup>*</sup> </a> </span> </div> <span class="citation_author_institution" id="aff1">[1], <span class="addr-line" title="addr-line">Biomolecular Science and Engineering Program, University of California Santa Barbara, Santa Barbara, California, United States of America </span> </span> <span class="citation_author_institution" id="aff2">[2], <span class="addr-line" title="addr-line">Department of Molecular, Cellular, and Developmental Biology, University of California, Santa Barbara, California, United States of America </span> </span> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ouzounis</span> <span class="given-names" tagx="given-names" title="given-names">Christos A.</span> </span> <span class="role" tagx="role" title="role">Editor</span> <a href="#edit1" /> </span> </div> <span class="citation_author_institution" id="edit1">[], </span> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">jcooper@lifesci.ucsb.edu</span> </div> <div class="fn-type-con" title="con"> <p>Conceived and designed the experiments: AMN JBC. Performed the experiments: AMN JBC. Analyzed the data: AMN JBC. Contributed reagents/materials/analysis tools: AMN. Wrote the paper: AMN JBC. Designed PlantPro20Fam website: AMN. </p> </div> <div class="fn-type-current-aff" title="current-aff"> <span class="label" tagx="label" title="label">Â¤</span> <p>Current address: Neuroscience Research Institute, Center for the Study of Macular Degeneration, University of California, Santa Barbara, California, United States of America </p> </div> </div> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2011</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2011-8-8</span> </span> <span class="volume" tagx="volume" title="volume">6</span> <span class="issue" tagx="issue" title="issue">8</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e23167</span> <span class="history" title="history"> <span class="received" title="received">received: 2011-3-2</span> <span class="accepted" title="accepted">accepted: 2011-7-13</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2011</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Cell walls, constructed by precisely choreographed changes in the plant secretome, play critical roles in plant cell physiology and development. Along with structural polysaccharides, secreted proline-rich Tandem Repeat Proteins (TRPs) are important for cell wall function, yet the evolutionary diversity of these structural TRPs remains virtually unexplored. Using a systems-level computational approach to analyze taxonomically diverse plant sequence data, we identified 31 distinct Pro-rich TRP classes targeted for secretion. This analysis expands upon the known phylogenetic diversity of extensins, the most widely studied class of wall structural proteins, and demonstrates that extensins evolved before plant vascularization. Our results also show that most Pro-rich TRP classes have unexpectedly restricted evolutionary distributions, revealing considerable differences in plant secretome signatures that define unexplored diversity. </p> </div> <div class="counts" title="counts"> <span class="page-count" title="page-count">page-count: </span> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Composed primarily of polysaccharides, plant cell walls provide critical structural support for terrestrial plant life, and play important roles in plant growth, development, and interactions with microbes. In addition, ligno-cellulosic plant cell walls provide the dietary fiber that facilitates mammalian digestion, represent an important source of textile materials and combustible biofuels, and play a major role in the global carbon cycle. Biochemical studies indicate that wall polysaccharide composition differs among plant lineages, and two distinct wall types have been characterized in higher plants to date <a href="#pone.0023167-Sarkar1">[1]</a>, <a href="#pone.0023167-Popper1">[2]</a>. Gymnosperms and most angiosperms are known to have Type I cell walls that have a distinctly different polysaccharide composition from Type II cell walls found only in a taxonomic group of advanced monocots that includes the grasses <a href="#pone.0023167-Vogel1">[3]</a>. Generally missing from current systems-level views of the plant cell wall <a href="#pone.0023167-Sarkar1">[1]</a>, <a href="#pone.0023167-Somerville1">[4]</a>, <a href="#pone.0023167-Cosgrove1">[5]</a>, however, are the secreted structural (glyco)protein elements containing 4-trans-hydroxyproline (Hyp) first discovered more than fifty years ago <a href="#pone.0023167-Dougall1">[6]</a>, <a href="#pone.0023167-Lamport1">[7]</a>. </p> <p>Nearly ubiquitous in green plants, Hyp-rich glycoproteins, commonly known as HRGPs or extensins <a href="#pone.0023167-Lamport2">[8]</a>, <a href="#pone.0023167-Kieliszewski1">[9]</a>, compose up to 10% of the cell wall mass of higher plants <a href="#pone.0023167-Dougall1">[6]</a>, <a href="#pone.0023167-Lamport1">[7]</a>, and have been shown to play critical roles in cell wall structure and function <a href="#pone.0023167-Cooper1">[10]</a>â <a href="#pone.0023167-Gille1">[13]</a>. Post-translationally modified from Pro-rich polypeptides, secreted HRGPs are generally grouped into three broad classes based on primary sequence architectures and glycosylation profiles <a href="#pone.0023167-Kieliszewski1">[9]</a>, <a href="#pone.0023167-Johnson1">[14]</a>â <a href="#pone.0023167-Ellis1">[16]</a>, and these three classes, the extensin glycoproteins, the proline-rich proteins (PRPs), and the highly glycosylated arabinogalactan proteins (AGPs), have been hypothesized to form a phylogenetic continuum ranging from green algae throughout land plants <a href="#pone.0023167-Kieliszewski1">[9]</a>, <a href="#pone.0023167-Lamport3">[17]</a>. </p> <p>Known HRGPs have highly biased amino acid compositions, and like numerous structural proteins found throughout nature, extensins and PRPs also have highly repetitive, tandem repeat (TR), sequence architectures. These sequence characteristics, along with extensive post-translational modifications often leading to insoluble cross-linked HRGP networks, have hindered broad phylogenetic analysis of plant HRGPs. Since commonly applied sequence analysis methods, such as BLAST <a href="#pone.0023167-Altschul1">[18]</a> or Hidden Markov Models <a href="#pone.0023167-Finn1">[19]</a>, have considerable limitations for analyzing sequences with biased amino acid content <a href="#pone.0023167-Showalter1">[15]</a>, several groups have used simple compositional filters or regular expression queries (based on previously characterized sequence motifs) to identify Pro-rich proteins from plant sequence databases. For example, Schultz et al. (2002) <a href="#pone.0023167-Schultz1">[20]</a> and Ma and Zhao (2010) <a href="#pone.0023167-Ma1">[21]</a> identified AGPs in <i>Arabidopsis</i> and rice, respectively, using a biased amino acid filter (50â55% or 35% Pro/Ala/Ser/Thr composition depending on protein length), and Graham et al. (2004) <a href="#pone.0023167-Graham1">[22]</a> used the pattern, PPV(E/Y/V)K, to identify novel PRPs in legumes. Showalter et al. (2010) <a href="#pone.0023167-Showalter1">[15]</a> developed BIO OHIO, a software tool that implements a variety of compositional and regular expression filters, to identify previously defined HRGP groups in the <i>Arabidopsis</i> genome. Unfortunately, these approaches all rely upon prior knowledge of repeat patterns and/or composition, limiting their utility for analyzing novel Pro-rich TRPs on a global multi-genomic scale. </p> <p>In this study, we leveraged computational methods specifically developed to analyze large sequence databases for TR and TRP content without prior knowledge, and report a considerable expansion in our knowledge of the number and diversity of Pro-rich TRP sequence classes targeted to plant secretomes. In contrast to previous analyses that have been limited to a handful of higher plant species (e.g., <a href="#pone.0023167-Showalter1">[15]</a>, <a href="#pone.0023167-Graham1">[22]</a>), we analyzed millions of plant secretome sequences spanning a broad phylogenetic range. Based on the inherent differences in primary sequence architecture among secreted Pro-rich TRPs, we propose a new taxonomy and nomenclature for 31 distinct classes of secreted Pro-rich TRPs. In addition to furthering our understanding of the phylogenetic distributions of canonical HRGP TRPs, our data indicate that nearly half of the identified TRP classes have very narrow phylogenetic distributions. Such diverse Pro-rich structural TRPs reveal phylogenetic distinctions that define an abundance of previously unrecognized secretome signatures. </p> </div> <div class="resultsanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
Results and Discussion</div> <div class="identificationandclassificationofpro-richtrpsinplantsecretomes" title="sec"> <div class="title" tagx="title" title="title">
Identification and classification of Pro-Rich TRPs in plant secretomes</div> <p>To explore the global diversity of Pro-rich TRPs, we constructed a database containing 8.3 M full and partial protein sequences, from genome and EST projects, spanning 36,815 plant and green algal species ( <a href="#pone.0023167.s011">Table S1</a>; note that to expand sequence diversity, most of data analyzed in this work are plant EST sequences). These data were analyzed for TR content using XTREAM (Newman and Cooper, 2007), and â¼210,000 Pro-rich TR motifs were identified, of which â¼90,000 are unique ( <a href="#pone.0023167.s012">Table S2</a>, footnoted). All TR-containing sequences with an N-terminal Met were analyzed for the presence of a secretory signal peptide using SignalP 3.0 <a href="#pone.0023167-Emanuelsson1">[23]</a> ( <a href="#pone.0023167.s012">Table S2</a>). From these sequences, unique TR motifs that compose protein domains of at least 100 amino acids, or 50 amino acids and at least 1/3 of the protein length ( <a href="#pone.0023167.s012">Table S2</a>), were clustered without prior knowledge into TR classes based on sequence similarity. Though necessarily arbitrary, these TR length criteria ensured that all analyzed TRs are sufficiently long to compose significant structural domains in their corresponding proteins. The results of this cluster analysis are presented in <a href="#pone-0023167-g001">Figure 1</a> as a network diagram depicting the landscape of the most abundant Pro-rich TRs in plant secretomes, where nodes correspond to TR motif clusters and edges reflect sequence similarity. A high-resolution view of this network, together with the individual TR motif consensus sequences, is shown in <a href="#pone.0023167.s001">Figure S1</a>. Using this network, we identified 38 distinct TR motif classes ( <a href="#pone.0023167.s013">Table S3</a>, <a href="#pone.0023167.s014">Table S4</a>, and <a href="#pone.0023167.s015">Table S5</a>) that define 31 unique classes of secreted Pro-rich TRPs ( <a href="#pone.0023167.s016">Table S6</a>, <a href="#pone.0023167.s017">Table S7</a>, and <a href="#pone.0023167.s018">Table S8</a>; also see <a href="#pone.0023167.s024">Text S1</a>, <a href="#pone.0023167.s025">Text S2</a>, and <a href="#pone.0023167.s026">Text S3</a>). Most of these TRP classes map to one of five large TR motif super-classes ( <a href="#pone-0023167-g001">Figures 1</a> and <a href="#pone.0023167.s002">S2</a>), and are distinctly different in TR sequence architecture from previously recognized HRGP groups <a href="#pone.0023167-Johnson1">[14]</a>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0023167.g001"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <div class="title" tagx="title" title="title">
Cluster landscape of Pro-rich TR motifs from plant secretome sequences.</div> <p>Each node represents a TR cluster, node labels denote the original cluster identifier (see <a href="#pone.0023167.s019">Tables S9</a>, <a href="#pone.0023167.s020">S10</a>), and edge thickness represents the fraction of times each pair of TR clusters was co-clustered over ensemble re-sampling (see <a href="#s3"> <i>Materials and Methods</i> </a>) (also see "pairwise affinity" defined in Figure 4 of <a href="#pone.0023167-Newman2">[49]</a>). Thin, dotted, edges indicate a co-clustering of &amp;lt;10%. Large labeled nodes in the network denote clusters containing secreted TRPs found in at least ten species and twenty protein sequences ( <a href="#pone.0023167.s020">Table S10</a>) while intermediate size labeled nodes satisfy one of these two criteria. Smaller unlabeled nodes do not meet either criterion, but are shown due to their similarity in motif content to larger, neighboring nodes. Major TRP classes from <a href="#pone.0023167.s016">Tables S6</a>, <a href="#pone.0023167.s017">S7</a> and <a href="#pone.0023167.s018">S8</a> are indicated around corresponding TR motif super-classes (circled in gray). Node color represents the retention rate of the TR taxonomy ( <a href="#pone.0023167.s013">Tables S3</a>, <a href="#pone.0023167.s014">S4</a>, <a href="#pone.0023167.s015">S5</a>), defined as the proportion of all protein sequences corresponding to each cluster that are captured by TR class definitions (for class definitions, see <a href="#pone.0023167.s021">Table S11</a>; for a quantitative version of the taxonomy retention rate, see <a href="#pone.0023167.s020">Table S10</a>). For a high-resolution version of this network showing all individual TR consensus sequences, see <a href="#pone.0023167.s001">Figure S1</a>, and see <a href="#pone.0023167.s002">Figure S2</a> for the same high-resolution network also showing TR super-classes. For details of network construction, see <a href="#s3"> <i>Materials and Methods</i> </a>. This network was rendered using Cytoscape 2.6.0 <a href="#pone.0023167-Shannon1">[50]</a>. </p>   </div> <p>To illustrate the sequence architectures of these TRP classes, representative examples of aligned TR domains are shown in <a href="#pone.0023167.s028">Dataset S1</a>, and multiple sequence alignments of the N-terminal and C-terminal regions of selected TRP classes are shown in <a href="#pone.0023167.s029">Dataset S2</a> and <a href="#pone.0023167.s003">Figure S3</a>. Aside from two heterogeneous classes (called EXTM and SPAP), all of the TRP classes we identified are highly conserved in sequence both within and outside of the Pro-rich TR domain, generally including the predicted N-terminal signal peptide ( <a href="#pone.0023167.s029">Dataset S2</a> and <a href="#pone.0023167.s003">Figure S3</a>). Complete data outputs of the TR motif cluster analysis are available in the supporting information online ( <a href="#pone.0023167.s019">Table S9</a> and <a href="#pone.0023167.s020">Table S10</a>), and all identified TRP sequences, including the TR motifs corresponding to each of the 38 TR classes, are available via a web database called PlantPro20Fam ( <a href="http://jimcooperlab.mcdb.ucsb.edu/plantpro20fam">http://jimcooperlab.mcdb.ucsb.edu/plantpro20fam</a>). </p> </div> <div class="extensins:" title="sec"> <div class="title" tagx="title" title="title">
Extensins: "extensively" distributed in the embryophytes</div> <p>Extensins, the first widely studied class of wall HRGPs, are generally defined as highly basic glycoproteins composed of canonical Ser-Pro <sub>4</sub> TR motifs, in which nearly every Pro residue is 4-trans-hydroxylated and O-arabinosylated <a href="#pone.0023167-Kieliszewski1">[9]</a>, <a href="#pone.0023167-Johnson1">[14]</a>, <a href="#pone.0023167-Lamport3">[17]</a>. In addition, extensin TR motifs are known to contain isodityrosine (Idt) crosslinking motifs: either YxY and/or VYK <a href="#pone.0023167-Epstein1">[24]</a>, <a href="#pone.0023167-Schnabelrauch1">[25]</a>. Previous work found Hyp-Arabinosides to be ubiquitous in green plants <a href="#pone.0023167-Lamport2">[8]</a>. However, due to the difficulty in characterizing full-length extensin glycoproteins, the evolutionary origins and distribution of these wall structural molecules has remained obscure <a href="#pone.0023167-Kieliszewski1">[9]</a>, <a href="#pone.0023167-Johnson1">[14]</a>, <a href="#pone.0023167-Schnabelrauch1">[25]</a>. Our global analysis identified YxY-containing extensins, referred to as Extensin type alpha (EXTA) ( <a href="#pone.0023167.s024">Text S1</a>), throughout most land plant divisions ( <a href="#pone-0023167-g002">Figures 2A</a>, <a href="#pone-0023167-g003">3A</a>, and <a href="#pone.0023167.s004">Figure S4</a>). In addition to the core eudicots and ferns <a href="#pone.0023167-Johnson1">[14]</a>, <a href="#pone.0023167-Schnabelrauch1">[25]</a>, <a href="#pone.0023167-Uchida1">[26]</a>, EXTAs are found in at least some non-grass monocots, including onion, orchids, and asparagus, and in a diversity of non-flowering vascular plants, ranging from gymnosperms to <i>Selaginella</i>, a member of the oldest extant vascular plant group <a href="#pone.0023167-Banks1">[27]</a>. Furthermore, although missing in mosses, transcripts encoding EXTAs were found in the non-vascular liverwort species <i>Marchantia polymorpha</i> demonstrating that EXTAs predate the evolution of plant vasculature ( <a href="#pone.0023167.s026">Text S3</a>, also see PlantPro20Fam). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0023167.g002"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <div class="title" tagx="title" title="title">
Phylogenetic distribution of Pro-rich TRP classes targeted to plant secretomes.</div> <p>(A) Dendrogram showing the evolutionary relationships among major plant divisions as well as the distribution of EXTAs identified in this study and in previous literature <a href="#pone.0023167-Johnson1">[14]</a>, <a href="#pone.0023167-Schnabelrauch1">[25]</a>, <a href="#pone.0023167-Uchida1">[26]</a>. (To reflect results of a recent phylogenetic analysis <a href="#pone.0023167-Moore1">[52]</a>, the large divisions, super-rosids and super-asterids are shown rather than rosids, asterids, and other phylogenetic groups). (B) Heat map showing phylogenetic distributions of 18 secreted Pro-rich TRP classes, 17 of which are represented by more than a single plant family (abbreviated names are described in <a href="#pone.0023167.s025">Text S2</a>, and <a href="#pone.0023167.s016">Tables S6</a>, <a href="#pone.0023167.s017">S7</a>, <a href="#pone.0023167.s018">S8</a>). TRP classes are divided into the <i>extensin superfamily</i>, the non-extensin <i>core modules</i>, and the less conserved TRP classes found in <i>&amp;gt;1 plant family</i>. Dark-green rectangles represent TRP classes in which at least one known protein sequence or full-length ORF with a predicted secretion signal was found in the corresponding TRP class and plant phylogenetic group. Light-green rectangles with a horizontal line pattern represent TRP classes in which at least one <i>putative</i> member is present in the corresponding phylogenetic group (e.g. lacking a full-length sequence). Gray rectangles with a diagonal line pattern represent the <i>putative</i> moss AGPA and EXTB sequences (the AGPA is not predicted to be GPI-anchored; the EXTB sequence is not predicted to be secreted and has a TR domain that encompasses only half of the sequence). (C) Phylogenetic sampling bias of all ESTs and protein sequences (from <a href="#pone.0023167.s011">Table S1</a>) either captured by our TR taxonomy ( <a href="#pone.0023167.s013">Tables S3</a>, <a href="#pone.0023167.s014">S4</a>, <a href="#pone.0023167.s015">S5</a>) or representing secreted Pro-rich TRPs ( <a href="#pone.0023167.s012">Table S2</a>), shown as a log-scale histogram. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0023167.g003"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <div class="title" tagx="title" title="title">
Phylogenetic distributions of three core module plant Pro-rich TRP classes.</div> <p>(AâC) Circular trees show genera distributions, with major plant divisions (same as <a href="#pone-0023167-g002">Figure 2A</a>) labeled around each tree. Versions of trees showing all genera are illustrated in the <a href="#pone.0023167.s004">Figures S4</a> (EXTA), <a href="#pone.0023167.s007">S7</a> (AGPA), and <a href="#pone.0023167.s008">S8</a> (PELPK). The 143 leaves in these trees represent the genera that are either captured by our TR taxonomy ( <a href="#pone.0023167.s013">Tables S3</a>, <a href="#pone.0023167.s014">S4</a>, <a href="#pone.0023167.s015">S5</a>) or represented by at least one secreted Pro-rich TRP in our analysis ( <a href="#pone.0023167.s012">Table S2</a>). Taxonomic data were downloaded from NCBI, and trees were rendered using PhyloWidget <a href="#pone.0023167-Jordan1">[53]</a>. </p>   </div> <p>While green algae are known to secrete HRGPs <a href="#pone.0023167-Lee1">[28]</a>, we found no evidence for EXTAs in Chlorophytes. Rather, we identified a wide variety of high molecular weight, low-complexity SP <sub>n</sub>-containing proteins with highly heterogeneous architectures generally lacking Tyr (see PlantPro20Fam), including the SPAP <sub>2</sub> containing mating-type agglutinins of <i>Chlamydomonas</i> <a href="#pone.0023167-Lee1">[28]</a> ( <a href="#pone.0023167.s016">Table S6</a>). These algal Pro-rich TRs generally compose limited sequence domains within large multi-domain non-TRPs, consistent with the hypothesis that the extensin class of HRGPs evolved in plants during the colonization of terrestrial ecosystems. </p> <p>Given the widespread occurrence of EXTAs in land plant secretomes, we next examined EXTA TR domains for evidence of structural conservation. We found that TR periodicities in EXTAs are highly conserved, with periods of 10 and 16 residues occurring in nearly two-thirds of EXTA TRs ( <a href="#pone.0023167.s005">Figure S5</a>). In addition, at least one of these two periodicities occurs in every major plant lineage containing an EXTA sequence ( <a href="#pone-0023167-g002">Figure 2A</a>). Circular dichroism data previously showed that carrot EXTA forms an extended polyproline II (PPII) helix and appears rod-like when imaged by electron microscopy <a href="#pone.0023167-vanHolst1">[29]</a>. Because of the trilateral symmetry of PPII helices, TR periodicity will determine the topological regularity of amino acid side groups along an extended PPII helix <a href="#pone.0023167-Williamson1">[30]</a>. Both 10 mers and 16 mers have periodicities of length <i>n</i>+1, where <i>n</i> is a multiple of 3. Lacking intramolecular Idt <a href="#pone.0023167-Schnabelrauch1">[25]</a>, <i>n</i>+1 (and <i>n</i>â1) TR repeats are predicted to have a spiraling pattern along the PPII helix whereby each repeat is rotated by 120Â° with respect to its upstream repeat. As shown in <a href="#pone.0023167.s005">Figure S5</a>, â¼92% of EXTA TRs have periodicities of <i>n</i>+1 (â¼77%) or <i>n</i>â1 (â¼15%), implying that natural selection has maintained such a genetically encoded structural feature in EXTAs throughout land plant evolution. </p> <p>In addition to conserved periodicity, most full-length EXTA sequences also have a conserved C-terminal SP <sub>n</sub> motif with a Tyr residue at or near the C-terminus ( <a href="#pone.0023167.s029">Dataset S2A</a>; also see PlantPro20Fam online), as was previously noted for 18 of 20 <i>Arabidopsis</i> extensins <a href="#pone.0023167-Cannon1">[12]</a>. Notably, this terminal SP <sub>n</sub>,Y domain may or may not be contiguous with the EXTA TR domain. </p> <p>As a class, EXTAs have a number of conserved sequence features, including SP <sub>n</sub>,Y-containing TRs with basic isoelectric points (e.g., K-rich), and the general occurrence of an adjacent or non-contiguous C-terminal SP <sub>n</sub>,Y domain. Allowing for the substitution of Thr for Ser, we identified three additional TRP classes containing all of these conserved EXTA sequence features that we term extensin subtypes beta, gamma, and delta ( <a href="#pone.0023167.s029">Dataset S2</a>, AâD). Based on EST data ( <a href="#pone.0023167.s023">Table S13</a>), each of these four EXT classes is expressed in all major plant organs at all stages of development (including both sporophyte and gametophyte for moss EXTDs). In addition, our analysis revealed a broadly distributed <div class="u_UNKNOWN">H</div>ybrid <div class="u_UNKNOWN">EX</div>tensin class (HEX type alpha) that includes the leucine-rich repeat HEXs previously called LRXs and PEXs and studied in <i>Arabidopsis</i> and rice <a href="#pone.0023167-Baumberger1">[31]</a> ( <a href="#pone.0023167.s029">Dataset S2E</a>), as well as a miscellaneous extensin-like class that generally lacks C-terminal SP <sub>n</sub>,Y (EXTM) ( <a href="#pone.0023167.s025">Text S2</a>). </p> <p>Among the additional extensin classes, type beta (EXTB) differs from EXTAs by the general absence of YxY motifs and the presence of the potential crosslinking motif VYK (and variants V/I-Y/H-K/H) ( <a href="#pone.0023167.s025">Text S2</a>). Unlike EXTAs, EXTBs include the P1-type extensins, while like EXTAs, EXTBs are typically composed of TRs with periods that are not a multiple of 3 ( <a href="#pone.0023167.s006">Figure S6</a>). Although found predominantly in the non-grass angiosperms and absent from grasses ( <a href="#pone-0023167-g002">Figure 2B</a>), a putative EXTB was also identified in moss. The completed <i>Arabidopsis</i> and <i>Medicago</i> genomes each contain a single EXTB protein, compared to eighteen and ten EXTAs, respectively. If a comparably low ratio of EXTB to EXTA proteins is common in other genomes, future genome and deep sequencing projects may reveal a much broader phylogenetic distribution of EXTBs. </p> <p>Previously called THRGPs, sequences that compose the extensin type gamma (EXTC) class are uniquely found in grass secretomes and have been well characterized in maize <a href="#pone.0023167-Kieliszewski2">[32]</a>, <a href="#pone.0023167-Stiefel1">[33]</a>. In addition to SP <sub>1</sub> (and sometimes SP <sub>2</sub>), EXTC TR domains are characterized by TP <sub>2</sub>TY motifs ( <a href="#pone.0023167.s013">Table S3</a>) that are highly homologous to hydroxylated SP <sub>n</sub>,Y motifs conserved within the other extensins. Like EXTAs and EXTBs, EXTCs are also basic (K-rich) and have a C-terminal SP <sub>n</sub>,Y domain ( <a href="#pone.0023167.s029">Dataset S2C</a>) indicating that these grass TRPs are <i>bona fide</i> members of the extensin superfamily. </p> <p>Although moss sequences in our database lack EXTAs ( <a href="#pone-0023167-g002">Figure 2, A and B</a>), analysis of the fully sequenced genome of <i>Physcomitrella patens</i>, together with EST data from several moss families, revealed a previously unrecognized extensin class, type delta (EXTD). Defined by the presence of TRs with SP <sub>2</sub> and SPVYX motifs (where Xâ=âK/E/T/S; <a href="#pone.0023167.s013">Table S3</a>), and a C-terminal SP <sub>n</sub>,Y domain (see <a href="#pone.0023167.s029">Dataset S2D</a>), EXTDs share conserved sequence features with other extensins, and like EXTBs, many EXTDs have at least one basic VYK crosslinking site. A partial sequence encoding a homologous EXTD, composed of SPPVYXAPP TRs with a carboxy terminal TP <sub>2</sub>,Y domain, was also found in the liverwort <i>M. polymorpha</i> indicating that the distribution of EXTDs may extend to all bryophytes (GenBank accession BJ851426, also see PlantPro20Fam). </p> </div> <div class="fourconservedclassesofnon-extensinpro-richtrps" title="sec"> <div class="title" tagx="title" title="title">
Four conserved classes of non-extensin Pro-Rich TRPs</div> <p>Like EXTAs, four non-extensin TRP classes were found broadly distributed throughout plant evolution and expressed throughout plant development, presumably representing important structural "core modules" ( <a href="#pone-0023167-g002">Figures 2B</a> and <a href="#pone-0023167-g003">3</a>, <a href="#pone.0023167.s023">Table S13</a>). One of these classes is defined by TRs enriched in a T/S-P <sub>3</sub>A motif that was previously identified in two <i>Arabidopsis</i> AGPs (AGP7 and AGP9, <a href="#pone.0023167-Schultz2">[34]</a>). Based on its broad distribution ( <a href="#pone-0023167-g003">Figures 3B</a> and <a href="#pone.0023167.s007">S7</a>), we refer to this TRP class as TR-AGP type alpha (AGPA). Though most AGPs are not TRPs <a href="#pone.0023167-Schultz2">[34]</a>, most AGPA sequences have a predicted GPI-anchor (like the classical AGPs), including AGPAs present in the major divisions of higher plants: conifers, eudicots, non-grass monocots, and grasses (76% of AGPA sequences in <a href="#pone.0023167.s026">Text S3</a>). A highly conserved glutamine residue found at the N-terminus of mature AGPAs (predicted signal peptide removed) is consistent with the possibility of a common ancestor. </p> <p>A second TRP core module class, characterized by PELPK-containing repeats ( <a href="#pone-0023167-g001">Figures 1</a> and <a href="#pone.0023167.s002">S2</a>), is distributed throughout grass and non-grass angiosperms ( <a href="#pone-0023167-g003">Figures 3C</a> and <a href="#pone.0023167.s008">S8</a>). Multiple sequence alignment analysis of PELPKs revealed conserved non-TR N-terminal and C-terminal sequence features consistent with the hypothesis that the PELPKs are an orthologous TRP class that arose sometime before or during the emergence of basal angiosperms. Although evidence for PELPK proteins was detected in sixty-four flowering plant species and 45 genera ( <a href="#pone-0023167-g003">Figures 3C</a> and <a href="#pone.0023167.s008">S8</a>), only two PELPKs have been previously reported (At5g09520 and At5g09530 in <i>Arabidopsis</i>, called putative PRPs by <a href="#pone.0023167-Johnson1">[14]</a>, PRP9 and PRP10 by <a href="#pone.0023167-Showalter1">[15]</a>, and PELPK2 and PELPK1 by <a href="#pone.0023167-Rashid1">[35]</a>). Recently, At5g09530 was shown to localize to seed aleurone cells and xylem cell walls, and found to be expressed in response to pathogen attack and stress <a href="#pone.0023167-Rashid1">[35]</a>. Using publicly available <i>Arabidopsis</i> microarray analysis tools <a href="#pone.0023167-Winter1">[36]</a>, <a href="#pone.0023167-Hruz1">[37]</a> we also found that both <i>Arabidopsis</i> PELPK genes are up-regulated during seed development and expressed in radicle, hypocotyl, and adult root tissues, and both are up-regulated in procambial tissue in response to osmotic stresses or ABA treatments. Together, these data are consistent with the hypothesis that PELPKs are a large, widely distributed class of plant cell wall proteins with potentially diverse physiological functions. </p> <p>Also broadly distributed in higher plants, we found two large groups of TRP classes each defined by the fusion of a distinct Pro-rich TR domain to highly conserved Cys-rich domains, either the Lipid Transfer (LT) protein domain or the Pollen Ole e I (PO) domain <a href="#pone.0023167-Silverstein1">[38]</a>, <a href="#pone.0023167-Dvokov1">[39]</a>. PO domains are considered a "domain/protein of unknown function" (termed DUF1210 in the Pfam database), while LT domains have been found in small, secreted proteins known to play roles in plant growth, defense and reproduction <a href="#pone.0023167-Kader1">[40]</a>, <a href="#pone.0023167-Chae1">[41]</a>. Among six subtypes of hybrid LT TRPs and three subtypes of hybrid PO TRPs identified in our analysis ( <a href="#pone.0023167.s017">Table S7</a>), two classes represent core modules in higher plant secretomes, HLT type alpha (HLTA) and HPO type alpha (HPOA). HLTAs have a Pro-rich TR domain containing variations on the motif, PPVTLPPVVK ( <a href="#pone.0023167.s015">Table S5</a>), and are found in 46 genera of non-grass seed-bearing plants ( <a href="#pone.0023167.s009">Figure S9</a>), while HPOAs have a TR domain containing variations on the motif, PPPVPVYKKPL ( <a href="#pone.0023167.s015">Table S5</a>), and are found in 37 genera from primitive to advanced non-grass angiosperms ( <a href="#pone.0023167.s010">Figure S10</a>). </p> </div> <div class="mostpro-richtrpclasseshavenarrowphylogeneticdistributions" title="sec"> <div class="title" tagx="title" title="title">
Most Pro-rich TRP classes have narrow phylogenetic distributions</div> <p>Remarkably, 81% of secreted Pro-rich structural TRP classes that we identified have very limited phylogenetic distributions ( <a href="#pone-0023167-g002">Figure 2B</a>, <a href="#pone-0023167-t001">Table 1</a>), with nearly half of the 31 TRP classes restricted to individual plant families ( <a href="#pone-0023167-t001">Table 1</a>). For example, the "classical" PRPs, defined by the conserved TR pentapeptide P <sub>2</sub>V(Y/E/H)K, are common throughout the Fabaceae but are limited to only three additional eudicot families in our database (see PRPA in <a href="#pone-0023167-g002">Figure 2B</a> and <a href="#pone.0023167.s025">Text S2</a>). A similar but distinct TRP class (PRPB) is found in conifers ( <a href="#pone.0023167.s017">Table S7</a>, <a href="#pone.0023167.s025">Text S2</a>). Two additional TRP classes are specific to legumes, and two other classes are only found in Brassicaceae or <i>Populus</i> species ( <a href="#pone-0023167-t001">Table 1</a>). Remarkably, the PEHK class, exclusive to the grape family, has 18 known and predicted genes arranged in a sequence continuum across chromosome five in <i>V. vinifera</i>. Such a physical arrangement indicates that this gene family is likely a product of recent tandem gene duplication ( <a href="#pone.0023167.s003">Figure S3</a>). (Several other Pro-rich TRP genes identified in this work also occur in closely linked gene clusters, including four different extensin gene clusters in <i>Arabidopsis</i>, and the PELPK genes in <i>Arabidopsis</i>, rice and <i>Sorghum bicolor</i> (see <a href="#pone.0023167.s025">Text S2</a>.) Finally, the grass family, known to have a unique Type II cell wall architecture <a href="#pone.0023167-Vogel1">[3]</a>, has a strikingly large group of unique Pro-rich TRP classes ( <a href="#pone-0023167-t001">Table 1</a>). Importantly, all plant families with a unique TRP class have at least one fully sequenced genome. The apparent absence of taxonomically restricted TRP classes in other analyzed plant families, however, may result from the unavoidable phylogenetic sampling bias of currently available sequence data (e.g., <a href="#pone-0023167-g002">Figure 2C</a>). Likewise, it is possible that some of the Pro-rich TR classes identified in this work span broader phylogenetic ranges than described. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0023167.t001">10.1371/journal.pone.0023167.t001</a> <span class="label" tagx="label" title="label">Table 1</span>  <div class="title" tagx="title" title="title">
Plant family-specific Pro-rich TRP classes.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <td>TRP Class</td> <td>Plant Family</td> <td>General Division(s)</td> </tr> </thead> <tbody> <tr> <td>HLTD</td> <td>Brassicaceae (e.g. <i>Arabidopsis</i>) </td> <td>Super-Rosids (Eudicots)</td> </tr> <tr> <td>PEHK</td> <td>Vitaceae (grape family)</td> <td>Super-Rosids (Eudicots)</td> </tr> <tr> <td>PHEK</td> <td>Fabaceae (legumes)</td> <td>Super-Rosids (Eudicots)</td> </tr> <tr> <td>KPIP</td> <td>Fabaceae</td> <td>Super-Rosids (Eudicots)</td> </tr> <tr> <td>HLTF</td> <td>Salicaceae (Populus species)</td> <td>Super-Rosids (Eudicots)</td> </tr> <tr> <td>AGPC</td> <td>Poaceae (grasses)</td> <td>Monocots</td> </tr> <tr> <td>MPAV</td> <td>Poaceae</td> <td>Monocots</td> </tr> <tr> <td>PEPKB</td> <td>Poaceae</td> <td>Monocots</td> </tr> <tr> <td>EXTC</td> <td>Poaceae</td> <td>Monocots</td> </tr> <tr> <td>HPOB</td> <td>Poaceae</td> <td>Monocots</td> </tr> <tr> <td>PEPKA</td> <td>Poaceae</td> <td>Monocots</td> </tr> <tr> <td>HLTE</td> <td>Poaceae (Non-Triticeae)</td> <td>Monocots</td> </tr> <tr> <td>QRA (LMW glutenins)</td> <td>Poaceae (Triticeae tribe)</td> <td>Monocots</td> </tr> <tr> <td>QRB (HMW glutenins)</td> <td>Poaceae (Triticeae tribe)</td> <td>Monocots</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label" /> <p>Each TRP class is described in the supporting information ( <a href="#pone.0023167.s024">Text S1</a> and <a href="#pone.0023167.s025">S2</a>; also see <a href="#pone.0023167.s016">Tables S6</a>, <a href="#pone.0023167.s017">S7</a>, <a href="#pone.0023167.s018">S8</a>). </p> </div> </div> </div> <p>In some cases, family-specific TRP classes correlate with distinct known secretome functions indicating that phylogenetically restricted TRPs, in general, may represent important evolutionary innovations. For example, the PHEK class, only found in Fabaceae, includes an "early nodulin" thought to be involved in legume-specific symbioses with rhizobia bacteria <a href="#pone.0023167-Brewin1">[42]</a>, while two unique TRP classes targeted to seed storage vacuoles in the Triticeae tribe of grasses (glutenins and gliadins) are known to impart the elastomeric properties that typify cereal grain flours <a href="#pone.0023167-Shewry1">[43]</a>. Though most TRP classes identified in our analysis have not yet been functionally characterized, like many TRPs throughout nature (e.g., mammalian mucins, spider silks, insect resilins, mollusc biomineralization proteins, mussel adhesive proteins), these plant structural molecules are likely to have evolutionarily important biomechanical properties that may underlie previously unexplored cell wall diversity. </p> <p>Proteins with highly biased amino acid composition, like the TRP classes identified in this work ( <a href="#pone.0023167.s002">Figure S2</a>), represent one important component of the poorly explored "dark matter" of the protein sequence universe <a href="#pone.0023167-Levitt1">[44]</a>. In this study, we developed algorithms for TR identification, architecture modeling, and clustering that do not depend upon prior knowledge of TR motifs. Applying these methods to a global computational analysis of Pro-rich structural TRPs in plant secretomes, 31 TRP classes with distinct TR architectures were identified, including all previously characterized extensin and PRP TRPs. In addition to showing that one TRP class, Extensin type alpha, is the most broadly conserved Pro-rich TRP class in plant secretomes, we identified an extensin superfamily that includes several additional TRP classes with well-defined, characteristic EXT sequence features. Beyond extensins, four TRP "core modules" in land plants were identified, along with more than twenty additional classes of Pro-rich TRPs, all of which have characteristic TR architectures and limited phylogenetic distributions (e.g. the "canonical" PRPs composed of P <sub>2</sub>V(Y/E/H)K TRs). Taken together, our results document large-scale diversity in cell wall extensins, identify conserved core modules and distinct phylogenetic signatures in higher plant secretomes, and provide a rational taxonomy and nomenclature for the diversity in secreted Pro-rich structural proteins. In addition, this work should have applications for developing comprehensive system-level cell wall models, which have heretofore focused almost exclusively on wall polysaccharide synthesis and architecture. </p> </div> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="masterdatabase" title="sec"> <div class="title" tagx="title" title="title">
Master Database</div> <p>To study the global diversity of Pro-rich TRPs targeted to plant secretomes, we built a large database containing both expressed sequence tag (EST) data and known/hypothetical protein sequences representing diverse plant species ( <a href="#pone.0023167.s011">Table S1</a>). All available plant EST assemblies and singletons were downloaded from two online repositories, the Gene Index Project (called "TCs") hosted at the Dana-Farber Cancer Institute (downloaded 10/06/09 from <a href="http://compbio.dfci.harvard.edu/tgi/plant.html">http://compbio.dfci.harvard.edu/tgi/plant.html</a>) <a href="#pone.0023167-Quackenbush1">[45]</a> and the TIGR Plant Transcript Assemblies database (called "TAs") (release 07/10/07, downloaded from <a href="ftp://ftp.tigr.org/pub/data/plantta/">ftp://ftp.tigr.org/pub/data/plantta/</a>) <a href="#pone.0023167-Childs1">[46]</a>. We also downloaded and extracted all known and predicted plant protein sequences from the NCBI Non-Redundant (NR) protein sequence database (release 10/04/09 from <a href="http://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/">http://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/</a>). ESTs were translated into forward reading frames ( <a href="#pone.0023167.s011">Table S1</a>), and complete taxonomies (i.e. phylum, class, family, etc.) were downloaded for all sequence data from NCBI ( <a href="http://www.ncbi.nlm.nih.gov/taxonomy">http://www.ncbi.nlm.nih.gov/taxonomy</a>). Of all plant species represented in the master database, 95% are severely under-sampled (at most 1â10 proteins and/or ESTs), and 0.6% of all species are represented by at least 1000 individual sequence samples (236/36,815 species). Notably, due to a lack of genome sequence data, most sequences analyzed in this work necessarily originate from specialist EST databases, not NCBI NR (see <a href="#pone.0023167.s011">Table S1</a>). For details of additional sequence data analyzed in this work, see <a href="#pone.0023167.s027">Text S4</a>. </p> </div> <div class="plantpro20databaseconstruction" title="sec"> <div class="title" tagx="title" title="title">
PlantPro20 Database Construction</div> <p>We used XSTREAM, a TR identification and architecture modeling tool <a href="#pone.0023167-Newman1">[47]</a>, to process the master database for TR content. The following algorithm parameters were used: Minimum Copy Number ( <i>minC</i>): 2, Minimum Character Identity ( <i>i</i>): 0.7, Minimum Consensus Matching ( <i>I</i>): 0.8, Maximum Consecutive Gaps ( <i>g</i>): 3, Minimum Domain Length ( <i>minD</i>): 12 amino acids, and default parameters (refer to <a href="#pone.0023167-Newman1">[47]</a> for details of these parameters). All identified TR motifs ( <a href="#pone.0023167.s011">Table S1</a>) were subsequently filtered for Pro-rich TRs, defined as TRs having at least 20% proline in their consensus sequences. The consensus sequence is a representative copy of the entire TR domain identified and modeled by XSTREAM. </p> <p>A new database, PlantPro20, was created to house all Pro-rich TRs along with additional sequence and phylogenetic data ( <a href="#pone.0023167.s012">Table S2</a>). Due to the high quality of TC ESTs, PlantPro20 was initially populated with Pro-rich TRs derived from translated TC sequences and singletons. Next, we added Pro-rich TRs identified from TA sequences representing plant species not already present in PlantPro20, and finally, all NR sequences with Pro-rich TRs were added (see <a href="#pone.0023167.s012">Table S2</a>). The longest sequence stretch containing an N-terminal methionine and downstream stop codon was identified for each EST. ORFs (full-length or partial ORFs, including NR sequences) were then individually scanned for a secretion signal and GPI anchor using SignalP 3.0 <a href="#pone.0023167-Emanuelsson1">[23]</a> and PredGPI <a href="#pone.0023167-Pierleoni1">[48]</a> web servers, respectively. For secretory peptide identification, we required a minimum HMM signal peptide probability of 0.7 or a âYesâ prediction for all Neural Network score components (i.e. max. C, max. Y, max. S, mean S, D). A minimum specificity of 99.5 (1-false positive rate) was used for the identification of GPI-anchored proteins. To focus on TR domains long enough to compose structural domains in TRPs, all TRs were additionally filtered for <i>TR Modules</i>, defined as TR domains that span at least 100 amino acids, or cover at least 33% of their parent protein sequence and span at least 50 amino acids (see <a href="#pone.0023167.s012">Table S2</a>). </p> </div> <div class="autosome-tr" title="sec"> <div class="title" tagx="title" title="title">
AutoSOME-TR</div> <p>We implemented an unsupervised clustering method, called AutoSOME <a href="#pone.0023167-Newman2">[49]</a>, within a software framework tailored to the unique properties of TR sequences. This new method, called AutoSOME-TR effectively clusters TR sequences with either high or low complexity amino acid compositions, captures sequence context dependency among adjacent residues, and recognizes TR phase variation using cyclical permutations (e.g., by defining a single equivalence class of TR consensus sequences: <span class="named-content" title="named-content">PVYKâ=âVYKPâ=âYKPVâ=âKPVY</span>). </p> <p>Prior to clustering, each TR domain is converted into a dipeptide compositional vector (of length 400 to accommodate all possible dipeptides). We found empirically that such dipeptide vectors provide for better discrimination of low-complexity sequences than single amino acid vectors (length 20). As an example, given a TR domain â <span class="named-content" title="named-content">PVPVKPVPVK</span>â with consensus sequence â <span class="named-content" title="named-content">PVPVK</span>â, the dipeptide vector would contain four copies of âPVâ, two copies of âVPâ, two copies of âVKâ and one copy of âKPâ. These compositional vectors are subsequently normalized by TR domain length (10 in the previous example) and used as input for the AutoSOME-TR algorithm. Importantly, the use of dipeptide compositional vectors allows AutoSOME-TR to both capture context dependence among adjacent residues and mitigate issues due to TR phase variation. </p> <p>Clustering of TRs is accomplished in two major phases. First, the entire input data set of dipeptide compositional vectors is clustered using the AutoSOME algorithm, and the resulting compositional clusters are output to memory and disk. Immediately thereafter, each TR compositional cluster is evaluated by a series of quality control procedures designed to enforce a user-specified threshold of internal homology. In brief, for each TR cluster, an alignment procedure performs an all-against-all comparison of each TR consensus sequence to all cyclical permutations of every other TR consensus sequence in the cluster, and an aggregate score records the total number of character matches. The TR consensus sequence alignment with maximal score is then used to build a master consensus sequence, which is compared to its constituents to determine a consensus error. The same consensus generation and error procedures as described for TRs in <a href="#pone.0023167-Newman1">[47]</a> are used here. Next, a procedure is invoked to flag and remove consensus sequences that are unlikely to belong to the cluster, and a consensus error is recomputed. If the lowest of the two consensus errors satisfies the user-defined homology threshold <i>E</i> (â=â0.4, by default), the corresponding TR cluster is output to file. Otherwise, AutoSOME-TR is rerun on the original compositional cluster, including any discarded consensus sequences from the cleansing step, to identify finer-grained partitions. This second stage is recursively repeated until each TR cluster either meets a stricter <i>E</i> (â=â0.3, by default) or the size of the cluster is below a minimum size threshold also modifiable by the user (10 by default). Software and pseudocode for our AutoSOME-TR implementation are available upon request. </p> </div> <div class="trclusteranalysis" title="sec"> <div class="title" tagx="title" title="title">
TR Cluster Analysis</div> <p>Of the 1240 unique TR Module consensus sequences found in protein sequences with a predicted signal peptide ( <a href="#pone.0023167.s012">Table S2</a>), 997 TR Module motifs were clustered by AutoSOME-TR. Each of these 997 TRs has periodicity&amp;gt;3, and lacks an âXâ character or stop codon. The following clustering parameters were used: 100 ensemble iterations for individual AutoSOME runs, <i>P</i>&amp;lt;0.01, self-organizing map (SOM) grid dimensions of 20Ã20 nodes for stage-one clustering and 10Ã10 nodes for stage-two clustering (see <i>AutoSOME-TR</i> for description of clustering stages), SOM topologyâ=âcircle, and cartogram resolution of 64Ã64 (parameters are defined and discussed in Newman and Cooper, 2010). AutoSOME-TR identified 104 TR clusters containing 816 distinct TR Module consensus sequences and 181 singleton TR consensus sequences. See <a href="#pone.0023167.s019">Table S9</a> for a comprehensive list of all clustered TR motifs, along with their corresponding proteins and plant taxa, and see <a href="#pone.0023167.s020">Table S10</a> for TR cluster summary statistics. </p> </div> <div class="constructionoftrmoduleclusternetwork" title="sec"> <div class="title" tagx="title" title="title">
Construction of TR Module Cluster Network</div> <p>Clusters with at least three members (73 of 104 clusters) were further analyzed for inter-cluster similarity using the AutoSOME fuzzy cluster network approach described in <a href="#pone.0023167-Newman2">[49]</a>. Both singleton TRs and TR clusters with two members ( <a href="#pone.0023167.s020">Table S10</a>) were analyzed separately. To prevent the abundance of proline residues from skewing fuzzy clustering results, proline residues were removed from TR consensus sequences prior to fuzzy clustering. As input to the AutoSOME algorithm, all TR consensus sequences (missing proline) from each cluster were individually converted into dipeptide compositional vectors, and then averaged together to form a compositional vector representing each TR cluster. The input data set of TR compositional vectors was then unit-variance normalized over each column, converted into an all-against-all distance matrix using Pearson's correlation, and clustered by AutoSOME using a p-value threshold of 0.01, 500 ensemble iterations, SOM grid size of 12Ã12 nodes, SOM topologyâ=âcircle, and cartogram resolution of 32Ã32 (parameters are described in Newman and Cooper, 2010). </p> <p>The resulting fuzzy cluster network was rendered using Cytoscape 2.6.0 <a href="#pone.0023167-Shannon1">[50]</a>. TR Module clusters (containing proline) were superimposed onto the network using a custom Cytoscape plug-in, and the Organic layout algorithm was used for spatial organization of clusters. Initially, the network display was highly interconnected, proving too dense for visual cluster analysis. Network density was significantly decreased by removing <i>insignificant</i> edges, defined as edges having cluster confidence less than 0.04, where confidenceâ=â1 denotes a co-clustering of 100% over all ensemble iterations (see <a href="#pone.0023167-Newman2">[49]</a>). This step resulted in the removal of 93% of all edges (2582 of 2775) and two nodes (clusters 42 and 60). The resulting cluster similarity network, showing 73 TR Module clusters and representing 747 unique TR Module consensus sequences, is depicted in high-resolution detail in <a href="#pone.0023167.s001">Figure S1</a> (also see <a href="#pone.0023167.s020">Table S10</a> for cluster network statistics). Original cluster identifiers ( <a href="#pone.0023167.s019">Tables S9</a>, <a href="#pone.0023167.s020">S10</a>) are provided in <a href="#pone.0023167.s001">Figure S1</a> next to the TR Module clusters. </p> </div> <div class="developmentoftrtaxonomyandnomenclature" title="sec"> <div class="title" tagx="title" title="title">
Development of TR Taxonomy and Nomenclature</div> <p>To focus on TR Module clusters representing a broad range of species and/or large number of protein sequences, a simple filter was applied to all cluster results, including the cluster network ( <a href="#pone.0023167.s001">Figure S1</a>). Clusters that passed the filter contain TR Modules that are together present in â¥20 secreted proteins <b>or</b> are present in secreted proteins that together span â¥10 species. These clusters are shown as numbered nodes in <a href="#pone-0023167-g001">Figure 1</a> (also see <a href="#pone.0023167.s020">Table S10</a>). Small, unlabeled nodes in <a href="#pone-0023167-g001">Figure 1</a> are also shown as they have similar TR content to clusters that passed the filter. </p> <p>In contrast to previous work that relied upon prior knowledge of TR sequences and composition (e.g. <a href="#pone.0023167-Showalter1">[15]</a>, <a href="#pone.0023167-Graham1">[22]</a>), we used unsupervised methods to identify clusters of abundant TR elements within the Pro-rich TR Module landscape ( <a href="#pone-0023167-g001">Figures 1</a> and <a href="#pone.0023167.s001">S1</a>), and subsequently devised a series of regular expression definitions to formally capture these prominent TR motifs. Regular expressions were used since, when rigorously defined, they allow for robust character matching with minimal noise. Every TR cluster satisfying the species and sequence filtration criteria or corresponding to a visible node in <a href="#pone-0023167-g001">Figure 1</a> was manually analyzed for motif homogeneity, both internally and compared to neighboring clusters in the network of <a href="#pone.0023167.s001">Figure S1</a> (see <a href="#pone.0023167.s020">Table S10</a> for all analyzed TR clusters). While our goal was to map one unique regular expression definition per TR cluster, some clusters were split to better capture motif diversity (e.g. cluster 3 in <a href="#pone.0023167.s001">Figure S1</a>), and other clusters, if highly related in TR content, were combined (e.g. PELPK clusters in <a href="#pone.0023167.s002">Figure S2</a>). Every regular expression was fine-tuned for maximum specificity and sensitivity across the entire PlantPro20 database ( <a href="#pone.0023167.s012">Table S2</a>), resulting in a final set of 38 distinct and highly specific regular expression definitions ( <a href="#pone.0023167.s021">Table S11</a>). </p> <p>Due to the unique structural and biochemical properties of proline residues, such as the ability to disrupt alpha helices, form polyproline helices <a href="#pone.0023167-Williamson1">[30]</a>, and serve as a substrate for post-translational modification <a href="#pone.0023167-Kieliszewski1">[9]</a>, we further classified each of the TR classes by proline sequence architecture. Based on prominent proline backbones observed in the cluster network of <a href="#pone.0023167.s001">Figure S1</a>, we selected the following proline backbone categories: SP <sub>n</sub>, TP <sub>n</sub>, interspersed P <sub>1</sub>, regular P <sub>2</sub> blocks, interspersed P <sub>2</sub>/P <sub>1</sub>, regular P <sub>2</sub> and P <sub>3</sub> blocks, and interspersed P <sub>2</sub>/P <sub>3</sub>/P <sub>1</sub>. </p> <p>Finally, rather than name each TR class using arbitrary alphanumeric nomenclature (e.g. class I, II, or A, B, etc.), or a generic name like âPRP motifâ, each TR class was named using the single letter amino acid code of a prevalent motif or sub-motif. By capturing an inherent property of the TR class within its name, this approach to nomenclature should eliminate ambiguity. </p> </div> <div class="trptaxonomyandnomenclature" title="sec"> <div class="title" tagx="title" title="title">
TRP Taxonomy and Nomenclature</div> <p>Using our TR taxonomy, the relevant published literature, multiple sequence alignments, and the presence or absence of conserved non-TR domains (identified using BLAST <a href="#pone.0023167-Altschul1">[18]</a> and SMART <a href="#pone.0023167-Letunic1">[51]</a>), all Pro-rich TRPs captured by the TR taxonomy were also classified and named. The TRP nomenclature that we developed includes both concise (3â5 letters) and descriptive names. When applicable, names were chosen to indicate hierarchical membership to larger classes, or "superfamilies" (e.g. all extensins are called âEXTsâ). Greek characters were used for superfamily sub-types, such as "extensin type alpha," or "TRP-AGP type gamma," and as a general rule, increasing Greek characters (i.e. alpha, beta, gamma, etc.) indicate decreasing phylogenetic coverage in the PlantPro20 database (e.g. 10 plant families, 3 families and 10 species, 3 families and 5 species, etc.) for a given TRP superfamily. </p> </div> <div class="sequenceredundancyandrevisions" title="sec"> <div class="title" tagx="title" title="title">
Sequence Redundancy and Revisions</div> <p>A significant amount of sequence redundancy was observed for full-length and partial protein sequences in the PlantPro20 database. To provide the research community with an uncluttered protein sequence resource, we attempted to eliminate overlapping sequences representing the same TRP gene sequence. Since single-read EST sequencing is prone to errors, particularly within TR regions (e.g. frameshifts), further computational assembly of identified complete/partial ORFs was bypassed, and instead, all sequences with hits to the TR taxonomy (i.e. ESTs, NR sequences, genomic sequences) were manually curated using basic alignment tools and online sequence databases. The well-annotated <i>Arabidopsis</i> genome was used as a benchmark to calibrate and fine-tune the accuracy of manual sequence assembly and clustering. Multiple Sequence Alignments (MSAs) of NR and genome sequences, translated ESTs, and 5â²/3â² UTRs of ESTs were all analyzed, when possible. If significant sequence homology was observed among all aligned proteins and/or transcript regions as judged by manual inspection, a master sequence was chosen to represent the group. When selecting a master sequence, precedence was given to genomic sequences, followed by NR sequences, and then translated ESTs. If more than one candidate master sequence of the same sequence type was identified (e.g. all ESTs), the longest sequence was designated as the master. In the absence of genomic or NR sequences, the "best" EST was chosen, where "best" is arbitrarily defined as the longest sequence stretch without observed errors or ambiguities (e.g. âXâ characters). To further increase the quality of the curated protein list, in some cases, the master sequence was revised on the basis of additional sequence information ( <a href="#pone.0023167.s022">Table S12</a>). Revisions included extending master sequences using significantly overlapping ESTs from the same species found by NCBI BLAST analysis and correction of suspected EST frameshifts by pairwise alignments of reading frames. All curated master sequences are provided in <a href="#pone.0023167.s026">Text S3</a>. For all master, partial, and redundant sequences, refer to the PlantPro20Fam online database. </p> </div> <div class="plantpro20famwebdatabase" title="sec"> <div class="title" tagx="title" title="title">
PlantPro20Fam Web Database</div> <p>The TRP sequence data classified and named in this work, along with all unclassified secreted Pro-rich TRPs within the PlantPro20 database, are freely available via an online repository ( <a href="http://jimcooperlab.mcdb.ucsb.edu/plantpro20fam">http://jimcooperlab.mcdb.ucsb.edu/plantpro20fam</a>). This web resource, called PlantPro20Fam, provides a facile interface for exploring the diverse Pro-rich TR architectures and phylogenetic diversity of Pro-rich TRPs spanning &amp;gt;250 species ranging from green algae to flowering plants. </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting Information</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S1</span>  <p> <b>Cluster landscape of Pro-rich TRs from plant secretomes.</b> High-resolution network representation of TR cluster results. Only unique consensus sequences corresponding to each TR domain are shown. Consensus motifs within each cluster were aligned as described in the <a href="#s3"> <i>Materials and Methods</i> </a>. To display amino acid physical chemical properties as RGB colors, we developed a three-dimensional representation of the following seven parameters: hydrophobicity <a href="#pone.0023167-Fauchere1">[54]</a>, alpha helix, beta sheet, beta turn, and coil conformational parameters <a href="#pone.0023167-Delage1">[55]</a>, Van der Waals volume, and isolectric point <a href="#pone.0023167-Creighton1">[56]</a>; Principal Components Analysis (PCA) was used to reduce these seven property scales into three dimensions (components 1, 2, and 3), and these components were normalized into R, G, and B color elements, respectively. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0023167.s001.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S2</span>  <p> <b>Super-classes of Pro-rich TRs, related to </b> <a href="#pone-0023167-g001"> <b>Figure 1</b> </a> <b>.</b> Super-classes of TRP clusters shown in <a href="#pone-0023167-g001">Figure 1</a> are indicated using the network of <a href="#pone.0023167.s001">Figure S1</a>. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0023167.s002.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S3</span>  <p> <b>Evolutionary relationships among 18 PEHK loci from the </b> <b> <i>V. vinifera</i> </b> <b> genome.</b> Major protein domains are indicated below the alignment, including part of the TR domain (for the TR taxonomy, see <a href="#pone.0023167.s015">Table S5</a>), and evolutionary relationships among corresponding PEHK loci are shown on the left (bootstrap values shown above branch points). Ten PEHK genes inferred by homology searching are indicated by an asterisk on the right side of the alignment while eight previously discovered PEHK loci are given as NCBI gene identifiers. For readability, only part of the original multiple sequence alignment of PEHK protein sequences is shown (aligned residues between positions 70 and 210 were removed; rendered with JalView <a href="#pone.0023167-Waterhouse1">[57]</a> and aligned using MUSCLE <a href="#pone.0023167-Edgar1">[58]</a>). Methods: tBLASTn <a href="#pone.0023167-Altschul1">[18]</a> was used to search the <i>V. vinifera</i> genome (12Ã database compiled 02/17/10, <a href="http://www.genoscope.cns.fr/cgi-bin/blast_server/projet_ML/blast.pl">http://www.genoscope.cns.fr/cgi-bin/blast_server/projet_ML/blast.pl</a>) for genes encoding proteins similar to the eight previously reported PEHKs <a href="#pone.0023167-Davies1">[59]</a>, <a href="#pone.0023167-Thomas1">[60]</a>, resulting in the identification of 10 putative PEHK genes. To analyze evolutionary relationships among all 18 PEHK genes, a multiple sequence alignment was built for all 18 known and predicted coding regions using CLUSTALW <a href="#pone.0023167-Larkin1">[61]</a>, and an evolutionary tree was constructed using the Neighbor-Joining method, with 10000 bootstrap iterations. Branches with &amp;lt;50% bootstrap confidence are collapsed. The Jukes-Cantor method was used to estimate evolutionary distance, and positions with alignment gaps and missing data were not considered during pairwise sequence comparisons. MEGA4 was used for the entire phylogenetic analysis <a href="#pone.0023167-Tamura1">[62]</a>. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0023167.s003.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S4</span>  <p> <b>Phylogenetic tree of the EXTA class, related to </b> <a href="#pone-0023167-g003"> <b>Figure 3A</b> </a> <b>.</b> Full-length and/or partial EXTAs from 112 species and 72 genera are shown (orange-colored branches). The 143 leaves in these trees represent the genera that are either captured by our TR taxonomy ( <a href="#pone.0023167.s013">Tables S3</a>, <a href="#pone.0023167.s014">S4</a>, <a href="#pone.0023167.s015">S5</a>) or represented by at least one secreted Pro-rich TRP in our analysis ( <a href="#pone.0023167.s012">Table S2</a>). The evolutionary relationships among genera were downloaded from the NCBI taxonomy browser, and the tree was rendered using PhyloWidget <a href="#pone.0023167-Jordan1">[53]</a>. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0023167.s004.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S5</span>  <p> <b>EXTA TRs have two major periodicities.</b> The periodicity distribution of 690 out of 698 EXTA TR domains is shown below (not shown are periods 44 (1Ã), 51 (1Ã), 53 (5Ã), and 76(1Ã)). Red, green, and blue-colored bars represent periods of length <i>n</i>, <i>n</i>â1, and <i>n</i>+1, respectively, where <i>n</i> is defined as all periodicities that are a multiple of 3. To ensure accurate characterization of TR periodicities, XSTREAM <a href="#pone.0023167-Newman1">[47]</a> was used to model the architectures of all 698 EXTA TR domains in the PlantPro20 database ( <a href="#pone.0023167.s012">Table S2</a>) without the inclusion of gap characters (otherwise, XSTREAM was run the same as described in <a href="#s3"> <i>Materials and Methods</i> </a>). </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0023167.s005.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S6</span>  <p> <b>EXTB TRs have two major periodicities.</b> This histogram shows the periodicity distribution of 172 out of 173 EXTB TR domains (one instance of periodâ=â40 is not shown). The histogram is colored identically to <a href="#pone.0023167.s005">Figure S5</a>. XSTREAM <a href="#pone.0023167-Newman1">[47]</a> was used to model all EXTB TR architectures the same as described in <a href="#pone.0023167.s005">Figure S5</a>. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0023167.s006.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S7</span>  <p> <b>Phylogenetic tree of the AGPA class, related to </b> <a href="#pone-0023167-g003"> <b>Figure 3B</b> </a> <b>.</b> AGPAs were found as partial or complete sequences in 38 plant genera (orange-colored branches and leaves). The tree was built identically to the tree in <a href="#pone.0023167.s004">Figure S4</a>. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0023167.s007.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S8</span>  <p> <b>Phylogenetic tree of the PELPK class, related to </b> <a href="#pone-0023167-g003"> <b>Figure 3C</b> </a> <b>.</b> PELPKs (partial or complete sequences) were found in 45 plant genera (orange-colored branches and leaves). The tree was built identically to the tree in <a href="#pone.0023167.s004">Figure S4</a>. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0023167.s008.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S9</span>  <p> <b>Phylogenetic tree of the HLTA class.</b> HLTAs were found as partial or complete sequences in 46 plant genera (orange-colored branches and leaves). The tree was built identically to the tree in <a href="#pone.0023167.s004">Figure S4</a>. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0023167.s009.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S10</span>  <p> <b>Phylogenetic tree of the HPOA class.</b> HPOAs (partial or complete sequences) were found in 37 plant genera (orange-colored branches and leaves). The tree was built identically to the tree in <a href="#pone.0023167.s004">Figure S4</a>. </p> <p>(TIF)</p>  <div class="media" title="media"> <a href="pone.0023167.s010.tif">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S1</span>  <p> <b>Summary statistics of plant master database.</b> Primary sequence data used to develop the TR and TRP taxonomies in this work. For further details, see <a href="#s3"> <i>Materials and Methods</i> </a> in the primary text. </p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pone.0023167.s011.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S2</span>  <p> <b>Summary statistics of PlantPro20 database.</b> Details of Pro-rich TRs with at least 20% proline content (see <a href="#s3"> <i>Materials and Methods</i> </a> in the primary text). </p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pone.0023167.s012.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S3</span>  <p> <b>Ser/Thr-(Pro) <sub>n</sub> TR classes and corresponding TRP classes. </b> </p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pone.0023167.s013.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S4</span>  <p> <b>Interspersed (Pro) <sub>1</sub> and (Pro) <sub>1</sub>/(Pro) <sub>2</sub> TR classes and corresponding TRP classes. </b> </p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pone.0023167.s014.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S5</span>  <p> <b>TR classes with regular (Pro) <sub>2</sub> blocks or (Pro) <sub>2</sub> interspersed with (Pro) <sub>1</sub> and/or (Pro) <sub>3</sub>, and corresponding TRP classes. </b> </p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pone.0023167.s015.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S6</span>  <p> <b>Ser/Thr-(Pro) <sub>n</sub>âcontaining TRP classes. </b> </p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pone.0023167.s016.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S7</span>  <p> <b>PRP and Hybrid PRP TRP classes.</b> </p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pone.0023167.s017.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S8</span>  <p> <b>Additional Pro-rich TRP classes.</b> </p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pone.0023167.s018.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S9</span>  <p> <b>Full details of all Pro-rich TR clusters found in the PlantPro20 database.</b> </p> <p>(XLS)</p>  <div class="media" title="media"> <a href="pone.0023167.s019.xls">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S10</span>  <p> <b>Summary statistics of all identified Pro-rich TR clusters with respect to fuzzy cluster diagrams and the TR taxonomy. </b> </p> <p>(XLS)</p>  <div class="media" title="media"> <a href="pone.0023167.s020.xls">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S11</span>  <p> <b>38 TR regular expression queries corresponding to TR taxonomy.</b> Each regular expression was used to scan TR consensus sequences in PlantPro20 ( <a href="#pone.0023167.s012">Table S2</a>) to identify protein sequences with similar TR content. Amino acids enclosed in square brackets represent more than one possible match (e.g. [ST] means either serine or threonine is a match), â.â denotes a wildcard character, ââ§â preceding an amino acid in closed square brackets denotes âNOTâ (e.g. [â§P], meaning that proline is excluded), and numbers in closed curly brackets indicate how many times a particular residue must be repeated for a match (e.g. P{2,4} means proline must be found 2â4 times in tandem for a match). Finally, when more than one motif is a match, each motif is separated by â|â and the entire expression is bound by square brackets (e.g. [AP|ST] means either AP or ST is a match). For example, the following regular expression for TR class tp3a, â[TS]P{3,4}[VA]{1,2}[TS].P and not [HY]â, should be read as: T or S, followed by 3â4 repeats of P, followed by 1â2 repeats of V or A, followed by T or S, any character followed by P, and never a match of H or Y. </p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pone.0023167.s021.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S12</span>  <p> <b>Documentation for revised master sequences.</b> Several master sequences in the curated sequence list ( <a href="#pone.0023167.s026">Text S3</a>) were derived from ESTs representing partial ORFs or ESTs with evidence of a frameshift. We attempted to revise these master sequences based on additional database searching, multiple sequence alignment, or combining reading frames with evidence of a frameshift. Actions taken to "fix" these sequences are given below. The number immediately following some sequence identifiers in <i>Revision(s)</i> denotes the forward reading frame used to revise the master sequence. </p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pone.0023167.s022.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Table S13</span>  <p> <b>EST tissue/organ source data for all 31 TRP classes.</b> </p> <p>(XLS)</p>  <div class="media" title="media"> <a href="pone.0023167.s023.xls">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Text S1</span>  <p> <b>TR and TRP taxonomies.</b> </p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pone.0023167.s024.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Text S2</span>  <p> <b>Additional details of plant Pro-rich TRP classes.</b> </p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pone.0023167.s025.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Text S3</span>  <p> <b>Manually curated list of Pro-rich TRPs.</b> A list of 912 non-redundant TRPs ("master sequences") catalogued in the TRP taxonomy ( <a href="#pone.0023167.s016">Tables S6</a>, <a href="#pone.0023167.s017">S7</a>, <a href="#pone.0023167.s018">S8</a>), 907 of which have a predicted signal peptide <a href="#pone.0023167-Emanuelsson1">[23]</a>. </p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pone.0023167.s026.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Text S4</span>  <p> <b>Additional genome sequence data.</b> </p> <p>(DOC)</p>  <div class="media" title="media"> <a href="pone.0023167.s027.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Dataset S1</span>  <p> <b>TR architectures of representative proteins from 31 Pro-rich TRP classes.</b> TR architectures of representative protein examples for each of 31 TRP classes ( <a href="#pone.0023167.s016">Tables S6</a>, <a href="#pone.0023167.s017">S7</a>, <a href="#pone.0023167.s018">S8</a>) are illustrated. All TRs were identified and aligned using XSTREAM <a href="#pone.0023167-Newman1">[47]</a> using the same parameters described in <a href="#s3"> <i>Materials and Methods</i> </a>. Each protein is shown N-terminus to C-terminus, from left to right and top to bottom. Major sequence features, in addition to TR domains, are indicated. For TR domains classified by the TR taxonomy (see <a href="#pone.0023167.s013">Tables S3</a>, <a href="#pone.0023167.s014">S4</a>, <a href="#pone.0023167.s015">S5</a>), the corresponding TR class is given in parentheses, e.g. TR_Domain1(mtp2). The scale-bar on top shows the number of amino acids from left to right in a given row. All 31 representative proteins are listed in alphabetical order and all images were rendered with JalView <a href="#pone.0023167-Waterhouse1">[57]</a>. </p> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0023167.s028.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Dataset S2</span>  <p> <b>Representative multiple sequence alignments.</b> Multiple sequence alignments of the 50 N-terminal and 50 C-terminal amino acids of full-length ORFs from 26 of 31 TRP classes (see <a href="#pone.0023167.s016">Tables S6</a>, <a href="#pone.0023167.s017">S7</a>, <a href="#pone.0023167.s018">S8</a>) are shown. Sequences were preprocessed to remove most of the TR domain to increase alignment quality. (PHEK and PRPB classes are not shown due to low quantities of complete ORFs; both EXTM and SPAP are heterogeneous protein classes, and are therefore not shown; PEHKs are shown in <a href="#pone.0023167.s003">Figure S3</a>). Each aligned TRP sequence (obtained from <a href="#pone.0023167.s026">Text S3</a>) has a predicted secretion signal, and is either derived from a genome sequence project, the NR database, or an EST, in which case a predicted stop codon was required. All sequence alignments were created with MUSCLE <a href="#pone.0023167-Edgar1">[58]</a> and rendered using JalView <a href="#pone.0023167-Waterhouse1">[57]</a>. </p> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0023167.s029.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank Drs. Ruth Finkelstein and Scott Hodges for critically reading this manuscript, and thank Dr. Scott Hodges for recommending the use of the PhyloWidget web tool for rendering taxonomic trees. </p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-COI-statement" title="COI-statement"> <p> <b>Competing Interests: </b>The authors have declared that no competing interests exist. </p> </div> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p> <b>Funding: </b>This work was partially supported by a University of California, Santa Barbara (UCSB) Dissertation Fellowship awarded to Dr. Newman. No additional external funding was received for this study. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. </p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="pone.0023167-Sarkar1" /> <span class="label" tagx="label" title="label">1</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sarkar</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bosneaga</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Auer</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Plant cell walls throughout evolution: towards a molecular understanding of their design principles. </span> <span class="source" tagx="source" title="source">J Exp Bot</span> <span class="volume" tagx="volume" title="volume">60</span> <span class="fpage" tagx="fpage" title="fpage">3615</span> <span class="lpage" tagx="lpage" title="lpage">3635</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19687127">19687127</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Popper1" /> <span class="label" tagx="label" title="label">2</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Popper</span> <span class="given-names" tagx="given-names" title="given-names">ZA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuohy</span> <span class="given-names" tagx="given-names" title="given-names">MG</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Beyond the green: understanding the evolutionary puzzle of plant and algal cell walls.</span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">153</span> <span class="fpage" tagx="fpage" title="fpage">373</span> <span class="lpage" tagx="lpage" title="lpage">383</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20421458">20421458</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Vogel1" /> <span class="label" tagx="label" title="label">3</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vogel</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Unique aspects of the grass cell wall.</span> <span class="source" tagx="source" title="source">Curr Opin Plant Biol</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">301</span> <span class="lpage" tagx="lpage" title="lpage">307</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18434239">18434239</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Somerville1" /> <span class="label" tagx="label" title="label">4</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Somerville</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bauer</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brininstool</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Facette</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hamann</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title">Toward a systems approach to understanding plant cell walls.</span> <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">306</span> <span class="fpage" tagx="fpage" title="fpage">2206</span> <span class="lpage" tagx="lpage" title="lpage">2211</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15618507">15618507</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Cosgrove1" /> <span class="label" tagx="label" title="label">5</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cosgrove</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span> </span> <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">Growth of the plant cell wall.</span> <span class="source" tagx="source" title="source">Nat Rev Mol Cell Biol</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">850</span> <span class="lpage" tagx="lpage" title="lpage">861</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16261190">16261190</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Dougall1" /> <span class="label" tagx="label" title="label">6</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dougall</span> <span class="given-names" tagx="given-names" title="given-names">DK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shimbayashi</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="year" tagx="year" title="year">1960</span> <span class="mixed-article-title" title="mixed-article-title">Factors affecting growth of tobacco callus tissue and its incorporation of tyrosine.</span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">35</span> <span class="fpage" tagx="fpage" title="fpage">396</span> <span class="lpage" tagx="lpage" title="lpage">404</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16655363">16655363</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Lamport1" /> <span class="label" tagx="label" title="label">7</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lamport</span> <span class="given-names" tagx="given-names" title="given-names">DTA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Northcote</span> <span class="given-names" tagx="given-names" title="given-names">DH</span> </span> </span> <span class="year" tagx="year" title="year">1960</span> <span class="mixed-article-title" title="mixed-article-title">Hydroxyproline in primary cell walls of higher plants.</span> <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">188</span> <span class="fpage" tagx="fpage" title="fpage">665</span> <span class="lpage" tagx="lpage" title="lpage">666</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/13757790">13757790</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Lamport2" /> <span class="label" tagx="label" title="label">8</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lamport</span> <span class="given-names" tagx="given-names" title="given-names">DTA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miller</span> <span class="given-names" tagx="given-names" title="given-names">DH</span> </span> </span> <span class="year" tagx="year" title="year">1971</span> <span class="mixed-article-title" title="mixed-article-title">Hydroxyproline-rich arabinosides in the plant kingdom.</span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">48</span> <span class="fpage" tagx="fpage" title="fpage">454</span> <span class="lpage" tagx="lpage" title="lpage">456</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16657818">16657818</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Kieliszewski1" /> <span class="label" tagx="label" title="label">9</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kieliszewski</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lamport</span> <span class="given-names" tagx="given-names" title="given-names">DTA</span> </span> </span> <span class="year" tagx="year" title="year">1994</span> <span class="mixed-article-title" title="mixed-article-title">Extensin: repetitive motifs, functional sites, post-translational codes, and phylogeny.</span> <span class="source" tagx="source" title="source">Plant J</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">157</span> <span class="lpage" tagx="lpage" title="lpage">172</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8148875">8148875</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Cooper1" /> <span class="label" tagx="label" title="label">10</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cooper</span> <span class="given-names" tagx="given-names" title="given-names">JB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heuser</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Varner</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> </span> <span class="year" tagx="year" title="year">1994</span> <span class="mixed-article-title" title="mixed-article-title">3,4-Dehydroproline inhibits cell wall assembly and cell division in tobacco protoplasts.</span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">104</span> <span class="fpage" tagx="fpage" title="fpage">747</span> <span class="lpage" tagx="lpage" title="lpage">752</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8159790">8159790</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Zhang1" /> <span class="label" tagx="label" title="label">11</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">XL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ren</span> <span class="given-names" tagx="given-names" title="given-names">YJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Roles of extensins in cotyledon primordium formation and shoot apical meristem activity in <i>Nicotiana tabacum</i>. </span> <span class="source" tagx="source" title="source">J Exp Bot</span> <span class="volume" tagx="volume" title="volume">59</span> <span class="fpage" tagx="fpage" title="fpage">4045</span> <span class="lpage" tagx="lpage" title="lpage">4058</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18931351">18931351</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Cannon1" /> <span class="label" tagx="label" title="label">12</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cannon</span> <span class="given-names" tagx="given-names" title="given-names">MC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Terneus</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hall</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tan</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Self-assembly of the plant cell wall requires an extensin scaffold.</span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="volume" tagx="volume" title="volume">105</span> <span class="fpage" tagx="fpage" title="fpage">2226</span> <span class="lpage" tagx="lpage" title="lpage">2231</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18256186">18256186</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Gille1" /> <span class="label" tagx="label" title="label">13</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gille</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">HÃ¤nsel</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ziemann</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pauly</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Identification of plant cell wall mutants by means of a forward chemical genetic approach using hydrolases. </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="volume" tagx="volume" title="volume">106</span> <span class="fpage" tagx="fpage" title="fpage">14699</span> <span class="lpage" tagx="lpage" title="lpage">14707</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19667208">19667208</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Johnson1" /> <span class="label" tagx="label" title="label">14</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">KL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schultz</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bacic</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="year" tagx="year" title="year">2003</span> <span class="mixed-article-title" title="mixed-article-title">Non-enzymic cell wall (glyco)proteins.</span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rose</span> <span class="given-names" tagx="given-names" title="given-names">JKC</span> </span> </span> <span class="source" tagx="source" title="source">The Plant Cell Wall, Vol. 8</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Florida</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">CRC Press</span> <span class="fpage" tagx="fpage" title="fpage">111</span> <span class="lpage" tagx="lpage" title="lpage">154</span> </span> </li> <li tag="ref"> <a name="pone.0023167-Showalter1" /> <span class="label" tagx="label" title="label">15</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Showalter</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keppler</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lichtenberg</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gu</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Welch</span> <span class="given-names" tagx="given-names" title="given-names">LR</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">A bioinformatics approach to the identification, classification, and analysis of hydroxyproline-rich glycoproteins. </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">153</span> <span class="fpage" tagx="fpage" title="fpage">485</span> <span class="lpage" tagx="lpage" title="lpage">513</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20395450">20395450</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Ellis1" /> <span class="label" tagx="label" title="label">16</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ellis</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Egelund</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schultz</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bacic</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Arabinogalactan-proteins: key regulators at the cell surface?</span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">153</span> <span class="fpage" tagx="fpage" title="fpage">403</span> <span class="lpage" tagx="lpage" title="lpage">419</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20388666">20388666</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Lamport3" /> <span class="label" tagx="label" title="label">17</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lamport</span> <span class="given-names" tagx="given-names" title="given-names">DT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kieliszewski</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cannon</span> <span class="given-names" tagx="given-names" title="given-names">MC</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Role of the extensin superfamily in primary cell wall architecture.</span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">156</span> <span class="fpage" tagx="fpage" title="fpage">11</span> <span class="lpage" tagx="lpage" title="lpage">19</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21415277">21415277</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Altschul1" /> <span class="label" tagx="label" title="label">18</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Altschul</span> <span class="given-names" tagx="given-names" title="given-names">SF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Madden</span> <span class="given-names" tagx="given-names" title="given-names">TL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">SchÃ¤ffer</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">1997</span> <span class="mixed-article-title" title="mixed-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">3389</span> <span class="lpage" tagx="lpage" title="lpage">3402</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694">9254694</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Finn1" /> <span class="label" tagx="label" title="label">19</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Finn</span> <span class="given-names" tagx="given-names" title="given-names">RD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mistry</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tate</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coggill</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heger</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">The Pfam protein families database.</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">38</span> <span class="fpage" tagx="fpage" title="fpage">D211</span> <span class="lpage" tagx="lpage" title="lpage">D222</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19920124">19920124</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Schultz1" /> <span class="label" tagx="label" title="label">20</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schultz</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rumsewicz</span> <span class="given-names" tagx="given-names" title="given-names">MP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">KL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gaspar</span> <span class="given-names" tagx="given-names" title="given-names">YM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">Using genomic resources to guide research directions. The arabinogalactan protein gene family as a test case. </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">129</span> <span class="fpage" tagx="fpage" title="fpage">1448</span> <span class="lpage" tagx="lpage" title="lpage">1463</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12177459">12177459</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Ma1" /> <span class="label" tagx="label" title="label">21</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Genome-wide identification, classification, and expression analysis of the arabinogalactan protein gene family in rice ( <i>Oryza sativa</i> L.). </span> <span class="source" tagx="source" title="source">J Exp Bot</span> <span class="volume" tagx="volume" title="volume">61</span> <span class="fpage" tagx="fpage" title="fpage">2647</span> <span class="lpage" tagx="lpage" title="lpage">2668</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20423940">20423940</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Graham1" /> <span class="label" tagx="label" title="label">22</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Graham</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Silverstein</span> <span class="given-names" tagx="given-names" title="given-names">KAT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cannon</span> <span class="given-names" tagx="given-names" title="given-names">SB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">VandenBosch</span> <span class="given-names" tagx="given-names" title="given-names">KA</span> </span> </span> <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title">Computational identification and characterization of novel genes from legumes.</span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">135</span> <span class="fpage" tagx="fpage" title="fpage">1179</span> <span class="lpage" tagx="lpage" title="lpage">1197</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15266052">15266052</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Emanuelsson1" /> <span class="label" tagx="label" title="label">23</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Emanuelsson</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brunak</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">von Heijne</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nielsen</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Locating proteins in the cell using TargetP, SignalP, and related tools.</span> <span class="source" tagx="source" title="source">Nat Protoc</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">953</span> <span class="lpage" tagx="lpage" title="lpage">971</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17446895">17446895</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Epstein1" /> <span class="label" tagx="label" title="label">24</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Epstein</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lamport</span> <span class="given-names" tagx="given-names" title="given-names">DTA</span> </span> </span> <span class="year" tagx="year" title="year">1984</span> <span class="mixed-article-title" title="mixed-article-title">An intramolecular linkage involving isodityrosine in extensin.</span> <span class="source" tagx="source" title="source">Phytochem</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">1241</span> <span class="lpage" tagx="lpage" title="lpage">1246</span> </span> </li> <li tag="ref"> <a name="pone.0023167-Schnabelrauch1" /> <span class="label" tagx="label" title="label">25</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schnabelrauch</span> <span class="given-names" tagx="given-names" title="given-names">LS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kieliszewski</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Upham</span> <span class="given-names" tagx="given-names" title="given-names">BL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alizedeh</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lamport</span> <span class="given-names" tagx="given-names" title="given-names">DTA</span> </span> </span> <span class="year" tagx="year" title="year">1996</span> <span class="mixed-article-title" title="mixed-article-title">Isolation of pI 4.6 extensin peroxidase from tomato cell suspension cultures and identification of Val-Tyr-Lys as putative intermolecular cross-link site. </span> <span class="source" tagx="source" title="source">Plant J</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">477</span> <span class="lpage" tagx="lpage" title="lpage">489</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8624511">8624511</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Uchida1" /> <span class="label" tagx="label" title="label">26</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Uchida</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takamichi</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jamet</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Furuya</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="year" tagx="year" title="year">1998</span> <span class="mixed-article-title" title="mixed-article-title">Control of expression of a gene encoding an extensin by phytochrome and a blue light receptor in spores of <i>Adiantum capillus-veneris</i> L. </span> <span class="source" tagx="source" title="source">Plant J</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">813</span> <span class="lpage" tagx="lpage" title="lpage">819</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9807820">9807820</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Banks1" /> <span class="label" tagx="label" title="label">27</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Banks</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Selaginella and 400 million years of separation.</span> <span class="source" tagx="source" title="source">Ann Rev Plant Biol</span> <span class="volume" tagx="volume" title="volume">60</span> <span class="fpage" tagx="fpage" title="fpage">223</span> <span class="lpage" tagx="lpage" title="lpage">238</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19575581">19575581</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Lee1" /> <span class="label" tagx="label" title="label">28</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">JH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Waffenschmidt</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Small</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goodenough</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Between-species analysis of short-repeat modules in cell wall and sex-related hydroxyproline-rich glycoproteins of <i>Chlamydomonas</i>. </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">144</span> <span class="fpage" tagx="fpage" title="fpage">1813</span> <span class="lpage" tagx="lpage" title="lpage">1826</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17573538">17573538</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-vanHolst1" /> <span class="label" tagx="label" title="label">29</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van Holst</span> <span class="given-names" tagx="given-names" title="given-names">GJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Varner</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span> </span> <span class="year" tagx="year" title="year">1984</span> <span class="mixed-article-title" title="mixed-article-title">Reinforced polyproline II conformation in a hydroxyproline-rich cell wall glycoprotein from carrot root. </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">74</span> <span class="fpage" tagx="fpage" title="fpage">247</span> <span class="lpage" tagx="lpage" title="lpage">251</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16663405">16663405</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Williamson1" /> <span class="label" tagx="label" title="label">30</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williamson</span> <span class="given-names" tagx="given-names" title="given-names">MP</span> </span> </span> <span class="year" tagx="year" title="year">1994</span> <span class="mixed-article-title" title="mixed-article-title">The structure and function of proline-rich regions in proteins.</span> <span class="source" tagx="source" title="source">Biochem J</span> <span class="volume" tagx="volume" title="volume">297</span> <span class="fpage" tagx="fpage" title="fpage">249</span> <span class="lpage" tagx="lpage" title="lpage">260</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8297327">8297327</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Baumberger1" /> <span class="label" tagx="label" title="label">31</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baumberger</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doesseger</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guyot</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Diet</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parsons</span> <span class="given-names" tagx="given-names" title="given-names">RL</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2003</span> <span class="mixed-article-title" title="mixed-article-title">Whole-genome comparison of leucine-rich repeat extensins in <i>Arabidopsis</i> and rice. A conserved family of cell wall proteins form a vegetative and a reproductive clade. </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">131</span> <span class="fpage" tagx="fpage" title="fpage">1313</span> <span class="lpage" tagx="lpage" title="lpage">1326</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12644681">12644681</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Kieliszewski2" /> <span class="label" tagx="label" title="label">32</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kieliszewski</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lamport</span> <span class="given-names" tagx="given-names" title="given-names">DTA</span> </span> </span> <span class="year" tagx="year" title="year">1987</span> <span class="mixed-article-title" title="mixed-article-title">Purification and partial characterization of a hydroxyproline-rich glycoprotein in a graminaceous monocot, <i>Zea mays</i>. </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">85</span> <span class="fpage" tagx="fpage" title="fpage">823</span> <span class="lpage" tagx="lpage" title="lpage">827</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16665784">16665784</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Stiefel1" /> <span class="label" tagx="label" title="label">33</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stiefel</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ruiz-Avila</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raz</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pilar VallÃ©s</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">GÃ³mez</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">1990</span> <span class="mixed-article-title" title="mixed-article-title">Expression of a maize cell wall hydroxyproline-rich glycoprotein gene in early leaf and root vascular differentiation. </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">785</span> <span class="lpage" tagx="lpage" title="lpage">793</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2152127">2152127</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Schultz2" /> <span class="label" tagx="label" title="label">34</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schultz</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">KL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Currie</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bacic</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="year" tagx="year" title="year">2000</span> <span class="mixed-article-title" title="mixed-article-title">The classical arabinogalactan protein gene family of <i>Arabidopsis</i>. </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">1751</span> <span class="lpage" tagx="lpage" title="lpage">1768</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11006345">11006345</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Rashid1" /> <span class="label" tagx="label" title="label">35</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rashid</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Functional characterization of a novel cell-wall annotated PELPK1 gene in <i>Arabidopsis thaliana</i>. </span> <span class="comment">Ph.D. dissertation, University of Alberta, Canada</span> </span> </li> <li tag="ref"> <a name="pone.0023167-Winter1" /> <span class="label" tagx="label" title="label">36</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Winter</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vinegar</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nahal</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ammar</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilson</span> <span class="given-names" tagx="given-names" title="given-names">GV</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">An "electronic fluorescent pictograph" browser for exploring and analyzing large-scale biological data sets. </span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">e718</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17684564">17684564</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Hruz1" /> <span class="label" tagx="label" title="label">37</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hruz</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laule</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Szabo</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wessendorp</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bleuler</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Genevestigator v3: a reference expression database for the meta-analysis of transcriptomes.</span> <span class="source" tagx="source" title="source">Adv Bioinformatics</span> <span class="volume" tagx="volume" title="volume">420747</span> </span> </li> <li tag="ref"> <a name="pone.0023167-Silverstein1" /> <span class="label" tagx="label" title="label">38</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Silverstein</span> <span class="given-names" tagx="given-names" title="given-names">KA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moskal</span> <span class="given-names" tagx="given-names" title="given-names">WA</span> <span class="suffix" tagx="suffix" title="suffix">Jr</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">HC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Underwood</span> <span class="given-names" tagx="given-names" title="given-names">BA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Graham</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Small cysteine-rich peptides resembling antimicrobial peptides have been under-predicted in plants. </span> <span class="source" tagx="source" title="source">Plant J</span> <span class="volume" tagx="volume" title="volume">51</span> <span class="fpage" tagx="fpage" title="fpage">262</span> <span class="lpage" tagx="lpage" title="lpage">280</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17565583">17565583</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Dvokov1" /> <span class="label" tagx="label" title="label">39</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">DvoÅÃ¡kovÃ¡</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">CvrÄkovÃ¡</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fischer</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Analysis of the hybrid proline-rich families from seven plant species suggests rapid diversification of their sequences and expression patterns. </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">412</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17997832">17997832</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Kader1" /> <span class="label" tagx="label" title="label">40</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kader</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span> </span> <span class="year" tagx="year" title="year">1996</span> <span class="mixed-article-title" title="mixed-article-title">Lipid-transfer proteins in plants.</span> <span class="source" tagx="source" title="source">Annu Rev Plant Phys</span> <span class="volume" tagx="volume" title="volume">47</span> <span class="fpage" tagx="fpage" title="fpage">627</span> <span class="lpage" tagx="lpage" title="lpage">654</span> </span> </li> <li tag="ref"> <a name="pone.0023167-Chae1" /> <span class="label" tagx="label" title="label">41</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chae</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kieslich</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morikis</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">SC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lord</span> <span class="given-names" tagx="given-names" title="given-names">EM</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">A gain-of-function mutation of Arabidopsis lipid transfer protein 5 disturbs pollen tube tip growth and fertilization. </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="volume" tagx="volume" title="volume">21</span> <span class="fpage" tagx="fpage" title="fpage">3902</span> <span class="lpage" tagx="lpage" title="lpage">3914</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20044438">20044438</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Brewin1" /> <span class="label" tagx="label" title="label">42</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brewin</span> <span class="given-names" tagx="given-names" title="given-names">NJ</span> </span> </span> <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title">Plant cell wall remodelling in the rhizobium-legume symbiosis.</span> <span class="source" tagx="source" title="source">CRC Cr Rev Plant Sci</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">293</span> <span class="lpage" tagx="lpage" title="lpage">316</span> </span> </li> <li tag="ref"> <a name="pone.0023167-Shewry1" /> <span class="label" tagx="label" title="label">43</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shewry</span> <span class="given-names" tagx="given-names" title="given-names">PR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Halford</span> <span class="given-names" tagx="given-names" title="given-names">NG</span> </span> </span> <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">Cereal seed storage proteins: structures, properties and role in grain utilization.</span> <span class="source" tagx="source" title="source">J Exp Bot</span> <span class="volume" tagx="volume" title="volume">53</span> <span class="fpage" tagx="fpage" title="fpage">947</span> <span class="lpage" tagx="lpage" title="lpage">958</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11912237">11912237</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Levitt1" /> <span class="label" tagx="label" title="label">44</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Levitt</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Nature of the protein universe.</span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="volume" tagx="volume" title="volume">106</span> <span class="fpage" tagx="fpage" title="fpage">11079</span> <span class="lpage" tagx="lpage" title="lpage">11084</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19541617">19541617</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Quackenbush1" /> <span class="label" tagx="label" title="label">45</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quackenbush</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liang</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holt</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pertea</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Upton</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="year" tagx="year" title="year">2000</span> <span class="mixed-article-title" title="mixed-article-title">The TIGR gene indices: reconstruction and representation of expressed gene sequences.</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">141</span> <span class="lpage" tagx="lpage" title="lpage">145</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10592205">10592205</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Childs1" /> <span class="label" tagx="label" title="label">46</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Childs</span> <span class="given-names" tagx="given-names" title="given-names">KL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hamilton</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ly</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheung</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">The TIGR plant transcript assemblies database.</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">35</span> <span class="fpage" tagx="fpage" title="fpage">D846</span> <span class="lpage" tagx="lpage" title="lpage">851</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17088284">17088284</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Newman1" /> <span class="label" tagx="label" title="label">47</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Newman</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cooper</span> <span class="given-names" tagx="given-names" title="given-names">JB</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">XSTREAM: a practical algorithm for identification and architecture modeling of tandem repeats in protein sequences. </span> <span class="source" tagx="source" title="source">BMC Bioinformatics</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">382</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17931424">17931424</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Pierleoni1" /> <span class="label" tagx="label" title="label">48</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pierleoni</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martelli</span> <span class="given-names" tagx="given-names" title="given-names">PL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Casadio</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">PredGPI: a GPI-anchor predictor.</span> <span class="source" tagx="source" title="source">BMC Bioinformatics</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">392</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18811934">18811934</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Newman2" /> <span class="label" tagx="label" title="label">49</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Newman</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cooper</span> <span class="given-names" tagx="given-names" title="given-names">JB</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">AutoSOME: A clustering method for identifying gene expression modules without prior knowledge of cluster number. </span> <span class="source" tagx="source" title="source">BMC Bioinformatics</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">117</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20202218">20202218</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Shannon1" /> <span class="label" tagx="label" title="label">50</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shannon</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Markiel</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ozier</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baliga</span> <span class="given-names" tagx="given-names" title="given-names">NS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">JT</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2003</span> <span class="mixed-article-title" title="mixed-article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks. </span> <span class="source" tagx="source" title="source">Genome Res</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">2498</span> <span class="lpage" tagx="lpage" title="lpage">2504</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14597658">14597658</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Letunic1" /> <span class="label" tagx="label" title="label">51</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Letunic</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doerks</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bork</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">SMART 6: recent updates and new developments.</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">37</span> <span class="fpage" tagx="fpage" title="fpage">D229</span> <span class="lpage" tagx="lpage" title="lpage">D232</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18978020">18978020</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Moore1" /> <span class="label" tagx="label" title="label">52</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moore</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soltis</span> <span class="given-names" tagx="given-names" title="given-names">PS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bell</span> <span class="given-names" tagx="given-names" title="given-names">CD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burleigh</span> <span class="given-names" tagx="given-names" title="given-names">JG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soltis</span> <span class="given-names" tagx="given-names" title="given-names">DE</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Phylogenetic analysis of 83 plastid genes further resolves the early diversification of eudicots. </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span> <span class="volume" tagx="volume" title="volume">107</span> <span class="fpage" tagx="fpage" title="fpage">4623</span> <span class="lpage" tagx="lpage" title="lpage">4628</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20176954">20176954</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Jordan1" /> <span class="label" tagx="label" title="label">53</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jordan</span> <span class="given-names" tagx="given-names" title="given-names">GE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Piel</span> <span class="given-names" tagx="given-names" title="given-names">WHJ</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">PhyloWidget: web-based visualizations for the tree of life.</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">24</span> <span class="fpage" tagx="fpage" title="fpage">1641</span> <span class="lpage" tagx="lpage" title="lpage">1642</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18487241">18487241</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Fauchere1" /> <span class="label" tagx="label" title="label">54</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fauchere</span> <span class="given-names" tagx="given-names" title="given-names">J-L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pliska</span> <span class="given-names" tagx="given-names" title="given-names">VE</span> </span> </span> <span class="year" tagx="year" title="year">1983</span> <span class="mixed-article-title" title="mixed-article-title">Hydrophobic parameters-pi of amino acid side chains from the partitioning of N-acetyl-amino-acid amides. </span> <span class="source" tagx="source" title="source">Eur J Med Chem</span> <span class="volume" tagx="volume" title="volume">18</span> <span class="fpage" tagx="fpage" title="fpage">369</span> <span class="lpage" tagx="lpage" title="lpage">375</span> </span> </li> <li tag="ref"> <a name="pone.0023167-Delage1" /> <span class="label" tagx="label" title="label">55</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">DelÃ©age</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roux</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> </span> <span class="year" tagx="year" title="year">1987</span> <span class="mixed-article-title" title="mixed-article-title">An algorithm for protein secondary structure prediction based on class prediction.</span> <span class="source" tagx="source" title="source">Protein Eng</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="fpage" tagx="fpage" title="fpage">289</span> <span class="lpage" tagx="lpage" title="lpage">294</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/3508279">3508279</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Creighton1" /> <span class="label" tagx="label" title="label">56</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Creighton</span> <span class="given-names" tagx="given-names" title="given-names">TE</span> </span> </span> <span class="year" tagx="year" title="year">1993</span> <span class="source" tagx="source" title="source">Proteins structures and molecular properties, 2 <sup>nd</sup> ed </span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">W.H. Freeman and Company</span> </span> </li> <li tag="ref"> <a name="pone.0023167-Waterhouse1" /> <span class="label" tagx="label" title="label">57</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Waterhouse</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Procter</span> <span class="given-names" tagx="given-names" title="given-names">JB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">DMA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clamp</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barton</span> <span class="given-names" tagx="given-names" title="given-names">GJ</span> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Jalview Version 2 â a multiple sequence alignment editor and analysis workbench.</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">1189</span> <span class="lpage" tagx="lpage" title="lpage">1191</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19151095">19151095</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Edgar1" /> <span class="label" tagx="label" title="label">58</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Edgar</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> </span> <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput.</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">32</span> <span class="fpage" tagx="fpage" title="fpage">1792</span> <span class="lpage" tagx="lpage" title="lpage">1797</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147">15034147</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Davies1" /> <span class="label" tagx="label" title="label">59</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davies</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robinson</span> <span class="given-names" tagx="given-names" title="given-names">SP</span> </span> </span> <span class="year" tagx="year" title="year">2000</span> <span class="mixed-article-title" title="mixed-article-title">Differential screening indicates a dramatic change in mRNA profiles during grape berry ripening. Cloning and characterization of cDNAs encoding putative cell wall stress response proteins. </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="volume" tagx="volume" title="volume">122</span> <span class="fpage" tagx="fpage" title="fpage">803</span> <span class="lpage" tagx="lpage" title="lpage">812</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10712544">10712544</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Thomas1" /> <span class="label" tagx="label" title="label">60</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thomas</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">MM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schiefelbein</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="year" tagx="year" title="year">2003</span> <span class="mixed-article-title" title="mixed-article-title">Molecular identification of proline-rich protein genes induced during root formation in grape ( <i>Vitis vinifera</i> L.) stem cuttings. </span> <span class="source" tagx="source" title="source">Plant Cell Environ</span> <span class="volume" tagx="volume" title="volume">26</span> <span class="fpage" tagx="fpage" title="fpage">1497</span> <span class="lpage" tagx="lpage" title="lpage">1504</span> </span> </li> <li tag="ref"> <a name="pone.0023167-Larkin1" /> <span class="label" tagx="label" title="label">61</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Larkin</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blackshields</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brown</span> <span class="given-names" tagx="given-names" title="given-names">NP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chenna</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McGettigan</span> <span class="given-names" tagx="given-names" title="given-names">PA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">ClustalW and ClustalX version 2.0.</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">2947</span> <span class="lpage" tagx="lpage" title="lpage">2948</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17846036">17846036</a> </span> </span> </li> <li tag="ref"> <a name="pone.0023167-Tamura1" /> <span class="label" tagx="label" title="label">62</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tamura</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dudley</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nei</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0.</span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="volume" tagx="volume" title="volume">24</span> <span class="fpage" tagx="fpage" title="fpage">1596</span> <span class="lpage" tagx="lpage" title="lpage">1599</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17488738">17488738</a> </span> </span> </li> </ul> </div> </div>  </body></html>