<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">BMC Genomics</span> <span class="iso-abbrev" title="iso-abbrev">BMC Genomics</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">BMC Genomics</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1471-2164</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5910586">5910586</a> </span> <span class="publisher-id" title="publisher-id">4626</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s12864-018-4626-9">10.1186/s12864-018-4626-9</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Comparative mitochondrial genomics of cryptophyte algae: gene shuffling and dynamic mobile genetic elements </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">Jong Im</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">jongim@cnu.ac.kr</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoon</span> <span class="given-names" tagx="given-names" title="given-names">Hwan Su</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">hwansu@gmail.com</span> </span> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yi</span> <span class="given-names" tagx="given-names" title="given-names">Gangman</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">gangman@dongguk.edu</span> </span> <a href="#Aff3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shin</span> <span class="given-names" tagx="given-names" title="given-names">Woongghi</span> </span> <span class="address" title="address"> <span class="phone" tagx="phone" title="phone">+1-902-494-2536</span> <span class="email" tagx="email" title="email">shinw@cnu.ac.kr</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Archibald</span> <span class="given-names" tagx="given-names" title="given-names">John M.</span> </span> <span class="address" title="address"> <span class="phone" tagx="phone" title="phone">+1-902-494-2536</span> <span class="email" tagx="email" title="email">John.Archibald@Dal.Ca</span> </span> <a href="#Aff4">4</a> </span> <span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0001 0722 6377</span> <span class="institution-id" title="institution-id">grid.254230.2</span> <span class="institution" title="institution">Department of Biology, </span> <span class="institution" title="institution">Chungnam National University, </span> </div> </span> <span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0001 2181 989X</span> <span class="institution-id" title="institution-id">grid.264381.a</span> <span class="institution" title="institution">Department of Biological Sciences, </span> <span class="institution" title="institution">Sungkyunkwan University, </span> </div> </span> <span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0001 0671 5021</span> <span class="institution-id" title="institution-id">grid.255168.d</span> <span class="institution" title="institution">Department of Multimedia Engineering, </span> <span class="institution" title="institution">Dongguk University, </span> </div> </span> <span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0004 1936 8200</span> <span class="institution-id" title="institution-id">grid.55602.34</span> <span class="institution" title="institution">Department of Biochemistry and Molecular Biology, </span> <span class="institution" title="institution">Dalhousie University, </span> </div> </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-4-4</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-4-4</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span> </span> <span class="volume" tagx="volume" title="volume">19</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">275</span> <span class="history" title="history"> <span class="received" title="received">received: 2017-10-24</span> <span class="accepted" title="accepted">accepted: 2018-3-27</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License ( <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver ( <a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Cryptophytes are an ecologically important group of algae comprised of phototrophic, heterotrophic and osmotrophic species. This lineage is of great interest to evolutionary biologists because their plastids are of red algal secondary endosymbiotic origin. Cryptophytes have a clear phylogenetic affinity to heterotrophic eukaryotes and possess four genomes: host-derived nuclear and mitochondrial genomes, and plastid and nucleomorph genomes of endosymbiotic origin. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>To gain insight into cryptophyte mitochondrial genome evolution, we sequenced the mitochondrial DNAs of five species and performed a comparative analysis of seven genomes from the following cryptophyte genera: <i>Chroomonas</i>, <i>Cryptomonas</i>, <i>Hemiselmis</i>, <i>Proteomonas</i>, <i>Rhodomonas</i>, <i>Storeatula</i> and <i>Teleaulax</i>. The mitochondrial genomes were similar in terms of their general architecture, gene content and presence of a large repeat region. However, gene order was poorly conserved. Characteristic features of cryptophyte mtDNAs included large syntenic clusters resembling α-proteobacterial operons that encode bacteria-like rRNAs, tRNAs, and ribosomal protein genes. The cryptophyte mitochondrial genomes retain almost all genes found in many other eukaryotes including the <i>nad</i>, <i>sdh</i>, <i>cox</i>, <i>cob</i>, and <i>atp</i> genes, with the exception of <i>sdh</i>2 and <i>atp</i>3. In addition, gene cluster analysis showed that cryptophytes possess a gene order closely resembling the jakobid flagellates <i>Jakoba</i> and <i>Reclinomonas</i>. Interestingly, the <i>cox</i>1 gene of <i>R. salina</i>, <i>T. amphioxeia</i>, and <i>Storeatula</i> species was found to contain group II introns encoding a reverse transcriptase protein, as did the <i>cob</i> gene of <i>Storeatula</i> species CCMP1868. </p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>These newly sequenced genomes increase the breadth of data available from algae and will aid in the identification of general trends in mitochondrial genome evolution. While most of the genomes were highly conserved, extensive gene arrangements have shuffled gene order, perhaps due to genome rearrangements associated with hairpin-containing mobile genetic elements, tRNAs with palindromic sequences, and tandem repeat sequences. The <i>cox</i>1 and <i>cob</i> gene sequences suggest that introns have recently been acquired during cryptophyte evolution. Comparison of phylogenetic trees based on plastid and mitochondrial genome data sets underscore the different evolutionary histories of the host and endosymbiont components of present-day cryptophytes. </p> </div> <div class="electronicsupplementarymaterial" title="sec"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <p>The online version of this article (10.1186/s12864-018-4626-9) contains supplementary material, which is available to authorized users. </p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Cryptophytes</div> <div class="kwd" title="kwd">
Genome re-arrangement</div> <div class="kwd" title="kwd">
Mitochondrial genome</div> <div class="kwd" title="kwd">
Mobile genetic elements</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100003725</span> <span class="institution" title="institution">National Research Foundation of Korea</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">NRF-2013R1A1A3012539, 2015R1D1A1A01057899</span> <span class="award-id" tagx="award-id" title="award-id">2017R1A2B3001923</span> <span class="award-id" tagx="award-id" title="award-id">2016R1D1A1A09919318</span> <span class="award-id" tagx="award-id" title="award-id">2015R1A2A2A01003192, 2015M1A5A1041808</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">Jong Im</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoon</span> <span class="given-names" tagx="given-names" title="given-names">Hwan Su</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yi</span> <span class="given-names" tagx="given-names" title="given-names">Gangman</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shin</span> <span class="given-names" tagx="given-names" title="given-names">Woongghi</span> </span> </span> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100003566</span> <span class="institution" title="institution">Ministry of Oceans and Fisheries</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">the Collaborative Genome Program (20140428)</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoon</span> <span class="given-names" tagx="given-names" title="given-names">Hwan Su</span> </span> </span> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100002471</span> <span class="institution" title="institution">Dongguk University</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">Dongguk University Research Fund of 2016</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yi</span> <span class="given-names" tagx="given-names" title="given-names">Gangman</span> </span> </span> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100000038</span> <span class="institution" title="institution">Natural Sciences and Engineering Research Council of Canada</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">an operating grant from the Natural Sciences and Engineering Research Council of Canada</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Archibald</span> <span class="given-names" tagx="given-names" title="given-names">John M.</span> </span> </span> </div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2018" /> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>The cryptophyte algae are an evolutionarily significant unicellular eukaryotic lineage inhabiting marine, brackish, and freshwater environments [ <a href="#CR1">1</a>, <a href="#CR2">2</a>]. Cryptophytes are comprised of photosyntheticic, heterotrophic and osmotrophic species [ <a href="#CR3">3</a>– <a href="#CR9">9</a>]; phototrophs contain plastids with chlorophyll- <i>a</i> and - <i>c</i>, as well as phycobilins as accessary pigments. Cryptophytes have a clear phylogenetic affinity to heterotrophic eukaryotes, including goniomonads, kathablepharids and <i>Palpitomonas bilix</i>, which collectively have been proposed to comprise a monophyletic assemblage known as Cryptista [ <a href="#CR10">10</a>– <a href="#CR12">12</a>]. Cryptophyte cells contain four genomes: host-derived nuclear and mitochondrial genomes, and plastid and nucleomorph genomes of endosymbiotic origin. Given this unusual feature, cryptophytes provide direct evidence for secondary endosymbiosis, a process whereby a photoautotrophic eukaryote is engulfed by a phagotrophic eukaryote [ <a href="#CR13">13</a>, <a href="#CR14">14</a>]. Secondary endosymbiosis has given rise to photosynthetic groups in other protist lineages as well (e.g., euglenoids and chlorarachniophytes from green-algal derived secondary endosymbioses, and cryptophytes haptophytes, stramenopiles, and dinoflagellates, whose plastids are of red-algal origin). </p> <p>Relatively little is known about mitochondrial diversity in cryptophytes and their non-photosynthetic relatives. Considering eukaryotes as a whole, mitochondrial genomes vary significantly in size, gene content, and genome organization [ <a href="#CR15">15</a>– <a href="#CR17">17</a>]. The least derived mitochondrial genomes known are found in an obscure protist group called jakobids and harbor up to 100 genes (including protein-coding genes and genes for noncoding RNA molecules) [ <a href="#CR18">18</a>], whereas the mitochondrial genomes of Myzozoa (a subgroup of alveolates) code for only three proteins [ <a href="#CR19">19</a>]. In terms of genome architecture, the most common structure is a contiguous circular-mapping DNA molecule, but a contiguous linear genome or genomes segmented into multiple circular or linear molecules are known from some taxa [ <a href="#CR15">15</a>, <a href="#CR19">19</a>]. Although considerable progress has been made in exploring the diversity and evolution of mitochondrial genomes across the full breadth of eukaryote phylogeny, representative genome sequences are still limited or altogether lacking for a large number of eukaryotic lineages, especial protists [ <a href="#CR16">16</a>]. Hence, further sampling is necessary to get a full picture of the evolutionary history of mitochondria and their genomes. </p> <p>To date, only two cryptophyte mitochondrial genomes have been sequenced, those of <i>Rhodomonas salina</i> and <i>Hemiselmis andersenii</i> [ <a href="#CR20">20</a>, <a href="#CR21">21</a>]. More recently, the mitochondrial genome of <i>Palpitomonas bilix</i>, a member of the Cryptista, was sequenced [ <a href="#CR22">22</a>]. Here we present the sequences of five mitochondrial genomes belonging to the following organisms: the blue/green-colored cryptophyte <i>Chroomonas placoidea</i>, the brown-colored <i>Cryptomonas curvata</i> and the red-colored <i>Proteomonas sulcata</i>, <i>Storeatula</i> sp. CCMP1868 and <i>Teleaulax amphioxeia</i>. We performed a comparative analysis of genome structure and gene re-arrangements and investigated the phylogeny of mitochondrial genes relative to nuclear and plastid genes. Our results provide important insights into the broad evolution of organelle genomes and fine-scale dynamics within cryptophytes. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="generalfeaturesofcryptophytemitochondrialgenomes" title="sec"> <div class="title" tagx="title" title="title">
General features of cryptophyte mitochondrial genomes</div> <p>Mitochondrial genomes (mtDNAs) were sequenced and analyzed from representatives of three different colored cryptophytes: the red-coloured <i>Proteomonas</i>, <i>Rhodomonas</i>, <i>Storeatula</i> and <i>Teleaulax</i>, the green-coloured <i>Chroomonas</i> and <i>Hemiselmis</i>, and the brown-colored <i>Cryptomonas</i> species (Table <a href="#Tab1">1</a>). The mitochondrial genomes of two species, <i>Storeatula</i> sp. CCMP1868 and <i>Teleaulax amphioxeia</i>, could not be completely assembled, due to the presence of complex repeat regions (Fig. <a href="#Fig1">1</a>), as seen previously (e.g., in the haptophyte <i>Phaeocystis</i>; [ <a href="#CR23">23</a>]). All of the newly sequenced cryptophyte mitochondrial genomes were found to have a highly repetitive non-coding region (Table <a href="#Tab1">1</a>), as previously described in two other cryptophyte mitochondrial genomes [ <a href="#CR20">20</a>, <a href="#CR21">21</a>]. The mitochondrial genomes ranged in size from ~ 37 Kbp ( <i>Cryptomonas curvata</i>) to ~ 54.5 Kbp ( <i>Storeatula</i> sp. CCMP 1868), not including repeat regions (Table <a href="#Tab1">1</a>). They share a core set of 2 rRNAs, 25~ 28 tRNAs and 42 protein-coding genes, including 18 components of the respiratory chain, 5 ATP synthase subunits, 16 ribosomal proteins and 2 subunits of the <i>tat</i> translocase (Tables <a href="#Tab2">2</a>, <a href="#Tab3">3</a>). Notably, the mitochondrion-encoded <i>rps</i>1 and <i>tat</i>A genes in cryptophytes were previously found to be present exclusively in jakobids and/or malawimonads, believed to possess the most ‘primitive’ mitochondrial genomes known [ <a href="#CR18">18</a>]. Minor variation was found in tRNA gene content (Table <a href="#Tab2">2</a>). For instance, <i>trn</i>K(UUU) is present in three red-colored cryptophytes, <i>R. salina</i>, <i>Storeatula</i> sp. CCMP1868, and <i>T. amphioxeia</i>, but not <i>Proteomonas sulcata</i>, while the blue/green-colored <i>Ch. placoidea</i> and <i>H. andersenii</i> and the brown-colored <i>Cr. curvata</i> have a unique isotype <i>trn</i>G(GCC) (Table <a href="#Tab2">2</a>). <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  </div></p><p>Characteristics of cryptophyte mitochondrial genomes analyzed in this study</p>  <table> <thead> <tr> <th>General characteristics</th> <th> <i>Proteomonas sulcata</i> <br /><br />CCMP705 </th> <th> <i>Teleaulax amphioxeia</i> <br /><br />HACCP CR01 </th> <th> <i>Storeatula</i> species <br /><br />CCMP1868 </th> <th> <i>Rhodomonas salina</i> <br /><br />CCMP1319 </th> <th> <i>Cryptomonas curvata</i> <br /><br />FBCC300012D </th> <th> <i>Hemiselmis andersenii</i> <br /><br />CCMP644 </th> <th> <i>Chroomonas placoidea</i> <br /><br />CCAP978/8 </th> </tr> </thead> <tbody> <tr> <td>Plastid color</td> <td>red</td> <td>red</td> <td>red</td> <td>red</td> <td>brown</td> <td>green</td> <td>green</td> </tr> <tr> <td>Genome size (bp)</td> <td>37,009</td> <td>43,442</td> <td>54,527</td> <td>43,375</td> <td>37,438</td> <td>40,898</td> <td>44,384</td> </tr> <tr> <td>(with Repeats)</td> <td>(48,238)</td> <td>(unknown)</td> <td>(unknown))</td> <td>(48,063)</td> <td>(39,464)</td> <td>(60,553)</td> <td>(46,700)</td> </tr> <tr> <td>G + C (%)</td> <td>31.2</td> <td>32.8</td> <td>29.0</td> <td>29.8</td> <td>25.81</td> <td>30.0</td> <td>29.4</td> </tr> <tr> <td>repeat region</td> <td>not inverted</td> <td>inverted</td> <td>inverted</td> <td>inverted</td> <td>not inverted</td> <td>not inverted</td> <td>not inverted</td> </tr> <tr> <td>present</td> <td>present</td> <td>present</td> <td>present</td> <td>present</td> <td>present</td> <td>present</td> </tr> <tr> <td>(1.1Kb)</td> <td /> <td /> <td>(4.7Kb)</td> <td>(2.03Kb)</td> <td>(19.7Kb)</td> <td>(2.3 Kb)</td> </tr> <tr> <td>Total gene (include RNAs)</td> <td>69</td> <td>73</td> <td>73</td> <td>71</td> <td>71</td> <td>74</td> <td>79</td> </tr> <tr> <td>Gene direction (+/−)</td> <td>69/0</td> <td>42/31</td> <td>40/33</td> <td>51/20</td> <td>29/42</td> <td>74/0</td> <td>79/0</td> </tr> <tr> <td>No. of protein-coding genes</td> <td>42</td> <td>45</td> <td>43</td> <td>42</td> <td>42</td> <td>44</td> <td>50</td> </tr> <tr> <td>Unknown ORFs</td> <td>1</td> <td>4</td> <td>2</td> <td>1</td> <td>1</td> <td>3</td> <td>6</td> </tr> <tr> <td>tRNAs</td> <td>25</td> <td>26</td> <td>28</td> <td>27</td> <td>27</td> <td>28</td> <td>27</td> </tr> <tr> <td>introns</td> <td /> <td>group II</td> <td>group II</td> <td>group II</td> <td /> <td /> <td /> </tr> <tr> <td>Genes with intron (no. of intron within gene)</td> <td /> <td>cox1(2)</td> <td>cox1(3)/cob(1)</td> <td>cox1(3)</td> <td /> <td /> <td>cox1*(2)</td> </tr> <tr> <td>gene fission</td> <td /> <td /> <td /> <td /> <td /> <td /> <td>cox1 (exon1-IEP-maturase)/cox1 (exon2)</td> </tr> <tr> <td>GenBank accession</td> <td>MG680945</td> <td>MG680944</td> <td>MG680943</td> <td>NC_002572</td> <td>MG680942</td> <td>NC_010637</td> <td>MG680941</td> </tr> </tbody> </table>  <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 1</span> </a> </div>   <p>Circular map of the mitochondrial genome of seven cryptophytes. The protein coding genes, rRNA and tRNA genes (single letter) are labeled outside the circle. The genes are color-coded according to the functional categories in the index </p>   </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  <p>Transfer RNAs (tRNAs) found in cryptophyte mitochondrial genomes</p>  <table> <thead> <tr> <th /> <th> <i>Chroomonas placoidea</i> </th> <th> <i>Hemiselmis andersenii</i> </th> <th> <i>Cryptomonas curvata</i> </th> <th> <i>Rhodomonas salina</i> </th> <th> <i>Storeatula</i> species </th> <th> <i>Teleaulax amphioxeia</i> </th> <th> <i>Proteomonas sulcata</i> </th> </tr> </thead> <tbody> <tr> <td>trnA(TGC)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnC(ACA)</td> <td /> <td /> <td /> <td>1</td> <td /> <td /> <td /> </tr> <tr> <td>trnC(GCA)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnD(GTC)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnE(TTC)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnF(GAA)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>2</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnG(GCC)</td> <td>1</td> <td>1</td> <td>1</td> <td /> <td /> <td /> <td /> </tr> <tr> <td>trnG(TCC)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnH(GTG)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnI(GAT)</td> <td>1</td> <td>2</td> <td>1</td> <td>2</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnI(TAT)</td> <td /> <td /> <td /> <td>1</td> <td /> <td /> <td /> </tr> <tr> <td>trnK(TTT)</td> <td /> <td /> <td /> <td>1</td> <td>1</td> <td>1</td> <td /> </tr> <tr> <td>trnfK(TTT)</td> <td /> <td>1</td> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>trnL(GAG)</td> <td>1</td> <td>1</td> <td>1</td> <td /> <td /> <td>1</td> <td /> </tr> <tr> <td>trnL(TAA)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnL(TAG)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnfM(CAT)</td> <td /> <td>1</td> <td /> <td /> <td /> <td /> <td /> </tr> <tr> <td>trnM(CAT)</td> <td>3</td> <td>1</td> <td>3</td> <td>2</td> <td>3</td> <td>3</td> <td>3</td> </tr> <tr> <td>trnN(GTT)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnP(TGG)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnQ(TTG)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnR(GCG)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td /> <td>1</td> </tr> <tr> <td>trnR(TCG)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnR(TCT)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnS(GCT)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnS(TGA)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnT(TGT)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnV(TAC)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnW(CCA)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnY(GTA)</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> <td>1</td> </tr> <tr> <td>trnY(ATA)</td> <td>1</td> <td>1</td> <td /> <td>1</td> <td /> <td /> <td /> </tr> <tr> <td>Total</td> <td>28</td> <td>29</td> <td>27</td> <td>28</td> <td>28</td> <td>26</td> <td>25</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3</span>  <p>Functional protein coding genes in the cryptophyte mitochondrial genome (41 total)</p>  <table> <thead> <tr> <th>Classification</th> <th>Genes</th> </tr> </thead> <tbody> <tr> <td>Electron transport and ATP synthesis</td> </tr> <tr> <td> NADH dehydrogenase (complex I) subunits</td> <td> <i>nad</i>1 </td> <td> <i>nad</i>2 </td> <td> <i>nad</i>3 </td> <td> <i>nad</i>4 </td> <td> <i>nad</i>4L </td> </tr> <tr> <td> <i>nad</i>5 </td> <td> <i>nad</i>6 </td> <td> <i>nad</i>7 </td> <td> <i>nad</i>8 </td> <td> <i>nad</i>9 </td> </tr> <tr> <td> <i>nad</i>10 </td> <td> <i>nad</i>11 </td> <td /> <td /> <td /> </tr> <tr> <td> Succinate dehydrogenase (complex II) subunits</td> <td> <i>sdh</i>3 </td> <td> <i>sdh</i>4 <br /><br />(HAM_72) </td> <td /> <td /> <td /> </tr> <tr> <td> Cytochrome <i>bc1</i> complex (complex III) subunits </td> <td> <i>cob</i> </td> <td /> <td /> <td /> <td /> </tr> <tr> <td> Cytochrome <i>c</i> oxidase (complex IV) subunits </td> <td> <i>cox</i>1 </td> <td> <i>cox</i>2 </td> <td> <i>cox</i>3 </td> <td /> <td /> </tr> <tr> <td> ATP synthase (complex V) subunits</td> <td> <i>atp</i>1 </td> <td> <i>atp</i>4 <br /><br />(ymf39) </td> <td> <i>atp</i>6 </td> <td> <i>atp</i>8 </td> <td> <i>atp</i>9 </td> </tr> <tr> <td>Translation</td> </tr> <tr> <td> SSU ribosomal proteins</td> <td> <i>rps</i>1 <br /><br />(orf207) </td> <td> <i>rps</i>2 </td> <td> <i>rps</i>3 </td> <td> <i>rps</i>4 </td> <td> <i>rps</i>7 </td> </tr> <tr> <td> <i>rps</i>8 </td> <td> <i>rps</i>11 </td> <td> <i>rps</i>12 </td> <td> <i>rps</i>13 </td> <td> <i>rps</i>14 </td> </tr> <tr> <td> <i>rps</i>19 </td> <td /> <td /> <td /> <td /> </tr> <tr> <td> LSU ribosomal proteins</td> <td> <i>rpl</i>5 </td> <td> <i>rpl</i>6 </td> <td> <i>rpl</i>14 </td> <td> <i>rpl</i>16 </td> <td> <i>rpl</i>31 <br /><br />(orf72) </td> </tr> <tr> <td>Protein import</td> </tr> <tr> <td> SecY-independent transporters</td> <td> <i>tat</i>A </td> <td> <i>tat</i>C <br /><br />(ymf16) </td> <td /> <td /> <td /> </tr> </tbody> </table> </div> <p /> <p>Further consideration of mitochondrial genome structure revealed that while all genes were located on the same strand in the <i>Chroomonas placoidea</i>, <i>Hemiselmis andersenii</i> and <i>Proteomonas sulcata</i> genomes, some genes were located on the opposite strand in <i>Cryptomonas curvata</i> (31 genes), <i>Storeatula</i> sp. CCMP1868 (41 genes), <i>Teleaulax amphioxeia</i> (39 genes), and <i>Rhodomonas salina</i> (53 genes) (Figs. <a href="#Fig1">1</a>, <a href="#Fig2">2</a>, Table <a href="#Tab1">1</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 2</span> </a> </div>   <p>Gene content and arrangement of cryptophyte mitochondrial genomes. Twelve synthetic blocks (A–L) and two rRNAs are described relative to a phylogenetic tree of cryptophytes. Putative gene rearrangements are indicated with numbers (1–23). Tandem repeat regions are marked with red triangles. The most conserved syntenic blocks among cryptophytes are marked; * = 100%, and † = broken by one gene. The hairpin structures with palindromic sequences are marked blue (hairpin symbol). The tRNAs are coded with single letters and tRNAs with palindromic sequences are highlighted red </p>   </div> <p /> </div> <div class="genomere-arrangements" title="sec"> <div class="title" tagx="title" title="title">
Genome re-arrangements</div> <p>The most evident feature of the cryptophyte mitochondrial genomes sequenced herein was that, although gene content is very stable, gene order is highly variable. The 42 protein-coding genes were arranged together and analyzed in the context of 12 syntenic blocks including rRNA and tRNA genes, each block consisting of 2–6 genes (Fig. <a href="#Fig2">2</a>). These blocks were as follows: A) <i>cox</i>1 <i>-cob-nad</i>11, B) <i>atp</i>8 <i>-atp</i>4 <i>-rps</i>4 <i>-atp</i>9, C) <i>nad</i>3 <i>-rps</i>2, D) <i>tat</i>C <i>-nad</i>7, E) <i>cox</i>2 <i>-tat</i>A <i>-cox</i>3 <i>-rpl</i>12 <i>-rpl</i>7 <i>-rps</i>19, F) <i>rps</i>3 <i>-rpl</i>16 <i>-rpl</i>14 <i>-rpl</i>5 <i>-rps</i>14, G) <i>nad</i>8 <i>-nad</i>6 <i>-nad</i>1, H) <i>sdh</i>3 <i>-sdh</i>4, I) <i>nad</i>4L <i>-nad</i>5 <i>-nad</i>4 <i>-nad</i>2, J) <i>atp</i>1 <i>-rps</i>1 <i>-orf</i>166 <i>, rpl</i>31, K) <i>rps</i>8 <i>-rpl</i>6 <i>-rps</i>13 <i>-rps</i>11 <i>-atp</i>6, L) <i>nad10-nad9</i>, and 2 rRNAs (Fig. <a href="#Fig2">2</a>). Two syntenic blocks were found to be common to all cryptophyte mitochondrial genomes (Fig. <a href="#Fig2">2</a>, J and L). </p> <p>To assess the extent of mitochondrial genome rearrangements more closely, genomes were aligned using the Mauve genome aligner (Additional file <a href="#MOESM1">1</a>: Figure S1). Synteny was broadly conserved within the three differently colored cryptophyte lineages. Overall, we documented 23 instances of mitochondrial genome rearrangements, suggesting that extensive scrambling has occurred since the evolutionary split of these species (Fig. <a href="#Fig2">2</a> and Additional file <a href="#MOESM1">1</a>: Figure S1). Comparing <i>Proteomonas sulcata</i> with the three other red-colored cryptophyte mitochondrial genomes, 14 gene-order rearrangements were detected (Fig. <a href="#Fig2">2</a> ①–⑭). The genes in the mitochondrial genome of <i>Teleaulax amphioxeia</i> appear to have been rearranged via transposition of synteny block C (①), an inversion of a set of consecutive genes in blocks G to L (②), and a combination of transposition and inversion involving synteny block H (③). Genes in the mitochondrial genome of <i>Rhodomonas salina</i> were rearranged with transposition of <i>nad</i>3 and <i>rps</i>2 from broken block C (in a manner different than that seen for <i>T. amphioxeia</i>) (④, ⑤), transposition of block D–E (⑥), an inversion of a set of consecutive genes in blocks G to J (⑦), and transposition of <i>atp</i>6 within block K (⑧). The mitochondrial genome of <i>Storeatula</i> sp. CCMP1868 was rearranged with transposition of synteny block G–L (⑨), transposition of <i>tat</i>C and <i>nad</i>7 from a broken block D (⑩), an inversion of block E (⑪), an inversion of block F (⑫), inversion of block B with rRNAs and <i>nad</i>11 being split from block A (⑬), and an inversion of block C (⑭). </p> <p>In terms of mitochondrial gene order, the brown-colored cryptophyte <i>Cryptomonas curvata</i> is more similar to the red-colored lineage than to the blue-greens. Relative to the genome of the red-colored <i>Proteomonas sulcata</i>, four gene-order rearrangements were detected (Fig. <a href="#Fig2">2</a> ⑮–⑱): a combination of an inversion and break in block E–F (⑮–⑯), an inversion of a string of genes in blocks G to L (⑰), and a combination of inversion and transposition block C (⑱). The gene order of blue/green-colored cryptophytes is very different from the other groups. Between the two blue/green-colored cryptophytes <i>Hemiselmis andersenii</i> and <i>Chroomonas placoidea</i>, five gene-order rearrangements were detected (Fig. <a href="#Fig2">2</a> ⑲– <span class="inline-graphic" title="inline-graphic" />): transposition of G and partial I block (⑲), transposition of <i>rps</i>4 (⑳) and <i>atp</i>9 ( <span class="inline-graphic" title="inline-graphic" />) from broken block B, transposition of <i>sdh</i>4 from broken block H ( <span class="inline-graphic" title="inline-graphic" />), and transposition of <i>cox</i>2 break free from the block E ( <span class="inline-graphic" title="inline-graphic" />). Clearly there have been extensive mitochondrial genome rearrangements in each of these lineages during cryptophyte evolution. </p> </div> <div class="palindromicsequencesandlargenon-codingregionswithtandemrepeats" title="sec"> <div class="title" tagx="title" title="title">
Palindromic sequences and large non-coding regions with tandem repeats</div> <p>We found between 3 and 20 A-T rich palindromic sequences in the intergenic regions of each of the mitochondrial genomes of cryptophytes (Fig. <a href="#Fig3">3</a>, Additional files <a href="#MOESM2">2</a> and <a href="#MOESM3">3</a>: Figures S2 and S3), similar to those reported previously in the large non-coding regions of <i>Rhodomonas salina</i> [ <a href="#CR20">20</a>] and <i>Hemiselmis andersenii</i> [ <a href="#CR21">21</a>]. Interestingly, between 4 and 9 tRNA genes are present at the palindromic endpoints (Additional file <a href="#MOESM3">3</a>: Figure S3). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 3</span> </a> </div>   <p>Representative tRNA palindromic sequences in the mitochondrial genome of <i>Proteomonas sulcata</i>. The non-tRNA sequences are marked with blue and palindromic sequences are marked with purple. All hairpin structures or tRNAs with palindromic sequences in cryptophtye mitochondrial genome are illustrated in Additional file <a href="#MOESM2">2</a>: Figure S2 and Additional file <a href="#MOESM3">3</a>: Figure S3 </p>   </div> <p /> <p>Cryptophyte mtDNA structure also appears to have been impacted by the presence of tandem repeats in the large non-coding region, with individual repeats ranging from 1.1Kbp ( <i>P. sulcata</i>) to 19.7Kbp ( <i>H. andersenii</i>), each of which contains multi-copy palindromic sequences [ <a href="#CR20">20</a>, <a href="#CR21">21</a>] (Figs. <a href="#Fig1">1</a> and <a href="#Fig2">2</a> marked as triangle). The repeats are inverted in the genomes of <i>R. salina</i>, <i>Storeatula</i> species CCMP1186 and <i>T. amphioxeia</i>, but in the other four completely assembled genomes, large non-coding regions are instead dispersed or arranged in tandem throughout the large non-coding region (Table <a href="#Tab1">1</a>). </p> </div> <div class="ribosomalproteinsandbacterial-likeoperons" title="sec"> <div class="title" tagx="title" title="title">
Ribosomal proteins and bacterial-like operons</div> <p>Ribosomal protein gene clusters have been suggested to represent vestiges of bacterial operons in the cryptophyte mitochondrial genome [ <a href="#CR20">20</a>]. Our comparisons of mitochondrial gene data provide further support for this idea. The ribosomal protein clusters <i>rps</i>3 <i>-rpl</i>16 <i>-rpl</i>14 <i>-rpl</i>5 <i>-rps</i>14 and <i>rps</i>8 <i>- rpl</i>6 <i>-rps</i>13 <i>-rps</i>11 in <i>Ch. placoidea</i>, <i>H. andersenii</i>, <i>Cr. curvata</i>, <i>R. salina</i> and <i>P. sulcata</i> exhibit the same relative gene order as in the operons of the bacteria <i>Escherichia coli</i> and <i>Rickettsia prowazekii</i>; this arrangement is also seen (with little variation) in the gene-rich mtDNAs of several other protists ( <i>Jakoba libera, Acanthamoeba catellanii</i>, <i>Phytophthora infestans</i>, and the green alga <i>Nephroselmis olivacea</i>) as well as the liverwort <i>M. polymorpha</i> (Additional file <a href="#MOESM4">4</a>: Figure S4). However, the bacterial-operon cluster <i>rps</i>8 <i>- rpl</i>6 <i>-rps</i>13 <i>-rps</i>11 was inverted in the cryptophytes <i>Storeatula</i> sp. CCMP 1868 and <i>T. amphioxeia</i> (Additional file <a href="#MOESM4">4</a>: Figure S4). Furthermore, the other conserved ribosomal protein gene cluster, <i>rps</i>12 <i>-rps</i>7 <i>-rps</i>19, is translocated in the mitochondrial genome of <i>Storeatula</i> sp. CCMP1868. </p> </div> <div class="phylogeny" title="sec"> <div class="title" tagx="title" title="title">
Phylogeny</div> <p>An ML tree was reconstructed using 4257 amino acids from 16 representative genes conserved in the mtDNAs of all chlorophyll- <i>c</i> containing algal groups (Fig. <a href="#Fig4">4</a>), as well as diverse photosynthetic and non-photosynthetic lineages from across the eukaryotic tree. The sequences of dinoflagellates and euglenoids were not included in this analysis due to the limited available data of mitochondrial genes. A monophyletic cryptophyte clade was strongly supported (MLB = 100%) and showed well-resolved internal relationships. The four red-colored taxa (i.e., <i>Storeatula</i>, <i>Rhodomonas</i>, <i>Teleaulax</i> and <i>Guillardia</i>) group together while the brown-colored <i>Cryptomonas curvata</i> forms a sister relationship with the blue/green-colored <i>Chroomonas</i> and <i>Hemiselmis</i> species (Fig. <a href="#Fig4">4</a>). Although our taxon sampling was limited, these results are consistent with earlier studies. In particular, the red-colored cryptophyte species ( <i>Rhodomonas</i>, <i>Storeatula</i>, <i>Teleaulax</i>, and <i>Proteomonas</i>) form a monophyletic lineage in both mitochondrial and plastid phylogenomic studies to the exclusion of other cryptophytes ([ <a href="#CR24">24</a>], this study), in contrast to single gene analyses of nuclear SSU rDNA [ <a href="#CR6">6</a>, <a href="#CR8">8</a>]. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 4</span> </a> </div>   <p>Phylogenetic tree of cryptophyte mitochondrial proteins. The tree was constructed using a dataset of 16 concatenated proteins (4257 amino acids). The numbers on each node represent RAxML bootstrap values. The scale bar indicates the inferred number of substitutions/site </p>   </div> <p /> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Mitochondrial DNA gene content was found to be essentially identical amongst all cryptophytes examined, although gene order rearrangements were detected. Similar mtDNA architectures, including gene-dense regions, a single large repetitive intergenic region and absolute strand polarity, are seen in the stramenopile <i>Thraustochytrium aureum</i> and the green algae <i>Pedinomonas minor</i> and <i>Pycnococcus provasolii</i> [ <a href="#CR25">25</a>, <a href="#CR26">26</a>]. Structurally speaking, the mitochondrial genomes of haptophycean algae are similar to those of cryptophytes, and show variation of strand polarity within the group. For example, absolute strand polarity is seen in the mtDNAs of <i>Chrysochromulina tobin</i> and <i>Emiliania huxleyi</i> but not in <i>Pavlova lutheri</i> (32/49 genes) and <i>Phaeocystis globosa</i> (40/45 genes) [ <a href="#CR23">23</a>, <a href="#CR27">27</a>, <a href="#CR28">28</a>]. </p> <p>Interestingly, the mitochondrial genomes of various other major algal lineages have also undergone extensive rearrangements. These include the green algae [ <a href="#CR29">29</a>, <a href="#CR30">30</a>], red algae [ <a href="#CR31">31</a>], Eustigmatophyceae [ <a href="#CR32">32</a>], haptophytes [ <a href="#CR28">28</a>], and chlorarachniophytes [ <a href="#CR33">33</a>]. The significant structural variation seen in cryptophyte mitochondrial genomes is interesting when compared to the plastid genomes of these same organisms, where gene order was recently shown to be nearly identical [ <a href="#CR24">24</a>]. </p> <div class="palindromicsequencesandrepeatstructuresrelativetorecombinationsites" title="sec"> <div class="title" tagx="title" title="title">
Palindromic sequences and repeat structures relative to recombination sites</div> <p>The fact that palindromic sequences are found concentrated near the ends of syntenic blocks in cryptophyte mitochondrial genomes suggests that they have played a role in mediating genomic rearrangements. In fungal mtDNAs, double-hairpin elements have been suggested to act as mobile DNA elements and to mediate lateral gene transfer [ <a href="#CR34">34</a>, <a href="#CR35">35</a>]. In any case, a notable consequence of the presence of such repeats is that they facilitate genome rearrangements [ <a href="#CR36">36</a>– <a href="#CR39">39</a>]. A number of possible mechanisms have been proposed. Inversions may occur in a specific location due to the presence of short repeat elements subject to homologous recombination [ <a href="#CR40">40</a>, <a href="#CR41">41</a>]. Even in un-rearranged plastid genomes, small inversions regularly occur in intergenic areas, caused by short inverted repeats forming hairpins that can easily flip the orientation of the intervening sequences [ <a href="#CR42">42</a>, <a href="#CR43">43</a>]. Such a mechanism could explain the highly shuffled genes seen in the cryptophytes mitochondrial genomes described herein (Fig. <a href="#Fig2">2</a>). In addition, tRNA genes associated with gene re-arrangement breakpoints have been reported in plastid genomes [ <a href="#CR41">41</a>, <a href="#CR44">44</a>]. It has been suggested that tRNA genes at inversion endpoints may be due to the presence of short repeats within or near the tRNA genes in the highly rearranged chloroplast genome of the charophyte <i>Chaetospheridium globosum</i> [ <a href="#CR45">45</a>], as well as in the flowering plant <i>Trachelium</i> [ <a href="#CR46">46</a>]. </p> <p>The direct repeat arrangements we observed are strikingly similar to the larger (35 kb) repeat structure found in the mitochondrial genome of the diatom <i>Phaeodactylum tricornutum</i> [ <a href="#CR47">47</a>]. The haptophytes <i>Chrysochromulina tobin</i>, <i>Emiliania hyxleyi</i>, <i>Phaeocystis antarctica</i>, <i>Phaeocystis globosa</i> and <i>Pavlova lutheri</i>, and the chlorophytes <i>Pedinomonas minor</i> and <i>Acutodesmus obliquus</i> also contain large tandem repeat regions (&amp;gt;4 kb) in their genomes [ <a href="#CR23">23</a>, <a href="#CR25">25</a>, <a href="#CR27">27</a>, <a href="#CR28">28</a>, <a href="#CR48">48</a>]. In animal mitochondrial genomes, the tandem repeats are located in the control region and help explain the evolutionary origins of tandem repeats among species, populations and even individuals [ <a href="#CR49">49</a>, <a href="#CR50">50</a>]. However, in most algal mitochondrial genomes the tandem repeat manifests itself in a pattern that is species-specific. The repeat sequences within the mitochondrial genomes of cryptophytes described herein do not retain obvious sequence or structural similarity across species bounds. </p> </div> <div class="genecontentofcryptophytemitochondrialgenomes" title="sec"> <div class="title" tagx="title" title="title">
Gene content of cryptophyte mitochondrial genomes</div> <p>The most gene-rich mitochondrial genomes reported to date are those of the excavate group Jakobida, with 61–69 protein-coding genes and 30–34 RNAs [ <a href="#CR18">18</a>]. Cryptophyte mitochondrial genomes are also noteworthy in retaining genes that are not found in most other eukaryotes (Fig. <a href="#Fig5">5</a>). The mtDNAs of cryptophytes contain 42 conserved protein genes (excluding non-conserved ORFs in each genome), more than found in glaucophytes (30 CDSs), Bacillariophyceae (34 CDSs), Eustigmatophyceae (36 CDSs), Phaeophyceae (34 CDSs), Rhaphidophyceae (34 CDSs), rhodophytes (22 CDSs), haptophytes (22 CDSs), Synurophyceae (32 CDSs), chlorarachniophytes (26 CDSs) and green algae (11–39 CDSs). Most of the cryptophyte mitochondrial genes are associated with electron transport systems belonging to a set of five complexes, as summarized below. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 5</span> </a> </div>   <p>Venn diagram comparing gene content among the mitochondrial genomes of chlorophyll- <i>c</i> containg algae. Only genes/proteins with predicted functions are included. Total protein gene numbers are indicated for each group (parentheses). The few genes found in specific subgroups are colored: light-blue in Cyanidiophyceae, green in Chryso/Bacillario/Raphiophyceae, light-green in Eustigmatophyceae, purple in the genus <i>Pavlova</i> (Haptophyta). The 16 genes conserved among all groups used to construct phylogenetic trees are highlighted red </p>   </div> <p /> <p>The genes encoding certain subunits of NADH dehydrogenase (complex I; <i>nad</i>7, <i>nad</i>8, <i>nad</i>9, <i>nad</i>10, and <i>nad</i>11) are only rarely found in mtDNA. The <i>nad</i>7, <i>nad</i>9 and <i>nad</i>11 genes are missing from all haptophyte and rhodophyte mitochondrial genomes. On the other hand, these three genes are present in all cryptophytes, <i>Palpitomonas</i>, stramenopiles and jakobid mitochondrial genomes (Fig. <a href="#Fig5">5</a>). Comparison of the mtDNA gene order in red-/brown-colored cryptophytes reveals the presence of three NADH dehydrogenase gene clusters (cluster G: <i>nad</i>8 <i>-nad</i>6 <i>-nad</i>1, cluster I: <i>nad</i>4L- <i>nad</i>5- <i>nad</i>4- <i>nad</i>2, and cluster L: <i>nad</i>10- <i>nad</i>9). However, two clusters (G and I) were separated into one or two genes, respectively (G: <i>nad</i>8- <i>nad</i>1 and <i>nad</i>6, cluster I: <i>nad</i>4L- <i>nad</i>5 and <i>nad</i>4- <i>nad</i>2), in blue/green-colored cryptophytes (Fig. <a href="#Fig2">2</a>). </p> <p>The succinate dehydrogenase (complex II) is made up of four protein subunits; two are hydrophilic and belong to the catalytic portion of the complex ( <i>sdh</i>1 and <i>sdh</i>2 genes) and the other two, encoded by the <i>sdh</i>3 and <i>sdh</i>4 genes, are hydrophobic and act as anchors to the entire complex [ <a href="#CR51">51</a>, <a href="#CR52">52</a>]. The <i>sdh</i>2, <i>sdh</i>3, and <i>sdh</i>4 genes are sometimes found in mitochondrial genomes, whereas <i>sdh</i>1 is transferred to the nuclear genome and its protein product is imported from the cytosol. The <i>sdh</i>2 gene is present in Rhodophyta except in <i>Galdieria sulphuraria</i> [ <a href="#CR31">31</a>]. Many other algae have also lost the <i>sdh</i>2 gene from their mitochondrial genomes. Mitochondrial genes for two subunits of succinate dehydrogenase ( <i>sdh</i>3 and <i>sdh</i>4) are present in only a few green algae, the liverwort <i>Marchantia polymorpha</i>, several red algae [ <a href="#CR31">31</a>, <a href="#CR53">53</a>], all jakobid flagellates including <i>Reclinomonas americana</i> [ <a href="#CR54">54</a>, <a href="#CR55">55</a>] and all known cryptophytes ([ <a href="#CR20">20</a>, <a href="#CR21">21</a>], this study). </p> <p>The genes for subunits of the cytochrome <i>bc</i> <sub> <i>1</i> </sub> complex (complex III) and cytochrome <i>c</i> oxidase (complex IV) are present in the mtDNAs of all cryptophytes and most algae. The <i>cob</i> and <i>cox</i>1 genes are always together in all cryptophyte genomes sequenced thus far (Fig. <a href="#Fig2">2</a>A). The <i>cox</i>2 and <i>cox</i>3 genes are found together in almost all cryptophytes, the exception being the <i>cox</i>2 gene in <i>Hemiselmis andersenii</i>, which is separated from cluster E (Fig. <a href="#Fig2">2</a>E). Interestingly, the <i>cox</i>1 gene of <i>Storeatula</i> sp. CCMP1868, <i>Teleaulax amphioxeia</i>, and <i>Rhodomonas salina</i> was found to contain a putative group II intron with a coding region for an intron encoded protein (IEP). In <i>Chroomonas placoidea</i>, the <i>cox</i>1 gene was split into two parts with 13 genes in between. The first exon encodes a maturase sequence and is located between the <i>nad</i>8- <i>trn</i>D and <i>atp</i>8 genes. The <i>cob</i> gene in <i>Storeatula</i> sp. CCMP1868 also has an intron with an IEP (Additional file <a href="#MOESM5">5</a>: Figure S5). These introns belong to group II, which often harbor three distinct protein-coding regions corresponding to reverse transcriptase, maturase, and C-terminal DNA binding domains [ <a href="#CR56">56</a>]. The <i>Storeatula</i> sp. intron coding regions show similarity to an IEP in the <i>atp</i>A gene in the mitochondrial genome of the liverwort <i>Marchantia polymorpha</i> [ <a href="#CR57">57</a>]. </p> <p>The mtDNAs of jakobid flagellates contain the biggest complement of ATP synthase genes ( <i>atp</i>1, 3, 4, 6, 8, 9). Three <i>atp</i> genes ( <i>atp</i>6, 8, 9) are present in the mitochondrial genomes of green algae, glaucophytes, haptophytes and stramenopiles (Fig. <a href="#Fig5">5</a>). The <i>atp</i>1 and <i>atp</i>4 genes are present in the mitochondrial genome of green algae, jakobids and cryptophytes. The heterolobosean <i>Naegleria fowleri</i> and <i>Tsukubamonas globose</i> (a member of the Tsukubamonadidae) have <i>atp</i>3 in their mtDNA but lack <i>atp</i>4. All cryptophytes and <i>Palpitomonas bilix</i> possess most of the <i>atp</i> gene set ( <i>atp</i>1, 4, 6, 8, 9) with the exception of <i>atp</i>3 ([ <a href="#CR22">22</a>], this study). </p> <p>The two genes for subunits of the twin-arginine protein translocation system transporters (encoded by the <i>tat</i>A and <i>tat</i>C genes) are generally conserved in diverse prokaryotes as well as in plastids [ <a href="#CR58">58</a>]. The <i>tat</i>C homologs are present in some mitochondrial genomes [ <a href="#CR59">59</a>], and the mitochondrial <i>tat</i>A gene was thought to be limited to the jakobids [ <a href="#CR60">60</a>]. However, a recent review of algal mitochondrial genomes [ <a href="#CR61">61</a>] noted the presence of <i>tat</i>A in some eukaryotic groups, including diatoms, raphidophytes, chrysophytes and cryptophytes; these genes are found in our newly sequenced cryptophyte genome as well (Fig. <a href="#Fig5">5</a>). </p> </div> <div class="phylogeneticrelationships" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic relationships</div> <p>Curiously, our 16-protein mitochondrial genome phylogeny shows a strong monophyletic relationship between cryptophytes and rhodophytes. This is unexpected given that the cryptophyte mitochondrion is derived from the host component of the original secondary endosymbiotic partnership that gave rise to modern-day cryptophytes. Based on the monophyletic relationship between chlorophyll- <i>c</i> containing groups and red algae seen in many (but not all) plastid genome phylogenies, the hypothesis that a single secondary endosymbiotic uptake of a red alga in a common ancestor of all ‘chromist’ algae has been explored [ <a href="#CR24">24</a>, <a href="#CR62">62</a>– <a href="#CR70">70</a>]. However, recent large-scale studies using nuclear genome data show topologies that are incongruent with this hypothesis. The occurrence of multiple secondary and serial endosymbioses has been proposed to reconcile the apparent incongruence between host and endosymbiont-associated phylogenies (e.g., [ <a href="#CR12">12</a>, <a href="#CR69">69</a>, <a href="#CR71">71</a>– <a href="#CR74">74</a>]). We examined phylogenies inferred from each of the single genes/proteins used in our multi-gene tree, and could see no consistent signal for a relationship between cryptophytes and rhodophytes to the exclusion of other eukaryotic groups (Additional file <a href="#MOESM6">6</a>: Figure S6). In only one of our protein trees ( <i>cob</i>) did the cryptophytes branch with <i>Palpitomonas bilix</i>, to which it is clearly related on the basis of nuclear multi-gene trees (e.g., [ <a href="#CR10">10</a>, <a href="#CR12">12</a>]). Clearly more mitochondrial genome data are needed from plastid-lacking lineages that are related to cryptophytes, such as goniomonads and kathablepharids. Nevertheless, our mitochondrial genome-based phylogeny is consistent with previously published phylogenies, including a 250 nuclear gene dataset [ <a href="#CR12">12</a>], in suggesting that the cryptophytes and haptophytes are not specifically related to one other. The monophyly of haptophytes + Opisthokonta/Amoebozoa + Stramenopiles/Rizaria + other relatives is supported by nuclear and mitochondrial analyses ([ <a href="#CR12">12</a>], this study). </p> </div> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>We have sequenced mitochondrial genomes from five cryptophyte algae with a wide range of color pigmentation: the red-colored <i>Proteomonas sulcata</i>, <i>Storeatula</i> sp. CCMP1868 and <i>Teleaulax amphioxeia</i>, the blue/green-colored <i>Chroomonas placoidea</i>, and the brown <i>Cryptomonas curvata</i>. These newly sequenced genomes will aid in the identification of general trends in mitochondrial genome evolution, not just within cryptophytes but between cryptophytes and their closest plastid-lacking relatives such as goniomonads and kathablepharids once these sequence data become available. While most of the cryptophyte genomes we have sequenced are highly conserved with respect to coding capacity, extensive gene arrangements have shuffled gene order. Such rearrangements appear to have been mediated by palindromic sequences, tRNAs, and/or repeat regions. This pattern lies in stark contrast to the high degree of synteny seen in the plastid genomes of these very same organisms, underscoring the dramatic differences in the tempo and mode of organellar genomes of different evolutionary history, even when they reside within the same cell. </p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="dnaisolationandsequencing" title="sec"> <div class="title" tagx="title" title="title">
DNA isolation and sequencing</div> <p>Cultures of <i>Chroomonas placoidea</i> CCAP 978/8 and <i>Proteomonas sulcata</i> CCMP 705 were obtained from the Culture Collection of Algae and Protozoa (CCAP), whereas <i>Storeatula</i> species CCMP 1868 came from the National Center for Marine Algae and Microbiota (NCMA). <i>Teleaulax amphioxeia</i> collected from Gomso Bay, Korea (35° 40′ N, 126° 40′ E) was established as clonal cultures from single-cell isolates and the strain is available from the Culture collection at the Chungnam National University, Korea. <i>Cryptomonas curvata</i> collected from Cheongyang pond, Korea (36° 30′ N, 126° 47′ E) was established as clonal cultures from single-cell isolates and the strain <i>Cryptomonas curvata</i> FBCC 300012D is available from the Freshwater Bioresources Culture Collection at the Nakdong-gang National Institute of Biological Resources Korea. All cultures were grown in AF-6 medium [ <a href="#CR75">75</a>] with distilled water for the freshwater strain ( <i>Cr. curvata</i>) or distilled seawater for marine strains, and were maintained at 20 °C under conditions of a 14:10 light:dark cycle with 30 μmol photons·m <sup>− 2</sup>·s <sup>− 1</sup> from cool white fluorescent tubes. DNA was extracted using the QIAGEN DNEasy Blood Mini Kit (QIAGEN, Valencia, CA, USA) as per manufacturer’s instructions. Next-generation sequencing (NGS) was carried out using the Ion Torrent PGM platform (Thermo Fisher Scientific, San Francisco, California, USA). Sequencing libraries were prepared using the Ion Xpress™ Plus gDNA Fragment Library Preparation Kit for 200 bp or 400 bp-sized sequencing library prearation and the Ion OneTouch™ 200 or 400 Template Kit (Thermo Fisher Scientific, San Francisco, CA, USA) according to the manufacturer’s protocol. Genomes were sequenced on an Ion Torrent Personal Genome Machine (PGM) using the Ion PGM sequencing 200 or 400 Kit (Thermo Fisher Scientific, San Francisco, CA, USA). On the MiSeq (Illumina, San Diego CA), the amplified DNA was fragmented and tagged using the NexteraXT protocol (Illumina), indexed, size selected, and pooled for sequencing using the small amplicon targeted resequencing run, which performs paired end 2 × 300 bp sequencing reads using the MiSeq Reagent Kit v3 (Illumina), according to the manufacturer’s recommendations. </p> </div> <div class="genomeassemblyandannotation" title="sec"> <div class="title" tagx="title" title="title">
Genome assembly and annotation</div> <p>The raw reads obtained from both NGS platforms (i.e., Ion Torrent and Illumina MiSeq) were trimmed separately using the following settings: base = 80 bp, error threshold = 0.05, n ambiguities = 2. Assemblies were also carried out separately. For the Ion Torrent data, the assembly was carried out using MIRA4 ( <a href="http://mira-assembler.sourceforge.net/docs/DefinitiveGuideToMIRA.html">http://mira-assembler.sourceforge.net/docs/DefinitiveGuideToMIRA.html</a>), whereas SPAdes 3.10 ( <a href="http://bioinf.spbau.ru/spades">http://bioinf.spbau.ru/spades</a>) was used for Illumina data. For each genome, the two assemblies were compared and the most ‘complete’ assembly was chosen. The mitochondrial origin of the assembled contigs was verified according to the following criteria and Jung et al. [ <a href="#CR76">76</a>]: (i) reads were mapped onto the final consensus contig for each cryptophyte mitochondrial genome using Bowtie2 (similarity = 95%, length fraction = 75%; <a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a>) with preset options (sensitive-local: -D 15 -R 2 -N 0 -L 20 -i s,1,0.75); (ii) BLAST searches using commonly known mitochondrial genes against the entire assembly produced hits to these contigs; and (iii) genome sizes consistent with those of other photosynthetic cryptophyte mitochondrial genomes were obtained. </p> <p>Protein coding genes as well as rRNA and tRNA genes were compiled from all previously sequenced cryptophyte mitochondrial genomes. Preliminary annotation of protein coding genes was performed using GeneMarkS ( <a href="http://opal.biology.gatech.edu/GeneMark/">http://opal.biology.gatech.edu/GeneMark/</a>). The final annotation file was checked in Geneious Pro 10.2.2 ( <a href="http://www.geneious.com">http://www.geneious.com</a>/) using the ORFfinder with genetic code 4 (Protozoan Mitochondrial Code). Predicted ORFs were checked manually and annotated accordingly. </p> <p>Transfer RNA genes were identified using the tRNAscan-SE version 2.0 server ( <a href="http://lowelab.ucsc.edu/tRNAscan-SE">http://lowelab.ucsc.edu/tRNAscan-SE/</a>) with default settings and the "Mold/Protozoan Mito" model. The rRNA genes were identified by BLASTn searches against a set of known, previously published mitochondrial rRNA sequences of cryptophytes. To determine intron types, mitochondrial genomes were submitted to the RNAweasel server ( <a href="http://megasun.bch.umontreal.ca/cgi-bin/RNAweasel/RNAweaselInterface.pl">http://megasun.bch.umontreal.ca/cgi-bin/RNAweasel/RNAweaselInterface.pl</a>). Physical maps were designed with the OrganellarGenomeDRAW program ( <a href="http://ogdraw.mpimp-golm.mpg.de">http://ogdraw.mpimp-golm.mpg.de</a>/). </p> <p>Palindromic sequence elements were detected using the EMBOSS explorer ( <a href="http://emboss.bioinformatics.nl/cgi-bin/emboss/palindrome">http://emboss.bioinformatics.nl/cgi-bin/emboss/palindrome</a>) with the following parameters: 8 for minimum length of palindromes, 100 for maximum length of palindromes, 10 for maximum gap between elements, and no mismatches within the palindrome. To find palindromic sequences in tRNAs, the maximum gap between elements was set at 100 bp (due to the normal tRNA gene length of less 90 bp). Some detected sequences were manually excluded if a loop region was longer than a stem region. Genome sequences were deposited into the NCBI GenBank database under the following accession numbers (Table <a href="#Tab1">1</a>). </p> </div> <div class="genearrangementcomparisons" title="sec"> <div class="title" tagx="title" title="title">
Gene arrangement comparisons</div> <p>Two previously published cryptophyte mitochondrial genome sequences were downloaded from GenBank [ <a href="#CR20">20</a>, <a href="#CR21">21</a>]. For structural and synteny comparisons, the genomes were aligned using the Mauve Genome Alignment tool version 2.2.0 [ <a href="#CR77">77</a>] with default settings. For the purposes of visualization, we arbitrarily designated the beginning of the <i>cox1</i> gene as position 1 in each genome (pointing in the direction of <i>cob</i>). </p> </div> <div class="phylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic analysis</div> <p>Phylogenetic trees were constructed from datasets created by combining amino acid sequences corresponding to 16 protein coding genes from 131 mitochondrial genomes, including those of seven cryptophytes, five haptophytes, 38 stramenopiles, and 36 red algae (Additional file <a href="#MOESM7">7</a>: Table S1). The 16 genes are as follows: <i>atp</i>6, <i>atp</i>8, <i>atp</i>9, <i>cob</i>, <i>cox</i>1, <i>cox</i>2, <i>cox</i>3, <i>nad</i>1, <i>nad</i>2, <i>nad</i>3, <i>nad</i>4, <i>nad</i>4L, <i>nad</i>5, <i>nad</i>6, <i>rps</i>12, and <i>rpl</i>16. The dataset was concatenated (4257 amino acid positions) and final refinements were performed by eyes in the MacGDE2.5 program [ <a href="#CR78">78</a>]. </p> <p>Maximum likelihood (ML) phylogenetic analyses were performed using RAxML version 8.0.0 [ <a href="#CR79">79</a>] with the Le and Gascuel gamma (LG + GAMMA) model [ <a href="#CR80">80</a>] for amino acid data chosen by ProtTest 3 [ <a href="#CR81">81</a>]. We used 1000 independent tree inferences using the -# option to identify the best tree. The model parameters with gamma correction values and the proportion of invariable sites in the combined dataset were obtained automatically by the RAxML program. Bootstrap support values (MLBS) were calculated using 1000 replicates with the same substitution model. </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Additional files</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"> <a href="12864_2018_4626_MOESM1_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 1:</span>  <p> <b>Figure S1.</b> Overview of cryptophyte mitochondrial genomes. Linearized maps of the five novel complete mitochondrial genomes are compared to those from previous studies. The color coded syntenic blocks are shown above each genome, and the gene maps are shown below. The syntenic blocks above the horizontal line are on the same strand, and those below the line are on the opposite strand. The horizontal bars inside the syntenic blocks show sequence conservation. The block boundaries correspond to sites where inversion events have occurred. In the gene maps, the genes above the horizontal line are transcribed from left to right, and those below the horizontal line are transcribed from right to left. The rRNA genes are shown in red. (PDF 1860 kb) </p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="12864_2018_4626_MOESM2_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 2:</span>  <p> <b>Figure S2.</b> Hairpin structures wih palindromic sequence in cryptophyte mitochondrial genomes. The non-tRNA sequences are marked with blue and palindromic sequences are marked with purple. (PDF 897 kb) </p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="12864_2018_4626_MOESM3_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 3:</span>  <p> <b>Figure S3.</b> All tRNAs wih palindromic sequences in cryptophyte mitochondrial genomes. The non-tRNA sequences are marked with blue and palindromic sequences are marked with purple. (PDF 1644 kb) </p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="12864_2018_4626_MOESM4_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 4:</span>  <p> <b>Figure S4.</b> Conservation of ribosomal protein gene organization. Gene order found in cryptophyte mitochondrial genomes compared with that of the contiguous bacterial str, S10 spec and alpha operons of <i>Escherichia coli</i> and <i>Rickettsia prowazekii</i>. (PDF 548 kb) </p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="12864_2018_4626_MOESM5_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 5:</span>  <p> <b>Figure S5.</b> Group II introns in cryptophyte <i>cox1</i> and <i>cob</i> mitochondrial genes. (PDF 352 kb) </p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="12864_2018_4626_MOESM6_ESM.pdf">LINK</a> <span class="label" tagx="label" title="label">Additional file 6:</span>  <p> <b>Figure S6.</b> Phylogenetic trees inferred from single genes. The trees were constructed using amino acid sequences of 16 genes: <i>atp</i>6, <i>atp</i>8, <i>atp</i>9, <i>cob</i>, <i>cox</i>1, <i>cox</i>2, <i>cox</i>3, <i>nad</i>1, <i>nad</i>2, <i>nad</i>3, <i>nad</i>4, <i>nad</i>4L, <i>nad</i>5, <i>nad</i>6, <i>rps</i>12, and <i>rpl</i>16. The numbers on each node represent ultrafast bootstrap approximation (UFBoot) using IQ-Tree. The scale bars indicate the number of substitutions/site. (PDF 2672 kb) </p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="12864_2018_4626_MOESM7_ESM.xlsx">LINK</a> <span class="label" tagx="label" title="label">Additional file 7:</span>  <p> <b>Table S1.</b> The concatenated data set of 16 protein coding genes for phylogenetic reconstruction. (XLSX 22 kb) </p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
CDS</div> <div class="def" title="def"> <p>Coding sequences</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
IEP</div> <div class="def" title="def"> <p>Intron encoded protein</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ML</div> <div class="def" title="def"> <p>Maximum likelihood</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MLB</div> <div class="def" title="def"> <p>Maximum likelihood bootstrap</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
mtDNAs</div> <div class="def" title="def"> <p>mitochondrial genomes</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NGS</div> <div class="def" title="def"> <p>Next-generation sequencing</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ORF</div> <div class="def" title="def"> <p>Open reading frame</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PGM</div> <div class="def" title="def"> <p>Personal Genome Machine</p> </div> </div> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Electronic supplementary material</b> </p> <p>The online version of this article (10.1186/s12864-018-4626-9) contains supplementary material, which is available to authorized users. </p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>The authors thank Jaehee Jung for bioinformatic assistance and two anonymous reviewers for their helpful comments. </p> <div class="funding" title="sec"> <div class="title" tagx="title" title="title">
Funding</div> <p>This research was supported by the National Research Foundation (NRF) of Korea funded by the Ministry of Science, ICT &amp;amp; Future Planning, Basic Science Research Program (MSIP; NRF-2013R1A1A3012539) and the Ministry of Education (2015R1D1A1A01057899) to JIK; NRF (2017R1A2B3001923) and the Collaborative Genome Program (20140428) funded by the Ministry of Oceans and Fisheries, Korea to HSY; NRF (2016R1D1A1A09919318), and the Dongguk University Research Fund of 2016 to GY; NRF (MSIP; 2015R1A2A2A01003192 and 2015M1A5A1041808) to WS; and an operating grant from the Natural Sciences and Engineering Research Council of Canada to JMA. </p> </div> <div class="availabilityofdataandmaterials" title="sec"> <div class="title" tagx="title" title="title">
Availability of data and materials</div> <p>The mitochondrial genome datasets in this project has been deposited in the NCBI GenBank database under the accession numbers MG680941, MG680942, MG680943, MG680944, MG680945. The sequence alignments used for phylogenetic analyses have been deposited in the Treebase repository (accession number 22274). Permission to sample from Gomso Bay and Cheongyang pond was obtained from the Korean government. The new strains described in this study are available from the corresponding author or Culture Collections. </p> </div> </div> <div class="title" tagx="title" title="title">
Authors’ contributions</div> <p>JIK, HSY, WS, JMA conceived and designed the experiments: JIK, GY performed the experiments and analyzed the data: JIK, HSY, WS, JMA wrote the manuscript: All authors have read and approved the final version of the manuscript. </p> <div class="ethicsapprovalandconsenttoparticipate" title="sec"> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p>Not applicable.</p> </div> <div class="consentforpublication" title="sec"> <div class="title" tagx="title" title="title">
Consent for publication</div> <p>Not applicable.</p> </div> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing interests</div> <p>The authors declare that they have no competing interests.</p> </div> <div class="publisher’snote" title="sec"> <div class="title" tagx="title" title="title">
Publisher’s Note</div> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Graham</span> <span class="given-names" tagx="given-names" title="given-names">LK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilcox</span> <span class="given-names" tagx="given-names" title="given-names">LW</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The origin of alteration of generations in land plants: a focus on matrotrophy and hexose transport </span> <span class="source" tagx="source" title="source">Philo Trans R Soc Lond B Biol Sci</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">255</span> <span class="fpage" tagx="fpage" title="fpage">757</span> <span class="lpage" tagx="lpage" title="lpage">766</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1098/rstb.2000.0614">10.1098/rstb.2000.0614</a> </span> </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shalchian-Tabrizi</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Minge</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Espelund</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Orr</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ruden</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Multigene phylogeny of choanozoa and the origin of animals</span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">e2098</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0002098">10.1371/journal.pone.0002098</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18461162">18461162</a> </span> </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hill</span> <span class="given-names" tagx="given-names" title="given-names">DRA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rowan</span> <span class="given-names" tagx="given-names" title="given-names">KS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The biliproteins of the Cryptophyceae</span> <span class="source" tagx="source" title="source">Phycologia</span> <span class="year" tagx="year" title="year">1989</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">455</span> <span class="lpage" tagx="lpage" title="lpage">463</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2216/i0031-8884-28-4-455.1">10.2216/i0031-8884-28-4-455.1</a> </span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McFadden</span> <span class="given-names" tagx="given-names" title="given-names">GI</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gilson</span> <span class="given-names" tagx="given-names" title="given-names">PR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hill</span> <span class="given-names" tagx="given-names" title="given-names">DRA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>Goniomonas</i>: rRNA sequences indicate that that this phagotrophic flagellate is a close relative to the host component of cryptomonads </span> <span class="source" tagx="source" title="source">Eur J Phycol</span> <span class="year" tagx="year" title="year">1994</span> <span class="volume" tagx="volume" title="volume">29</span> <span class="fpage" tagx="fpage" title="fpage">29</span> <span class="lpage" tagx="lpage" title="lpage">32</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1080/09670269400650451">10.1080/09670269400650451</a> </span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoef-Emden</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marin</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Melkonian</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Nuclear and nucleomorph SSU rDNA phylogeny in the Cryptophyta and the evolution of cryptophyte diversity </span> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">55</span> <span class="fpage" tagx="fpage" title="fpage">161</span> <span class="lpage" tagx="lpage" title="lpage">179</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00239-002-2313-5">10.1007/s00239-002-2313-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12107593">12107593</a> </span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deane</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hill</span> <span class="given-names" tagx="given-names" title="given-names">DRA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brett</span> <span class="given-names" tagx="given-names" title="given-names">SJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McFadden</span> <span class="given-names" tagx="given-names" title="given-names">GI</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cryptomonad evolution: nuclear 18S rDNA phylogeny versus cell morphology and pigmentation</span> <span class="source" tagx="source" title="source">J Phycol</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">38</span> <span class="fpage" tagx="fpage" title="fpage">1236</span> <span class="lpage" tagx="lpage" title="lpage">1244</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1529-8817.2002.01250.x">10.1046/j.1529-8817.2002.01250.x</a> </span> </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">von der Heyden</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chao</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cavalier-Smith</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genetic diversity of goniomonads: an ancient divergence between marine and freshwater species </span> <span class="source" tagx="source" title="source">Eur J Phycol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">39</span> <span class="fpage" tagx="fpage" title="fpage">343</span> <span class="lpage" tagx="lpage" title="lpage">350</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1080/09670260400005567">10.1080/09670260400005567</a> </span> </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoef-Emden</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular phylogeny of phycocyanin-containing cryptophytes: evolution of biliproteins and geographical distribution </span> <span class="source" tagx="source" title="source">J Phycol</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">44</span> <span class="fpage" tagx="fpage" title="fpage">985</span> <span class="lpage" tagx="lpage" title="lpage">993</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1529-8817.2008.00530.x">10.1111/j.1529-8817.2008.00530.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27041617">27041617</a> </span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Donaher</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanifuji</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Onodera</span> <span class="given-names" tagx="given-names" title="given-names">NT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Malfatti</span> <span class="given-names" tagx="given-names" title="given-names">SA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chain</span> <span class="given-names" tagx="given-names" title="given-names">PS</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The complete plastid genome sequence of the secondarily nonphotosynthetic alga <i>Cryptomonas paramecium</i>: reduction, compaction, and accelerated evolutionary rate </span> <span class="source" tagx="source" title="source">Genome Biol Evol</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="fpage" tagx="fpage" title="fpage">439</span> <span class="lpage" tagx="lpage" title="lpage">448</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/gbe/evp047">10.1093/gbe/evp047</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20333213">20333213</a> </span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yabuki</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>Palpitomonas bilix</i> represents a basal cryptist lineage: insight into the character evolution in Cryptista </span> <span class="source" tagx="source" title="source">Sci Rep</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">4641</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep04641">10.1038/srep04641</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24717814">24717814</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cavalier-Smith</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chao</span> <span class="given-names" tagx="given-names" title="given-names">EE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lewis</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Multiple origins of Heliozoa from flagellate ancestors: new cryptist subphylum Corbihelia, superclass Corbistoma, and monophyly of Haptista, Cryptista, Hacrobia and Chromista </span> <span class="source" tagx="source" title="source">Mol Phylogenet Evol</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">93</span> <span class="fpage" tagx="fpage" title="fpage">331</span> <span class="lpage" tagx="lpage" title="lpage">362</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ympev.2015.07.004">10.1016/j.ympev.2015.07.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26234272">26234272</a> </span> </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burki</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaplan</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tikhonenkov</span> <span class="given-names" tagx="given-names" title="given-names">DV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zlatogursky</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Minh</span> <span class="given-names" tagx="given-names" title="given-names">BQ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Radaykina</span> <span class="given-names" tagx="given-names" title="given-names">LV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smirnov</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mylnikov</span> <span class="given-names" tagx="given-names" title="given-names">AP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keeling</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Untangling the early diversification of eukaryotes: a phylogenomic study of the evolutionary origins of Centrohelida, Haptophyta and Cryptista </span> <span class="source" tagx="source" title="source">Proc R Soc B</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">283</span> <span class="fpage" tagx="fpage" title="fpage">20152802</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1098/rspb.2015.2802">10.1098/rspb.2015.2802</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26817772">26817772</a> </span> </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Douglas</span> <span class="given-names" tagx="given-names" title="given-names">SE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Murphy</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Spencer</span> <span class="given-names" tagx="given-names" title="given-names">DF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gray</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cryptomonad algae are evolutionary chimaeras of two phylogenetically distinct unicellular eukaryotes </span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">1991</span> <span class="volume" tagx="volume" title="volume">350</span> <span class="fpage" tagx="fpage" title="fpage">148</span> <span class="lpage" tagx="lpage" title="lpage">151</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/350148a0">10.1038/350148a0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2005963">2005963</a> </span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McFadden</span> <span class="given-names" tagx="given-names" title="given-names">GI</span> </span> </span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Callow</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Second-hand chloroplasts: evolution of cryptomonad algae</span> <span class="source" tagx="source" title="source">Advances in botanical research</span> <span class="year" tagx="year" title="year">1993</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Academic Press Limited</span> <span class="fpage" tagx="fpage" title="fpage">189</span> <span class="lpage" tagx="lpage" title="lpage">230</span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gray</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lang</span> <span class="given-names" tagx="given-names" title="given-names">BF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burger</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mitochondria of protists</span> <span class="source" tagx="source" title="source">Ann Rev Genet</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">38</span> <span class="fpage" tagx="fpage" title="fpage">477</span> <span class="lpage" tagx="lpage" title="lpage">524</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev.genet.37.110801.142526">10.1146/annurev.genet.37.110801.142526</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15568984">15568984</a> </span> </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">DR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The past, present and future of mitochondrial genomics: have we sequenced enough mtDNAs?</span> <span class="source" tagx="source" title="source">Brief Funct Genomics</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">47</span> <span class="lpage" tagx="lpage" title="lpage">54</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26117139">26117139</a> </span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grover</span> <span class="given-names" tagx="given-names" title="given-names">CE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nie</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wendel</span> <span class="given-names" tagx="given-names" title="given-names">JF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hua</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Plant mitochondrial genome evolution and cytoplasmic male sterility</span> <span class="source" tagx="source" title="source">Crit Rev Plant Sci</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">36</span> <span class="fpage" tagx="fpage" title="fpage">55</span> <span class="lpage" tagx="lpage" title="lpage">69</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1080/07352689.2017.1327762">10.1080/07352689.2017.1327762</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burger</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gray</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Forget</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lang</span> <span class="given-names" tagx="given-names" title="given-names">BF</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Strikingly bacteria-like and gene-rich mitochondrial genomes throughout Jakobid protists</span> <span class="source" tagx="source" title="source">Genome Biol Evol</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">418</span> <span class="lpage" tagx="lpage" title="lpage">438</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/gbe/evt008">10.1093/gbe/evt008</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23335123">23335123</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burger</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jackson</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Waller</span> <span class="given-names" tagx="given-names" title="given-names">RF</span> </span> </span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bullerwell</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Unusual mitochondrial genomes and genes</span> <span class="source" tagx="source" title="source">Organelle genetics</span> <span class="year" tagx="year" title="year">2012</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span> <span class="fpage" tagx="fpage" title="fpage">41</span> <span class="lpage" tagx="lpage" title="lpage">77</span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hauth</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maier</span> <span class="given-names" tagx="given-names" title="given-names">UG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lang</span> <span class="given-names" tagx="given-names" title="given-names">BF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burger</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The <i>Rhodomonas salina</i> mitochondrial genome: bacteria-like operons, compact gene arrangement and complex repeat region </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">33</span> <span class="fpage" tagx="fpage" title="fpage">4433</span> <span class="lpage" tagx="lpage" title="lpage">4442</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gki757">10.1093/nar/gki757</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16085754">16085754</a> </span> </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lane</span> <span class="given-names" tagx="given-names" title="given-names">CE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Curtis</span> <span class="given-names" tagx="given-names" title="given-names">BA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Complete sequence and analysis of the mitochondrial genome of <i>Hemiselmis andersenii</i> CCMP644 (Cryptophyceae) </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">215</span> <span class="lpage" tagx="lpage" title="lpage">227</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-9-215">10.1186/1471-2164-9-215</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18474103">18474103</a> </span> </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nishimura</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanifuji</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kamikawa</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yabuki</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hashimoto</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Inagaki</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mitochondrial genome of <i>Palpitomonas bilix</i>: derived genome structure and ancestral system for cytochrome c maturation </span> <span class="source" tagx="source" title="source">Genome Biol Evol</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">3090</span> <span class="lpage" tagx="lpage" title="lpage">3098</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/gbe/evw217">10.1093/gbe/evw217</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">DR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arrigo</span> <span class="given-names" tagx="given-names" title="given-names">KR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alderkamp</span> <span class="given-names" tagx="given-names" title="given-names">AC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Allen</span> <span class="given-names" tagx="given-names" title="given-names">AE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Massive difference in synonymous substitution rates among mitochondrial, plastid, and nuclear genes of <i>Phaeocystis</i> algae </span> <span class="source" tagx="source" title="source">Mol Phylogenet Evol</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">71</span> <span class="fpage" tagx="fpage" title="fpage">30</span> <span class="lpage" tagx="lpage" title="lpage">40</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ympev.2013.10.018">10.1016/j.ympev.2013.10.018</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">JI</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moore</span> <span class="given-names" tagx="given-names" title="given-names">CE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Archibald</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhattacharya</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yi</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoon</span> <span class="given-names" tagx="given-names" title="given-names">HS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shin</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evolutionary dynamics of cryptophyte plastid genomes</span> <span class="source" tagx="source" title="source">Genome Biol Evol</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">1859</span> <span class="lpage" tagx="lpage" title="lpage">1872</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/gbe/evx123">10.1093/gbe/evx123</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28854597">28854597</a> </span> </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turmel</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lemieux</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burger</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lang</span> <span class="given-names" tagx="given-names" title="given-names">BF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Otis</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Plante</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gray</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The complete mitochondrial DNA sequences of <i>Nephroselmis olivacea</i> and <i>Pedinomonas minor</i>: two radically different evolutionary patterns within green algae </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">1717</span> <span class="lpage" tagx="lpage" title="lpage">1729</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10488238">10488238</a> </span> </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turmel</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Otis</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lemieux</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A deviant genetic code in the reduced mitochondrial genome of the picoplaktonic green alga <i>Pycnococcus provasolii</i> </span> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">70</span> <span class="fpage" tagx="fpage" title="fpage">203</span> <span class="lpage" tagx="lpage" title="lpage">214</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00239-010-9322-6">10.1007/s00239-010-9322-6</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20135105">20135105</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sánchez Puetra</span> <span class="given-names" tagx="given-names" title="given-names">MV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bachvaroff</span> <span class="given-names" tagx="given-names" title="given-names">TR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Delwiche</span> <span class="given-names" tagx="given-names" title="given-names">CF</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The complete mitochondrial genome sequence of the haptophyte <i>Emiliania huxleyi</i> and its relation to heterokonts </span> <span class="source" tagx="source" title="source">DNA Res</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">10</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/dnares/11.1.1">10.1093/dnares/11.1.1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15141941">15141941</a> </span> </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hovde</span> <span class="given-names" tagx="given-names" title="given-names">BT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Starkenburg</span> <span class="given-names" tagx="given-names" title="given-names">SR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hunsperger</span> <span class="given-names" tagx="given-names" title="given-names">HM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mercer</span> <span class="given-names" tagx="given-names" title="given-names">LD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deodato</span> <span class="given-names" tagx="given-names" title="given-names">CR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jha</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The mitochondrial and chloroplast genomes of the haptophyte <i>Chrysochromulina tobin</i> contain unique repeat structures and gene profiles </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">604</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-15-604">10.1186/1471-2164-15-604</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25034814">25034814</a> </span> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xue</span> <span class="given-names" tagx="given-names" title="given-names">JY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiu</span> <span class="given-names" tagx="given-names" title="given-names">YL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The mitochondrial genomes of the early land plants <i>Treubia lacunose</i> and <i>Anomodon rugelii</i>: dynamic and conservative evolution </span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">e25836</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0025836">10.1371/journal.pone.0025836</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21998706">21998706</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Farwagi</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fučíková</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McManus</span> <span class="given-names" tagx="given-names" title="given-names">HA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogenetic patterns of gene rearrangements in four mitochondrial genomes from the green algal family Hydrodictyaceae (Sphaeropleales, Chlorophyceae) </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">826</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12864-015-2056-5">10.1186/s12864-015-2056-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26486870">26486870</a> </span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Highly conserved mitochondrial genomes among multicellular red algae of the Florideophyceae</span> <span class="source" tagx="source" title="source">Genome Biol Evol</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">2394</span> <span class="lpage" tagx="lpage" title="lpage">2406</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/gbe/evv147">10.1093/gbe/evv147</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26245677">26245677</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ševčíková</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zbránková</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Klimeš</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strnad</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hroudová</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vlček</span> <span class="given-names" tagx="given-names" title="given-names">Č</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eliáš</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A comparative analysis of mitochondrial genomes in eustigmatophyte algae</span> <span class="source" tagx="source" title="source">Genome Biol Evol</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">705</span> <span class="lpage" tagx="lpage" title="lpage">722</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/gbe/evw027">10.1093/gbe/evw027</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26872774">26872774</a> </span> </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanifuji</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Archibald</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hashimoto</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Comparative genomics of mitochondria in chlorarachniophyte algae: endosymbiotic gene transfer and organellar genome dynamics </span> <span class="source" tagx="source" title="source">Sci Rep</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">21016</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep21016">10.1038/srep21016</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26888293">26888293</a> </span> </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paquin</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laforest</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lang</span> <span class="given-names" tagx="given-names" title="given-names">BF</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Double-hairpin elements in the mitochondrial DNA of Allomyces: evidence for mobility</span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">1760</span> <span class="lpage" tagx="lpage" title="lpage">1768</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/oxfordjournals.molbev.a026274">10.1093/oxfordjournals.molbev.a026274</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11070063">11070063</a> </span> </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bullerwell</span> <span class="given-names" tagx="given-names" title="given-names">CE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leigh</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Forget</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lang</span> <span class="given-names" tagx="given-names" title="given-names">BF</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A comparison of three fission yeast mitochondrial genomes</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="fpage" tagx="fpage" title="fpage">759</span> <span class="lpage" tagx="lpage" title="lpage">768</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkg134">10.1093/nar/gkg134</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12527786">12527786</a> </span> </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nedelcu</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">RW</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Short repetitive sequences in green algal mitochondrial genomes: potential roles in mitochondrial genome evolution </span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">690</span> <span class="lpage" tagx="lpage" title="lpage">701</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/oxfordjournals.molbev.a025972">10.1093/oxfordjournals.molbev.a025972</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9615450">9615450</a> </span> </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartoszewski</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Katzir</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harvey</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Organization of repetitive DNAs and the genomic regions carrying ribosomal RNA, <i>cob</i>, and <i>atp</i>9 genes in the cucurbit mitochondrial genomes </span> <span class="source" tagx="source" title="source">Theor Appl Genet</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">108</span> <span class="fpage" tagx="fpage" title="fpage">982</span> <span class="lpage" tagx="lpage" title="lpage">992</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00122-003-1516-5">10.1007/s00122-003-1516-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15067383">15067383</a> </span> </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beaudet</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mitochondrial genome rearrangements in <i>Glomus</i> species triggered by homologous recombination between distinct mtDNA haplotypes </span> <span class="source" tagx="source" title="source">Genome Biol Evol</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">1628</span> <span class="lpage" tagx="lpage" title="lpage">1643</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/gbe/evt120">10.1093/gbe/evt120</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23925788">23925788</a> </span> </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kamikawa</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shiratori</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishida</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miyashita</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roger</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Group II intron-mediated trans-splicing in the gene-rich mitochondrial genome of an enigmatic eukaryote, <i>Diphylleia rotans</i> </span> <span class="source" tagx="source" title="source">Genome Biol Evol</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">458</span> <span class="lpage" tagx="lpage" title="lpage">466</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/gbe/evw011">10.1093/gbe/evw011</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26833505">26833505</a> </span> </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bogorad</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vasil</span> <span class="given-names" tagx="given-names" title="given-names">IK</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Plastid chromosomes: structure and evolution</span> <span class="source" tagx="source" title="source">Cell culture and somatic genetics of plant</span> <span class="year" tagx="year" title="year">1991</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Diego</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Molecular Biology of Plastids, Academic Press</span> <span class="fpage" tagx="fpage" title="fpage">5</span> <span class="lpage" tagx="lpage" title="lpage">53</span> </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knox</span> <span class="given-names" tagx="given-names" title="given-names">EB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Downie</span> <span class="given-names" tagx="given-names" title="given-names">SR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Chloroplast genome rearrangements and the evolution of giant lobelias from herbaceous ancestors </span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">1993</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">414</span> <span class="lpage" tagx="lpage" title="lpage">430</span> </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kelchner</span> <span class="given-names" tagx="given-names" title="given-names">SA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wendel</span> <span class="given-names" tagx="given-names" title="given-names">JF</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Hairpins create minute inversions in non-coding regions of chloroplast DNA</span> <span class="source" tagx="source" title="source">Curr Genet</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">259</span> <span class="lpage" tagx="lpage" title="lpage">262</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s002940050130">10.1007/s002940050130</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8753656">8753656</a> </span> </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kelchner</span> <span class="given-names" tagx="given-names" title="given-names">SA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The evolution of non-coding chloroplast DNA and its application in plant systematics</span> <span class="source" tagx="source" title="source">Ann Mo Bot Gard</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">87</span> <span class="fpage" tagx="fpage" title="fpage">482</span> <span class="lpage" tagx="lpage" title="lpage">498</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2307/2666142">10.2307/2666142</a> </span> </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoot</span> <span class="given-names" tagx="given-names" title="given-names">SB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Structural rearrangements, including parallel inversions, within the chloroplast genome of Anemone and related genera </span> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="year" tagx="year" title="year">1994</span> <span class="volume" tagx="volume" title="volume">38</span> <span class="fpage" tagx="fpage" title="fpage">274</span> <span class="lpage" tagx="lpage" title="lpage">281</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/BF00176089">10.1007/BF00176089</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8006994">8006994</a> </span> </span> </li> <li tag="ref"> <a name="CR45" /> <span class="label" tagx="label" title="label">45.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turmel</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Otis</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lemieux</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The complete mitochondrial DNA sequence of <i>Mesostigma viride</i> identifies this green alga as the earliest green plant divergence and predicts a highly compact mitochondrial genome in the ancestor of all green plants </span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">24</span> <span class="lpage" tagx="lpage" title="lpage">38</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/oxfordjournals.molbev.a003979">10.1093/oxfordjournals.molbev.a003979</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11752187">11752187</a> </span> </span> </li> <li tag="ref"> <a name="CR46" /> <span class="label" tagx="label" title="label">46.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haberle</span> <span class="given-names" tagx="given-names" title="given-names">RC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fourcade</span> <span class="given-names" tagx="given-names" title="given-names">HM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boore</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Extensive rearrangements in the chloroplast genome of <i>Trachelium caeruleum</i> are associated with repeats and tRNA genes </span> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">66</span> <span class="fpage" tagx="fpage" title="fpage">350</span> <span class="lpage" tagx="lpage" title="lpage">361</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00239-008-9086-4">10.1007/s00239-008-9086-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18330485">18330485</a> </span> </span> </li> <li tag="ref"> <a name="CR47" /> <span class="label" tagx="label" title="label">47.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oudot-Le Secq</span> <span class="given-names" tagx="given-names" title="given-names">M-P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Green</span> <span class="given-names" tagx="given-names" title="given-names">BR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Complex repeat structures and novel features in the mitochondrial genomes of the diatoms <i>Phaeodactylum tricornutum</i> and <i>Thalassiosira pseudonana</i> </span> <span class="source" tagx="source" title="source">Gene</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">476</span> <span class="fpage" tagx="fpage" title="fpage">20</span> <span class="lpage" tagx="lpage" title="lpage">26</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.gene.2011.02.001">10.1016/j.gene.2011.02.001</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21320580">21320580</a> </span> </span> </li> <li tag="ref"> <a name="CR48" /> <span class="label" tagx="label" title="label">48.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nedelcu</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">RW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lemieux</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gray</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burger</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The complete mitochondrial DNA sequence of <i>Scenedesmus obliquus</i> reflects an intermediate stage in the evolution of the green algal mitochondrial genome </span> <span class="source" tagx="source" title="source">Genome Res</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">819</span> <span class="lpage" tagx="lpage" title="lpage">831</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.10.6.819">10.1101/gr.10.6.819</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10854413">10854413</a> </span> </span> </li> <li tag="ref"> <a name="CR49" /> <span class="label" tagx="label" title="label">49.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lunt</span> <span class="given-names" tagx="given-names" title="given-names">DH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Whipple</span> <span class="given-names" tagx="given-names" title="given-names">LE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hyman</span> <span class="given-names" tagx="given-names" title="given-names">BC</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mitochondrial DNA variable number tandem repeats (VNTRs): utility and problems in molecular ecology </span> <span class="source" tagx="source" title="source">Mol Ecol</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">1441</span> <span class="lpage" tagx="lpage" title="lpage">1455</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1365-294x.1998.00495.x">10.1046/j.1365-294x.1998.00495.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9819900">9819900</a> </span> </span> </li> <li tag="ref"> <a name="CR50" /> <span class="label" tagx="label" title="label">50.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Casane</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dennebouy</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">de Rochambeau</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mounolou</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Monnerot</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Nonneutral evolution of tandem repeats in the mitochondrial DNA control region of lagomorphs </span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">779</span> <span class="lpage" tagx="lpage" title="lpage">789</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/oxfordjournals.molbev.a025818">10.1093/oxfordjournals.molbev.a025818</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9254915">9254915</a> </span> </span> </li> <li tag="ref"> <a name="CR51" /> <span class="label" tagx="label" title="label">51.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elorza</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Nuclear SDH2-1 and SDH2-2 genes, encoding the iron-sulfur subunit of mitochondrial complex II in <i>Arabidopsis</i>, have distinct cell-specific expression patterns and promoter activities </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">136</span> <span class="fpage" tagx="fpage" title="fpage">4072</span> <span class="lpage" tagx="lpage" title="lpage">4087</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.104.049528">10.1104/pp.104.049528</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15563621">15563621</a> </span> </span> </li> <li tag="ref"> <a name="CR52" /> <span class="label" tagx="label" title="label">52.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bayley</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Devilee</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taschner</span> <span class="given-names" tagx="given-names" title="given-names">PE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The SDH mutation database: an online resource for succinate dehydrogenase sequence variants involved in pheochromocytoma, paraganglioma and mitochondrial complex II deficiency </span> <span class="source" tagx="source" title="source">BMC Med Genet</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">39</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2350-6-39">10.1186/1471-2350-6-39</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16288654">16288654</a> </span> </span> </li> <li tag="ref"> <a name="CR53" /> <span class="label" tagx="label" title="label">53.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salomaki</span> <span class="given-names" tagx="given-names" title="given-names">ED</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lane</span> <span class="given-names" tagx="given-names" title="given-names">CE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Red algal mitochondrial genomes are more complete than previously reported</span> <span class="source" tagx="source" title="source">Genome Biol Evol</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">48</span> <span class="lpage" tagx="lpage" title="lpage">63</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28175279">28175279</a> </span> </span> </li> <li tag="ref"> <a name="CR54" /> <span class="label" tagx="label" title="label">54.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lang</span> <span class="given-names" tagx="given-names" title="given-names">BF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burger</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Okelly</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cedergren</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Golding</span> <span class="given-names" tagx="given-names" title="given-names">GB</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">An ancestral mitochondrial DNA resembling a eubacterial genome in miniature</span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">387</span> <span class="fpage" tagx="fpage" title="fpage">493</span> <span class="lpage" tagx="lpage" title="lpage">497</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/387493a0">10.1038/387493a0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9168110">9168110</a> </span> </span> </li> <li tag="ref"> <a name="CR55" /> <span class="label" tagx="label" title="label">55.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burger</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gray</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lang</span> <span class="given-names" tagx="given-names" title="given-names">BF</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mitochondrial genomes: anything goes</span> <span class="source" tagx="source" title="source">Trends Genet</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">709</span> <span class="lpage" tagx="lpage" title="lpage">716</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tig.2003.10.012">10.1016/j.tig.2003.10.012</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14642752">14642752</a> </span> </span> </li> <li tag="ref"> <a name="CR56" /> <span class="label" tagx="label" title="label">56.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zimmerly</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hausner</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogenetic relationships among group II intron ORFs</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">29</span> <span class="fpage" tagx="fpage" title="fpage">1238</span> <span class="lpage" tagx="lpage" title="lpage">1250</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/29.5.1238">10.1093/nar/29.5.1238</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11222775">11222775</a> </span> </span> </li> <li tag="ref"> <a name="CR57" /> <span class="label" tagx="label" title="label">57.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oda</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamato</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohta</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakamura</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Gene organization deduced from the complete sequence of liverwort <i>Marchantia polymorpha</i> mitochondrial DNA. A primitive form of plant mitochondrial genome </span> <span class="source" tagx="source" title="source">J Mol Biol</span> <span class="year" tagx="year" title="year">1992</span> <span class="volume" tagx="volume" title="volume">223</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">7</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0022-2836(92)90708-R">10.1016/0022-2836(92)90708-R</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1731062">1731062</a> </span> </span> </li> <li tag="ref"> <a name="CR58" /> <span class="label" tagx="label" title="label">58.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berks</span> <span class="given-names" tagx="given-names" title="given-names">BC</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The twin-arginine protein translocation pathway</span> <span class="source" tagx="source" title="source">Annu Rev Biochem</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">84</span> <span class="fpage" tagx="fpage" title="fpage">843</span> <span class="lpage" tagx="lpage" title="lpage">864</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev-biochem-060614-034251">10.1146/annurev-biochem-060614-034251</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25494301">25494301</a> </span> </span> </li> <li tag="ref"> <a name="CR59" /> <span class="label" tagx="label" title="label">59.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bogsch</span> <span class="given-names" tagx="given-names" title="given-names">EG</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">An essential component of a novel bacterial protein export system with homologues in plastids and mitochondria </span> <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">273</span> <span class="fpage" tagx="fpage" title="fpage">18003</span> <span class="lpage" tagx="lpage" title="lpage">18006</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.273.29.18003">10.1074/jbc.273.29.18003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9660752">9660752</a> </span> </span> </li> <li tag="ref"> <a name="CR60" /> <span class="label" tagx="label" title="label">60.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacob</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seif</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paquet</span> <span class="given-names" tagx="given-names" title="given-names">P-O</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lang</span> <span class="given-names" tagx="given-names" title="given-names">BF</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Loss of the mRNA-like region in mitochondrial tmRNAs of jakobids</span> <span class="source" tagx="source" title="source">RNA</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">605</span> <span class="lpage" tagx="lpage" title="lpage">614</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1261/rna.5227904">10.1261/rna.5227904</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15037770">15037770</a> </span> </span> </li> <li tag="ref"> <a name="CR61" /> <span class="label" tagx="label" title="label">61.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burger</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nedelcu</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> </span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bock</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knoop</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mitochondrial genomes of algae</span> <span class="source" tagx="source" title="source">Genomics of chloroplasts and mitochondria (advances in photosynthesis and respiration)</span> <span class="year" tagx="year" title="year">2012</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Dordrecht</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span> <span class="fpage" tagx="fpage" title="fpage">127</span> <span class="lpage" tagx="lpage" title="lpage">157</span> </span> </li> <li tag="ref"> <a name="CR62" /> <span class="label" tagx="label" title="label">62.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cavalier-Smith</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Principles of protein and lipid targeting in secondary symbiogenesis: euglenoid, dinoflagellate, and sporozoan plastid origins and the eukaryotic family tree </span> <span class="source" tagx="source" title="source">J Eukaryot Microbiol</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">46</span> <span class="fpage" tagx="fpage" title="fpage">347</span> <span class="lpage" tagx="lpage" title="lpage">366</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1550-7408.1999.tb04614.x">10.1111/j.1550-7408.1999.tb04614.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18092388">18092388</a> </span> </span> </li> <li tag="ref"> <a name="CR63" /> <span class="label" tagx="label" title="label">63.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoon</span> <span class="given-names" tagx="given-names" title="given-names">HS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hackett</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhattacharya</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A single origin of the peridinin—and fucoxanthin-containing plastids in dinoflagellates through tertiary endosymbiosis </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">99</span> <span class="fpage" tagx="fpage" title="fpage">11724</span> <span class="lpage" tagx="lpage" title="lpage">11729</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.172234799">10.1073/pnas.172234799</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12172008">12172008</a> </span> </span> </li> <li tag="ref"> <a name="CR64" /> <span class="label" tagx="label" title="label">64.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoon</span> <span class="given-names" tagx="given-names" title="given-names">HS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hackett</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pinto</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhattacharya</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The single, ancient origin of chromist plastids</span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">99</span> <span class="fpage" tagx="fpage" title="fpage">15507</span> <span class="lpage" tagx="lpage" title="lpage">15512</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.242379899">10.1073/pnas.242379899</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12438651">12438651</a> </span> </span> </li> <li tag="ref"> <a name="CR65" /> <span class="label" tagx="label" title="label">65.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keeling</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Diversity and evolutionary history of plastids and their hosts</span> <span class="source" tagx="source" title="source">Am J Bot</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">91</span> <span class="fpage" tagx="fpage" title="fpage">1481</span> <span class="lpage" tagx="lpage" title="lpage">1493</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3732/ajb.91.10.1481">10.3732/ajb.91.10.1481</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21652304">21652304</a> </span> </span> </li> <li tag="ref"> <a name="CR66" /> <span class="label" tagx="label" title="label">66.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Archibald</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keeling</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span> </span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saap</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="source" tagx="source" title="source">On the origin and evolution of plastids. Microbial phylogeny and evolution</span> <span class="year" tagx="year" title="year">2005</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span> <span class="fpage" tagx="fpage" title="fpage">238</span> <span class="lpage" tagx="lpage" title="lpage">260</span> </span> </li> <li tag="ref"> <a name="CR67" /> <span class="label" tagx="label" title="label">67.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhattacharya</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoon</span> <span class="given-names" tagx="given-names" title="given-names">HS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hackett</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Photosynthetic eukaryotes unite: endosymbiosis connects the dots</span> <span class="source" tagx="source" title="source">BioEssays</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">26</span> <span class="fpage" tagx="fpage" title="fpage">50</span> <span class="lpage" tagx="lpage" title="lpage">60</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/bies.10376">10.1002/bies.10376</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14696040">14696040</a> </span> </span> </li> <li tag="ref"> <a name="CR68" /> <span class="label" tagx="label" title="label">68.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhattacharya</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Archibald</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">APM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reyes-Prieto</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">How do endosymbionts become organelles? Understanding early events in plastid evolution</span> <span class="source" tagx="source" title="source">BioEssays</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">29</span> <span class="fpage" tagx="fpage" title="fpage">1239</span> <span class="lpage" tagx="lpage" title="lpage">1246</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/bies.20671">10.1002/bies.20671</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18027391">18027391</a> </span> </span> </li> <li tag="ref"> <a name="CR69" /> <span class="label" tagx="label" title="label">69.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Archibald</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The puzzle of plastid evolution</span> <span class="source" tagx="source" title="source">Curr Biol</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">R81</span> <span class="lpage" tagx="lpage" title="lpage">R88</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cub.2008.11.067">10.1016/j.cub.2008.11.067</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19174147">19174147</a> </span> </span> </li> <li tag="ref"> <a name="CR70" /> <span class="label" tagx="label" title="label">70.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">JI</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoon</span> <span class="given-names" tagx="given-names" title="given-names">HS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yi</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">HS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yih</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shin</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The plastid genome of the cryptomonad <i>Teleaulax amphioxeia</i> </span> <span class="source" tagx="source" title="source">PLoS One</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">e0129284</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0129284">10.1371/journal.pone.0129284</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26047475">26047475</a> </span> </span> </li> <li tag="ref"> <a name="CR71" /> <span class="label" tagx="label" title="label">71.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baurain</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brinkmann</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Petersen</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodríguez-Ezpeleta</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stechmann</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogenomic evidence for separate acquisition of plastids in cryptophytes, haptophytes, and stramenopiles </span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="fpage" tagx="fpage" title="fpage">1698</span> <span class="lpage" tagx="lpage" title="lpage">1709</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msq059">10.1093/molbev/msq059</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20194427">20194427</a> </span> </span> </li> <li tag="ref"> <a name="CR72" /> <span class="label" tagx="label" title="label">72.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burki</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Okamoto</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pombert</span> <span class="given-names" tagx="given-names" title="given-names">J-F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keeling</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The evolutionary history of haptophytes and cryptophytes: phylogenomic evidence for separate origins </span> <span class="source" tagx="source" title="source">Proc Biol Sci</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">279</span> <span class="fpage" tagx="fpage" title="fpage">2246</span> <span class="lpage" tagx="lpage" title="lpage">2254</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1098/rspb.2011.2301">10.1098/rspb.2011.2301</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22298847">22298847</a> </span> </span> </li> <li tag="ref"> <a name="CR73" /> <span class="label" tagx="label" title="label">73.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stiller</span> <span class="given-names" tagx="given-names" title="given-names">JW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schreiber</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yue</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ding</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The evolution of photosynthesis in chromist algae through serial endosymbioses</span> <span class="source" tagx="source" title="source">Nat Commun</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">5764</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncomms6764">10.1038/ncomms6764</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25493338">25493338</a> </span> </span> </li> <li tag="ref"> <a name="CR74" /> <span class="label" tagx="label" title="label">74.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Archibald</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genomic perspectives on the birth and spread of plastids</span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">112</span> <span class="fpage" tagx="fpage" title="fpage">10147</span> <span class="lpage" tagx="lpage" title="lpage">10153</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1421374112">10.1073/pnas.1421374112</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25902528">25902528</a> </span> </span> </li> <li tag="ref"> <a name="CR75" /> <span class="label" tagx="label" title="label">75.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Watanabe</span> <span class="given-names" tagx="given-names" title="given-names">MM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hiroki</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="source" tagx="source" title="source">NIES-collection list of strains</span> <span class="year" tagx="year" title="year">1997</span> <span class="edition" tagx="edition" title="edition">5</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Tsukuba</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">National Institute for Environmental Studies</span> <span class="fpage" tagx="fpage" title="fpage">127</span> </span> </li> <li tag="ref"> <a name="CR76" /> <span class="label" tagx="label" title="label">76.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jung</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">JI</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jeong</span> <span class="given-names" tagx="given-names" title="given-names">Y-S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yi</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A robust method for finding the automated best matched genes based on grouping similar fragments of large-scale references for genome assembly </span> <span class="source" tagx="source" title="source">Symmetry</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">192</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3390/sym9090192">10.3390/sym9090192</a> </span> </span> </li> <li tag="ref"> <a name="CR77" /> <span class="label" tagx="label" title="label">77.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Darling</span> <span class="given-names" tagx="given-names" title="given-names">AC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mau</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blattner</span> <span class="given-names" tagx="given-names" title="given-names">FR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perna</span> <span class="given-names" tagx="given-names" title="given-names">NT</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mauve: multiple alignment of conserved genomic sequence with rearrangements</span> <span class="source" tagx="source" title="source">Genome Res</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">1394</span> <span class="lpage" tagx="lpage" title="lpage">1403</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.2289704">10.1101/gr.2289704</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15231754">15231754</a> </span> </span> </li> <li tag="ref"> <a name="CR78" /> <span class="label" tagx="label" title="label">78.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">SW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Overbeek</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Woese</span> <span class="given-names" tagx="given-names" title="given-names">CR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gilbert</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gillevet</span> <span class="given-names" tagx="given-names" title="given-names">PM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The genetic data environment: an expandable GUI for multiple sequence analysis</span> <span class="source" tagx="source" title="source">Comput Appl Biosci</span> <span class="year" tagx="year" title="year">1994</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">671</span> <span class="lpage" tagx="lpage" title="lpage">675</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7704666">7704666</a> </span> </span> </li> <li tag="ref"> <a name="CR79" /> <span class="label" tagx="label" title="label">79.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stamatakis</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">1312</span> <span class="lpage" tagx="lpage" title="lpage">1313</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btu033">10.1093/bioinformatics/btu033</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24451623">24451623</a> </span> </span> </li> <li tag="ref"> <a name="CR80" /> <span class="label" tagx="label" title="label">80.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Le</span> <span class="given-names" tagx="given-names" title="given-names">SQ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gascuel</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">An improved general amino acid replacement matrix</span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">1307</span> <span class="lpage" tagx="lpage" title="lpage">1320</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msn067">10.1093/molbev/msn067</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18367465">18367465</a> </span> </span> </li> <li tag="ref"> <a name="CR81" /> <span class="label" tagx="label" title="label">81.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Darriba</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taboada</span> <span class="given-names" tagx="given-names" title="given-names">GL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doallo</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Posada</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">ProtTest 3: fast selection of best-fit models of protein evolution</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="fpage" tagx="fpage" title="fpage">1164</span> <span class="lpage" tagx="lpage" title="lpage">1165</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btr088">10.1093/bioinformatics/btr088</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21335321">21335321</a> </span> </span> </li> </ul> </div> </div>  </body></html>