<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Biol Direct</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Biology Direct</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1745-6150</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/2881121">2881121</a> </span> <span class="publisher-id" title="publisher-id">1745-6150-5-34</span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20459727">20459727</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/1745-6150-5-34">10.1186/1745-6150-5-34</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Discovery notes</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Lack of conservation of bacterial type promoters in plastids of Streptophyta</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lyubetsky</span> <span class="given-names" tagx="given-names" title="given-names">Vassily A</span> </span> <a href="#I1">1</a> <span class="email" tagx="email" title="email">lyubetsk@iitp.ru</span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rubanov</span> <span class="given-names" tagx="given-names" title="given-names">Lev I</span> </span> <a href="#I1">1</a> <span class="email" tagx="email" title="email">rubanov@iitp.ru</span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seliverstov</span> <span class="given-names" tagx="given-names" title="given-names">Alexandr V</span> </span> <a href="#I1">1</a> <span class="email" tagx="email" title="email">slvstv@iitp.ru</span> </span> </div> <span class="citation_author_institution" id="I1">[1], </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2010</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2010-5-5</span> </span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">34</span> <span class="lpage" tagx="lpage" title="lpage">34</span> <span class="history" title="history"> <span class="received" title="received">received: 2010-4-20</span> <span class="accepted" title="accepted">accepted: 2010-5-10</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Lyubetsky et al; licensee BioMed Central Ltd.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2010</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License ( <a href="http://creativecommons.org/licenses/by/2.0">http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. </span> </span> </div> <a class="self-uri" href="http://www.biology-direct.com/content/5/1/34" title="self-uri">http://www.biology-direct.com/content/5/1/34</a> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="" title="sec"> <div class="title" tagx="title" title="title" /> <p>We demonstrate the scarcity of conserved bacterial-type promoters in plastids of Streptophyta and report widely conserved promoters only for genes <i>psaA, psbA, psbB, psbE, rbcL</i>. Among the reasonable explanations are: evolutionary changes of sigma subunit paralogs and phage-type RNA polymerases possibly entailing the loss of corresponding nuclear genes, <i>de novo </i>emergence of the promoters, their loss together with plastome genes; functional substitution of the promoter boxes by transcription activation factor binding sites. </p> </div> <div class="reviewers" title="sec"> <div class="title" tagx="title" title="title">
Reviewers</div> <p>This article was reviewed by Dr. Arcady Mushegian, and by Dr. Alexander Bolshoy and Dr. Yuri Wolf (both nominated by Dr. Purificación López-García). </p> </div> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Genes evolve at different rates. Various hypotheses try to explain, or at least to correlate, the evolutionary rate (sequence conservation) and the functional properties of the protein-coding gene. As far as we know, there is no published evidence on searching for the plastid promoters at the genome scale. This problem should probably be addressed separately for nuclear, plastome and mitochondrial genomes, different taxonomic lineages and different RNA polymerase types. In particular, multisubunit RNA polymerase (PEP), which has the core enzyme encoded in plastome and the sigma subunit in nucleome, binds bacterial type promoters (PEP-promoters); and monosubunit RNA polymerases (NEP), which is nucleome-encoded, binds NEP-promoters. Here we report a study of PEP-promoters of plastome genes in representatives of the green line (Viridiplantae, including Chlorophyta and Streptophyta; Euglenozoa, Rhizaria, in particular Cercozoa; Glaucocystophyceae) and the red line (Rhodophyta, stramenopiles, including Bacillariophyta, Pelagophyceae, Raphidophyceae, Xanthophyceae; Cryptophyta, Haptophyceae, Apicomplexa). Add. file 1 describes the complete list of studied species with plastids, organized according to the NCBI Taxonomy. Plastid genes are believed to be evolutionarily conserved across large taxonomic lineages [[ <a href="#B1">1</a>], section 9.7c], although the authors are unaware of systematic studies on their promoters conservation. Instead, there is ample published research on the promoter comparisons within small lineages, largely the studies of the promoters and their transcription factors in gamma- and alpha-proteobacteria [ <a href="#B2">2</a>]. Further, some pairs of closely related species have been shown to possess largely diverged promoters [ <a href="#B3">3</a>, <a href="#B4">4</a>]. We have reported an evolutionary labile promoter for the <i>ndhF </i>gene in a narrow lineage of dicotyledonous angiosperm plants and described four different promoter types, which are likely to have replaced each other during evolution [ <a href="#B5">5</a>]. </p> <p>In this study we aimed at searching for widely conserved PEP-promoters in plastomes of the above mentioned taxa. By "widely conserved" we mean the cases when the regions upstream of orthologous genes across the high-level taxonomic divisions can be aligned. The promoters confined to only vascular plants or the red line lineages are not examined here (e.g., the NEP-promoter of gene <i>clpP </i>in vascular plants). In our analyses using the fixed consensus as a query produced massive under-predictions, or, alternatively, massive over-predictions, which suggests that querying without taking into account the alignment of 5'-leader regions is obviously misleading. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <p>The regions of up to 1000 bp length upstream from all protein-coding genes (90 genes per species at average) in plastomes of species listed in Additional file <a href="#S1">1</a> were extracted from GenBank, and multiple alignments of the regions were constructed. Searches of promoters were conducted using two original algorithms: the <i>first </i>to pre-select leader regions with candidate PEP-promoters (several candidates were found per region), and the <i>second </i>to build a multiple alignment keeping one of the candidate promoters in each of the regions. The alignment was constructed to reveal the two bacterial type boxes and cover the taxonomic diversity of the above mentioned lineages as wide as possible. In a <i>positive </i>prediction, the alignment of the boxes, linker and some flanking regions was required to have a good quality (see below). Otherwise, a <i>negative </i>prediction is produced and a PEP-promoter is not detected with our method. Evidently "positive prediction" means the prediction of a PEP-promoter and "negative prediction" means the lack of positive prediction. Notably, the positive predictions contained experimentally proved PEP-promoters and often their TG-extensions, which indicates that these are not false positives. Also, in all negative predictions the alignment had a considerably lower quality compared to the minimal quality among all positive predictions. All predicted PEP-promoters were located within approximately 40 bp-long highly conserved regions flanked by less conserved 3'-areas and highly variable 5'-areas. </p> <p>The idea of the <i>first </i>algorithm. Given is a set of <i>n </i>leader regions. The goal is to find a subset of the set with one potential promoter in each region such that their total pair-wise similarity is maximal comparing to any other collection of potential promoters in that subset; the subset size is simultaneously maximized. In order to increase search speed, randomly selected regions are set as "linked" and the promoter similarity is estimated only within the linked pairs of regions. It formally means that we consider a graph with <i>n </i>vertices, each assigned a leader region, but only linked regions are connected by an edge in the graph. As a result, the complexity of comparing all pairs of candidate promoters to determine their total similarity is reduced in our algorithm by means of considering a large number of randomly defined sets of edges, i.e. randomly constructed graphs with <i>n </i>vertices assigned the same regions but connected by different edges. By doing so, the computing time becomes square to number <i>n </i>of the regions and cubic to their average length. The algorithm is designed for effective parallelization to enable mass processing of large amounts of long regions in feasible time. The enhanced performance of the parallel implementation allows to compute a solution closer to the maximum quality of the alignment. The algorithm is highly scalable and provides for the approximately linear growth of performance with the number of available processors up to 2000. </p> <p>The idea of the <i>second </i>algorithm. Along a fixed phylogenetic species tree, the algorithm aligns leader regions with respect to one of the candidate promoters selected by the first algorithm, from the promoter start up to the start codon. It uses a common observation that promoters, as well as transcribed regions, can be well aligned, in contrast to the region upstream of the promoter. The algorithm takes a non-binary (which is often the case) species tree and during the run reduces it to a binary tree in a variety (or even all) possible ways. Each leaf of the tree bears an orthologous gene leader region from the corresponding species. The alignment is constructed as follows. First, each leaf is assigned a nucleotide frequency distribution at each position of the sequence: the distribution contains a unity for the observed nucleotide type and three zeros for the unobserved. A zero distribution contains four zeros. Then, at each inner node, <i>two </i>distribution sequences at its descendant nodes are aligned by any applicable algorithm, with an award for matching two distributions not pre-defined, but calculated anew at each position <i>j </i>taking into account the length of each descendant branch. The award is estimated as a scalar square of the difference between two nonzero distributions weighted for different nucleotide types. The penalty for inserting a gap symbol (i.e., for the alignment of zero and nonzero distributions) is a decreasing function of the number of contiguous gaps: the longer the gap region, the lower the penalty. Two zero distributions are forbidden to align. At each position of the alignment, the distribution in the ancestral sequence is a half-sum of the two distributions in the descendants. When the root distribution sequence is constructed, the algorithm projects the gaps along the tree to its leaves onto the extant sequences, thus obtaining the final multiple alignment. The complexity is linear to the number of leaves. Different binary tree resolutions are compared on the basis of the corresponding <i>alignment quality</i>, which is estimated as follows: <span class="inline-formula" title="inline-formula"> <span class="inline-graphic" title="inline-graphic" /> </span>, where <i>N</i> <sub> <i>a </i> </sub>is the number of totally conserved (containing the same character) single columns, <i>N</i> <sub> <i>s </i> </sub>- the number of totally conserved regions (two or more contiguous totally conserved columns, <i>l</i> <sub> <i>i </i> </sub>is the number of columns), <i>N</i> <sub> <i>b </i> </sub>- the number of "nearly" conserved columns (with one non-matching character); <i>b, c </i>and <i>s </i>are parameters. Computing an alignment of 16 sequences with the length of 120-223 bases requires less than one second on a 3 GHz Pentium-4 PC. The automatically computed alignments were manually checked and minor corrections were introduced if so required. Both algorithms are implemented as 32-bit command line utilities written in ANSI C, which can be compiled with many popular compilers and run under Windows or Linux. The algorithms and their detailed descriptions are available from [ <a href="#B6">6</a>, <a href="#B7">7</a>]. </p> <p>Testing of the algorithms and their comparison with "common" local alignment algorithms (see the introduction and the list of references in [ <a href="#B8">8</a>]) are described in [ <a href="#B9">9</a>- <a href="#B11">11</a>]. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>Table <a href="#T1">1</a> contains the species from add. file 1 predicted to possess at least one <i>widely </i>conserved promoter in the plastome. Predictions are identical for their close relatives with a corresponding orthologous gene (not shown). Within flowering plants the promoter sequences are similar and well aligned, therefore we illustrate results on <i>Arabidopsis thaliana </i>and <i>Spinacia oleracea </i>only. The five positive predictions are described below. Our analyses suggest that <i>widely </i>conserved promoters are absent elsewhere in streptophyte plastomes. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <p>Estimated coordinates of the transcription initiation sites of the predicted PEP-promoters</p>  <table> <thead> <tr> <th>Species</th> <th> <i>psaA</i> </th> <th> <i>psbA</i> </th> <th> <i>psbB</i> </th> <th> <i>psbE</i> </th> <th> <i>rbcL</i> </th> </tr> </thead> <tbody> <tr> <td> <i>Arabidopsis thaliana</i> </td> <td>Ex -188</td> <td>Ex -77</td> <td>-170</td> <td>-125</td> <td>-177</td> </tr> <tr> <td> <i>Spinacia oleracea</i> </td> <td>Ex -179</td> <td>Ex -82</td> <td>-175</td> <td>-150</td> <td>-176</td> </tr> <tr> <td> <i>Cycas taitungensis</i> </td> <td>Ex -156</td> <td>-60</td> <td>Ex -170</td> <td>-141</td> <td>-156</td> </tr> <tr> <td> <i>Cryptomeria japonica</i> </td> <td>Ex -142</td> <td>-58</td> <td>Ex -142</td> <td>-137</td> <td>-161</td> </tr> <tr> <td> <i>Pinus koraiensis</i> </td> <td>Ex -158</td> <td>-52</td> <td>-193</td> <td>-148</td> <td>-136</td> </tr> <tr> <td> <i>Pinus thunbergii</i> </td> <td>Ex -158</td> <td>-52</td> <td>-180</td> <td>-145</td> <td>-127</td> </tr> <tr> <td> <i>Welwitschia mirabilis</i> </td> <td>Ex -156</td> <td>Ex -51</td> <td>-271</td> <td>-31</td> <td>-136</td> </tr> <tr> <td> <i>Adiantum capillus-veneris</i> </td> <td>Ex -163</td> <td>Ex -55</td> <td>Ex -291</td> <td>-191</td> <td>-157</td> </tr> <tr> <td> <i>Angiopteris evecta</i> </td> <td>Ex -152</td> <td>Ex -69</td> <td>Ex -181</td> <td>-142</td> <td>-148</td> </tr> <tr> <td> <i>Psilotum nudum</i> </td> <td>Ex -147</td> <td>Ex -53</td> <td>Ex -178</td> <td>-127</td> <td>-140</td> </tr> <tr> <td> <i>Huperzia lucidula</i> </td> <td>Ex -153</td> <td>Ex -55</td> <td>Ex -187</td> <td>-134</td> <td>-150</td> </tr> <tr> <td> <i>Anthoceros formosae</i> </td> <td>Ex -155</td> <td>=</td> <td>=</td> <td>-143</td> <td>-160</td> </tr> <tr> <td> <i>Aneura mirabilis</i> </td> <td>Pseudo</td> <td>Ex -54</td> <td>Pseudo</td> <td>Pseudo</td> <td>-148</td> </tr> <tr> <td> <i>Marchantia polymorpha</i> </td> <td>Ex -149</td> <td>Ex -53</td> <td>=</td> <td>-132</td> <td>-124</td> </tr> <tr> <td> <i>Physcomitrella patens</i> </td> <td>Ex -161</td> <td>Ex -53</td> <td>=</td> <td>-145</td> <td>-143</td> </tr> <tr> <td> <i>Chara vulgaris</i> </td> <td>Ex -199</td> <td>-121</td> <td>Ex -179</td> <td>=</td> <td>-154</td> </tr> <tr> <td> <i>Chaetosphaeridium globosum</i> </td> <td>Ex -154</td> <td>Ex -57</td> <td>Ex -161</td> <td>-119</td> <td>-102</td> </tr> <tr> <td> <i>Staurastrum punctulatum</i> </td> <td>Ex -235</td> <td>Ex -59</td> <td>-190</td> <td>-154</td> <td>-219</td> </tr> <tr> <td> <i>Zygnema circumcarinatum</i> </td> <td>Ex -157</td> <td>Ex -58</td> <td>-159</td> <td>-122</td> <td>-168</td> </tr> <tr> <td> <i>Chlorokybus atmophyticus</i> </td> <td>=</td> <td>=</td> <td>-266</td> <td>=</td> <td>=</td> </tr> <tr> <td> <i>Mesostigma viride</i> </td> <td>=</td> <td>Ex -53</td> <td>-89</td> <td>=</td> <td>=</td> </tr> <tr> <td> <i>Bigelowiella natans</i> </td> <td>=</td> <td>-136</td> <td>=</td> <td>=</td> <td>=</td> </tr> <tr> <td> <i>Cyanophora paradoxa</i> </td> <td>=</td> <td>-61</td> <td>=</td> <td>=</td> <td>=</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Coordinates are relative to the start codon. The "Ex" means the presence of the 5'-extension TG of the "-10" box, "Pseudo" marks a negative prediction for the pseudogene, "=" - a negative prediction for the functioning gene. </p> </div> </div> <p> <b>Gene <i>psbA </i> </b>(protein D1 of the photosystem II active center) in plastomes. Promoters of this chloroplast gene were experimentally studied in selected species, including <i>Arabidopsis</i>, mustard, and spinach [ <a href="#B3">3</a>, <a href="#B12">12</a>, <a href="#B13">13</a>], for which our predictions are in good agreement with the experiment. The algorithm predicted candidate conserved promoters upstream of this gene in most Streptophyta, primary and secondary endosymbionts, <i>Bigelowiella natans </i>from the Chlorarachniophyceae, and <i>Cyanophora paradoxa </i>from the Glaucocystophyceae (ref. to Fig. <a href="#F1">1</a>, <i>psbA</i>). The gene alignments are given in Fig. <a href="#F1">1</a>, per-site nucleotide frequency distributions are given in Fig. <a href="#F2">2</a> (constructed with the Weblogo program [ <a href="#B14">14</a>]). We suggest that this ancient promoter with the consensus TTGACA-15-TGTwATAmT is ancestral for at least all Streptophyta. The linker between the boxes is usually 18 bases long, but is 17 bases in <i>Cycas taitungensis</i>, <i>Adiantum capillus-veneris</i>, <i>Staurastrum punctulatum</i>, <i>Mesostigma viride </i>and <i>B. natans</i>. Many predictions possess the 5'-extension (TG or TGTG) of the "-10" box, which enhances the promoter efficiency. In the gymnosperm <i>C. taitungensis</i>, the predicted "-35" box essentially differs from the alignment consensus and the bacterial-like promoter. The <i>psbA </i>promoter was not found in the hornworts <i>Anthoceros formosae</i>, although in other bryophytes it is highly conserved. In the early emerging alga <i>Chlorokybus atmophyticus </i>only the "-35" box was identified, while the complete promoter was found in <i>M. viride</i>. Two dodder species ( <i>Cuscuta gronovii</i>, <i>C. obtusiflora</i>) with a largely reduced plastome also lack the <i>psbA </i>promoter, which, however is found in their close relatives ( <i>C. exaltata</i>, <i>C. reflexa</i>) and most angiosperm plants. The lack of promoters correlates with the reduction of genomes: <i>Cuscuta gronovii </i>and <i>C. obtusiflora </i>do not photosynthesize and lack most of the photosynthetic genes. Although the <i>psbA </i>gene retains an open reading frame, it lacks the PEP-promoter and is probably poorly expressed compared to photosynthetic species. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p> <b>Predicted promoters upstream of genes <i>psbA</i>, <i>psbB</i>, <i>psbE</i>, <i>rbcL</i>, <i>psaA</i> </b>. In the cells of first column only first occurrences of each taxon name are given. In yellow are the promoter boxes and the 5'-extension of the "-10" box. Numbers are the distance to the start codon; its location is given in the last column, prepended with "c" for complement sequences. In violet are the experimentally identified transcription initiation sites in <i>Arabidopsis thaliana </i>and <i>Spinacia oleracea </i>upstream of <i>psbA</i>, <i>psbB</i>, <i>rbcL</i>, <i>psaA</i>. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p> <b>Nucleotide frequency distribution for the alignments shown in Fig. 1</b>. </p>   </div> <p> <b>Gene <i>psbB </i> </b>(a chlorophyll apoprotein of photosystem II CP47) in plastomes of Streptophyta. For this gene, the transcription start is experimentally identified in spinach ( <i>S. oleracea</i>) [ <a href="#B15">15</a>]; it adjoins the 3'-end of the accordingly named sequence in Fig. <a href="#F1">1</a>, <i>psbB</i>. A conserved promoter is predicted in most vascular plants: in angiosperms ( <i>A. thaliana</i>, <i>S. oleracea</i>), gymnosperms ( <i>Cycas taitungensis</i>, <i>Cryptomeria japonica</i>, <i>Welwitschia mirabilis</i>, <i>Pinus </i>spp.) and pteridophytes ( <i>Adiantum capillus-veneris</i>, <i>Angiopteris evecta</i>, <i>Psilotum nudum</i>, <i>Huperzia lucidula</i>). A related promoter is predicted in some algae ( <i>Chaetosphaeridium globosum</i>, <i>Chara vulgaris</i>, <i>Staurastrum punctulatum</i>, <i>Zygnema circumcarinatum</i>, <i>Chlorokybus atmophyticus</i>, <i>Mesostigma viride</i>), ref. to Fig. <a href="#F1">1</a>, <i>psbB</i>. This promoter is highly conserved in <i>C. taitungensis</i>, <i>C. japonica</i>, pteridophytes and streptophyte algae <i>C. globosum</i>, <i>C. vulgaris</i>, <i>S. punctulatum</i>, and less conserved in <i>Z. circumcarinatum</i>, <i>C. atmophyticus </i>and <i>M. viride</i>. It possesses the "-10" box TG-extension. In the early branching <i>C. atmophyticus </i>and <i>M. viride</i>, several potential promoters are predicted in 5'-leader regions; however these cannot be unambiguously added to the alignment of Streptophytina (Fig. <a href="#F1">1</a>, <i>psbB</i>), especially in the regions between the boxes and start codons. Therefore, the promoters closest to the start codon are selected and shown for <i>C. atmophyticus </i>and <i>M. viride</i>. In bryophytes ( <i>Aneura mirabilis</i>, <i>Anthoceros formosae</i>, <i>Marchantia polymorpha</i>, <i>Physcomitrella patens</i>), a conserved promoter was not found. Notably, the <i>psbB </i>sequence of <i>A. mirabilis </i>is annotated as a pseudogene in NCBI GenBank. The usual linker of 18 bp between the boxes is reduced to 17 bp in <i>W. mirabilis </i>and some algae ( <i>C. atmophyticus</i>, <i>S. punctulatum</i>, <i>Z. circumcarinatum</i>). In the pines <i>Pinus koraiensis </i>and <i>P. thunbergii</i>, the sequence differences are not shown (they occur in between the end of the sequence in Fig. <a href="#F1">1</a>, <i>psbB </i>and the conserved processing site shown in Fig. <a href="#F3">3</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p> <b>The 5'-leader regions upstream of gene <i>psbB</i> </b>. In the cells of first column only first occurrences of each taxon name are given. Numbers to the left of the sequences are distances from the 5'-edge to the start codon, which location is specified in the last column ("c" stands for complement sequences). In spinach the region is located precisely between the mRNA cleavage site and the start codon. Conserved putative mRNA-protein binding sites downstream of the cleavage site are shown in green. Conserved putative ribosome binding sites close to the start codon are in yellow. </p>   </div> <p> <b>Gene <i>psbE </i> </b>(photosystem II cytochrome b559 protein alpha subunit) in plastomes of Streptophyta. Promoters were predicted in most land plants and the algae <i>Chaetosphaeridium globosum</i>, <i>Staurastrum punctulatum</i>, <i>Zygnema circumcarinatum</i>, ref. to Fig. <a href="#F1">1</a>, <i>psbE</i>. Negative predictions were obtained for the algae <i>Chara vulgaris</i>, <i>Chlorokybus atmophyticus </i>and <i>Mesostigma viride</i>, even though the region is conserved in their closer relatives. This gene is a pseudogene in the <i>Aneura mirabilis </i>plastome. </p> <p> <b>Gene <i>rbcL </i> </b>(the large subunit of ribulose-1,5-bisphosphate carboxylase) in plastomes of Streptophyta. The promoter was experimentally characterized in spinach ( <i>S. oleracea</i>) [ <a href="#B13">13</a>], and mustard ( <i>Sinapis alba</i>) [ <a href="#B12">12</a>]. It was predicted in all land plants and in the streptophyte algae <i>Chaetosphaeridium globosum</i>, <i>Chara vulgaris</i>, <i>Staurastrum punctulatum</i>, <i>Zygnema circumcarinatum</i>, ref. to Fig. <a href="#F1">1</a>, <i>rbcL</i>. </p> <p> <b>Gene <i>psaA </i> </b>(apoprotein A1 of photosystem I P700) in plastomes of Streptophyta. Promoter and the transcription initiation site for this gene were experimentally characterized in <i>Arabidopsis thaliana </i>[ <a href="#B16">16</a>]. In <i>Aneura mirabilis </i>it is a pseudogene. The promoter was predicted in almost all land plants and streptophyte algae, except for <i>Chlorokybus atmophyticus </i>and <i>Mesostigma viride</i>, see Fig. <a href="#F1">1</a>, <i>psaA</i>. This promoter differs from all other predictions and the bacterial σ-70 promoter. Its "-10" box consensus is CATAAT, which differs from the bacterial type at the first position. At the 5'-end of the box a conserved putative extension is found with the consensus TrTGT. The predicted "-35" box is even more divergent from its counterparts, despite being located within a long conserved region. </p> <p>Although the alignments shown Fig. <a href="#F1">1</a> are unambiguous within the lineages, neither can be extended onto the Euglenozoa, Chlorophyta, Rhodophyta, Cryptophyta, diatom and other algae with plastids similar to those of the Rhodophyta, see add. file 1. </p> <p>Normally, the entire promoter region, not only the boxes, is more conserved comparing to the rest of the leader region, which hampers distinguishing between regulated and non-regulated promoters. </p> <p>We illustrate the comparison between wide and local conservations on the PEP-promoters of genes <i>ycf1</i>, <i>rps4 </i>and <i>psaJ</i>. The promoters were experimentally identified in <i>Arabidopsis thaliana</i>. These genes are among the 85 protein-coding genes in the plastome of <i>A. thaliana</i>. They are not widely conserved. </p> <p>The <i>ycf1 </i>gene encodes an unknown function protein and has PEP-promoter <i>ycf1</i>-34 with a smaller distance between the "-35" and "-10" boxes than normally [ <a href="#B3">3</a>]. This promoter overlaps with NEP-promoter <i>ycf1</i>-39. PEP-promoters very similar to <i>ycf1</i>-34 with unambiguous multiple alignments of the 5'-UTR regions are found in most eudicotyledonous, magnoliid and basal magnoliophyte plants. Some species (including <i>Cucumis sativus</i>) possess a much longer 5'-UTR region, while in others (including <i>Ranunculus macranthus</i>) the <i>ycf1 </i>PEP-promoter is not found. In monocotyledonous (Liliopsida), gymnosperm and pteridophyte plants possessing the <i>ycf1 </i>gene, its putative PEP-promoters are found but differ considerably from those in eudicotyledons, magnoliids and the basal Magnoliophyta. The promoter in <i>A. thaliana </i>is most similar to that from the cycadophyte <i>Cycas taitungensis</i>. </p> <p>In <i>A. thaliana </i>the gene <i>rps4 </i>encoding ribosomal protein S4 has PEP-promotor <i>rps4</i>-123 [ <a href="#B3">3</a>]. Similar promoters with unambiguous 5'-UTR multiple alignment are found only in selected species of Brassicaceae: <i>Arabis hirsuta</i>, <i>Barbarea verna</i>, <i>Crucihimalaya wallichii</i>, <i>Draba nemorosa</i>, <i>Lepidium virginicum</i>, <i>Lobularia maritima</i>, <i>Nasturtium officinale </i>and <i>Olimarabidopsis pumila</i>. The plastomes of <i>B. verna</i>, <i>D. nemorosa</i>, <i>L. maritima </i>and <i>O. pumila </i>contain single nucleotide insertions in between the boxes; <i>Arabis hirsute </i>has a single nucleotide deletion. The promoter region is variable even across close species ( <i>Aethionema cordifolium</i>, <i>A. grandiflorum</i>, <i>Carica papaya</i>, <i>Citrus sinensis</i>) but their 5'-UTR regions can still be well aligned. </p> <p> <i>A. thaliana </i>was experimentally found to possess a Sig2-dependent promoter upstream of gene <i>psaJ </i>encoding photosystem I active center subunit IX, with a 37 nucleotide-long 5'-UTR [ <a href="#B17">17</a>]. Although well aligned across all eurosids II, its 5'-UTR regions are conserved only within Brassicaceae and diverge already in <i>C. papaya</i>. </p> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Conserved promoters are found in the monophyletic Streptophyta and in two distant species, <i>B. natans </i>and <i>C. paradoxa</i>. Notably, even though <i>B. natans </i>belongs to the Cercozoa, its plastome is similar to that of green algae [ <a href="#B18">18</a>]. On the contrary, the plastome of <i>C. paradoxa </i>is different in many respects [ <a href="#B19">19</a>, <a href="#B20">20</a>]. </p> <p>There are many reasons why PEP-promoters upstream of the protein-coding plastome genes are scarce. Their loss may be related to the evolutionary changes of sigma subunit paralogs and phage-type RNA polymerases that lead to rapid replacements of the PEP-promoter. Indeed, the PEP sigma subunits vary already between maize, poplar and thale cress: e.g., maize possesses two Sig2 paralogs and lacks Sig4, while in poplar <i>sig4 </i>is a pseudogene, and thale cress possesses a Sig4 and only one Sig2, [ <a href="#B21">21</a>]. Also, promoters can be lost with their nuclear sigma subunit-encoding genes, such as the Sig4-dependent <i>ndhF </i>promoter in poplar [ <a href="#B5">5</a>]. Some dicotyledonous plants, including <i>Arabidopsis </i>and <i>Nicotiana</i>, have gained the additional phage-type RNA polymerase RpoTmp, which is active in chloroplasts and mitochondria of these plants but is missing from monocotyledonous plants (unpublished dissertation by K. Kühn, 2006). Only one phage-type RNA polymerase, RpoTp, is known from plastids of monocots ( <i>Zea</i>, <i>Triticum</i>), two phage-type RNA polymerases - from plastids of dicots ( <i>Arabidopsis</i>, <i>Nicotiana</i>): RpoTp in chloroplasts and RpoTmp in both chloroplasts and mitochondria. The moss <i>Physcomitrella patens </i>also has two phage-type polymerases, RpoT1 and RpoT2, which target both chloroplasts and mitochondria [ <a href="#B22">22</a>]. Promoters can emerge <i>de novo</i>, as has been shown, e.g., for the <i>ndhF </i>promoter [ <a href="#B5">5</a>]. Others are lost together with plastome genes, e.g., the <i>chlL </i>promoter in flowering and some other plants (according to the GenBank records). Another possible factor in rapid promoter turnover in plastids may be tissue-specific differentiation of plastid types, especially in vascular and, particularly, flowering plants, which evolved a rich diversity of sigma subunits [ <a href="#B21">21</a>] and phage type RNA polymerases. Often the promoter boxes are functionally substituted by the transcription activation factor binding sites [ <a href="#B4">4</a>]. </p> <p>In parasitic, non-photosynthesizing plants, such as dicotyledonous dodder ( <i>Cuscuta </i>spp.) and liverwort <i>Aneura mirabilis</i>, many chloroplast genes are pseudogenes [ <a href="#B23">23</a>] and promoters of these genes are lost too. The promoter conservation might become lower in the presence of alternative promoters. The promoter might have undergone rapid evolution [ <a href="#B3">3</a>, <a href="#B5">5</a>] and become unrecognizable. It also might be located beyond the 1000 bp distance from the start codon and thus be overlooked in our analyses. </p> <p>Given these multiple reasons to expect fast evolution and rapid turnover of the chloroplast promoters, one may ask why some of them, such as the five promoters described above, are so widely conserved? One possible explanation is that three of the conserved promoters regulate the expression of the photosystem components and that the stability of the promoter structure is important to maintain high expression of genes <i>psbA</i>, <i>psbB</i>, <i>psaA</i>; due to the light-dependent translation regulation of <i>psbA</i>, a high amount of mRNA is built up in the dark and translated under light [ <a href="#B24">24</a>]. Conserved promoters upstream of <i>psbA </i>and <i>psaA </i>may also be required to form polycistronic mRNAs, which encode, along with the photosystem components, tRNA and proteins involved in translation that also have to be expressed at high levels: <i>psbA </i>appears to belong to the same operon as histidine tRNA, while <i>psaAB </i>and <i>rps14 </i>are in an operon with methionine tRNA. The <i>psbEFLJ </i>operon and <i>psbBTHpetBD </i>operon might be formed likewise. The other conserved promoter regulates <i>rbcL</i>, the large subunit of a key enzyme involved in the carbon dioxide fixation during the Calvin cycle, the most abundant enzyme in the biosphere, whose gene also must be highly expressed. When a gene is highly transcribed and regulated by a single promoter, the selection pressure prevents any considerable change in the promoter's structure to provide for its effective binding to the polymerase. </p> <p>Relatively lower conservation of the PEP-promoters of housekeeping genes (viz., tRNA, rRNA, ribosomal protein and PEP subunit-encoding genes, etc.) might be explained by the presence of NEP transcription: e.g., the <i>rpoB </i>transcription is entirely NEP-mediated, although most genes possess both PEP and NEP-promoters. This is the case of the <i>ycf1 </i>and <i>clpP </i>genes, which were experimentally shown in <i>Arabidopsis thaliana </i>to be under several promoters recognized by PEP with different subunits and two NEP, RpoTp and RpoTmp, [ <a href="#B22">22</a>]. </p> <p>Operonic organization and RNA polymerase competition are important factors explaining the effect of genome rearrangements on the evolution of promoters. Thus, the loss of the common <i>ndhF </i>promoter and the emergence of a new one upstream of gene <i>ndhF </i>in poplar ( <i>Populus alba</i>, <i>P. trichocarpa</i>) concur with the deletion of a neighboring gene [ <a href="#B5">5</a>]. </p> <p>Some conserved promoters might be overlooked. For instance, the well studied <i>psbC </i>promoter is located within a coding region of other gene (according to the GenBank records) and its conservation cannot be assessed without estimating the synonymous vs. non-synonymous substitutions ratio, which is yet to be incorporated in our approach. Similar promoter-like regions were observed within other coding areas (unpublished data), but their role awaits explanation. </p> </div> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing interests</div> <p>The authors declare that they have no competing interests.</p> </div> <div class="authors'contributions" title="sec"> <div class="title" tagx="title" title="title">
Authors' contributions</div> <p>VAL and AVS performed the analyses, interpreted the results and developed the algorithms. LIR programmed the algorithms and ran computations. All authors contributed equally to preparing the manuscript. </p> </div> <div class="reviewers'comments" title="sec"> <div class="title" tagx="title" title="title">
Reviewers' comments</div> <p>Reviewer's report 1</p> <p>Arcady Mushegian, Stowers Institute</p> <p>The manuscript by Lyubetsky et al. examines the conservation of promoters in the choroplast genes of Streptophyta. The evidence is presented that, across large evolutionary distances (i.e., larger than the flowering plants clade) only a handful of promoter sequences contains conserved regions. This is an interesting observation suitable for publication in the Discovery Notes section of Biology Direct. </p> <p>1) 1st paragraph: the authors assert that there is no published evidence on searching for promoters at the genome scale. This is not true and needs to be qualified: there are many papers about eukaryotes and several about either methods to detect or databases of detected promotors in various groups of bacteria, some of which have been obtained using intergenomic conservation as one of the criteria. Citing the research behind J.Collado-Vides databases or RegulonDB might be in order. </p> <p>Response: This sentence lacks the word "..plastid.." which occurs widely in our text and is present in the title. We now refer to the works by professor Collado-Vides [ <a href="#B2">2</a>], which contain references to databases on promoters and regulation factors including the RegulonDB database. These databases and other citations in [ <a href="#B2">2</a>] are related to selected gamma-, alpha-proteobacteria and eukaryotic nucleoms. We do not see them as directly related to the "searching for the plastid promoters at the genomic scale". Particularly, the RegulonDB database does not contain photosynthesis and many other plastome genes because they lack in <i>E. coli</i>. The intergenomic conservation ideology is used in our algorithms [ <a href="#B6">6</a>, <a href="#B7">7</a>] but in a form different from that in [ <a href="#B2">2</a>]. </p> <p>2) Methods: references 4 and 5 are links to the authors' website with the documentation of their software. Why the reliance on the original code instead of the established methods of motif search and sequence alignment? Please explain crucial differences in the algorithms and how the homegrown ones were tested. </p> <p>Response: Studies [ <a href="#B9">9</a>, <a href="#B10">10</a>] report testing of the "first" algorithm in our approach in the comparison with established local alignment algorithms. The "second" algorithm and its testing was reported during a conference [ <a href="#B11">11</a>]. Widely used "standard" programs did not produce better promoter predictions (they are described in [ <a href="#B8">8</a>] and many related references). An explanation might be that we define a PEP-promoter as two boxes separated by a region (sometimes with a TG extension) variable in terms of structure and length; the imposed requirements are the degree of the variability of this region, the linker between the "-10" box and the start codon and the 5'-end of the "-35" box. The alignment of leader regions was built based on the precomputed two-boxed structures. It is more efficient to build it along a (usually known) species tree and not construct the alignment and the tree anew together as some approaches do. Ideologically the algorithms are described in the text, full details are given in [ <a href="#B6">6</a>, <a href="#B7">7</a>] and demonstrate their different performance comparing to other published methods. </p> <p>3) A suggestion that may help to provide a more complete picture of the evolutionary trends in chloroplast promoter conservation: <i>A. thaliana </i>chloroplast has 85 protein-coding genes. Can we have a table that shows, for each gene, how broadly its promoter is conserved? </p> <p>Response: The "Results" section now contains an analysis of PEP-promoter conservation upstream some coding genes in <i>A. thaliana</i>. An analysis of all 85 genes would be a subject for a separate publication. We show (as also noted in [ <a href="#B5">5</a>]) a typical problem in finding non-widely conserved promoters. Thus, well studied gene <i>ndhF </i>in <i>A. thaliana </i>is found to have only one PEP-promoter out of the four types known in Magnoliophyta, which is conserved across the Brassicaceae and predicted in all sequenced eurosids II and in <i>Vitis vinifera </i>[ <a href="#B5">5</a>]. Chloroplast PEP-promoters are experimentally unidentified for many coding genes in <i>A. thaliana</i>, while for many they are [ <a href="#B3">3</a>]. These promoters are conserved also in the Brassicaceae but already in eurosids II their recognition depends on imposed cut-offs and requires biological validation. For widely conserved promoters over-prediction is much lower than for promoters conserved within a thin lineage where the leader regions did not diverge to a noticeable extent. </p> </div> <div class="reviewer'sreport2" title="sec"> <div class="title" tagx="title" title="title">
Reviewer's report 2</div> <p>Alexander Bolshoy, University of Haifa (nominated by Purificación López-García, Université Paris-Sud) </p> <p>In the paper of Lyubetsky et al. conservation and variability of the plastid promoters is studied, and, to the best of my knowledge, for the first time at the whole genome level. Undoubtedly, the problem is important and non-trivial. The authors obtained unexpected result: promoter regions in plastids are less conservative than corresponding coding sequences. To identify promoters the authors proposed an original method of searching short motifs surrounded by certain other motifs. Thus, the proposed article includes an interesting problem, original methods to solve it and non-trivial results of analysis of promoter regions. It makes this article suitable for publication in the Discovery Notes section of Biology Direct. </p> <p>My remarks:</p> <p>1) In Background section you use a term "lower conservation". Can you show how have you compared protein conservation with promoter conservation? Response: Comparing to the PEP-promoters, their regulated proteins are always widely conserved and well aligned. A family present in vascular pants is almost ubiquitous, while known widely conserved PEP-promoters are only five. PEP-promoters might be more abundant than NEP-promoters: the knockout of RpoTp-NEP is not lethal for <i>A. thaliana</i>, while the PEP-promoter loss (e.g. in <i>Epifagus virginiana</i>) entails the loss of numerous genes. The authors are unaware of detailed estimates. </p> <p>2) In Background section you use the term "widely" to indicate that the leader region sequences upstream orthologous genes can be aligned across high-level taxonomic divisions. Please, give some details for better understanding of the term "widely conserved"? </p> <p>Please refer to Response #3 to Yu.W.</p> <p>3) In Background section the following phrase "... using the fixed consensus as a query produced massive under-predictions, or, alternatively, massive over-predictions..." needs some explanation. </p> <p>Response: A simple approach to the promoter search is to define a conserved query mask. Using masks very close to, e.g., the bacterial sigma-70 consensus, will lead to under-predictions because reliable PEP-promoters of different structure will be overlooked. Using diverged masks will lead to numerous false predictions. We believe that using a fixed per-site nucleotide frequency queries is not a perspective. </p> <p>4) Materials and methods. Please, give a short description of your algorithms.</p> <p>Response: We developed an original approach to the promoters search. At the first stage we find a two-boxed signal via local multiple alignment (the first algorithm, ref. to Response to A.M #2). For each leader region the algorithm predicts a number of candidate "-35" and "-10" boxes. The second algorithm aligns the promoter region, about 20 nucleotides upstream its "-35" box and the transcribed region up to the start codon (the part of the alignment is given in Fig. <a href="#F1">1</a>) and chooses the putative boxes taking into account the distance between them (typically 17-18 nucleotides) and their affinity on the species tree (closer species have more similar sequences). The algorithms are described in detail in [ <a href="#B6">6</a>, <a href="#B7">7</a>]. </p> <p>5) Results. Why the authors insist to strengthen differences between plastid REP-promoter of <i>psaA </i>gene and bacterial σ-70 promoters? </p> <p>Response: The <i>psaA </i>leader regions have a reliable long alignment, which accents the fact that this promoter considerably differs from the bacterial sigma-70 consensus. </p> </div> <div class="reviewer'sreport3" title="sec"> <div class="title" tagx="title" title="title">
Reviewer's report 3</div> <p>Yuri I. Wolf, National Center for Biotechnology Information (nominated by Purificación López-García, Université Paris-Sud) </p> <p>The authors report the virtual lack of conservation of Plastid-Encoded Polymerase promoters among the various lineages of plants. The finding is quite noteworthy and would be of interest to those who study the evolution of regulatory elements and plastid genomes. </p> <p>1) p. 2. "Plastid genes and their promoters are believed to be evolutionarily conserved across large taxonomic lineages". This is a strong statement that requires at least a couple of references, indicated who, when and in what form expressed these beliefs. </p> <p>Response: In [2, 9.7c] (this reference is added) the authors state that "The structure of chloroplast... genes is widely conserved across lineages. Their evolutionary rate is much lower than that of nuclear genes." This seems to be a common knowledge from textbooks (references can be added if necessary). Our logic was first straight: highly conserved genes cannot have low conserved promoters. But out results show the opposite. The phrase "and their promoters" is now removed. </p> <p>2) p.3 and throughout. "The term "widely" is used to indicate...". The authors attempt to clarify the usage of the term "widely", but actually just substitute it by no less vague "across high-level taxonomic divisions". I suggest to specify the "high-level taxonomic divisions" used in the definition of "widely" and avoid the italicized usage of this term further in the text. </p> <p>Response: An alignment was called "widely conserved" when included the Magnoliophyta and at least two representatives (at least one must not be a vascular plant) from Cycadophyta, Coniferophyta, Gnetophyta, Moniliformopses, Lycopodiophyta, Marchantiophyta, Bryophyta, Charophyceae, Coleochaetophyceae, Zygnemophyceae, Mesostigmatophyceae, Chlorarachniophyceae or Glaucocystophyceae. Each high lineage from Fig. <a href="#F1">1</a> is represented by few species because other species can usually be unambiguously aligned. These lineages are unbalanced in terms of molecular taxon sampling and are here represented by similar numbers of species. The term "widely conserved" will hopefully be given a more precise definition in the future. </p> <p>3) pp. 4-6. The gene-specific section of the Results reads like a verbal narration of the content of the Table <a href="#T1">1</a>. It is not clear why the authors need such a detailed listing of facts that don't seem to lead to any particular conclusions. I would recommend considering the possibility of removing this part from Results altogether, joining Results and Discussion and use the extra available space to somewhat expand the Methods section. </p> <p>Response: The "Results" do not just state the fact of the widely conserved promoter and its distance from the gene (which is indeed evident from Table <a href="#T1">1</a>) but also comparisons of the orthologous gene promoters supported by the alignment analyses and interpretations of published data. The authors believe this section should be kept at least structurally. It might be technically merged with the Discussion but its contents should remain. Discussion elements in the Results are directly related to the details described. If the note is to be reduced, we argue for moving Fig. <a href="#F2">2</a> (and, if needed, Table <a href="#T1">1</a>) into the supplementary data. </p> <p>4) Promoter blocks for different genes seem to be aligned, but all shown sequences have different lengths. This leads to a seemingly paradoxical result - the magenta mark for the experimentally identified transcription initiation site in <i>psbB </i>of <i>Spinacia oleracea </i>highlights an empty space. </p> <p>Response: The Figure <a href="#F1">1</a> shows a good alignment, which length cannot be amended. If the <i>psbB </i>alignment is appended some columns to the right, its quality will decrease. In magenta is now a character existing in this position in spinach an experimentally proved to be at the transcription start. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Additional file 1</div> <p> <b>The list of plastomes examined for conserved PEP promoters</b>. The data were extracted from GenBank, NCBI. </p>  <div class="media" title="media"> <a href="1745-6150-5-34-S1.XLS">LINK</a>  <p>Click here for file</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="acknowledgements" title="sec"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>The authors are grateful to Leonid Rusin for valuable discussion and help in manuscript preparation. This study was partly supported by the International Science &amp;amp; Technology Center (project no. 3807) and the Federal Agency for Education (grant no. P2370). </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"> <a name="B1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singer</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berg</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="source" tagx="source" title="source">Genes and Genomes</span> <span class="year" tagx="year" title="year">1991</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Blackwell Scientific Publications Ltd. Oxford</span> </span> </li> <li tag="ref"> <a name="B2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Collado-Vides</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salgado</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morett</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gama-Castro</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiménez-Jacinto</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martínez-Flores</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Medina-Rivera</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Muñiz-Rascado</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peralta-Gil</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Santos-Zavaleta</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="mixed-article-title" title="mixed-article-title">Bioinformatics resources for the study of gene regulation in bacteria</span> <span class="source" tagx="source" title="source">J Bacteriol</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">191</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">23</span> <span class="lpage" tagx="lpage" title="lpage">31</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JB.01017-08">10.1128/JB.01017-08</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18978060">18978060</a> </span> </span> </li> <li tag="ref"> <a name="B3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Swiatecka-Hagenbruch</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liere</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borner</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="mixed-article-title" title="mixed-article-title">High diversity of plastidial promoters in <i>Arabidopsis thaliana</i> </span> <span class="source" tagx="source" title="source">Mol Genet Genomics</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">277</span> <span class="fpage" tagx="fpage" title="fpage">725</span> <span class="lpage" tagx="lpage" title="lpage">734</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00438-007-0222-4">10.1007/s00438-007-0222-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17333279">17333279</a> </span> </span> </li> <li tag="ref"> <a name="B4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoffer</span> <span class="given-names" tagx="given-names" title="given-names">PH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Christopher</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span> <span class="mixed-article-title" title="mixed-article-title">Structure and blue-light-responsive transcription of a chloroplast <i>psbD </i>promoter from <i>Arabidopsis thaliana</i> </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">115</span> <span class="fpage" tagx="fpage" title="fpage">213</span> <span class="lpage" tagx="lpage" title="lpage">222</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.115.1.213">10.1104/pp.115.1.213</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9306699">9306699</a> </span> </span> </li> <li tag="ref"> <a name="B5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seliverstov</span> <span class="given-names" tagx="given-names" title="given-names">AV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lysenko</span> <span class="given-names" tagx="given-names" title="given-names">EA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lyubetsky</span> <span class="given-names" tagx="given-names" title="given-names">VA</span> </span> <span class="mixed-article-title" title="mixed-article-title">Rapid evolution of promoters for the plastome gene <i>ndhF </i>in flowering plants </span> <span class="source" tagx="source" title="source">Russian Journal of Plant Physiology</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">56</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">837</span> <span class="lpage" tagx="lpage" title="lpage">844</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1134/S1021443709060144">10.1134/S1021443709060144</a> </span> </span> </li> <li tag="ref"> <a name="B6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="mixed-article-title" title="mixed-article-title">Finding of multi-box regulatory signal in the set of unaligned sequences</span> <a href="http://lab6.iitp.ru/en/twobox/">http://lab6.iitp.ru/en/twobox/</a> </span> </li> <li tag="ref"> <a name="B7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="mixed-article-title" title="mixed-article-title">A tree-based method of sequence alignment</span> <a href="http://lab6.iitp.ru/en/treeal/">http://lab6.iitp.ru/en/treeal/</a> </span> </li> <li tag="ref"> <a name="B8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Favorov</span> <span class="given-names" tagx="given-names" title="given-names">AV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gelfand</span> <span class="given-names" tagx="given-names" title="given-names">MS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gerasimova</span> <span class="given-names" tagx="given-names" title="given-names">AV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ravcheev</span> <span class="given-names" tagx="given-names" title="given-names">DA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mironov</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Makeev</span> <span class="given-names" tagx="given-names" title="given-names">VYu</span> </span> <span class="mixed-article-title" title="mixed-article-title">A Gibbs sampler for identification of symmetrically structured, spaced DNA motifs with improved estimation of the signal length </span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">21</span> <span class="issue" tagx="issue" title="issue">10</span> <span class="fpage" tagx="fpage" title="fpage">2240</span> <span class="lpage" tagx="lpage" title="lpage">5</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/bti336">10.1093/bioinformatics/bti336</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15728117">15728117</a> </span> </span> </li> <li tag="ref"> <a name="B9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Danilova</span> <span class="given-names" tagx="given-names" title="given-names">LV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gorbunov</span> <span class="given-names" tagx="given-names" title="given-names">KIu</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gelfand</span> <span class="given-names" tagx="given-names" title="given-names">MS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liubetskiĭ (Lyubetsky)</span> <span class="given-names" tagx="given-names" title="given-names">VA</span> </span> <span class="mixed-article-title" title="mixed-article-title">Algorithms for isolating regulatory signals in DNA sequences</span> <span class="source" tagx="source" title="source">Mol Biol (Mosk)</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">35</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">987</span> <span class="lpage" tagx="lpage" title="lpage">95</span> <span class="comment">[Article in Russian].</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1023/A:1013282101105">10.1023/A:1013282101105</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11771146">11771146</a> </span> </span> </li> <li tag="ref"> <a name="B10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Danilova</span> <span class="given-names" tagx="given-names" title="given-names">LV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lyubetsky</span> <span class="given-names" tagx="given-names" title="given-names">VA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gelfand</span> <span class="given-names" tagx="given-names" title="given-names">MS</span> </span> <span class="mixed-article-title" title="mixed-article-title">An algorithm for identification of regulatory signals in unaligned DNA sequences, its testing and parallel implementation </span> <span class="source" tagx="source" title="source">In Silico Biol</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="issue" tagx="issue" title="issue">1-2</span> <span class="fpage" tagx="fpage" title="fpage">33</span> <span class="lpage" tagx="lpage" title="lpage">47</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12762844">12762844</a> </span> </span> </li> <li tag="ref"> <a name="B11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rubanov</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seliverstov</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lyubetsky</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="mixed-article-title" title="mixed-article-title">Multiple alignment based on species tree</span> <span class="source" tagx="source" title="source">Abstracts of the Sixth International Conference on Bioinformatics of Genome Regulation and Structure, BGRS'2008: 22-28 June 2008; Novosibirsk </span> <span class="year" tagx="year" title="year">2008</span> <span class="fpage" tagx="fpage" title="fpage">212</span> </span> </li> <li tag="ref"> <a name="B12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Homann</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Link</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="mixed-article-title" title="mixed-article-title">DNA-binding and transcription characteristics of three cloned sigma factors from mustard ( <i>Sinapis alba </i>L.) suggest overlapping and distinct roles in plastid gene expression </span> <span class="source" tagx="source" title="source">Eur J Biochem</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">270</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">1288</span> <span class="lpage" tagx="lpage" title="lpage">300</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1432-1033.2003.03494.x">10.1046/j.1432-1033.2003.03494.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12631287">12631287</a> </span> </span> </li> <li tag="ref"> <a name="B13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gruissem</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zurawski</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="mixed-article-title" title="mixed-article-title">Analysis of promoter regions for the spinach chloroplast <i>rbcL</i>, <i>atpB </i>and <i>psbA </i>genes </span> <span class="source" tagx="source" title="source">The EMBO Journal</span> <span class="year" tagx="year" title="year">1985</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="issue" tagx="issue" title="issue">13A</span> <span class="fpage" tagx="fpage" title="fpage">3375</span> <span class="lpage" tagx="lpage" title="lpage">3383</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2868888">2868888</a> </span> </span> </li> <li tag="ref"> <a name="B14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crooks</span> <span class="given-names" tagx="given-names" title="given-names">GE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hon</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chandonia</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brenner</span> <span class="given-names" tagx="given-names" title="given-names">SE</span> </span> <span class="mixed-article-title" title="mixed-article-title">WebLogo: A sequence logo generator</span> <span class="source" tagx="source" title="source">Genome Research</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">1188</span> <span class="lpage" tagx="lpage" title="lpage">1190</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.849004">10.1101/gr.849004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15173120">15173120</a> </span> </span> </li> <li tag="ref"> <a name="B15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Westhoff</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Herrmann</span> <span class="given-names" tagx="given-names" title="given-names">RG</span> </span> <span class="mixed-article-title" title="mixed-article-title">Complex RNA maturation in chloroplasts. The <i>psbB </i>operon from spinach </span> <span class="source" tagx="source" title="source">Eur J Biochem</span> <span class="year" tagx="year" title="year">1988</span> <span class="volume" tagx="volume" title="volume">171</span> <span class="fpage" tagx="fpage" title="fpage">551</span> <span class="lpage" tagx="lpage" title="lpage">564</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1432-1033.1988.tb13824.x">10.1111/j.1432-1033.1988.tb13824.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2831053">2831053</a> </span> </span> </li> <li tag="ref"> <a name="B16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fey</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wagner</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Braütigam</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wirtz</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hell</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dietzmann</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leister</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oelmüller</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pfannschmidt</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="mixed-article-title" title="mixed-article-title">Retrograde plastid redox signals in the expression of nuclear genes for chloroplast proteins of <i>Arabidopsis thaliana</i> </span> <span class="source" tagx="source" title="source">The Journal of Biological Chemistry</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">280</span> <span class="fpage" tagx="fpage" title="fpage">5318</span> <span class="lpage" tagx="lpage" title="lpage">5328</span> <span class="comment"> <b>Additions and Corrections</b>, <i>ibid</i>, <b>280(17): </b>17572 </span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.M406358200">10.1074/jbc.M406358200</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15561727">15561727</a> </span> </span> </li> <li tag="ref"> <a name="B17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagashima</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hanaoka</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Motohashi</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seki</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanamaru</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takahashi</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanaka</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="mixed-article-title" title="mixed-article-title">DNA microarray analysis of plastid gene expression in <i>Arabidopsis </i>mutant deficient in a plastid transcription factor sigma, SIG2 </span> <span class="source" tagx="source" title="source">Biosci Biotechnol Biochem</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">68</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">694</span> <span class="lpage" tagx="lpage" title="lpage">704</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1271/bbb.68.694">10.1271/bbb.68.694</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15056905">15056905</a> </span> </span> </li> <li tag="ref"> <a name="B18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rogers</span> <span class="given-names" tagx="given-names" title="given-names">MB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gilson</span> <span class="given-names" tagx="given-names" title="given-names">PR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Su</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McFadden</span> <span class="given-names" tagx="given-names" title="given-names">GI</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keeling</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span> <span class="mixed-article-title" title="mixed-article-title">The complete chloroplast genome of the chlorarachniophyte <i>Bigelowiella natans</i>: evidence for independent origins of Chlorarachniophyte and Euglenid secondary endosymbionts </span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">24</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">54</span> <span class="lpage" tagx="lpage" title="lpage">62</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msl129">10.1093/molbev/msl129</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16990439">16990439</a> </span> </span> </li> <li tag="ref"> <a name="B19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lemieux</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Otis</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turmel</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="mixed-article-title" title="mixed-article-title">A clade uniting the green algae <i>Mesostigma viride </i>and <i>Chlorokybus atmophyticus </i>represents the deepest branch of the Streptophyta in chloroplast genome-based phylogenies </span> <span class="source" tagx="source" title="source">BMC Biology</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">17</span> </span> </li> <li tag="ref"> <a name="B20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nosenko</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hackett</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhattacharya</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogenomic analysis identifies red algal genes of endosymbiotic origin in the Chromalveolates</span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">663</span> <span class="lpage" tagx="lpage" title="lpage">674</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msj075">10.1093/molbev/msj075</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16357039">16357039</a> </span> </span> </li> <li tag="ref"> <a name="B21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lysenko</span> <span class="given-names" tagx="given-names" title="given-names">EA</span> </span> <span class="mixed-article-title" title="mixed-article-title">Plant sigma factors and their role in plastid transcription</span> <span class="source" tagx="source" title="source">Plant Cell Rep</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">26</span> <span class="fpage" tagx="fpage" title="fpage">845</span> <span class="lpage" tagx="lpage" title="lpage">859</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00299-007-0318-7">10.1007/s00299-007-0318-7</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17356883">17356883</a> </span> </span> </li> <li tag="ref"> <a name="B22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Swiatecka-Hagenbruch</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Emanuel</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hedtke</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liere</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borner</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="mixed-article-title" title="mixed-article-title">Impaired function of the phage-type RNA polymerase RpoTp in transcription of chloroplast genes is compensated by a second phage-type RNA polymerase </span> <span class="source" tagx="source" title="source">Nucleic Acids Research</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">36</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">785</span> <span class="lpage" tagx="lpage" title="lpage">792</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkm1111">10.1093/nar/gkm1111</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18084023">18084023</a> </span> </span> </li> <li tag="ref"> <a name="B23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wickett</span> <span class="given-names" tagx="given-names" title="given-names">NJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hansen</span> <span class="given-names" tagx="given-names" title="given-names">SK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roper</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuehl</span> <span class="given-names" tagx="given-names" title="given-names">JV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Plock</span> <span class="given-names" tagx="given-names" title="given-names">SA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolf</span> <span class="given-names" tagx="given-names" title="given-names">PG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Depamphilis</span> <span class="given-names" tagx="given-names" title="given-names">CW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boore</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goffinet</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="mixed-article-title" title="mixed-article-title">Functional Gene Losses Occur with Minimal Size Reduction in the Plastid Genome of the Parasitic Liverwort <i>Aneura mirabilis</i> </span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">393</span> <span class="lpage" tagx="lpage" title="lpage">401</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msm267">10.1093/molbev/msm267</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18056074">18056074</a> </span> </span> </li> <li tag="ref"> <a name="B24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seliverstov</span> <span class="given-names" tagx="given-names" title="given-names">AV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lyubetsky</span> <span class="given-names" tagx="given-names" title="given-names">VA</span> </span> <span class="mixed-article-title" title="mixed-article-title">Translation regulation of intron containing genes in chloroplasts</span> <span class="source" tagx="source" title="source">Journal of Bioinformatics and Computational Biology</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="fpage" tagx="fpage" title="fpage">783</span> <span class="lpage" tagx="lpage" title="lpage">793</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1142/S0219720006002235">10.1142/S0219720006002235</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17007067">17007067</a> </span> </span> </li> </ul> </div> </div>  </body></html>