<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">PLoS One</span> <span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span> <span class="publisher-id" title="publisher-id">plos</span> <span class="pmc" title="pmc">plosone</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6007896">6007896</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0199356">10.1371/journal.pone.0199356</a> </span> <span class="publisher-id" title="publisher-id">PONE-D-18-08257</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span>: <span class="subject" title="subject">Gene Expression</span> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Transcriptome Analysis</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Transcriptome Analysis</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">People and Places</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Geographical Locations</span>: <span class="subject" title="subject">Antarctica</span> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Physical Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Physics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Classical Mechanics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Mechanical Stress</span>: <span class="subject" title="subject">Thermal Stresses</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Eukaryota</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plants</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Nonvascular Plants</span>: <span class="subject" title="subject">Mosses</span> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span>: <span class="subject" title="subject">Gene Amplification</span> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Techniques</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Artificial Gene Amplification and Extension</span>: <span class="subject" title="subject">Polymerase Chain Reaction</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Techniques</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Artificial Gene Amplification and Extension</span>: <span class="subject" title="subject">Polymerase Chain Reaction</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Processes</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cellular Stress Responses</span>: <span class="subject" title="subject">Heat Shock Response</span> </div> </div> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Identification of reference genes for RT-qPCR in the Antarctic moss <i>Sanionia uncinata</i> under abiotic stress conditions </div> <div class="alt-title" title="alt-title">
Novel reference genes for RT-qPCR in <i>Sanionia uncinata</i> </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">Mira</span> </span> <span class="role" tagx="role" title="role">Data curation</span> <span class="role" tagx="role" title="role">Formal analysis</span> <span class="role" tagx="role" title="role">Investigation</span> <span class="role" tagx="role" title="role">Validation</span> <span class="role" tagx="role" title="role">Writing – original draft</span> <span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span> <a href="#aff001"> <sup>1</sup> </a> <a href="#aff002"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hong</span> <span class="given-names" tagx="given-names" title="given-names">Soon Gyu</span> </span> <span class="role" tagx="role" title="role">Conceptualization</span> <span class="role" tagx="role" title="role">Data curation</span> <span class="role" tagx="role" title="role">Formal analysis</span> <span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span> <a href="#aff003"> <sup>3</sup> </a> <a href="#aff004"> <sup>4</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">Hyun</span> </span> <span class="role" tagx="role" title="role">Data curation</span> <span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span> <a href="#aff001"> <sup>1</sup> </a> <a href="#aff004"> <sup>4</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">Byeong-ha</span> </span> <span class="role" tagx="role" title="role">Conceptualization</span> <span class="role" tagx="role" title="role">Writing – original draft</span> <span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span> <a href="#aff002"> <sup>2</sup> </a> <a href="#cor001">*</a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-5831-6345</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">Hyoungseok</span> </span> <span class="role" tagx="role" title="role">Conceptualization</span> <span class="role" tagx="role" title="role">Investigation</span> <span class="role" tagx="role" title="role">Writing – original draft</span> <span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span> <a href="#aff001"> <sup>1</sup> </a> <a href="#aff004"> <sup>4</sup> </a> <a href="#cor001">*</a> </span> </div> <span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">Unit of Polar Genomics, Korea Polar Research Institute, Incheon, South Korea</span> </span> <span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Department of Life Sciences, Sogang University, Seoul, South Korea</span> </span> <span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">Division of Life Sciences, Korea Polar Research Institute, Incheon, South Korea</span> </span> <span class="citation_author_institution" id="aff004">[4], <span class="addr-line" title="addr-line">Polar Science, University of Science &amp;amp; Technology, Daejeon, South Korea</span> </span> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baruffini</span> <span class="given-names" tagx="given-names" title="given-names">Enrico</span> </span> <span class="role" tagx="role" title="role">Editor</span> <a href="#edit1" /> </span> </div> <span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">University of Parma, ITALY</span> </span> <div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p> <b>Competing Interests: </b>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest. </p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">soulaid@kopri.re.kr</span> (HL); <span class="email" tagx="email" title="email">byeongha@sogang.ac.kr</span> (B-hL) </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-6-6</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span> </span> <span class="volume" tagx="volume" title="volume">13</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e0199356</span> <span class="history" title="history"> <span class="received" title="received">received: 2018-3-19</span> <span class="accepted" title="accepted">accepted: 2018-6-6</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Park et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited. </span> </span> </div> <a class="self-uri" href="pone.0199356.pdf" title="self-uri">pone.0199356.pdf</a> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p> <i>Sanionia uncinata</i> is a dominant moss species in the maritime Antarctic. Due to its high adaptability to harsh environments, this extremophile plant has been considered a good target for studying the molecular adaptation mechanisms of plants to a variety of environmental stresses. Despite the importance of <i>S</i>. <i>uncinata</i> as a representative Antarctic plant species for the identification and characterization of genes associated with abiotic stress tolerance, suitable reference genes, which are critical for RT-qPCR analyses, have not yet been identified. In this report, 11 traditionally used and 6 novel candidate reference genes were selected from transcriptome data of <i>S</i>. <i>uncinata</i> and the expression stability of these genes was evaluated under various abiotic stress conditions using three statistical algorithms; geNorm, NormFinder, and BestKeeper. The stability ranking analysis selected the best reference genes depending on the stress conditions. Among the 17 candidates, the most stable references were <i>POB1</i> and <i>UFD2</i> for cold stress, <i>POB1</i> and <i>AKB</i> for drought treatment, and <i>UFD2</i> and <i>AKB</i> for the field samples from a different water contents in Antarctica. Overall, novel genes <i>POB1</i> and <i>AKB</i> were the most reliable references across all samples, irrespective of experimental conditions. In addition, 6 novel candidate genes including <i>AKB</i>, <i>POB1</i> and <i>UFD2</i>, were more stable than the housekeeping genes traditionally used for internal controls, indicating that transcriptome data can be useful for identifying novel robust normalizers. The reference genes validated in this study will be useful for improving the accuracy of RT-qPCR analysis for gene expression studies of <i>S</i>. <i>uncinata</i> in Antarctica and for further functional genomic analysis of bryophytes. </p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100004230</span> <span class="institution" title="institution">Korea Polar Research Institute</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">PE18080</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">Hyun</span> </span> </span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100004230</span> <span class="institution" title="institution">Korea Polar Research Institute</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">PE18090</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hong</span> <span class="given-names" tagx="given-names" title="given-names">Soon Gyu</span> </span> </span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100003627</span> <span class="institution" title="institution">Rural Development Administration</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">PJ01330101</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">Byeong-ha</span> </span> </span> </div> <div class="funding-statement" title="funding-statement">
This work was supported by grants ‘Modeling responses of terrestrial organisms to environmental changes on King George Island’ (PE18090) to SGH, ‘The Polar Genomics 101 Project: Genome analysis of polar organisms and establishment of application platform’ (PE18080) to HP, funded by the Korea Polar Research Institute, and ‘Next-Generation BioGreen21 Program’ (PJ01330101) to BhL, funded by the Rural Development Administration, South Korea. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. </div> </div> <div class="counts" title="counts"> <span class="fig-count" title="fig-count">fig-count: </span> <span class="table-count" title="table-count">table-count: </span> <span class="page-count" title="page-count">page-count: </span> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="All relevant data are within the paper and its Supporting Information files." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>All relevant data are within the paper and its Supporting Information files.</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Bryophytes dominate polar regions by developing adaptive strategies, as reflected in their poikilohydric lifestyle, to withstand an array of harsh conditions [ <a href="#pone.0199356.ref001">1</a>]. In extremely cold and/or dry conditions, temperature and water availability are the most influential factors with respect to plant growth and carbon gain, which determine bryophyte productivity in the Antarctic [ <a href="#pone.0199356.ref002">2</a>, <a href="#pone.0199356.ref003">3</a>]. These plants have evolved desiccation tolerance by rapidly recovering net positive photosynthesis within minutes of rehydration [ <a href="#pone.0199356.ref001">1</a>]. </p> <p>The moss <i>Sanionia uncinata</i> (Hedw.) Loeske is one of the most dominant species of moss in both the Arctic and Antarctic regions and is widely distributed over coastal areas [ <a href="#pone.0199356.ref004">4</a>, <a href="#pone.0199356.ref005">5</a>]. This pleurocarpous moss species forms carpets and occurs in a wide variety of habitats, ranging from dry sites such as exposed rock surfaces to wet sites near streams and swampy ground [ <a href="#pone.0199356.ref006">6</a>– <a href="#pone.0199356.ref008">8</a>]. It has recently been reported that <i>S</i>. <i>uncinata</i> is more tolerant to desiccation stress than <i>Polytrichastrum alpinum</i>. <i>S</i>. <i>uncinata</i> showed a more rapid rate of recovery from complete water loss and could retain water in its tissues for a longer period of time compared with <i>P</i>. <i>alpinum</i> [ <a href="#pone.0199356.ref009">9</a>]. However, the molecular mechanisms underlying stress tolerance have not yet been identified. Understanding the expression patterns of stress response genes can help discover the tolerance mechanisms of <i>S</i>. <i>uncinata</i> in response to various abiotic stresses in extreme environments. </p> <p>Gene expression analysis is essential for uncovering the molecular mechanisms underlying various stress tolerances. Reverse transcription quantitative real-time polymerase chain reaction (RT-qPCR) is one of the most widely used techniques for gene expression studies due to its high sensitivity and specificity, with high-throughput capabilities and cost-effectiveness [ <a href="#pone.0199356.ref010">10</a>, <a href="#pone.0199356.ref011">11</a>]. However, when conducting gene expression analyses by RT-qPCR, various factors such as RNA quality, reverse transcription efficiency, cDNA quality, and amplification efficiency can significantly influence the reliability of the results [ <a href="#pone.0199356.ref012">12</a>]. To reduce the influence of these factors, selection of internal reference genes with stable expression levels is critical for obtaining reliable results [ <a href="#pone.0199356.ref013">13</a>]. Some reports have suggested that suitable reference genes should be selected in a species-specific manner and tested under particular experimental conditions when validating their reliability [ <a href="#pone.0199356.ref014">14</a>, <a href="#pone.0199356.ref015">15</a>].To ensure more precise results, it is necessary to use one or more stable reference genes to normalize the expression data of target genes [ <a href="#pone.0199356.ref016">16</a>, <a href="#pone.0199356.ref017">17</a>]. </p> <p>Housekeeping genes, such as β-actin ( <i>ACT</i>), glyceraldehyde-3-phosphate dehydrogenase ( <i>GAPDH</i>), α- or β-tubulin ( <i>TUB</i>), 18S ribosomal RNA ( <i>18S</i>), and ubiquitin ( <i>UBQ</i>), have been widely used as reference genes for RT-qPCR in plants [ <a href="#pone.0199356.ref018">18</a>, <a href="#pone.0199356.ref019">19</a>]. However, recent studies have shown that these commonly used reference genes might not be stably expressed under different experimental conditions [ <a href="#pone.0199356.ref013">13</a>, <a href="#pone.0199356.ref020">20</a>– <a href="#pone.0199356.ref023">23</a>] and under various abiotic stress conditions [ <a href="#pone.0199356.ref013">13</a>, <a href="#pone.0199356.ref020">20</a>, <a href="#pone.0199356.ref021">21</a>, <a href="#pone.0199356.ref024">24</a>], suggesting that the use of unauthenticated references could greatly affect the quantification of expression levels of a target gene. However, few reference genes have been tested in <i>Physcomitrella patens</i> [ <a href="#pone.0199356.ref025">25</a>] and even in extremely stress-tolerant dessert moss <i>Syntrichia caninervis</i> and the model liverwort <i>Marchantia polymorpha</i> [ <a href="#pone.0199356.ref026">26</a>, <a href="#pone.0199356.ref027">27</a>]. Therefore, it is necessary to identify and validate suitable reference genes under different experimental conditions prior to gene expression studies in bryophytes. </p> <p>In this study, we assessed 17 reference genes (11 traditional and 6 novel ones) from the transcriptome data of the Antarctic moss <i>S</i>. <i>uncinata</i>, and evaluated their reliabilities as internal controls for RT-qPCR analysis under three different experimental conditions; axenically cultured moss gametophytes under cold or drought treatments, and natural field samples with a water-level gradient in the maritime Antarctic. To select the most suitable reference genes, three different statistical tools, geNorm [ <a href="#pone.0199356.ref012">12</a>], NormFinder [ <a href="#pone.0199356.ref028">28</a>], and BestKeeper [ <a href="#pone.0199356.ref029">29</a>] were used to evaluate the expression stability of the candidate reference genes. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Material and methods</div> <div class="plantmaterialsandstresstreatments" title="sec"> <div class="title" tagx="title" title="title">
Plant materials and stress treatments</div> <p>Samples of <i>S</i>. <i>uncinata</i> were collected from a coastal area (62°13’13"S 58°46’18"W) on the Barton Peninsula of King George Island in January 2015. To prepare samples from various water gradients in the field (WGF), <i>S</i>. <i>uncinata</i> patches of 5 cm x 5 cm containing at least 50 gametophytes were collected from three Antarctic field populations exhibiting a series of water contents. They were designated W68, W48, and W32, representing <i>S</i>. <i>uncinata</i> gametophytes with water contents of 68%, 48%, and 32%, respectively. Four biological replicates were prepared for each condition. Some of the moss gametophytes were transported to the laboratory at the Korea Polar Research Institute and cultivated axenically on BCDAT solid media [ <a href="#pone.0199356.ref026">26</a>] in a climate chamber at 15°C with a 16-h light/8-h dark cycle. To prepare the cold experimental set, gametophyte colonies grown at 15°C were transferred to chambers at 2°C and incubated for 1, 3, and 8 h. For the drought experimental set, drought treatment was carried out by transferring colonies grown at 15°C to clean dishes, followed by air-drying at room temperature for 6, 12, and 24 h. For each treatment, four biological replicates were prepared and each replicate contained &amp;gt;20 gametophytes to avoid possible variations among individuals. Immediately after the stress treatments, samples were frozen in liquid nitrogen and stored at –80°C until further use ( <a href="#pone.0199356.t001">Table 1</a>). </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0199356.t001">10.1371/journal.pone.0199356.t001</a> <span class="label" tagx="label" title="label">Table 1</span>  <div class="title" tagx="title" title="title">
Growth and abiotic stress conditions tested on Antarctic moss <i>S</i>. <i>uncinata</i>. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>Experimental set</th> <th>Tissue</th> <th>condition</th> <th>Age of culture</th> <th>Medium</th> </tr> </thead> <tbody> <tr> <td> <b>Cold</b> </td> <td>gametophore</td> <td>cold control, 15°C</td> <td>40 days</td> <td>BCDAT</td> </tr> <tr> <td>gametophore</td> <td>cold 2°C, 1 h</td> <td>40 days</td> <td>BCDAT</td> </tr> <tr> <td>gametophore</td> <td>cold 2°C, 3 h</td> <td>40 days</td> <td>BCDAT</td> </tr> <tr> <td>gametophore</td> <td>cold 2°C, 8 h</td> <td>40 days</td> <td>BCDAT</td> </tr> <tr> <td> <b>Drought</b> </td> <td>gametophore</td> <td>drought 0 h, control</td> <td>40 days</td> <td>BCDAT</td> </tr> <tr> <td>gametophore</td> <td>drought 6 h</td> <td>40 days</td> <td>BCDAT</td> </tr> <tr> <td>gametophore</td> <td>drought 12 h</td> <td>40 days</td> <td>BCDAT</td> </tr> <tr> <td>gametophore</td> <td>drought 24 h</td> <td>40 days</td> <td>BCDAT</td> </tr> <tr> <td> <b>Water gradient in the field (WGF)</b> </td> <td>gametophore</td> <td>68% water content (W68)</td> <td /> <td /> </tr> <tr> <td>gametophore</td> <td>48% water content (W48)</td> <td /> <td /> </tr> <tr> <td>gametophore</td> <td>32% water content (W32)</td> <td /> <td /> </tr> </tbody> </table> </div> </div> </div> <div class="totalrnaextractionandcdnasynthesis" title="sec"> <div class="title" tagx="title" title="title">
Total RNA extraction and cDNA synthesis</div> <p>Total RNA was isolated using the RNeasy Plant Mini Kit (Qiagen). Genomic DNA contamination was eliminated using RNase-free DNaseI (Qiagen). RNA concentration, purity, and integrity were determined using a NanoDrop ND-2000 spectrophotometer (Thermo Fisher Scientific) and visually assessed via gel electrophoresis ( <a href="#pone.0199356.s001">S1 Fig</a>). Each RNA sample (2 μg) was reverse-transcribed using the M-MLV transcriptase enzyme (Enzynomics) according to the manufacturer’s instructions. Reverse transcription was performed at 42°C for 60 min in a final volume of 20 μl, and inactivation of the enzyme was achieved at 72°C for 5 min. All cDNA were stored at –20°C until further use. </p> </div> <div class="candidatereferencegeneselectionfromtranscriptomedata" title="sec"> <div class="title" tagx="title" title="title">
Candidate reference gene selection from transcriptome data</div> <p>Transcriptome sequencing was carried out using total RNA from the cold experimental set, which comprised gametophyte colonies grown at 15°C, transferred to chambers at 2°C, and incubated for 1, 3, and 8 h. Gametophyte samples grown at 15°C were used as a control. Four different biological replicates were prepared. To construct the sequencing library, 2 μg of total RNA of each sample was used as the input for the TruSeq RNA sample prep kit v2 (Illumina). Following validation and quantification, libraries were applied to a MiSeq Sequencer system (Illumina) using the Illumina MiSeq Reagent Kit v3 (600 cycles). After assembly using the CLC Genomics Workbench v8 module (Qiagen), the transcriptome dataset composed of 21,993 unigenes was employed to analyze the read counts, which were converted to fragments per kilobase per million mapped reads (FPKM values) [ <a href="#pone.0199356.ref030">30</a>]. To estimate the expression stability of each gene, the following indices of FPKM values, including mean expression value (MV), standard deviation (SD), and coefficient of variation (CV) value, which is obtained by dividing SD by MV, were calculated [ <a href="#pone.0199356.ref031">31</a>]. The genes with an MV above the half value of the average MV for total unigenes and a CV below 0.3 were considered to be stably expressed. In addition to the 12 stable candidates with low CV, 5 additional candidate genes that showed high similarity to known housekeeping genes were included, which showed CV values above 0.3 from the transcriptome analysis. Finally, 17 candidate reference genes were listed with a transcriptome CV value, gene annotation in the NCBI non-redundant database, and feasibility of primer design ( <a href="#pone.0199356.s005">S1 Table</a>). </p> </div> <div class="primerdesignandevaluationofcandidatereferencegenes" title="sec"> <div class="title" tagx="title" title="title">
Primer design and evaluation of candidate reference genes</div> <p>Sequences of the 17 candidate reference genes are provided in <a href="#pone.0199356.s004">S1 File</a> and were used to design primers via the PrimerQuest program on the Integrated DNA Technologies website ( <a href="http://sg.idtdna.com/Primerquest/">http://sg.idtdna.com/Primerquest/</a>). The criteria for primer design were as follows: melting temperature (T <sub>m</sub>), 59–65°C; primer length, 18–25 base pairs (bp); GC content, 40–55%; and amplicon length, 100–200 bp. Each primer set returned by the program was tested for homology to other genes using BLASTN. To confirm primer specificity, all primer pairs were initially tested by RT-PCR, and the amplification product for each gene was verified by 2% agarose gel electrophoresis. Amplification efficiency (E) was evaluated using a standard curve generated by RT-qPCR using a 10-fold dilution series (1, 1/10, 1/100, 1/1000, and 1/10,000). Primer specificity was confirmed using melting-curve analysis following RT-qPCR and gel electrophoresis analysis of the unique amplicon. A no template control (NTC) reaction was included in every PCR run for each primer set. </p> </div> <div class="rt-qpcr" title="sec"> <div class="title" tagx="title" title="title">
RT-qPCR</div> <p>RT-qPCR was performed following the guidelines of the Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) [ <a href="#pone.0199356.ref032">32</a>]. Each reaction was performed in 10 μL reactions including 2 μL of a 1:30 dilution of cDNA template, 2 μM of each primer, and 5 μL of SYBR Green (Takara). Amplified signals were monitored continuously with the Rotor Gene 3000 Real-Time Thermal Cycler (Qiagen). The amplification protocol was as follows: 5 min of denaturation and enzyme activation at 95°C, followed by 40 cycles at 95°C for 15 s, 55°C for 20 s, and 72°C for 15 s. Three biological replicates for each sample, and four technical replicates of each biological replicate, were used. </p> </div> <div class="rt-qpcrdataanalysisofreferencegenestability" title="sec"> <div class="title" tagx="title" title="title">
RT-qPCR data analysis of reference gene stability</div> <p>Quantification cycle (Cq) values indicating the level of gene expression were determined by RT-qPCR amplification of each candidate reference gene from all RNA samples. The RT-qPCR Cq value of each gene was obtained by calculating the arithmetic mean from four technological and three biological replicates. The correlation coefficients (R values), slopes, and corresponding RT-qPCR efficiency (E) were obtained from standard curves. To rank the reference genes based on gene expression stability under the different experimental conditions, we used three different algorithms: geNorm [ <a href="#pone.0199356.ref012">12</a>], NormFinder [ <a href="#pone.0199356.ref028">28</a>], and BestKeeper [ <a href="#pone.0199356.ref029">29</a>]. The Cq value of each reference gene was converted into an input file according to the respective software manuals. For geNorm and NormFinder analysis, the raw Cq values were converted into relative value using the formula Q = 2 <sup>−ΔCq</sup>, in which ΔCq equals to each corresponding Cq minus minimum Cq. These values were then submitted to the Excel-based geNorm applet to calculate the expression stability value (M), which is described as the average of the pairwise variation (V) of a candidate reference gene [ <a href="#pone.0199356.ref012">12</a>]. Also NormFinder was used to calculate the stability value using an ANOVA-based model to consider intra-group and inter-group variation of the candidate reference genes, with the lowest value representing the highest stability [ <a href="#pone.0199356.ref028">28</a>]. For BestKeeper, the raw data (Cq values) were used to calculate the coefficients of variance (CV) and the standard deviation (SD), with the lowest CV representing the highest stability [ <a href="#pone.0199356.ref029">29</a>]. Finally, three results of the stability rankings of 17 candidate reference genes were integrated to determine a comprehensive ranking by calculating the geometric mean of three rankings [ <a href="#pone.0199356.ref033">33</a>]. </p> </div> <div class="validationofidentifiedreferencegenes" title="sec"> <div class="title" tagx="title" title="title">
Validation of identified reference genes</div> <p>Two widely known abiotic stress-related genes, late embryogenesis abundant ( <i>LEA</i>; (MG020651; forward <tt>5’-TGCTCAATGAGTTTCGTGACTA-3’</tt> and reverse <tt>5’-ATGCAAGACCACCACCGATCATAG-3’</tt>) and heat shock protein 70 ( <i>HSP70</i>; MG721539; forward <tt>5’-AAAGAAGGAGCAGGTGTTCTC-3’</tt> and reverse <tt>5’-TTGCCGAGCAGGTTGTT-3’</tt>), were selected as target genes to measure the expression stability of the candidate reference genes under cold, drought, and WGF conditions. The gene expression levels were normalized with the two most stable, as well as the two least stable, candidate reference genes identified from this study. </p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="selectionofcandidatereferencegenes" title="sec"> <div class="title" tagx="title" title="title">
Selection of candidate reference genes</div> <p>To select the candidate reference genes, we first chose stably expressed genes from the <i>S</i>. <i>uncinata</i> transcriptome data obtained following treatment at 2°C for 1, 3, and 8 h. After transcriptome analysis with the CLC Genomics Workbench v8 module, we defined stably expressed genes as those fulfilling two conditions: coefficient of variation (CV) below 0.3 and mean expression value (MV) above half of the MV of the total genes. From this dataset of stably expressed genes, we identified 6 housekeeping genes with homology to 60S ribosomal protein L23A ( <i>60S897</i>), ubiquitin-conjugating enzyme E2 ( <i>E2</i>. <i>299</i>), E3 ubiquitin ligase ( <i>E3</i>. <i>528</i>), elongation factor 1-delta ( <i>EF1</i>. <i>278</i>), histone H3.3 isoform X1 ( <i>HIS</i>), and translation initiation factor eIF-2B ( <i>tIF</i>), which are commonly used as reference genes in plants. The six genes were designated as the gene group 1. </p> <p>Meanwhile, we selected 6 novel genes that have not been widely used as stably expressed genes, homologous to adenylate kinase 4 ( <i>AKB</i>), histone H3 methyltransferase complex subunit ( <i>HMT</i>), BTB/POZ domain-containing protein ( <i>POB1</i>), serine palmitoyltransferase 2 ( <i>SPT</i>), ubiquitin protein ligase ( <i>UFD2</i>), and an uncharacterized protein ( <i>UK552</i>), with stable expression patterns under control and cold stress treatments ( <a href="#pone.0199356.s005">S1 Table</a>). These genes were designated as the gene group 2. </p> <p>Additionally, 5 housekeeping genes that are traditionally used as internal reference genes were selected, which were not stably expressed in <i>S</i>. <i>uncinata</i> transcriptome data with CV above 0.3: Actin 5 ( <i>ACT5</i>), actin-related protein 9 ( <i>ARP</i>), glyceraldehyde-3-phosphate dehydrogenase ( <i>GAPDH</i>), alpha tubulin ( <i>SuTub</i>), and gamma tubulin ( <i>TUB250</i>). These genes were designated as the gene group 3 for using as control to evaluate the efficiency of reference genes depending on CV variation. Accession numbers and the expression values from the transcriptome analysis for all 17 candidates are listed in <a href="#pone.0199356.s005">S1 Table</a>. </p> </div> <div class="rt-qpcramplificationefficiencyandprimerspecificity" title="sec"> <div class="title" tagx="title" title="title">
RT-qPCR amplification efficiency and primer specificity</div> <p>The RT-qPCR primers were designed using PrimerQuest for the 17 candidate reference genes to generate an amplicon ranging in size from 101 to 152 bp for each reference gene ( <a href="#pone.0199356.t002">Table 2</a>). The PCR products were amplified using cDNA made from total RNA of <i>S</i>. <i>uncinata</i> gametophytes grown at 15°C as a template, and the specificity and accuracy of the primers were verified by 2% agarose gel electrophoresis ( <a href="#pone.0199356.s002">S2A Fig</a>). The dissociation curves of each PCR product showed a unique melting peak of fluorescence signatures, suggesting that only gene-specific DNA fragments were amplified during the PCR reaction with primer pairs of each candidate gene ( <a href="#pone.0199356.s002">S2B Fig</a>). The PCR efficiency of each primer pair was greater than 90%, and ranged from 94.17% ( <i>TUB250</i>) to 104.26% ( <i>GAPDH</i>); correlation coefficients ( <i>R</i> <sup>2</sup>) of the standard curve were between 0.96 ( <i>TUB250</i>) and 0.999 ( <i>UK552</i>) ( <a href="#pone.0199356.t002">Table 2</a> and <a href="#pone.0199356.s003">S3 Fig</a>). </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0199356.t002">10.1371/journal.pone.0199356.t002</a> <span class="label" tagx="label" title="label">Table 2</span>  <div class="title" tagx="title" title="title">
Primer sequences and amplicon characteristics of 17 candidate reference genes for RT-qPCR analysis. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>Gene group</th> <th>Gene symbol</th> <th>Accession number</th> <th>Gene name</th> <th>Primer sequence <br /><br />(Forward /Reverse) </th> <th>Amplicon size (bp)</th> <th>PCR efficiency (%)*</th> <th> <i>R</i> <sup>2</sup>* </th> </tr> </thead> <tbody> <tr> <td>1</td> <td> <i>60S897</i> </td> <td>MG020635</td> <td>60s ribosomal protein L23A</td> <td> <tt>CTAGAGGCTTGGCCTGTTT</tt> </td> <td>106</td> <td>105.72</td> <td>0.996</td> </tr> <tr> <td> <tt>TGACAAGGTAGGAAGCAATGA</tt> </td> </tr> <tr> <td>1</td> <td> <i>E2</i>. <i>299</i> </td> <td>MG020639</td> <td>Ubiquitin-conjugating enzyme E2</td> <td> <tt>CCAGTGATCTCACTCACATGC</tt> </td> <td>124</td> <td>101.05</td> <td>0.998</td> </tr> <tr> <td> <tt>TGTCTATGGCAGTCCCTTCT</tt> </td> </tr> <tr> <td>1</td> <td> <i>E3</i>. <i>528</i> </td> <td>MG020640</td> <td>E3 ubiquitin ligase</td> <td> <tt>GCAAGTTGGCTGCTTTCATTT</tt> </td> <td>109</td> <td>102.17</td> <td>0.993</td> </tr> <tr> <td> <tt>ACTTTAAGCGCTCACCAGAAG</tt> </td> </tr> <tr> <td>1</td> <td> <i>EF1</i>. <i>278</i> </td> <td>MG020641</td> <td>Elongation factor 1-delta</td> <td> <tt>CGCTTCTTCTTTCTGGGATACA</tt> </td> <td>121</td> <td>97.2</td> <td>0.984</td> </tr> <tr> <td> <tt>GTGCAAGGTTGAACACACATAC</tt> </td> </tr> <tr> <td>1</td> <td> <i>HIS</i> </td> <td>MG020644</td> <td>Histone H3.3 isoform X1</td> <td> <tt>TGTCAACATATCCGAAGCTAGT</tt> </td> <td>116</td> <td>99.83</td> <td>0.996</td> </tr> <tr> <td> <tt>TGAATAGAGTTCTCTTGTAGATTGC</tt> </td> </tr> <tr> <td>1</td> <td> <i>tIF</i> </td> <td>MG020647</td> <td>Translation initiation factor eIF-2B</td> <td> <tt>GGGTTCCTCGGTTCATGTT</tt> </td> <td>136</td> <td>99.79</td> <td>0.985</td> </tr> <tr> <td> <tt>CAAGGTGGTAGCCCTGTATT</tt> </td> </tr> <tr> <td>2</td> <td> <i>AKB</i> </td> <td>MG020637</td> <td>Adenylate kinase 4</td> <td> <tt>GCAAGCAAGCCATCCTATTTG</tt> </td> <td>101</td> <td>100.25</td> <td>0.994</td> </tr> <tr> <td> <tt>TGTTCAACAAGGCACACTTAAAC</tt> </td> </tr> <tr> <td>2</td> <td> <i>HMT</i> </td> <td>MG020643</td> <td>Histone H3 methyltransferase complex</td> <td> <tt>GAGGGTGGATCCAACAGATATT</tt> </td> <td>135</td> <td>101.39</td> <td>0.995</td> </tr> <tr> <td> <tt>CCTGATTTCAGGACTCCTATCG</tt> </td> </tr> <tr> <td>2</td> <td> <i>POB1</i> </td> <td>MG020645</td> <td>BTB/POZ domain-containing protein</td> <td> <tt>AGCGTAGATTAGGTTTCCAGTTC</tt> </td> <td>111</td> <td>94.96</td> <td>0.997</td> </tr> <tr> <td> <tt>TACGAAGCTGCGCAAAGT</tt> </td> </tr> <tr> <td>2</td> <td> <i>SPT</i> </td> <td>MG020646</td> <td>Serine palmitoyltransferase 2</td> <td> <tt>GAGACGCCCTACAGGATTTG</tt> </td> <td>108</td> <td>103.05</td> <td>0.997</td> </tr> <tr> <td> <tt>ATTCCTATGCCTTTGCTCACT</tt> </td> </tr> <tr> <td>2</td> <td> <i>UFD2</i> </td> <td>MG020649</td> <td>Ubiquitin-protein ligase</td> <td> <tt>GAGTTGATATGCTGCAATGCC</tt> </td> <td>103</td> <td>95.82</td> <td>0.997</td> </tr> <tr> <td> <tt>CTGGAAGCAACTCCTCACAA</tt> </td> </tr> <tr> <td>2</td> <td> <i>UK552</i> </td> <td>MG020650</td> <td>Uncharacterized protein</td> <td> <tt>AGATCTGACGTAAGCGTTGG</tt> </td> <td>101</td> <td>97.63</td> <td>0.999</td> </tr> <tr> <td> <tt>GCACAGGTTCTGAGGATCAA</tt> </td> </tr> <tr> <td>3</td> <td> <i>ACT5</i> </td> <td>MG020636</td> <td>Actin 5</td> <td> <tt>AGCATGAAGATCAAGGTGGTG</tt> </td> <td>101</td> <td>96.03</td> <td>0.982</td> </tr> <tr> <td> <tt>ATCCACATCTGCTGGAAAGTG</tt> </td> </tr> <tr> <td>3</td> <td> <i>ARP9</i> </td> <td>MG020638</td> <td>Actin-related protein 9</td> <td> <tt>GGCAGTGGCAGGAAGTATAG</tt> </td> <td>117</td> <td>99.54</td> <td>0.996</td> </tr> <tr> <td> <tt>TCTGTTCGTCGTCCACAAAT</tt> </td> </tr> <tr> <td>3</td> <td> <i>GAPDH</i> </td> <td>MG020642</td> <td>Glyceraldehyde-3-phosphate dehydrogenase</td> <td> <tt>ATCTGTTGGGACCAGTTTCAG</tt> </td> <td>109</td> <td>104.26</td> <td>0.998</td> </tr> <tr> <td> <tt>ACACGTGAGATCCAATGACAG</tt> </td> </tr> <tr> <td>3</td> <td> <i>SuTub</i> </td> <td>MG020651</td> <td>Alpha tubulin</td> <td> <tt>GCCACCATCAAGACCAAGA</tt> </td> <td>152</td> <td>95.34</td> <td>0.992</td> </tr> <tr> <td> <tt>CTGGTGCTGTTCGAGATCAT</tt> </td> </tr> <tr> <td>3</td> <td> <i>TUB250</i> </td> <td>MG020648</td> <td>Gamma tubulin</td> <td> <tt>AGGTCAGTCCGTCCAGAA</tt> </td> <td>105</td> <td>94.17</td> <td>0.96</td> </tr> <tr> <td> <tt>CCTGATGCCAGAGTCGTTAAA</tt> </td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>* Mean of 4 technical replicates</p> </div> </div> </div> </div> <div class="expressionofthecandidatereferencegenes" title="sec"> <div class="title" tagx="title" title="title">
Expression of the candidate reference genes</div> <p>We prepared three experimental sets to verify the stability of candidate reference genes under various abiotic stress conditions. Since the <i>S</i>. <i>uncinata</i> culture samples used in this study were originally grown in the Antarctic, we selected cold and drought as the representative abiotic stressors for this species. To verify gene expression patterns under cold stress, axenic cultured <i>S</i>. <i>uncinata</i> gametophytes were transferred from 15°C, which is close to the optimum temperature for net photosynthesis [ <a href="#pone.0199356.ref034">34</a>], to 2°C, which is close to the average temperature of the summer season in the Barton Peninsula. After incubating for 1, 3, and 8 h, total RNAs were prepared from each condition. To verify gene expression patterns under drought stress, cultured <i>S</i>. <i>uncinata</i> gametophytes were transferred from agar plates to clean dishes followed by air-drying at room temperature for 6, 12, and 24 h. In addition to the cultured gametophytes, Antarctic field samples were prepared to test the applicability of the selected reference genes to naturally growing plants. For this reason, <i>S</i>. <i>uncinata</i> gametophyte plants were collected from three Antarctic field populations with a series of water contents, designated as W68, W48, and W32 and representing <i>S</i>. <i>uncinata</i> gametophytes with 68%, 48%, and 32% water contents, respectively; this comprised the WGF experimental setup ( <a href="#pone.0199356.t001">Table 1</a>). </p> <p>Using the 11 different RNA samples from the three experimental conditions, the expression levels of the 17 candidate reference genes were analyzed. Following amplification of each candidate gene with the 11 different RNA samples, the median Cq values of the 11 PCR amplifications, and the difference between the minimal and maximal Cq values, were obtained ( <a href="#pone.0199356.g001">Fig 1</a>). The median Cq values for the reference genes ranged from 16 to 25. <i>Tub250</i> (24.25 ± 0.79) had the maximum median Cq value and <i>GAPDH</i> (17.18 ± 1.12) had the minimum, which indicates the lowest and highest expression, respectively. <i>ACT5</i> (18.95 ± 1.31) and <i>POB1</i> (21.5 ± 1.61) showed low gene expression variation, while <i>E3</i>. <i>528</i> (21.8 ± 3.98) and <i>SuTub</i> (21.8 ± 3.89) displayed high variation. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0199356.g001"> <span class="label" tagx="label" title="label">Fig 1</span> </a> </div>   <div class="title" tagx="title" title="title">
Distribution of the quantification cycle (Cq) values of the 17 candidate reference genes in <i>Sanionia uncinata</i> samples. </div> <p>The median Cq values for 11 samples are displayed in a boxplot indicating the interquartile range. Error bars represent the minimum and maximum Cq values and the middle panes show the mean values. Three biological replicates per sample and four technical replicates per biological replicate were used. </p>   </div> </div> <div class="referencegenestabilityanalysiswithgenorm" title="sec"> <div class="title" tagx="title" title="title">
Reference gene stability analysis with geNorm</div> <p>The gene expression stability of the 17 candidate reference genes was further evaluated by RT-qPCR Cq values and ranked using geNorm, NormFinder, and BestKeeper. The geNorm statistical algorithm determines gene expression stability (M) based on the average pairwise variations of all tested genes [ <a href="#pone.0199356.ref012">12</a>]. In this analysis, lower M values indicate more stable gene expression. When the geNorm algorithm was applied to the 17 candidate references from <i>S</i>. <i>uncinata</i>, they all exhibited M values lower than the suggested threshold value of 0.5 ( <a href="#pone.0199356.g002">Fig 2</a>). Under the cold conditions, <i>POB1</i> and <i>UFD2</i> (M = 0.151 for both) were identified as the most stably expressed genes and <i>EF1</i>. <i>278</i> (M = 0.418) showed the least stability. Under drought conditions, <i>tIF</i> and <i>AKB</i> (M = 0.028 and 0.035, respectively) were among the most stable genes, while <i>TUB250</i> (M = 0.172) was the least stable. <i>E2</i>. <i>299</i> and <i>AKB</i> (M = 0.06 for both) were ranked as the most stable in the WGF experimental set, whereas <i>SuTub</i> (M = 0.383) was determined to be the least stable. When the 11 RNA samples from all three experimental sets were considered together, the most stable genes were <i>POB1</i> and <i>AKB</i> (M = 0.205 and 0.213, respectively), whereas the most unstable gene was <i>GAPDH</i> (M = 0.509) ( <a href="#pone.0199356.g002">Fig 2</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0199356.g002"> <span class="label" tagx="label" title="label">Fig 2</span> </a> </div>   <div class="title" tagx="title" title="title">
Expression stability values (M) of the candidate reference genes based on the geNorm algorithm. </div> <p>The cutoff for reference gene stability was proposed to be an M value of 0.5. Candidate reference genes were ranked in ascending order of expression stability, from the least stable on the left to the most stable gene on the right. (A) Cold experimental set, (B) drought experimental set, (C) WGF experimental set, and (D) total experimental set. </p>   </div> <p>Notably, some candidate genes showed highly variable expression depending on the experimental conditions. For example, <i>GAPDH</i> had a low M value in the cold (M = 0.221), drought (M = 0.079), and WGF (M = 0.106) experimental sets ( <a href="#pone.0199356.g002">Fig 2A–2C</a>), but showed the highest score when tested with the total set (M = 0.509) ( <a href="#pone.0199356.g002">Fig 2D</a>). <i>HIS</i> had a low score under drought (M = 0.108) and WGF (M = 0.113) conditions, but exhibited a high score in the total set (M = 0.485) ( <a href="#pone.0199356.g002">Fig 2D</a>). These results indicated that there could be a large difference in stability rankings between a single stressor and multiple stressors, underscoring the importance of validation of the reference genes in the intended experimental conditions. </p> <p>The geNorm also calculates a normalization factor to determine the optimal number of reference genes for accurate normalization, based on pairwise variations (Vn/n + 1) between each combination of sequential normalization factors. A cutoff value of 0.15 is the recommended threshold, which indicates that an additional reference gene will make no remarkable contribution to the normalization [ <a href="#pone.0199356.ref012">12</a>]. The V2/3 values in the total, cold, drought, and WGF experimental sets were less than 0.15 ( <a href="#pone.0199356.g003">Fig 3</a>), suggesting that the two most stable reference genes are sufficient for accurate normalization. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0199356.g003"> <span class="label" tagx="label" title="label">Fig 3</span> </a> </div>   <div class="title" tagx="title" title="title">
Pairwise variation (Vn/Vn+1) analysis of the candidate reference genes.</div> <p>The pairwise variation (Vn/Vn+1) was analyzed based on the geNorm algorithm to determine the optimal number of reference genes for accurate normalization. A cutoff value of 0.15 was set for the pairwise variation, as recommended by Vandesompele et al (2002) [ <a href="#pone.0199356.ref012">12</a>]. </p>   </div> </div> <div class="referencegenestabilityanalysiswithnormfinder" title="sec"> <div class="title" tagx="title" title="title">
Reference gene stability analysis with NormFinder</div> <p>The 17 candidate reference genes were further evaluated with the NormFinder algorithm to identify the optimal normalization genes under the different experimental conditions. NormFinder evaluates the stability of reference genes with the expression variations of candidate genes, using a model-based approach to rank candidate reference genes based on intra- and inter-group expression variation [ <a href="#pone.0199356.ref028">28</a>]. Similar to the geNorm analysis, lower stability values indicate more stable gene expression. While NormFinder predicted <i>POB1</i> as the most stable gene under the cold, drought, and total experimental set, <i>POB1</i> was the fourth most stable gene in the WGF experimental sets ( <a href="#pone.0199356.g004">Fig 4</a>). Although the rankings of <i>POB1</i>, <i>UFD2</i>, and <i>AKB</i> varied, they were commonly predicted as the top five stable genes in all four experimental sets. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0199356.g004"> <span class="label" tagx="label" title="label">Fig 4</span> </a> </div>   <div class="title" tagx="title" title="title">
Expression stability values and ranking of the candidate reference genes based on the NormFinder algorithm. </div> <p>Candidate reference genes were ranked in ascending order of expression stability, from the least stable on the left to the most stable gene on the right. (A) Cold experimental set, (B) drought experimental set, (C) WGF experimental set, and (D) total experimental set. </p>   </div> </div> <div class="referencegenestabilityanalysiswithbestkeeper" title="sec"> <div class="title" tagx="title" title="title">
Reference gene stability analysis with BestKeeper</div> <p>BestKeeper uses the SD and CV of Cq values to calculate the expression stability of candidate reference genes [ <a href="#pone.0199356.ref029">29</a>]. When the expression stability of the 17 candidate genes was analyzed via BestKeeper ( <a href="#pone.0199356.t003">Table 3</a>), two genes showing the lowest CV±SD values [ <i>HMT</i> (1.69±0.38) and <i>AKB</i> (2.05±0.45)] were confirmed as the most stable genes in total experimental set. Under cold condition, <i>TUB250</i> and <i>AKB</i>, which showed the lowest CV±SD values of 2.07±0.51 and 2.49±0.54, respectively, were ranked as the top two most stable genes. The results under drought conditions revealed that <i>POB1</i> (CV±SD = 0.71±0.15) and <i>ARP9</i> (CV±SD = 0.86±0.19) were the most stable genes. Furthermore, in the WGF experimental set, <i>UFD2</i> (CV±SD = 0.17±0.04) and <i>SPT</i> (CV±SD = 0.20±0.05) were identified as the most stable genes by BestKeeper. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0199356.t003">10.1371/journal.pone.0199356.t003</a> <span class="label" tagx="label" title="label">Table 3</span>  <div class="title" tagx="title" title="title">
Stability analysis of 17 candidate reference genes using BestKeeper.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>Rank</th> <th>Cold</th> <th>Drought</th> <th>WGF</th> <th>Total</th> </tr> <tr> <th>Gene</th> <th>SD</th> <th>CV</th> <th>Gene</th> <th>SD</th> <th>CV</th> <th>Gene</th> <th>SD</th> <th>CV</th> <th>Gene</th> <th>SD</th> <th>CV</th> </tr> </thead> <tbody> <tr> <td>1</td> <td> <i>TUB250</i> </td> <td>0.51</td> <td>2.07</td> <td> <i>POB1</i> </td> <td>0.15</td> <td>0.71</td> <td> <i>UFD2</i> </td> <td>0.04</td> <td>0.17</td> <td> <i>HMT</i> </td> <td>0.38</td> <td>1.69</td> </tr> <tr> <td>2</td> <td> <i>AKB</i> </td> <td>0.54</td> <td>2.49</td> <td> <i>ARP9</i> </td> <td>0.19</td> <td>0.86</td> <td> <i>SPT</i> </td> <td>0.05</td> <td>0.20</td> <td> <i>AKB</i> </td> <td>0.45</td> <td>2.05</td> </tr> <tr> <td>3</td> <td> <i>SuTub</i> </td> <td>0.57</td> <td>2.57</td> <td> <i>AKB</i> </td> <td>0.22</td> <td>1.00</td> <td> <i>GAPDH</i> </td> <td>0.05</td> <td>0.25</td> <td> <i>POB1</i> </td> <td>0.51</td> <td>2.32</td> </tr> <tr> <td>4</td> <td> <i>ARP9</i> </td> <td>0.60</td> <td>2.68</td> <td> <i>SuTub</i> </td> <td>0.23</td> <td>1.08</td> <td> <i>60S897</i> </td> <td>0.14</td> <td>0.61</td> <td> <i>ARP9</i> </td> <td>0.57</td> <td>2.50</td> </tr> <tr> <td>5</td> <td> <i>UFD2</i> </td> <td>0.62</td> <td>2.87</td> <td> <i>GAPDH</i> </td> <td>0.24</td> <td>1.46</td> <td> <i>HMT</i> </td> <td>0.14</td> <td>0.62</td> <td> <i>E3</i>. <i>528</i> </td> <td>0.57</td> <td>2.64</td> </tr> <tr> <td>6</td> <td> <i>SPT</i> </td> <td>0.68</td> <td>3.07</td> <td> <i>ACT5</i> </td> <td>0.26</td> <td>1.45</td> <td> <i>POB1</i> </td> <td>0.16</td> <td>0.74</td> <td> <i>E2</i>. <i>299</i> </td> <td>0.58</td> <td>2.53</td> </tr> <tr> <td>7</td> <td> <i>POB1</i> </td> <td>0.70</td> <td>3.22</td> <td> <i>E2</i>. <i>299</i> </td> <td>0.27</td> <td>1.21</td> <td> <i>E3</i>. <i>528</i> </td> <td>0.18</td> <td>0.82</td> <td> <i>UFD2</i> </td> <td>0.58</td> <td>2.70</td> </tr> <tr> <td>8</td> <td> <i>E3</i>. <i>528</i> </td> <td>0.70</td> <td>3.26</td> <td> <i>60S897</i> </td> <td>0.30</td> <td>1.39</td> <td> <i>EF1</i>. <i>278</i> </td> <td>0.19</td> <td>0.75</td> <td> <i>UK552</i> </td> <td>0.58</td> <td>2.89</td> </tr> <tr> <td>9</td> <td> <i>EF1</i>. <i>278</i> </td> <td>0.71</td> <td>3.00</td> <td> <i>HMT</i> </td> <td>0.30</td> <td>1.33</td> <td> <i>AKB</i> </td> <td>0.22</td> <td>1.00</td> <td> <i>ACT5</i> </td> <td>0.59</td> <td>3.14</td> </tr> <tr> <td>10</td> <td> <i>HMT</i> </td> <td>0.71</td> <td>3.16</td> <td> <i>UFD2</i> </td> <td>0.30</td> <td>1.44</td> <td> <i>TUB250</i> </td> <td>0.23</td> <td>0.93</td> <td> <i>TUB250</i> </td> <td>0.60</td> <td>2.45</td> </tr> <tr> <td>11</td> <td> <i>GAPDH</i> </td> <td>0.71</td> <td>4.27</td> <td> <i>EF1</i>. <i>278</i> </td> <td>0.31</td> <td>1.36</td> <td> <i>HIS</i> </td> <td>0.23</td> <td>0.97</td> <td> <i>SPT</i> </td> <td>0.63</td> <td>2.88</td> </tr> <tr> <td>12</td> <td> <i>E2</i>. <i>299</i> </td> <td>0.73</td> <td>3.23</td> <td> <i>UK552</i> </td> <td>0.33</td> <td>1.64</td> <td> <i>ARP9</i> </td> <td>0.24</td> <td>1.04</td> <td> <i>SuTub</i> </td> <td>0.66</td> <td>2.98</td> </tr> <tr> <td>13</td> <td> <i>60S897</i> </td> <td>0.73</td> <td>3.27</td> <td> <i>TUB250</i> </td> <td>0.34</td> <td>1.46</td> <td> <i>SuTub</i> </td> <td>0.24</td> <td>1.05</td> <td> <i>60S897</i> </td> <td>0.72</td> <td>3.21</td> </tr> <tr> <td>14</td> <td> <i>ACT5</i> </td> <td>0.73</td> <td>3.80</td> <td> <i>HIS</i> </td> <td>0.35</td> <td>1.57</td> <td> <i>E2</i>. <i>299</i> </td> <td>0.27</td> <td>1.15</td> <td> <i>GAPDH</i> </td> <td>0.72</td> <td>4.20</td> </tr> <tr> <td>15</td> <td> <i>UK552</i> </td> <td>0.80</td> <td>4.10</td> <td> <i>SPT</i> </td> <td>0.38</td> <td>1.78</td> <td> <i>UK552</i> </td> <td>0.34</td> <td>1.65</td> <td> <i>EF1</i>. <i>278</i> </td> <td>0.80</td> <td>3.38</td> </tr> <tr> <td>16</td> <td> <i>HIS</i> </td> <td>0.83</td> <td>3.64</td> <td> <i>E3</i>. <i>528</i> </td> <td>0.70</td> <td>3.29</td> <td> <i>tIF</i> </td> <td>0.35</td> <td>1.92</td> <td> <i>HIS</i> </td> <td>0.90</td> <td>3.94</td> </tr> <tr> <td>17</td> <td> <i>tIF</i> </td> <td>1.15</td> <td>6.18</td> <td> <i>tIF</i> </td> <td>0.91</td> <td>5.34</td> <td> <i>ACT5</i> </td> <td>0.43</td> <td>2.22</td> <td> <i>tIF</i> </td> <td>0.94</td> <td>5.26</td> </tr> </tbody> </table> </div> </div> </div> <div class="comprehensivereferencegeneselection" title="sec"> <div class="title" tagx="title" title="title">
Comprehensive reference gene selection</div> <p>To determine the most stable genes predicted by the three different algorithms, we integrated the results from geNorm, NormFinder, and BestKeeper to determine comprehensive ranking for each candidate reference gene based on the geometric mean of rankings from three algorithms [ <a href="#pone.0199356.ref035">35</a>]. As shown in <a href="#pone.0199356.t004">Table 4</a>, <i>POB1</i> and <i>UFD2</i> were recommended as the most stable reference genes under cold conditions. <i>AKB</i> and <i>POB1</i> were chosen as the most stable genes for the drought and all experimental sets. Moreover, <i>UFD2</i> and <i>AKB</i> were predicted as the most stable genes in the WGF experimental set. In reverse, we could select the least stable genes for each experimental set. Overall, traditional housekeeping genes belonging to the gene group 3, such as <i>ACT5</i>, <i>GAPDH</i>, <i>SuTub</i>, and <i>TUB250</i>, showed low stability in experimental sets tested ( <a href="#pone.0199356.t004">Table 4</a>). Interestingly, <i>UFD2</i> was ranked to the most stable genes in the cold and WGF experimental sets, however, it was one of the two least stable genes in the drought experimental set. </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0199356.t004">10.1371/journal.pone.0199356.t004</a> <span class="label" tagx="label" title="label">Table 4</span>  <div class="title" tagx="title" title="title">
Comprehensive stability ranking of the candidate reference genes.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <tbody> <tr> <td> <b>Method</b> </td> <td> <b>1</b> </td> <td> <b>2</b> </td> <td> <b>3</b> </td> <td> <b>4</b> </td> <td> <b>5</b> </td> <td> <b>6</b> </td> <td> <b>7</b> </td> <td> <b>8</b> </td> <td> <b>9</b> </td> </tr> <tr> <td> <b>Ranking order for the cold experimental set (Better—Good—Average)</b> </td> </tr> <tr> <td>geNorm</td> <td> <i>POB1/UFD2</i> </td> <td> <i>UK552</i> </td> <td> <i>HMT</i> </td> <td> <i>GAPDH</i> </td> <td> <i>ACT5</i> </td> <td> <i>SuTub</i> </td> <td> <i>E2</i>. <i>299</i> </td> <td> <i>TUB250</i> </td> </tr> <tr> <td>NormFinder</td> <td> <i>POB1</i> </td> <td> <i>UK552</i> </td> <td> <i>UFD2</i> </td> <td> <i>E2</i>. <i>299</i> </td> <td> <i>AKB</i> </td> <td> <i>60S897</i> </td> <td> <i>TUB250</i> </td> <td> <i>HMT</i> </td> <td> <i>SuTub</i> </td> </tr> <tr> <td>BestKeeper</td> <td> <i>TUB250</i> </td> <td> <i>AKB</i> </td> <td> <i>SuTub</i> </td> <td> <i>ARP9</i> </td> <td> <i>UFD2</i> </td> <td> <i>SPT</i> </td> <td> <i>POB1</i> </td> <td> <i>E3</i>. <i>528</i> </td> <td> <i>EF1</i>. <i>278</i> </td> </tr> <tr> <td>Comprehensive ranking</td> <td> <i>POB1</i> </td> <td> <i>UFD2</i> </td> <td> <i>TUB250</i> </td> <td> <i>UK552</i> </td> <td> <i>AKB</i> </td> <td> <i>SuTub</i> </td> <td> <i>HMT</i> </td> <td> <i>E2</i>. <i>299</i> </td> <td> <i>GAPDH</i> </td> </tr> <tr> <td> <b>Ranking order for the drought experimental set (Better—Good—Average)</b> </td> </tr> <tr> <td>geNorm</td> <td> <i>tIF</i> </td> <td> <i>AKB</i> </td> <td> <i>POB1</i> </td> <td> <i>E2</i>. <i>299</i> </td> <td> <i>HMT</i> </td> <td> <i>ACT5</i> </td> <td> <i>GAPDH</i> </td> <td> <i>UK552</i> </td> <td> <i>SPT</i> </td> </tr> <tr> <td>NormFinder</td> <td> <i>POB1</i> </td> <td> <i>tIF</i> </td> <td> <i>E2</i>. <i>299</i> </td> <td> <i>AKB</i> </td> <td> <i>GAPDH</i> </td> <td> <i>UK552</i> </td> <td> <i>SPT</i> </td> <td> <i>ACT5</i> </td> <td> <i>HMT</i> </td> </tr> <tr> <td>BestKeeper</td> <td> <i>POB1</i> </td> <td> <i>ARP9</i> </td> <td> <i>AKB</i> </td> <td> <i>SuTub</i> </td> <td> <i>GAPDH</i> </td> <td> <i>ACT5</i> </td> <td> <i>E2</i>. <i>299</i> </td> <td> <i>60S897</i> </td> <td> <i>HMT</i> </td> </tr> <tr> <td>Comprehensive ranking</td> <td> <i>POB1</i> </td> <td> <i>AKB</i> </td> <td> <i>tIF</i> </td> <td> <i>E2</i>. <i>299</i> </td> <td> <i>GAPDH</i> </td> <td> <i>ARP9</i> </td> <td> <i>ACT5</i> </td> <td> <i>HMT</i> </td> <td> <i>UK552</i> </td> </tr> <tr> <td> <b>Ranking order for the WGF experimental set (Better—Good—Average)</b> </td> </tr> <tr> <td>geNorm</td> <td> <i>E2</i>. <i>299</i> </td> <td> <i>AKB</i> </td> <td> <i>UFD2</i> </td> <td> <i>GAPDH</i> </td> <td> <i>HIS</i> </td> <td> <i>TUB250</i> </td> <td> <i>tIF</i> </td> <td> <i>SPT</i> </td> <td> <i>POB1</i> </td> </tr> <tr> <td>NormFinder</td> <td> <i>POB1</i> </td> <td> <i>UFD2</i> </td> <td> <i>E2</i>. <i>299</i> </td> <td> <i>AKB</i> </td> <td> <i>HIS</i> </td> <td> <i>ARP9</i> </td> <td> <i>GAPDH</i> </td> <td> <i>HMT</i> </td> <td> <i>tIF</i> </td> </tr> <tr> <td>BestKeeper</td> <td> <i>UFD2</i> </td> <td> <i>SPT</i> </td> <td> <i>GAPDH</i> </td> <td> <i>60S897</i> </td> <td> <i>HMT</i> </td> <td> <i>POB1</i> </td> <td> <i>E3</i>. <i>528</i> </td> <td> <i>EF1</i>. <i>278</i> </td> <td> <i>AKB</i> </td> </tr> <tr> <td>Comprehensive ranking</td> <td> <i>UFD2</i> </td> <td> <i>AKB</i> </td> <td> <i>E2</i>. <i>299</i> </td> <td> <i>POB1</i> </td> <td> <i>GAPDH</i> </td> <td> <i>SPT</i> </td> <td> <i>HIS</i> </td> <td> <i>HMT</i> </td> <td> <i>TUB250</i> </td> </tr> <tr> <td> <b>Ranking order for the total experimental set (Better—Good—Average)</b> </td> </tr> <tr> <td>geNorm</td> <td> <i>POB1</i> </td> <td> <i>AKB</i> </td> <td> <i>UFD2</i> </td> <td> <i>E2</i>. <i>299</i> </td> <td> <i>ARP9</i> </td> <td> <i>TUB250</i> </td> <td> <i>60S897</i> </td> <td> <i>tIF</i> </td> <td> <i>HMT</i> </td> </tr> <tr> <td>NormFinder</td> <td> <i>E2</i>. <i>299</i> </td> <td> <i>UFD2</i> </td> <td> <i>POB1</i> </td> <td> <i>AKB</i> </td> <td> <i>tIF</i> </td> <td> <i>TUB250</i> </td> <td> <i>ARP9</i> </td> <td> <i>HMT</i> </td> <td> <i>60S897</i> </td> </tr> <tr> <td>BestKeeper</td> <td> <i>HMT</i> </td> <td> <i>AKB</i> </td> <td> <i>POB1</i> </td> <td> <i>ARP9</i> </td> <td> <i>E3</i>. <i>528</i> </td> <td> <i>E2</i>. <i>299</i> </td> <td> <i>UFD2</i> </td> <td> <i>UK552</i> </td> <td> <i>ACT5</i> </td> </tr> <tr> <td>Comprehensive ranking</td> <td> <i>POB1</i> </td> <td> <i>AKB</i> </td> <td> <i>E2</i>. <i>299</i> </td> <td> <i>UFD2</i> </td> <td> <i>HMT</i> </td> <td> <i>ARP9</i> </td> <td> <i>TUB250</i> </td> <td> <i>E3</i>. <i>528</i> </td> <td> <i>tIF</i> </td> </tr> <tr> <td> <b>Method</b> </td> <td /> <td> <b>10</b> </td> <td> <b>11</b> </td> <td> <b>12</b> </td> <td> <b>13</b> </td> <td> <b>14</b> </td> <td> <b>15</b> </td> <td> <b>16</b> </td> <td> <b>17</b> </td> </tr> <tr> <td> <b>Ranking order for the cold experimental set (Better—Good—Average)</b> </td> </tr> <tr> <td>geNorm</td> <td /> <td> <i>AKB</i> </td> <td> <i>60S897</i> </td> <td> <i>SPT</i> </td> <td> <i>tIF</i> </td> <td> <i>ARP9</i> </td> <td> <i>E3</i>. <i>528</i> </td> <td> <i>HIS</i> </td> <td> <i>EF1</i>. <i>278</i> </td> </tr> <tr> <td>NormFinder</td> <td /> <td> <i>ACT5</i> </td> <td> <i>GAPDH</i> </td> <td> <i>SPT</i> </td> <td> <i>tIF</i> </td> <td> <i>ARP9</i> </td> <td> <i>E3</i>. <i>528</i> </td> <td> <i>HIS</i> </td> <td> <i>EF1</i>. <i>278</i> </td> </tr> <tr> <td>BestKeeper</td> <td /> <td> <i>HMT</i> </td> <td> <i>GAPDH</i> </td> <td> <i>E2</i>. <i>299</i> </td> <td> <i>60S897</i> </td> <td> <i>ACT5</i> </td> <td> <i>UK552</i> </td> <td> <i>HIS</i> </td> <td> <i>tIF</i> </td> </tr> <tr> <td>Comprehensive ranking</td> <td /> <td> <i>ARP9</i> </td> <td> <i>ACT5</i> </td> <td> <i>60S897</i> </td> <td> <i>SPT</i> </td> <td> <i>E3</i>. <i>528</i> </td> <td> <i>EF1</i>. <i>278</i> </td> <td> <i>tIF</i> </td> <td> <i>HIS</i> </td> </tr> <tr> <td> <b>Ranking order for the drought experimental set (Better—Good—Average)</b> </td> </tr> <tr> <td>geNorm</td> <td /> <td> <i>E3</i>. <i>528</i> </td> <td> <i>HIS</i> </td> <td> <i>ARP9</i> </td> <td> <i>EF1</i>. <i>278</i> </td> <td> <i>UFD2</i> </td> <td> <i>60S897</i> </td> <td> <i>SuTub</i> </td> <td> <i>TUB250</i> </td> </tr> <tr> <td>NormFinder</td> <td /> <td> <i>HIS</i> </td> <td> <i>E3</i>. <i>528</i> </td> <td> <i>ARP9</i> </td> <td> <i>EF1</i>. <i>278</i> </td> <td> <i>UFD2</i> </td> <td> <i>SuTub</i> </td> <td> <i>60S897</i> </td> <td> <i>TUB250</i> </td> </tr> <tr> <td>BestKeeper</td> <td /> <td> <i>UFD2</i> </td> <td> <i>EF1</i>. <i>278</i> </td> <td> <i>UK552</i> </td> <td> <i>TUB250</i> </td> <td> <i>HIS</i> </td> <td> <i>SPT</i> </td> <td> <i>E3</i>. <i>528</i> </td> <td> <i>tIF</i> </td> </tr> <tr> <td>Comprehensive ranking</td> <td /> <td> <i>SPT</i> </td> <td> <i>SuTub</i> </td> <td> <i>HIS</i> </td> <td> <i>E3</i>. <i>528</i> </td> <td> <i>EF1</i>. <i>278</i> </td> <td> <i>60S897</i> </td> <td> <i>UFD2</i> </td> <td> <i>TUB250</i> </td> </tr> <tr> <td> <b>Ranking order for the WGF experimental set (Better—Good—Average)</b> </td> </tr> <tr> <td>geNorm</td> <td /> <td> <i>ARP9</i> </td> <td> <i>HMT</i> </td> <td> <i>E3</i>. <i>528</i> </td> <td> <i>60S897</i> </td> <td> <i>UK552</i> </td> <td> <i>EF1</i>. <i>278</i> </td> <td> <i>ACT5</i> </td> <td> <i>SuTub</i> </td> </tr> <tr> <td>NormFinder</td> <td /> <td> <i>E3</i>. <i>528</i> </td> <td> <i>TUB250</i> </td> <td> <i>SPT</i> </td> <td> <i>UK552</i> </td> <td> <i>60S897</i> </td> <td> <i>EF1</i>. <i>278</i> </td> <td> <i>ACT5</i> </td> <td> <i>SuTub</i> </td> </tr> <tr> <td>BestKeeper</td> <td /> <td> <i>TUB250</i> </td> <td> <i>HIS</i> </td> <td> <i>ARP9</i> </td> <td> <i>SuTub</i> </td> <td> <i>E2</i>. <i>299</i> </td> <td> <i>UK552</i> </td> <td> <i>tIF</i> </td> <td> <i>ACT5</i> </td> </tr> <tr> <td>Comprehensive ranking</td> <td /> <td> <i>ARP9</i> </td> <td> <i>60S897</i> </td> <td> <i>E3</i>. <i>528</i> </td> <td> <i>tIF</i> </td> <td> <i>EF1</i>. <i>278</i> </td> <td> <i>UK552</i> </td> <td> <i>SuTub</i> </td> <td> <i>ACT5</i> </td> </tr> <tr> <td> <b>Ranking order for the total experimental set (Better—Good—Average)</b> </td> </tr> <tr> <td>geNorm</td> <td /> <td> <i>SPT</i> </td> <td> <i>E3</i>. <i>528</i> </td> <td> <i>ACT5</i> </td> <td> <i>EF1</i>. <i>278</i> </td> <td> <i>SuTub</i> </td> <td> <i>UK552</i> </td> <td> <i>HIS</i> </td> <td> <i>GAPDH</i> </td> </tr> <tr> <td>NormFinder</td> <td /> <td> <i>E3</i>. <i>528</i> </td> <td> <i>ACT5</i> </td> <td> <i>EF1</i>. <i>278</i> </td> <td> <i>SPT</i> </td> <td> <i>SuTub</i> </td> <td> <i>HIS</i> </td> <td> <i>UK552</i> </td> <td> <i>GAPDH</i> </td> </tr> <tr> <td>BestKeeper</td> <td /> <td> <i>TUB250</i> </td> <td> <i>SPT</i> </td> <td> <i>SuTub</i> </td> <td> <i>60S897</i> </td> <td> <i>GAPDH</i> </td> <td> <i>EF1</i>. <i>278</i> </td> <td> <i>HIS</i> </td> <td> <i>tIF</i> </td> </tr> <tr> <td>Comprehensive ranking</td> <td /> <td> <i>60S897</i> </td> <td> <i>ACT5</i> </td> <td> <i>SPT</i> </td> <td> <i>UK552</i> </td> <td> <i>EF1</i>. <i>278</i> </td> <td> <i>SuTub</i> </td> <td> <i>HIS</i> </td> <td> <i>GAPDH</i> </td> </tr> </tbody> </table> </div> </div> <p>To better clarify the pattern of comprehensive rankings of candidates depending on the gene group, we calculated averages of geomean values from three different algorithms for each gene group. As shown in <a href="#pone.0199356.g005">Fig 5</a>, although the slight difference between the values of the gene group 1 and the gene group 3, the gene group 2 showed constantly lowest values by all three algorithms, irrespective of the program used. This result indicates that genes in the group 2, which were novel genes with the low CV of transcriptome, were ranked higher and were more stable compared to the other housekeeping genes with low CV (group 1) or high CV (group 3). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0199356.g005"> <span class="label" tagx="label" title="label">Fig 5</span> </a> </div>   <div class="title" tagx="title" title="title">
Distribution of average geomean rankings for candidate reference genes depending of the gene group. </div> <p>Values were obtained by calculating an average comprehensive ranking for the genes in the gene group 1, 2, and 3, respectively. Four datasets from different experimental sets were used, and error bars represent standard deviation. </p>   </div> </div> <div class="validationoftherecommendedreferencegenes" title="sec"> <div class="title" tagx="title" title="title">
Validation of the recommended reference genes</div> <p>To validate the best-ranked candidate reference genes, the selected genes were used to normalize the expression patterns of the genes <i>LEA</i> and <i>HSP70</i>. Since <i>LEA</i> and <i>HSP70</i> are known to be upregulated by various environmental [ <a href="#pone.0199356.ref035">35</a>– <a href="#pone.0199356.ref037">37</a>], they are good markers of abiotic stress responses in plants. We identified a homologous sequence for each gene in the <i>S</i>. <i>uncinata</i> transcriptome dataset of this study. The <i>LEA</i> gene identified from <i>S</i>. <i>uncinata</i> showed 47% sequence identity with <i>Triticum aestivum LEA3</i> (AAN74639) at both the nucleotide and amino acid levels. In addition, the transcript levels of the <i>S</i>. <i>uncinata LEA</i> homolog exhibited large, positive fold changes under cold conditions: an approximately 20-, 25-, and 60-fold increase after 1, 3, and 8 h, respectively. The <i>HSP70</i> gene from <i>S</i>. <i>uncinata</i> showed high sequence identity at both the nucleotide and amino acid levels (81.6% and 92%, respectively) with the <i>Bryum argenteum HSP70</i> gene (KP087877). The transcript levels of the <i>S</i>. <i>uncinata HSP70</i> homolog exhibited positive fold changes under cold conditions: a 4.8-, 5.6-, and 1.9-fold increase after 1, 3, and 8 h, respectively. Thus, we concluded that the <i>LEA</i> and <i>HSP70</i> homologs from <i>S</i>. <i>uncinata</i> are <i>bona fide</i> stress-responsive genes suitable for reference gene evaluations ( <a href="#pone.0199356.g006">Fig 6A</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0199356.g006"> <span class="label" tagx="label" title="label">Fig 6</span> </a> </div>   <div class="title" tagx="title" title="title">
Relative expression levels of the late embryogenesis abundant ( <i>LEA</i>) and heat shock protein 70 ( <i>HSP70</i>) genes under different stress conditions when the selected reference genes were used for normalization. </div> <p>(A) <i>LEA</i> and <i>HSP70</i> expression in the cold experimental set of <i>S</i>. <i>uncinata</i> was normalized using a combination of the two most stable genes <i>POB1</i> and <i>UFD2</i>, and the least stable genes <i>HIS</i> and <i>tIF</i>. Red bar indicates fold-change values of gene expression, which were calculated based on the normalized FPKM from the RNA-Seq analysis of the cold experimental set ( <a href="#pone.0199356.s006">S2 Table</a>). (B) The expression of <i>LEA</i> and <i>HSP70</i> in the drought experimental set of <i>S</i>. <i>uncinata</i> was normalized using a combination of the two most stable genes <i>POB1</i> and <i>AKB</i>, and the least stable genes <i>UFD2</i> and <i>TUB250</i>. (C) The expression of <i>LEA</i> and <i>HSP70</i> in the WGF experimental set of <i>S</i>. <i>uncinata</i> was normalized using a combination of the two most stable genes <i>UFD2</i> and <i>AKB</i>, and the least stable genes <i>SuTub</i> and <i>ACT5</i>. Error bars represent the standard deviation of the mean of three biological replicates. </p>   </div> <p>For the quantification of target gene expression by RT-qPCR in the cold, drought, and WGF experimental sets, we used the top two most stable genes and the two least stable genes as references. Induction patterns similar to RNA-Seq under cold stress condition were observed when the Cq values of marker genes were normalized to the best candidate genes <i>POB1</i> or <i>UFD2</i> or both of <i>POB1/UFD2</i>, with only small variations in fold change values in each normalization ( <a href="#pone.0199356.g006">Fig 6A</a>). On the other hand, when data normalization was performed with the least stable genes, the values differed substantially. For example, the ranges of fold change for <i>LEA</i> was 20–60 and 60–120 when <i>POB1</i> and <i>HIS</i> were used as the reference gene, respectively. In the drought and WGF experimental sets, the normalized expression levels of the target genes also differed from the results obtained with the most stable genes ( <i>POB1</i> and <i>AKB</i> for drought; <i>UFD2</i> and <i>AKB</i> for WGF) and the most variable genes ( <i>UFD2</i> and <i>TUB250</i> for drought; <i>SuTub</i> and <i>ACT5</i> for WGF) ( <a href="#pone.0199356.g006">Fig 6B and 6C</a>). Overall, our validation suggested that the expression profiles of target genes are strongly affected by the choice of reference gene. </p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>The analysis of gene expression is a common strategy used to identify genes involved in a specific biological process. As data normalization is an essential step in gene expression analysis, the accuracy of RT-qPCR data can be greatly improved by the inclusion of a reference gene whose transcriptional levels are maintained across different experimental conditions [ <a href="#pone.0199356.ref038">38</a>]. Therefore, the selection of suitable reference genes for RT-qPCR is crucial for analyzing the expression of key genes correlated with various biological conditions. The present study assessed the stability of candidate reference genes for <i>S</i>. <i>uncinata</i> under different stress conditions. </p> <p>We selected 17 candidate reference genes for <i>S</i>. <i>uncinata</i> under several experimental conditions, especially for representative abiotic stresses such as cold and drought, from axenic culture to Antarctic field samples. The expression stability of the candidate reference genes was evaluated using geNorm, NormFinder, and BestKeeper. The results of our conjoint analysis could provide a reliable basis for the selection of reference genes for RT-qPCR analysis with various treatments. A set of reference genes varied slightly in ranking, although some genes were consistently found in the upper ranks of the list of stably expressed genes ( <a href="#pone.0199356.t004">Table 4</a>). </p> <p>When we compared results from the three different analysis algorithms, the best reference genes differed depending on the experimental set ( <a href="#pone.0199356.t004">Table 4</a>). <i>POB1</i> and <i>UFD2</i> were the most stable reference genes in the cold experimental set, while <i>POB1</i> and <i>AKB</i> were the most stable in the drought experimental set. In the WGF experimental set, <i>UFD2</i> and <i>AKB</i> showed maximum stability. Overall, the three novel reference genes, <i>AKB</i>, <i>POB1</i>, and <i>UFD2</i>, showed the most robust stability in multiple experimental sets, while traditionally used reference genes, such as <i>ACT5</i>, <i>GAPDH</i>, <i>SuTub</i>, and <i>TUB250</i>, were the least stable ( <a href="#pone.0199356.t004">Table 4</a>). Also, when we compared the distribution of comprehensive rankings of the gene groups, the novel candidates genes in the group 2 were constantly more stable than the other two groups ( <a href="#pone.0199356.g005">Fig 5</a>), indicating that traditionally used reference genes are often not the best choice for the normalization of RT-qPCR data in mosses under conditions of abiotic stress. The usefulness of transcriptome data for identifying robust normalizers is consistent with previous reports on the validation of novel reference genes based on transcriptome data [ <a href="#pone.0199356.ref039">39</a>, <a href="#pone.0199356.ref040">40</a>]. However, it is still necessary to validate the gene reliability individually even after the candidate gene selection from transcriptome data. </p> <p>To validate the suitability of the identified reference genes, the expression patterns of the genes <i>LEA</i> and <i>HSP70</i> were investigated under different experimental conditions using different reference genes. The data once again showed that reference genes play an essential role in the normalization of RT-qPCR data, and the use of inappropriate reference genes may produce unreliable results. Significant differences were observed in the expression patterns of the target genes, <i>LEA</i> and <i>HSP70</i>, between the normalized values with the two most stable genes and the two least stable genes in most of the experimental conditions tested ( <a href="#pone.0199356.g006">Fig 6</a>). </p> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>In this study, we evaluated the expression stability of 17 candidate reference genes suitable for <i>S</i>. <i>uncinata</i> under a wide range of environmental conditions using three widely used applications, and the comprehensive stability ranking by the geometric mean. We found the best reference genes for the gene expression studies in abiotic stress responses of <i>S</i>. <i>uncinata</i>; <i>POB1</i> and <i>UFD2</i> for cold stress, <i>POB1</i> and <i>AKB</i> for drought stress, and <i>UFD2</i> and <i>AKB</i> for populations from different water gradients in the Antarctic field. For the preliminary study on the general abiotic stresses, <i>POB1</i> and <i>AKB</i> pair should be the first choice. In addition, we also confirmed that the novels genes with the low CV from the transcriptome data were more stable than the traditionally used reference genes. The stable reference genes for abiotic stresses identified in this study will facilitate future expression studies in mosses under various abiotic stresses. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 Fig</span>  <div class="title" tagx="title" title="title">
Agarose gel electrophoresis showing total RNA extracted from <i>S. uncinata</i> under various conditions. </div> <p>(A) cold stress, (B) drought stress, (C) water gradients in the field.</p> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0199356.s001.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S2 Fig</span>  <div class="title" tagx="title" title="title">
Verification of primer specificity for reverse transcription quantitative real-time polymerase chain reaction (RT-qPCR) amplification. </div> <p>(A) Agarose gel (2.0%) showing the amplified products of the 17 candidate reference genes at the expected sizes. (B) Melting curves of the 17 candidate reference genes. </p> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0199356.s002.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S3 Fig</span>  <div class="title" tagx="title" title="title">
Amplification efficiencies of primers used in this study.</div> <p>(PDF)</p>  <div class="media" title="media"> <a href="pone.0199356.s003.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 File</span>  <div class="title" tagx="title" title="title">
The contig sequences for candidate genes.</div> <p>The regions for RT-qPCR amplification were highlighted in red color.</p> <p>(DOCX)</p>  <div class="media" title="media"> <a href="pone.0199356.s004.docx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S1 Table</span>  <div class="title" tagx="title" title="title">
Expression values of the 17 candidate genes of <i>S. uncinata</i> from the transcriptome under cold stress. </div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0199356.s005.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">S2 Table</span>  <div class="title" tagx="title" title="title">
Expression values of <i>LEA</i> and <i>Hsp70</i> in <i>S. uncinata</i> from the transcriptome under cold stress. </div> <p>(XLSX)</p>  <div class="media" title="media"> <a href="pone.0199356.s006.xlsx">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="pone.0199356.ref001" /> <span class="label" tagx="label" title="label">1</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bramley-Alves</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">King</span> <span class="given-names" tagx="given-names" title="given-names">DH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robinson</span> <span class="given-names" tagx="given-names" title="given-names">SA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miller</span> <span class="given-names" tagx="given-names" title="given-names">RE</span> </span>. <span class="chapter-title" tagx="chapter-title" title="chapter-title">Dominating the Antarctic environment: bryophytes in a time of change</span> In: <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hanson</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rice</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, editors. <span class="source" tagx="source" title="source">Photosynthesis in bryophytes and early land plants</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Dordrecht</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>; <span class="year" tagx="year" title="year">2014</span> pp. <span class="fpage" tagx="fpage" title="fpage">309</span>– <span class="lpage" tagx="lpage" title="lpage">324</span>. </span> </li> <li tag="ref"> <a name="pone.0199356.ref002" /> <span class="label" tagx="label" title="label">2</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lenne</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bryant</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hocart</span> <span class="given-names" tagx="given-names" title="given-names">CH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">CX</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ball</span> <span class="given-names" tagx="given-names" title="given-names">MC</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Freeze avoidance: a dehydrating moss gathers no ice</span>. <span class="source" tagx="source" title="source">Plant Cell Environ</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">33</span>: <span class="fpage" tagx="fpage" title="fpage">1731</span>– <span class="lpage" tagx="lpage" title="lpage">1741</span>. <span class="comment">doi: <a href="https://doi.org/10.1111/j.1365-3040.2010.02178.x">10.1111/j.1365-3040.2010.02178.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20525002">20525002</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref003" /> <span class="label" tagx="label" title="label">3</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Melick</span> <span class="given-names" tagx="given-names" title="given-names">DR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seppelt</span> <span class="given-names" tagx="given-names" title="given-names">RD</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Seasonal investigations of soluble carbohydrates and pigment levels in Antarctic bryophytes and lichens </span>. <span class="source" tagx="source" title="source">Bryologist</span>. <span class="year" tagx="year" title="year">1994</span>; <span class="volume" tagx="volume" title="volume">97</span>: <span class="fpage" tagx="fpage" title="fpage">13</span>– <span class="lpage" tagx="lpage" title="lpage">19</span>. </span> </li> <li tag="ref"> <a name="pone.0199356.ref004" /> <span class="label" tagx="label" title="label">4</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">RIL</span> </span>. <span class="chapter-title" tagx="chapter-title" title="chapter-title">Terrestrial and freshwater biotic components of the western Antarctic Peninsula</span> In: <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ross</span> <span class="given-names" tagx="given-names" title="given-names">RM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hofmann</span> <span class="given-names" tagx="given-names" title="given-names">EE</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quetin</span> <span class="given-names" tagx="given-names" title="given-names">LB</span> </span>, editors. <span class="source" tagx="source" title="source">Foundations for ecological research west of the Antarctic Peninsula</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Washington DC</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">American Geophysical Union</span>; <span class="year" tagx="year" title="year">1996</span> <span class="comment">doi: <a href="https://doi.org/10.1029/AR070p0015">10.1029/AR070p0015</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref005" /> <span class="label" tagx="label" title="label">5</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Virtanen</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lundberg</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moen</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oksanen</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Topographic and altitudinal patterns in plant communities on European arctic islands</span>. <span class="source" tagx="source" title="source">Polar Biol</span>. <span class="year" tagx="year" title="year">1997</span>; <span class="volume" tagx="volume" title="volume">17</span>: <span class="fpage" tagx="fpage" title="fpage">95</span>– <span class="lpage" tagx="lpage" title="lpage">113</span>. </span> </li> <li tag="ref"> <a name="pone.0199356.ref006" /> <span class="label" tagx="label" title="label">6</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakatsubo</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Predicting the impact of climatic warming on the carbon balance of the moss <i>Sanionia uncinata</i> on a maritime Antarctic island </span>. <span class="source" tagx="source" title="source">J Plant Res</span>. <span class="year" tagx="year" title="year">2002</span>; <span class="volume" tagx="volume" title="volume">115</span>: <span class="fpage" tagx="fpage" title="fpage">99</span>– <span class="lpage" tagx="lpage" title="lpage">106</span>. <span class="comment">doi: <a href="https://doi.org/10.1007/s102650200014">10.1007/s102650200014</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12884132">12884132</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref007" /> <span class="label" tagx="label" title="label">7</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hokkanen</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Environmental patterns and gradients in the vascular plants and bryophytes of eastern Fennoscandian herb-rich forests </span>. <span class="source" tagx="source" title="source">For Ecol Manage</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">229</span>: <span class="fpage" tagx="fpage" title="fpage">73</span>– <span class="lpage" tagx="lpage" title="lpage">87</span>. </span> </li> <li tag="ref"> <a name="pone.0199356.ref008" /> <span class="label" tagx="label" title="label">8</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kushnevskaya</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mirin</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shorohova</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Patterns of epixylic vegetation on spruce logs in late-successional boreal forests</span>. <span class="source" tagx="source" title="source">For Ecol Manage</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">250</span>: <span class="fpage" tagx="fpage" title="fpage">25</span>– <span class="lpage" tagx="lpage" title="lpage">33</span>. </span> </li> <li tag="ref"> <a name="pone.0199356.ref009" /> <span class="label" tagx="label" title="label">9</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zúñiga-González</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zúñiga</span> <span class="given-names" tagx="given-names" title="given-names">GE</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pizarro</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Casanova-Katny</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Soluble carbohydrate content variation in <i>Sanionia uncinata</i> and <i>Polytrichastrum alpinum</i>, two Antarctic mosses with contrasting desiccation capacities </span>. <span class="source" tagx="source" title="source">Biol Res</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">49</span>: <span class="fpage" tagx="fpage" title="fpage">6</span> <span class="comment">doi: <a href="https://doi.org/10.1186/s40659-015-0058-z">10.1186/s40659-015-0058-z</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26823072">26823072</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref010" /> <span class="label" tagx="label" title="label">10</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bustin</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Benes</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nolan</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pfaffl</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Quantitative real-time RT-PCR–a perspective</span>. <span class="source" tagx="source" title="source">J Mol Endocrinol</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">34</span>: <span class="fpage" tagx="fpage" title="fpage">597</span>– <span class="lpage" tagx="lpage" title="lpage">601</span>. <span class="comment">doi: <a href="https://doi.org/10.1677/jme.1.01755">10.1677/jme.1.01755</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15956331">15956331</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref011" /> <span class="label" tagx="label" title="label">11</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huggett</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dheda</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bustin</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zumla</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Real-time RT-PCR normalisation; strategies and considerations</span>. <span class="source" tagx="source" title="source">Genes Immun</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">279</span>– <span class="lpage" tagx="lpage" title="lpage">284</span>. <span class="comment">doi: <a href="https://doi.org/10.1038/sj.gene.6364190">10.1038/sj.gene.6364190</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15815687">15815687</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref012" /> <span class="label" tagx="label" title="label">12</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vandesompele</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De Preter</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pattyn</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Poppe</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Roy</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De Paepe</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Accurate normalization of real-time quantitative RT-PCR data by geometric averaging of multiple internal control genes </span>. <span class="source" tagx="source" title="source">Genome Biol</span>. <span class="year" tagx="year" title="year">2002</span>; 3: research0034.1. </span> </li> <li tag="ref"> <a name="pone.0199356.ref013" /> <span class="label" tagx="label" title="label">13</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gutierrez</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mauriat</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guénin</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pelloux</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lefebvre</span> <span class="given-names" tagx="given-names" title="given-names">JF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Louvet</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">The lack of a systematic validation of reference genes: a serious pitfall undervalued in reverse transcription‐polymerase chain reaction (RT‐PCR) analysis in plants </span>. <span class="source" tagx="source" title="source">Plant Biotechnol</span>. <span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">609</span>– <span class="lpage" tagx="lpage" title="lpage">618</span>. </span> </li> <li tag="ref"> <a name="pone.0199356.ref014" /> <span class="label" tagx="label" title="label">14</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boava</span> <span class="given-names" tagx="given-names" title="given-names">LP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laia</span> <span class="given-names" tagx="given-names" title="given-names">ML</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacob</span> <span class="given-names" tagx="given-names" title="given-names">TR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dabbas</span> <span class="given-names" tagx="given-names" title="given-names">KM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gonçalves</span> <span class="given-names" tagx="given-names" title="given-names">JF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ferro</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Selection of endogenous genes for gene expression studies in <i>Eucalyptus</i> under biotic ( <i>Puccinia psidii</i>) and abiotic (acibenzolar-S-methyl) stresses using RT-qPCR </span>. <span class="source" tagx="source" title="source">BMC Res Notes</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">43</span> <span class="comment">doi: <a href="https://doi.org/10.1186/1756-0500-3-43">10.1186/1756-0500-3-43</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20181283">20181283</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref015" /> <span class="label" tagx="label" title="label">15</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">XS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">HL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">DY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">YM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wood</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Reference gene selection in the desert plant <i>Eremosparton songoricum</i> </span>. <span class="source" tagx="source" title="source">Int J Mol Sci</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">6944</span>– <span class="lpage" tagx="lpage" title="lpage">6963</span>. <span class="comment">doi: <a href="https://doi.org/10.3390/ijms13066944">10.3390/ijms13066944</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22837673">22837673</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref016" /> <span class="label" tagx="label" title="label">16</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hong</span> <span class="given-names" tagx="given-names" title="given-names">SM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bahn</span> <span class="given-names" tagx="given-names" title="given-names">SC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lyu</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jung</span> <span class="given-names" tagx="given-names" title="given-names">HS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ahn</span> <span class="given-names" tagx="given-names" title="given-names">JH</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Identification and testing of superior reference genes for a starting pool of transcript normalization in Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant Cell Physiol</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">51</span>: <span class="fpage" tagx="fpage" title="fpage">1694</span>– <span class="lpage" tagx="lpage" title="lpage">1706</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/pcp/pcq128">10.1093/pcp/pcq128</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20798276">20798276</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref017" /> <span class="label" tagx="label" title="label">17</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacob</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guertler</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Naim</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nixdorf</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fedier</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hacker</span> <span class="given-names" tagx="given-names" title="given-names">NF</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Careful selection of reference genes is required for reliable performance of RT-qPCR in human normal and cancer cell lines </span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">e59180</span> <span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0059180">10.1371/journal.pone.0059180</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23554992">23554992</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref018" /> <span class="label" tagx="label" title="label">18</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iskandar</span> <span class="given-names" tagx="given-names" title="given-names">HM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simpson</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Casu</span> <span class="given-names" tagx="given-names" title="given-names">RE</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bonnett</span> <span class="given-names" tagx="given-names" title="given-names">GD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maclean</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manners</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Comparison of reference genes for quantitative real-time polymerase chain reaction analysis of gene expression in sugarcane </span>. <span class="source" tagx="source" title="source">Plant Mol Bio Rep</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">325</span>– <span class="lpage" tagx="lpage" title="lpage">337</span>. </span> </li> <li tag="ref"> <a name="pone.0199356.ref019" /> <span class="label" tagx="label" title="label">19</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jain</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nijhawan</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tyagi</span> <span class="given-names" tagx="given-names" title="given-names">AK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khurana</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Validation of housekeeping genes as internal control for studying gene expression in rice by quantitative real-time PCR </span>. <span class="source" tagx="source" title="source">Biochem Biophys Res Commun</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">345</span>: <span class="fpage" tagx="fpage" title="fpage">646</span>– <span class="lpage" tagx="lpage" title="lpage">651</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.bbrc.2006.04.140">10.1016/j.bbrc.2006.04.140</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16690022">16690022</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref020" /> <span class="label" tagx="label" title="label">20</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Borowski</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Galli</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">da Silva Messias</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perin</span> <span class="given-names" tagx="given-names" title="given-names">EC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buss</span> <span class="given-names" tagx="given-names" title="given-names">JH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Silva</span> <span class="given-names" tagx="given-names" title="given-names">SDDAE</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Selection of candidate reference genes for real-time PCR studies in lettuce under abiotic stresses </span>. <span class="source" tagx="source" title="source">Planta</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">239</span>: <span class="fpage" tagx="fpage" title="fpage">1187</span>– <span class="lpage" tagx="lpage" title="lpage">1200</span>. <span class="comment">doi: <a href="https://doi.org/10.1007/s00425-014-2041-2">10.1007/s00425-014-2041-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24573225">24573225</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref021" /> <span class="label" tagx="label" title="label">21</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Czechowski</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stitt</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Altmann</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Udvardi</span> <span class="given-names" tagx="given-names" title="given-names">MK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scheible W-</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Genome-wide identification and testing of superior reference genes for transcript normalization in Arabidopsis </span>. <span class="source" tagx="source" title="source">Plant Physiol</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">139</span>: <span class="fpage" tagx="fpage" title="fpage">5</span>– <span class="lpage" tagx="lpage" title="lpage">17</span>. <span class="comment">doi: <a href="https://doi.org/10.1104/pp.105.063743">10.1104/pp.105.063743</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16166256">16166256</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref022" /> <span class="label" tagx="label" title="label">22</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manoli</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sturaro</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trevisan</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quaggiotti</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nonis</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Evaluation of candidate reference genes for qPCR in maize</span>. <span class="source" tagx="source" title="source">J Plant Physiol</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">169</span>: <span class="fpage" tagx="fpage" title="fpage">807</span>– <span class="lpage" tagx="lpage" title="lpage">815</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.jplph.2012.01.019">10.1016/j.jplph.2012.01.019</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22459324">22459324</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref023" /> <span class="label" tagx="label" title="label">23</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moura</span> <span class="given-names" tagx="given-names" title="given-names">JCMS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Araújo</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">dos Santos Brito</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Souza</span> <span class="given-names" tagx="given-names" title="given-names">UR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Viana</span> <span class="given-names" tagx="given-names" title="given-names">JOF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mazzafera</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Validation of reference genes from <i>Eucalyptus spp</i>. under different stress conditions </span>. <span class="source" tagx="source" title="source">BMC Res Notes</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">634</span> <span class="comment">doi: <a href="https://doi.org/10.1186/1756-0500-5-634">10.1186/1756-0500-5-634</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23148685">23148685</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref024" /> <span class="label" tagx="label" title="label">24</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Løvdal</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lillo</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Reference gene selection for quantitative real-time PCR normalization in tomato subjected to nitrogen, cold, and light stress </span>. <span class="source" tagx="source" title="source">Anal Biochem</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">387</span>: <span class="fpage" tagx="fpage" title="fpage">238</span>– <span class="lpage" tagx="lpage" title="lpage">242</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.ab.2009.01.024">10.1016/j.ab.2009.01.024</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19454243">19454243</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref025" /> <span class="label" tagx="label" title="label">25</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Le Bail</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scholz</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kost</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Evaluation of reference genes for RT qPCR analyses of structure-specific and hormone regulated gene expression in <i>Physcomitrella patens</i> gametophytes </span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">e70998</span> <span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0070998">10.1371/journal.pone.0070998</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23951063">23951063</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref026" /> <span class="label" tagx="label" title="label">26</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Characterization of reference genes for RT-qPCR in the desert moss <i>Syntrichia caninervis</i> in response to abiotic stress and desiccation/rehydration </span>. <span class="source" tagx="source" title="source">Front Plant Sci</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">38</span> <span class="comment">doi: <a href="https://doi.org/10.3389/fpls.2015.00038">10.3389/fpls.2015.00038</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25699066">25699066</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref027" /> <span class="label" tagx="label" title="label">27</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saint-Marcoux</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Proust</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dolan</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Langdale</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Identification of reference genes for real-time quantitative PCR experiments in the liverwort <i>Marchantia polymorpha</i> </span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">e0118678</span> <span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0118678">10.1371/journal.pone.0118678</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25798897">25798897</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref028" /> <span class="label" tagx="label" title="label">28</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Andersen</span> <span class="given-names" tagx="given-names" title="given-names">CL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jensen</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ørntoft</span> <span class="given-names" tagx="given-names" title="given-names">TF</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Normalization of real-time quantitative reverse transcription-PCR data: a model-based variance estimation approach to identify genes suited for normalization, applied to bladder and colon cancer data sets </span>. <span class="source" tagx="source" title="source">Cancer Res</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">64</span>: <span class="fpage" tagx="fpage" title="fpage">5245</span>– <span class="lpage" tagx="lpage" title="lpage">5250</span>. <span class="comment">doi: <a href="https://doi.org/10.1158/0008-5472.CAN-04-0496">10.1158/0008-5472.CAN-04-0496</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15289330">15289330</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref029" /> <span class="label" tagx="label" title="label">29</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pfaffl</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tichopad</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prgomet</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neuvians</span> <span class="given-names" tagx="given-names" title="given-names">TP</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Determination of stable housekeeping genes, differentially regulated target genes and sample integrity: BestKeeper–Excel-based tool using pair-wise correlations </span>. <span class="source" tagx="source" title="source">Biotechnol Lett</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">26</span>: <span class="fpage" tagx="fpage" title="fpage">509</span>– <span class="lpage" tagx="lpage" title="lpage">515</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15127793">15127793</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref030" /> <span class="label" tagx="label" title="label">30</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trapnell</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williams</span> <span class="given-names" tagx="given-names" title="given-names">BA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pertea</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mortazavi</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kwan</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Baren</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation </span>. <span class="source" tagx="source" title="source">Nat Biotechnol</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">28</span>: <span class="fpage" tagx="fpage" title="fpage">511</span>– <span class="lpage" tagx="lpage" title="lpage">515</span>. <span class="comment">doi: <a href="https://doi.org/10.1038/nbt.1621">10.1038/nbt.1621</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20436464">20436464</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref031" /> <span class="label" tagx="label" title="label">31</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">González-Agüero</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">García-Rojas</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Di Genova</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Correa</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maass</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Orellana</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Identification of two putative reference genes from grapevine suitable for gene expression analysis in berry and related tissues derived from RNA-Seq data </span>. <span class="source" tagx="source" title="source">BMC Genomics</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">878</span> <span class="comment">doi: <a href="https://doi.org/10.1186/1471-2164-14-878">10.1186/1471-2164-14-878</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24330674">24330674</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref032" /> <span class="label" tagx="label" title="label">32</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bustin</span> <span class="given-names" tagx="given-names" title="given-names">SA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Benes</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garson</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hellemans</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huggett</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kubista</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">The MIQE guidelines: minimum information for publication of quantitative real-time PCR experiments </span>. <span class="source" tagx="source" title="source">Clin Chem</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">55</span>: <span class="fpage" tagx="fpage" title="fpage">611</span>– <span class="lpage" tagx="lpage" title="lpage">622</span>. <span class="comment">doi: <a href="https://doi.org/10.1373/clinchem.2008.112797">10.1373/clinchem.2008.112797</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19246619">19246619</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref033" /> <span class="label" tagx="label" title="label">33</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiao</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yao</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Validation of suitable reference genes for gene expression analysis in the halophyte <i>Salicornia europaea</i> by real-time quantitative PCR </span>. <span class="source" tagx="source" title="source">Front Plant Sci</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">788</span> <span class="comment">doi: <a href="https://doi.org/10.3389/fpls.2014.00788">10.3389/fpls.2014.00788</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25653658">25653658</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref034" /> <span class="label" tagx="label" title="label">34</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ueno</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bekku</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Uchida</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanda</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Photosynthetic light responses of a widespread moss, <i>Sanionia uncinata</i>, from contrasting water regimes in the high Arctic tundra, Svalbard, Norway </span>. <span class="source" tagx="source" title="source">J Bryol</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">28</span>: <span class="fpage" tagx="fpage" title="fpage">345</span>– <span class="lpage" tagx="lpage" title="lpage">349</span>. </span> </li> <li tag="ref"> <a name="pone.0199356.ref035" /> <span class="label" tagx="label" title="label">35</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoekstra</span> <span class="given-names" tagx="given-names" title="given-names">FA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Golovina</span> <span class="given-names" tagx="given-names" title="given-names">EA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buitink</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Mechanisms of plant desiccation tolerance</span>. <span class="source" tagx="source" title="source">Trends Plant Sci</span>. <span class="year" tagx="year" title="year">2001</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">431</span>– <span class="lpage" tagx="lpage" title="lpage">438</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11544133">11544133</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref036" /> <span class="label" tagx="label" title="label">36</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vinocur</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Altman</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Plant responses to drought, salinity and extreme temperatures: towards genetic engineering for stress tolerance </span>. <span class="source" tagx="source" title="source">Planta</span>. <span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">218</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">14</span>. <span class="comment">doi: <a href="https://doi.org/10.1007/s00425-003-1105-5">10.1007/s00425-003-1105-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14513379">14513379</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref037" /> <span class="label" tagx="label" title="label">37</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamaguchi-Shinozaki</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Molecular responses to dehydration and low temperature: differences and cross-talk between two stress signaling pathways </span>. <span class="source" tagx="source" title="source">Curr Opin Plant Biol</span>. <span class="year" tagx="year" title="year">2000</span>; <span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">217</span>– <span class="lpage" tagx="lpage" title="lpage">223</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10837265">10837265</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref038" /> <span class="label" tagx="label" title="label">38</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huggett</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dheda</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bustin</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zumla</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Real-time RT-PCR normalisation; strategies and considerations</span>. <span class="source" tagx="source" title="source">Genes Immun</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">279</span>– <span class="lpage" tagx="lpage" title="lpage">284</span>. <span class="comment">doi: <a href="https://doi.org/10.1038/sj.gene.6364190">10.1038/sj.gene.6364190</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15815687">15815687</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref039" /> <span class="label" tagx="label" title="label">39</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Selection and validation of appropriate reference genes for quantitative real-time PCR normalization in staminate and perfect flowers of andromonoecious <i>Taihangia rupestris</i> </span>. <span class="source" tagx="source" title="source">Front Plant Sci</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">729</span> <span class="comment">doi: <a href="https://doi.org/10.3389/fpls.2017.00729">10.3389/fpls.2017.00729</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28579993">28579993</a> </span> </span> </li> <li tag="ref"> <a name="pone.0199356.ref040" /> <span class="label" tagx="label" title="label">40</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ding</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Selection and evaluation of new reference genes for RT-qPCR analysis in <i>Epinephelus akaara</i> based on transcriptome data </span>. <span class="source" tagx="source" title="source">PLoS ONE</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">e0171646</span> <span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0171646">10.1371/journal.pone.0171646</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28182746">28182746</a> </span> </span> </li> </ul> </div> </div>  </body></html>