<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">PLoS One</span> <span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span> <span class="publisher-id" title="publisher-id">plos</span> <span class="pmc" title="pmc">plosone</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5919044">5919044</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/29694421">29694421</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0196473">10.1371/journal.pone.0196473</a> </span> <span class="publisher-id" title="publisher-id">PONE-D-17-42055</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Genomics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Genomes</span>: <span class="subject" title="subject">Chloroplast Genome</span> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biotechnology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Biotechnology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Genomics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Genomes</span>: <span class="subject" title="subject">Chloroplast Genome</span> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Science</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Biotechnology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Genomics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Genomes</span>: <span class="subject" title="subject">Chloroplast Genome</span> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Genomics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Genomes</span>: <span class="subject" title="subject">Chloroplast Genome</span> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Science</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Genomics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Genomes</span>: <span class="subject" title="subject">Chloroplast Genome</span> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Eukaryota</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plants</span>: <span class="subject" title="subject">Fruits</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Phylogenetics</span>: <span class="subject" title="subject">Phylogenetic Analysis</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Taxonomy</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Phylogenetics</span>: <span class="subject" title="subject">Phylogenetic Analysis</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computer and Information Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Data Management</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Taxonomy</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Phylogenetics</span>: <span class="subject" title="subject">Phylogenetic Analysis</span> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular biology techniques</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">DNA construction</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">DNA library construction</span>: <span class="subject" title="subject">Genomic Library Construction</span> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and analysis methods</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular biology techniques</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">DNA construction</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">DNA library construction</span>: <span class="subject" title="subject">Genomic Library Construction</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span>: <span class="subject" title="subject">Comparative Genomics</span> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span>: <span class="subject" title="subject">Comparative Genomics</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span>: <span class="subject" title="subject">Phylogenetics</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Taxonomy</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span>: <span class="subject" title="subject">Phylogenetics</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computer and Information Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Data Management</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Taxonomy</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span>: <span class="subject" title="subject">Phylogenetics</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Complexity</span>: <span class="subject" title="subject">Introns</span> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Complexity</span>: <span class="subject" title="subject">Introns</span> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Nucleic acids</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA</span> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Non-coding RNA</span>: <span class="subject" title="subject">Transfer RNA</span> </div> </div> </div> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
The chloroplast genome of <i>Cerasus humilis</i>: Genomic characterization and phylogenetic analysis </div> <div class="alt-title" title="alt-title">
The first chloroplast genome of <i>Cerasus humilis</i> </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mu</span> <span class="given-names" tagx="given-names" title="given-names">Xiaopeng</span> </span> <span class="role" tagx="role" title="role">Formal analysis</span> <span class="role" tagx="role" title="role">Investigation</span> <span class="role" tagx="role" title="role">Writing – original draft</span> <a href="#aff001"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Pengfei</span> </span> <span class="role" tagx="role" title="role">Data curation</span> <span class="role" tagx="role" title="role">Formal analysis</span> <span class="role" tagx="role" title="role">Validation</span> <a href="#aff001"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-9200-1099</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Du</span> <span class="given-names" tagx="given-names" title="given-names">Junjie</span> </span> <span class="role" tagx="role" title="role">Data curation</span> <span class="role" tagx="role" title="role">Funding acquisition</span> <span class="role" tagx="role" title="role">Methodology</span> <span class="role" tagx="role" title="role">Project administration</span> <span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span> <a href="#aff001"> <sup>1</sup> </a> <a href="#aff002"> <sup>2</sup> </a> <a href="#cor001">*</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">Yu Gary</span> </span> <span class="role" tagx="role" title="role">Validation</span> <span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span> <a href="#aff003"> <sup>3</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Jiancheng</span> </span> <span class="role" tagx="role" title="role">Data curation</span> <span class="role" tagx="role" title="role">Methodology</span> <span class="role" tagx="role" title="role">Resources</span> <a href="#aff001"> <sup>1</sup> </a> </span> </div> <span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">College of Horticulture, Shanxi Agricultural University, Jinzhong, Shanxi, P. R. China</span> </span> <span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Shanxi Key Laboratory of Germplasm Improvement and Utilization in Pomology, Jinzhong, Shanxi, P. R. China </span> </span> <span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">OSU South Centers, College of Food, Agricultural, and Environmental Sciences, The Ohio State University, Shyville Road, Piketon, Ohio, United States of America </span> </span> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">Shashi</span> </span> <span class="role" tagx="role" title="role">Editor</span> <a href="#edit1" /> </span> </div> <span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">International Centre for Genetic Engineering and Biotechnology, INDIA</span> </span> <div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p> <b>Competing Interests: </b>The authors have declared that no competing interests exist. </p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">djj738@163.com</span> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-4-4</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span> </span> <span class="volume" tagx="volume" title="volume">13</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e0196473</span> <span class="history" title="history"> <span class="received" title="received">received: 2017-11-29</span> <span class="accepted" title="accepted">accepted: 2018-4-13</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Mu et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited. </span> </span> </div> <a class="self-uri" href="pone.0196473.pdf" title="self-uri">pone.0196473.pdf</a> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p> <i>Cerasus humilis</i> is endemic to China and is a new fruit tree species with economic and environmental benefits, with potential developmental and utilization applications. We report the first complete chloroplast genome sequence of <i>C</i>. <i>humilis</i>. Its genome is 158,084 bp in size, and the overall GC content is 36.8%. An inverted repeats (IR) of 52,672 bp in size is separated by a large single-copy (LSC) region of 86,374 bp and a small single-copy (SSC) region of 19,038 bp. The chloroplast genome of <i>C</i>. <i>humilis</i> contains 131 genes including 90 protein-coding genes, 33 transfer RNA genes, and 8 ribosomal RNA genes. The genome has a total 510 simple sequence repeats (SSRs). Of these, 306, 149, and 55 were found in the LSC, IR, and SSC regions, respectively. In addition, a comparison of the boundaries of the LSC, SSC, and IR regions of ten other <i>Prunus</i> species exhibited an overall high degree of sequence similarity, with slight variations in the IR boundary region which included gene deletions, insertions, expansions, and contractions. <i>C</i>. <i>humilis</i> lost the <i>ycf1</i> gene at the IRA/SSC border and it has the largest <i>ycf1</i> gene at the IRB/SSC border among these <i>Prunus</i> species, whereas the <i>rps19</i> gene was inserted at the IRB/LSC junction. Furthermore, phylogenetic reconstruction using 61 conserved coding-protein genes clustered <i>C</i>. <i>humilis</i> with <i>Prunus tomentosa</i>. Thus, the complete chloroplast genome sequence of <i>C</i>. <i>humilis</i> provides a rich source of genetic information for studies on <i>Prunus</i> taxonomy, phylogeny, and evolution, as well as lays the foundation for further development and utilization of <i>C</i>. <i>humilis</i>. </p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="institution" title="institution">the Major Subject of Shanxi Science and Technology Research</span> </div> <span class="award-id" tagx="award-id" title="award-id">20121101010</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-9200-1099</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Du</span> <span class="given-names" tagx="given-names" title="given-names">Junjie</span> </span> </span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="institution" title="institution">the Doctoral Research Fund of Shanxi Agriculture University</span> </div> <span class="award-id" tagx="award-id" title="award-id">2015ZZ19</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Pengfei</span> </span> </span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="institution" title="institution">Project funded by China Postdoctoral Science Foundation</span> </div> <span class="award-id" tagx="award-id" title="award-id">2017M621106</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mu</span> <span class="given-names" tagx="given-names" title="given-names">Xiaopeng</span> </span> </span> </div> <div class="funding-statement" title="funding-statement">
This study was performed with the support of the Major Subject of Shanxi Science and Technology Research (Grant No. 20121101010), the Doctoral Research Fund of Shanxi Agriculture University (Grant No. 2015ZZ19) and China Postdoctoral Science Foundation (Grant No.2017M621106). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. </div> </div> <div class="counts" title="counts"> <span class="fig-count" title="fig-count">fig-count: </span> <span class="table-count" title="table-count">table-count: </span> <span class="page-count" title="page-count">page-count: </span> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="The complete C. humilis chloroplast genome was submitted to GenBank, as Accession Number MF405921." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>The complete C. humilis chloroplast genome was submitted to GenBank, as Accession Number MF405921. </p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p> <i>Cerasus humilis</i> (Bge.) Sok is a bush fruit tree that is endemic to China; its fruits are rich in calcium and are thus also known as "Calcium fruit" [ <a href="#pone.0196473.ref001">1</a>]. It is mainly distributed in the northeast, northwest, north, and other northern areas of China [ <a href="#pone.0196473.ref002">2</a>]. <i>C</i>. <i>humilis</i> has long existed in the wild, and studies on this species were only initiated in the 1990s [ <a href="#pone.0196473.ref003">3</a>]. After nearly 30 years of research studies, identification of varieties, and establishment of cultivation techniques, our understanding of this fruit tree species has improved and facilitated its large-scale artificial cultivation. It has a strong root system and it shows strong adaptability to saline soil, harsh winter and drought [ <a href="#pone.0196473.ref004">4</a>]. Furthermore, it has strong soil erosion resistance, therefore it can be used for the improvement of damaged soil and environmental greening [ <a href="#pone.0196473.ref005">5</a>]. <i>C</i>. <i>humilis</i> is also used as an ornamental plant, generating striking white or red flowers. Its fruits look like cherries but have various colors including red, yellow, white, green, purple, etc. Moreover, it is a high-yield crop and its fruit has a unique flavor. The fruits can be eaten fresh, preserved, or processed into wine or other by-products. Its stems can be used in weaving and as high-protein cattle feed. The seed kernels have been traditionally used in Chinese medicine as "Yuliren" [ <a href="#pone.0196473.ref005">5</a>, <a href="#pone.0196473.ref006">6</a>]. Based on its ecological and economic benefits, <i>C</i>. <i>humilis</i> has become an emerging multi-purpose fruit trees with a broad developmental and utilization potential. </p> <p>Similar to the mitochondria and nuclei, chloroplasts are semi-autonomous organelles that function in the transfer and expression of genetic material DNA [ <a href="#pone.0196473.ref007">7</a>, <a href="#pone.0196473.ref008">8</a>]. Chloroplasts are generally situated in the cytoplasmic matrix and play an important role in photosynthesis and fatty acid, starch, and amino acid synthesis [ <a href="#pone.0196473.ref009">9</a>]. In 1986, the complete chloroplast genome sequence of <i>Marchantia polymorpha</i> and tobacco ( <i>Nicotiana tabacum</i>) were released and to date, the chloroplast whole genome sequence of 700 plant species has been completed [ <a href="#pone.0196473.ref010">10</a>]. The chloroplast genome is a closed cyclic structure that consists of large single copy (LSC), small single copy (SSC), and two reverse repeats (IR); its genome size ranges from 100 kb to 200 kb. Protein-coding genes are important in photosynthesis and other functions of the chloroplast and are further divided into nine categories: Photosystem I, cytochrome b6-f complex, Photosystem II, ATP synthase, ribulose bis-phosphate carboxylase, ribosome large subunit, ribosome small subunit, RNA polymerase, and a hypothetical protein [ <a href="#pone.0196473.ref011">11</a>]. The overall structure and genome of the chloroplast are relatively conserved, although in-depth studies in chloroplast genomics and biology have identified several mutations and small structural changes among various species including insertions, deletions, reversals, and translocations [ <a href="#pone.0196473.ref012">12</a>, <a href="#pone.0196473.ref013">13</a>]. </p> <p>A famous Chinese fruit scientist earlier classified <i>C</i>. <i>humilis</i> as <i>Rosaceae cerasus</i> [ <a href="#pone.0196473.ref014">14</a>], whereas peach, apricot, plum, and cherry are classified as <i>Prunus</i>. Although <i>Prunus</i> consists of numerous species, only the chloroplast genome of ten species including <i>Prunus kansuensis</i>, <i>Prunus maximowiczii</i>, <i>Prunus pseudocerasus</i>, <i>Prunus yedoensis</i>, <i>Prunus persica</i>, <i>Prunus tomentosa</i>, <i>Prunus serotina</i>, <i>Prunus dulcis</i>, <i>Prunus cerasoides</i> and <i>Prunus padus</i> have been completely sequenced to date, with genome sizes ranging from 157,685 bp–158,955 bp. <i>C</i>. <i>humilis</i> may be further developed and potentially utilized in various industries. The complete chloroplast genome sequence of <i>C</i>. <i>humilis</i> could be applied in facilitating evolutionary analysis, accelerating breeding, identification of commercial cultivars and determination of varieties’ purity. In this study, the whole genome sequence of the chloroplast of <i>C</i>. <i>humilis</i> was obtained for the first time, and its structural characteristics and phylogenetic relationships were analyzed. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="plantmaterialanddnaextraction" title="sec"> <div class="title" tagx="title" title="title">
Plant material and DNA extraction</div> <p>Fresh and young leaves of the upper part of the branches from a cultivated variety of <i>C</i>. <i>humilis</i> (cv. Nongda 4, <a href="#pone.0196473.g001">Fig 1</a>) were collected from the germplasm resources of <i>C</i>. <i>humilis</i> of Shanxi Agricultural University (37°26'N, 112°32'E). Using 100 mg of a fresh plant sample, total genomic DNA was isolated using HP Plant DNA Kit D2485-01 (Omega Bio-Tek, Santa Clara, CA, USA), according to the manufacturer’s protocol. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0196473.g001"> <span class="label" tagx="label" title="label">Fig 1</span> </a> </div>   <div class="title" tagx="title" title="title">
Whole plant of <i>C</i>. <i>humilis</i> (cv. ‘Nongda 4’). </div> <p>(A) Young fruit stage. (B) Mature fruit stage.</p>   </div> </div> <div class="chloroplastgenomesequencing,assembly,andannotation" title="sec"> <div class="title" tagx="title" title="title">
Chloroplast genome sequencing, assembly, and annotation</div> <p>After quantification and qualification of total genomic DNA of the sample, a paired-end library of 2 × 150 bp and insert size of ~350 bp was constructed according to the Illumina standard library, which was then sequenced on an Illumina Hiseq 2500 platform. The resulting low-quality raw data were filtered, and the linker sequences were removed, thereby resulting in 20.86 Gb of clean data, with a quality value ≥ Q30, accounting for 91.37%. The genome of <i>P</i>. <i>pseudocerasus</i> was used as reference (GenBank Accession No. NC_030599.1), and NOVOPlasty software was used to assemble the chloroplast genome, the mistake parameter was set by default [ <a href="#pone.0196473.ref015">15</a>]. Three software were used for genome degenerate base correction, namely, bwa, samtools, and bcftools [ <a href="#pone.0196473.ref016">16</a>, <a href="#pone.0196473.ref017">17</a>]. The complete <i>C</i>. <i>humilis</i> chloroplast genome sequence was annotated using the online software CpGAVAS developed by Liu et al [ <a href="#pone.0196473.ref018">18</a>] ( <a href="http://www.herbalgenomics.org/0506/cpgavas/analyzer/annotate">http://www.herbalgenomics.org/0506/cpgavas/analyzer/annotate</a>). TRNAscan-SE was used in tRNA confirmation [ <a href="#pone.0196473.ref019">19</a>]; <a href="http://lowelab.ucsc.edu/tRNAscan-SE/">http://lowelab.ucsc.edu/tRNAscan-SE/</a>). Genomic circle graphs were drawn using OGDRAW [ <a href="#pone.0196473.ref020">20</a>] ( <a href="http://ogdraw.mpimp-golm.mpg.de/">http://ogdraw.mpimp-golm.mpg.de/</a>). The complete <i>C</i>. <i>humilis</i> chloroplast genome was submitted to GenBank, as Accession Number MF405921. </p> </div> <div class="comparativegenomeanalysis" title="sec"> <div class="title" tagx="title" title="title">
Comparative genome analysis</div> <p>To investigate the differences between the <i>C</i>. <i>humilis</i> genome sequence and those of the other members of the same genus, we used the LAGAN alignment strategy as implemented in the mVISTA software to compare the whole genome sequence of ten species of <i>Prunus</i> with <i>C</i>. <i>humilis</i>, using <i>C</i>. <i>humilis</i> as reference [ <a href="#pone.0196473.ref021">21</a>] ( <a href="http://genome.lbl.gov/cgi-bin/VistaInput?num_seqs=2">http://genome.lbl.gov/cgi-bin/VistaInput?num_seqs=2</a>). In addition, the differences in the types and gene sizes of the IR border genes of five species were analyzed. </p> </div> <div class="simplesequencerepeat(ssr)analysis" title="sec"> <div class="title" tagx="title" title="title">
Simple sequence repeat (SSR) analysis</div> <p>Microsatellites are also known as SSRs, which occur as mono-, di-, tri-, tetra-, penta-, and hexa-nucleotide repeats. The SSRs in the chloroplast genome of <i>C</i>. <i>humilis</i> were identified using PHOBOS 3.3.12, and the alignment was screened in terms of matches, mismatches, gaps, and N positions using the scores of 1, -5, -5, and 0, respectively [ <a href="#pone.0196473.ref022">22</a>]. At the same time, the SSR of the IR, LSC, SSC, and coding regions, introns, and intergenic regions corresponding to different regions were analyzed. </p> </div> <div class="phylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic analysis</div> <p>The chloroplast genome sequences of ten <i>Prunus</i> species were downloaded from NCBI. Approximately 61 coding-protein genes were identified in <i>C</i>. <i>humilis</i> and the other ten <i>Prunus</i> species, which were then aligned using mafft software's FFT-NS-2 algorithm [ <a href="#pone.0196473.ref023">23</a>]. After the results were integrated, phylogenetic reconstruction was performed using RAxML Version 8 tool as implemented in CIPRES [ <a href="#pone.0196473.ref024">24</a>]. </p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="generalfeaturesofthec.humilischloroplastgenome" title="sec"> <div class="title" tagx="title" title="title">
General features of the <i>C</i>. <i>humilis</i> chloroplast genome </div> <p>The genome of the <i>C</i>. <i>humilis</i> circular chloroplast is 158,084 bp in size and exhibits the typical quadripartite structure found in most land plants ( <a href="#pone.0196473.g002">Fig 2</a>). The IR region spans 52,672 bp, whereas the large single copy (LSC) and small single copy (SSC) region span 86,374 bp and 19,038 bp, respectively. GC contents of the complete genome is 36.8%, the LSC and SSC region are 34.5% and 30.5%, respectively, whereas that of the IR regions is 42.6% ( <a href="#pone.0196473.t001">Table 1</a>). Similar to other terrestrial plants such as <i>Cynara humilis</i>, a species of family Asteraceae, the IRS showed higher GC content, which was mainly due to the rRNA genes <i>rrn16</i>, <i>rrn23</i>, <i>rrn4</i>. <i>5</i>, and <i>rrn5</i> [ <a href="#pone.0196473.ref025">25</a>]. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0196473.g002"> <span class="label" tagx="label" title="label">Fig 2</span> </a> </div>   <div class="title" tagx="title" title="title">
Gene map of the <i>C</i>. <i>humilis</i> chloroplast genome. </div> <p>Genes drawn inside the circle are transcribed clockwise, whereas genes outside are transcribed counterclockwise. Genes are color coded by their function in the legend. The area in darker gray and lighter gray in the inner circle indicates GC content and AT content, respectively. </p>   </div> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0196473.t001">10.1371/journal.pone.0196473.t001</a> <span class="label" tagx="label" title="label">Table 1</span>  <div class="title" tagx="title" title="title">
Summary statistics for assembly of <i>C</i>. <i>humilis</i> chloroplast genomes. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>Genome features</th> <th> <i>Cerasus humilis</i> </th> </tr> </thead> <tbody> <tr> <td>Size (bp)</td> <td>158,084</td> </tr> <tr> <td>LSC length (bp)</td> <td>86,374</td> </tr> <tr> <td>SSC length (bp)</td> <td>19,038</td> </tr> <tr> <td>IR length (bp)</td> <td>52,672</td> </tr> <tr> <td>Number of gene</td> <td>131</td> </tr> <tr> <td>Protein-coding genes</td> <td>90</td> </tr> <tr> <td>tRNA genes</td> <td>33</td> </tr> <tr> <td>rRNA genes</td> <td>8</td> </tr> <tr> <td>GC content (%)</td> <td>36.8</td> </tr> <tr> <td>GC content in LSC (%) </td> <td>34.5</td> </tr> <tr> <td>GC content in SSC (%)</td> <td>30.5</td> </tr> <tr> <td>GC content in IR (%)</td> <td>42.6</td> </tr> </tbody> </table> </div> </div> <p>There are a total of 131 genes, including 90 protein-coding genes, 33 transfer RNA (tRNA), and 8 ribosomal RNA (rRNA) genes. Among these, 17 genes have two copies while <i>rps15</i> and <i>rps12</i> have more than copies (Tables <a href="#pone.0196473.t001">1</a> and <a href="#pone.0196473.t002">2</a>). In the functional classification of genes, 44 genes (unique) are associated with photosynthetic functions such as <i>atpH</i>, <i>ndhD</i>, and <i>rbcL</i>. The function of the conserved open reading frames of <i>ycf</i> ( <i>ycf1</i>, <i>ycf2</i>, <i>ycf3</i>, <i>ycf4</i> and <i>ycf15</i>) is unknown. The nine genes, namely, <i>atpF</i>, <i>ndhA</i>, <i>ndhB</i>, <i>petB</i>, <i>rpl16</i>, <i>rpl2</i>, <i>rpoC1</i>, <i>rps16</i>, and <i>ycf1</i>, contain a single intron. The two copies of the <i>rpl2</i> and <i>ndhB</i> genes all have one intron. In addition, there are two introns in the <i>ycf3</i> and c <i>lpP</i> genes ( <a href="#pone.0196473.t002">Table 2</a>). </p> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0196473.t002">10.1371/journal.pone.0196473.t002</a> <span class="label" tagx="label" title="label">Table 2</span>  <div class="title" tagx="title" title="title">
Gene content in <i>C</i>. <i>humilis</i> chloroplast genome. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>Function</th> <th>Family</th> <th>Genes</th> </tr> </thead> <tbody> <tr> <td> <b>Genes for photosynthesis</b> </td> <td>Subunits of ATP synthase</td> <td> <i>atpA</i>, <i>atpB</i>, <i>atpE</i>, <i>atpF</i> <sup> <i>a</i> </sup>, <i>atpH</i>, <i>atpI</i> </td> </tr> <tr> <td>Subunits of NADH-dehydrogenase</td> <td> <i>ndhA</i> <a href="#t002fn001"> <sup>a</sup> </a>, <i>ndhB</i> <a href="#t002fn001"> <sup>a</sup> </a> (2), <i>ndhC</i>, <i>ndhD</i>, <i>ndhE</i>, <i>ndhF</i>, <i>ndhG</i>, <i>ndhH</i>, <i>ndhI</i>, <i>ndhJ</i>, <i>ndhK</i> </td> </tr> <tr> <td>Subunits of cytochrome b/f complex</td> <td> <i>petA</i>, <i>petB</i> <a href="#t002fn001"> <sup>a</sup> </a> <i>petD</i>, <i>petG</i>, <i>petL</i>, <i>petN</i> </td> </tr> <tr> <td>Subunits of photosystem I</td> <td> <i>psaA</i>, <i>psaB</i>, <i>psaC</i>, <i>psaI</i>, <i>psaJ</i> </td> </tr> <tr> <td>Subunits of photosystem II</td> <td> <i>psbA</i>, <i>psbB</i>, <i>psbC</i>, <i>psbD</i>, <i>psbE</i>, <i>psbF</i>, <i>psbH</i>, <i>psbI</i>, <i>psbJ</i>, <i>psbK</i>, <i>psbL</i>, <i>psbM</i>, <i>psbN</i>, <i>psbT</i>, <i>psbZ</i> </td> </tr> <tr> <td>Subunit of RuBisCO</td> <td> <i>rbcL</i> </td> </tr> <tr> <td> <b>Other genes</b> </td> <td>Subunit of Acetyl-CoA-carboxylase</td> <td> <i>accD</i> </td> </tr> <tr> <td>c-type cytochrome synthesis gene</td> <td> <i>ccsA</i> </td> </tr> <tr> <td>Envelop membrane protein</td> <td> <i>cemA</i> </td> </tr> <tr> <td>Protease</td> <td> <i>clpP</i> <a href="#t002fn002"> <sup>b</sup> </a> </td> </tr> <tr> <td>Maturase</td> <td> <i>matK</i> </td> </tr> <tr> <td> <b>Self-replication</b> </td> <td>Large subunit of ribosome</td> <td> <i>rpl14</i>, <i>rpl16</i> <a href="#t002fn001"> <sup>a</sup> </a>, <i>rpl20</i>, <i>rpl22</i>, <i>rpl23</i> (2), <i>rpl2</i> <a href="#t002fn001"> <sup>a</sup> </a> (2), <i>rpl32</i>, <i>rpl33</i>, <i>rpl36</i> </td> </tr> <tr> <td>DNA-dependent RNA polymerase</td> <td> <i>rpoA</i>, <i>rpoB</i>, <i>rpoC1</i> <a href="#t002fn001"> <sup>a</sup> </a>, <i>rpoC2</i> </td> </tr> <tr> <td>Small subunit of ribosome</td> <td> <i>rps11</i>, <i>rps12</i> (3), <i>rps14</i>, <i>rps15</i>, <i>rps16</i> <a href="#t002fn001"> <sup>a</sup> </a>, <i>rps18</i>, <i>rps19</i> (2), <i>rps2</i>, <i>rps3</i>, <i>rps4</i>, <i>rps7</i> (2), <i>rps8</i> </td> </tr> <tr> <td>rRNA Genes</td> <td> <i>rrn4</i>. <i>5S</i> (2), <i>rrn5S</i> (2), <i>rrn23S</i> (2), <i>rrn16S</i> (2) </td> </tr> <tr> <td>tRNA genes</td> <td> <i>trnR-ACG</i> (2), <i>trnS-TGA</i>, <i>trnT-TGT</i>, <i>trnS-GCT</i>, <i>trnF-GAA</i>, <i>trnH-GTG</i>, <i>trnD-GTC</i>, <i>trnL-CAA</i> (2), <i>trnE-TTC</i>, <i>trnG-GCC</i>, <i>trnI-CAT</i> (2), <i>trnI-TAT</i> (2), <i>trnR-TCT</i>, <i>trnL-TAG</i>, <i>trnM-CAT</i>, <i>trnY-GTA</i>, <i>trnQ-TTG</i>, <i>trnT-GGT</i>, <i>trnS-AGA</i> (2), <i>trnN-GTT</i> (2), <i>trnW-CCA</i>, <i>trnfM-CAT</i>, <i>trnS-GGA</i>, <i>trnP-TGG</i>, <i>trnV-GAC</i> (2), <i>trnC-GCA</i> </td> </tr> <tr> <td> <b>Unknown function</b> </td> <td>Conserved open reading frames</td> <td> <i>ycf1</i> <a href="#t002fn001"> <sup>a</sup> </a>, <i>ycf2</i> (2), <i>ycf3</i> <a href="#t002fn002"> <sup>b</sup> </a>, <i>ycf4</i>, <i>ycf15</i> (4) </td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p> <sup>a</sup>: Gene containing a single intron </p> </div> <div class="fn-type-" title=""> <p> <sup>b</sup>: Gene containing two introns. </p> </div> </div> </div> </div> <div class="comparativegenomeanalysis" title="sec"> <div class="title" tagx="title" title="title">
Comparative genome analysis</div> <p>To further analyze the characteristics of the <i>C</i>. <i>humilis</i> chloroplast genome, we compared the assembled genomes with the chloroplast genomes of ten other <i>Prunus</i> species in NCBI. First, we compared the basic characteristics of their genomes. The size of the <i>C</i>. <i>humilis</i> chloroplast genome is smaller than those of <i>P</i>. <i>padus, <a href="https://www.ncbi.nlm.nih.gov/genome/45388?genome_assembly_id=279000">P. pseudocerasus</a> </i> and <i>P</i>. <i>tomentosa</i>. However, it has the highest GC content and the second largest number of encoded proteins (N = 90). In addition, it has 131 genes in total and the number of tRNA genes is the lowest among these 11 members of the <i>Prunus</i> genus ( <a href="#pone.0196473.t003">Table 3</a>). The sequence identity was analysed using program mVISTA by aligning the ten <i>Prunus</i> species’ chloroplast genomes with <i>C</i>. <i>humilis </i>( <a href="#pone.0196473.g003">Fig 3</a>). As expected, the sequence identity of the eleven chloroplast genome sequences was high, indicating that these are highly conserved. However, several variations were found in noncoding and single copy regions of these eleven chloroplast genome ( <a href="#pone.0196473.g003">Fig 3</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0196473.g003"> <span class="label" tagx="label" title="label">Fig 3</span> </a> </div>   <div class="title" tagx="title" title="title">
Visualization of genome alignment of the chloroplast genomes of ten <i>Prunus</i> species using <i>C</i>. <i>humilis</i> as reference using mVISTA. </div> <p>Vertical scale indicates the percentage of identity, which ranges from 50% to 100%. Coding regions are marked in blue, and non-coding regions are indicated in red. Gray arrows indicate the position and direction of each gene. </p>   </div> <div class="table-wrap_UNKNOWN"> <a href="https://dx.doi.org/10.1371/journal.pone.0196473.t003">10.1371/journal.pone.0196473.t003</a> <span class="label" tagx="label" title="label">Table 3</span>  <div class="title" tagx="title" title="title">
Comparison of chloroplast genome characteristics in various <i>Prunus</i> species. </div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" /> <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>Species</th> <th>GenBank Acc. No.</th> <th>Size (kb)</th> <th>GC%</th> <th>Protein</th> <th>rRNA</th> <th>tRNA</th> <th>Gene</th> <th>Pseudogene</th> </tr> </thead> <tbody> <tr> <td> <i>Cerasus humilis</i> </td> <td>MF405921</td> <td>158.084</td> <td>36.80</td> <td>90</td> <td>8</td> <td>33</td> <td>131</td> <td>-</td> </tr> <tr> <td> <i>Prunus kansuensis</i> </td> <td>NC_023956.1</td> <td>157.736</td> <td>36.76</td> <td>86</td> <td>8</td> <td>39</td> <td>133</td> <td>-</td> </tr> <tr> <td> <i>Prunus maximowiczii</i> </td> <td>NC_026981.1</td> <td>157.852</td> <td>36.73</td> <td>84</td> <td>8</td> <td>37</td> <td>129</td> <td>-</td> </tr> <tr> <td> <i>Prunus persica</i> </td> <td>NC_014697.1</td> <td>157.79</td> <td>36.76</td> <td>85</td> <td>8</td> <td>37</td> <td>131</td> <td>1</td> </tr> <tr> <td> <i>Prunus pseudocerasus</i> </td> <td>NC_030599.1</td> <td>157.834</td> <td>36.73</td> <td>86</td> <td>8</td> <td>37</td> <td>131</td> <td>-</td> </tr> <tr> <td> <i>Prunus yedoensis</i> </td> <td>NC_026980.1</td> <td>157.859</td> <td>36.73</td> <td>84</td> <td>8</td> <td>37</td> <td>129</td> <td>-</td> </tr> <tr> <td> <i>Prunus serotina</i> </td> <td>NC_036133.1</td> <td>158,778</td> <td>36.60</td> <td>82</td> <td>8</td> <td>36</td> <td>126</td> <td>-</td> </tr> <tr> <td> <i>Prunus dulcis</i> </td> <td>NC_034696.1</td> <td>157,723</td> <td>36.80</td> <td>86</td> <td>0</td> <td>38</td> <td>124</td> <td>-</td> </tr> <tr> <td> <i>Prunus cerasoides</i> </td> <td>NC_035891.1</td> <td>157,685</td> <td>36.70</td> <td>84</td> <td>8</td> <td>37</td> <td>129</td> <td>-</td> </tr> <tr> <td> <i>Prunus tomentosa</i> </td> <td>NC_036394.1</td> <td>158,356</td> <td>36.80</td> <td>91</td> <td>0</td> <td>36</td> <td>127</td> <td>-</td> </tr> <tr> <td> <i>Prunus padus</i> </td> <td>NC_026982.1</td> <td>158.955</td> <td>36.59</td> <td>84</td> <td>8</td> <td>37</td> <td>129</td> <td>-</td> </tr> </tbody> </table> </div> </div> <p>In addition, we compared the SSC region and the IR-LSC and IR-SSC boundaries of <i>C</i>. <i>humilis</i> with other four species of <i>Prunus: <a href="https://www.ncbi.nlm.nih.gov/genome/37250?genome_assembly_id=230545">P. maximowiczii</a>, <a href="https://www.ncbi.nlm.nih.gov/genome/31803?genome_assembly_id=57073">P. kansuensis</a>, <a href="https://www.ncbi.nlm.nih.gov/genome/388?genome_assembly_id=312190">P. persica</a> and <a href="https://www.ncbi.nlm.nih.gov/genome/45388?genome_assembly_id=279000">P. pseudocerasus</a> </i> ( <a href="#pone.0196473.g004">Fig 4</a>). In the SSC region, all genes are conserved except tmS-AGA. The <i>rps19</i> gene is present at the LSC/IRA junction of the <i>C</i>. <i>humilis</i>, with a size of 142 bp which is shorter than other species’ <i>rps19</i> gene. The IR boundary regions also showed slight variations, which included gene deletions, insertions, expansions, and contractions ( <a href="#pone.0196473.g004">Fig 4</a>). <i>C</i>. <i>humilis</i> and <i>P</i>. <i>maximowiczii</i> lost the <i>ycf1</i> gene in IRA/SSC boundary region. Moreover, <i>rps19</i> gene was deleted in <i>P</i>. <i>maximowiczii</i>, but inserted in <i>C</i>. <i>humilis</i> at the IRB/LSC boundary. The size of these bordering genes also varied. The small <i>ycf1</i> gene ranged in size from 1,003 bp to 1,059 bp, and that of the larger <i>ycf1</i> gene ranged from 5,607 bp to 5,664 bp, with the largest observed in <i>C</i>. <i>humilis</i>. At the IRB/LSC junction, the <i>rps19</i> gene was within the range of 183 bp–210 bp, whereas <i>P</i>. <i>maximowiczii</i> has no <i>rps19</i> gene at this area. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0196473.g004"> <span class="label" tagx="label" title="label">Fig 4</span> </a> </div>   <div class="title" tagx="title" title="title">
Comparison of the boundaries of the LSC, SSC, and IR regions in the <i>C</i>. <i>humilis</i> chloroplast genome with those of four other <i>Prunus</i> species. </div>   </div> <p>No major errors in genome assembly were observed in this study, the <i>ycf1</i> and <i>rps19</i> genes showed deletions, insertions, expansions and contractions among the five species, indicating that these genes may be used as markers in studying the evolution of the IR/SSC and IR/LSC junction in the <i>Prunus</i>. </p> </div> <div class="ssranalysis" title="sec"> <div class="title" tagx="title" title="title">
SSR analysis</div> <p>A total of 510 SSRs were identified in the chloroplast genome of <i>C</i>. <i>humilis</i>. Then, we studied the distribution, presence, and types of SSRs ( <a href="#pone.0196473.g005">Fig 5</a>). Of the 510 SSRs, 306 (60%), 55 (11%), and 149 (29%) were discovered within the LSC, SSC, and IR regions, respectively ( <a href="#pone.0196473.g005">Fig 5A</a>). In addition, 263, 510, and 19 SSRs were situated within the protein-coding regions, intergenic spacer regions, and introns, respectively. The presence of SSRs in the SSC and IR regions was the same as that above except for the LSC, i.e., the SSR number in the protein-coding regions was the highest, followed by intergenic spacers, and then introns. Furthermore, the predominant SSRs included dinucleotides, trinucleotides, and tetranucleotides, with the repeat-number equal to three, and dinucleotide span the repeat-number range from 3 to 9. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0196473.g005"> <span class="label" tagx="label" title="label">Fig 5</span> </a> </div>   <div class="title" tagx="title" title="title">
The distribution and type of SSRs in the chloroplast genome of <i>C</i>. <i>humilis</i>. </div> <p>(A) Percentage of SSRs in the LSC, SSC, and IR regions. (B) Distribution of SSRs in the protein-coding regions, introns, and intergenic spacers of the LSC, SSC, and IR regions. (C) Presence of polymers in the chloroplast genome of <i>C</i>. <i>humilis</i>. </p>   </div> </div> <div class="phylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic analysis</div> <p>The 61 conservative coding-protein genes were used to analyze the phylogenetic relationships among the members of <i>Prunus</i> ( <a href="#pone.0196473.g006">Fig 6</a>). The phylogenetic tree demonstrated that <i>P</i>. <i>persica</i> and <i>P</i>. <i>kansuensis</i> formed a monophyletic clade while <i>C</i>. <i>humilis</i> and <i>P</i>. <i>tomentosa</i> formed another monophyletic clade. These two clades were then clustered with <i>P</i>. <i>dulcis</i> to form a larger branch ( <a href="#pone.0196473.g006">Fig 6</a>). <i>C</i>. <i>humilis</i> and <i>P</i>. <i>tomentosa</i> were previously classified to the subgenus <i>Microcerasus</i> based on their morphological characteristics [ <a href="#pone.0196473.ref014">14</a>]. The phylogenetic results of this study provide new evidence for this classification at the genome level. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0196473.g006"> <span class="label" tagx="label" title="label">Fig 6</span> </a> </div>   <div class="title" tagx="title" title="title">
Phylogenetic reconstruction of the <i>Prunus</i> species based on 61 conserved coding-protein genes using the maximum likelihood approach. </div>   </div> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>In this study, the complete chloroplast genome of <i>C</i>. <i>humilis</i> was sequenced and annotated, and the SSRs and comparative genomics of the genome were analyzed. Moreover, phylogenetic reconstruction of <i>C</i>. <i>humilis</i> and ten other <i>Prunus</i> species was performed. Although the chloroplast genomes are highly conserved in terms of genomic structure and size, the IR/SC border genes varied in size and in type, which is a typical feature of chloroplast genomes [ <a href="#pone.0196473.ref010">10</a>, <a href="#pone.0196473.ref026">26</a>, <a href="#pone.0196473.ref027">27</a>]. The genomic structure of ten <i>Prunus </i>species and <i>C</i>. <i>humilis</i> are also relatively conserved, whereas differences in the presence of genes in the IR/SC border region were observed. <i>P</i>. <i>kansuensis</i> and <i>C</i>. <i>humilis</i> lost the <i>ycf1</i> gene in the IRA/SSC border region. <i>Ycf1</i> is one of the giant ORFs in most higher plants of chloroplast genomes and it usually spans the boundary of the IR and SSC regions of the plastid genome [ <a href="#pone.0196473.ref028">28</a>]. However, in the orchid genus <i>Phalaenopsis</i>, the entire <i>ycf1</i> gene is situated within the SSC region [ <a href="#pone.0196473.ref029">29</a>, <a href="#pone.0196473.ref030">30</a>]. In addition, the function of the <i>ycf1</i> gene remains elusive [ <a href="#pone.0196473.ref031">31</a>]. In tobacco, it has been suggested that the <i>ycf1</i> gene does not encode a functional protein while some other studies have indicated that it encodes a protein that is essential for cell survival or related to the ABC-transporters [ <a href="#pone.0196473.ref031">31</a>, <a href="#pone.0196473.ref032">32</a>]. The higher variation in the <i>ycf1</i> gene could provide superior resolution and support at lower taxonomic levels in <i>Orchidaceae</i> [ <a href="#pone.0196473.ref028">28</a>]. In our study, the small size of the <i>ycf1</i> gene in IRA/SSC boundary ranges from 1,003 bp to 1,059 bp which is significantly smaller than the larger <i>ycf1</i> gene (size range: 5,607 bp-5,662 bp). The role of this gene deletion in the evolution of chloroplast genome of <i>Prunus</i> requires further investigations. </p> <p>In this study, two copies of <i>C</i>. <i>humilis rps19</i> gene were found in the boundaries of IRA/LSC and IRB/SSC repectively, whereas the <i>rps19</i> gene was deleted from the IRB/SSC boundary in <i>P</i>. <i>kansuensis</i>. Similar results were found in two leguminous plants <i>Cajanus</i> and <i>Millettia</i> [ <a href="#pone.0196473.ref033">33</a>– <a href="#pone.0196473.ref035">35</a>]. In addition, it was reported that one of the two <i>rps19</i> genes in the IR/SC boundaries is usually a pseudogene. For example, <i>Dianthus</i> encodes one copy of the <i>rps19</i> gene at the IRB/SSC junction and a pseudogene <i>rps19</i> at IRA/LSC junction while the size of the pseudogene <i>rps19</i> is shorter than that of the regular <i>rps19</i> gene [ <a href="#pone.0196473.ref034">34</a>]. In three <i>Cardiocrinum</i> (Liliaceae) species, the <i>rps19</i> gene located in the boundary between LSC and IRA apparently lost its protein-coding ability due to partial gene duplication [ <a href="#pone.0196473.ref036">36</a>]. The <i>rps19</i> gene of <i>C</i>. <i>humilis</i> located in the boundary between LSC/IRA is much shorter than those of other four <i>Prunus</i> species which means it could also be a pseudogene. Roles of the <i>rps19</i> gene’s insertion and deletion observed in the chloroplast genomes of <i>Prunus</i> species remains to be elucidated. </p> <p> <i>C</i>. <i>humilis</i> belongs to the genus <i>Prunus</i> which consists of approximately 250 species. Many economically important fruit crops such as cherry, plum, apricot, almond, peach are in this genus. However, there are different opinions about how <i>C</i>. <i>humilis</i> should be grouped and named. A famous Chinese plant taxonomist earlier classified <i>C</i>. <i>humilis</i> as <i>Rosaceae cerasus</i> because he believed <i>C</i>. <i>humilis</i> is a close relative to <i>P</i>. <i>pseudocerasus</i> (cherry) [ <a href="#pone.0196473.ref014">14</a>]. However, the phylogenetic trees we built in this study showed that <i>C</i>. <i>humilis</i> is closer to <i>P</i>. <i>tomentosa</i> than to <i>P</i>. <i>pseudocerasus</i>. Our results agree with another classification which put <i>C</i>. <i>humilis</i> and <i>P</i>. <i>tomentosa</i> to the same <i>subgenus Microcerasus</i> based on their morphological characteristics [ <a href="#pone.0196473.ref037">37</a>]. Therefore, the results of this study might draw attentions of other scientists who have been working on clarifying the phylogenetic relationships among species of genus <i>Prunus</i>. More chloroplast genome sequences will be needed to improve our understanding of the evolution of <i>Prunus</i> species. Our findings not only provide the foundation for further studies on the evolution of chloroplast genome of <i>C</i>. <i>humilis</i> and <i>Prunus</i> species, but also serve as the basis for the molecular identification of <i>Prunus</i> species. </p> <p> <i>C</i>. <i>humilis</i> was a wild species 30 years ago and it had sour and astringent taste which was quite different for people to eat. Our team started a program including domestication and breeding of this species in 1987 aiming at improving its fruit characters for commercial cultivation. One part of this program is to introduce specific advantageous traits into <i>C</i>. <i>humilis</i> from cultivated crops by crossing. However, little has been accomplished despite a lot of effort. Success in breeding is determined by genetic compatibility and chloroplast genomes serve as a valuable tool for identifying plants that are likely to be closely related and, therefore, genetically compatible [ <a href="#pone.0196473.ref038">38</a>]. We believe that chloroplast genome of <i>C</i>. <i>humilis</i> can provide useful information for us to select proper parental combinations to increase breeding efficiency. Another key application of the chloroplast genome of <i>C</i>. <i>humilis</i> is the identification of commercial cultivars and the determination of their purity. DNA barcodes could be developed from its chloroplast genome and be used to identify varieties and their offsprings. </p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>The authors would like to acknowledge Ms. Luting Jia and Ms. Xuanxuan Han who provided invaluable assistance in the samples collection. Dr. Juan Wang provided critical comments on an earlier version of the manuscript. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="pone.0196473.ref001" /> <span class="label" tagx="label" title="label">1</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mu</span> <span class="given-names" tagx="given-names" title="given-names">XP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aryal</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Du</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Du</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Oil content and fatty acid composition of the kernels of 31 genotypes of Chinese dwarf cherry [ <i>Cerasus humilis</i> (Bge.) Sok.] </span>. <span class="source" tagx="source" title="source">Journal of Horticultural Science &amp;amp; Biotechnology</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">90</span>: <span class="fpage" tagx="fpage" title="fpage">525</span>– <span class="lpage" tagx="lpage" title="lpage">529</span>. <span class="comment">doi: <a href="https://doi.org/10.1080/14620316.2015.11668709">10.1080/14620316.2015.11668709</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref002" /> <span class="label" tagx="label" title="label">2</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">XS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shang</span> <span class="given-names" tagx="given-names" title="given-names">ZW</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yin</span> <span class="given-names" tagx="given-names" title="given-names">ZP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ren</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">MC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">XL</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Mechanism of xanthophyll-cycle-mediated photoprotection in <i>Cerasus humilis</i> seedlings under water stress and subsequent recovery </span>. <span class="source" tagx="source" title="source">Photosynthetica</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">49</span>: <span class="fpage" tagx="fpage" title="fpage">523</span>– <span class="lpage" tagx="lpage" title="lpage">530</span>. <span class="comment">doi: <a href="https://doi.org/10.1007/s11099-011-0065-4">10.1007/s11099-011-0065-4</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref003" /> <span class="label" tagx="label" title="label">3</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Du</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chi</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Distribution and groups of Chinese dwarf cherry ( <i>Cerasus humilis</i>) in Shanxi province </span>. <span class="source" tagx="source" title="source">Crop Variety Resources</span>. <span class="year" tagx="year" title="year">1993</span>; <span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">6</span>– <span class="lpage" tagx="lpage" title="lpage">7</span>. </span> </li> <li tag="ref"> <a name="pone.0196473.ref004" /> <span class="label" tagx="label" title="label">4</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yin</span> <span class="given-names" tagx="given-names" title="given-names">ZP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ren</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">XS</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Role of spermidine and spermine in alleviation of drought-induced oxidative stress and photosynthetic inhibition in Chinese dwarf cherry ( <i>Cerasus humilis</i>) seedlings </span>. <span class="source" tagx="source" title="source">Plant Growth Regulation</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">74</span>: <span class="fpage" tagx="fpage" title="fpage">209</span>– <span class="lpage" tagx="lpage" title="lpage">218</span>. <span class="comment">doi: <a href="https://doi.org/10.1007/s10725-014-9912-1">10.1007/s10725-014-9912-1</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref005" /> <span class="label" tagx="label" title="label">5</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ha</span> <span class="given-names" tagx="given-names" title="given-names">XL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">ZJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liang</span> <span class="given-names" tagx="given-names" title="given-names">JY</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Comparison on photosynthetic and water use characteristics of different varieties (Lines) in <i>Prunus humilis</i> </span>. <span class="source" tagx="source" title="source">Northern Horticulture</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="fpage" tagx="fpage" title="fpage">7</span>– <span class="lpage" tagx="lpage" title="lpage">11</span>. <span class="comment">doi: <a href="https://doi.org/10.11937/bfyy.201702002">10.11937/bfyy.201702002</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref006" /> <span class="label" tagx="label" title="label">6</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hao</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jin</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Genotypic diversity of phenolic compounds and antioxidant capacity of Chinese dwarf cherry ( <i>Cerasus humilis</i> (Bge.) Sok.) in China </span>. <span class="source" tagx="source" title="source">Scientia Horticulturae</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">175</span>: <span class="fpage" tagx="fpage" title="fpage">208</span>– <span class="lpage" tagx="lpage" title="lpage">213</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.scienta.2014.06.015">10.1016/j.scienta.2014.06.015</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref007" /> <span class="label" tagx="label" title="label">7</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leliaert</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">DR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moreau</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Herron</span> <span class="given-names" tagx="given-names" title="given-names">MD</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Verbruggen</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Delwiche</span> <span class="given-names" tagx="given-names" title="given-names">CF</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogeny and molecular evolution of the green Algae</span>. <span class="source" tagx="source" title="source">Critical Reviews in Plant Sciences</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">31</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">46</span>. <span class="comment">doi: <a href="https://doi.org/10.1080/07352689.2011.615705">10.1080/07352689.2011.615705</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref008" /> <span class="label" tagx="label" title="label">8</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolfe</span> <span class="given-names" tagx="given-names" title="given-names">KH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">WH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharp</span> <span class="given-names" tagx="given-names" title="given-names">PM</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Rates of nucleotide substitution vary greatly among plant mitochondrial, chloroplast and nuclear DNA </span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences</span>. <span class="year" tagx="year" title="year">1988</span>; <span class="volume" tagx="volume" title="volume">84</span>: <span class="fpage" tagx="fpage" title="fpage">9054</span>– <span class="lpage" tagx="lpage" title="lpage">9058</span>. </span> </li> <li tag="ref"> <a name="pone.0196473.ref009" /> <span class="label" tagx="label" title="label">9</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neuhaus</span> <span class="given-names" tagx="given-names" title="given-names">HE</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Emes</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Nonphotosynthetic metabolism in plastids</span>. <span class="source" tagx="source" title="source">Annual Review of Plant Physiology &amp;amp; Plant Molecular Biology</span>. <span class="year" tagx="year" title="year">2000</span>; <span class="volume" tagx="volume" title="volume">51</span>: <span class="fpage" tagx="fpage" title="fpage">111</span> <span class="comment">doi: <a href="https://doi.org/10.1146/annurev.arplant.51.1.111">10.1146/annurev.arplant.51.1.111</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15012188">15012188</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref010" /> <span class="label" tagx="label" title="label">10</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zuo</span> <span class="given-names" tagx="given-names" title="given-names">LH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shang</span> <span class="given-names" tagx="given-names" title="given-names">AQ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">XY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ren</span> <span class="given-names" tagx="given-names" title="given-names">YC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">MS</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">The first complete chloroplast genome sequences of <i>Ulmus</i> species by <i>de novo</i> sequencing: Genome comparative and taxonomic position analysis </span>. <span class="source" tagx="source" title="source">PLoS ONE</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">e0171264</span> <span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0171264">10.1371/journal.pone.0171264</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28158318">28158318</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref011" /> <span class="label" tagx="label" title="label">11</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Howe</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Chloroplast Genome</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="source" tagx="source" title="source">eLS</span>. <span class="comment">doi: <a href="https://doi.org/10.1038/npg.els.0002016">10.1038/npg.els.0002016</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref012" /> <span class="label" tagx="label" title="label">12</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Complete plastome sequence of <i>Thalictrum coreanum</i> (Ranunculaceae) and transfer of the <i>rpl32</i> gene to the nucleus in the ancestor of the subfamily Thalictroideae </span>. <span class="source" tagx="source" title="source">BMC Plant Biology</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">40</span> <span class="comment">doi: <a href="https://doi.org/10.1186/s12870-015-0432-6">10.1186/s12870-015-0432-6</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25652741">25652741</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref013" /> <span class="label" tagx="label" title="label">13</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reith</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Munholland</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Complete nucleotide sequence of the <i>Porphyra pupurea</i> chloroplast </span>. <span class="source" tagx="source" title="source">Plant Molecular Biology Reporter</span>. <span class="year" tagx="year" title="year">1995</span>; <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">333</span>– <span class="lpage" tagx="lpage" title="lpage">335</span>. <span class="comment">doi: <a href="https://doi.org/10.1007/BF02669187">10.1007/BF02669187</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref014" /> <span class="label" tagx="label" title="label">14</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span>. <span class="source" tagx="source" title="source">Chinese fruit taxonomy</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">China agriculture Publishing</span>; <span class="year" tagx="year" title="year">1979</span>. </span> </li> <li tag="ref"> <a name="pone.0196473.ref015" /> <span class="label" tagx="label" title="label">15</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dierckxsens</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mardulyn</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smits</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>. <span class="mixed-article-title" title="mixed-article-title">NOVOPlasty: <i>de novo</i> assembly of organelle genomes from whole genome data </span>. <span class="source" tagx="source" title="source">Nucleic Acids Research</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">45</span>( <span class="issue" tagx="issue" title="issue">4</span>): <span class="fpage" tagx="fpage" title="fpage">e18</span> <span class="comment">doi: <a href="https://doi.org/10.1093/nar/gkw955">10.1093/nar/gkw955</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28204566">28204566</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref016" /> <span class="label" tagx="label" title="label">16</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>. <span class="mixed-article-title" title="mixed-article-title">A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data </span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">27</span>: <span class="fpage" tagx="fpage" title="fpage">2987</span>– <span class="lpage" tagx="lpage" title="lpage">2993</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/bioinformatics/btr509">10.1093/bioinformatics/btr509</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21903627">21903627</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref017" /> <span class="label" tagx="label" title="label">17</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Durbin</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">1754</span>– <span class="lpage" tagx="lpage" title="lpage">1760</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/bioinformatics/btp324">10.1093/bioinformatics/btp324</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168">19451168</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref018" /> <span class="label" tagx="label" title="label">18</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shi</span> <span class="given-names" tagx="given-names" title="given-names">LC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">YJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">HM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">JH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">XH</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">CpGAVAS, an integrated web server for the annotation, visualization, analysis, and GenBank submission of completely sequenced chloroplast genome sequences </span>. <span class="source" tagx="source" title="source">BMC Genomics</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">715</span> <span class="comment">doi: <a href="https://doi.org/10.1186/1471-2164-13-715">10.1186/1471-2164-13-715</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23256920">23256920</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref019" /> <span class="label" tagx="label" title="label">19</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lowe</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>. <span class="mixed-article-title" title="mixed-article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">1997</span>; <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">955</span>– <span class="lpage" tagx="lpage" title="lpage">964</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/nar/25.5.955">10.1093/nar/25.5.955</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104">9023104</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref020" /> <span class="label" tagx="label" title="label">20</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lohse</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drechsel</span> <span class="given-names" tagx="given-names" title="given-names">O</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kahlau</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bock</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>. <span class="mixed-article-title" title="mixed-article-title">OrganellarGenomeDRAW-a suite of tools for generating physical maps of plastid and mitochondrial genomes and visualizing expression data sets </span>. <span class="source" tagx="source" title="source">Nucleic Acids Research</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">41</span>: <span class="fpage" tagx="fpage" title="fpage">W575</span>– <span class="lpage" tagx="lpage" title="lpage">581</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/nar/gkt289">10.1093/nar/gkt289</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23609545">23609545</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref021" /> <span class="label" tagx="label" title="label">21</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frazer</span> <span class="given-names" tagx="given-names" title="given-names">KA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pachter</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Poliakov</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rubin</span> <span class="given-names" tagx="given-names" title="given-names">EM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dubchak</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>. <span class="mixed-article-title" title="mixed-article-title">VISTA: Computational tools for comparative genomics</span>. <span class="source" tagx="source" title="source">Nucleic Acids Research</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">32</span>: <span class="fpage" tagx="fpage" title="fpage">W273</span>– <span class="lpage" tagx="lpage" title="lpage">279</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/nar/gkh458">10.1093/nar/gkh458</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15215394">15215394</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref022" /> <span class="label" tagx="label" title="label">22</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leese</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mayer</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Held</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Isolation of microsatellites from unknown genomes using known genomes as enrichment templates </span>. <span class="source" tagx="source" title="source">Limnology &amp;amp; Oceanography Methods</span>. <span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">6</span>( <span class="issue" tagx="issue" title="issue">5</span>): <span class="fpage" tagx="fpage" title="fpage">412</span>– <span class="lpage" tagx="lpage" title="lpage">426</span>. </span> </li> <li tag="ref"> <a name="pone.0196473.ref023" /> <span class="label" tagx="label" title="label">23</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Katoh</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Misawa</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuma</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miyata</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Mafft: a novel method for rapid multiple sequence alignment based on fast fourier transform </span>. <span class="source" tagx="source" title="source">Nucleic Acids Research</span>. <span class="year" tagx="year" title="year">2002</span>; <span class="volume" tagx="volume" title="volume">30</span>( <span class="issue" tagx="issue" title="issue">14</span>): <span class="fpage" tagx="fpage" title="fpage">3059</span>– <span class="lpage" tagx="lpage" title="lpage">3066</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/nar/gkf436">10.1093/nar/gkf436</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12136088">12136088</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref024" /> <span class="label" tagx="label" title="label">24</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stamatakis</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>. <span class="mixed-article-title" title="mixed-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">1312</span>– <span class="lpage" tagx="lpage" title="lpage">1313</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/bioinformatics/btu033">10.1093/bioinformatics/btu033</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24451623">24451623</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref025" /> <span class="label" tagx="label" title="label">25</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Curci</span> <span class="given-names" tagx="given-names" title="given-names">PL</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De Paola</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Danzi</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vendramin</span> <span class="given-names" tagx="given-names" title="given-names">GG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sonnante</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Complete chloroplast genome of the multifunctional crop globe artichoke and comparison with other Asteraceae </span>. <span class="source" tagx="source" title="source">PLoS ONE</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">e0120589</span> <span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0120589">10.1371/journal.pone.0120589</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25774672">25774672</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref026" /> <span class="label" tagx="label" title="label">26</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">HL</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Complete chloroplast genome sequences from Korean ginseng ( <i>Panax schinseng</i> Nees) and comparative analysis of sequence evolution among 17 vascular plants </span>. <span class="source" tagx="source" title="source">DNA Research</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">247</span>– <span class="lpage" tagx="lpage" title="lpage">261</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/dnares/11.4.247">10.1093/dnares/11.4.247</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15500250">15500250</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref027" /> <span class="label" tagx="label" title="label">27</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Silva</span> <span class="given-names" tagx="given-names" title="given-names">SR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Diaz</span> <span class="given-names" tagx="given-names" title="given-names">YC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Penha</span> <span class="given-names" tagx="given-names" title="given-names">HA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pinheiro</span> <span class="given-names" tagx="given-names" title="given-names">DG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fernandes</span> <span class="given-names" tagx="given-names" title="given-names">CC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miranda</span> <span class="given-names" tagx="given-names" title="given-names">VF</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">The chloroplast genome of <i>Utricularia reniformis</i> sheds light on the evolution of the <i>ndh</i> gene complex of terrestrial carnivorous plants from the Lentibulariaceae Family </span>. <span class="source" tagx="source" title="source">PLoS ONE</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">e0165176</span> <span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0165176">10.1371/journal.pone.0165176</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27764252">27764252</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref028" /> <span class="label" tagx="label" title="label">28</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neubig</span> <span class="given-names" tagx="given-names" title="given-names">KM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Whitten</span> <span class="given-names" tagx="given-names" title="given-names">WM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carlsward</span> <span class="given-names" tagx="given-names" title="given-names">BS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blanco</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Endara</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williams</span> <span class="given-names" tagx="given-names" title="given-names">NH</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogenetic utility of <i>ycf1</i> in orchids: a plastid gene more variable than <i>matK</i> </span>. <span class="source" tagx="source" title="source">Plant Systematics and Evolution</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">277</span>: <span class="fpage" tagx="fpage" title="fpage">77</span>– <span class="lpage" tagx="lpage" title="lpage">84</span>. <span class="comment">doi: <a href="https://doi.org/10.1007/s00606-008-0105-0">10.1007/s00606-008-0105-0</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref029" /> <span class="label" tagx="label" title="label">29</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chang</span> <span class="given-names" tagx="given-names" title="given-names">CC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">HC</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chow</span> <span class="given-names" tagx="given-names" title="given-names">TY</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">HH</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">WH</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">The chloroplast genome of <i>Phalaenopsis aphrodite</i> (Orchidaceae): Comparative analysis of evolutionary rate with that of grasses and its phylogenetic implications </span>. <span class="source" tagx="source" title="source">Molecular Biology &amp;amp; Evolution</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">23</span>: <span class="fpage" tagx="fpage" title="fpage">279</span> <span class="comment">doi: <a href="https://doi.org/10.1093/molbev/msj029">10.1093/molbev/msj029</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16207935">16207935</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref030" /> <span class="label" tagx="label" title="label">30</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raubeson</span> <span class="given-names" tagx="given-names" title="given-names">LA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span>. <span class="chapter-title" tagx="chapter-title" title="chapter-title">Plant diversity and evolution: genotypic and phenotypic variation in higher plants</span> In: <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Henry</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span>, ed. <span class="source" tagx="source" title="source">Biologist</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Cambridge</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">CABI Publishing</span> <span class="year" tagx="year" title="year">2005</span>; p <span class="fpage" tagx="fpage" title="fpage">45</span>– <span class="lpage" tagx="lpage" title="lpage">78</span>. </span> </li> <li tag="ref"> <a name="pone.0196473.ref031" /> <span class="label" tagx="label" title="label">31</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doerks</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Copley</span> <span class="given-names" tagx="given-names" title="given-names">RR</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schultz</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ponting</span> <span class="given-names" tagx="given-names" title="given-names">CP</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bork</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Systematic identification of novel protein domain families associated with nuclear functions </span>. <span class="source" tagx="source" title="source">Genome Research</span>. <span class="year" tagx="year" title="year">2002</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">47</span>– <span class="lpage" tagx="lpage" title="lpage">56</span>. <span class="comment">doi: <a href="https://doi.org/10.1101/gr.203201">10.1101/gr.203201</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11779830">11779830</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref032" /> <span class="label" tagx="label" title="label">32</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drescher</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ruf</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Calsa</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carrer</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bock</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span>. <span class="mixed-article-title" title="mixed-article-title">The two largest chloroplast genome-encoded open reading frames of higher plants are essential genes </span>. <span class="source" tagx="source" title="source">Plant Journal</span>. <span class="year" tagx="year" title="year">2002</span>; <span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">97</span>– <span class="lpage" tagx="lpage" title="lpage">104</span>. <span class="comment">doi: <a href="https://doi.org/10.1046/j.1365-313x.2000.00722.x">10.1046/j.1365-313x.2000.00722.x</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref033" /> <span class="label" tagx="label" title="label">33</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaila</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chaduvla</span> <span class="given-names" tagx="given-names" title="given-names">PK</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saxena</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bahadur</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gahukar</span> <span class="given-names" tagx="given-names" title="given-names">SJ</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chaudhury</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Chloroplast genome sequence of Pigeonpea ( <i>Cajanus cajan</i> (L.) Millspaugh) and <i>Cajanus scarabaeoides</i> (L.) Thouars: Genome organization and comparison with other Legumes </span>. <span class="source" tagx="source" title="source">Frontier in Plant Science</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">1847</span> <span class="comment">doi: <a href="https://doi.org/10.3389/fpls.2016.01847">10.3389/fpls.2016.01847</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28018385">28018385</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref034" /> <span class="label" tagx="label" title="label">34</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raman</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Analysis of the complete chloroplast genome of a medicinal plant, <i>Dianthus superbus var</i>. <i>longicalyncinus</i>, from a comparative genomics perspective </span>. <span class="source" tagx="source" title="source">PLoS ONE</span>. <span class="year" tagx="year" title="year">2015</span>’ <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">e0141329</span> <span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0141329">10.1371/journal.pone.0141329</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26513163">26513163</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref035" /> <span class="label" tagx="label" title="label">35</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williams</span> <span class="given-names" tagx="given-names" title="given-names">AV</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boykin</span> <span class="given-names" tagx="given-names" title="given-names">LM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Howell</span> <span class="given-names" tagx="given-names" title="given-names">KA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nevill</span> <span class="given-names" tagx="given-names" title="given-names">PG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Small</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Correction: The complete sequence of the <i>Acacia ligulata</i> chloroplast genome reveals a highly divergent <i>clpP1</i> gene </span>. <span class="source" tagx="source" title="source">PLoS ONE</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">e0125768</span> <span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0125768">10.1371/journal.pone.0125768</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25955637">25955637</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref036" /> <span class="label" tagx="label" title="label">36</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">RS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiu</span> <span class="given-names" tagx="given-names" title="given-names">YX</span> </span>. <span class="mixed-article-title" title="mixed-article-title">The complete chloroplast genomes of three Cardiocrinum (Liliaceae) species: Comparative genomic and phylogenetic analyses </span>. <span class="source" tagx="source" title="source"> <i>Front</i> Plant Science </span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">2054</span> <span class="comment">doi: <a href="https://doi.org/10.3389/fpls.2016.02054">10.3389/fpls.2016.02054</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28119727">28119727</a> </span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref037" /> <span class="label" tagx="label" title="label">37</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Studies on peroxidase isozymes in <i>cerasus humilis</i> (Bge.) Sok </span>. <span class="source" tagx="source" title="source">Acta Horticulturae Sinica</span>. <span class="year" tagx="year" title="year">1994</span>, <span class="volume" tagx="volume" title="volume">1</span>: <span class="fpage" tagx="fpage" title="fpage">94</span>– <span class="lpage" tagx="lpage" title="lpage">96</span> </span> </li> <li tag="ref"> <a name="pone.0196473.ref038" /> <span class="label" tagx="label" title="label">38</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Daniell</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chang</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span>. <span class="mixed-article-title" title="mixed-article-title">Chloroplast genomes: diversity, evolution, and applications in genetic engineering</span>. <span class="source" tagx="source" title="source">Genome Biology</span>. <span class="year" tagx="year" title="year">2016</span>, <span class="volume" tagx="volume" title="volume">17</span>: <span class="fpage" tagx="fpage" title="fpage">134</span> <span class="comment">doi: <a href="https://doi.org/10.1186/s13059-016-1004-2">10.1186/s13059-016-1004-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27339192">27339192</a> </span> </span> </li> </ul> </div> </div>  </body></html>