<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Sci Rep</span> <span class="iso-abbrev" title="iso-abbrev">Sci Rep</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5575236">5575236</a> </span> <span class="publisher-id" title="publisher-id">10434</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-017-10434-7">10.1038/s41598-017-10434-7</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
The extraordinary variation of the organellar genomes of the Aneura pinguis revealed advanced cryptic speciation of the early land plants </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Myszczyński</span> <span class="given-names" tagx="given-names" title="given-names">Kamil</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">kamil.myszczynski@gmail.com</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bączkiewicz</span> <span class="given-names" tagx="given-names" title="given-names">Alina</span> </span> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buczkowska</span> <span class="given-names" tagx="given-names" title="given-names">Katarzyna</span> </span> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ślipiko</span> <span class="given-names" tagx="given-names" title="given-names">Monika</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Szczecińska</span> <span class="given-names" tagx="given-names" title="given-names">Monika</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sawicki</span> <span class="given-names" tagx="given-names" title="given-names">Jakub</span> </span> <a href="#Aff1">1</a> </span> <span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0001 2149 6795</span> <span class="institution-id" title="institution-id">grid.412607.6</span> <span class="institution" title="institution">Department of Botany and Nature Protection, </span> <span class="institution" title="institution">University of Warmia and Mazury, </span> </div> </span> <span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0001 2097 3545</span> <span class="institution-id" title="institution-id">grid.5633.3</span> <span class="institution" title="institution">Department of Biology, </span> <span class="institution" title="institution">Institute of Experimental Biology, Adam Mickiewicz University in Poznań, </span> </div> </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-8-8</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2017-8-8</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2017</span> </span> <span class="volume" tagx="volume" title="volume">7</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">9804</span> <span class="history" title="history"> <span class="received" title="received">received: 2016-12-12</span> <span class="accepted" title="accepted">accepted: 2017-8-9</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p> <i>Aneura pinguis</i> is known as a species complex with several morphologically indiscernible species, which are often reproductively isolated from each other and show distinguishable genetic differences. Genetic dissimilarity of cryptic species may be detected by genomes comparison. This study presents the first complete sequences of chloroplast and mitochondrial genomes of six cryptic species of <i>A. pinguis</i> complex: <i>A. pinguis</i> A, B, C, E, F, J. These genomes have been compared to each other in order to reconstruct phylogenetic relationships and to gain better understanding of the evolutionary process of cryptic speciation in this complex. The chloroplast genome with the nucleotide diversity 0.05111 and 1537 indels is by far more variable than mitogenome with π value 0.00233 and number of indels 1526. Tests of selection evidenced that on about 36% of chloroplast genes and on 10% of mitochondrial genes of <i>A. pinguis</i> acts positive selection. It suggests an advanced speciation of species. The phylogenetic analyses based on genomes show that <i>A. pinguis</i> is differentiated and forms three distinct clades. Moreover, on the cpDNA trees, <i>Aneura mirabilis</i> is nested among the cryptic species of <i>A. pinguis</i>. This indicates that the <i>A. pinguis</i> cryptic species do not derive directly from one common ancestor. </p> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2017" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Complexes of cryptic species are groups of related species that are virtually identical or morphologically very similar, with unclear morphological boundaries between them. On the other hand, cryptic species often are reproductively isolated from each other and show distinguishable genetic differences, as great as those observed in taxonomically distinct species, with clear morphological differences <sup> <a href="#CR1">1</a>– <a href="#CR3">3</a> </sup>. Complexes of cryptic species occur in many groups of organisms, for instance, in bryophytes <sup> <a href="#CR4">4</a> </sup>, which include three early diverging lineages of land plants – liverworts, mosses, and hornworts. In the bryophyte group, cryptic speciation was detected in many seemingly widespread species such as, e.g.: <i>Mielichhoferia elongata</i> (Hoppe &amp;amp; Hornsch) Ness &amp;amp; Hornsch <sup> <a href="#CR2">2</a> </sup>, <i>Hamatocaulis vernicosus</i> (Mitt.) Hedenäs <sup> <a href="#CR5">5</a> </sup>, <i>Platyh</i> <b> <i>y</i> </b> <i>pnidum riparioides</i> (Hedw.) Dixon <sup> <a href="#CR6">6</a> </sup>, in the genera <i>Orthotrichum</i> (Hedw.) Sawicki <sup> <a href="#CR7">7</a> </sup> and <i>Scleropodium</i> (Bruch &amp;amp; Schimp) <sup> <a href="#CR8">8</a> </sup>, <i>Targonia lorbeeriana</i> <sup> <a href="#CR9">9</a> </sup>, in the genus of <i>Herbertus</i> <sup> <a href="#CR10">10</a> </sup>, <i>Ptilidium ciliare</i> (L.) Hampe <sup> <a href="#CR11">11</a> </sup>, <i>Porella platyphylla</i> (L.) Pfeiff. <sup> <a href="#CR12">12</a> </sup>, <i>Frulania tamarisci</i> (L.) Dumort <sup> <a href="#CR13">13</a> </sup> and in <i>Aneura pinguis</i> (L.) Dumort <sup> <a href="#CR3">3</a>, <a href="#CR14">14</a> </sup>. </p> <p>Genetic differences between species in the complexes of cryptic species may be detected by comparing their genomes. Next-generation sequencing enables sequencing complete genomes instead of a few genes and detecting the position of genes in the genome and differences between the nucleotides in genes. Also, by comparing mitochondrial or chloroplast genomes of cryptic species, it is possible to observe which genes are important for the speciation and evolution of bryophytes and which genes are subject to selection. However, only a few complete mitochondrial <sup> <a href="#CR15">15</a>– <a href="#CR19">19</a> </sup> and chloroplast <sup> <a href="#CR20">20</a>– <a href="#CR22">22</a> </sup> genomes have been published for bryophytes. Comparative analyses of mtDNA and cpDNA genomes in this group showed that bryophytes possess similar gene order and content <sup> <a href="#CR16">16</a>– <a href="#CR18">18</a> </sup>. Moreover, it is found that organelle genomes in bryophytes evolve slowly and show similarity to organelle genomes of charophyte algae apart from some features indicating dynamics, such as: moderate increase in genome size, large-scale intron gains and occurrence of RNA editing <sup> <a href="#CR23">23</a>, <a href="#CR24">24</a> </sup>. In contrast, bryophyte genomes are dramatically different from the genomes of angiosperm plants that exhibit dynamic evolution of many features <sup> <a href="#CR17">17</a>, <a href="#CR25">25</a>, <a href="#CR26">26</a> </sup>. </p> <p> <i>A. pinguis</i> is a thalloid liverwort with a simple morphological and anatomical structure and broad holarctic distribution – it ranges from Europe, Asia, Australia and New Zealand to North America and Mexico <sup> <a href="#CR27">27</a>, <a href="#CR28">28</a> </sup>. It is common from lowlands up to the high mountain zone and grows in various habitats: on calcareous rocks, basic humus, peat bogs, wet sand on lake shores, and fallen decorticated logs <sup> <a href="#CR29">29</a> </sup>. It is now known as a species complex with several morphologically indistinguishable species temporarily named: <i>A. pinguis</i> species from A to L <sup> <a href="#CR3">3</a>, <a href="#CR14">14</a>, <a href="#CR30">30</a> </sup>. Genetic differences between them are clear and species may be distinguished by isozyme markers, ISSR markers and DNA barcodes <sup> <a href="#CR3">3</a>, <a href="#CR14">14</a> </sup>. </p> <p>In this study, is presented the first complete sequences of chloroplast and mitochondrial genomes of six cryptic species of <i>A. pinguis</i> complex: <i>A. pinguis</i> A, B, C, E, F and J. Additionally, these genomes have been compared to each other in order to reconstruct phylogenetic relationships between the studied species and to gain a better understanding of the evolutionary process of cryptic speciation in this complex. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="organellargenomesequencingandannotation" title="sec"> <div class="title" tagx="title" title="title">
Organellar genome sequencing and annotation</div> <p>Total genomic DNA of six <i>Aneura pinguis</i> cryptic species was extracted from <i>in vitro</i> culture using the Zymo Plant/Seed DNA kit (Zymo, Hilden, Germany). Fresh thalli were ground with silica beads in a MiniBead-Beater tissue disruptor for 50 s, and they were subsequently processed using the manufacturer’s protocols. DNA quantity was estimated with the use of the Qubit fluorometer system (Invitrogen, Carlsbad, NM, USA) and the Quant-IT ds-DNA BR Assay kit (Invitrogen). </p> <p>A genomic library for MiSeq sequencing was developed with the use of the Nextera XT Kit. DNA in the amount of 1 ng was used in the procedure described in the Nextera XT protocol (Illumina, San Diego, CA, USA). The number and accuracy of libraries was verified with the use of primers whose sequences are given in the Sequencing Library qPCR Quantification Guide (Illumina). PCR reactions were performed in 20 µL of a reaction mixture containing 3 µL of library genomes, 1.0 µM of each primer, 1.5 mM MgCl <sub>2</sub>, 200 µL M dNTP (dATP, dGTP, dCTP, dTTP), 1 × PCR buffer and 1 U OpenExTaq polymerase (Open Exome, Warsaw, Poland). PCR reactions were performed under the following thermal conditions: (1) initial denaturation, 5 min at a temperature of 94 °C; (2) denaturation, 30 s at 94 °C; (3) annealing, 30 s at 52 °C; (4) elongation, 1 min at 72 °C and final elongation, 7 min at 72 °C. Stages 2–4 were repeated 34 times. The products of the PCR reaction were separated in the QIAxcel capillary electrophoresis system (Qiagen). Electrophoresis was performed using the QIAxcel High Resolution Kit with the 15–1000 bp alignment marker (Qiagen) and the 25–1000 bp DNA size marker (Qiagen). Standard OL500 settings were used as the electrophoresis program. Validated libraries were pooled according to the Nextera XT protocol. Genomic libraries were sequenced using the MiSeq 500v2 cartridge that supported the acquisition of 2 × 250 bp pair-end reads. The resulting reads were preliminarily assembled using the Velvet <i>de novo</i> assembler implemented in the Geneious R8 software <sup> <a href="#CR31">31</a> </sup>. First, the reads were cleaned by removing adaptor sequences and low quality reads with ambiguous sequences. Afterwards reads were assembled with k-mer length of 31 bp and minimum contig length of 100 bp along with other default parameters. Finally, produced contigs with similarity less than 70% to reference organellar genomes such as: <i>Aneura mirabilis, Pellia endiviifolia</i>, <i>Ptilidium pulcherrimum, Pleurozia purpurea, Treubia lacunosa</i> and <i>Marchantia polymorpha</i> were discarded from further analyses. </p> <p>The flow chart for the <i>in silico</i> construction of the <i>A. pinguis</i> organellar genomes was identical to that presented in the previous study <sup> <a href="#CR32">32</a> </sup>. </p> <p>The four junctions between single-copy segments and inverted repeats were confirmed using PCR-based product sequencing of the assembled genomes. Purified PCR products were sequenced in both directions using the ABI BigDye 3.1 Terminator Cycle Kit (Applied Biosystems, Foster City, CA, USA) and visualized with the ABI Prism 3130 Automated DNA Sequencer (Applied Biosystems). The sequences obtained with the Sanger method were aligned with the assembled genomes using the Geneious R8 assembly software <sup> <a href="#CR31">31</a> </sup> to check for any differences. </p> <p>The assembled organellar genomes were annotated with DOGMA <sup> <a href="#CR33">33</a> </sup> and Geneious R8 <sup> <a href="#CR31">31</a> </sup>. The specimens and sequencing details with GenBank numbers are given in Table <a href="#Tab1">1</a>. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  </div></p><p>Sequencing details and specimens used in this study.</p>  <table> <thead> <tr> <th>Sample ID</th> <th>Cryptic species</th> <th>POZW voucher number</th> <th>Locality</th> <th>Habitat</th> <th>Geographic coordinates</th> <th>Collection date and collector</th> <th>Number of sequencing reads</th> <th>Mitogenome length [bp]</th> <th>Plastome length [bp]</th> <th>Mitogenome accession number</th> <th>Plastome accession number</th> </tr> </thead> <tbody> <tr> <td>T199-1</td> <td>A</td> <td>42826</td> <td>S Poland, Tatra Mts, NE slope of Skupniów Upłaz Mt, slope above road</td> <td>Humus over lime rocks</td> <td>49.16°N 20.00°E</td> <td>18.09.2012 AB, KB</td> <td>19,196,548</td> <td>121,105</td> <td>165,867</td> <td>KY702722</td> <td>KY702721</td> </tr> <tr> <td>A18-1</td> <td>B</td> <td>42793</td> <td>Poland, Bieszczady Mts, valley of Beskidnik stream, sparsely used road along the steam</td> <td>Clay soil</td> <td>49.08°N 20.29°E</td> <td>10.08.2012 KB</td> <td>1,037,898</td> <td>121,140</td> <td>164,047</td> <td>KU140427</td> <td>KY242383</td> </tr> <tr> <td>T178-1</td> <td>C</td> <td>42819</td> <td>Poland, Tatra Mts, Sucha Woda Valley, slope above road along the stream</td> <td>Humus</td> <td>49.28°N 20.03°E</td> <td>22.09.2012 AB, KB</td> <td>4,075,008</td> <td>120,927</td> <td>164,989</td> <td>NC026901</td> <td>KY242382</td> </tr> <tr> <td>T184-3</td> <td>E</td> <td>42815</td> <td>S Poland, Tatra Mts, Valley of Biały Potok stream</td> <td>Limestone rock in flowing water</td> <td>49.16°N 19.57°E</td> <td>16.09.2012 KB, AB</td> <td>17,743,042</td> <td>120,698</td> <td>167,033</td> <td>KY702723</td> <td>KY702720</td> </tr> <tr> <td>BS1-3</td> <td>F</td> <td>42818</td> <td>Poland, Beskid Sądecki Mts, Kozłecki stream, moist slopes of the stream</td> <td>Clay soil</td> <td>49.26°N 20.26°E</td> <td>3.09.2012 KB</td> <td>3,793,416</td> <td>120,898</td> <td>164,984</td> <td>KR817582</td> <td>KY242384</td> </tr> <tr> <td>JAP1-1</td> <td>J</td> <td>41053</td> <td>Japan, Mount Lide, N slope</td> <td>Humus</td> <td>37.86°N 139.76°E</td> <td>10.07.2005 MI</td> <td>4,166,004</td> <td>120,898</td> <td>164,699</td> <td>KZ242386</td> <td>KY242385</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Collectors: KB - Katarzyna Buczkowska, AB – Alina Bączkiewicz, MI – Misao Itouga.</p> </div>  <p /> </div> <div class="polymorphismanalyses" title="sec"> <div class="title" tagx="title" title="title">
Polymorphism analyses</div> <p>Chloroplast genomes of six cryptic species of <i>Aneura pinguis</i> were aligned using the MAFFT genome aligner <sup> <a href="#CR34">34</a> </sup>. Afterwards based on alignment of genomes polymorphism analysis was conducted separately for each coding sequence, intron and intergenic spacer. Every variation within aforementioned regions was identified as single nucleotide polymorphism (SNP) or insertion/deletion (indel) and counted using custom Python script. The nucleotide diversity values (π) were calculated for each coding and noncoding region using MEGA7 software <sup> <a href="#CR35">35</a> </sup>. Because the nucleotide diversity is based only on substitutions, number of indels and percent of polymorphic sites value <sup> <a href="#CR36">36</a> </sup> are given for each region, however section ‘Results and Discussion’ refers only to the π value. Each SNP within coding sequence was tested if it affects the protein sequence and defined as synonymous or nonsynonymous SNP. Finally, variations were visualized using Circos software <sup> <a href="#CR37">37</a> </sup> combined with Python script. The number of synonymous differences per synonymous site (dS), the number of nonsynonymous differences per nonsynonymous site (dN) and dN/dS ratio <sup> <a href="#CR38">38</a> </sup> were also calculated for CDSs (Supplementary Table <a href="#MOESM1">S2</a>). For each gene with dN/dS ratio larger than 1, codon-based Z-test of selection was done. The above evolutionary analyses were conducted in MEGA7 <sup> <a href="#CR35">35</a> </sup>. Using RAxML tree (100 bootstrap replications) and alignment of all liverworts species mentioned in Supplementary Table <a href="#MOESM1">S1</a>, we carried out branch-site statistical test for positive selection in HyPhy <sup> <a href="#CR39">39</a> </sup> using the BUSTED algorithm <sup> <a href="#CR40">40</a> </sup>. The same polymorphism analyses strategy was used also for mitochondrial genomes. For comparison of mt and cp genomes and to examine if they are consistent with neutral evolution, the HKA test was done <sup> <a href="#CR41">41</a> </sup>. As outgroups mitogenome and plastome of <i>Marchantia polymorpha</i> were used. The HKA test was conducted in DnaSp v5 software <sup> <a href="#CR42">42</a> </sup>. </p> </div> <div class="analysisofprotein-codinggenes" title="sec"> <div class="title" tagx="title" title="title">
Analysis of protein-coding genes</div> <p>Protein-coding genes were predicted based on the closest known genomes of related species to the <i>Aneura pinguis</i> i.e. <i>Aneura mirabilis, Pellia endiviifolia</i>, <i>Ptilidium pulcherrimum</i> for plastome and <i>Pleurozia purpurea, Treubia lacunosa</i> and <i>Marchantia polymorpha</i> for mitogenome. Predictions were made using Geneious R8 software <sup> <a href="#CR31">31</a> </sup> and BLAST tool <sup> <a href="#CR43">43</a> </sup>. </p> </div> <div class="phylogenomicsreconstruction" title="sec"> <div class="title" tagx="title" title="title">
Phylogenomics reconstruction</div> <p>Bayesian inference and maximum likelihood methods were applied to infer phylogenetic relationships. To construct phylogenetic trees 19 mitochondrial and 17 plastid genomes of bryophytes were used. The specimens and taxonomy status details with GenBank numbers are given in Supplementary Table <a href="#MOESM1">S1</a>. <i>Marchantia polymorpha</i> genomes were used as root of generated phylogenetic trees. Bayesian inference phylogenetic analyses were estimated for 1,500,000 generations, sampling one out of every 100 generations of random trees using MrBayes v. 3.2.6 <sup> <a href="#CR44">44</a> </sup>. Maximum likelihood analyses were conducted using RAxML <sup> <a href="#CR45">45</a> </sup> where the stability of clades was assessed by 100 bootstrap replications. Beside whole genome sequences, phylogenetic reconstructions were also performed for partitioned dataset including protein-coding genes. The ML trees were calculated separately for synonymous and nonsynonymous sites using HyPhy software <sup> <a href="#CR39">39</a> </sup>. </p> <p>Relative-rate test was conducted using Phyltest 2.0 software <sup> <a href="#CR46">46</a> </sup>. This two-cluster test <sup> <a href="#CR47">47</a> </sup> examines the constancy of evolutionary rate for two lineages where an outgroup lineage is given. Because of input data limit only plastomic and mitogenomic protein-coding sequences were used for relative-rate test. Phyltest allows multiple sequences to be included in each of the lineages. In our dataset <i>A. pinguis</i> is represented by six different mitogenomic and plastomic protein-coding sequences. Poisson correction distance was used for relative-rate comparisons of protein sequences. Species selected for relative-rate test are shown in Table <a href="#Tab2">2</a>. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  </div></p><p>Results of relative-rate test for <i>A. pinguis</i> and other species lineages. </p>  <table> <thead> <tr> <th>Genome</th> <th>Species</th> <th>Outgroup</th> <th>La-Lb</th> <th>Evolutionary rate</th> <th>Z</th> </tr> </thead> <tbody> <tr> <td> mt</td> <td> <i>Pleurozia purpurea</i> </td> <td> <i>Treubia lacunosa</i> </td> <td>0.0810309</td> <td>+</td> <td> <b>27.9378</b> </td> </tr> <tr> <td> mt</td> <td> <i>Treubia lacunosa</i> </td> <td> <i>Marchantia polymorpha</i> </td> <td>0.0873281</td> <td>+</td> <td> <b>24.0475</b> </td> </tr> <tr> <td> mt</td> <td> <i>Tetraphis pellucida</i> </td> <td> <i>Marchantia polymorpha</i> </td> <td>−0.00486008</td> <td>−</td> <td>1.00038</td> </tr> <tr> <td> mt</td> <td> <i>Orthotrichum stellatum</i> </td> <td> <i>Marchantia polymorpha</i> </td> <td>−0.00653954</td> <td>−</td> <td>1.30667</td> </tr> <tr> <td> mt</td> <td> <i>Syntrichia filaris</i> </td> <td> <i>Marchantia polymorpha</i> </td> <td>−0.00451914</td> <td>−</td> <td>0.910135</td> </tr> <tr> <td> <b>cp</b> </td> <td> <i>Aneura mirabilis</i> </td> <td> <i>Ptilidium pulcherrimum</i> </td> <td>−0.0472961</td> <td>−</td> <td>−*</td> </tr> <tr> <td> cp</td> <td> <i>Ptilidium pulcherrimum</i> </td> <td> <i>Pellia endiviifolia</i> </td> <td>0.0926284</td> <td>+</td> <td> <b>25.0532</b> </td> </tr> <tr> <td> cp</td> <td> <i>Pellia endiviifolia</i> </td> <td> <i>Marchantia polymorpha</i> </td> <td>0.0833034</td> <td>+</td> <td> <b>21.8773</b> </td> </tr> <tr> <td> cp</td> <td> <i>Physcomitrella patens</i> </td> <td> <i>Marchantia polymorpha</i> </td> <td>0.033927</td> <td>+</td> <td> <b>7.9098</b> </td> </tr> <tr> <td> cp</td> <td> <i>Takakia lepidozioides</i> </td> <td> <i>Marchantia polymorpha</i> </td> <td>−0.0135805</td> <td>−</td> <td> <b>2.94158</b> </td> </tr> <tr> <td> cp</td> <td> <i>Nyholmiella obtusifolia</i> </td> <td> <i>Marchantia polymorpha</i> </td> <td>0.0351483</td> <td>+</td> <td> <b>8.18193</b> </td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>La and Lb are the average numbers of substitutions per site (branch lengths) from the common ancestor (outgroup) of cluster A and B. "+" - faster evolutionary rate of protein-coding sequences of <i>A. pinguis</i> than that of the compared species, "−" - slower rate of evolution of <i>A. pinguis</i>. The bold font in Z column depict the statistically significant differences in evolutionary rates at the 5% level. "*" - two-tailed test cannot be computed probably because of substantial genetic differences between <i>A. mirabilis</i> and other bryophytes (besides <i>A. pinguis</i>). </p> </div>  <p /> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results and Discussion</div> <div class="structureandpolymorphismofchloroplastgenome" title="sec"> <div class="title" tagx="title" title="title">
Structure and polymorphism of chloroplast genome</div> <p>The chloroplast genomes of six cryptic species of <i>Aneura pinguis</i> length range from 120,698 to 121,140 bp and display the typical structure of most land plants, consisting of a pair of IRs (each of 8,575 bp) separated by LSC (83,632 bp) and SSC (20,145 bp) regions (Fig. <a href="#Fig1">1A</a>). Overall GC content of the cpDNA is 39.3%, which is similar to other known Jungermanniopsida class chloroplast genomes (33–41%) <sup> <a href="#CR48">48</a>, <a href="#CR49">49</a> </sup>. The IR gene content is identical to that of the liverworts <i>Aneura mirabilis</i>, <i>Pellia endiviifolia</i> and <i>Ptilidium pulcherrimum</i>, including the <i>trn</i>V-GAC, <i>rrn</i>16, <i>trn</i>I-GAU, <i>trn</i>A-UGC, <i>rrn</i>23, <i>rrn</i>4.5, <i>rrn</i>5, <i>trn</i>R-ACG and <i>trn</i>N-GUU genes. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>Genome maps of <i>Aneura pinguis</i> chloroplast ( <b>A</b>) and mitochondrial ( <b>B</b>) genomes. OGDraw genome maps <sup> <a href="#CR72">72</a> </sup>. Genes in the clockwise direction are on the inside of the map, and genes in the counterclockwise direction are on the outside of the map. Annotated genes are colored according to the functional categories shown in the legend. The inner circle visualizes GC content with midpoint line which indicates 50% GC content. Histogram presents GC content per 50 bp. </p>   </div> <p /> <p>Of the 122 unique genes (i. e., including one copy of the inverted repeats) identified in <i>Aneura pinguis</i> plastome, 81 are protein coding genes, 5 genes of unknown function ( <i>ycf</i> genes), 4 ribosomal RNAs and 32 transfer RNAs (Table <a href="#Tab3">3</a>). Ten genes such as: <i>atp</i>F, <i>ndh</i>A, <i>ndh</i>B, <i>pet</i>B, <i>pet</i>D, <i>rpl</i>2, <i>rpl</i>16, <i>rpo</i>C1, <i>rps</i>12 and <i>ycf</i>3 contain one intron and <i>clp</i>P contains two introns. The IRs and LSC gene content is identical to that of closely related <i>Aneura mirabilis</i>, the only member of Aneuraceae family with known chloroplast genome structure <sup> <a href="#CR49">49</a> </sup>. However, SSC gene content of <i>Aneura mirabilis</i> lacks such genes as <i>ndh</i>A, <i>ndh</i>G, <i>ndh</i>H and <i>ndh</i>I in comparison with <i>Aneura pinguis</i>. These chlororespiratory genes encode subunits of the NADH dehydrogenase complex in plant chloroplast genomes and play a role in photosynthesis <sup> <a href="#CR50">50</a> </sup>. The <i>ndh</i> genes are considered to be lacking or nonfunctional in heterotrophic plants <sup> <a href="#CR51">51</a> </sup> and <i>Aneura mirabilis</i> is the only bryophyte species known to receive carbon from other source than photosynthesis <sup> <a href="#CR52">52</a> </sup>. Therefore <i>Aneura pinguis</i> plastome is 12,920 bp longer than that of <i>Aneura mirabilis</i> (108,007 bp). <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3</span>  </div></p><p>Genes contained within the chloroplast and mitochondrial genome of <i>Aneura pinguis</i>. </p>  <table> <thead> <tr> <th /> <th>Gene names</th> <th>Type of gene</th> </tr> </thead> <tbody> <tr> <td> <b>cpDNA</b> </td> <td>accD</td> <td>Acetyl-CoA carboxylase</td> </tr> <tr> <td>atpA, atpB, atpE, atpF, atpH, atpI</td> <td>ATP synthase</td> </tr> <tr> <td>clpP</td> <td>Clp protease</td> </tr> <tr> <td>petA, petB, petD, petG, petL, petN</td> <td>Cytochrome b/f complex</td> </tr> <tr> <td>ccsA</td> <td>Cytochrome c biogenesis protein</td> </tr> <tr> <td>cemA</td> <td>Envelope membrane protein</td> </tr> <tr> <td>ycf1, ycf2, ycf3, ycf4, ycf12</td> <td>Hypothetical genes</td> </tr> <tr> <td>chlB, chlL, chlN</td> <td>Light-independent protochlorophyllide oxidoreductase subunit</td> </tr> <tr> <td>matK</td> <td>Maturase</td> </tr> <tr> <td>ndhA, ndhB, ndhC, ndhD, ndhE, ndhF, ndhG, ndhH, ndhI, ndhJ, ndhK</td> <td>NADH dehydrogenase</td> </tr> <tr> <td>psaA, psaB, psaC, psaI, psaJ, psaM</td> <td>Photosystem I</td> </tr> <tr> <td>psbA, psbB, psbC, psbD, psbE, psbF, psbH, psbI, psbJ, psbK, psbL, psbM, psbN, psbT, psbZ </td> <td>Photosystem II</td> </tr> <tr> <td>rpl2, rpl14, rpl16, rpl20, rpl21, rpl22, rpl23, rpl32, rpl33, rpl36</td> <td>Large ribosomal protein units</td> </tr> <tr> <td>rps2, rps3, rps4, rps7, rps8, rps11, rps12, rps14, rps15, rps18, rps19</td> <td>Small ribosomal protein units</td> </tr> <tr> <td>rrn4.5, rrn5, rrn16, rrn23</td> <td>Ribosomal RNAs</td> </tr> <tr> <td>rpoA, rpoB, rpoC1, rpoC2</td> <td>RNA polymerase</td> </tr> <tr> <td>rbcL</td> <td>Rubisco large subunit</td> </tr> <tr> <td>cysA, cysT</td> <td>Sulphate ABC transporter subunit</td> </tr> <tr> <td>trnA-UGC, trnC-GCA, trnD-GUC, trnE-UUC, trnF-GAA, trnfM-CAU, trnG-GCC, trnG-UCC, trnH-GUG, trnI-CAU, trnI-GAU, trnK-UUU, trnL-CAA, trnL-UAA, trnM-CAU, trnN-GUU, trnP-UGG, trnQ-UUG, trnR-ACG, trnR-CCG, trnR-UCU, trnS-GCU, trnS-UGA, trnT-GGU, trnT-UGU, trnV-GAC, trnV-UAC, trnW-CCA, trnY-GUA </td> <td>Transfer RNAs</td> </tr> <tr> <td>infA</td> <td>Translational initiation factor</td> </tr> <tr> <td> <b>mtDNA</b> </td> <td>atp1, atp4, atp6, atp8, atp9</td> <td>ATP synthase</td> </tr> <tr> <td>ccmB, ccmC, ccmFC, ccmFN</td> <td>Cytochrome c biogenesis</td> </tr> <tr> <td>cox1, cox2, cox3</td> <td>Cytochrome c oxidase</td> </tr> <tr> <td>cob</td> <td>Cytochrome c reductase</td> </tr> <tr> <td>rpl2, rpl5, rpl6</td> <td>Large ribosomal protein units</td> </tr> <tr> <td>nad1, nad2, nad3, nad4, nad4L, nad5, nad6, nad9</td> <td>NADH dehydrogenase</td> </tr> <tr> <td>rrn5, rrn18, rrn26</td> <td>Ribosomal RNAs</td> </tr> <tr> <td>rps1, rps2, rps3, rps4, rps7, rps8, rps10, rps11, rps12, rps13, rps14, rps19</td> <td>Small ribosomal protein units</td> </tr> <tr> <td>sdh3, sdh4</td> <td>Succinate dehydrogenase</td> </tr> <tr> <td>trnA-UGC, trnC-GCA, trnC-UCC, trnD-GUC, trnE-UUC, trnF-GAA, trnG-GCC, trnG-UCC, trnH-GUG, trnI-CAU, trnK-UUU, trnL-CAA, trnL-UAA, trnL-UAG, trnM-CAU, trnMf-CAU, trnN-GUU, trnP-UGG, trnQ-UUG, trnR-ACG, trnR-UCU, trnS-GCU, trnS-UGA, trnV-UAC, trnW-CCA, trnY-GUA </td> <td>Transfer RNAs</td> </tr> <tr> <td>tatC</td> <td>Twin arginine subunit c</td> </tr> </tbody> </table>  <p /> <p>A total of 182 regions were identified in the chloroplast genomes of the analysed cryptic species: 86 CDSs, 12 introns and 84 intergenic spacers. 10,169 SNPs and 1,537 indels were found across whole plastome of <i>Aneura pinguis</i>. Among the 5,455 SNPs identified in coding sequences, 2,232 are nonsynonymous and 3,223 are synonymous. Also 54 indels were found inside coding sequences (Fig. <a href="#Fig2">2</a>). A mean value of π for plastome was 0.05111. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>Circos graph representing SNP and indel variation among plastomes of six cryptic species of <i>A. pinguis</i>. Track A shows nonsynonymous SNP occurence within genes. Track B and C represent identified SNP (light blue spots) and indel (purple spots) per 100 bp, respectively. Track D represents percent of SNPs per CDS length while track E represents percent of indels per CDS length. Black plot represents π value (maximum value = 0.06) for each CDS. Track F represents percent of SNPs per noncoding region length while track G represents percent of indels per noncoding region length. Black plot represents π value (maximum value = 0.2) for each noncoding region. </p>   </div> <p /> <p>In noncoding regions longer than 100 bp (to eliminate bias) the π value was the highest in <i>ycf</i>1 <i>-chl</i>N spacer (0.230) (Supplementary Table <a href="#MOESM1">S4</a>). In this relatively short fragment (142 bp) 36 SNPs and 19 indels occured. The second most polymorphic spacer (π = 0.149), between <i>pet</i>L and <i>pet</i>G genes, contained 57 SNPs and 21 indels while 194 bp long. The <i>rpl</i>14 <i>-rpl</i>16 spacer had slightly lower diversity (π = 0.131), however in this, 122 bp long spacer, only 15 SNPs and 9 indels occured. </p> <p>Among introns the highest diversity (π = 0.073) was present in intron 1 of <i>ndh</i>A gene (800 bp) with 104 SNPs and 26 insertions and deletions (Supplementary Table <a href="#MOESM1">S4</a>). <i>ndh</i>B intron 1 showed a similar level of diversity (π = 0.068), although it was much shorter (624 bp). This region contained 77 SNPs and 3 indels. The third most polymorphic intron (π = 0.061) was <i>clp</i>P intron 2. In this, 423 bp long fragment, 52 SNPs and 10 indels occured. </p> <p>In every coding region substitutions were detected. The nucleotide diversity varied from 0.014 to 0.062 (Supplementary Table <a href="#MOESM1">S2</a>). The most variable plastid genes were <i>ycf</i>1 and <i>ycf</i>2, coding proteins of unknown function. The <i>ycf</i>1 seems to be the most variable in the plastid genome and is repeatedly used in phylogenetic studies <sup> <a href="#CR53">53</a> </sup> as well as DNA barcode <sup> <a href="#CR54">54</a> </sup>. Recently <i>ycf</i>2 gene is also recommended for this type of analysis showing a promising level of variation <sup> <a href="#CR55">55</a>– <a href="#CR57">57</a> </sup>. In those genes respectively 389 and 671 SNPs were found including 255 and 442 nonsynonymous substitutions. For comparison, 29 nonsynonymous substitutions have been found in <i>ycf</i>1 of genus <i>Pulsatilla</i> <sup> <a href="#CR55">55</a> </sup>, but only 3 nonsynonymous substitutions have occurred in this gene of <i>Arabis alpina</i> <sup> <a href="#CR58">58</a> </sup>. Similarly, few nonsynonymous substitutions have been found in the <i>ycf</i>2, for instance: three in <i>Tortula ruralis</i> <sup> <a href="#CR59">59</a> </sup> or only one in <i>Tetraplodon fuegianus</i> <sup> <a href="#CR60">60</a> </sup>. In comparison, a variability of <i>ycf</i>1 and <i>ycf</i> 2 genes in <i>Aneura pinguis</i> is huge. The other genes of this group ( <i>ycf</i>4, <i>ycf</i>3 and <i>ycf</i>12) appeared to be much less polymorphic. A high level of nonsynonymous SNPs has also been identified in genes such as: <i>rpl</i>32 (11 SNPs, 7 nonsynonymous), <i>ndh</i>G (47 SNPs, 27 nonsynonymous), <i>mat</i>K (143 SNPs, 82 nonsynonymous), <i>psb</i>K (14 SNPs, 8 nonsynonymous), <i>rpl</i>20 (27 SNPs, 15 nonsynonymous), <i>rpl</i>33 (9 SNPs, 5 nonsynonymous). Analysis of nonsynonymous substitutions occurrence showed a group of 4 genes, which were identified as highly conserved: <i>psa</i>M, <i>psb</i>I, <i>psb</i>L, <i>psb</i>F. In their coding sequences nonsynonymous substitutions were not identified. Also other genes of <i>psa</i> and <i>psb</i> family showed little polymorphism, probably due to the importance of its function, which is encoding subunits of photosystem I and II <sup> <a href="#CR52">52</a> </sup>. </p> </div> <div class="structureandpolymorphismofmitochondrialgenome" title="sec"> <div class="title" tagx="title" title="title">
Structure and polymorphism of mitochondrial genome</div> <p>The mitochondrial genomes of six cryptic species of <i>Aneura pinguis</i> length range from 164,047 to 167,033 bp (Fig. <a href="#Fig1">1B</a>), which is slightly shorter than the closest related species with known mitogenome structure – <i>Pleurozia purpurea</i> (168,526 bp) <sup> <a href="#CR18">18</a> </sup>. The two other known mitochondrial genomes of the liverwort species, <i>Treubia lacunosa</i> and <i>Marchantia polymorpha</i>, differ more and are composed of 151,983 and 186,609 bp, respectively. Overall GC content of <i>Aneura pinguis</i> is 47.4%, which is similar to other liverworts (42–45%) <sup> <a href="#CR15">15</a>, <a href="#CR48">48</a> </sup>. </p> <p>68 unique genes have been identified in <i>Aneura pinguis</i> mitogenome, including 40 protein coding genes, 3 ribosomal RNAs and 25 transfer RNAs (Table <a href="#Tab3">3</a>). Nine genes such as: <i>atp</i>9, <i>nad</i>2, <i>nad</i>3, <i>nad</i>4, <i>nad</i>5, <i>rpl</i>2, <i>rps</i>14, <i>rrn</i>18 and <i>trn</i>S-GCU contain one intron, four genes: <i>atp</i>1, <i>cox</i>2, <i>cox</i>3, <i>nad</i>4L contain two introns, <i>cob</i> contains 3 introns and <i>cox</i>1 contains 9 introns. Gene order is identical to the three aforementioned liverworts mitogenomes. However one exception has been observed. The <i>nad</i>7 gene was not identified in the obtained mitochondrial genome. The earlier analyses conducted on bryophytes have shown that <i>nad</i>7 in hornworts and in most liverworts is missing or frequently occurs as partial pseudogene with degenerated structure <sup> <a href="#CR17">17</a>, <a href="#CR61">61</a>, <a href="#CR62">62</a> </sup> in contrast to mosses, in which apparently this gene is functional <sup> <a href="#CR63">63</a> </sup>. The only liverwort species with non-pseudogenised <i>nad</i>7 are <i>Haplomitrium</i> <sup> <a href="#CR62">62</a> </sup> and <i>Treubia</i> <sup> <a href="#CR48">48</a> </sup>, which form a common clade, Haplomitriopsida, regarded as a sister to the rest of the liverworts (Marchantiopsida and Jungermanniopsida) <sup> <a href="#CR64">64</a> </sup> having pseudogene <i>nad</i>7. The studied species <i>Aneura pinguis</i> (Jungermanniopsida) has a nonfunctional mitochondrial copy of <i>nad</i>7, which is in line with the above division of Marchantiophyta. Pseudogenization of this gene in <i>Aneura pinguis</i> seems to rely on the loss of exon 1 occurring in <i>nad</i>7 gene of <i>Treubia lacunosa</i> and in pseudogene of <i>Pleurozia purpurea</i>. Exons 2 and 3 are present and very similar to the exons 2 and 3 of <i>Treubia lacunosa</i>, to 80% and 94% respectively. The biggest changes occur in intron 2. This segment is 162 bp shorter than intron 2 of <i>Treubia lacunosa</i> and similar to it only to about 40%, whereas intron 1 of <i>Aneura pinguis</i> has 80% paired identity with the same segment in <i>Pleurozia purpurea</i>. </p> <p>103 regions have been identified in the mitochondrial genome of <i>Aneura pinguis</i>: 40 CDSs, 27 introns and 36 intergenic spacers. 953 SNPs and 1,940 indels have been identified in total. Among the 84 SNPs identified in coding sequences, 72 are nonsynonymous and 12 are synonymous. 43 indels were also found inside coding sequences (Fig. <a href="#Fig3">3</a>). A mean value of π for mitogenome was 0.00233. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>Circos graph representing SNP and indel variation among mitogenomes of six cryptic species of <i>A. pinguis</i>. Track A shows nonsynonymous SNP occurence within genes. Track B and C represent identified SNP (light blue spots) and indel (purple spots) per 100 bp, respectively. Track D represents percent of SNPs per CDS length while track E represents percent of indels per CDS length. Black plot represents π value (maximum value = 0.004) for each CDS. Track F represents percent of SNPs per noncoding region length while track G represents percent of indels per noncoding region length. Black plot represents π value (maximum value = 0.01) for each noncoding region. </p>   </div> <p /> <p>The nucleotide diversity value among noncoding regions longer than 100 bp was the greatest in <i>sdh</i>4 <i>-nad</i>4L spacer (π = 0.012407) (Supplementary Table <a href="#MOESM1">S5</a>). This spacer contained 5 SNPs and 14 indels while 412 bp long. The second most polymorphic spacer (π = 0.009675) is adjacent to <i>rps</i> gene family. The 482 bp <i>rps</i>14 <i>-rps</i>8 spacer comprised 9 SNPs and 15 indels. Another one of the most diverse regions is <i>nad</i>9 <i>-atp</i>1 spacer (π = 0.007692) with 4 SNPs and 8 indels on 390 bp long section. A slightly lower polymorphism (π = 0.006709) was found in <i>nad</i>5 <i>-nad</i>4 spacer. This 1,008 bp long region contained 12 SNPs and 26 indels. </p> <p>The highest diversity among introns (π = 0.006938) was identified in the first intron of <i>nad</i>5 gene (Supplementary Table <a href="#MOESM1">S5</a>). This intron contained 11 SNPs and 5 indels. The second most polymorphic intron (π = 0.006395) was <i>cox</i>2 intron 1. In this, 1061 bp long section, 13 SNPs and 13 indels occured. The intron 1 and intron 3 of <i>cob</i> gene showed a similar level of polymorphism (π = 0.004842, π = 0.004828 respectively) and took place the third position among the most polymorphic introns. </p> <p>The nucleotide diversity values in coding regions varied between 0 to 0.004087 (Supplementary Table <a href="#MOESM1">S3</a>). By far the most diverse mitogenome gene was <i>sdh</i>4 responsible for succinate dehydrogenase (π = 0.004087). In its coding sequence 2 SNPs were found and all of them were nonsynonymous substitutions. Similar level of polymorphism was detected in <i>tat</i>C gene coding sequence (π = 0.004082). In this region 5 SNPs were identified with 4 nonsynonymous SNPs. Whereas in other gene of <i>sdh</i> family – <i>sdh</i>3 no polymorphism was identified. The third most polymorphic coding sequence in mt genome was <i>rps</i>19 with 2 SNPs. High level of nucleotide diversity was also observed in <i>ccm</i>C gene, where all 4 SNPs were nonsynonymous. Other coding sequences of <i>Aneura pinguis</i> mitochondrial genome presented the π values below 0.003. A group of 8 CDSs contained neither SNPs nor indels. Those highly conserved coding sequences were: <i>atp</i>4, <i>ccm</i>B, <i>cox</i>3, <i>nad</i>4L, <i>rpl</i>6, <i>rps</i>1, <i>rps</i>10 and <i>rps</i>14. The last one was reported to be the most polymorphic as it revealed comparative mitogenomics between <i>Physcomitrella patens</i> and <i>Marchantia polymorpha</i> <sup> <a href="#CR65">65</a> </sup>. </p> <p>In comparison with the genome of the plastid, the mitochondrial genome is less variable. Generally, in the cp coding sequences of <i>Anuera</i> there are far more substitutions (5,455) than in mt sequences encode proteins (84). Furthermore, the mean value of nucleotide diversity per genome (cp: π = 0.05111; mt: π = 0.00233) also indicated that more permanent is mt genome. However, the total number of indels in both genomes is similar (cp: 1537; mt: 1526). The results are consistent with a common observation that mitochondrial genomes of bryophytes are stable, in contrast to mt chromosomes of angiosperms <sup> <a href="#CR66">66</a> </sup>. The HKA test evidenced that these two genomes of <i>Aneura</i> differ from one another in diversity level and that at least one of the mentioned genomes deviates from neutral expectation (χ <sup>2</sup> = 5.381; p = 0.0204, p &amp;lt; 0.05). The dN/dS ratio, calculated for each protein coding sequence of mt and cp genome, indicated which genes are probably under positive selection within <i>A. pinguis</i> complex. The Z-test showed, which values of dN/dS ratios are significantly larger than 1. In plastome 31 genes were under positive selection. For 24 genes the dN/dS ratio was significantly higher than 1 and in 7 genes only nonsynonymous substitutions occurred (Supplementary Table <a href="#MOESM1">S3</a>). The dN/dS ratio is also associated with the strength of selection. Values over 1 indicate positive selection, whereas much higher values indicate stronger selection <sup> <a href="#CR67">67</a> </sup>. The dN/dS ratio values of six of the chloroplast genes: <i>psa</i>C, <i>chl</i>L, <i>psb</i>N, <i>psb</i>E, <i>clp</i>P and <i>psb</i>K range between 3.8 and 29 (Supplementary Table <a href="#MOESM1">S2</a>). All of them are directly or indirectly associated with photosynthesis. </p> <p>Photosynthesis genes of <i>Arabis alpina</i> was also suggested to be under selection, but negative. In these protein coding sequences only synonymous substitutions were found <sup> <a href="#CR58">58</a> </sup>. Although mt genomes of bryophytes are quite conservative, in the case of mitogenome of <i>A. pinguis</i> the number of nonsynonymous substitutions in protein encoding genes was even 3–4 times greater than in the case of the families Orthotrichaceae <sup> <a href="#CR68">68</a> </sup>, or Grimmiaceae <sup> <a href="#CR69">69</a> </sup>. However, only four mitochondrial genes within studied <i>A. pinguis</i> complex display positive selection (Supplementary Table <a href="#MOESM1">S3</a>). Three of them play important role in the respiratory chain: <i>ccm</i>FC, <i>cox</i>1and <i>nad</i>1, while <i>rps</i>3 encodes small ribosomal protein unit. </p> <p>We also conducted branch-site test for positive selection among mitochondrial and chloroplast genes of liverworts (species specified in Supplementary Table <a href="#MOESM1">S1</a>) using BUSTED algorithm <sup> <a href="#CR40">40</a> </sup>. The test, combining diversity and divergence data, indicated that 3 mitochondrial and 9 chloroplast genes were under positive selection within liverworts clade (Supplementary Tables <a href="#MOESM1">S2</a> and <a href="#MOESM1">S3</a>). These results are consistent with aforementioned diversity analyses conducted among <i>A. pinguis</i> cryptic species supporting hypothesis that plastid genomes are more variable than mitochondrial genomes. Branch-site test depicted that on about 11% of chloroplast and 7% of mitochondrial genes of liverworts acts positive selection (Supplementary Tables <a href="#MOESM1">S2</a> and <a href="#MOESM1">S3</a>). </p> </div> <div class="phylogenomicsrelationships" title="sec"> <div class="title" tagx="title" title="title">
Phylogenomics relationships</div> <p>The phylogenetic trees based on chloroplast and mitochondrial genomes of 17 and 19 bryophytes based on Bayesian analysis are shown in Fig. <a href="#Fig4">4</a>. Both phylogenetic trees show that <i>Aneura pinguis</i> is differentiated and forms three distinct clades, very well being supported by Bayesian posterior probabilities (PP). The topology of trees generated by RAxML methods confirm the same relationships within the genus <i>Aneura</i> (Fig. <a href="#Fig4">4</a>). These analyses and many others studies indicated that <i>A. pinguis</i> is a paraphyletic taxon <sup> <a href="#CR49">49</a>, <a href="#CR70">70</a>, <a href="#CR71">71</a> </sup> consisting of several evolutionary lineages corresponding to the previously detected cryptic species of <i>A. pinguis</i> <sup> <a href="#CR3">3</a>, <a href="#CR70">70</a> </sup>. The partitioned datasets based on synonymous and nonsynonymous substitutions revealed trees of identical topology as whole genome analyses (Fig. <a href="#Fig5">5</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>Phylogenetic relationships among <i>Aneura pinguis</i> cryptic species and other bryophytes based on complete sequences of plastomes ( <b>A</b>) and complete sequences of mitogenomes ( <b>B</b>). Bayesian inference phylogenetic trees of the 17 complete plastome ( <b>A</b>) and 19 mitogenome ( <b>B</b>) sequences of bryophytes. All resolved clades have maximum values of posterior probabilities. In case of the maximum likelihood analysis the values lower than 100% of bootstrap support are given at the nodes. The moss clade of the complete mitogenome analysis is scaled (six times shortened) for better representation of the phylogram. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <p>Phylogenetic relationships among <i>Aneura pinguis</i> cryptic species and other bryophytes based on partitioned plastome ( <b>A</b> and <b>B</b>) and mitogenome ( <b>C</b> and <b>D</b>) protein-coding datasets. Maximum likelihood analysis of protein-coding dataset was performed separately for synonymous (plastome - A, mitogenome - C) and nonsynonymous (plastome - B, mitogenome - D) sites. All resolved clades have maximum values of the maximum likelihood bootstrap support. </p>   </div> <p /> <p>Relative-rate test revealed that both mitogenome and plastome of <i>A. pinguis</i> exhibits faster evolutionary rate than other tested liverworts, supporting our finding about extraordinary variation of the organellar genomes of this species within liverworts and formerly detected cryptic speciation of <i>A. pinguis</i> <sup> <a href="#CR3">3</a>, <a href="#CR70">70</a> </sup>. In comparison with moss species evolutionary rate of <i>A. pinguis</i> mitogenome did not differ significantly. However <i>A. pinguis</i> plastome evolutionary rate is higher than two of moss species ( <i>Physcomitrella patens</i> and <i>Nyholmiella obtusifolia</i>) and lower than one moss species - <i>Takakia lepidozioides</i> (Table <a href="#Tab2">2</a>). These results are in accordance with the stability of the bryophyte mitogenome hypothesis and, on the other hand, proved higher plasticity of the bryophytes chloroplast genome. </p> <p> <i>Aneura mirabilis</i> (previously belonging to the genus <i>Cryptothallus</i>) is a taxonomically uniform species in our studies. On the cpDNA trees, it is nested among the cryptic species of <i>A. pinguis</i> (Figs <a href="#Fig4">4a</a> and <a href="#Fig5">5a,b</a>). Moreover, molecular studies based both on coding ( <i>rbc</i>L, <i>mat</i>K, <i>rpo</i>C1) and noncoding ( <i>trn</i>L <i>-trn</i>F, <i>trn</i>H <i>-psb</i>A) barcode sequences also show that <i>A. mirabilis</i>, as well as <i>A. maxima</i>, are nested among the cryptic species of the <i>A. pinguis</i> complex (unpublished data). This indicates that the cryptic species of the <i>A. pinguis</i> complex do not derive directly from one common ancestor, but their evolutionary history is more complex. </p> </div> </div> <div class="conclusion" title="conclusion"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>Due to next-generation sequencing one is able to present the first complete sequences of chloroplast and mitochondrial genomes of six cryptic species of <i>A. pinguis</i> complex. This method enabled the identification of highly variable sequences that could potentially be used as DNA barcode (e.g <i>. ycf</i>1, <i>ycf</i>2 or <i>sdh</i>4). The results also show that the aforementioned organellar genomes are extraordinarily variable, especially the chloroplast genome. About 36% of chloroplast genes in <i>A. pinguis</i> is under positive selection (based on Z-test). Surprisingly, taking into account the stability of mitogenome in bryophytes, on 10% of mitochondrial genes also acts directional selection. Moreover branch-site test showed that on about 11% of chloroplast and 7% of mitochondrial genes of studied liverworts acts positive selection. The above results indicate an advanced speciation of species. These findings confirm phylogenomic analyses, which divide <i>A. pinguis</i> into three distinct clades. The inclusion of the plastome sequence of <i>A. mirabilis</i> revealed paraphyly of <i>A. pinguis</i> as the former resolved as a sister to the cryptic species C and F. <i>A. pinguis</i> has a broad distribution and grows in various habitats, which might induce higher positive selection. Ecological processes are essential for the formation of new species, when barriers to gene flow (reproductive isolation) evolve between populations as a result of ecologically-based divergent selection. Although there are numerous studies providing evidence for the presence of ecological speciation, further lines of research are needed to explore the mechanisms underlying this process. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41598_2017_10434_MOESM1_ESM.doc">LINK</a>  <p>Supplementary materials</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Electronic supplementary material</b> </p> <p> <b>Supplementary information</b> accompanies this paper at doi:10.1038/s41598-017-10434-7 </p> </div> <div class="fn-type-" title=""> <p> <b>Publisher's note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>K.M. assembled genome sequences, analyzed data, wrote the main manuscript text and prepared all figures, A.B. and K.B. collected samples, prepared material and performed the experiments, M.Ś. and M.S. analyzed data, wrote the paper, and J.S. oversaw the project, analyzed data, wrote the paper and reviewed drafts of the paper. </p> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing Interests</div> <p>The authors declare that they have no competing interests.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wyatt</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Population genetics of bryophytes in relation to their reproductive biology</span> <span class="source" tagx="source" title="source">J. Hattori Bot. Lab.</span> <span class="year" tagx="year" title="year">1994</span> <span class="volume" tagx="volume" title="volume">76</span> <span class="fpage" tagx="fpage" title="fpage">147</span> <span class="lpage" tagx="lpage" title="lpage">157</span> </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Shaw, A. J. Population ecology, population genetics, and microevolution in B ryophyte Biology (ed. Shaw, A. J. &amp;amp; Goffinet, B.) 369–402 (Cambridge University Press, 2000). </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bączkiewicz</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buczkowska</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Differentiation and genetic variability of three cryptic species within the <i>Aneura pinguis</i> complex ( <i>Jungermanniidae, Marchantiophyta</i>) </span> <span class="source" tagx="source" title="source">Cryptogam., Bryol.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">37</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">16</span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shaw</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Biogeographic patterns and cryptic speciation in bryophytes</span> <span class="source" tagx="source" title="source">J. Biogeogr.</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">253</span> <span class="lpage" tagx="lpage" title="lpage">261</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1365-2699.2001.00530.x">10.1046/j.1365-2699.2001.00530.x</a> </span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hedenäs</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eldenäs</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cryptic speciation, habitat differentiation, and geography in <i>Hamatocaulis vernicosus</i> (Calliergonaceae, Bryophyta) </span> <span class="source" tagx="source" title="source">Plant Syst. Evol.</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">268</span> <span class="fpage" tagx="fpage" title="fpage">131</span> <span class="lpage" tagx="lpage" title="lpage">145</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00606-007-0529-y">10.1007/s00606-007-0529-y</a> </span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huttunen</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ignatov</span> <span class="given-names" tagx="given-names" title="given-names">MS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evolution and taxonomy of aquatic species in the genus <i>Rhynchostegium</i> (Brachytheciaceae, Bryophyta) </span> <span class="source" tagx="source" title="source">Taxon</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">59</span> <span class="fpage" tagx="fpage" title="fpage">791</span> <span class="lpage" tagx="lpage" title="lpage">808</span> </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sawicki</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Plášek</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Szczecińka</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular data do not support the current division of <i>Orthotrichum</i> (Bryophyta) species with immersed stomata </span> <span class="source" tagx="source" title="source">Plant Syst. Evol.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">50</span> <span class="fpage" tagx="fpage" title="fpage">12</span> <span class="lpage" tagx="lpage" title="lpage">24</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1759-6831.2011.00168.x">10.1111/j.1759-6831.2011.00168.x</a> </span> </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carter</span> <span class="given-names" tagx="given-names" title="given-names">BE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Species delimitation and cryptic diversity in the moss genus <i>Scleropodium</i> (Brachytheciaceae) </span> <span class="source" tagx="source" title="source">Mol. Phylogenet. Evol.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">63</span> <span class="fpage" tagx="fpage" title="fpage">891</span> <span class="lpage" tagx="lpage" title="lpage">903</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ympev.2012.03.002">10.1016/j.ympev.2012.03.002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22421213">22421213</a> </span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boisselier-Dubayle</span> <span class="given-names" tagx="given-names" title="given-names">MC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bischler</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genetic relationships between haploid and triploid <i>Targonia</i> (Targoniaceae, Hepaticae) </span> <span class="source" tagx="source" title="source">Int. J. Plant Sci.</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">160</span> <span class="fpage" tagx="fpage" title="fpage">1163</span> <span class="lpage" tagx="lpage" title="lpage">1169</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1086/314199">10.1086/314199</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10568784">10568784</a> </span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feldberg</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Groth</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilson</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schäfer-Verwimp</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heinrichs</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cryptic speciation in <i>Herbertus</i> (Herbertaceae, Jungermanniopsida): range and morphology of <i>Herbertus sendtneri</i> inferred from nrITS sequences </span> <span class="source" tagx="source" title="source">Plant Syst. Evol.</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">249</span> <span class="fpage" tagx="fpage" title="fpage">247</span> <span class="lpage" tagx="lpage" title="lpage">261</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00606-004-0221-4">10.1007/s00606-004-0221-4</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kreier</span> <span class="given-names" tagx="given-names" title="given-names">HP</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogeny of the leafy liverwort <i>Ptilidium</i>: cryptic speciation and shared haplotypes between the Northern and Southern Hemispheres </span> <span class="source" tagx="source" title="source">Mol. Phylogenet. Evol.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">57</span> <span class="fpage" tagx="fpage" title="fpage">1260</span> <span class="lpage" tagx="lpage" title="lpage">1267</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ympev.2010.10.002">10.1016/j.ympev.2010.10.002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20950690">20950690</a> </span> </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heinrichs</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Formalizing morphologically cryptic biological entities: New insights from DNA taxonomy, hybridization, and biogeography in the leafy liverwort <i>Porella platyphylla</i> (Jungermanniopsida, Porellales) </span> <span class="source" tagx="source" title="source">Am. J. Bot.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">98</span> <span class="issue" tagx="issue" title="issue">8</span> <span class="fpage" tagx="fpage" title="fpage">1252</span> <span class="lpage" tagx="lpage" title="lpage">1262</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3732/ajb.1100115">10.3732/ajb.1100115</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21788532">21788532</a> </span> </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heinrichs</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">One species or at least eight? Delimitation and distribution of <i>Frullania tamarisci</i> (L.) Dumort. s. l. ( <i>Jungermanniopsida, Porellales</i>) inferred from nuclear and chloroplast DNA markers </span> <span class="source" tagx="source" title="source">Mol. Phylogenet. Evol.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">56</span> <span class="fpage" tagx="fpage" title="fpage">1105</span> <span class="lpage" tagx="lpage" title="lpage">1114</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ympev.2010.05.004">10.1016/j.ympev.2010.05.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20460161">20460161</a> </span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wawrzyniak</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wasiak</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bączkiewicz</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buczkowska</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Volatile compounds in cryptic species of the <i>Aneura pinguis</i> complex and <i>Aneura maxima</i> ( <i>Marchantiophyta, Metzgeriidae</i>) </span> <span class="source" tagx="source" title="source">Phytochemistry</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">105</span> <span class="fpage" tagx="fpage" title="fpage">115</span> <span class="lpage" tagx="lpage" title="lpage">122</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.phytochem.2014.06.010">10.1016/j.phytochem.2014.06.010</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25034615">25034615</a> </span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oda</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Gene organization deduced from the complete sequence of liverwort <i>Marchantia polymorpha</i> mitochondrial DNA—a primitive form of plant mitochondrial genome </span> <span class="source" tagx="source" title="source">J. Mol. Biol.</span> <span class="year" tagx="year" title="year">1992</span> <span class="volume" tagx="volume" title="volume">223</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">7</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0022-2836(92)90708-R">10.1016/0022-2836(92)90708-R</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1731062">1731062</a> </span> </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Terasawa</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The mitochondrial genome of the moss <i>Physcomitrella patens</i> sheds new light on mitochondrial evolution in land plants </span> <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">24</span> <span class="fpage" tagx="fpage" title="fpage">699</span> <span class="lpage" tagx="lpage" title="lpage">709</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msl198">10.1093/molbev/msl198</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17175527">17175527</a> </span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiu</span> <span class="given-names" tagx="given-names" title="given-names">Y-L</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The complete mitochondrial genome sequence of the hornwort Megaceros aenigmaticus shows a mixed mode of conservative yet dynamic evolution in early land plant mitochondrial genomes </span> <span class="source" tagx="source" title="source">J. Mol. Evol.</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">68</span> <span class="fpage" tagx="fpage" title="fpage">665</span> <span class="lpage" tagx="lpage" title="lpage">678</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00239-009-9240-7">10.1007/s00239-009-9240-7</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19475442">19475442</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xue</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiu</span> <span class="given-names" tagx="given-names" title="given-names">YL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The complete mitochondrial genome sequence of the liverwort <i>Pleurozia purpurea</i> reveals extremely conservative mitochondrial genome evolution in liverworts </span> <span class="source" tagx="source" title="source">Curr. Genet.</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">55</span> <span class="fpage" tagx="fpage" title="fpage">601</span> <span class="lpage" tagx="lpage" title="lpage">609</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00294-009-0273-7">10.1007/s00294-009-0273-7</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19756627">19756627</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Myszczyński</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The complete mitochondrial genome of the cryptic species C of <i>Aneura pinguis</i> </span> <span class="source" tagx="source" title="source">Mitochondrial DNA.</span> <span class="year" tagx="year" title="year">2015</span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sugiura</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kobayashi</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aoki</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sugita</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sugita</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Complete chloroplast DNA sequence of the moss <i>Physcomitrella patens</i>: evidence for the loss and relocation of <i>rpo</i>A from the chloroplast to the nucleus </span> <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="issue" tagx="issue" title="issue">18</span> <span class="fpage" tagx="fpage" title="fpage">5324</span> <span class="lpage" tagx="lpage" title="lpage">5331</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkg726">10.1093/nar/gkg726</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12954768">12954768</a> </span> </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kugita</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamamoto</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujikawa</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matsumoto</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoshinaga</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">RNA editing in hornwort chloroplasts makes more than half the genes functional</span> <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="fpage" tagx="fpage" title="fpage">2417</span> <span class="lpage" tagx="lpage" title="lpage">2423</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkg327">10.1093/nar/gkg327</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12711687">12711687</a> </span> </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolf</span> <span class="given-names" tagx="given-names" title="given-names">PG</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The first complete chloroplast genome sequence of a lycophyte, <i>Huperzia lucidula</i> (Lycopodiaceae) </span> <span class="source" tagx="source" title="source">Gene</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">117</span> <span class="lpage" tagx="lpage" title="lpage">128</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.gene.2005.01.018">10.1016/j.gene.2005.01.018</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turmel</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Otis</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lemieux</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The chloroplast and mitochondrial genome sequences of the charophyte <i>Chaetosphaeridium globosum</i>: Insights into the timing of the events that restructured organelle DNAs within the green algal lineage that led to land plants </span> <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">99</span> <span class="fpage" tagx="fpage" title="fpage">11275</span> <span class="lpage" tagx="lpage" title="lpage">11280</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.162203299">10.1073/pnas.162203299</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12161560">12161560</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turmel</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Otis</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lemieux</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The mitochondrial genome of <i>Chara vulgaris</i>: Insights into the mitochondrial DNA architecture of the last common ancestor of green algae and land plants </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">1888</span> <span class="lpage" tagx="lpage" title="lpage">1903</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.013169">10.1105/tpc.013169</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12897260">12897260</a> </span> </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gray</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lang</span> <span class="given-names" tagx="given-names" title="given-names">BF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burger</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mitochondria of protists</span> <span class="source" tagx="source" title="source">Annu Rev Genet</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">38</span> <span class="fpage" tagx="fpage" title="fpage">477</span> <span class="lpage" tagx="lpage" title="lpage">524</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev.genet.37.110801.142526">10.1146/annurev.genet.37.110801.142526</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15568984">15568984</a> </span> </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kubo</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Newton</span> <span class="given-names" tagx="given-names" title="given-names">KJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Angiosperm mitochondrial genomes and mutations</span> <span class="source" tagx="source" title="source">Mitochondrion</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">5</span> <span class="lpage" tagx="lpage" title="lpage">14</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.mito.2007.10.006">10.1016/j.mito.2007.10.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18065297">18065297</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Paton, J. A. <i>The liverwort flora of the British Isles</i> (Harley Books, Colchester, 1999). </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Damsholt, K. <i>Illustrated Flora of Nordic liverworts and hornworts</i> (Nordic Bryological Society, Lund, 2002). </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Szweykowski, J. An annotated checklist of Polish liverworts and hornworts in: <i>Biodiversity of</i> Poland (ed. Mirek, Z.). Vol. 4. (W. Szafer Institute of Botany, Polish Academy of Sciences, Cracow, 2006). </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buczkowska</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Adamczyk</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bączkiewicz</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Morphological and anatomical differentiation within the <i>Aneura pinguis</i> complex (Metzgeriales, Hepaticae) </span> <span class="source" tagx="source" title="source">Biol. Lett.</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">43</span> <span class="fpage" tagx="fpage" title="fpage">51</span> <span class="lpage" tagx="lpage" title="lpage">68</span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kearse</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Geneious Basic: An integrated and extendable desktop software platform for the organization and analysis of sequence data </span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">1647</span> <span class="lpage" tagx="lpage" title="lpage">1649</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/bts199">10.1093/bioinformatics/bts199</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22543367">22543367</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Szczecińska</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gomolińska</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Szkudlarz</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sawicki</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Plastid and nuclear genomic resources of a relict and endangered plant species: <i>Chamaedaphne calyculata</i> (L.) Moench (Ericaceae) </span> <span class="source" tagx="source" title="source">Turk. J. Bot.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">38</span> <span class="fpage" tagx="fpage" title="fpage">1229</span> <span class="lpage" tagx="lpage" title="lpage">1238</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3906/bot-1405-80">10.3906/bot-1405-80</a> </span> </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wyman</span> <span class="given-names" tagx="given-names" title="given-names">SK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jansen</span> <span class="given-names" tagx="given-names" title="given-names">RK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boore</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Automatic annotation of organellar genomes with DOGMA</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">20</span> <span class="issue" tagx="issue" title="issue">17</span> <span class="fpage" tagx="fpage" title="fpage">3252</span> <span class="lpage" tagx="lpage" title="lpage">3255</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/bth352">10.1093/bioinformatics/bth352</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15180927">15180927</a> </span> </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Katoh</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Standley</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability </span> <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">772</span> <span class="lpage" tagx="lpage" title="lpage">780</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/mst010">10.1093/molbev/mst010</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23329690">23329690</a> </span> </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stecher</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tamura</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">MEGA7: Molecular Evolutionary Genetics Analysis version 7.0 for bigger datasets</span> <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">33</span> <span class="fpage" tagx="fpage" title="fpage">1870</span> <span class="lpage" tagx="lpage" title="lpage">1874</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msw054">10.1093/molbev/msw054</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27004904">27004904</a> </span> </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shaw</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Chloroplast DNA sequence utility for lowest phylogenetic inferences in angiosperms: The tortoise and the hare IV </span> <span class="source" tagx="source" title="source">Am. J. Bot.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">101</span> <span class="issue" tagx="issue" title="issue">11</span> <span class="fpage" tagx="fpage" title="fpage">1987</span> <span class="lpage" tagx="lpage" title="lpage">2004</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3732/ajb.1400398">10.3732/ajb.1400398</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25366863">25366863</a> </span> </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krzywinski</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">An information Aesthetic for Comparative Genomics</span> <span class="source" tagx="source" title="source">Genome Res.</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">1639</span> <span class="lpage" tagx="lpage" title="lpage">1645</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.092759.109">10.1101/gr.092759.109</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19541911">19541911</a> </span> </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nei</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gojobori</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions </span> <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="year" tagx="year" title="year">1986</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">418</span> <span class="lpage" tagx="lpage" title="lpage">426</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/3444411">3444411</a> </span> </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kosakovsky Pond</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frost</span> <span class="given-names" tagx="given-names" title="given-names">SDW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Muse</span> <span class="given-names" tagx="given-names" title="given-names">SV</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">HyPhy: hypothesis testing using phylogenies</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">21</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">676</span> <span class="lpage" tagx="lpage" title="lpage">679</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/bti079">10.1093/bioinformatics/bti079</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15509596">15509596</a> </span> </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Murrell</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Gene-wide identification of episodic selection</span> <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">32</span> <span class="fpage" tagx="fpage" title="fpage">1365</span> <span class="lpage" tagx="lpage" title="lpage">1371</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msv035">10.1093/molbev/msv035</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25701167">25701167</a> </span> </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hudson</span> <span class="given-names" tagx="given-names" title="given-names">RR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kreitman</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aguadé</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A Test of Neutral Molecular Evolution Based on Nucleotide Data</span> <span class="source" tagx="source" title="source">Genetics</span> <span class="year" tagx="year" title="year">1987</span> <span class="volume" tagx="volume" title="volume">116</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">153</span> <span class="lpage" tagx="lpage" title="lpage">159</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/3110004">3110004</a> </span> </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Librado</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rozas</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">DnaSP v5: A software for comprehensive analysis of DNA polymorphism data</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">1451</span> <span class="lpage" tagx="lpage" title="lpage">1452</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btp187">10.1093/bioinformatics/btp187</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19346325">19346325</a> </span> </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">NCBI BLAST: a better web inerface</span> <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">36</span> <span class="fpage" tagx="fpage" title="fpage">W5</span> <span class="lpage" tagx="lpage" title="lpage">W9</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkn201">10.1093/nar/gkn201</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18440982">18440982</a> </span> </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huelsenbeck</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ronquist</span> <span class="given-names" tagx="given-names" title="given-names">FR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">MRBAYES: Bayesian inference of phylogenetic trees</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">754</span> <span class="lpage" tagx="lpage" title="lpage">755</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/17.8.754">10.1093/bioinformatics/17.8.754</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11524383">11524383</a> </span> </span> </li> <li tag="ref"> <a name="CR45" /> <span class="label" tagx="label" title="label">45.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stamatakis</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">RAxML Version 8: A tool for phylogenetic analysis and post-analysis of large phylogenies</span> <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">1312</span> <span class="lpage" tagx="lpage" title="lpage">1313</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btu033">10.1093/bioinformatics/btu033</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24451623">24451623</a> </span> </span> </li> <li tag="ref"> <a name="CR46" /> <span class="label" tagx="label" title="label">46.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kumar, S. PHYLTEST: a program for testing phylogenetic hypothesis. Version 2.0. Institute of Molecular and Evolutionary State University, University Park, Pennsylvania (1996). </span> </li> <li tag="ref"> <a name="CR47" /> <span class="label" tagx="label" title="label">47.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takezaki</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Razhetsky</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nei</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogenetic test of the molecular clock and linearized trees</span> <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="year" tagx="year" title="year">1995</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">823</span> <span class="lpage" tagx="lpage" title="lpage">833</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7476128">7476128</a> </span> </span> </li> <li tag="ref"> <a name="CR48" /> <span class="label" tagx="label" title="label">48.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xue</span> <span class="given-names" tagx="given-names" title="given-names">J-Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiu</span> <span class="given-names" tagx="given-names" title="given-names">Y-L</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The mitochondrial genomes of the early land plants <i>Treubia lacunosa</i> and <i>Anomodon rugelii</i>: Dynamic and conservative evolution </span> <span class="source" tagx="source" title="source">PLoS ONE</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="issue" tagx="issue" title="issue">10</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0025836">10.1371/journal.pone.0025836</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21998706">21998706</a> </span> </span> </li> <li tag="ref"> <a name="CR49" /> <span class="label" tagx="label" title="label">49.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wickett</span> <span class="given-names" tagx="given-names" title="given-names">NJ</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Functional gene losses occur minimal size reduction in the plastid genome of the parasitic liverwort <i>Aneura mirabilis</i> </span> <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">393</span> <span class="lpage" tagx="lpage" title="lpage">401</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msm267">10.1093/molbev/msm267</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18056074">18056074</a> </span> </span> </li> <li tag="ref"> <a name="CR50" /> <span class="label" tagx="label" title="label">50.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">CS</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The location and translocation of <i>ndh</i> genes of chloroplast origin in the Orchidaceae family </span> <span class="source" tagx="source" title="source">Sci. Rep.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep09040">10.1038/srep09040</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25761566">25761566</a> </span> </span> </li> <li tag="ref"> <a name="CR51" /> <span class="label" tagx="label" title="label">51.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barrett</span> <span class="given-names" tagx="given-names" title="given-names">CF</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Investigating the path of plastid genome degradation in an early-transitional clade of heterotrophic orchids, and implications for heterotrophic angiosperms </span> <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="fpage" tagx="fpage" title="fpage">3095</span> <span class="lpage" tagx="lpage" title="lpage">3112</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msu252">10.1093/molbev/msu252</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25172958">25172958</a> </span> </span> </li> <li tag="ref"> <a name="CR52" /> <span class="label" tagx="label" title="label">52.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bidartondo</span> <span class="given-names" tagx="given-names" title="given-names">MI</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bruns</span> <span class="given-names" tagx="given-names" title="given-names">TD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weiss</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sérgio</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Read</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Specialized cheating of the ectomycorrhizal symbiosis by an epiparasitic liverwort</span> <span class="source" tagx="source" title="source">Proc. R. Soc. Lond., B, Biol. Sci.</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">270</span> <span class="issue" tagx="issue" title="issue">1517</span> <span class="fpage" tagx="fpage" title="fpage">835</span> <span class="lpage" tagx="lpage" title="lpage">842</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1098/rspb.2002.2299">10.1098/rspb.2002.2299</a> </span> </span> </li> <li tag="ref"> <a name="CR53" /> <span class="label" tagx="label" title="label">53.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neubig</span> <span class="given-names" tagx="given-names" title="given-names">KM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogenetic utility of <i>ycf</i>1 in orchids: a plastid gene more variable than <i>mat</i>K </span> <span class="source" tagx="source" title="source">Plant Syst. Evol.</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">277</span> <span class="fpage" tagx="fpage" title="fpage">75</span> <span class="lpage" tagx="lpage" title="lpage">84</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00606-008-0105-0">10.1007/s00606-008-0105-0</a> </span> </span> </li> <li tag="ref"> <a name="CR54" /> <span class="label" tagx="label" title="label">54.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dong</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>ycf</i>1, the most promising plastid DNA barcode of land plants </span> <span class="source" tagx="source" title="source">Sci. Rep.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep08348">10.1038/srep08348</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25672218">25672218</a> </span> </span> </li> <li tag="ref"> <a name="CR55" /> <span class="label" tagx="label" title="label">55.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Szczecińska</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sawicki</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genomic resources of three <i>Pulsatilla</i> species reveal evolutionary hotspots, species-specific sites and variable plastid structure in the family Ranunculaceae </span> <span class="source" tagx="source" title="source">Int. J. Mol. Sci.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">22258</span> <span class="lpage" tagx="lpage" title="lpage">22279</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3390/ijms160922258">10.3390/ijms160922258</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26389887">26389887</a> </span> </span> </li> <li tag="ref"> <a name="CR56" /> <span class="label" tagx="label" title="label">56.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">GF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hinsinger</span> <span class="given-names" tagx="given-names" title="given-names">DD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strijk</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Comparison of intraspecific, interspecific and intergeneric chloroplast diversity in Cycads </span> <span class="source" tagx="source" title="source">Sci. Rep.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep31473">10.1038/srep31473</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27558458">27558458</a> </span> </span> </li> <li tag="ref"> <a name="CR57" /> <span class="label" tagx="label" title="label">57.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yi</span> <span class="given-names" tagx="given-names" title="given-names">DK</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The complete chloroplast genome sequence of <i>Abies nephrolepis</i> (Pinaceae: Abietoideae) </span> <span class="source" tagx="source" title="source">J. Asia Pac. Biodivers.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">245</span> <span class="lpage" tagx="lpage" title="lpage">249</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.japb.2016.03.014">10.1016/j.japb.2016.03.014</a> </span> </span> </li> <li tag="ref"> <a name="CR58" /> <span class="label" tagx="label" title="label">58.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Melodelima</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lobréaux</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Complete <i>Arabis alpina</i> chloroplast genome sequence and insight into its polymorphism </span> <span class="source" tagx="source" title="source">Meta Gene</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="fpage" tagx="fpage" title="fpage">65</span> <span class="lpage" tagx="lpage" title="lpage">75</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.mgene.2013.10.004">10.1016/j.mgene.2013.10.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25606376">25606376</a> </span> </span> </li> <li tag="ref"> <a name="CR59" /> <span class="label" tagx="label" title="label">59.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oliver</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Chloroplast genome sequence of the moss <i>Tortula ruralis</i>: gene content, polymorphism, and structural arrangement relative to other green plant chloroplast genomes </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-11-143">10.1186/1471-2164-11-143</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20187961">20187961</a> </span> </span> </li> <li tag="ref"> <a name="CR60" /> <span class="label" tagx="label" title="label">60.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lewis</span> <span class="given-names" tagx="given-names" title="given-names">LR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rozzi</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goffinet</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Infraspecific variation within and across complete organellar genomes and nuclear ribosomal repeats in a moss </span> <span class="source" tagx="source" title="source">Mol. Phylogenet. Evol.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">96</span> <span class="fpage" tagx="fpage" title="fpage">195</span> <span class="lpage" tagx="lpage" title="lpage">199</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ympev.2015.12.005">10.1016/j.ympev.2015.12.005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26724407">26724407</a> </span> </span> </li> <li tag="ref"> <a name="CR61" /> <span class="label" tagx="label" title="label">61.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xue</span> <span class="given-names" tagx="given-names" title="given-names">JY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiu</span> <span class="given-names" tagx="given-names" title="given-names">YL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The complete mitochondria genome sequence of the hornwort <i>Phaeoceres laevis</i>: retention of many ancient pseudogenes and conservative evolution of mitochondrial genomes in hornworts </span> <span class="source" tagx="source" title="source">Curr. Genet.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">56</span> <span class="fpage" tagx="fpage" title="fpage">53</span> <span class="lpage" tagx="lpage" title="lpage">62</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00294-009-0279-1">10.1007/s00294-009-0279-1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19998039">19998039</a> </span> </span> </li> <li tag="ref"> <a name="CR62" /> <span class="label" tagx="label" title="label">62.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Groth-Malonek</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wahrmund</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Polsakiewicz</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knoop</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evolution of a pseudogene: exlusive survival of a functional mitochondria <i>nad</i>7 gene supports <i>Haplomitrium</i> as the earliest liverwort lineage and proposes a secondary loss of RNA editing in Marchantiidae </span> <span class="source" tagx="source" title="source">Miol Biol Evol</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">24</span> <span class="fpage" tagx="fpage" title="fpage">1068</span> <span class="lpage" tagx="lpage" title="lpage">1074</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/molbev/msm026">10.1093/molbev/msm026</a> </span> </span> </li> <li tag="ref"> <a name="CR63" /> <span class="label" tagx="label" title="label">63.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bell</span> <span class="given-names" tagx="given-names" title="given-names">NE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boore</span> <span class="given-names" tagx="given-names" title="given-names">JL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brent</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hyvönen</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Organellar genomes of the four-toothed moss, <i>Tetraphis pellucida</i> </span> <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-15-383">10.1186/1471-2164-15-383</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24884426">24884426</a> </span> </span> </li> <li tag="ref"> <a name="CR64" /> <span class="label" tagx="label" title="label">64.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crandall-Stotler</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stotler</span> <span class="given-names" tagx="given-names" title="given-names">RE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Long</span> <span class="given-names" tagx="given-names" title="given-names">DG</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Phylogeny and classification of the Marchantiophyta</span> <span class="source" tagx="source" title="source">Edinb. J. Bot.</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">66</span> <span class="fpage" tagx="fpage" title="fpage">155</span> <span class="lpage" tagx="lpage" title="lpage">198</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1017/S0960428609005393">10.1017/S0960428609005393</a> </span> </span> </li> <li tag="ref"> <a name="CR65" /> <span class="label" tagx="label" title="label">65.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cui</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A complete mitochondrial genome of wheat ( <i>Triticum aestivum</i> cv. Chinese Yumai), and fast evolving mitochondrial genes in higher plants </span> <span class="source" tagx="source" title="source">J. Genet.</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">88</span> <span class="fpage" tagx="fpage" title="fpage">299</span> <span class="lpage" tagx="lpage" title="lpage">307</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s12041-009-0043-9">10.1007/s12041-009-0043-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20086295">20086295</a> </span> </span> </li> <li tag="ref"> <a name="CR66" /> <span class="label" tagx="label" title="label">66.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Medina</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goffinet</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">350 my of mitochondrial genome stasis in mosses, an early land plant lineage</span> <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="fpage" tagx="fpage" title="fpage">8</span> <span class="lpage" tagx="lpage" title="lpage">13</span> </span> </li> <li tag="ref"> <a name="CR67" /> <span class="label" tagx="label" title="label">67.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matute</span> <span class="given-names" tagx="given-names" title="given-names">DR</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evidence for positive selection in putative virulence factors within the <i>Paracoccidioides brasiliensis</i> Species Complex </span> <span class="source" tagx="source" title="source">PLoS Negl. Trop. Dis.</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="issue" tagx="issue" title="issue">9</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pntd.0000296">10.1371/journal.pntd.0000296</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18820744">18820744</a> </span> </span> </li> <li tag="ref"> <a name="CR68" /> <span class="label" tagx="label" title="label">68.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vigalondo</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Comparing three complete mitochondrial genomes of the moss genus <i>Orthotrichum</i> Hedw </span> <span class="source" tagx="source" title="source">Mitochondrial DNA B Resour.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">168</span> <span class="lpage" tagx="lpage" title="lpage">170</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1080/23802359.2016.1149784">10.1080/23802359.2016.1149784</a> </span> </span> </li> <li tag="ref"> <a name="CR69" /> <span class="label" tagx="label" title="label">69.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sawicki, J., Szczecińska, M., Bednarek-Ochyra, H., Ochyra, R. Mitochondrial phylogenomics supports splitting the traditionally conceived genus <i>Racomitrium</i> (Bryophyta: Grimmiaceae). <i>Nova Hedwigia</i> <b>100</b> <b>(</b> <b>2</b> <b>)</b>, doi:10.1127/nova_hedwigia/2015/0248 (2015). </span> </li> <li tag="ref"> <a name="CR70" /> <span class="label" tagx="label" title="label">70.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wachowiak</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bączkiewicz</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chudzińska</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buczkowska</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cryptic speciation in liverworts – a case study in the <i>Aneura pinguis</i> complex </span> <span class="source" tagx="source" title="source">Bot. J. Linn. Soc.</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">155</span> <span class="fpage" tagx="fpage" title="fpage">273</span> <span class="lpage" tagx="lpage" title="lpage">282</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1095-8339.2007.00692.x">10.1111/j.1095-8339.2007.00692.x</a> </span> </span> </li> <li tag="ref"> <a name="CR71" /> <span class="label" tagx="label" title="label">71.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Preussing</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">New insights in the evolution of the liverwort family Aneuraceae (Metzgeriales, Marchantiophyta), with emphasis on the genus <i>Lobatiriccardia</i> </span> <span class="source" tagx="source" title="source">Taxon</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">59</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">1424</span> <span class="lpage" tagx="lpage" title="lpage">1440</span> </span> </li> <li tag="ref"> <a name="CR72" /> <span class="label" tagx="label" title="label">72.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lohse</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">OrganellarGenomeDRAW—a suite of tools for generating physical maps of plastid and mitochondrial genomes and visualizing expression data sets </span> <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">41</span> <span class="fpage" tagx="fpage" title="fpage">W575</span> <span class="lpage" tagx="lpage" title="lpage">W581</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkt289">10.1093/nar/gkt289</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23609545">23609545</a> </span> </span> </li> </ul> </div> </div>  </body></html>