<?xml version="1.0" encoding="UTF-8"?><html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
   <div class="sec" title="sec">
      <div xmlns="http://www.w3.org/1999/xhtml">ZFN-mediated targeted sequence replacement</div>
      <p xmlns="http://www.w3.org/1999/xhtml">We then tested if ZFN-mediated targeted sequence replacement was feasible directly
         in the oocyte. A gene targeting vector, IgME1, was designed to replace IgM exon 1
         with a PGK neo cassette, providing ∼1.9 kb of novel sequence. The strategy is illustrated
         in 
         <a>Figure 3</a>; the full sequence of the gene-targeting vector is shown in 
         <a>Figure S2</a>. Although the presence of the PGK neo gene was used for screening, no antibiotic
         selection was used to enrich for neo-positive embryos.
         
      </p>
      <div xmlns="http://www.w3.org/1999/xhtml"><span>10.1371/journal.pone.0021045.g003</span><span>Figure 3</span><div>
            <div>Targeted sequence replacement at the IgM locus.</div>
            <p>The structure of the IgME1 gene targeting vector, regions of homology with the IgM
               locus, and the predicted structure of the targeted locus are shown. Exons are marked
               and numbered as open boxes. The ZFN 18255/18257 cleavage site within exon 1 is indicated
               by a vertical arrow. The 5′ and 3′ junction PCR products are indicated; with positions
               of primers marked by arrows, and restriction sites for 
               <span>MscI</span> and 
               <span>ApaI</span> marked by (M) and (A), respectively. The hybridization probe and the diagnostic fragments
               used for Southern analysis are indicated in the lower part of the diagram. 
               <span>HinDIII</span> (H) and 
               <span>BspHI</span> (B) restriction sites are marked.
               
            </p>
         </div>
         <div></div>
      </div>
      <p xmlns="http://www.w3.org/1999/xhtml">Mixtures of 
         <span>MluI</span> linearised IgME1 DNA (concentrations ranging from 5 to 15 ng/µl) and ZFN 18255/18257
         mRNAs (concentrations ranging from 3 to 9 ng/µl of each) were injected into pronuclear
         stage fertilized oocytes, see 
         <a>Table 3</a>. Since ZFN mRNAs are most effective when injected into the cytoplasm, but IgME1 targeting
         vector DNA must be located in the nucleus to serve as the donor during homology-directed
         repair, we employed a two-step microinjection procedure similar to that described
         by Meyer et al. 
         <a>[23]</a>. A portion of the mRNA/DNA mixture was first injected into a pronucleus to locate
         the vector DNA with the nuclear DNA. Then, on removing the injection needle, a second
         dose was injected into the cytoplasm to enable translation of ZFN mRNA. Microinjected
         oocytes were cultured to blastocyst stage and a total of 242 blastocysts were analyzed
         by PCR for targeted gene replacement events. Correct gene editing was detected in
         three blastocysts, as judged by the presence of a diagnostic 1.758 kb PCR product
         spanning the 5′ junction of the target site, see 
         <a>Figure 4</a>. Data summarized in 
         <a>Table 3</a> show that concentrations of ZFN mRNA slightly higher than those used for mutagenesis
         were most effective in driving targeted sequence replacement. Combined data for all
         mRNA and targeting vector DNA concentrations indicated an overall rate of 1.2% targeted
         sequence replacement, but efficiency at the most effective mRNA/DNA concentrations
         was higher (17% at 9 ng/µl ZFN mRNA plus 10–15 ng/µl IgME1 DNA). Positive blastocysts
         were also screened for the presence of an intact IgM exon 1 by PCR. All gave a positive
         result, indicating that targeting had occurred at just one IgM allele.
         
      </p>
      <div xmlns="http://www.w3.org/1999/xhtml"><span>10.1371/journal.pone.0021045.g004</span><span>Figure 4</span><div>
            <div>PCR analysis of gene targeted blastocysts and fetuses.</div>
            <p>Upper panel, representative 5′ junction PCR products amplified from blastocysts. Upper
               panel left, 5′ junction PCR from fetuses. Predicted 5′ PCR fragment size: 1.758 kb.
               Upper panel right, restriction digests of 5′ PCR fragment amplified from fetus 7.
               Predicted restriction fragment sizes: 
               <span>ApaI</span> - 1242, 516 bp; 
               <span>MscI</span> - 853, 687, 219 bp. Middle and bottom panels, 5′ and 3′ junction PCR products amplified
               from fetuses 1–12. Predicted 3′ PCR fragment size: 1.683 kb. M - size markers.
               
            </p>
         </div>
         <div></div>
      </div>
      <div xmlns="http://www.w3.org/1999/xhtml"><span>10.1371/journal.pone.0021045.t003</span><span>Table 3</span><div>
            <div>Screening of rabbit blastocysts for targeted gene replacement.</div>
         </div>
         <div>
            <div></div>
            <table>
               <colgroup>
                  <col />
                  <col />
                  <col />
                  <col />
                  <col />
               </colgroup>
               <thead>
                  <tr>
                     <td>Concentration of ZFN mRNA (ng/
                        <span>µ</span>l)
                        
                     </td>
                     <td>Concentration of vector DNA (ng/
                        <span>µ</span>l)
                        
                     </td>
                     <td>Oocytes injected</td>
                     <td>Blastocysts screened</td>
                     <td>Blastocysts positive for targeting</td>
                  </tr>
               </thead>
               <tbody>
                  <tr>
                     <td>3</td>
                     <td>5</td>
                     <td>102</td>
                     <td>45</td>
                     <td>0</td>
                  </tr>
                  <tr>
                     <td>3</td>
                     <td>10</td>
                     <td>39</td>
                     <td>26</td>
                     <td>0</td>
                  </tr>
                  <tr>
                     <td>6</td>
                     <td>5</td>
                     <td>76</td>
                     <td>43</td>
                     <td>0</td>
                  </tr>
                  <tr>
                     <td>6</td>
                     <td>10</td>
                     <td>118</td>
                     <td>76</td>
                     <td>1</td>
                  </tr>
                  <tr>
                     <td>6</td>
                     <td>15</td>
                     <td>28</td>
                     <td>18</td>
                     <td>0</td>
                  </tr>
                  <tr>
                     <td>9</td>
                     <td>10</td>
                     <td>17</td>
                     <td>6</td>
                     <td>1</td>
                  </tr>
                  <tr>
                     <td>9</td>
                     <td>15</td>
                     <td>60</td>
                     <td>28</td>
                     <td>1</td>
                  </tr>
               </tbody>
            </table>
         </div>
      </div>
      <p xmlns="http://www.w3.org/1999/xhtml">Oocytes were then injected with 9 ng/µl ZFN 18255/18257 mRNA plus 10 ng/µl IgME1 DNA,
         and transferred to recipients. Seventeen fetuses were removed at day 15 of gestation
         and analyzed. All showed normal development, and examination of the uterine horns
         showed no evidence of degenerate or absorbed conceptuses. DNA was prepared from each
         fetus and screened by PCR. Three fetuses (2, 7 and 9) showed amplification of a 1.758
         kb product across the 5′ junction and a 1.683 kb fragment across the 3′ junction of
         the target site, consistent with replacement of IgM exon 1 with the PGK neo cassette
         by homologous recombination, see 
         <a>Figure 4</a>. The identity of the 5′ PCR product was confirmed by restriction digestion, see 
         <a>Figure 4</a>. In addition, the DNA sequence of the 5′ PCR product from fetus 7 was determined
         and found to be identical to that predicted for the targeted locus, see 
         <a>Figure S2</a>. Southern hybridization analysis was carried out using genomic DNA from the three
         PCR positive and five PCR negative fetuses. The Southern blotting scheme is outlined
         in 
         <a>Figure 3</a> and results shown in 
         <a>Figure S3</a>. Each of the PCR positive fetuses showed a diagnostic restriction fragment spanning
         the 3′ junction of the target site, consistent with targeted replacement at the first
         exon, and also a fragment corresponding to a wild type IgM allele. The five negative
         fetuses showed only the wild type allele.
         
      </p>
      <p xmlns="http://www.w3.org/1999/xhtml">In summary, our data show efficient ZFN-induced gene disruption and homology-directed
         sequence replacement through microinjection in rabbit oocytes. We have obtained the
         first rabbits genetically engineered at a specific endogenous locus, the immunoglobulin
         M locus.
      </p>
   </div>
</html>