<?xml version="1.0" encoding="UTF-8"?><html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
   <div class="sec" title="sec">
      <div xmlns="http://www.w3.org/1999/xhtml">Results</div>
      <div class="sec" title="sec">
         <div xmlns="http://www.w3.org/1999/xhtml">Phlyogenetic analyses</div>
         <p xmlns="http://www.w3.org/1999/xhtml">All three methods of phylogenetic inference (NJ, ML and MP) identified two major lineages
            (African and Asian) (
            <a>Figure 1</a>). The most recent common ancestor of MR 766 (Uganda, 1947) diverged first, followed
            by the divergence of the most recent common ancestor of the ArD 41519 (Senegal, 1984)
            and IbH 30656 (Nigeria, 1968) strains, the P6-740 strain (Malaysia, 1966), and lastly
            the EC Yap (Micronesia, 2007) and FS13025 (Cambodia, 2010) strains.
            
         </p>
         <div xmlns="http://www.w3.org/1999/xhtml"><span>10.1371/journal.pntd.0001477.g001</span><span>Figure 1</span><div>
               <div>Zika virus nucleotide and amino acid alignments.</div>
               <p>Neighbor-joining phylogeny generated from open reading frame nucleotide sequences
                  of Zika virus strains. The tree was rooted with Spondweni virus (GenBank accession
                  number DQ859064). The scale at the bottom of the tree represents genetic distance
                  in nucleotide substitutions per site. Numbers at the nodes represent percent bootstrap
                  support values based on 1,000 replicates. Isolates are represented according to strain
                  name, country of origin, and year of isolation. The lineage of each virus is indicated
                  to the right of the tree. *Strains sequenced in this study.
               </p>
            </div>
            <div></div>
         </div>
      </div>
      <div class="sec" title="sec">
         <div xmlns="http://www.w3.org/1999/xhtml">Nucleotide and amino acid sequence variation among African and Asian strains</div>
         <p xmlns="http://www.w3.org/1999/xhtml">Based on nucleotide and amino acid sequence composition, the African strains were
            the most divergent from the Asian strains, and strains from the same geographic regions
            were the least divergent (Africa and Asia) (
            <a>Table 3</a>). There were several deduced amino acid differences among the strains, which in turn
            correlated to geographic area of virus collection.
            
         </p>
         <div xmlns="http://www.w3.org/1999/xhtml"><span>10.1371/journal.pntd.0001477.t003</span><span>Table 3</span><div>
               <div>Pairwise comparisons of African and Asian Zika virus strains.
                  <a>*</a></div>
            </div>
            <div>
               <div></div>
               <table>
                  <colgroup>
                     <col />
                     <col />
                     <col />
                     <col />
                     <col />
                     <col />
                     <col />
                  </colgroup>
                  <thead>
                     <tr>
                        <td>Sequence</td>
                        <td>MR 766
                           <a>†</a></td>
                        <td>IbH 30656</td>
                        <td>ArD 41519</td>
                        <td>P6-740</td>
                        <td>FSS13025</td>
                        <td>EC Yap</td>
                     </tr>
                  </thead>
                  <tbody>
                     <tr>
                        <td>MR 766
                           <a>†</a></td>
                        <td></td>
                        <td><span>2.2</span></td>
                        <td><span>1.7</span></td>
                        <td><span>3.1</span></td>
                        <td><span>3.7</span></td>
                        <td><span>3.7</span></td>
                     </tr>
                     <tr>
                        <td>IbH 30656</td>
                        <td>7.0</td>
                        <td></td>
                        <td><span>1.2</span></td>
                        <td><span>3.1</span></td>
                        <td><span>3.6</span></td>
                        <td><span>3.9</span></td>
                     </tr>
                     <tr>
                        <td>ArD 41519</td>
                        <td>7.0</td>
                        <td>3.2</td>
                        <td></td>
                        <td><span>2.7</span></td>
                        <td><span>3.1</span></td>
                        <td><span>3.4</span></td>
                     </tr>
                     <tr>
                        <td>P6-740</td>
                        <td>10.2</td>
                        <td>10.3</td>
                        <td>10.1</td>
                        <td></td>
                        <td><span>1.2</span></td>
                        <td><span>1.5</span></td>
                     </tr>
                     <tr>
                        <td>FSS13025</td>
                        <td>11.6</td>
                        <td>11.7</td>
                        <td>11.6</td>
                        <td>4.3</td>
                        <td></td>
                        <td><span>0.8</span></td>
                     </tr>
                     <tr>
                        <td>EC Yap</td>
                        <td>11.4</td>
                        <td>11.7</td>
                        <td>11.5</td>
                        <td>4.3</td>
                        <td>1.8</td>
                        <td></td>
                     </tr>
                  </tbody>
               </table>
            </div>
            <div>
               <div xmlns="" class="fn-type-" title=""><span xmlns="http://www.w3.org/1999/xhtml"></span><p xmlns="http://www.w3.org/1999/xhtml">*Lightface type = Percent nucleotide divergence; Boldface type = Percent amino acid
                     divergence.
                  </p>
               </div>
               <div xmlns="" class="fn-type-" title="">
                  <p xmlns="http://www.w3.org/1999/xhtml"><span>†:</span> GenBank accession no. AY632535.
                     
                  </p>
               </div>
            </div>
         </div>
      </div>
      <div class="sec" title="sec">
         <div xmlns="http://www.w3.org/1999/xhtml">Genetic variation among three MR 766 sequences</div>
         <p xmlns="http://www.w3.org/1999/xhtml">The two MR 766 strains that had been previously sequenced exhibited extensive genetic
            variation (6.3% nucleotide, 1.8% amino acid divergence) (
            <a>Table 2</a>). To investigate this discrepancy (
            <a>Table 2</a>), we sequenced an additional strain of MR 766 from the WRCEVA collection. The MR
            766 sequence with accession number AY632535 
            <a>[16]</a>, was ultimately chosen for use in our analyses due to its low passage history, nucleotide
            and amino acid similarity to the high passage MR 766 strain that we sequenced (0.4%
            nucleotide and 0.6% amino acid divergence), and its position closest to the root of
            the MR 766 lineage in a tree including all three sequences (not shown).
            
         </p>
      </div>
      <div class="sec" title="sec">
         <div xmlns="http://www.w3.org/1999/xhtml">Glycosylation site variation between African and Asian strains</div>
         <p xmlns="http://www.w3.org/1999/xhtml">Deletions in a potential glycosylation site of several strains were observed following
            their alignment (
            <a>Figure 2</a>). A 4-codon deletion was observed beginning at amino acid position 153 of the E protein
            of the MR766 strain (GenBank accession number AY632535), a 6-codon deletion at position
            156 of another MR 766 strain (GenBank accession number DQ859059), and a 6-codon deletion
            at position 156 of the IbH 30656 strain sequenced in this study. The MR 766 strain
            (Passage 147) sequenced here did not exhibit any deletions in the predicted amino
            acid sequence and provided evidence of passage-associated changes in potential glycosylation
            site(s).
            
         </p>
         <div xmlns="http://www.w3.org/1999/xhtml"><span>10.1371/journal.pntd.0001477.g002</span><span>Figure 2</span><div>
               <div>Zika virus phylogeny.</div>
               <p>Nucleotide (top) and amino acid (bottom) sequences of the envelope protein/gene of
                  Zika virus strains showing the deletions in the potential glycosylation sites of the
                  MR 766 (Uganda, 1947) and the IbH 30656 (Nigeria, 1968) strains. Deletions are indicated
                  by dashes. The “N” at position 467 of the P6-740 strain (Malaysia, 1966) represents
                  an equally weighted double population of the nucleotides “C” and “T”. This translates
                  to an “X” at position 165 of the amino acid alignment.
               </p>
            </div>
            <div></div>
         </div>
      </div>
   </div>
</html>