<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Front Plant Sci</span> <span class="iso-abbrev" title="iso-abbrev">Front Plant Sci</span> <span class="publisher-id" title="publisher-id">Front. Plant Sci.</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Frontiers in Plant Science</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1664-462X</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Frontiers Media S.A.</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5954354">5954354</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3389/fpls.2018.00537">10.3389/fpls.2018.00537</a> </span> <div class="article-categories" title="article-categories"> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Science</span>: <span class="subject" title="subject">Original Research</span> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Identification of Putative Precursor Genes for the Biosynthesis of Cannabinoid-Like Compound in <i>Radula marginata</i> </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hussain</span> <span class="given-names" tagx="given-names" title="given-names">Tajammul</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#c001"> <sup>*</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/522045/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Plunkett</span> <span class="given-names" tagx="given-names" title="given-names">Blue</span> </span> <a href="#aff2"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ejaz</span> <span class="given-names" tagx="given-names" title="given-names">Mahwish</span> </span> <a href="#aff3"> <sup>3</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Espley</span> <span class="given-names" tagx="given-names" title="given-names">Richard V.</span> </span> <a href="#aff2"> <sup>2</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/172189/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kayser</span> <span class="given-names" tagx="given-names" title="given-names">Oliver</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#c002"> <sup>*</sup> </a> </span> </div> <span class="citation_author_institution" id="aff1">[], <sup>1</sup> <span class="institution" title="institution">Department of Technical Biochemistry, TU Dortmund University</span> <span class="addr-line" title="addr-line">Dortmund</span> <span class="country" tagx="country" title="country">Germany</span> </span> <span class="citation_author_institution" id="aff2">[], <sup>2</sup> <span class="institution" title="institution">The New Zealand Institute for Plant &amp;amp; Food Research Limited (PFR)</span> <span class="addr-line" title="addr-line">Auckland</span> <span class="country" tagx="country" title="country">New Zealand</span> </span> <span class="citation_author_institution" id="aff3">[], <sup>3</sup> <span class="institution" title="institution">Max Planck Institute for Plant Breeding Research</span> <span class="addr-line" title="addr-line">Cologne</span> <span class="country" tagx="country" title="country">Germany</span> </span> <div class="author-notes" title="author-notes"> <div class="fn-type-edited-by" title="edited-by"> <p>Edited by: Marco Fondi, Università degli Studi di Firenze, Italy</p> </div> <div class="fn-type-edited-by" title="edited-by"> <p>Reviewed by: Hamed Bostan, North Carolina State University, United States; Xiyin Wang, North China University of Science and Technology, China </p> </div> <div class="corresp" title="corresp">
*Correspondence: Tajammul Hussain <span class="email" tagx="email" title="email">tajammul.hussain@tu-dortmund.de</span> </div> <div class="corresp" title="corresp">
Oliver Kayser <span class="email" tagx="email" title="email">oliver.kayser@tu-dortmund.de</span> </div> <div class="fn-type-other" title="other"> <p>This article was submitted to Bioinformatics and Computational Biology, a section of the journal Frontiers in Plant Science </p> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-5-5</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span> </span> <span class="volume" tagx="volume" title="volume">9</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">537</span> <span class="history" title="history"> <span class="received" title="received">received: 2018-1-29</span> <span class="accepted" title="accepted">accepted: 2018-4-06</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Hussain, Plunkett, Ejaz, Espley and Kayser</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The liverwort <i>Radula marginata</i> belongs to the bryophyte division of land plants and is a prospective alternate source of cannabinoid-like compounds. However, mechanistic insights into the molecular pathways directing the synthesis of these cannabinoid-like compounds have been hindered due to the lack of genetic information. This prompted us to do deep sequencing, <i>de novo</i> assembly and annotation of <i>R. marginata</i> transcriptome, which resulted in the identification and validation of the genes for cannabinoid biosynthetic pathway. In total, we have identified 11,421 putative genes encoding 1,554 enzymes from 145 biosynthetic pathways. Interestingly, we have identified all the upstream genes of the central precursor of cannabinoid biosynthesis, cannabigerolic acid (CBGA), including its two first intermediates, stilbene acid (SA) and geranyl diphosphate (GPP). Expression of all these genes was validated using quantitative real-time PCR. We have characterized the protein structure of stilbene synthase (STS), which is considered as a homolog of olivetolic acid in <i>R. marginata</i>. Moreover, the metabolomics approach enabled us to identify CBGA-analogous compounds using electrospray ionization mass spectrometry (ESI-MS/MS) and gas chromatography mass spectrometry (GC-MS). Transcriptomic analysis revealed 1085 transcription factors (TF) from 39 families. Comparative analysis showed that six TF families have been uniquely predicted in <i>R. marginata</i>. In addition, the bioinformatics analysis predicted a large number of simple sequence repeats (SSRs) and non-coding RNAs (ncRNAs). Our results collectively provide mechanistic insights into the putative precursor genes for the biosynthesis of cannabinoid-like compounds and a novel transcriptomic resource for <i>R. marginata</i>. The large-scale transcriptomic resource generated in this study would further serve as a reference transcriptome to explore the <i>Radulaceae</i> family. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd"> <i>Radula marginata</i> </div> <div class="kwd" title="kwd"> <i>de novo</i> transcriptomic assembly </div> <div class="kwd" title="kwd">
biosynthetic metabolomics pathways</div> <div class="kwd" title="kwd">
plant cannabinoids</div> <div class="kwd" title="kwd"> <i>Radula</i> </div> <div class="kwd" title="kwd">
a prospective alternate to cannabinoids</div> </div> <div class="counts" title="counts"> <span class="fig-count" title="fig-count">fig-count: </span> <span class="table-count" title="table-count">table-count: </span> <span class="equation-count" title="equation-count">equation-count: </span> <span class="ref-count" title="ref-count">ref-count: </span> <span class="page-count" title="page-count">page-count: </span> <span class="word-count" title="word-count">word-count: </span> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>The liverwort <i>Radula marginata</i> belongs to the bryophyte division of land plants. Bryophytes are a distinctive group of an early-diverged lineage of non-vascular land plants comprising around 20,000 species. This division includes mosses, hornworts, and liverworts. Liverworts, or Marchantiophyta, are the most abundant phylum, comprising almost 6,000–8,000 species with high diversity in their ecology, morphology and genetic variation (Qiu et al., <a href="#B65">2007</a>; Rubinstein et al., <a href="#B69">2010</a>; Ruhfel et al., <a href="#B71">2014</a>). These non-vascular plants are very simple, with small flattened bodies with overlapping scales (Buck, <a href="#B15">1986</a>). Liverworts evolved (from algae) as the earliest bryophyte group and thus are considered ancestors of the mosses, hornworts, and land plants (embryophytes) (Willis and McElwain, <a href="#B92">2002</a>). Because of their distinct evolutionary position as a basal group for the colonization of land plants, bryophytes are ideal for exploring the evolution of plants, genetic networks, and developmental variation (Bowman et al., <a href="#B13">2007</a>; Sharma et al., <a href="#B75">2014</a>). </p> <p>Among three classes of liverworts, the Jungermanniopsida encompasses 85% of known liverwort species. This is further subdivided into three subclasses and each subclass has three orders. The order Jungermanniales of the subclass Jungermanniidae consists of 13 suborders and 47 families (Stotler and Crandall-Stotler, <a href="#B81">1977</a>; Ruggiero et al., <a href="#B70">2015</a>). The <i>Radulaceae</i> family comprises the <i>Radula</i> genus, with 283 species. The most important <i>Radula</i> species are <i>R. complanta, R. demissa</i> (Renner et al., <a href="#B66">2013</a>) <i>, R. jonesii</i> (Bouman et al., <a href="#B12">1998</a>), <i>R. kojana, R. laxiramea, R. visianica, R. perrotteii</i>, and <i>R. marginata</i> (Losada-Lima et al., <a href="#B52">2007</a>). <i>Radula</i> species have been reported to be rich in habitual diversity and are found in almost all ecosystems such as trees, rocks and soils throughout the world, from Antarctica's coastal area to the northern hemisphere and from Australian semi-arid regions to the Amazon rainforest (Hallingbäck and Hodgetts, <a href="#B35">2001</a>). </p> <p>Two decades ago, <i>Radula</i> species were shown to be rich in secondary metabolites, such as terpenoids and phenolics. For example, prenyl bibenzyls were identified in <i>R. perrottetti, R. complanta</i>, and <i>R. kojana</i> species (Asakawa et al., <a href="#B5">1991</a>; Toyota et al., <a href="#B86">1994</a>). These compounds have a distinct carbon backbone structures which acts as a marker to differentiate <i>Radula</i> species (Ludwiczuk and Asakawa, <a href="#B53">2008</a>). These compounds have suggested biological as well as pharmaceutical significance, having antifungal, antioxidant, antimicrobial and cytotoxic activities. Perrottetinene and its acid (perrottetinenic acid) have been identified in <i>R. marginata</i> (Toyota et al., <a href="#B87">2002</a>; Park and Lee, <a href="#B62">2010</a>). Interestingly, these compounds are structural analogs of tetrahydrocannabinol (Δ9-THC), a psychopharmacological compound in <i>Cannabis sativa L</i>. </p> <p>Cannabinoids are plant secondary metabolites and belong to the class of terpenophenolics which are predominantly found in <i>C. sativa</i>. These compounds have proven to have a wide-ranging role in numerous clinical applications. These compounds accumulate in specialized glandular structures known as trichomes (Flemming et al., <a href="#B28">2009</a>; Happyana et al., <a href="#B36">2013</a>). Since cannabinoids have been recognized for their clinical value, research has also been carried out on plants other than <i>C. sativa</i> that contain cannabinoid-like compounds. This has led to the identification of cannabidiol-like (CBD-like) compounds in <i>Linum usitatissimum</i> (Styrczewska et al., <a href="#B83">2012</a>). Likewise, cannabigerol-like (CBG-like) compounds were found in the South African flowering plant <i>Helichrysum umbraculigerum</i> (Bohlmann and Hoffmann, <a href="#B10">1979</a>). In addition, sesquiterpenoid-like β-caryophyllene was discovered in <i>Piper nigrum</i> (Tisserand and Young, <a href="#B85">2014</a>). Despite some reports on the identification of cannabinoid-like compounds in plants, their complex plant architecture makes it challenging as an alternative source of cannabinoids. Although cannabinoids have been identified in different plant species, there has been no report on psychopharmacologically essential compounds except the THC-like metabolites found in <i>R. marginata</i>. Recently, the reported agonistic activity of THC-like natural products from <i>R. marginata</i> with the cannabinoid receptor 1 (CB1) further confirmed its significance (Russo, <a href="#B72">2016</a>; Gachet et al., <a href="#B30">2017</a>; Soethoudt et al., <a href="#B78">2017</a>). Therefore, <i>R. marginata</i> could be a suitable alternate source because of its relatively simple architecture and a diversity of natural habitats. Thus a genetic-level understanding of secondary metabolic pathways that lead to the synthesis of cannabinoid-like natural compounds would be desirable. </p> <p>Since whole genome sequencing is resource-intensive, we performed a <i>de novo</i> approach to assemble the <i>R. marginata</i> transcriptome to establish a reference dataset. In addition to the transcriptome, micro-transcriptomic traits like transcription factors (TFs), simple sequence repeats (SSRs), and non-coding RNAs (ncRNAs) were also identified to understand the regulation of these genes. Interestingly, candidate genes for almost all the enzymes required for the conversion of primary metabolites into geranyl diphosphate (GPP) were identified. Expression of all the genes was confirmed by quantitative real time PCR (RT-qPCR). In addition, stilbene synthase (STS) was identified, structurally analyzed, functionally validated and considered as the first intermediate rather than olivetolic acid for the production of CBGA-like compound. This study was designed firstly to discover genes, identify biosynthetic pathways and the regulation of these genes to understand the biological processes involved. Secondly, being the first such gene expression profiling for this family, it would also serve as the reference transcriptome in future exploration of the <i>Radulaceae</i>. Furthermore, it would provide the link to study the transition from liverworts to higher plants during evolution. </p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="illuminangssequencinganddenovotranscriptomeassembly" title="sec"> <div class="title" tagx="title" title="title">
Illumina NGS sequencing and <i>de novo</i> transcriptome assembly </div> <p>In the absence of a reference genome, <i>de novo</i> assembly of <i>R. marginata</i> was performed to obtain a comprehensive reference transcriptome and to identify novel genes for the biosynthesis of secondary metabolites, in particular, cannabinoid biosynthesis. For this purpose, we performed paired-end RNA sequencing to determine the distant connections between the transcripts. RNA-sequencing yielded ~30 million raw reads for each pair with an average length of 250 bp. After filtering the raw reads for low base quality, trimming and adapter removal (see section Materials and Methods for details), we selected 91% (28,231,052) reads for the assembly (Table <a href="#T1">1</a>, Supplementary Table <a href="#SM1">1</a>). Raw reads were <i>in-silico</i> normalized and assembled using Trinity (version 2.2.0). The <i>de novo</i> assembly resulted a total of 1,580,612 transcripts, with a median contig size of 303 bp, and a maximum of 24,899 bp. Since <i>de novo</i> assembly also generates many isoforms of a gene, particularly at high coverage areas, we only selected the longest isoform per gene, resulting in 1,482,641 transcripts (Supplementary Table <a href="#SM2">2</a>). In addition, clustering analysis resulted in 501,849 non-redundant transcripts of an average length of 484 bp and N50 of 459 bp. To perform the assembly assessment, raw reads were mapped back to the <i>de novo</i> assembled transcriptome using Bowtei2 (version 2.2.6) (Langmead and Salzberg, <a href="#B45">2012</a>). In total, 98% of raw reads successfully mapped back to the transcripts, of which 72% of the reads were mapped to both forward and reverse reads (Supplementary Table <a href="#SM3">3</a>). </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <p>Sequencing summary.</p>  <table> <tbody> <tr> <td>No of raw reads</td> <td>30,981,418</td> </tr> <tr> <td>No of filtered reads</td> <td>28,231,052 (91%)</td> </tr> <tr> <td>No of assembled bases</td> <td>608,201,850</td> </tr> <tr> <td>No of assembled transcripts</td> <td>1,580,612</td> </tr> <tr> <td>Read mapped</td> <td>20,315,784 (72%)</td> </tr> <tr> <td>Maximum length (bp)</td> <td>23,874</td> </tr> <tr> <td>Median contig length</td> <td>303</td> </tr> <tr> <td>Minimum length (bp)</td> <td>278</td> </tr> </tbody> </table> </div> <p>To determine the potential of all the putative genes to translate into full-length protein, open reading frames (ORFs) of these putative genes were predicted using the TransDecoder software package (Haas et al., <a href="#B34">2013</a>). The homology of these genes was searched from the protein database UniProtKB/Swiss-Prot ( <a href="http://www.uniprot.org/">http://www.uniprot.org/</a>) followed by the identification of the protein domains by the Hidden Markov Model based approach using PFAM ( <a href="http://pfam.xfam.org/">http://pfam.xfam.org/</a>). Results from both searches were incorporated with the longest ORFs. The ORFs showing homology to known proteins were selected. Based on the gene structure, we have predicted four types of ORFs: complete, three prime partial (3′p), five prime partial (5′p) and the internal ORFs. The complete ORFs were defined as the target genes which were transcribed with a start codon followed by five prime UTR, exon(s) and stop codon with three prime UTR (5′UTR+exon+3′UTR). The 3′p ORFs lacked a stop codon consisting of only 5′UTR and exon, while the start codon was absent in 5′p ORFs, only having exon and 3′UTR. In contrast, the internal ORFs were transcribed from the start to the end of the transcripts and did not contain start and stop codons. We have identified 2,880, 2,977, and 5,104 ORFs that belonged to complete, 3′p and 5′p ORF types, respectively. The highest number of the ORFs (76,499) belonged to the internal type. Moreover, for each type of ORF, the number of ORFs identified on the positive and negative strands was similar as shown in the percentages (Supplementary Table <a href="#SM4">4</a>). </p> <p>Finally, after the selection of the longest isoform, clustering of assembled transcripts, mapping of raw reads, quantification and retention of the longest ORF peptide candidates resulted in a total of 87,460 transcripts. This high quality unique transcriptomic dataset were named as unigenes. The length of unigenes ranged from 278 to 23,874 bp, with most of the unigenes (93%) having a length of up to 1 kb (Table <a href="#T2">2</a>). These unigenes were further used for the functional annotation and other downstream analysis. Furthermore, these unigenes were considered as the tentative consensus transcripts (Tct) and were assigned unique identifiers with the prefix <i>Rm-Tct</i> ( <i>R. marginata</i> tentative consensus transcripts). The raw sequencing data have been submitted to NCBI under the BioProject No. <a href="PRJNA430820">PRJNA430820</a>, BioSample No. 08378951 and Sequence Read Archive No <a href="SRR6489347">SRR6489347</a>. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  <p>Assembly statistics.</p>  <table> <thead> <tr> <th /> <th> <b>Transcripts</b> </th> <th> <b>Contigs</b> </th> <th> <b>Unigenes</b> </th> </tr> </thead> <tbody> <tr> <td>Number</td> <td>1,580,612</td> <td>217,702</td> <td>87,460</td> </tr> <tr> <td>Length</td> <td>881,230,701</td> <td>77,717,062</td> <td>36,765,277</td> </tr> <tr> <td>N50 (bp)</td> <td>604</td> <td>314</td> <td>324</td> </tr> <tr> <td>Percent GC</td> <td>54.25</td> <td>55.7</td> <td>55.75</td> </tr> <tr> <td>Median length (bp)</td> <td>311</td> <td>308</td> <td>316</td> </tr> <tr> <td>Averager length (bp)</td> <td>557</td> <td>359</td> <td>425</td> </tr> <tr> <td>200–500 nt</td> <td>1,262,140</td> <td>209,643</td> <td>81,695</td> </tr> <tr> <td>500–1,000 nt</td> <td>205,061</td> <td>3,788</td> <td>1,984</td> </tr> <tr> <td>1,000–2,000 nt</td> <td>49,077</td> <td>1,699</td> <td>1,356</td> </tr> <tr> <td>2,000–5,000 nt</td> <td>43,227</td> <td>2,205</td> <td>2,064</td> </tr> <tr> <td>5,000–10,000 nt</td> <td>17,345</td> <td>331</td> <td>326</td> </tr> <tr> <td>&amp;gt;10,000 nt</td> <td>49</td> <td>35</td> <td>35</td> </tr> </tbody> </table> </div> </div> <div class="functionalannotation" title="sec"> <div class="title" tagx="title" title="title">
Functional annotation</div> <div class="blastfunctionalanalysis" title="sec"> <div class="title" tagx="title" title="title">
BLAST functional analysis</div> <p>Annotation is one of the most critical steps for interpreting and precisely evaluating the transcriptomic assembly. For the preliminary functional analysis, homology of all the unigenes was searched using the blastx algorithm (Altschul et al., <a href="#B2">1990</a>) against the National Centre for Biotechnology Information (NCBI) <a href="https://www.ncbi.nlm.nih.gov/">https://www.ncbi.nlm.nih.gov/</a> non-redundant (nr) protein database. High quality identifiers for 68,715 (78%) unigenes were found with known protein coding potential with a cut-off <i>E</i>-value of 1e-05. The remaining 18,746 (21.5%) unigenes could not be matched with known protein entries in the database, because of lack of an existing genome sequence and EST information for <i>R. marginata</i>. Currently there are only 34 nucleotide and 13 protein entries available at NCBI. Percentage identity and <i>E</i>-values are two important parameters determining the quality of BLAST results. In our analysis, we found 98% of the blasted unigenes had sequence similarity between 50 and 90%, except 1.76% of the transcripts, which had a minimum sequence similarity of 31% (Figure <a href="#F1">1B</a>). Similarly, 23, 31, and 24% of unigenes had an <i>E</i>-value of up to 1e-15, 1e-30, and 1e-45, respectively. Only 2% of the sequences had an <i>E</i>-value equal to zero (Figure <a href="#F1">1C</a>). Higher percentage identity as well as a lower <i>E</i>-value of the majority of the blasted unigenes reflects the strength of the blast analysis. Furthermore, blast species distribution (Supplementary Data <a href="#SM10">1</a>) showed that <i>Marchantia polymorphia</i> and <i>Physcomitrella patens</i> were the most abundant homologous species. Indeed, these two species are the closest relatives to <i>R. marginata</i>. The most represented species in blast hits are shown in Figure <a href="#F1">1A</a>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>BLAST analysis. <b>(A)</b> Blast species distribution of closely related bryophyte species against NCBI non redundant (nr) database with an <i>E</i>-value of 1e-03. <b>(B)</b> Sequence similarity and <b>(C)</b> <i>E</i>-value distribution of BLAST results. </p>   </div> </div> <div class="interproscanfunctionalclassification" title="sec"> <div class="title" tagx="title" title="title">
InterProScan functional classification</div> <p>To enable a functional classification of unigenes, InterProScan (InterPro) <a href="https://www.ebi.ac.uk/interpro/">https://www.ebi.ac.uk/interpro/</a> (release 60) was used. From this analysis, 65,415 (74%) unigenes matched with at least one protein signature. In total 3,487 protein family, 3,529 domains, 515 sites and 108 repeats had significant similarity, with 19,760, 60,021, 1,922, and 4,378 unigenes, respectively. "Heat shock protein 70 family" was the most abundant, with 399 unigenes, followed by the Short-chain dehydrogenase/reductase SDR and "Chaperonin Cpn60/TCP-1 family" that had 357 and 284 unigenes for each category (Figure <a href="#F2">2A</a>). Within the domain signatures, 2,935 unigenes had the "P-loop containing nucleoside triphosphate hydrolase" domain (Figure <a href="#F2">2B</a>). The Tetra-tri-co-peptide repeat, the Leucine-rich repeat and the WD40 repeat were the most recurrent that were found in 232, 200 and 186 transcripts, respectively (Figure <a href="#F2">2C</a>). Furthermore, out of the predicted site signatures, 68% (349) of the signatures were from the conserved category, followed by 19% (96) that belonged to the active sites. The serine/threonine-protein kinase active site was the most prevalent site present in 205 unigenes. The top most representative protein signatures for the four protein classification databases are shown in Figures <a href="#F2">2A–D</a>, Supplementary Figure <a href="#SM18">1A</a> and Supplementary Data <a href="#SM11">2</a>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>InterProScan functional analysis. <b>(A)</b> Annotation distribution of most significant protein families <b>(B)</b> domain identified <b>(C)</b> prominent repeats observed and <b>(D)</b> important sites determined. X-axis represents the number of genes associated with the respective protein signatures on Y-axis. </p>   </div> </div> </div> <div class="geneontologyclassification" title="sec"> <div class="title" tagx="title" title="title">
Gene ontology classification</div> <p>To determine the shared dynamic proteins among organisms and to understand the putative biological functions, gene ontology (GO) was analyzed (Ashburner et al., <a href="#B6">2000</a>). Based on the homology results from the NCBI nr database, GO terms were mapped to each unigene using a comprehensive annotation suite BLAST2GO (Conesa et al., <a href="#B17">2005</a>). Predicted InterPro scan protein signatures were also mapped for GO terms and finally an integrated annotation of unigenes was obtained after merging with blast-derived GO annotation. From the analysis, a total of 116,543 GOs were predicted from 4,820 functions. These GO terms were divided into three major categories: molecular function, biological process and cellular component. Fifty-one percent of the GOs (59,279) were predicted for the molecular function category (MF) followed by 34% (39,950) for biological processes (BP) and 15% (17,314) for cellular components (CC). Within the MF category, ATP binding, DNA binding, and oxidoreductase activity were significantly represented, with 5,359, 2,957, and 2,606 GOs for each type of function. Metabolic process (1,884) and regulation of transcription, DNA templated (1,744) were the most active BPs. CC were enriched with the membrane, membrane and cytoplasm integral components, with 4,080, 3,445, and 1,386 GOs. In total, 4,820 GO categories from 2,353 molecular functions, 1,849 biological processes and 618 cellular components were significantly present in the liverwort transcriptome. The total numbers of unigenes mapped with each GO category were 39,864, 31,268, and 14,369 for MF, BP, and CC, respectively. The ratio of GOs/unigenes was 1.49, with a minimum of one GO term per unigene to 27 GOs per unigenes (Figure <a href="#F3">3A</a>, Supplementary Figures <a href="#SM1">1B,C</a> and Supplementary Data <a href="#SM12">3</a>). From the complete annotation process we were able to generate 49,167 unigenes with functional annotations. In summary, 56% of the <i>de novo</i> assembled transcriptomic pool was annotated (Figure <a href="#F3">3B</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>Gene Ontology analysis. <b>(A)</b> Gene ontology functional annotation for biological process (BP), Molecular function (MF), and Cellular component (CC). <b>(B)</b> Overall annotation results from BLAST, InterProScan (IPR), Gene ontology (GO), and KEGG pathway databases. </p>   </div> <p>The remaining 44% of the non-annotated were considered as unknown or orphan genes. Despite the lack of significant annotation, these unknown genes may have a possible functional potential as they were generated from the expressed transcripts (mRNA). They might either have potential binding sites for transcription factors (TFs), SSRs, or be ncRNAs. </p> </div> <div class="keggpathwaynetworkanalysis" title="sec"> <div class="title" tagx="title" title="title">
KEGG pathway network analysis</div> <p>One of the objectives in establishing a reference liverwort ( <i>R. marginata</i>) transcriptome was to provide a resource for further NGS and proteomics studies. We further estimated the coverage of our newly assembled annotated reference transcriptome by determining the minimum coverage of general pathways involved in the MF and BP in <i>R. marginata</i>. To determine the rate of coverage, we analyzed more than 518 pathways listed in the Kyoto Encyclopedia of Genes and Genomes (KEGG) database. To identify active biological pathways, unigenes were scanned for the pathways based on analysis using KEGG ( <a href="http://www.genome.jp/kegg/">http://www.genome.jp/kegg/</a>). As a result, 11,421 candidate genes were identified that encoded 1,554 enzymes in 145 biosynthesis pathways. These pathways belonged to 15 functional categories from "network of metabolism," "environmental information processing," "organismal processes" and "genetic information processing" (Figure <a href="#F4">4A</a> and Supplementary Data <a href="#SM13">4</a>). Among the identified pathways, the carbohydrate metabolism pathway was mapped with the highest number of 323 enzymes. We identified 53 enzymes for secondary metabolite biosynthesis and 29 from the terpenoid metabolism and polyketide pathways (Supplementary Table <a href="#SM5">5</a>). Moreover, the identified enzymes were broadly distributed into six major enzyme classifications (Figure <a href="#F4">4B</a>, Supplementary Table <a href="#SM6">6</a>). Transferases were the most abundant enzymes, followed by the oxidoreductases and hydrolases. Distribution of the identified enzymes per pathways is shown in Figure <a href="#F4">4C</a>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>KEGG Pathway analysis. <b>(A)</b> Identification of genes encoding enzymes in 15 functional pathways among four main categories. <b>(B)</b> Distribution of enzymes per pathway, X-axis shows the number of pathways, while Y-axis represents the number of identified enzymes for each pathway. <b>(C)</b> Enzyme code distribution according to the enzyme class (EC) type. </p>   </div> </div> <div class="candidategenesforthebiosynthesisofcannabinoid-likecompoundsinr.marginata" title="sec"> <div class="title" tagx="title" title="title">
Candidate genes for the biosynthesis of cannabinoid-like compounds in <i>R. marginata</i> </div> <p>In <i>C. sativa</i> the biosynthesis of cannabinoids, such as tetrahydrocannabinolic acid (THCA), is the result of oxidative cyclization of cannabigerolic acid (CBGA) (Sirikantaramas et al., <a href="#B77">2004</a>; Taura et al., <a href="#B84">2007</a>; Stout et al., <a href="#B82">2012</a>; Andre et al., <a href="#B3">2016</a>; Zirpel et al., <a href="#B98">2017</a>). CBGA is the central precursor for cannabinoid biosynthesis, which is catalyzed by the alkylation of olivetolic acid (OA) with geranyl diphosphate (GPP). OA and GPP are derived from the polyketide and 2-C-Methyl-D-erthritol 4-phosphate (MEP) pathways, respectively. GPP is derived from a condensation reaction between two isoprene units, dimethylallyl pyrophosphate and isopentenyl pyrophosphate (Croteau and Purkett, <a href="#B18">1989</a>). Remarkably, we have identified all the potential homologous genes in <i>R. marginata</i> encoding the enzymes required for the conversions of pyruvate and D glyceraldehyde 3 phosphate into GPP, and also confirmed the expression through qRT-PCR (Figures <a href="#F5">5A–C</a>). The genes identified for GPP synthase had homology of 79.7% at the protein level with the GPP synthase in <i>Capsicum annuum</i>, and 75% with the large subunit of GPP synthase in <i>C. sativa</i>. In addition, we found that all the 15 potential binding sites of both isoprene units were conserved. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <p>Candidate precursor genes for cannabinoid biosynthesis in <i>Radula marginata</i>. <b>(A)</b> Identification of candidate genes for cannabinoid biosynthesis in <i>R. marginata</i>. GPP transcripts as well as all the upstream genes were identified. Genes with Rm-tct_xxx prefix were identified in this study. <b>(B)</b> Transcriptomic expression of identified genes determined from FPKM. <b>(C)</b> Relative expression of candidate gene by real time quantitative PCR (RT-qPCR). </p>   </div> <p>In contrast to OA in <i>C. sativa</i>, which is synthesized from the condensation of aliphatic CoA-tethered hexanoyl (hexanoyl-CoA), we found an analogous compound, stilbene acid (SA), in <i>R. marginata</i>. Compared with OA, SA catalyzes the condensation reaction from the CoA-tethered phenyl propanoid i.e., coumaroyl (P-coumaroyl-CoA). However, both require three molecules of malonyl-CoA by polyketide synthase (Fellermeier et al., <a href="#B23">2001</a>; Austin et al., <a href="#B7">2004</a>). Acetyl CoA carboxylase is the enzyme required for the carboxylation of acetyl-CoA to form malonyl-CoA, and has been identified in this study. Also, unlike the hexanoyl-acyl carrier protein, P-coumaroyl-CoA provides the "shuttle service" for the intermediate polyketide as well as substrates in the formation of SA by the enzymatic reaction of STS. The STS identified from <i>R. marginata</i> had 85% homology with chalcone synthase (CHS) and 84% homology with stilbencarboxylate synthase from <i>M. polymorph</i>. STS diverged from CHS as a result of gene duplication, from an evolutionary perspective (Tropf et al., <a href="#B88">1994</a>). Both catalyze the same reaction with different cyclization mechanisms, which lead to different products (Austin and Noel, <a href="#B8">2003</a>). </p> <p>The CBGA analog was identified with the HPLC-ESI-MS-MS approach (Supplementary Figure <a href="#SM19">2A</a>) and the metabolite with the same m/z was also observed from GC-MS analysis (Supplementary Figure <a href="#SM19">2B</a>, Supplementary Data <a href="#SM14">5</a>). The positive ESI mass spectrum of the CBGA bibenzyl analog showed an [M+H]+ ion at m/z 395, a predominant [M+H-H2O]+ ion at m/z 377, and fragment ions at m/z 271 [M+H-C9H16] and m/z 267 [M+H-H2O-C8H4]. The ion m/z 253 is probably the result of degradation processes eliminating two protons, resulting in [M+H-H2O-C9H16]. Together these indicate the presence of the CBGA analogue's bibenzyl structure in the methanolic extracts of <i>R. marginata</i>. </p> <p>To gain insights into the STS identified in <i>R. marginata</i>, we carried out structural analysis of the protein sequence. For this purpose, we used homology modeling to develop the structure of STS in comparison with the CHS-like PKS (PDB ID code 3awk) from <i>Huperzia serrata</i> and STS (PDB ID code 2p0u) from <i>Marchantia polymorpha</i>. Structural analysis revealed that the <i>R. marginata</i> STS shared 64 and 60% homology with these, respectively. In the model prediction, Lys-Ala was replaced with Thr-Ser, Thr-Glu, Arg-Lys, and Ala-Lys at positions 20–21, 46–47, 74–75, 312–313, respectively. Similarly, these variable sites except 20–21 were also found in STS from <i>M. polymorpha</i>, where they code Pro-Ala, Glu-Ser, and Ala-Lys amino acids. Likewise, OA in <i>C. sativa</i> also showed the same variable sites with the amino acid changes Thr-Ala, Thr-Gln, Ala-Gln, and Glu-Ala. Since STS diverged from CHS and OA was identified as the homolog of CHS, we speculate that these variable sites might have the potential for the diversification of the parental gene. However, the mechanism of the divergence and their catalytic activity of these potential sites need to be explored (Figure <a href="#F6">6</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 6</span> </a> </div>   <p> <b>(A)</b> Crystal structure of stilbene synthase model compared with Chalcone synthase- like polyketide (Left, 3awk.1) and Stilbene carboxylate synthase from <i>Marchantia polymorpha</i> (right, 2p0u.1. A). <b>(B)</b> Quality estimate of local similarity, Z score of the predicted STS model in <i>Radula marginata</i>. <b>(C)</b> Alignment of the identified stilbene synthase in <i>Radula marginata</i> with the respective template model, blue boxes are the beta sheets, yellow boxes in the alignment are the variable sites for both models used as well as in olivetolic acid from <i>Cannabis sativa</i>. Taken with permission from " <a href="https://swissmodel.expasy.org">https://swissmodel.expasy.org</a>" (Arnold et al., <a href="#B4">2006</a>; Biasini et al., <a href="#B9">2014</a>). </p>   </div> </div> <div class="identificationoftranscriptionfactors" title="sec"> <div class="title" tagx="title" title="title">
Identification of transcription factors</div> <p>To understand gene regulation, unigenes were scanned using a Hidden Markov Models (HMM) (Finn et al., <a href="#B26">2011</a>)-based approach against the pfam (Finn et al., <a href="#B27">2016</a>) as well as the PlnTFDB (Pérez-Rodríguez et al., <a href="#B63">2009</a>) and PlantTFDB (Jin et al., <a href="#B40">2014</a>) databases. TFs were predicted according to the family assessment rule as described in PlantTFDB. To increase the accuracy, TFs with an <i>E</i>-value less than 1e-05 as well as an identity percentage &amp;lt; 50 were excluded from further analysis. After filtering we identified 1,085 TFs belonging to 39 TF families (Figure <a href="#F7">7</a>). We found that putative TF encoding regions were present in 3,449 unigenes. The total number of TFs identified in our study was higher than previously reported TFs in two desiccation-tolerant moss species, <i>Syntrichia caninervis</i> and <i>Bryum argenteum</i>, which contained 778 and 770 TFs, respectively. However, the number of TFs was less than in the model moss <i>P. patens</i>, which contains 1,156 TFs. Moreover, the 39 identified TF families in <i>R. marginata</i> were less than in the above-mentioned species, which contained 49, 50, and 53 TF families for each, respectively. Furthermore, the number of transcripts having a TF-encoding region was considerably higher than those in already-reported transcripts. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 7</span> </a> </div>   <p>Comparative analysis of Transcription Factor families from <i>Radula marginata</i> and related moss species. TF families are on X-axis while number of TFs for each family shown on Y-axis. Green dotted box represents abundant TF families; those in red dotted box are less abundant, green solid-line box is for TF families that are exclusively observed in <i>R. marginata</i> compared with related moss species, while red solid-line box represents TF families that were not observed in <i>R. marginata</i>. </p>   </div> <p>TF families are highly conserved in eukaryotic organisms, especially in plants. Despite the sequence conservation, the number of TFs for specific families varies among different species. This variation might be due to evolution or species specification. As for other plant species, evolutionary expansion/contraction was also observed in this study. For example, comparative analysis of three closely related moss species showed that the number of TFs within the ARF, C2H2, and CH3 families was higher in our data than in <i>B. argenteum</i>. In contrast, the B3 and bHLH families were relatively less abundant in the <i>R. marginata</i> transcriptome than in <i>S. caninervis</i> and <i>B. argenteum</i>. Moreover, six TF families—BSD, CSD, DbpA, FHA, LIM, tify, and TIG—were explicitly predicted in our data, confirming evolutionary expansion of TFs. On the other hand, we could not find any target sequences from B3 and trihelix TF families, which might have been due to evolutionary contraction. On this basis we might assume that the prediction of some TF families, as well as the apparent lack of other TF families, was due to evolutionary expansion and contraction, respectively. </p> <p>The TF families: MYBs, Basic Leucine Zipper Domain (bZIP), AP2/ERF family proteins, NAC, Basic helix-loop-helix (bHLH), and DNA-binding One Zinc Finger (DOF), are considered those most closely involved in the regulation of secondary metabolism, development and growth in many plant species (Vom Endt et al., <a href="#B90">2002</a>; Li et al., <a href="#B48">2015</a>; Zhu et al., <a href="#B96">2015</a>). Interestingly, we found 156 TFs belonging to the above-mentioned families, with 66, 39, 9/22, 10, 8, and 2 TFs, respectively. Of these, MYB TFs were the most abundant, contributing up to 42% of the total TFs, some of which are proposed to regulate secondary metabolism by forming regulatory complexes with other TFs, such as bHLH and WD40 repeats in the regulation of phenylpropanoids (Espley et al., <a href="#B21">2007</a>; Liu et al., <a href="#B51">2015</a>). Moreover, the regulation of terpenoids has been associated with the APETALA2 (AP2), WRKY, and MYC families (Broun et al., <a href="#B14">2006</a>; Spyropoulou et al., <a href="#B79">2014</a>). In total, nine TFs from AP2 and three from WRKY were identified from our data. These predictive TFs might regulate aspects of secondary metabolism and terpenoid biosynthesis in <i>R. marginata</i> (Supplementary Table <a href="#SM7">7</a> and Supplementary Data <a href="#SM15">6</a>). </p> </div> <div class="simplesequencerepeats(ssrs)identification" title="sec"> <div class="title" tagx="title" title="title">
Simple sequence repeats (SSRs) identification</div> <p>To identify SSRs, all the unigenes were screened using a microsatellite satellite identification tool (MISA) ( <a href="http://pgrc.ipk-gatersleben.de/misa/">http://pgrc.ipk-gatersleben.de/misa/</a>). A total of 2,041 SSRs (mono to hexa-) were identified from 87,460 assembled transcripts (Figure <a href="#F8">8A</a>). Mononucleotide repeats can encounter a problem with a higher rate of homo-polymer errors associated with genotyping as well as sequencing (Gilles et al., <a href="#B33">2011</a>; Zhang and Huang, <a href="#B95">2016</a>). Hence, these were excluded from further SSR analysis. Of the remaining 1,577 di-nucleotide to hexa-nucleotide repeats, 90% of the SSRs belonged were di-nucleotides and tri-nucleotides. Tetra-, penta-, and hexa-nucleotide SSRs were found only in trace amounts, with 4, 3, and 3%, respectively (Figure <a href="#F8">8B</a>). The linkage potential of SSRs to loci was much higher with those SSRs identified from the annotated unigenes than for non-annotated (Li et al., <a href="#B49">2014</a>). The 1,577 SSRs were annotated with 1,217 unigenes with a minimum length of 12 bp. Moreover, 20% of the annotated unigenes (254) were found to have more than one SSR and 7% of the annotated SSRs (126) were found in compound formation. Based on sequence complementation, a total of 198 identified motifs were classified into 85 repeat types. Analysis showed that for di-nucleotide repeats AG/CT (12%) and AT/AT (5%) were the most abundant, while for trinucleotide repeats AGC/CTG (19%) as well as CCG/CGG (17%) showed the highest frequency. These specific repeats accounted for 53% of the total repeats found (Figure <a href="#F8">8C</a>, Supplementary Table <a href="#SM8">8</a> and Supplementary Data <a href="#SM16">7</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 8</span> </a> </div>   <p>Identification of simple sequence repeats. <b>(A)</b> Statistic of simple sequence repeats. <b>(B)</b> Frequency of repeat number from 5 to 86 for di-, tri-, tetra-, Penta-, and hexa- type of SSR repeats. Y-axis represents the number of genes for each category of repeats type according to the repeat number on X-axis. <b>(C)</b> Most abundant di- and tri- repeats type distribution. </p>   </div> <p>Repetition of the SSR motif varied from a minimum of five times up to 46 (86 for compound SSR). Although the repeat frequency was observed up to 86, the significance proportion of the motifs were below the repetition value of 10. Only 0.01% of the motifs were above 50, which was almost negligible as well as not being from the annotated SSRs. Similarly, a small fraction of motifs, 4.03%, were found to have a repetition of above 10. A significant proportion of the motifs lay within the repetition unit of 5 (48.54%), followed by the repetition of 6, 7, 8, and 9 with 26.46, 11.90, 5.65, and 1.46%, respectively. Furthermore, distribution of the repeat number within the repetition of 5 was decreased with the increase of motif length from tri to hexanucleotide. Trinucleotide motifs comprised the highest proportion, with 88%, followed by the tetra- (4%), penta- (3.4%), and hexa-nucleotides (1.4%) (La Rota et al., <a href="#B46">2005</a>; Hisano et al., <a href="#B37">2008</a>; Cloutier et al., <a href="#B16">2009</a>). This highest proportion of trinucleotide repeats was directly proportional to the accuracy. The possible reason for this is that expansion and contraction in tetra-, penta-, and hexa-nucleotide repeats may lead to a frame-shift mutation in the coding region (Metzgar et al., <a href="#B56">2000</a>; Morgante et al., <a href="#B58">2002</a>). </p> </div> <div class="predictionofnon-codingrnas" title="sec"> <div class="title" tagx="title" title="title">
Prediction of non-coding RNAs</div> <p>To identify non-coding RNA genes (ncRNA), non-annotated unigenes were scanned for functional RNAs against Rfam database. Rfam <a href="http://rfam.xfam.org/">http://rfam.xfam.org/</a> is a collection of RNA families and has three functional categories of non-coding RNA genes, structured <i>cis</i>-regulatory elements and self-splicing RNAs. ncRNA has been demonstrated to play roles in the regulation of gene expression at the post transcriptional level (Eddy and Hughes, <a href="#B20">2001</a>) as well as in maintaining genome stability by guiding RNA modification (Moazed, <a href="#B57">2009</a>). ncRNA genes produce a direct functional RNA molecule rather than being translated into proteins (Mattick and Mattick, <a href="#B55">2010</a>). From the analysis we predicted 2,043 ncRNA genes that belong to rRNA (1985), tRNA (46), sRNA (9), and snRNA (3) types (Supplementary Data <a href="#SM17">8</a>). For the category of <i>cis-</i> regulation nine genes have significant homology. In order to determine the functionality of these predicted non coding genes further studies are required. </p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Next generation sequencing has become an essential tool for studying the genomic constitution of living organisms, especially those with high genome complexity. Despite the relatively low cost of sequencing, <i>de novo</i> assembly of whole genomes without prior sequence information is still costly and is computationally resource intensive. Compared with whole genome sequencing, <i>de novo</i> transcriptome analysis has made it possible to understand the genetic architecture of those organisms without a reference genome, at a low cost, and it is also computationally less intensive. Thus, a <i>de novo</i> transcriptome study is a valuable tool for identifying new genes, molecular markers, regulatory elements and the expression profile of genes (Verk et al., <a href="#B89">2013</a>). Indeed, this advancement has made it possible to study less well characterized species, for example, mosses and liverworts that have not been extensively studied compared with higher land plants such as rice (Zhang et al., <a href="#B94">2010</a>), poplar (Qiu et al., <a href="#B64">2011</a>) sesamum (Wei et al., <a href="#B91">2011</a>) as well as yeasts (Nagalakshmi et al., <a href="#B59">2008</a>) and animals (Feldmeyer et al., <a href="#B22">2011</a>). A <i>de novo</i> approach has previously been used for the moss species <i>S. caninervis</i> (Gao et al., <a href="#B32">2014</a>) and <i>B. argenteum</i> (Gao et al., <a href="#B31">2015</a>). Therefore, a <i>de novo</i> assembled transcriptome can be used to quantify the expression of genes and identify new genes. </p> <p> <i>Radula marginata</i> (Liverworts), have been identified to contain a diverse array of secondary metabolites of high structural and medicinal properties for over the past two decades, such as perrottetinene and its acid (Toyota et al., <a href="#B86">1994</a>, <a href="#B87">2002</a>). However, there are only a few studies on liverwort species and most of those are related to biodiversity and molecular clock to infer the evolutionary relationships to land plants (Alaba et al., <a href="#B1">2015</a>; Singh et al., <a href="#B76">2015</a>; Honkanen et al., <a href="#B38">2016</a>). While <i>Radula</i> is an important plant order, it has not been investigated at the molecular and genomic level. Until now, only 34 nucleotide and 13 proteins have been reported for this liverwort species within the NCBI public database. Here, we report for the first time a <i>de novo</i> assembled transcriptome of <i>R. marginata</i>. In the RNA-seq analysis, we have identified all the upstream genes of the central precursor in cannabinoid biosynthesis. We have also validated the presence of CBGA-analogous by using HPLC ESI/MS-MS and GC-MS approach. We could identify all the prerequisite enzymes for the precursor which are likely to be conserved. However, we have identified stilbene synthase (SA) instead of olivetolic acid (OA), one of the CBGA precursor in <i>C. sativa</i>. Additionally, transcription factors (TFs), microsatellite markers (SSRs) and ncRNAs have also been identified. This <i>de novo</i> assembled reference transcriptomic dataset would provide a resource for exploring the <i>R. marginata</i>. </p> <p>The sequence contents and quality of <i>de novo</i> assembled transcriptomes essentially depend on input material and further bioinformatics processing of the data. Assembly of the raw reads into contigs is the first step in the <i>de novo</i> transcriptomic study. We assembled 217,702 contigs from 30,981,418 raw reads. Because of splice site variation, assembly from Illumina sequencing generates many transcripts of different lengths for a single gene, where selection and identification of a single full-length transcript is a critical step in <i>de novo</i> assembly (Schliesky et al., <a href="#B74">2012</a>; Steijger et al., <a href="#B80">2013</a>). To remove this transcript redundancy (see section Materials and Methods) we finally obtained 87,460 unigenes. The total number of contigs and unigenes identified in our study was higher than the number of contigs (106,066) and unigenes (57,247) described in previously <i>de novo</i> assembled moss species ( <i>B. argenteum</i>). This further illustrates the quality and accuracy of our assembly. Furthermore, the average length, GC contents and N50 value of these unigenes were comparable with the results from previous studies (Liang et al., <a href="#B50">2013</a>). </p> <p>To estimate the quality of <i>de novo</i> assembly, raw reads were mapped back to the contigs generated from sequence assembly using bowtie2 with default parameters. Seventy-two percent of reads were successfully mapped back to the contigs and these mapping results are congruent with previous studies. However, the parameters used to check the quality of <i>de novo</i> assembly provide only rough estimates. Therefore, it is important to select all the transcripts that have coding potential and annotate all these transcripts to assign them a putative gene function. For this purpose, we used BLAST algorithms, such as blastn, blastx, and blastp with databases including NCBI non-redundant protein (nr), nucleotide (nt), Swiss-Prot (UniProtKB), cosmos (v1.6, <i>P</i>. patens proteins) and <i>M. polymorphia</i> genomes using a cut off value for homology search of at least 65% of identity and a stringent <i>E</i>-value of 1e-05. This allowed for 78% protein-coding transcripts with different degrees of homology. </p> <p>Functional characterization of unigenes provides insights into the active BP, MF, and CC within an organism. To determine the functionality of genes, regarding regulation and expression, annotation is desirable. Therefore, all the unigenes were blasted with InterProScan (IPR), Gene ontology (GO), and Kyoto Encyclopedia of Genes and Genomes (KEGG) databases. In contrast to BLAST, 74% of the unigenes were mapped with the protein signature from 14 databases allied with IPR. Out of the total assembled unigenes scanned for GO terms, 56% of the unigenes had at least one GO which was more than the identified GO terms reported in the bryophyte moss species <i>S. caninervis</i> (48%) and in the non-model transcriptome of <i>Bacopa monnieri</i> (47%) (Jeena et al., <a href="#B39">2017</a>). However, <i>B. argenteum</i> and <i>P. patens</i> (Rensing et al., <a href="#B67">2008</a>; Zimmer et al., <a href="#B97">2013</a>) have 64 and 58% of GOs which is little higher than in our study. This might be due to an incomplete transcriptome or different cut-off values for transcriptome assembly processes. </p> <p>To determine the interaction of functionally annotated unigenes of <i>R. marginata</i>, we performed KEGG pathway analysis (Kanehisa et al., <a href="#B41">2006</a>). This analysis elucidated the potential signal transduction biosynthesis pathways that were abundantly represented from our transcriptomic data. Out of 49,167 annotated unigenes, 11,421 were mapped with 145 signal transduction pathways which were greater than in other bryophyte species. For instance, <i>D. hirsute, S. caninervis</i>, and <i>B. argenteum</i> revealed the presence of 95, 119, and 127 pathways, respectively (Gao et al., <a href="#B32">2014</a>, <a href="#B31">2015</a>; Singh et al., <a href="#B76">2015</a>). Our results are also in line with those for other non-model organisms like <i>Sesamum indicum L</i>., <i>Camelina sativa</i>, and <i>Callosobruchus maculatus</i> which showed 116, 119, and 119 pathways identified from their respective transcriptomes (Wei et al., <a href="#B91">2011</a>; Liang et al., <a href="#B50">2013</a>; Sayadi et al., <a href="#B73">2016</a>). The biosynthesis of the terpenoid backbone was the most significant pathway in the metabolism of terpenoids and polyketides with 84 unigenes identified that encoded 12 enzymes. Overall, network analysis revealed the presence of almost all the pathways involved in metabolism, of which carbohydrate metabolism was the most dominant with 323 enzymes followed by amino acid metabolism (275), metabolism of co-factors and vitamins (135) and lipid metabolism with 114 enzymes mapped to all of the pathways for each. However, we could not identify some of the secondary metabolic pathways, for example, zeatin, tetra cycline, and brassino steroid biosynthesis. It might be because the bryophyte transcriptomes have greater metabolic versatility than land plants which might have favored alternative metabolic pathways during evolution (Rensing et al., <a href="#B68">2002</a>; Oliver et al., <a href="#B60">2004</a>; Lang et al., <a href="#B44">2005</a>; Wood and Duff, <a href="#B93">2009</a>). </p> <p>We have also identified the precursor genes for the synthesis of SA and GPP. Alkylation of OA with GPP results in the formation of CBGA which is the primary precursor for the cannabinoid biosynthesis in <i>C. sativa</i> (Fellermeier and Zenk, <a href="#B24">1998</a>). In contrast, we have identified SA in <i>R</i>. marginata and it is likely that SA acts in <i>R</i>. marginata as a first intermediate in place of OA to form CBGA-analogous. A metabolomics study also revealed the identification of the CBGA analogous and confirms our hypothesis that SA acts as the first intermediate where stilbene is the backbone of the identified compound. It is worth noting that the THCA like compound identified in <i>R. marginata</i> (Toyota et al., <a href="#B87">2002</a>) also has the stilbene backbone. </p> <p>In our analysis, we were unable to annotate 44% of the transcriptome, and these were named as unknown genes. A similar percentage of unknown genes has also been observed in other moss species such as <i>S. caninervis</i> (41%), <i>B. argenteum</i> (36%) as well as other closely related liverwort species like <i>M. polymorpha</i> (43%) (Sharma et al., <a href="#B75">2014</a>). These high numbers of unknown genes might be due to the unavailability of sufficient genomic information for mosses in general and <i>R. marginata</i> specifically. To date, the only available reference genome for bryophytes is the moss species <i>P. patens</i> which also has 42% of unknown genes (Rensing et al., <a href="#B67">2008</a>; Ortiz-Ramírez et al., <a href="#B61">2016</a>). Therefore, we speculate that these unknown genes might play a role in gene regulation since they were generated from the expressed transcripts (mRNA). These unknown genes might be orphan genes and may have binding sites for TF, SSRs or ncRNAs. </p> <p>The identified unigenes that have potential binding sites for 1088 TFs are consistent with those in other moss species such as <i>S. caninervis</i> with 778 and <i>B. argenteum</i> with 778 TFs, while <i>P. patens</i> has 1,156 TFs. However, the numbers of identified TF families are less than in the other related moss species. This variable number of TFs might be due to speciation events during evolution which explains the phenomenon of evolutionary expansion/contraction of TFs. We found six TF families that are unique to <i>R. marginata</i> which may be a result of expansion and three TF families that could not be detected from comparative analysis with related moss species. Although the number of transcription factors within a family varies, the sequence tends to remain conserved because of the conserved nature of TFs. </p> <p>Our analysis of SSRs also revealed a large number of di- to hexa- repeats, of which 90% belongs to the di- and tri- repeat types. These data are correlated with <i>P. patens</i> where 91% of the SSRs comprised of di- and tri-nucleotide repeats. Some plant species such as sunflower, sesame, canola, <i>Arabidopsis</i>, peanut, sugar beet, cabbage, soybean, sweet potato, pea, and grape have a higher number of dinucleotide repeat motifs while trinucleotide type repeats are more frequent SSR motifs in cereals such as barley, rice, and wheat (Kumpatla and Mukhopadhyay, <a href="#B43">2005</a>; La Rota et al., <a href="#B46">2005</a>; Wei et al., <a href="#B91">2011</a>). Although repeat units were observed with a minimum of five to a maximum of 46 times, significant proportions of the repeat types were associated with five repeat units, as also observed in moss species. </p> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>We describe the first high quality <i>de novo</i> assembled transcriptome, and have annotated the highest number of genes to date in <i>R. marginata</i>. Moreover, identification of the precursor genes in the cannabinoid biosynthesis pathway suggests that the cannabinoid pathway is likely to be conserved in lower and high land plants with the exception of first intermediate. These findings require further experimental work to confirm this proposed novelty. Overall, this study would serve as a new transcriptomic resource among the bryophyte species and also proposes <i>R. marginata</i> as an alternate to <i>C. sativa</i> for cannabinoid-like compounds. </p> </div> <div class="materials and methods" title="materials and methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="liverwortscollection" title="sec"> <div class="title" tagx="title" title="title">
Liverworts collection</div> <p> <i>Radula marginata</i> (liverworts) samples were collected from their basal habitat at Waitakere Ranges Regional Park, New Zealand. For metabolite extraction, material was transported in sealed plastic zip lock bags in liquid nitrogen to the Technical University Dortmund, Germany. For RNA extraction, samples were collected in liquid nitrogen and then stored at −80°C at The New Zealand Institute for Plant &amp;amp; Food Research Limited (PFR), Auckland, 1142, New Zealand. </p> </div> <div class="extractionofrnas" title="sec"> <div class="title" tagx="title" title="title">
Extraction of RNAs</div> <p>Total RNA was extracted from six samples using RNeasy™ Plant Mini Kit (Qiagen N.V., The Netherlands) according to the manufacturer's recommendations. To eliminate possible DNA contamination On-column DNaseI (Qiagen) digestion was added to the extraction protocol. RNA concentration was measured with Nanodrop ND-1000 spectrophotometer (NanoDrop Technologies, Wilmington, DE, USA) and quality was checked on agarose gel electrophoresis using Bioanalyzer agilant 2100 (Agilent Technologies, Santa Clara, CA, USA) was used to determine the integrity by RNA integrity number (RIN). Enrichment of mRNA by removing rRNA from total RNA was carried out by oligo (d) T beads (Qiagen, Hilden Germany). Purified mRNA was then transported in RNAStable plate (Biomatrica, San Diego, USA) to IGA Technology Services Udine, Italy. </p> </div> <div class="cdnalibrarypreparationandilluminangssequencing" title="sec"> <div class="title" tagx="title" title="title">
cDNA library preparation and illumina NGS sequencing</div> <p>The total RNA from each of six samples was pooled and 2.5 μg of the pooled RNA was used to prepare cDNA according to the instructions of Illumina® (Illumina, San Diego, USA). The further steps in the preparation such as adapter insertion, interruption of the fragment, size selection and PCR amplification were performed at IGA Technology, Italy. Sixty million paired end reads of a length 250 bp were generated using Illumina HiSeq-2000 platform. </p> </div> <div class="qualitycontrol" title="sec"> <div class="title" tagx="title" title="title">
Quality control</div> <p>Illumina Hiseq-2000 sequencing platform (Illumina, San Diego, CA, USA) was used to sequence the transcriptome. FASTQC (version 0.11.24) ( <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>) was used to determine the quality of the raw read. The base quality score was set to (Q20 = 1%) and all the reads below Q20 were considered as low quality (Andrews). Trimmomatics (version 0.36) was used to trim the low quality reads from 3′ and 5′ end using sliding window, leading and trailing to 4, 5, and 5 bases, respectively (Bolger et al., <a href="#B11">2014</a>). Adapter sequences present in the sequences were removed with cutadapt (version 1.9.1) ( <a href="http://code.google.com/p/cutadapt/">http://code.google.com/p/cutadapt/</a>) (Martin, <a href="#B54">2011</a>). Read lengths with &amp;lt; 50 bp and reads with ambiguous "N" &amp;gt;5% were also dropped. </p> </div> <div class="dataprocessinganddenovotranscriptomicassembly" title="sec"> <div class="title" tagx="title" title="title">
Data processing and <i>de novo</i> transcriptomic assembly </div> <p>The sequencing quality of raw reads was determined with FASTQC ( <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>) software package. To avoid any false positive gene prediction adapters, low-quality base (higher than 20% nucleotides having a quality value ≤ 10) and reads (ambiguous nucleotide &amp;gt;5%) were removed with cutadapt and trimmomatic, respectively (Martin, <a href="#B54">2011</a>; Bolger et al., <a href="#B11">2014</a>). <i>In silico</i> read normalization was done before the assembly and then filtered normalized reads were <i>de novo</i> assembled using Trinity software package with default k-mer size of 25 (Haas et al., <a href="#B34">2013</a>). Inchworm, Chrysalis and Butterfly are the three independent modules for the assembly in Trinity. At the first step of assembly, Inchworm, raw reads are used to form a k-mer catalog followed by Chrysalis, using de Bruijn graph approach (based on distance and relation) to constructs the contigs. Finally, Butterfly connects all possible de Bruijn graph to construct the full-length transcripts as well as their splice variants as a result of alternative splicing. </p> <p>To remove redundancy, assembled reads were clustered using CD-HIT-EST tool (Fu et al., <a href="#B29">2012</a>). To determine the protein coding region each transcript (putative gene) was <i>in silico</i> translated using TransDecoder ( <a href="http://transdecoder.github.io">http://transdecoder.github.io</a>). Among all the predicted frames, ORFs (open reading frames) peptide candidates were selected. Subsequently, the quality of the transcriptome was accessed by mapping raw reads back to the assembled reads using Bowtie2 (Langmead and Salzberg, <a href="#B45">2012</a>). Since <i>de novo</i> assembly is challenging for accurate mapping quantification, RNA-seq by Expectation-Maximization (RSEM) was used to determine the relative abundance of transcripts (Li and Dewey, <a href="#B47">2011</a>) which quantifies assembled transcripts with a high degree of accuracy by first preparing a reference set followed by the read mapping and abundance estimation. Afterwards, gene expression was calculated and reported as Transcripts per million (TPM) and fragment per kilobase of transcript per million mapped reads (FPKM) expected count numbers. Finally, the clustering of different isoforms of the transcripts (genes), selection of the longest putative ORF peptide candidate and the transcripts with at least TPM value &amp;gt; 1 were defined as a unigene. </p> </div> <div class="functionalannotationandcharacterizationofunigenes" title="sec"> <div class="title" tagx="title" title="title">
Functional annotation and characterization of unigenes</div> <p>All the unigenes were searched against different public databases. BLASTx was used to determine the similarity against the NCBI non-redundant protein database (nr), Swiss-Prot, NCBI nucleotide database (nt) with stringent parameters ( <i>E</i>-value 1e-05). For the functional characterization of unigenes, ORFs were scanned against InterProScan (IPR) release 60 (Finn et al., <a href="#B25">2017</a>). IPR suite scans unigenes against protein signatures (predictive models) from 14 affiliated databases resulting in a comprehensive protein annotation that includes protein superfamily classification, specific protein domains, repeats, and sites. Based on the homology results from nr databases, GO terms were mapped to each unigene using a comprehensive annotation suite BLAST2GO (Conesa et al., <a href="#B17">2005</a>). IPR protein annotation signatures were also mapped to GO terms and finally an integrated annotation was obtained after merging it with blast-derived GO annotation. Metabolic pathway analysis was done by similarity searching using blastx against the KEGG database (Kanehisa et al., <a href="#B42">2004</a>). The resulting unigenes and enzyme code were assigned and mapped to their respective pathway map. </p> </div> <div class="identificationoftranscriptionfactors(tfs)" title="sec"> <div class="title" tagx="title" title="title">
Identification of transcription factors (TFS)</div> <p>Unigenes were scanned using Hidden Markov Model approach (HMM) in the HMMER software package against the Pfam, PlnTFDB, and PlantTFDB databases (Eddy, <a href="#B19">2011</a>; Finn et al., <a href="#B26">2011</a>, <a href="#B27">2016</a>). TFs were predicted according to the family assessment rule as described in PlantTFDB (Pérez-Rodríguez et al., <a href="#B63">2009</a>; Jin et al., <a href="#B40">2014</a>). The TFs retention criterion was set to an <i>E</i>-value less than 1e-05 and percentage identity of 50%. </p> </div> <div class="identificationofsimplesequencerepeats(ssrs)" title="sec"> <div class="title" tagx="title" title="title">
Identification of simple sequence repeats (SSRs)</div> <p>To identify SSRs, we used microsatellite identification tool (MISA) ( <a href="http://pgrc.ipk-gatersleben.de/misa/">http://pgrc.ipk-gatersleben.de/misa/</a>). All the unigenes were screened to identify potential sites for SSRs. Criteria were set such as minimum repeat unit number of 5 for di- to hexa- nucleotide repeats and a minimum repeat overlapping length of 100 bp. Mono nucleotide repeats were not included because of the higher error rate of homo polymer variation. Compound SSRs were also predicted with additional parameters as described above. </p> </div> <div class="quantitativereal-timepcranalysis" title="sec"> <div class="title" tagx="title" title="title">
Quantitative real-time PCR analysis</div> <p>For quantitative real time PCR (qRT-PCR), the total RNA was extracted from four biological replicates of <i>R. marginata</i>. cDNA was synthesized according to the manufacturer's recommendations (QuantiTect® Reverse Transcription, QIAGEN Group). qRT-PCR assay was performed, using the LightCycler® SYBR Green I Master (Roche Diagnostics, Germany). Four technical replicates were used for each sample and quantified based on relative expression levels and normalized against the housekeeping gene <i>Actin</i> of <i>R. marginata</i>, using LightCycler® Software (Roche; version 1.5). Gene specific primers used for the qRT-PCR are listed in the Supplementary Table <a href="#SM9">9</a>. </p> </div> <div class="hplcesi-ms/ms" title="sec"> <div class="title" tagx="title" title="title">
HPLC ESI-MS/MS</div> <p>One hundred milligrams of dried <i>R. marginata</i> material was grinded in 1 mL 80% methanol using a glass homogenizer. After centrifugation of the cell debris at 13,100 g at room temperature, the supernatant was analyzed by HPLC-ESI MS/MS analysis. Separation of the liverwort extract was performed by RP-HPLC (DAD) (Agilent 1260 Infinity HPLC, Waldbronn, Germany) equipped with a Poroshell 120 EC-C18, 2.1 × 100 mm, 2.7 μm column (Agilent, Waldbronn, Germany) with a flow rate of 0.7 mL min <sup>−1</sup> at 40°C under isocratic conditions [50% (v/v) H <sub>2</sub>O with 0.1% (v/v) FA, 50% (v/v) ACN]. The identity of the liverwort cannabinoids was confirmed by mass and tandem mass spectra using a Bruker compact™ ESI-Q-TOF (Bruker, Bremen, Germany) operating at a positive ionization mode. Fragmentation was achieved at collision energy of 20.0 eV. The methodology was described in detail (Zirpel et al., <a href="#B98">2017</a>). </p> </div> <div class="gc-msanalysis" title="sec"> <div class="title" tagx="title" title="title">
GC-MS analysis</div> <p>Fifty mg of the liquid nitrogen-ground <i>Radula</i> tissue was suspended in 100 μL of hexane, vortexed vigorously, and sonicated for 30 min. The samples were centrifuged at 16,000 g, for 10 min. The resulting supernatant was collected and analyzed. The GC/MS analysis was performed using TRACE 1310 gas chromatograph connected to a TSQ8000 triplequad MS (both Thermo Scientific). A DB-5 bonded-phase fused-silica capillary column (30 m length, 0.25 mm inner diameter, 0.25 μm film thickness) (J&amp;amp;W Scientific Co., USA) was used for separation. The GC oven temperature program was as follows: 2 min at 70°C, raised by 10°C/min to 300°C, and held for 10 min at 300°C. The total time of GC analysis was 36 min. Helium was used as the carrier gas at a flow rate of 1 mL/min. One microliter of each sample was injected in splitless mode. The initial injector temperature was 40°C for 0.1 min and after that time raised by 600°C/min to 350°C. The septum purge flow rate was 3 mL/min and the purge was turned on after 60 s. The transfer line and ion source temperatures were set to 250°C. Ion-source fragmentation was performed with 70 eV energy. Mass spectra were recorded in the mass range 35–650 m/z. </p> <p>Data acquisition, automatic peak detection, mass spectrum deconvolution, retention index calculation, and library search were done by XCalibur and AMDIS software. The metabolites were automatically identified by library search (NIST library); the analyte was considered as identified when it passed a quality threshold: i.e., similarity index (SI) above 700 and matching retention index ± 10. The artifacts (alkanes, column bleed, plasticizers, MSTFA, and reagents) were identified analogously, and then excluded from further analysis. To obtain accurate peak areas for the deconvoluted components, unique quantification masses for each component were specified and the samples were reprocessed. The obtained profiles were normalized against the sum of chromatographic peak area (using the TIC approach). </p> </div> <div class="structuralanalysis" title="sec"> <div class="title" tagx="title" title="title">
Structural analysis</div> <p>A STS structural model was built with SWISS-MODEL by comparative approach using PDB ID code 3awk and 2p0u as a template (Arnold et al., <a href="#B4">2006</a>; Biasini et al., <a href="#B9">2014</a>). </p> </div> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author contributions</div> <p>TH and OK: Wrote the first draft; TH, RE, and OK: Designed the experiment; TH and BP: Performed the experiments; TH and OK: Analyzed the data. TH, ME, RE, and OK: Wrote the manuscript. All authors contributed in the final version of the manuscript. </p> <div class="conflictofintereststatement" title="sec"> <div class="title" tagx="title" title="title">
Conflict of interest statement</div> <p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest. </p> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>This study was supported by the DISCO project that received funding from the European Union's Seventh Framework Programme for research, technological development and demonstration under the grant agreement 613513. Computational analyses were performed using Linux-HPC-Cluster at Technical University Dortmund (LiDong). We thank Mr. Sven Buijssen for his valuable help and repeated support regarding numerous aspects while using the compute cluster LiDong partially funded in the course of the Large-Scale Equipment Initiative by the German Research Foundation (DFG) as project 49560908. We also thank Dr. Felix Stehle for ESI-MS/MS and Dr. Pawel Rodziewicz for GC/MS analysis. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary material</div> <p>The Supplementary Material for this article can be found online at: <a href="https://www.frontiersin.org/articles/10.3389/fpls.2018.00537/full#supplementary-material">https://www.frontiersin.org/articles/10.3389/fpls.2018.00537/full#supplementary-material</a> </p> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Table 1</span>  <p>Filtering summary of raw reads.</p>  <div class="media" title="media"> <a href="Table_1.PDF">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Table 2</span>  <p>Isoform per gene summary.</p>  <div class="media" title="media"> <a href="Table_1.PDF">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Table 3</span>  <p>Read mapping summary.</p>  <div class="media" title="media"> <a href="Table_1.PDF">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Table 4</span>  <p>Predicted ORFs summary.</p>  <div class="media" title="media"> <a href="Table_1.PDF">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Table 5</span>  <p>Enzymes identified related to secondary metabolites and "terpenoids and polykettides."</p>  <div class="media" title="media"> <a href="Table_1.PDF">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Table 6</span>  <p>Enzyme distribution according to subclass category.</p>  <div class="media" title="media"> <a href="Table_1.PDF">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Table 7</span>  <p>Comparative summary of identified TFs.</p>  <div class="media" title="media"> <a href="Table_1.PDF">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Table 8</span>  <p>Frequency of repeat number.</p>  <div class="media" title="media"> <a href="Table_1.PDF">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Table 9</span>  <p>Oligonucleotide primers used for qPCR.</p>  <div class="media" title="media"> <a href="Table_1.PDF">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Data 1</span>  <p>BLAST analysis summary.</p>  <div class="media" title="media"> <a href="Data_Sheet_1.XLSX">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Data 2</span>  <p>IntrerProScan protein signature list.</p>  <div class="media" title="media"> <a href="Data_Sheet_2.XLSX">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Data 3</span>  <p>List of gene ontologies (GO) identified from three categories.</p>  <div class="media" title="media"> <a href="Data_Sheet_3.XLSX">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Data 4</span>  <p>List of identified enzymes associated with genes.</p>  <div class="media" title="media"> <a href="Data_Sheet_4.XLSX">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Data 5</span>  <p>List of GC-MS detected compounds.</p>  <div class="media" title="media"> <a href="Data_Sheet_5.XLSX">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Data 6</span>  <p>List of transcription factors (TFs).</p>  <div class="media" title="media"> <a href="Data_Sheet_6.XLSX">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Data 7</span>  <p>List of simple sequence repeats (SSRs).</p>  <div class="media" title="media"> <a href="Data_Sheet_7.XLSX">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Data 8</span>  <p>List of non-coding RNAs <b>(</b>ncRNAs). </p>  <div class="media" title="media"> <a href="Data_Sheet_8.XLSX">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Figure 1</span>  <p> <b>(A)</b> Association of protein signatures for a specific transcript among PANTHER, GENE3D SMART, TIGERFAM, and SIGNALP protein databases. <b>(B)</b> Distribution of gene ontologies annotation from GO-levels 2 to 15 for each category of biological process (BP) molecular function (MF) and cellular components (CC). <b>(C)</b> Gene ontology distribution among three main categories of molecular function (MF), biological process (BP), and cellular components (CC). Y-axis represents the number of genes for their respective function/process/component as on X-axis. </p>  <div class="media" title="media"> <a href="Image_1.PDF">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Figure 2</span>  <p> <b>(A)</b> Tandem mass spectra of the bibenzyl analog of cannabigerolic acid with m/z 395.214 <b>(B)</b> GC-MS chromatogram of <i>Radula marginata</i>. </p>  <div class="media" title="media"> <a href="Image_2.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="B1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alaba</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Piszczalka</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pietrykowska</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pacak</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sierocka</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nuc</span> <span class="given-names" tagx="given-names" title="given-names">P. W.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">The liverwort Pellia endiviifolia shares microtranscriptomic traits that are common to green algae and land plants </span>. <span class="source" tagx="source" title="source">New Phytol.</span> <span class="volume" tagx="volume" title="volume">206</span>, <span class="fpage" tagx="fpage" title="fpage">352</span>– <span class="lpage" tagx="lpage" title="lpage">367</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/nph.13220">10.1111/nph.13220</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25530158">25530158</a> </span> </span> </li> <li tag="ref"> <a name="B2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Altschul</span> <span class="given-names" tagx="given-names" title="given-names">S. F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gish</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miller</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Myers</span> <span class="given-names" tagx="given-names" title="given-names">E. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lipman</span> <span class="given-names" tagx="given-names" title="given-names">D. J.</span> </span> </span> ( <span class="year" tagx="year" title="year">1990</span>). <span class="mixed-article-title" title="mixed-article-title">Basic local alignment search tool</span>. <span class="source" tagx="source" title="source">J. Mol. Biol.</span> <span class="volume" tagx="volume" title="volume">215</span>, <span class="fpage" tagx="fpage" title="fpage">403</span>– <span class="lpage" tagx="lpage" title="lpage">410</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0022-2836(05)80360-2">10.1016/S0022-2836(05)80360-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712">2231712</a> </span> </span> </li> <li tag="ref"> <a name="B3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Andre</span> <span class="given-names" tagx="given-names" title="given-names">C. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hausman</span> <span class="given-names" tagx="given-names" title="given-names">J. F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guerriero</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title"> <i>Cannabis sativa</i>: the plant of the thousand and one molecules </span>. <span class="source" tagx="source" title="source">Front. Plant Sci.</span> <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">19</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.3389/fpls.2016.00019">10.3389/fpls.2016.00019</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26870049">26870049</a> </span> </span> </li> <li tag="ref"> <a name="B4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arnold</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bordoli</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kopp</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schwede</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">The SWISS-MODEL workspace: a web-based environment for protein structure homology modelling </span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">22</span>, <span class="fpage" tagx="fpage" title="fpage">195</span>– <span class="lpage" tagx="lpage" title="lpage">201</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/bti770">10.1093/bioinformatics/bti770</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16301204">16301204</a> </span> </span> </li> <li tag="ref"> <a name="B5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Asakawa</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hashimoto</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takikawa</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tori</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ogawa</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> ( <span class="year" tagx="year" title="year">1991</span>). <span class="mixed-article-title" title="mixed-article-title">Prenyl bibenzyls from the liverworts <i>Radula perrottetii</i> and <i>Radula complanata</i> </span>. <span class="source" tagx="source" title="source">Phytochemistry</span> <span class="volume" tagx="volume" title="volume">30</span>, <span class="fpage" tagx="fpage" title="fpage">235</span>– <span class="lpage" tagx="lpage" title="lpage">251</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0031-9422(91)84130-K">10.1016/0031-9422(91)84130-K</a> </span> </span> </li> <li tag="ref"> <a name="B6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ashburner</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ball</span> <span class="given-names" tagx="given-names" title="given-names">C. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blake</span> <span class="given-names" tagx="given-names" title="given-names">J. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Botstein</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Butler</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cherry</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">Gene Ontology: tool for the unification of biology</span>. <span class="source" tagx="source" title="source">Nat. Genet.</span> <span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">25</span>– <span class="lpage" tagx="lpage" title="lpage">29</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/75556">10.1038/75556</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651">10802651</a> </span> </span> </li> <li tag="ref"> <a name="B7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Austin</span> <span class="given-names" tagx="given-names" title="given-names">M. B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bowman</span> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ferrer</span> <span class="given-names" tagx="given-names" title="given-names">J. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schröder</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Noel</span> <span class="given-names" tagx="given-names" title="given-names">J. P.</span> </span> </span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">An aldol switch discovered in stilbene synthases mediates cyclization specificity of type III polyketide synthases </span>. <span class="source" tagx="source" title="source">Chem. Biol.</span> <span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">1179</span>– <span class="lpage" tagx="lpage" title="lpage">1194</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.chembiol.2004.05.024">10.1016/j.chembiol.2004.05.024</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15380179">15380179</a> </span> </span> </li> <li tag="ref"> <a name="B8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Austin</span> <span class="given-names" tagx="given-names" title="given-names">M. B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Noel</span> <span class="given-names" tagx="given-names" title="given-names">J. P.</span> </span> </span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">The chalcone synthase superfamily of type III polyketide synthases</span>. <span class="source" tagx="source" title="source">Nat. Prod. Rep.</span> <span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">79</span>– <span class="lpage" tagx="lpage" title="lpage">110</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1039/b100917f">10.1039/b100917f</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12636085">12636085</a> </span> </span> </li> <li tag="ref"> <a name="B9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Biasini</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bienert</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Waterhouse</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arnold</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Studer</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmidt</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">42</span>, <span class="fpage" tagx="fpage" title="fpage">252</span>– <span class="lpage" tagx="lpage" title="lpage">258</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gku340">10.1093/nar/gku340</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24782522">24782522</a> </span> </span> </li> <li tag="ref"> <a name="B10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bohlmann</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoffmann</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> </span> ( <span class="year" tagx="year" title="year">1979</span>). <span class="mixed-article-title" title="mixed-article-title">Cannabigerol-ähnliche verbindungen aus <i>Helichrysum umbraculigerum</i> </span>. <span class="source" tagx="source" title="source">Phytochemistry</span> <span class="volume" tagx="volume" title="volume">18</span>, <span class="fpage" tagx="fpage" title="fpage">1371</span>– <span class="lpage" tagx="lpage" title="lpage">1374</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0031-9422(79)83025-3">10.1016/0031-9422(79)83025-3</a> </span> </span> </li> <li tag="ref"> <a name="B11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bolger</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lohse</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Usadel</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">30</span>, <span class="fpage" tagx="fpage" title="fpage">2114</span>– <span class="lpage" tagx="lpage" title="lpage">2120</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btu170">10.1093/bioinformatics/btu170</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404">24695404</a> </span> </span> </li> <li tag="ref"> <a name="B12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bouman</span> <span class="given-names" tagx="given-names" title="given-names">A. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dirkse</span> <span class="given-names" tagx="given-names" title="given-names">G. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamada</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> ( <span class="year" tagx="year" title="year">1998</span>). <span class="mixed-article-title" title="mixed-article-title"> <i>Radula jonesii</i> spec. nov. (Hepaticae), a new species from Tenerife </span>. <span class="source" tagx="source" title="source">J. Bryol.</span> <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">161</span>– <span class="lpage" tagx="lpage" title="lpage">164</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1179/jbr.1988.15.1.161">10.1179/jbr.1988.15.1.161</a> </span> </span> </li> <li tag="ref"> <a name="B13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bowman</span> <span class="given-names" tagx="given-names" title="given-names">J. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Floyd</span> <span class="given-names" tagx="given-names" title="given-names">S. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sakakibara</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Green genes-comparative genomics of the green branch of life</span>. <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">129</span>, <span class="fpage" tagx="fpage" title="fpage">229</span>– <span class="lpage" tagx="lpage" title="lpage">234</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cell.2007.04.004">10.1016/j.cell.2007.04.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17448980">17448980</a> </span> </span> </li> <li tag="ref"> <a name="B14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Broun</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Queen</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schwarz</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leibman</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Importance of transcription factors in the regulation of plant secondary metabolism and their relevance to the control of terpenoid accumulation </span>. <span class="source" tagx="source" title="source">Phytochem. Rev.</span> <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">27</span>– <span class="lpage" tagx="lpage" title="lpage">38</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11101-006-9000-x">10.1007/s11101-006-9000-x</a> </span> </span> </li> <li tag="ref"> <a name="B15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buck</span> <span class="given-names" tagx="given-names" title="given-names">W. R.</span> </span> </span> ( <span class="year" tagx="year" title="year">1986</span>). <span class="mixed-article-title" title="mixed-article-title">Introduction to bryology</span>. <span class="source" tagx="source" title="source">Brittonia</span> <span class="volume" tagx="volume" title="volume">38</span>, <span class="fpage" tagx="fpage" title="fpage">94</span>– <span class="lpage" tagx="lpage" title="lpage">95</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.2307/2807430">10.2307/2807430</a> </span> </span> </li> <li tag="ref"> <a name="B16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cloutier</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">NiuRaju</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Duguid</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Development and analysis of EST-SSRs for flax ( <i>Linum usitatissimum</i> L.) </span>. <span class="source" tagx="source" title="source">Theor. Appl. Genet.</span> <span class="volume" tagx="volume" title="volume">119</span>, <span class="fpage" tagx="fpage" title="fpage">53</span>– <span class="lpage" tagx="lpage" title="lpage">63</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00122-009-1016-3">10.1007/s00122-009-1016-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19357828">19357828</a> </span> </span> </li> <li tag="ref"> <a name="B17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Conesa</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Götz</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">García-Gómez</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Terol</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Talón</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robles</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research </span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">21</span>, <span class="fpage" tagx="fpage" title="fpage">3674</span>– <span class="lpage" tagx="lpage" title="lpage">3676</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/bti610">10.1093/bioinformatics/bti610</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16081474">16081474</a> </span> </span> </li> <li tag="ref"> <a name="B18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Croteau</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Purkett</span> <span class="given-names" tagx="given-names" title="given-names">P. T.</span> </span> </span> ( <span class="year" tagx="year" title="year">1989</span>). <span class="mixed-article-title" title="mixed-article-title">Geranyl pyrophosphate synthase: characterization of the enzyme and evidence that this chain-length specific prenyltransferase is associated with monoterpene biosynthesis in Sage ( <i>Salvia officinalis</i>) </span>. <span class="source" tagx="source" title="source">Arch. Biochem. Biophys.</span> <span class="volume" tagx="volume" title="volume">271</span>, <span class="fpage" tagx="fpage" title="fpage">524</span>– <span class="lpage" tagx="lpage" title="lpage">535</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0003-9861(89)90304-4">10.1016/0003-9861(89)90304-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2730002">2730002</a> </span> </span> </li> <li tag="ref"> <a name="B19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eddy</span> <span class="given-names" tagx="given-names" title="given-names">S. R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Accelerated profile HMM searches</span>. <span class="source" tagx="source" title="source">PLoS Comput. Biol.</span> <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">e1002195</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pcbi.1002195">10.1371/journal.pcbi.1002195</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22039361">22039361</a> </span> </span> </li> <li tag="ref"> <a name="B20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eddy</span> <span class="given-names" tagx="given-names" title="given-names">S. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hughes</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">Non-coding RNA genes</span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">2</span>, <span class="fpage" tagx="fpage" title="fpage">919</span>– <span class="lpage" tagx="lpage" title="lpage">929</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/35103511">10.1038/35103511</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11733745">11733745</a> </span> </span> </li> <li tag="ref"> <a name="B21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Espley</span> <span class="given-names" tagx="given-names" title="given-names">R. V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hellens</span> <span class="given-names" tagx="given-names" title="given-names">R. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Putterill</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stevenson</span> <span class="given-names" tagx="given-names" title="given-names">D. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kutty-Amma</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Allan</span> <span class="given-names" tagx="given-names" title="given-names">A. C.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Red colouration in apple fruit is due to the activity of the MYB transcription factor, MdMYB10 </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">49</span>, <span class="fpage" tagx="fpage" title="fpage">414</span>– <span class="lpage" tagx="lpage" title="lpage">427</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2006.02964.x">10.1111/j.1365-313X.2006.02964.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17181777">17181777</a> </span> </span> </li> <li tag="ref"> <a name="B22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feldmeyer</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wheat</span> <span class="given-names" tagx="given-names" title="given-names">C. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krezdorn</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rotter</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pfenninger</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Short read Illumina data for the <i>de novo</i> assembly of a non-model snail species transcriptome <i>(Radix balthica</i>, Basommatophora, Pulmonata), and a comparison of assembler performance </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">317</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-12-317">10.1186/1471-2164-12-317</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21679424">21679424</a> </span> </span> </li> <li tag="ref"> <a name="B23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fellermeier</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eisenreich</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bacher</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zenk</span> <span class="given-names" tagx="given-names" title="given-names">M. H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">Biosynthesis of cannabinoids</span>. <span class="source" tagx="source" title="source">Eur. J. Biochem.</span> <span class="volume" tagx="volume" title="volume">268</span>, <span class="fpage" tagx="fpage" title="fpage">1596</span>– <span class="lpage" tagx="lpage" title="lpage">1604</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1432-1327.2001.02030.x">10.1046/j.1432-1327.2001.02030.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11248677">11248677</a> </span> </span> </li> <li tag="ref"> <a name="B24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fellermeier</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zenk</span> <span class="given-names" tagx="given-names" title="given-names">M. H.</span> </span> </span> ( <span class="year" tagx="year" title="year">1998</span>). <span class="mixed-article-title" title="mixed-article-title">Prenylation of olivetolate by a hemp transferase yields cannabigerolic acid, the precursor of tetrahydrocannabinol </span>. <span class="source" tagx="source" title="source">FEBS Lett.</span> <span class="volume" tagx="volume" title="volume">427</span>, <span class="fpage" tagx="fpage" title="fpage">283</span>– <span class="lpage" tagx="lpage" title="lpage">285</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0014-5793(98)00450-5">10.1016/S0014-5793(98)00450-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9607329">9607329</a> </span> </span> </li> <li tag="ref"> <a name="B25" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Finn</span> <span class="given-names" tagx="given-names" title="given-names">R. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Attwood</span> <span class="given-names" tagx="given-names" title="given-names">T. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Babbitt</span> <span class="given-names" tagx="given-names" title="given-names">P. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bateman</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bork</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bridge</span> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">InterPro in 2017-beyond protein family and domain annotations</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">45</span>, <span class="fpage" tagx="fpage" title="fpage">D190</span>– <span class="lpage" tagx="lpage" title="lpage">D199</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkw1107">10.1093/nar/gkw1107</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27899635">27899635</a> </span> </span> </li> <li tag="ref"> <a name="B26" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Finn</span> <span class="given-names" tagx="given-names" title="given-names">R. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clements</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eddy</span> <span class="given-names" tagx="given-names" title="given-names">S. R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">HMMER web server: interactive sequence similarity searching</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">39</span>, <span class="fpage" tagx="fpage" title="fpage">29</span>– <span class="lpage" tagx="lpage" title="lpage">37</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkr367">10.1093/nar/gkr367</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21593126">21593126</a> </span> </span> </li> <li tag="ref"> <a name="B27" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Finn</span> <span class="given-names" tagx="given-names" title="given-names">R. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coggill</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eberhardt</span> <span class="given-names" tagx="given-names" title="given-names">R. Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eddy</span> <span class="given-names" tagx="given-names" title="given-names">S. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mistry</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mitchell</span> <span class="given-names" tagx="given-names" title="given-names">A. L.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">The Pfam protein families database: towards a more sustainable future</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">44</span>, <span class="fpage" tagx="fpage" title="fpage">D279</span>– <span class="lpage" tagx="lpage" title="lpage">D285</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkv1344">10.1093/nar/gkv1344</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26673716">26673716</a> </span> </span> </li> <li tag="ref"> <a name="B28" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Flemming</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Muntendam</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steup</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kayser</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Chemistry and biological activity of tetrahydrocannabinol and its derivatives</span> in <span class="source" tagx="source" title="source">Topics in Heterocyclic Chemistry</span>, ed <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">R. R.</span> </span> </span> ( <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Berlin; Heidelberg</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>), <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">42</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/7081_2007_084">10.1007/7081_2007_084</a> </span> </span> </li> <li tag="ref"> <a name="B29" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fu</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Niu</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">CD-HIT: accelerated for clustering the next-generation sequencing data</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">28</span>, <span class="fpage" tagx="fpage" title="fpage">3150</span>– <span class="lpage" tagx="lpage" title="lpage">3152</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/bts565">10.1093/bioinformatics/bts565</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23060610">23060610</a> </span> </span> </li> <li tag="ref"> <a name="B30" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gachet</span> <span class="given-names" tagx="given-names" title="given-names">M. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schubert</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Calarco</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boccard</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gertsch</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Targeted metabolomics shows plasticity in the evolution of signaling lipids and uncovers old and new endocannabinoids in the plant kingdom </span>. <span class="source" tagx="source" title="source">Sci. Rep.</span> <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">41177</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep41177">10.1038/srep41177</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28120902">28120902</a> </span> </span> </li> <li tag="ref"> <a name="B31" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wood</span> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> transcriptome characterization and gene expression profiling of the desiccation tolerant following remoss <i>Bryum argenteum</i> hydration </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">416</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12864-015-1633-y">10.1186/s12864-015-1633-y</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26016800">26016800</a> </span> </span> </li> <li tag="ref"> <a name="B32" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wood</span> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> assembly and characterization of the transcriptome in the desiccation-tolerant moss <i>Syntrichia caninervis</i> </span>. <span class="source" tagx="source" title="source">BMC Res. Notes</span> <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">490</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1756-0500-7-490">10.1186/1756-0500-7-490</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25086984">25086984</a> </span> </span> </li> <li tag="ref"> <a name="B33" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gilles</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meglécz</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pech</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ferreira</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Malausa</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">J. F.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Accuracy and quality assessment of 454 GS-FLX Titanium pyrosequencing</span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">245</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-12-245">10.1186/1471-2164-12-245</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21592414">21592414</a> </span> </span> </li> <li tag="ref"> <a name="B34" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haas</span> <span class="given-names" tagx="given-names" title="given-names">B. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Papanicolaou</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yassour</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grabherr</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blood</span> <span class="given-names" tagx="given-names" title="given-names">P. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bowden</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis </span>. <span class="source" tagx="source" title="source">Nat. Protoc.</span> <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">1494</span>– <span class="lpage" tagx="lpage" title="lpage">1512</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nprot.2013.084">10.1038/nprot.2013.084</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23845962">23845962</a> </span> </span> </li> <li tag="ref"> <a name="B35" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hallingbäck</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hodgetts</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> </span> ( <span class="year" tagx="year" title="year">2001</span>). <span class="source" tagx="source" title="source">Mosses, Liverworts and Hornworts</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">IUCN/SSC Bryophyte Specialist Group</span>. </span> </li> <li tag="ref"> <a name="B36" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Happyana</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Agnolet</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Muntendam</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Dam</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schneider</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kayser</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Analysis of cannabinoids in laser-microdissected trichomes of medicinal <i>Cannabis sativa</i> using LCMS and cryogenic NMR </span>. <span class="source" tagx="source" title="source">Phytochemistry</span> <span class="volume" tagx="volume" title="volume">87</span>, <span class="fpage" tagx="fpage" title="fpage">51</span>– <span class="lpage" tagx="lpage" title="lpage">59</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.phytochem.2012.11.001">10.1016/j.phytochem.2012.11.001</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23280038">23280038</a> </span> </span> </li> <li tag="ref"> <a name="B37" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hisano</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sato</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Isobe</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sasamoto</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wada</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matsuno</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Characterization of the soybean genome using EST-derived microsatellite markers</span>. <span class="source" tagx="source" title="source">DNA Res.</span> <span class="volume" tagx="volume" title="volume">14</span>, <span class="fpage" tagx="fpage" title="fpage">271</span>– <span class="lpage" tagx="lpage" title="lpage">281</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/dnares/dsm025">10.1093/dnares/dsm025</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18192281">18192281</a> </span> </span> </li> <li tag="ref"> <a name="B38" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Honkanen</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">V. A. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morieri</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Champion</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hetherington</span> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kelly</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">The mechanism forming the cell surface of tip-growing rooting cells is conserved among land plants </span>. <span class="source" tagx="source" title="source">Curr. Biol.</span> <span class="volume" tagx="volume" title="volume">26</span>, <span class="fpage" tagx="fpage" title="fpage">3238</span>– <span class="lpage" tagx="lpage" title="lpage">3244</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cub.2016.09.062">10.1016/j.cub.2016.09.062</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27866889">27866889</a> </span> </span> </li> <li tag="ref"> <a name="B39" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jeena</span> <span class="given-names" tagx="given-names" title="given-names">G. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fatima</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tripathi</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Upadhyay</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shukla</span> <span class="given-names" tagx="given-names" title="given-names">R. K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Comparative transcriptome analysis of shoot and root tissue of Bacopa monnieri identifies potential genes related to triterpenoid saponin biosynthesis </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">490</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12864-017-3865-5">10.1186/s12864-017-3865-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28659188">28659188</a> </span> </span> </li> <li tag="ref"> <a name="B40" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jin</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kong</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luo</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">PlantTFDB 3.0: a portal for the functional and evolutionary study of plant transcription factors </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">42</span>, <span class="fpage" tagx="fpage" title="fpage">1182</span>– <span class="lpage" tagx="lpage" title="lpage">1187</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkt1016">10.1093/nar/gkt1016</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24174544">24174544</a> </span> </span> </li> <li tag="ref"> <a name="B41" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanehisa</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goto</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hattori</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aoki-Kinoshita</span> <span class="given-names" tagx="given-names" title="given-names">K. F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Itoh</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kawashima</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">From genomics to chemical genomics: new developments in KEGG</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">34</span>, <span class="fpage" tagx="fpage" title="fpage">D354</span>– <span class="lpage" tagx="lpage" title="lpage">D357</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkj102">10.1093/nar/gkj102</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16381885">16381885</a> </span> </span> </li> <li tag="ref"> <a name="B42" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanehisa</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goto</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kawashima</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Okuno</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hattori</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">The KEGG resource for deciphering the genome</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">32</span>, <span class="fpage" tagx="fpage" title="fpage">277</span>– <span class="lpage" tagx="lpage" title="lpage">280</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkh063">10.1093/nar/gkh063</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14681412">14681412</a> </span> </span> </li> <li tag="ref"> <a name="B43" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumpatla</span> <span class="given-names" tagx="given-names" title="given-names">S. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mukhopadhyay</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Mining and survey of simple sequence repeats in expressed sequence tags of dicotyledonous species </span>. <span class="source" tagx="source" title="source">Genome</span> <span class="volume" tagx="volume" title="volume">48</span>, <span class="fpage" tagx="fpage" title="fpage">985</span>– <span class="lpage" tagx="lpage" title="lpage">998</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1139/g05-060">10.1139/g05-060</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16391668">16391668</a> </span> </span> </li> <li tag="ref"> <a name="B44" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lang</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eisinger</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reski</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rensing</span> <span class="given-names" tagx="given-names" title="given-names">S. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Representation and high-quality annotation of the <i>Physcomitrella patens</i> transcriptome demonstrates a high proportion of proteins involved in metabolism in mosses </span>. <span class="source" tagx="source" title="source">Plant Biol.</span> <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">238</span>– <span class="lpage" tagx="lpage" title="lpage">250</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1055/s-2005-837578">10.1055/s-2005-837578</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15912443">15912443</a> </span> </span> </li> <li tag="ref"> <a name="B45" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Langmead</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salzberg</span> <span class="given-names" tagx="given-names" title="given-names">S. L.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Fast gapped-read alignment with Bowtie 2</span>. <span class="source" tagx="source" title="source">Nat. Methods</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">357</span>– <span class="lpage" tagx="lpage" title="lpage">359</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth.1923">10.1038/nmeth.1923</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286">22388286</a> </span> </span> </li> <li tag="ref"> <a name="B46" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">La Rota</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kantety</span> <span class="given-names" tagx="given-names" title="given-names">R. V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">J. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sorrells</span> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> </span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Nonrandom distribution and frequencies of genomic and EST-derived microsatellite markers in rice, wheat, and barley </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">23</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-6-23">10.1186/1471-2164-6-23</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15720707">15720707</a> </span> </span> </li> <li tag="ref"> <a name="B47" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dewey</span> <span class="given-names" tagx="given-names" title="given-names">C. N.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome </span>. <span class="source" tagx="source" title="source">BMC Bioinformatics</span> <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">323</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2105-12-323">10.1186/1471-2105-12-323</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21816040">21816040</a> </span> </span> </li> <li tag="ref"> <a name="B48" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">C. F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Global transcriptome and gene regulation network for secondary metabolite biosynthesis of tea plant ( <i>Camellia sinensis</i>) </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">560</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12864-015-1773-0">10.1186/s12864-015-1773-0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26220550">26220550</a> </span> </span> </li> <li tag="ref"> <a name="B49" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">M. Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiong</span> <span class="given-names" tagx="given-names" title="given-names">A. S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Identification of SSRs and differentially expressed genes in two cultivars of celery ( <i>Apium graveolens</i> L.) by deep transcriptome sequencing </span>. <span class="source" tagx="source" title="source">Hortic. Res.</span> <span class="volume" tagx="volume" title="volume">1</span>: <span class="fpage" tagx="fpage" title="fpage">10</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/hortres.2014.10">10.1038/hortres.2014.10</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26504532">26504532</a> </span> </span> </li> <li tag="ref"> <a name="B50" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liang</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yiu</span> <span class="given-names" tagx="given-names" title="given-names">S. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">B. L.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> assembly and characterization of <i>Camelina sativa</i> transcriptome by paired-end sequencing </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">146</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-14-146">10.1186/1471-2164-14-146</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23496985">23496985</a> </span> </span> </li> <li tag="ref"> <a name="B51" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Osbourn</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">MYB transcription factors as regulators of phenylpropanoid metabolism in plants</span>. <span class="source" tagx="source" title="source">Mol. Plant</span> <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">689</span>– <span class="lpage" tagx="lpage" title="lpage">708</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.molp.2015.03.012">10.1016/j.molp.2015.03.012</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25840349">25840349</a> </span> </span> </li> <li tag="ref"> <a name="B52" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Losada-Lima</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodriguez-Nunez</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dirkse</span> <span class="given-names" tagx="given-names" title="given-names">G. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Bibliographical references on the bryophyte flora of the Canary Islands</span>. <span class="source" tagx="source" title="source">Arch. Biol.</span> <span class="volume" tagx="volume" title="volume">24</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">27</span>. Available online at: <a href="http://www.archive-for-bryology.com/Archive%2024.pdf">http://www.archive-for-bryology.com/Archive%2024.pdf</a> </span> </li> <li tag="ref"> <a name="B53" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ludwiczuk</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Asakawa</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Distribution of terpenoids and aromatic compounds in selected southern hemispheric liverworts </span>. <span class="source" tagx="source" title="source">Fieldiana Bot.</span> <span class="volume" tagx="volume" title="volume">47</span>: <span class="fpage" tagx="fpage" title="fpage">37</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3158/0015-0746-47.1.37">10.3158/0015-0746-47.1.37</a> </span> </span> </li> <li tag="ref"> <a name="B54" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads</span>. <span class="source" tagx="source" title="source">EMBnet</span> <span class="volume" tagx="volume" title="volume">17</span>: <span class="fpage" tagx="fpage" title="fpage">10</span> <span class="pub-id"> <a href="https://dx.doi.org/10.14806/ej.17.1.200">10.14806/ej.17.1.200</a> </span> </span> </li> <li tag="ref"> <a name="B55" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mattick</span> <span class="given-names" tagx="given-names" title="given-names">J. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mattick</span> <span class="given-names" tagx="given-names" title="given-names">J. S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">The functional genomics of noncoding RNA</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">1527</span>, <span class="fpage" tagx="fpage" title="fpage">1527</span>– <span class="lpage" tagx="lpage" title="lpage">1529</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1117806">10.1126/science.1117806</a> </span> </span> </li> <li tag="ref"> <a name="B56" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Metzgar</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bytof</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wills</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> </span> ( <span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">Selection against frameshift mutations limits microsatellite expansion in coding DNA</span>. <span class="source" tagx="source" title="source">Genome Res.</span> <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">72</span>– <span class="lpage" tagx="lpage" title="lpage">80</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.10.1.72">10.1101/gr.10.1.72</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10645952">10645952</a> </span> </span> </li> <li tag="ref"> <a name="B57" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moazed</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Small RNAs in transcriptional gene silencing and genome defence</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">457</span>, <span class="fpage" tagx="fpage" title="fpage">413</span>– <span class="lpage" tagx="lpage" title="lpage">420</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature07756">10.1038/nature07756</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19158787">19158787</a> </span> </span> </li> <li tag="ref"> <a name="B58" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morgante</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hanafey</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Powell</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Microsatellites are preferentially associated with nonrepetitive DNA in plant genomes</span>. <span class="source" tagx="source" title="source">Nat. Genet.</span> <span class="volume" tagx="volume" title="volume">30</span>, <span class="fpage" tagx="fpage" title="fpage">194</span>– <span class="lpage" tagx="lpage" title="lpage">200</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ng822">10.1038/ng822</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11799393">11799393</a> </span> </span> </li> <li tag="ref"> <a name="B59" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagalakshmi</span> <span class="given-names" tagx="given-names" title="given-names">U.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Waern</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shou</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raha</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gerstein</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">The transcriptional landscape of the yeast genome defined by RNA sequencing</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">320</span>, <span class="fpage" tagx="fpage" title="fpage">1344</span>– <span class="lpage" tagx="lpage" title="lpage">1350</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1158441">10.1126/science.1158441</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18451266">18451266</a> </span> </span> </li> <li tag="ref"> <a name="B60" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oliver</span> <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dowd</span> <span class="given-names" tagx="given-names" title="given-names">S. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zaragoza</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mauget</span> <span class="given-names" tagx="given-names" title="given-names">S. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Payton</span> <span class="given-names" tagx="given-names" title="given-names">P. R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">The rehydration transcriptome of the desiccation-tolerant bryophyte <i>Tortula ruralis</i>: transcript classification and analysis </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">89</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-5-89">10.1186/1471-2164-5-89</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15546486">15546486</a> </span> </span> </li> <li tag="ref"> <a name="B61" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ortiz-Ramírez</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hernandez-Coronado</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thamm</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Catarino</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dolan</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">A transcriptome atlas of <i>Physcomitrella patens</i> provides insights into the evolution and development of land plants </span>. <span class="source" tagx="source" title="source">Mol. Plant</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">205</span>– <span class="lpage" tagx="lpage" title="lpage">220</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.molp.2015.12.002">10.1016/j.molp.2015.12.002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26687813">26687813</a> </span> </span> </li> <li tag="ref"> <a name="B62" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">B. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">Y. R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Concise synthesis of perrottetinene with bibenzyl cannabinoid</span>. <span class="source" tagx="source" title="source">Bull. Korean Chem. Soc.</span> <span class="volume" tagx="volume" title="volume">31</span>, <span class="fpage" tagx="fpage" title="fpage">2712</span>– <span class="lpage" tagx="lpage" title="lpage">2714</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.5012/bkcs.2010.31.9.2712">10.5012/bkcs.2010.31.9.2712</a> </span> </span> </li> <li tag="ref"> <a name="B63" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pérez-Rodríguez</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Riaño-Pachón</span> <span class="given-names" tagx="given-names" title="given-names">D. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Corrêa</span> <span class="given-names" tagx="given-names" title="given-names">L. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rensing</span> <span class="given-names" tagx="given-names" title="given-names">S. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kersten</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mueller-Roeber</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">PlnTFDB: updated content and new features of the plant transcription factor database</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="volume" tagx="volume" title="volume">38</span>, <span class="fpage" tagx="fpage" title="fpage">822</span>– <span class="lpage" tagx="lpage" title="lpage">827</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkp805">10.1093/nar/gkp805</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19934264">19934264</a> </span> </span> </li> <li tag="ref"> <a name="B64" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiu</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Genome-scale transcriptome analysis of the desert poplar, <i>Populus euphratica</i> </span>. <span class="source" tagx="source" title="source">Tree Physiol.</span> <span class="volume" tagx="volume" title="volume">31</span>, <span class="fpage" tagx="fpage" title="fpage">452</span>– <span class="lpage" tagx="lpage" title="lpage">461</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/treephys/tpr015">10.1093/treephys/tpr015</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21427158">21427158</a> </span> </span> </li> <li tag="ref"> <a name="B65" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qiu</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dombrovska</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">A nonflowering land plant phylogeny inferred from nucleotide sequences of seven chloroplast, mitochondrial, and nuclear genes </span>. <span class="source" tagx="source" title="source">Int. J. Plant Sci.</span> <span class="volume" tagx="volume" title="volume">168</span>, <span class="fpage" tagx="fpage" title="fpage">691</span>– <span class="lpage" tagx="lpage" title="lpage">708</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1086/513474">10.1086/513474</a> </span> </span> </li> <li tag="ref"> <a name="B66" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Renner</span> <span class="given-names" tagx="given-names" title="given-names">M. A. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Devos</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brown</span> <span class="given-names" tagx="given-names" title="given-names">E. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Orme</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elgey</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilson</span> <span class="given-names" tagx="given-names" title="given-names">T. C.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Integrative taxonomy resolves the cryptic and pseudo-cryptic</span>. <span class="source" tagx="source" title="source">PhytoKeys</span> <span class="volume" tagx="volume" title="volume">113</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">113</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.3897/phytokeys.27.5523">10.3897/phytokeys.27.5523</a> </span> </span> </li> <li tag="ref"> <a name="B67" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rensing</span> <span class="given-names" tagx="given-names" title="given-names">S. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lang</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zimmer</span> <span class="given-names" tagx="given-names" title="given-names">A. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Terry</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salamov</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shapiro</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">The physcomitrella genome reveals evolutionary insights into the conquest of land by plants stefan </span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">319</span>, <span class="fpage" tagx="fpage" title="fpage">64</span>– <span class="lpage" tagx="lpage" title="lpage">69</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1150646">10.1126/science.1150646</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18079367">18079367</a> </span> </span> </li> <li tag="ref"> <a name="B68" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rensing</span> <span class="given-names" tagx="given-names" title="given-names">S. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rombauts</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hohe</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lang</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Duwenig</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rouze</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">The transcriptome of the moss <i>Physcomitrella patens</i>: comparative analysis reveals a rich source of new genes </span>. <span class="source" tagx="source" title="source">Plant Biotech</span> <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">18</span>. Available online at: <a href="http://www.plant-biotech.net/Rensing_et_al_transcriptome2002.pdf">http://www.plant-biotech.net/Rensing_et_al_transcriptome2002.pdf</a> </span> </li> <li tag="ref"> <a name="B69" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rubinstein</span> <span class="given-names" tagx="given-names" title="given-names">C. V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gerrienne</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">de la Puente</span> <span class="given-names" tagx="given-names" title="given-names">G. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Astini</span> <span class="given-names" tagx="given-names" title="given-names">R. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steemans</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Early middle ordovician evidence for land plants in Argentina (eastern Gondwana)</span>. <span class="source" tagx="source" title="source">New Phytol.</span> <span class="volume" tagx="volume" title="volume">188</span>, <span class="fpage" tagx="fpage" title="fpage">365</span>– <span class="lpage" tagx="lpage" title="lpage">369</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1469-8137.2010.03433.x">10.1111/j.1469-8137.2010.03433.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20731783">20731783</a> </span> </span> </li> <li tag="ref"> <a name="B70" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ruggiero</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gordon</span> <span class="given-names" tagx="given-names" title="given-names">D. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Orrell</span> <span class="given-names" tagx="given-names" title="given-names">T. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bailly</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bourgoin</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brusca</span> <span class="given-names" tagx="given-names" title="given-names">R. C.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">A higher level classification of all living organisms</span>. <span class="source" tagx="source" title="source">PLoS ONE</span> <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">e0119248</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0119248">10.1371/journal.pone.0119248</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25923521">25923521</a> </span> </span> </li> <li tag="ref"> <a name="B71" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ruhfel</span> <span class="given-names" tagx="given-names" title="given-names">B. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gitzendanner</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soltis</span> <span class="given-names" tagx="given-names" title="given-names">P. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soltis</span> <span class="given-names" tagx="given-names" title="given-names">D. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burleigh</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">From algae to angiosperms–inferring the phylogeny of green plants (Viridiplantae) from 360 plastid genomes </span>. <span class="source" tagx="source" title="source">BMC Evol. Biol.</span> <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">23</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2148-14-23">10.1186/1471-2148-14-23</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24533922">24533922</a> </span> </span> </li> <li tag="ref"> <a name="B72" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Russo</span> <span class="given-names" tagx="given-names" title="given-names">E. B.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Beyond cannabis: plants and the endocannabinoid system</span>. <span class="source" tagx="source" title="source">Trends Pharmacol. Sci.</span> <span class="volume" tagx="volume" title="volume">37</span>, <span class="fpage" tagx="fpage" title="fpage">594</span>– <span class="lpage" tagx="lpage" title="lpage">605</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tips.2016.04.005">10.1016/j.tips.2016.04.005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27179600">27179600</a> </span> </span> </li> <li tag="ref"> <a name="B73" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sayadi</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Immonen</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bayram</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arnqvist</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">The <i>de novo</i> transcriptome and its functional annotation in the seed beetle callosobruchus maculatus </span>. <span class="source" tagx="source" title="source">PLoS ONE</span> <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">e158565</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0158565">10.1371/journal.pone.0158565</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27442123">27442123</a> </span> </span> </li> <li tag="ref"> <a name="B74" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schliesky</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gowik</span> <span class="given-names" tagx="given-names" title="given-names">U.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">A. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bräutigam</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">RNA-seq assembly – are we there yet?</span> <span class="source" tagx="source" title="source">Front. Plant Sci.</span> <span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">220</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.3389/fpls.2012.00220">10.3389/fpls.2012.00220</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23056003">23056003</a> </span> </span> </li> <li tag="ref"> <a name="B75" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharma</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jung</span> <span class="given-names" tagx="given-names" title="given-names">C. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhalla</span> <span class="given-names" tagx="given-names" title="given-names">P. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">M. B.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">RNA sequencing analysis of the gametophyte transcriptome from the liverwort, Marchantia polymorpha </span>. <span class="source" tagx="source" title="source">PLoS ONE</span> <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">e97497</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0097497">10.1371/journal.pone.0097497</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24841988">24841988</a> </span> </span> </li> <li tag="ref"> <a name="B76" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rai</span> <span class="given-names" tagx="given-names" title="given-names">K. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Upadhyay</span> <span class="given-names" tagx="given-names" title="given-names">S. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pant</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Verma</span> <span class="given-names" tagx="given-names" title="given-names">P. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">A. P.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Transcriptome sequencing of a thalloid bryophyte; <i>Dumortiera hirsuta</i> (Sw) Nees: assembly, annotation, and marker discovery </span>. <span class="source" tagx="source" title="source">Sci. Rep.</span> <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">15350</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep15350">10.1038/srep15350</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26481431">26481431</a> </span> </span> </li> <li tag="ref"> <a name="B77" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sirikantaramas</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morimoto</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shoyama</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishikawa</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wada</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shoyama</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">The gene controlling marijuana psychoactivity</span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="volume" tagx="volume" title="volume">279</span>, <span class="fpage" tagx="fpage" title="fpage">39767</span>– <span class="lpage" tagx="lpage" title="lpage">39774</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.M403693200">10.1074/jbc.M403693200</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15190053">15190053</a> </span> </span> </li> <li tag="ref"> <a name="B78" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soethoudt</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grether</span> <span class="given-names" tagx="given-names" title="given-names">U.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grim</span> <span class="given-names" tagx="given-names" title="given-names">T. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fezza</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perret</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gils</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Cannabinoid CB <sub>2</sub> receptor ligand profiling reveals biased signalling and off-target activity </span>. <span class="source" tagx="source" title="source">Nat. Commun</span>. <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">13958</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncomms13958">10.1038/ncomms13958</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28045021">28045021</a> </span> </span> </li> <li tag="ref"> <a name="B79" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Spyropoulou</span> <span class="given-names" tagx="given-names" title="given-names">E. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haring</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schuurink</span> <span class="given-names" tagx="given-names" title="given-names">R. C.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">RNA sequencing on Solanum lycopersicum trichomes identifies transcription factors that activate terpene synthase promoters </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">402</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-15-402">10.1186/1471-2164-15-402</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24884371">24884371</a> </span> </span> </li> <li tag="ref"> <a name="B80" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steijger</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abril</span> <span class="given-names" tagx="given-names" title="given-names">J. F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Engström</span> <span class="given-names" tagx="given-names" title="given-names">P. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kokocinski</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abril</span> <span class="given-names" tagx="given-names" title="given-names">J. F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Akerman</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Assessment of transcript reconstruction methods for RNA-seq</span>. <span class="source" tagx="source" title="source">Nat. Methods</span> <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">1177</span>– <span class="lpage" tagx="lpage" title="lpage">1184</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth.2714">10.1038/nmeth.2714</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24185837">24185837</a> </span> </span> </li> <li tag="ref"> <a name="B81" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stotler</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crandall-Stotler</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> </span> ( <span class="year" tagx="year" title="year">1977</span>). <span class="mixed-article-title" title="mixed-article-title">A checklist of the liverworts and hornworts of North America</span>. <span class="source" tagx="source" title="source">Bryologist</span> <span class="volume" tagx="volume" title="volume">80</span>, <span class="fpage" tagx="fpage" title="fpage">405</span>– <span class="lpage" tagx="lpage" title="lpage">428</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.2307/3242017">10.2307/3242017</a> </span> </span> </li> <li tag="ref"> <a name="B82" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stout</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boubakir</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ambrose</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Purves</span> <span class="given-names" tagx="given-names" title="given-names">R. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Page</span> <span class="given-names" tagx="given-names" title="given-names">J. E.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">The hexanoyl-CoA precursor for cannabinoid biosynthesis is formed by an acyl-activating enzyme in <i>Cannabis sativa</i> trichomes </span>. <span class="source" tagx="source" title="source">Plant J.</span> <span class="volume" tagx="volume" title="volume">71</span>, <span class="fpage" tagx="fpage" title="fpage">353</span>– <span class="lpage" tagx="lpage" title="lpage">365</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2012.04949.x">10.1111/j.1365-313X.2012.04949.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22353623">22353623</a> </span> </span> </li> <li tag="ref"> <a name="B83" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Styrczewska</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kulma</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ratajczak</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Amarowicz</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Szopa</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Cannabinoid-like anti-inflammatory compounds from flax fiber</span>. <span class="source" tagx="source" title="source">Cell. Mol. Biol. Lett.</span> <span class="volume" tagx="volume" title="volume">17</span>, <span class="fpage" tagx="fpage" title="fpage">479</span>– <span class="lpage" tagx="lpage" title="lpage">499</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.2478/s11658-012-0023-6">10.2478/s11658-012-0023-6</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22706678">22706678</a> </span> </span> </li> <li tag="ref"> <a name="B84" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taura</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sirikantaramas</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shoyama</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yoshikai</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shoyama</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morimoto</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Cannabidiolic-acid synthase, the chemotype-determining enzyme in the fiber-type <i>Cannabis sativa</i> </span>. <span class="source" tagx="source" title="source">FEBS Lett.</span> <span class="volume" tagx="volume" title="volume">581</span>, <span class="fpage" tagx="fpage" title="fpage">2929</span>– <span class="lpage" tagx="lpage" title="lpage">2934</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.febslet.2007.05.043">10.1016/j.febslet.2007.05.043</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17544411">17544411</a> </span> </span> </li> <li tag="ref"> <a name="B85" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tisserand</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Young</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="source" tagx="source" title="source">Essential Oil Safety</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Churchill Livingstone Elsevier</span> Available online at: <a href="https://www.elsevier.com/books/essential-oil-safety/tisserand/978-0-443-06241-4">https://www.elsevier.com/books/essential-oil-safety/tisserand/978-0-443-06241-4</a> </span> </li> <li tag="ref"> <a name="B86" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Toyota</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kinugawa</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Asakawa</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> </span> ( <span class="year" tagx="year" title="year">1994</span>). <span class="mixed-article-title" title="mixed-article-title">Bibenzyl cannabinoid and bisbibenzyl derivative from the liverwort <i>Radula perrottetii</i> </span>. <span class="source" tagx="source" title="source">Phytochemistry</span> <span class="volume" tagx="volume" title="volume">37</span>, <span class="fpage" tagx="fpage" title="fpage">859</span>– <span class="lpage" tagx="lpage" title="lpage">862</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0031-9422(00)90371-6">10.1016/S0031-9422(00)90371-6</a> </span> </span> </li> <li tag="ref"> <a name="B87" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Toyota</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shimamura</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ishii</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Renner</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Braggins</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Asakawa</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">New bibenzyl cannabinoid from the New Zealand liverwort <i>Radula marginata</i> </span>. <span class="source" tagx="source" title="source">Chem. Pharm. Bull.</span> <span class="volume" tagx="volume" title="volume">50</span>, <span class="fpage" tagx="fpage" title="fpage">1390</span>– <span class="lpage" tagx="lpage" title="lpage">1392</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1248/cpb.50.1390">10.1248/cpb.50.1390</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12372871">12372871</a> </span> </span> </li> <li tag="ref"> <a name="B88" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tropf</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lanz</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rensing</span> <span class="given-names" tagx="given-names" title="given-names">S. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schröder</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schröder</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> </span> ( <span class="year" tagx="year" title="year">1994</span>). <span class="mixed-article-title" title="mixed-article-title">Evidence that stilbene synthases have developed from chalcone synthases several times in the course of evolution </span>. <span class="source" tagx="source" title="source">J. Mol. Evol.</span> <span class="volume" tagx="volume" title="volume">38</span>, <span class="fpage" tagx="fpage" title="fpage">610</span>– <span class="lpage" tagx="lpage" title="lpage">618</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/BF00175881">10.1007/BF00175881</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8083886">8083886</a> </span> </span> </li> <li tag="ref"> <a name="B89" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Verk</span> <span class="given-names" tagx="given-names" title="given-names">M. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Hickman</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Wees</span> <span class="given-names" tagx="given-names" title="given-names">S. C. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">RNA-Seq: revelation of the messengers</span>. <span class="source" tagx="source" title="source">Trends Plant Sci.</span> <span class="volume" tagx="volume" title="volume">18</span>, <span class="fpage" tagx="fpage" title="fpage">175</span>– <span class="lpage" tagx="lpage" title="lpage">179</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tplants.2013.02.001">10.1016/j.tplants.2013.02.001</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23481128">23481128</a> </span> </span> </li> <li tag="ref"> <a name="B90" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vom Endt</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kijne</span> <span class="given-names" tagx="given-names" title="given-names">J. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Memelink</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Transcription factors controlling plant secondary metabolism: what regulates the regulators?</span> <span class="source" tagx="source" title="source">Phytochemistry</span> <span class="volume" tagx="volume" title="volume">61</span>, <span class="fpage" tagx="fpage" title="fpage">107</span>– <span class="lpage" tagx="lpage" title="lpage">114</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0031-9422(02)00185-1">10.1016/S0031-9422(02)00185-1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12169302">12169302</a> </span> </span> </li> <li tag="ref"> <a name="B91" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wei</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qi</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hua</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Characterization of the sesame ( <i>Sesamum indicum</i> L.) global transcriptome using Illumina paired-end sequencing and development of EST-SSR markers </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">451</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-12-451">10.1186/1471-2164-12-451</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21929789">21929789</a> </span> </span> </li> <li tag="ref"> <a name="B92" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Willis</span> <span class="given-names" tagx="given-names" title="given-names">K. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McElwain</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="source" tagx="source" title="source">The Evolution of Plants</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxford</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span>. </span> </li> <li tag="ref"> <a name="B93" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wood</span> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Duff</span> <span class="given-names" tagx="given-names" title="given-names">R. J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">The aldehyde dehydrogenase (ALDH) gene superfamily of the moss <i>Physcomitrella patens</i> and the algae <i>Chlamydomonas reinhardtii</i> and <i>Ostreococcus tauri</i> </span>. <span class="source" tagx="source" title="source">Bryologist</span> <span class="volume" tagx="volume" title="volume">112</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">11</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1639/0007-2745-112.1.1">10.1639/0007-2745-112.1.1</a> </span> </span> </li> <li tag="ref"> <a name="B94" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Deep RNA sequencing at single base-pair resolution reveals high complexity of the rice transcriptome </span>. <span class="source" tagx="source" title="source">Genome Res.</span> <span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">646</span>– <span class="lpage" tagx="lpage" title="lpage">654</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1101/gr.100677.109">10.1101/gr.100677.109</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20305017">20305017</a> </span> </span> </li> <li tag="ref"> <a name="B95" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> transcriptome analysis and molecular marker development of two hemarthria species </span>. <span class="source" tagx="source" title="source">Front. Plant Sci.</span> <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">496</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.3389/fpls.2016.00496">10.3389/fpls.2016.00496</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27148320">27148320</a> </span> </span> </li> <li tag="ref"> <a name="B96" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leng</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mu</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Discovery of conservation and diversification of genes by phylogenetic analysis based on global genomes </span>. <span class="source" tagx="source" title="source">Plant Genome</span> <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">11</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.3835/plantgenome2014.10.0076">10.3835/plantgenome2014.10.0076</a> </span> </span> </li> <li tag="ref"> <a name="B97" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zimmer</span> <span class="given-names" tagx="given-names" title="given-names">A. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lang</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buchta</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rombauts</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nishiyama</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hasebe</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span>. ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Reannotation and extended community resources for the genome of the non-seed plant <i>Physcomitrella patens</i> provide insights into the evolution of plant gene structures and functions </span>. <span class="source" tagx="source" title="source">BMC Genomics</span> <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">498</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2164-14-498">10.1186/1471-2164-14-498</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23879659">23879659</a> </span> </span> </li> <li tag="ref"> <a name="B98" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zirpel</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Degenhardt</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kayser</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stehle</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> </span> ( <span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Engineering yeasts as platform organisms for cannabinoid biosynthesis</span>. <span class="source" tagx="source" title="source">J. Biotechnol.</span> <span class="volume" tagx="volume" title="volume">259</span>, <span class="fpage" tagx="fpage" title="fpage">204</span>– <span class="lpage" tagx="lpage" title="lpage">212</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jbiotec.2017.07.008">10.1016/j.jbiotec.2017.07.008</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28694184">28694184</a> </span> </span> </li> </ul> </div> <div class="glossary" title="glossary"> <div class="def-list" title="def-list"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-item" title="def-item"> <div class="term" title="term">
BLAST</div> <div class="def" title="def"> <p>Basic local alignment search tool</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GO</div> <div class="def" title="def"> <p>Gene ontology</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
BP</div> <div class="def" title="def"> <p>Biological Process</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MF</div> <div class="def" title="def"> <p>Molecular Function</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CC</div> <div class="def" title="def"> <p>Cellular Components</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
KEGG</div> <div class="def" title="def"> <p>Kyoto Encyclopedia of Genes and Genomes</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NGS</div> <div class="def" title="def"> <p>Next Generation Sequencing</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
RT-qPCR</div> <div class="def" title="def"> <p>Real-time quantitative polymerase chain reaction</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
RPKM</div> <div class="def" title="def"> <p>Reads per kilobases per million reads</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ncRNA</div> <div class="def" title="def"> <p>non-coding RNA</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
RSEM</div> <div class="def" title="def"> <p>RNA-seq by Expectation-Maximization</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
FPKM</div> <div class="def" title="def"> <p>Fragment per kilobase of transcript per million mapped reads</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
TPM</div> <div class="def" title="def"> <p>Transcripts per million</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
TFs</div> <div class="def" title="def"> <p>transcription factors</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SSRs</div> <div class="def" title="def"> <p>simple sequence repeats</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term"> <i>Rm-Tct</i> </div> <div class="def" title="def"> <p> <i>Radula marginata</i> tentative consensus transcripts </p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SA</div> <div class="def" title="def"> <p>Stilbene acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
OA</div> <div class="def" title="def"> <p>Olivetolic acid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
STS</div> <div class="def" title="def"> <p>Stilbene synthase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CHS</div> <div class="def" title="def"> <p>Chalcone synthase</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
PDB</div> <div class="def" title="def"> <p>Protein databank.</p> </div> </div> </div> </div> </div>  </body></html>