<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">PLoS One</span> <span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span> <span class="publisher-id" title="publisher-id">plos</span> <span class="pmc" title="pmc">plosone</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, USA</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/2987802">2987802</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21124976">21124976</a> </span> <span class="publisher-id" title="publisher-id">10-PONE-RA-20757R1</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0014048">10.1371/journal.pone.0014048</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Genetics and Genomics/Genomics</span> <span class="subject" title="subject">Genetics and Genomics/Microbial Evolution and Genomics</span> <span class="subject" title="subject">Microbiology/Microbial Evolution and Genomics</span> <span class="subject" title="subject">Molecular Biology/Chromosome Structure</span> <span class="subject" title="subject">Molecular Biology/Molecular Evolution</span> <span class="subject" title="subject">Molecular Biology/RNA Splicing</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
The <i>Agaricus bisporus cox1</i> Gene: The Longest Mitochondrial Gene and the Largest Reservoir of Mitochondrial Group I Introns </div> <div class="alt-title" title="alt-title">
Mitochondrial Intron Reservoir</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Férandon</span> <span class="given-names" tagx="given-names" title="given-names">Cyril</span> </span> <a href="#aff1"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moukha</span> <span class="given-names" tagx="given-names" title="given-names">Serge</span> </span> <a href="#aff2"> <sup>2</sup> </a> <a href="#aff3"> <sup>3</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Callac</span> <span class="given-names" tagx="given-names" title="given-names">Philippe</span> </span> <a href="#aff3"> <sup>3</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Benedetto</span> <span class="given-names" tagx="given-names" title="given-names">Jean-Pierre</span> </span> <a href="#aff1"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Castroviejo</span> <span class="given-names" tagx="given-names" title="given-names">Michel</span> </span> <a href="#aff1"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barroso</span> <span class="given-names" tagx="given-names" title="given-names">Gérard</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#cor1"> <sup>*</sup> </a> </span> </div> <span class="citation_author_institution" id="aff1">[1], <span class="addr-line" title="addr-line">UMR 5234 CNRS (Centre National de la Recherche Scientifique) – Université Victor Segalen Bordeaux 2, Bordeaux, France </span> </span> <span class="citation_author_institution" id="aff2">[2], <span class="addr-line" title="addr-line">Laboratoire de Toxicologie et Hygiène Appliquée, UFR des Sciences Pharmaceutiques, Université Victor Segalen Bordeaux 2, Bordeaux, France </span> </span> <span class="citation_author_institution" id="aff3">[3], <span class="addr-line" title="addr-line">INRA (Institut National de la Recherche Agronomique) UR 1264 Mycologie et Sécurité des Aliments, Villenave d'Ornon, France </span> </span> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Badger</span> <span class="given-names" tagx="given-names" title="given-names">Jonathan H.</span> </span> <span class="role" tagx="role" title="role">Editor</span> <a href="#edit1" /> </span> </div> <span class="citation_author_institution" id="edit1">[], </span> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">gerard.barroso@u-bordeaux2.fr</span> </div> <div class="fn-type-con" title="con"> <p>Conceived and designed the experiments: CF SM GB. Performed the experiments: CF SM GB. Analyzed the data: CF SM PC JPB MC GB. Contributed reagents/materials/analysis tools: CF SM PC GB. Wrote the paper: CF SM PC JPB MC GB. </p> </div> </div> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2010</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2010-11-11</span> </span> <span class="volume" tagx="volume" title="volume">5</span> <span class="issue" tagx="issue" title="issue">11</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e14048</span> <span class="history" title="history"> <span class="received" title="received">received: 2010-7-7</span> <span class="accepted" title="accepted">accepted: 2010-10-29</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2010</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>In eukaryotes, introns are located in nuclear and organelle genes from several kingdoms. Large introns (up to 5 kbp) are frequent in mitochondrial genomes of plant and fungi but scarce in Metazoa, even if these organisms are grouped with fungi among the Opisthokonts. Mitochondrial introns are classified in two groups (I and II) according to their RNA secondary structure involved in the intron self-splicing mechanism. Most of these mitochondrial group I introns carry a "Homing Endonuclease Gene" ( <i>heg</i>) encoding a DNA endonuclease acting in transfer and site-specific integration ("homing") and allowing intron spreading and gain after lateral transfer even between species from different kingdoms. Opposed to this gain mechanism, is another which implies that introns, which would have been abundant in the ancestral genes, would mainly evolve by loss. The importance of both mechanisms (loss and gain) is matter of debate. Here we report the sequence of the <i>cox1</i> gene of the button mushroom <i>Agaricus bisporus</i>, the most widely cultivated mushroom in the world. This gene is both the longest mitochondrial gene (29,902 nt) and the largest group I intron reservoir reported to date with 18 group I and 1 group II. An exhaustive analysis of the group I introns available in <i>cox1</i> genes shows that they are mobile genetic elements whose numerous events of loss and gain by lateral transfer combine to explain their wide and patchy distribution extending over several kingdoms. An overview of intron distribution, together with the high frequency of eroded <i>heg</i>, suggests that they are evolving towards loss. In this landscape of eroded and lost intron sequences, the <i>A. bisporus cox1</i> gene exhibits a peculiar dynamics of intron keeping and catching, leading to the largest collection of mitochondrial group I introns reported to date in a Eukaryote. </p> </div> <div class="counts" title="counts"> <span class="page-count" title="page-count">page-count: </span> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Introns have been described in prokaryotes, eukaryotes and even in viral genomes <a href="#pone.0014048-Haugen1">[1]</a>, <a href="#pone.0014048-Lambowitz1">[2]</a>, <a href="#pone.0014048-Nesbo1">[3]</a>, <a href="#pone.0014048-Sandegren1">[4]</a>. In eukaryotes, they are present in nuclear, plastidial and mitochondrial genes from several kingdoms and with various sizes and frequencies <a href="#pone.0014048-Lang1">[5]</a>. Large introns (from 0.1 to 5 kbp) are frequently found in mitochondrial genes of plant and fungal kingdoms <a href="#pone.0014048-Lang1">[5]</a>. However, they are scarce in the mitochondrial genomes of Metazoa, despite being grouped with fungi among the Fungi/Metazoa (Opisthokonts) super-kingdom. Mitochondrial introns are classified in two main groups (group I and II) according to their RNA secondary structure involved in the intron self-splicing mechanism <a href="#pone.0014048-Lang1">[5]</a>, <a href="#pone.0014048-Saldanha1">[6]</a>. Mitochondrial group I and group II introns can be found in the same species and even in the same gene. Fungal mitochondrial genes tend to bear group I introns while plant mitochondrial genes tend to bear group II introns <a href="#pone.0014048-Lang1">[5]</a>. For example, the <i>cox1</i> gene of the fungus <i>Podospora anserina</i> possesses 15 group I introns and one group II <a href="#pone.0014048-Cummings1">[7]</a> and the complete mitochondrial genome of the liverwort <i>Marchantia polymorpha</i> carries 25 group II and seven group I <a href="#pone.0014048-Oda1">[8]</a>. Most of the mitochondrial group I introns carry a "Homing Endonuclease Gene" ( <i>heg</i>) encoding a DNA endonuclease (HE) acting in the transfer and site-specific integration ("homing") of the intron <a href="#pone.0014048-Lambowitz2">[9]</a>, <a href="#pone.0014048-Lazowska1">[10]</a>, <a href="#pone.0014048-Pellenz1">[11]</a>. The presence of these <i>heg</i> allowed the raising of a theory where introns might be mobile elements able to spread and to be gained by genes after lateral transfer even between phylogenetically distant species from different kingdoms <a href="#pone.0014048-Gonzalez1">[12]</a>, <a href="#pone.0014048-Vaughn1">[13]</a>. However, this theory does not infer the origin of the mobile intron. Another theory suggests that introns were abundant in the ancestral genes and mainly evolve by loss <a href="#pone.0014048-Goddard1">[14]</a>, <a href="#pone.0014048-Gonzalez2">[15]</a>. However, both hypotheses are matters of much debate about the dynamics of intron loss and gain during evolution <a href="#pone.0014048-Cusimano1">[16]</a>, <a href="#pone.0014048-SanchezPuerta1">[17]</a>. For example, two recent studies dealing with a group I intron widely distributed in plant angiosperms led to conflicting conclusions <a href="#pone.0014048-Cusimano1">[16]</a>, <a href="#pone.0014048-SanchezPuerta1">[17]</a>. Indeed, the history of these sequences is thought to be dominated by numerous events of loss <a href="#pone.0014048-Cusimano1">[16]</a> or due to frequent gains after a lateral transfer from a fungal donor <a href="#pone.0014048-SanchezPuerta1">[17]</a>. </p> <p>The <i>cox1</i> gene (encoding the subunit 1 of the cytochrome c oxydase) was widely used in a successful "DNA barcoding" taxonomic method in animals <a href="#pone.0014048-Hebert1">[18]</a>. However, in the related fungal kingdom, this approach is hampered by the presence of several large group I introns <a href="#pone.0014048-Santamaria1">[19]</a>. Indeed, the fungal <i>cox1</i> gene is the mitochondrial gene showing the highest number of introns <a href="#pone.0014048-Santamaria1">[19]</a>. Owing to this wealth of introns, few fungal complete <i>cox1</i> gene sequences are available in databases, especially for species belonging to the Basidiomycota division where only nine sequences ( <i>Cryptococcus neoformans</i> var. <i>neoformans</i>, <i>Cryptococcus neoformans var. grubii</i>, <i>Ustilago maydis</i>, <i>Tilletia indica</i>, <i>Moniliophtora perniciosa</i>, <i>Pleurotus ostreatus</i>, <i>Agrocybe aegerita</i>, <i>Schizophyllum commune</i> and <i>Trametes cingulata</i>) have been reported and correctly annotated to date. </p> <p>To understand the evolution and dynamics of mitochondrial introns in fungi, we decided to determine the sequence and molecular organization of the commonly used <i>cox1</i> model gene in the most widely cultivated mushroom in the world: <i>Agaricus bisporus</i>. </p> </div> <div class="resultsanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
Results and Discussion</div> <div class="thea.bisporuscox1gene:thelargestmitochondrialintronreservoir" title="sec"> <div class="title" tagx="title" title="title">
The <i>A. bisporus cox1</i> gene: the largest mitochondrial intron reservoir </div> <p>The complete sequence (29.902 nt, GenBank Accession number: EU314927) of the <i>cox1</i> gene of the button mushroom <i>Agaricus bisporus (Abi)</i> was determined for a subculture (Bs 518) of the traditional brown cultivar C9 from overlapping cloned <i>Bam</i>HI and <i>Eco</i>RI restriction fragments and cloned PCR products (Supplementary information file, <a href="#pone.0014048.s001">Figure S1</a>). An alignment of this gene with the <i>cox1</i> exon sequence (CDS) of the related Agaricales <i>Agrocybe aegerita</i> <a href="#pone.0014048-Gonzalez1">[12]</a> shows that the <i>Abi cox1</i> CDS is split by 19 large (&amp;gt;1 kbp) introns. The borders of these introns, their group and sub-group were deduced from the design of each secondary structure (Supplementary information file, <a href="#pone.0014048.s002">Figure S2</a>) and from a comparison of the gene with the <i>Abi</i> cDNA (1,587 nt). The 19 introns are scattered over the whole gene sequence ( <a href="#pone-0014048-g001">Figure 1</a>). The size of the exons varies from 3 nt (exon 2) to 288 nt (exon 20), contrasting with the large size of the introns that ranges from 1,057 nt ( <i>iAbi</i>13) to 2,736 nt ( <i>iAbi</i>2). The intron sequences represent 28,318 nt, i.e. 94.7% of the gene. The <i>Abi cox1</i> gene is the longest mitochondrial gene reported to date in all kingdoms. Indeed, the next longest mitochondrial genes are the <i>cox1</i> gene of the Ascomycota <i>Podospora anserina</i> (24,507 nt) which carries 14 group I and one group II introns <a href="#pone.0014048-Cummings1">[7]</a> and the Basidiomycota <i>Trametes cingulata</i> (22,370 nt) with 15 group I introns <a href="#pone.0014048-Haridas1">[20]</a>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0014048.g001"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <div class="title" tagx="title" title="title">
Molecular organization of the <i>Abi cox1</i> gene. </div> <p>Vertical lines represent exons and their nucleotide (nt) sizes are indicated below the exon E1 to E20. The name and nt size of the introns are indicated in the empty boxes. The putative functional <i>heg</i> in group I introns are shown by vertical hachured boxes, and the eroded reverse transcriptase gene carried by the <i>iAbi2</i> group II intron by a diagonal hachured box. The invading <i>heg</i> with a <i>cob</i> origin harbored by the <i>iAbi11</i> intron is shown by a grey box. </p>   </div> </div> <div class="intactversuserodedintronencodedopenreadingframes(orfs)" title="sec"> <div class="title" tagx="title" title="title">
Intact versus eroded intron encoded Open Reading Frames (ORFs)</div> <p>Only one ( <i>iAbi2)</i> of the 19 introns belongs to group II (subgroup IIA1). This large intron (2,736 nt) contains an eroded ORF (402 aa) encoding a putative protein possessing significant amino acid (aa) sequence identity (up to 38%) with the central and C-terminal regions of several reverse transcriptases (RT) encoded by plant and fungal mitochondrial group II introns. However, this encoded protein lacks the first 200 aa of the RT N-terminal region. All these RT genes are carried by group II introns possessing the same insertion site in the orthologous <i>cox1</i> genes. The highest amino acid (aa) identities (id.) and similarities (sim.) of the <i>iAbi2</i> putatively encoded RT were obtained with the phylogenetically distant Ascomycota <i>Cryphonectria parasitica</i> (38% aa id., 56% aa sim.) and the liverwort (Viridiplantae) <i>Marchantia polymorpha</i> (37% aa id., 54% aa sim.). In <i>Abi</i>, this intron might be undergoing an elimination process, as suggested by the erosion of the encoded reverse transcriptase. </p> <p>Fifteen out of the 18 group I introns of the <i>Abi cox1</i> gene harbor intact ORF encoding a putative functional HE, while only three ( <i>iAbi6</i>, <i>iAbi11</i> and <i>iAbi14)</i> contain eroded <i>heg</i> leading to non-functional HE ( <a href="#pone-0014048-g001">Figure 1</a>). The eroded <i>heg</i> are characterized by the presence of several length and point mutations leading to frameshifts and to the appearance of stop codons interrupting the ORF. </p> </div> <div class="evidenceforrecentinvasionofana.bisporuscox1intronbyahegarisingfromanothergene" title="sec"> <div class="title" tagx="title" title="title">
Evidence for recent invasion of an <i>A. bisporus cox1</i> intron by a <i>heg</i> arising from another gene </div> <p>The <i>heg</i> encoded by the <i>iAbi11</i> intron exhibits a peculiar organization, resulting from the insertion in a <i>cox1 heg</i> of an other invading <i>heg</i> ( <a href="#pone-0014048-g001">Figure 1</a>). This intron ORF of 1,800 nt (nt 16,249–18,478) is constituted by the insertion (at nt 16,848) in the <i>iAbi11 cox1</i> intron <i>heg</i> (recipient) of a second complete <i>heg</i> (invader). Both <i>heg</i> are in an open reading frame. </p> <p>The <i>iAbi11</i> recipient <i>heg</i> possesses a percentage of aa identity ranging from 73% with <i>heg</i> encoded by the Ascomycota <i>Podospora anserina i9</i> or <i>Gibberella zeae i5</i> to 40% for an orthologous group I intron widely distributed in plants. All these orthologous <i>heg</i> are carried by introns located at the same position in fungal and plant <i>cox1</i> genes. </p> <p>The <i>iAbi11</i> invading <i>heg</i> is orthologous to a <i>heg</i> carried by an intron located in another mitochondrial gene: the <i>cob</i> gene of several Ascomycota species. The highest percentage of aa identity is obtained with <i>G. zeae</i> (50%) and <i>P. anserina</i> (47%). </p> <p>In these two species, the invading and recipient <i>heg</i> are located in introns of the <i>cob</i> and <i>cox1</i> genes, respectively. On the contrary, in <i>A. bisporus</i>, both heg are located in the same intron of the <i>cox1</i> gene. </p> <p>As the recipient and invading <i>heg</i> do not possess sequence homology, the <i>iAbi11</i> chimeric <i>heg</i> can not be generated by a recombinational event. Hence, the invasion of the <i>iAbi11</i> intron <i>heg</i> by a <i>heg</i> arising from an other gene ( <i>cob</i>) suggests a recent transfer activity of the enzyme encoded by the mobile invading element, as previously reported in the T4 phage and in the fungus <i>Podospora curvicolla</i> <a href="#pone.0014048-Loizos1">[21]</a>, <a href="#pone.0014048-Saguez1">[22]</a>. </p> </div> <div class="landscapeofeukaryoticmitochondrialgroupiintrons:apatchydistribution" title="sec"> <div class="title" tagx="title" title="title">
Landscape of eukaryotic mitochondrial group I introns: a patchy distribution</div> <p>To understand the origin and dynamics of the <i>Abi</i> group I introns, all the complete <i>cox1</i> genes in databases were compiled ( <a href="#pone-0014048-g002">Figure 2</a>) as described in the <a href="#s3">Materials and Methods</a> section. This showed that the mitochondrial group I introns have been reported in three kingdoms: Fungi/Metazoa, Viridiplantae and Amoebozoa. In the Fungi/Metazoa, mitochondrial group I introns have been described in a single species of the Choanoflagellida order ( <i>Monosiga brevicollis</i>) <a href="#pone.0014048-Lang2">[23]</a>, in one species of the Porifera phylum (Metazoa kingdom) <a href="#pone.0014048-Wang1">[24]</a> and in numerous species of the fungal kingdom <a href="#pone.0014048-Santamaria1">[19]</a>. In contrast, all the <i>cox1</i> genes from the Eumetazoa lack group I intron, except two different intron types described in Cnidaria <a href="#pone.0014048-Fukami1">[25]</a>, <a href="#pone.0014048-Goddard2">[26]</a>, <a href="#pone.0014048-Rot1">[27]</a>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0014048.g002"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <div class="title" tagx="title" title="title">
Schematic representation of the distribution of group I introns, according to their Pcl (row) and to the species harboring them (column). </div> <p>Row "a" indicates the number of introns per species while column "b" shows the number of fungal introns per Pcl. All introns of the same Pcl are in the same row and all introns carried by a given species are in the same column. The first 35 Pcls were named by increasing alphabetic characters from the 5′-end of the <i>cox1</i> CDS. The 37 rows of the Pcls are organized according to the decreasing number of fungal representatives they contain from the most distributed Pcl K (25 members) to the 12 Pcls of "rare" introns containing either a single member or two (E and AF Pcls) to four members (in both additional Pcls CO1 and CO2) shared by closely related species. Presence of intact and eroded/or absent <i>heg</i> are shown by black and hachured boxes, respectively. Species were organized according to their taxonomic (superkingdoms and kingdoms, phylla, orders) and their phylogenetic positions, as indicated in <a href="#pone-0014048-g004">Figure 4</a> A and B. Fungi Dikarya Basidiomycota: <i>Agaricus bisporus</i> (EU314927), Aae: <i>Agrocybe aegerita</i> (AF010257), Pos: <i>Pleurotus ostreatus</i> (NC_009905), Mpe: <i>Moniliophthora perniciosa</i> (NC_005927), Tci <i>: Trametes cingulata</i> (NC_013933), Uma: <i>Ustilago maydis</i> (NC_008368), Tin: <i>Tilletia indica</i> (NC_009880), Cnevne: <i>Cryptococcus neoformans</i> var <i>neoformans</i> (AY560609). Ascomycota: Sja: <i>Schizosaccharomyces japonicum</i> (NC_004332), Soc: <i>Schizosaccharomyces octosporus</i> (NC_004312), Spo: <i>Schizosaccharomyces pombe</i> (NC_001326), Cze: <i>Candida zemplinina</i> (NC_005972), Yli: <i>Yarrowia lipolytica</i> (NC_002659), Hur: <i>Hanseniaspora urvum</i> (NC_007780), Dha: <i>Debaryomyces hansenii</i> (NC_010166), Cnee: <i>Candida neerlandica</i> (YP_002122387), Cor: <i>Candida orthopsilosis</i> (DQ026513), Cpa: <i>Candida parapsilosis</i> (NC_005253), Pca: <i>Pichia canadensis</i> (NC_001762), Kla: <i>Kluyveromyces lactis</i> (NC_006077), Kth: <i>Kluyveromyces thermotholerans</i> (NC_006626), Cgl: <i>Candida glabrata</i> (NC_004691), Sca: <i>Saccharomyces castellii</i> (NC_003920), Sce: <i>Saccharomyces cerevisiae</i> (NC_001224), Sse: <i>Saccharomyces servazii</i> (NC_004918), Ncr: <i>Neurospora crassa</i> (X14669), Pan: <i>Podospora anserina</i> (NC_001329), Hje: <i>Hypocrea jecorina</i> (NC_003388), Gze: <i>Gibberella zeae</i> (NC_009493), Cba: <i>Cordyceps bassiana</i> (ABU50156), Cbr: <i>Cordyceps brongniartii</i> (YP_002213602), Efl: <i>Epidermophyton floccosum</i> (NC_007394), Pbr: <i>Paracoccioides brasiliensis</i> (NC_007935), Pma: <i>Penicillium marneffei</i> (NC_005256), Eni: <i>Emericella nidulans</i> (X00790), Atu: <i>Aspergillus tubingensis</i> (NC_007597). Glomeromycota: Gin: <i>Glomus intraradices</i> (NC_012056). Blastocladiomycota: Ama: <i>Allomyces macrogynus</i> (NC_001715), Bem: <i>Blastocladiella emersonii</i> (YP_002274319). Chytridiomycota: Spu: <i>Spizellomyces punctatus</i> (NC_003052), Rsp: <i>Rhizophydium sp</i>. 136 (NC_003053), Msp: <i>Monoblepharella sp.</i> (NC_004624). Fungi incertae sedis: Scu: <i>Smittium culisetae</i> (NC_006837), Mve: <i>Mortierella verticillata</i> (NC_006838), Ror: <i>Rhizopus oryzae</i> (NC_006836). Ichthyosporea: Apa: <i>Amoebidium parasiticum</i> (AAN04062). Eumetazoa: Pjv: <i>Palythoa sp</i>. JVK-2006 (ABF67639). Corals: Mse: <i>Metridium senile</i> (NP_009253), Sra: <i>Siderastrea radians</i> (YP_654418), Rspc: <i>Rhodactis sp</i>. CASIZ 171755 (YP_654290), Rfl: <i>Ricordea florida</i> (YP_654303), Pang: <i>Plakortis angulospiculatus</i> (YP_001648679). Choanoflagellida: Mbr: <i>Monosiga brevicollis</i> (NP_696984). Dictyosteliida: Dci: <i>Dictyostelium citrinum</i> (NC_007787). Viridiplantae: Pwi: <i>Prototheca wickeramii</i> (NP_042244), Cat: <i>Chlorokybus atmophyticus</i> (YP_001315139), Cglo: <i>Chaetosphaeridium globosum</i> (NP_689386), Cvu: <i>Chara vulgaris</i> (NP_943703), Mpo: <i>Marchantia polymorpha</i> (NC_001660), Osat: <i>Oryza sativa</i> (BAD38494), Bvu: <i>Beta vulgaris</i> (NP_064063), Oel: <i>Ochrosia elliptica</i> (ABY83864), Patr: <i>Plantago atrata</i> (ABY83853). The following fungal species have intronless <i>cox1</i> genes: <i>Schizophyllum commune</i> (NC_003049), <i>Cryptococcus neoformans</i> var <i>grubii</i> (NC_004336), <i>Ashbya gossypii</i> (NC_005789), <i>Beauveria bassiana</i> (YP_001876504), <i>Mycosphaerella graminicola</i> (NC_010222), <i>Hyaloraphidium curvatum</i> (NP_150103), <i>Harpochytrium sp</i>. JEL94 (NC_004760), <i>Harpochytrium sp</i>. JEL105 (NC_004623). </p>   </div> <p>240 group I introns were found in 53 fungal <i>cox1</i> genes (including three species of the Zygomycota phylum, now fungi incertae sedis). They were sorted in 35 position classes (Pcls) defined first by the same location in the <i>cox1</i> CDS sequence. This distribution, which is based on the precise insertion site of each intron, is correlated with the highest sequence identities at the nucleotide level as well as with the amino acid identities of the putative encoded HE. Hence, each Pcl is also a sequence identity class. Indeed, each Pcl was shown to be constituted by introns inserted at the same position in the <i>cox1</i> CDS and, when present, carrying an orthologous <i>he</i>g. The orthologous character was deduced first from BLASTx analysis. When using a <i>heg</i> described in a <i>cox1</i> intron as a query in a BLASTx search, the homologous sequences with the higher scores (always higher than 80) and consequently, the lowest E-values (<div class="e-20)_UNKNOWN" were="" analyzed.="" in="" the="" large="" majority="" of="" these="" sequences="" corresponded="" to="">
heg located in introns with the same location in the <i>cox1</i> CDS. If the sequence of the <i>cox1</i> gene was complete and fully annotated, these <i>heg</i> were included in the same Pcl. Moreover, the analysis of all these complete <i>cox1</i> genes allowed the highlight of some additional <i>heg</i> whose encoded protein was eroded or undescribed. The corresponding introns were added to the corresponding Pcl. In a few cases, a sequence with similar E-values but located in a mitochondrial gene different from the <i>cox1</i> gene ( <i>nad5</i> or <i>cob</i>) was present in the BLAStx results. These sequences have been considered as an invader with a <i>cox1</i> intron origin and not included in our analysis of the <i>cox1</i> introns. In all cases, we did not find any non-homologous intron <i>heg</i> located at the same precise site of the <i>cox1</i> CDS. This is in accordance with the fact that the insertion site of the intron appears to be determined by the DNA sequence precisely recognized by its encoded HE <a href="#pone.0014048-Lambowitz1">[2]</a>, <a href="#pone.0014048-Seligman1">[28]</a>. As expected, the BLASTx results were confirmed by ClustalW alignments of the HE showing that the percentages of aa identity of all the intact HE belonging to a Pcl were always higher than those obtained when aligned with HE from the other Pcls (Supplementary information file, <a href="#pone.0014048.s003">Figure S3</a>). </div></p> <p>The 35 Pcls were named in alphabetic order starting from the 5′-end of the <i>cox1</i> CDS. Since all the 35 Pcls are represented in at least one of the six longest fungal <i>cox1</i> genes, an alignment of the corresponding COXI proteins is used in <a href="#pone-0014048-g003">Figure 3</a> to represent their relative positions. The six longest <i>cox1</i> genes are found in the Basidiomycota <i>Abi</i> (18 group I introns), in the Ascomycota <i>Podospora anserina</i> ( <i>Pan</i>: 14 introns) and <i>Gibberella zeae</i> ( <i>Gze</i>: 12 introns), in the Chytridiomycota <i>Rhyzophidium sp.</i> ( <i>Rsp</i>: 14 introns), in the Blastocladiomycota <i>Allomyces macrogynus</i> ( <i>Ama</i>: 11 introns) and in the Glomeromycota <i>Glomus intraradices</i> ( <i>Gin</i>: 11 introns). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0014048.g003"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <div class="title" tagx="title" title="title">
Comparison of group I intron insertion sites in the amino acid alignment (Clustal W) of the COX1 protein encoded by six long <i>cox1</i> genes of fungi. </div> <p>Introns are designated by their Pcl name (From A to Z and from AA to AI) and by their number in each fungal gene starting from the 5′ end of the gene. Closed and open symbols (triangles) indicate introns with intact and eroded <i>heg</i>, respectively. The insertion sites of the coral introns Pcls CO1 and CO2 never reported to date in the fungal kingdom are indicated by an arrow. The symbols "+1" and "+2" indicate that the insertion of the intron occurs inside the indicated codon: between the nt 1 and nt 2 of this codon for "+1" and between the nt 2 and nt 3 for "+2". </p>   </div> <p>The number of fungal <i>cox1</i> introns assigned to each Pcl varied greatly from 25 in the Pcl K to a single one in 8 different Pcls ( <a href="#pone-0014048-g002">Figure 2</a>). Hence, these introns can be classified as being either rare or widely distributed. Rare introns are those detected either in a single species (8 Pcls) or in two closely related species for the Pcls E and AF, which both contain introns reported in two Blastocladiales. The introns of the remaining 25 Pcls show a wide and patchy distribution ranging from 2 in the Pcl W to 25 in the Pcl K, and they are present in different phyla of the fungal kingdom. At the same time, phylogenetically close species differ by the presence/absence of these introns. For instance, the two representatives of Pcl W have been described in the Glomeromycota <i>Gin</i> and in the distant Ascomycota <i>Pan</i> ( <a href="#pone-0014048-g002">Figure 2</a>). The 25 representatives of the largest Pcl K are present in one species ( <i>Rsp</i>) out of six Chytridiomycota, in one species out of two Blastocladiella, in the Glomeromycota species, in one out of three species of the basal fungal lineages, in seven out of ten Basidiomycota and in 14 out of 31 Ascomycota. <i>Abi</i> contains15 introns belonging to the 25 widely distributed Pcls, while <i>Schizophyllum commune</i> ( <i>Sco</i>) which belongs to the same Agaricales order is intronless. Thus, the observed wide and patchy distribution of the group I fungal introns appears to result from numerous losses and gains of these mobile genetic elements. </p> <p>Among the 35 fungal intron Pcls, 17 contained orthologous introns defined by the same position in the <i>cox1</i> CDS in the Viridiplantae, Amoebozoa or Metazoa kingdoms. As previously established with fungal introns (i.e. deduced from BLASTx results and confirmed by percentages of aa identities of the encoded HE that were always higher inside a Pcl than between Pcls), the <i>heg</i> of the introns from all these kingdoms belonging to the same Pcl are orthologous genes. These Pcls represent 49% (17/35) of the Pcls and up to 60% (15/25) of the widely distributed ones. Except for two group I introns specifically harbored by coral species (classes Co1 and Co2), all the <i>cox1</i> group I introns belong to Pcls previously defined by representatives from the fungal kingdom. </p> <p>In <a href="#pone-0014048-g002">Figure 2</a>, the species were ordered according to their systematic position indicated in the taxonomic and phylogenetic trees in <a href="#pone-0014048-g004">Figure 4A and B</a>. The phylogenetic relationships of the species were deduced from the six-gene phylogeny established by James et al. <a href="#pone.0014048-James1">[29]</a>. The occurrence of the orthologous introns of the most widely distributed Pcl K is shown in the trees ( <a href="#pone-0014048-g004">Figure 4</a>). This clearly shows that the patchy distribution of the species with and without orthologous introns requires a combination of multiple events of intron loss and gain. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0014048.g004"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <div class="title" tagx="title" title="title">
Taxonomic and phylogenetic position of species with and without a Pcl K intron.</div> <p>In Figure 4A and 4B, the species harboring a Pcl K orthologous intron with an intact or an eroded <i>heg</i> are in orange and green, respectively. The species lacking an orthologous intron are in black. The phylogenetic position of each fungal species was established according to the phylogenetic tree reported by James et al. <a href="#pone.0014048-James1">[29]</a> for the fungal kingdom using a six-gene phylogeny. When not present in the tree, species were located according to the nearest taxon (same genus or order) found in the tree. </p>   </div> <p>A high sequence divergence can be seen between the <i>heg</i> sequences of orthologous introns of the same Pcl. The HE encoded by the introns of Pcl K show percentages of aa identity (and similarity) ranging from 31% aa id. (54% aa sim.) between the fungal Ascomycota <i>Gze</i> and the Viridiplantae <i>Mpo</i> to 56% aa id (73% aa sim.) between the Basidiomycota <i>Abi</i> and <i>Aae</i>. These sequence divergences between orthologous introns of the same Pcl, which are distributed over up to four kingdoms but are also observed between closely related species, can be explained by (i) a high divergence of vertically inherited introns maintained in the compared species during the course of evolution or (ii) independent gains due to lateral transfer. </p> <p>Moreover, most of the Pcls of the widely distributed introns ( <a href="#pone-0014048-g002">Figure 2</a>) contain introns with a potentially functional <i>heg</i> and others with an eroded <i>heg</i>. The Pcl AG is an exception containing four representatives with only intact <i>heg</i>. Erosion of the <i>heg</i> is thought to be a preliminary step before the complete elimination of the intron <a href="#pone.0014048-Goddard1">[14]</a>. For each Pcl, the presence of intact and eroded <i>heg</i>, even in closely related species strengthens the hypothesis that numerous events of loss and gain occurred during evolution. </p> <p>Most <i>cox1</i> genes of the Viridiplantae, Amoebozoa and Metazoa kingdoms lack intron and only a small number of species harbor 1 to 6 group I introns. On the contrary, the majority (45 genes) of the 53 <i>cox1</i> sequences of the fungal kingdom possess one to 18 introns, while only eight genes do not contain any intron. </p> </div> <div class="dynamicsofintronmobility" title="sec"> <div class="title" tagx="title" title="title">
Dynamics of intron mobility</div> <p>To analyse the dynamics (loss and gain) of group I introns in eukaryotes, a curve ( <a href="#pone-0014048-g005">Figure 5</a>) representing the number of species according to the number of introns of their <i>cox1</i> gene was established. This curve does not fit with the Gaussian distribution that would be expected if the numbers of introns were to reflect random events of losses and gains. The plots can be represented by a theoretical curve showing a logarithmic decrease. Together with the high frequency of eroded <i>heg</i> carried by these introns (68 eroded <i>heg</i> among the 240 fungal introns analyzed), the curve can be interpreted as a trend of these group I introns towards loss from the <i>cox1</i> genes. However, the six long genes distributed over five different fungal divisions behave as group I intron reservoirs, with 11 to 18 introns. They contain at least one representative of each Pcl, except for the introns reported in corals. As the eight rare introns detected once were reported in five of these six long genes, the ability to bear specific or rare introns appears to be a characteristic of these reservoir species. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/10.1371/journal.pone.0014048.g005"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <div class="title" tagx="title" title="title">
Graphical representation of number of eukaryotic species (purple curve) and fungal species (blue curve) as a function of number of introns carried by their <i>cox1</i> gene. </div> <p>The red curve shows the logarithmic regression model (y = −5,0316Ln(x) +13,507; R <sup>2</sup> = 0.83) with all the eukaryotic species analyzed. </p>   </div> <p>Moreover, most of the introns described in the long <i>cox1</i> genes from the basal fungal lineages carry eroded <i>heg</i>: 10 introns out of the 11 of <i>Ama</i>, 9 out of the 11 of <i>Gin</i> and 7 out of the 14 of <i>Rsp.</i> This suggests that these introns are being eliminated from these species. In principle, the erosion of the <i>heg</i> should definitively prevent the harboring intron from spreading. On the contrary, most of the introns of the long <i>cox1</i> genes of Dikarya (16/18, 12/14 and 12/12 for <i>Abi</i>, <i>Pan</i> and <i>Gze</i>, respectively) maintain a putative functional <i>heg</i>. </p> <p>In this context, the <i>Abi cox1</i> gene has to be considered as the largest group I intron reservoir described to date. It possesses representatives of more than half (18/35 = 51%) of the Pcls, and more than 64% (16/25) of the widely distributed ones and all but two of its 18 group I introns possess intact and potentially functional <i>heg</i>. </p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>It appears that the evolution of the mitochondrial genome of eukaryotes is still moving in the direction of group I intron loss. However, in some species and particularly in fungi, complete elimination of the group I intron seems counterbalanced by the gain mechanism resulting from the lateral transfer of these mobile genetic elements. Furthermore, the long genes described in the Dikarya fungi, whose the <i>Abi cox1</i> gene is the most amazing representative to date, pose an appealing and still unresolved question about why and how some fungal species organize their mitochondrial genes in such an expensive manner. </p> </div> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="dnamanipulation,molecularcloningandsequencing" title="sec"> <div class="title" tagx="title" title="title">
DNA manipulation, molecular cloning and sequencing</div> <p>The <i>Abi cox1</i> complete gene was sequenced by conventional procedures <a href="#pone.0014048-Maniatis1">[30]</a> from cloned mitochondrial restriction fragments and from PCR products as indicated in Supplementary information files <a href="#pone.0014048.s001">Figure S1</a>. All new data sequencing (i.e the complete sequence of the <i>Abi cox1</i> gene) has been deposited in the GenBank (Accession number EU314927). </p> <p>The <i>Agaricus bisporus</i> strain Bs518 was cultured in Roux vials containing complete CYM medium supplemented with 10% (v/v) compost extract <a href="#pone.0014048-Raper1">[31]</a>. Mitochondrial DNA was purified from 5 g of vegetative mycelium according to Fukumasa-Nakaï et al. <a href="#pone.0014048-FukumasaNakai1">[32]</a>. <i>Bam</i>H1 and <i>Eco</i>RI mitochondrial DNA libraries were constructed in <i>E. coli</i> XL1-Blue <a href="#pone.0014048-Bullock1">[33]</a> using pGEM-7Zf(+) from Promega, according to <a href="#pone.0014048-Maniatis1">[30]</a>. Both libraries were screened by hybridization on colonies with probes constituted first by an <i>Abi cox1</i> cDNA and by cloned PCR products (Supplementary information files <a href="#pone.0014048.s001">Figure S1</a>). </p> <p> <i>Abi cox1</i> cDNA was obtained by RT-PCR from 1.5 µg of total RNAs extracted by the "Hot Phenol" procedure <a href="#pone.0014048-Wessels1">[34]</a> using the Access RT-PCR kit from Promega and a set of specific primers (2.5 pmol) designed from the <i>A. aegerita cox1</i> gene sequence (GenBank Accession number AF010257). </p> <p>PCR amplifications were carried out using the Go <i>Taq</i> polymerase from Promega Corp. and couples of primers (Supplementary information files <a href="#pone.0014048.s001">Figure S1</a>) deduced from the sequences and synthesized by MWG (Seraing, Germany). PCR were performed in a Programmable Thermal Cycler PTC 200 (MJ Research Inc., Watertown, Mass., USA). Each reaction contained 10 to 30 ng of fungal genomic DNA, 4 µM of both primers, 200 µM of each dNTP, 1 unit of <i>Taq</i> DNA polymerase, in a final volume of 50 µl of the appropriate buffer. Reactions were run for 40 cycles at 95°C for 1 min, then two degrees below the Tm of both oligonucleotides for 1 min, 72°C for 1 min, and one final cycle at 72°C for 5 min. The PCR products were cloned on PGEMT-easy vector from Promega Corp. (Madison, Wisc, USA). </p> <p>The different probes were recovered after agarose gel electrophoresis by using the "Wizard SV gel Kit", then labeled with 925 kBq of [α <sup>32</sup>P] dCTP (110 TBq/mmol) from Amersham Pharmacia Biotech., using the Promega Corp. "Random Primers DNA Labeling Kit". The probes had a specific radioactivity higher than 10 <sup>8</sup> cpm/µg DNA. </p> <p>Cloned PCR products and the isolated mitochondrial genomic fragments were sequenced by the primer walking methods using the Big Dye® Terminator Cycle Sequencing kit v 1.1 (Applied Biosystems). Sequencing was performed in the genotyping and sequencing facility at University Victor Segalen Bordeaux 2. </p> </div> <div class="sequenceanalyses" title="sec"> <div class="title" tagx="title" title="title">
Sequence analyses</div> <p>The overlapping <i>cox1</i> sequences were compiled and analyzed with the DNA Strider 1.2 software (Commissariat à l'énergie atomique, Gif-sur-Yvette, France). Comparisons with sequences of the GenBank and EMBL databases were performed using the BLAST search algorithm <a href="#pone.0014048-Altschul1">[35]</a>. </p> <p>Compilation of the complete <i>cox1</i> gene sequences was performed by compiling the results (orthologous sequences) given by a BLASTx analysis using first the <i>A. bisporus cox1</i> CDS as query. This analysis revealed most of the fungal <i>cox1</i> sequences available in the Genbank and also numerous <i>cox1</i> genes from plants. Second, all the intron <i>heg</i> carried by these sequences were used independently as a query for BLASTx analysis. This allowed the recovery of some additional split <i>cox1</i> sequences from plants and also from other distant species such as Palythoa sp. JVK-2006 (Porifera), and species belonging to the Ichthyosporea, Choanoflagellida and Dictyosteliida phylla, which have been added to our analysis. Finally, the coral (Cnidaria) <i>cox1</i> genes were added to the analysis because they contain the only introns reported to date in the Eumetazoa kingdom without homology with introns from other kingdoms. Alignments of nucleotide and protein sequences were carried out with Clustal W (1.8) software <a href="#pone.0014048-Thompson1">[36]</a>. </p> <p>The intron secondary structures (Supplementary information files <a href="#pone.0014048.s002">Figure S2</a>) were deduced from the determination of the conserved sequences P, Q, R and S of the intron cores and of the base-paired regions P1–P9 with the help of RNA-Weasel software, and by direct comparison with the standard scheme for group I introns <a href="#pone.0014048-Michel1">[37]</a> and with the secondary structures of the database established by Damberger and Gutell. <a href="#pone.0014048-Damberger1">[38]</a>. </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting Information</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S1</span>  <p>Restriction map of the Abi mitochondrial genome region carrying the cox1 gene. The exon sequences are represented by coloured boxes. The cloned BamH1 and EcoRI restriction fragments used to determine the Abi cox1 gene sequence are indicated and named according to the restriction map. The sequenced PCR products and the location of the corresponding primers are indicated (CoxU: <span class="named-content" title="named-content">ATGAATTGGTTAAATTCTAC</span>; 5'R: <span class="named-content" title="named-content">TTAAAAATGTAAACTCCTG</span>, CoxR: <span class="named-content" title="named-content">TTATTAAGATTGCGCAGGT</span>, B17U: <span class="named-content" title="named-content">TTAGTAGGATCCTCAGAG</span>, E21R: <span class="named-content" title="named-content">AATATAAAGTACCTAAGGC</span>; B13U: <span class="named-content" title="named-content">CCTTAAGATTGTAGAGTAG</span>; B6R: <span class="named-content" title="named-content">ACAACAGATTATTTCTGGC</span>; B6U: <span class="named-content" title="named-content">AATAAACTAACCCTACCAG</span>; B15R: <span class="named-content" title="named-content">CTCTTAATATGATAAAGGTG</span>; E19U: <span class="named-content" title="named-content">ATAAGATACTTAAGTCCCC</span>; E14R: <span class="named-content" title="named-content">ATAAACTTAGCTACAGCC</span>). </p> <p>(0.18 MB PPT)</p>  <div class="media" title="media"> <a href="pone.0014048.s001.ppt">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S2</span>  <p>Secondary structures of the A. bisporus introns iAbi1 to iAbi19. Exonic sequences and intronic sequences are in lower-case and upper-case letters, respectively. The base-paired regions P1 to P9 are shown on the secondary structures, according to the standard scheme for group I introns described by Michel and Westhof <a href="#pone.0014048-Michel1">[37]</a>. For the group II intron iAbi2, the structural domains are indicated by roman numerals. </p> <p>(0.74 MB PPT)</p>  <div class="media" title="media"> <a href="pone.0014048.s002.ppt">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Figure S3</span>  <p>Alignement (A) and representation of the Total Score (B) obtained when comparing the 21 intact HE of the most widely distributed Pcl K. The Total Scores higher than 200 are in red, those between 80 and 200 in pink. In C are reported the percentages of aa identity (a) and aa similarity (b) between I-AbiVII-P and the 20 others orthologous HE. </p> <p>(0.07 MB DOC)</p>  <div class="media" title="media"> <a href="pone.0014048.s003.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>The authors are grateful to James B. Anderson and Jianping Xu for providing the cloned mitochondrial fragments used during preliminary studies. </p> <p>DNA Sequencing was performed in the Genotyping and Sequencing facility at University Victor Segalen Bordeaux 2. </p> <p>They also thank Dr. Ray Cooke for editing the English.</p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-COI-statement" title="COI-statement"> <p> <b>Competing Interests: </b>The authors have declared that no competing interests exist. </p> </div> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p> <b>Funding: </b>The authors have no support or funding to report. </p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="pone.0014048-Haugen1" /> <span class="label" tagx="label" title="label">1</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haugen</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simon</span> <span class="given-names" tagx="given-names" title="given-names">DM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhattacharya</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">The natural history of group I introns.</span> <span class="source" tagx="source" title="source">Trends Genet</span> <span class="volume" tagx="volume" title="volume">21</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">111</span> <span class="lpage" tagx="lpage" title="lpage">119</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15661357">15661357</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Lambowitz1" /> <span class="label" tagx="label" title="label">2</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lambowitz</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Belfort</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="year" tagx="year" title="year">1993</span> <span class="mixed-article-title" title="mixed-article-title">Introns as mobile genetic elements.</span> <span class="source" tagx="source" title="source">Annu Rev Biochem</span> <span class="volume" tagx="volume" title="volume">62</span> <span class="fpage" tagx="fpage" title="fpage">587</span> <span class="lpage" tagx="lpage" title="lpage">622</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8352597">8352597</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Nesbo1" /> <span class="label" tagx="label" title="label">3</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nesbo</span> <span class="given-names" tagx="given-names" title="given-names">CL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doolittle</span> <span class="given-names" tagx="given-names" title="given-names">WF</span> </span> </span> <span class="year" tagx="year" title="year">2003</span> <span class="mixed-article-title" title="mixed-article-title">Active self-splicing group I introns in 23S rRNA genes of hyperthermophilic bacteria, derived from introns in eukaryotic organelles. </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="volume" tagx="volume" title="volume">100</span> <span class="issue" tagx="issue" title="issue">19</span> <span class="fpage" tagx="fpage" title="fpage">10806</span> <span class="lpage" tagx="lpage" title="lpage">10811</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12947037">12947037</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Sandegren1" /> <span class="label" tagx="label" title="label">4</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sandegren</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sjoberg</span> <span class="given-names" tagx="given-names" title="given-names">BM</span> </span> </span> <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title">Distribution, sequence homology, and homing of group I introns among T-even-like bacteriophages: evidence for recent transfer of old introns. </span> <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="volume" tagx="volume" title="volume">279</span> <span class="issue" tagx="issue" title="issue">21</span> <span class="fpage" tagx="fpage" title="fpage">22218</span> <span class="lpage" tagx="lpage" title="lpage">22227</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15026408">15026408</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Lang1" /> <span class="label" tagx="label" title="label">5</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lang</span> <span class="given-names" tagx="given-names" title="given-names">BF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laforest</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burger</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Mitochondrial introns: a critical view.</span> <span class="source" tagx="source" title="source">Trends Genet</span> <span class="volume" tagx="volume" title="volume">23</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">119</span> <span class="lpage" tagx="lpage" title="lpage">125</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17280737">17280737</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Saldanha1" /> <span class="label" tagx="label" title="label">6</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saldanha</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mohr</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Belfort</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lambowitz</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> </span> <span class="year" tagx="year" title="year">1993</span> <span class="mixed-article-title" title="mixed-article-title">Group I and group II introns.</span> <span class="source" tagx="source" title="source">FASEB J</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">15</span> <span class="lpage" tagx="lpage" title="lpage">24</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8422962">8422962</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Cummings1" /> <span class="label" tagx="label" title="label">7</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cummings</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Michel</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McNally</span> <span class="given-names" tagx="given-names" title="given-names">KL</span> </span> </span> <span class="year" tagx="year" title="year">1989</span> <span class="mixed-article-title" title="mixed-article-title">DNA sequence analysis of the 24.5 kilobase pair cytochrome oxidase subunit I mitochondrial gene from <i>Podospora anserina</i>: a gene with sixteen introns. </span> <span class="source" tagx="source" title="source">Curr Genet</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">381</span> <span class="lpage" tagx="lpage" title="lpage">406</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2558809">2558809</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Oda1" /> <span class="label" tagx="label" title="label">8</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oda</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamato</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohta</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakamura</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takemura</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">1992</span> <span class="mixed-article-title" title="mixed-article-title">Gene organization deduced from the complete sequence of liverwort <i>Marchantia polymorpha</i> mitochondrial DNA. A primitive form of plant mitochondrial genome. </span> <span class="source" tagx="source" title="source">J Mol Biol</span> <span class="volume" tagx="volume" title="volume">223</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">7</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1731062">1731062</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Lambowitz2" /> <span class="label" tagx="label" title="label">9</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lambowitz</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perlman</span> <span class="given-names" tagx="given-names" title="given-names">PS</span> </span> </span> <span class="year" tagx="year" title="year">1990</span> <span class="mixed-article-title" title="mixed-article-title">Involvement of aminoacyl-tRNA synthetases and other proteins in group I and group II intron splicing. </span> <span class="source" tagx="source" title="source">Trends Biochem Sci</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="issue" tagx="issue" title="issue">11</span> <span class="fpage" tagx="fpage" title="fpage">440</span> <span class="lpage" tagx="lpage" title="lpage">444</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2278103">2278103</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Lazowska1" /> <span class="label" tagx="label" title="label">10</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lazowska</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacq</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Slonimski</span> <span class="given-names" tagx="given-names" title="given-names">PP</span> </span> </span> <span class="year" tagx="year" title="year">1980</span> <span class="mixed-article-title" title="mixed-article-title">Sequence of introns and flanking exons in wild-type and box3 mutants of cytochrome b reveals an interlaced splicing protein coded by an intron. </span> <span class="source" tagx="source" title="source">Cell</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="issue" tagx="issue" title="issue">2 Pt 2</span> <span class="fpage" tagx="fpage" title="fpage">333</span> <span class="lpage" tagx="lpage" title="lpage">348</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7004642">7004642</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Pellenz1" /> <span class="label" tagx="label" title="label">11</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pellenz</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harington</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dujon</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolf</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schafer</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> </span> <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">Characterization of the I-Spom I endonuclease from fission yeast: insights into the evolution of a group I intron-encoded homing endonuclease. </span> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="volume" tagx="volume" title="volume">55</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">302</span> <span class="lpage" tagx="lpage" title="lpage">313</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12187383">12187383</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Gonzalez1" /> <span class="label" tagx="label" title="label">12</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gonzalez</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barroso</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Labarere</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="year" tagx="year" title="year">1998</span> <span class="mixed-article-title" title="mixed-article-title">Molecular analysis of the split <i>cox1</i> gene from the Basidiomycota <i>Agrocybe aegerita</i>: relationship of its introns with homologous Ascomycota introns and divergence levels from common ancestral copies. </span> <span class="source" tagx="source" title="source">Gene</span> <span class="volume" tagx="volume" title="volume">220</span> <span class="issue" tagx="issue" title="issue">1–2</span> <span class="fpage" tagx="fpage" title="fpage">45</span> <span class="lpage" tagx="lpage" title="lpage">53</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9767103">9767103</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Vaughn1" /> <span class="label" tagx="label" title="label">13</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vaughn</span> <span class="given-names" tagx="given-names" title="given-names">JC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mason</span> <span class="given-names" tagx="given-names" title="given-names">MT</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sper-Whitis</span> <span class="given-names" tagx="given-names" title="given-names">GL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuhlman</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <span class="year" tagx="year" title="year">1995</span> <span class="mixed-article-title" title="mixed-article-title">Fungal origin by horizontal transfer of a plant mitochondrial group I intron in the chimeric CoxI gene of <i>Peperomia</i>. </span> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="volume" tagx="volume" title="volume">41</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">563</span> <span class="lpage" tagx="lpage" title="lpage">572</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7490770">7490770</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Goddard1" /> <span class="label" tagx="label" title="label">14</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goddard</span> <span class="given-names" tagx="given-names" title="given-names">MR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burt</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="year" tagx="year" title="year">1999</span> <span class="mixed-article-title" title="mixed-article-title">Recurrent invasion and extinction of a selfish gene.</span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="volume" tagx="volume" title="volume">96</span> <span class="issue" tagx="issue" title="issue">24</span> <span class="fpage" tagx="fpage" title="fpage">13880</span> <span class="lpage" tagx="lpage" title="lpage">13885</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10570167">10570167</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Gonzalez2" /> <span class="label" tagx="label" title="label">15</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gonzalez</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barroso</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Labarere</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="year" tagx="year" title="year">1999</span> <span class="mixed-article-title" title="mixed-article-title">Molecular gene organisation and secondary structure of the mitochondrial large subunit ribosomal RNA from the cultivated Basidiomycota <i>Agrocybe aegerita</i>: a 13 kb gene possessing six unusual nucleotide extensions and eight introns. </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="issue" tagx="issue" title="issue">7</span> <span class="fpage" tagx="fpage" title="fpage">1754</span> <span class="lpage" tagx="lpage" title="lpage">1761</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10076008">10076008</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Cusimano1" /> <span class="label" tagx="label" title="label">16</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cusimano</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">LB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Renner</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Reevaluation of the cox1 group I intron in Araceae and angiosperms indicates a history dominated by loss rather than horizontal transfer. </span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">265</span> <span class="lpage" tagx="lpage" title="lpage">276</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18158323">18158323</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-SanchezPuerta1" /> <span class="label" tagx="label" title="label">17</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sanchez-Puerta</span> <span class="given-names" tagx="given-names" title="given-names">MV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cho</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mower</span> <span class="given-names" tagx="given-names" title="given-names">JP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alverson</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmer</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Frequent, phylogenetically local horizontal transfer of the cox1 group I Intron in flowering plant mitochondria. </span> <span class="source" tagx="source" title="source">Mol Biol Evol</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="issue" tagx="issue" title="issue">8</span> <span class="fpage" tagx="fpage" title="fpage">1762</span> <span class="lpage" tagx="lpage" title="lpage">1777</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18524785">18524785</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Hebert1" /> <span class="label" tagx="label" title="label">18</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hebert</span> <span class="given-names" tagx="given-names" title="given-names">PD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ratnasingham</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">deWaard</span> <span class="given-names" tagx="given-names" title="given-names">JR</span> </span> </span> <span class="year" tagx="year" title="year">2003</span> <span class="mixed-article-title" title="mixed-article-title">Barcoding animal life: cytochrome c oxidase subunit 1 divergences among closely related species. </span> <span class="source" tagx="source" title="source">Proc Biol Sci</span> <span class="volume" tagx="volume" title="volume">270</span> <span class="issue" tagx="issue" title="issue">Suppl 1</span> <span class="fpage" tagx="fpage" title="fpage">S96</span> <span class="lpage" tagx="lpage" title="lpage">9</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12952648">12952648</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Santamaria1" /> <span class="label" tagx="label" title="label">19</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Santamaria</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vicario</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pappadà</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scioscia</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scazzocchio</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Towards barcode markers in Fungi: an intron map of Ascomycota mitochondria.</span> <span class="source" tagx="source" title="source">BMC Bioinformatics</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="issue" tagx="issue" title="issue">Suppl 6</span> <span class="fpage" tagx="fpage" title="fpage">S15</span> </span> </li> <li tag="ref"> <a name="pone.0014048-Haridas1" /> <span class="label" tagx="label" title="label">20</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haridas</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gantt</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span> </span> <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">The mitochondrial genome of the wood-degrading basidiomycete <i>Trametes cingulata</i>. </span> <span class="source" tagx="source" title="source">FEMS Microbiol Lett</span> <span class="volume" tagx="volume" title="volume">308</span> <span class="fpage" tagx="fpage" title="fpage">29</span> <span class="lpage" tagx="lpage" title="lpage">34</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20455947">20455947</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Loizos1" /> <span class="label" tagx="label" title="label">21</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Loizos</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tillier</span> <span class="given-names" tagx="given-names" title="given-names">ER</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Belfort</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="year" tagx="year" title="year">1994</span> <span class="mixed-article-title" title="mixed-article-title">Evolution of mobile group I introns: recognition of intron sequences by an intron-encoded endonuclease. </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span> <span class="volume" tagx="volume" title="volume">91</span> <span class="issue" tagx="issue" title="issue">25</span> <span class="fpage" tagx="fpage" title="fpage">11983</span> <span class="lpage" tagx="lpage" title="lpage">11987</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7991569">7991569</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Saguez1" /> <span class="label" tagx="label" title="label">22</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saguez</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lecellier</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koll</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> </span> <span class="year" tagx="year" title="year">2000</span> <span class="mixed-article-title" title="mixed-article-title">Intronic GIY-YIG endonuclease gene in the mitochondrial genome of <i>Podospora curvicolla</i>: evidence for mobility. </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">1299</span> <span class="lpage" tagx="lpage" title="lpage">1306</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10684923">10684923</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Lang2" /> <span class="label" tagx="label" title="label">23</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lang</span> <span class="given-names" tagx="given-names" title="given-names">BF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O'Kelly</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nerad</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gray</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burger</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> </span> <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">The closest unicellular relatives of animals.</span> <span class="source" tagx="source" title="source">Curr Biol</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">1773</span> <span class="lpage" tagx="lpage" title="lpage">1778</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12401173">12401173</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Wang1" /> <span class="label" tagx="label" title="label">24</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">X</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lavrov</span> <span class="given-names" tagx="given-names" title="given-names">DV</span> </span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Seventeen new complete mtDNA sequences reveal extensive mitochondrial genome evolution within the Demospongiae. </span> <span class="source" tagx="source" title="source">PLoS ONE</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">e2723</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18628961">18628961</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Fukami1" /> <span class="label" tagx="label" title="label">25</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fukami</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chiou</span> <span class="given-names" tagx="given-names" title="given-names">CY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knowlton</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> </span> <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Novel group I introns encoding a putative homing endonuclease in the mitochondrial <i>cox1</i> gene of <i>Scleractinian</i> corals. </span> <span class="source" tagx="source" title="source">J Mol Evol</span> <span class="volume" tagx="volume" title="volume">64</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">591</span> <span class="lpage" tagx="lpage" title="lpage">600</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17437148">17437148</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Goddard2" /> <span class="label" tagx="label" title="label">26</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goddard</span> <span class="given-names" tagx="given-names" title="given-names">MR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leigh</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roger</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pemberton</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> </span> <span class="year" tagx="year" title="year">2006</span> <span class="mixed-article-title" title="mixed-article-title">Invasion and persistence of a selfish gene in the Cnidaria.</span> <span class="source" tagx="source" title="source">PLoS ONE</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="volume" tagx="volume" title="volume">e3</span> </span> </li> <li tag="ref"> <a name="pone.0014048-Rot1" /> <span class="label" tagx="label" title="label">27</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rot</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goldfarb</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ilan</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huchon</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="year" tagx="year" title="year">2006</span> <span class="mixed-article-title" title="mixed-article-title">Putative cross-kingdom horizontal gene transfer in sponge (Porifera) mitochondria.</span> <span class="source" tagx="source" title="source">BMC Evol Biol</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">71</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16972986">16972986</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Seligman1" /> <span class="label" tagx="label" title="label">28</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seligman</span> <span class="given-names" tagx="given-names" title="given-names">LM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stephens</span> <span class="given-names" tagx="given-names" title="given-names">KM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Savage</span> <span class="given-names" tagx="given-names" title="given-names">JH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Monnat</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> <span class="suffix" tagx="suffix" title="suffix">Jr</span> </span> </span> <span class="year" tagx="year" title="year">1997</span> <span class="mixed-article-title" title="mixed-article-title">Genetic analysis of the <i>Chlamydomonas reinhardtii</i> I-CreI mobile intron homing system in Escherichia coli. </span> <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">147</span> <span class="fpage" tagx="fpage" title="fpage">1653</span> <span class="lpage" tagx="lpage" title="lpage">1664</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9409828">9409828</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-James1" /> <span class="label" tagx="label" title="label">29</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">James</span> <span class="given-names" tagx="given-names" title="given-names">TY</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kauff</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schoch</span> <span class="given-names" tagx="given-names" title="given-names">CL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matheny</span> <span class="given-names" tagx="given-names" title="given-names">PB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hofstetter</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="year" tagx="year" title="year">2006</span> <span class="mixed-article-title" title="mixed-article-title">Reconstructing the early evolution of Fungi using a six-gene phylogeny.</span> <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">443</span> <span class="issue" tagx="issue" title="issue">7113</span> <span class="fpage" tagx="fpage" title="fpage">818</span> <span class="lpage" tagx="lpage" title="lpage">822</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17051209">17051209</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Maniatis1" /> <span class="label" tagx="label" title="label">30</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maniatis</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frisch</span> <span class="given-names" tagx="given-names" title="given-names">EF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoffman</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="year" tagx="year" title="year">1982</span> <span class="mixed-article-title" title="mixed-article-title">Molecular cloning: a laboratory manual.</span> <span class="source" tagx="source" title="source">Cold Spring Harbor Laboratory Press</span> </span> </li> <li tag="ref"> <a name="pone.0014048-Raper1" /> <span class="label" tagx="label" title="label">31</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raper</span> <span class="given-names" tagx="given-names" title="given-names">JR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoffman</span> <span class="given-names" tagx="given-names" title="given-names">RM</span> </span> </span> <span class="year" tagx="year" title="year">1974</span> <span class="mixed-article-title" title="mixed-article-title"> <i>Schizophyllum commune</i>. </span> <span class="source" tagx="source" title="source">Handbook of genetics</span> <span class="volume" tagx="volume" title="volume">1</span> <span class="fpage" tagx="fpage" title="fpage">597</span> <span class="lpage" tagx="lpage" title="lpage">626</span> </span> </li> <li tag="ref"> <a name="pone.0014048-FukumasaNakai1" /> <span class="label" tagx="label" title="label">32</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fukumasa-Nakai</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matsumoto</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fukuda</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="year" tagx="year" title="year">1992</span> <span class="mixed-article-title" title="mixed-article-title">Efficient isolation of mitochondrial DNA from higher Basidiomycetes for restriction endonuclease analysis. </span> <span class="source" tagx="source" title="source">Rept Tottori Mycol Inst</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">60</span> <span class="lpage" tagx="lpage" title="lpage">68</span> </span> </li> <li tag="ref"> <a name="pone.0014048-Bullock1" /> <span class="label" tagx="label" title="label">33</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bullock</span> <span class="given-names" tagx="given-names" title="given-names">WO</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fernandez</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stuart</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> </span> <span class="year" tagx="year" title="year">1987</span> <span class="mixed-article-title" title="mixed-article-title">XL1-Blue: a high efficiency plasmid transforming recA <i>Escherichia coli</i> strain with beta-galactosidase selection. </span> <span class="source" tagx="source" title="source">Bio.Techniques</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">376</span> <span class="lpage" tagx="lpage" title="lpage">337</span> </span> </li> <li tag="ref"> <a name="pone.0014048-Wessels1" /> <span class="label" tagx="label" title="label">34</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wessels</span> <span class="given-names" tagx="given-names" title="given-names">JGH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mulder</span> <span class="given-names" tagx="given-names" title="given-names">GH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Springer</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="year" tagx="year" title="year">1987</span> <span class="mixed-article-title" title="mixed-article-title">Expression of dikaryon-specific and non-specific mRNAs of <i>Schizophyllum commune</i> in relation to environmental conditions and fruiting. </span> <span class="source" tagx="source" title="source">J Gen Microbiol</span> <span class="volume" tagx="volume" title="volume">133</span> <span class="fpage" tagx="fpage" title="fpage">2557</span> <span class="lpage" tagx="lpage" title="lpage">2561</span> </span> </li> <li tag="ref"> <a name="pone.0014048-Altschul1" /> <span class="label" tagx="label" title="label">35</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Altschul</span> <span class="given-names" tagx="given-names" title="given-names">SF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gish</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miller</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Myers</span> <span class="given-names" tagx="given-names" title="given-names">EW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lipman</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">DJ</span> </span> </span> <span class="year" tagx="year" title="year">1990</span> <span class="mixed-article-title" title="mixed-article-title">Basic local alignment search tool.</span> <span class="source" tagx="source" title="source">J Mol Biol</span> <span class="volume" tagx="volume" title="volume">215</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">403</span> <span class="lpage" tagx="lpage" title="lpage">410</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712">2231712</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Thompson1" /> <span class="label" tagx="label" title="label">36</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Higgins</span> <span class="given-names" tagx="given-names" title="given-names">DG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gibson</span> <span class="given-names" tagx="given-names" title="given-names">TJ</span> </span> </span> <span class="year" tagx="year" title="year">1994</span> <span class="mixed-article-title" title="mixed-article-title">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. </span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="issue" tagx="issue" title="issue">2222</span> <span class="fpage" tagx="fpage" title="fpage">4673</span> <span class="lpage" tagx="lpage" title="lpage">4680</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7984417">7984417</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Michel1" /> <span class="label" tagx="label" title="label">37</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Michel</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Westhof</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> </span> <span class="year" tagx="year" title="year">1990</span> <span class="mixed-article-title" title="mixed-article-title">Modelling of the three-dimensional architecture of group I catalytic introns based on comparative sequence analysis. </span> <span class="source" tagx="source" title="source">J Mol Biol</span> <span class="volume" tagx="volume" title="volume">216</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">585</span> <span class="lpage" tagx="lpage" title="lpage">610</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2258934">2258934</a> </span> </span> </li> <li tag="ref"> <a name="pone.0014048-Damberger1" /> <span class="label" tagx="label" title="label">38</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Damberger</span> <span class="given-names" tagx="given-names" title="given-names">SH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gutell</span> <span class="given-names" tagx="given-names" title="given-names">RR</span> </span> </span> <span class="year" tagx="year" title="year">1994</span> <span class="mixed-article-title" title="mixed-article-title">A comparative database of group I intron structures.</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="issue" tagx="issue" title="issue">17</span> <span class="fpage" tagx="fpage" title="fpage">3508</span> <span class="lpage" tagx="lpage" title="lpage">3510</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7937050">7937050</a> </span> </span> </li> </ul> </div> </div>  </body></html>